<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Genetic variation, population structure and linkage disequilibrium in Switchgrass with ISSR, SCoT and EST-SSR markers | Hereditas | Full Text</title>
<meta name="citation_abstract" content="To evaluate genetic variation, population structure, and the extent of linkage disequilibrium (LD), 134 switchgrass (Panicum virgatum L.) samples were analyzed with 51 markers, including 16 ISSRs, 20 SCoTs, and 15 EST-SSRs. In this study, a high level of genetic variation was observed in the switchgrass samples and they had an average Neiâs gene diversity index (H) of 0.311. A total of 793 bands were obtained, of which 708 (89.28Â %) were polymorphic. Using a parameter marker index (MI), the efficiency of the three types of markers (ISSR, SCoT, and EST-SSR) in the study were compared and we found that SCoT had a higher marker efficiency than the other two markers. The 134 switchgrass samples could be divided into two sub-populations based on STRUCTURE, UPGMA clustering, and principal coordinate analyses (PCA), and upland and lowland ecotypes could be separated by UPGMA clustering and PCA analyses. Linkage disequilibrium analysis revealed an average r2 of 0.035 across all 51 markers, indicating a trend of higher LD in sub-population 2 than that in sub-population 1 (Pâ&lt;â0.01). The population structure revealed in this study will guide the design of future association studies using these switchgrass samples.">
<meta name="description" content="To evaluate genetic variation, population structure, and the extent of linkage disequilibrium (LD), 134 switchgrass (Panicum virgatum L.) samples were analyzed with 51 markers, including 16 ISSRs, 20 SCoTs, and 15 EST-SSRs. In this study, a high level of genetic variation was observed in the switchgrass samples and they had an average Neiâs gene diversity index (H) of 0.311. A total of 793 bands were obtained, of which 708 (89.28Â %) were polymorphic. Using a parameter marker index (MI), the efficiency of the three types of markers (ISSR, SCoT, and EST-SSR) in the study were compared and we found that SCoT had a higher marker efficiency than the other two markers. The 134 switchgrass samples could be divided into two sub-populations based on STRUCTURE, UPGMA clustering, and principal coordinate analyses (PCA), and upland and lowland ecotypes could be separated by UPGMA clustering and PCA analyses. Linkage disequilibrium analysis revealed an average r2 of 0.035 across all 51 markers, indicating a trend of higher LD in sub-population 2 than that in sub-population 1 (Pâ&lt;â0.01). The population structure revealed in this study will guide the design of future association studies using these switchgrass samples.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Genetic variation, population structure and linkage disequilibrium in Switchgrass with ISSR, SCoT and EST-SSR markers">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-04-19">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 Zhang et al.">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="To evaluate genetic variation, population structure, and the extent of linkage disequilibrium (LD), 134 switchgrass (Panicum virgatum L.) samples were analyzed with 51 markers, including 16 ISSRs, 20 SCoTs, and 15 EST-SSRs. In this study, a high level of genetic variation was observed in the switchgrass samples and they had an average Nei’s gene diversity index (H) of 0.311. A total of 793 bands were obtained, of which 708 (89.28 %) were polymorphic. Using a parameter marker index (MI), the efficiency of the three types of markers (ISSR, SCoT, and EST-SSR) in the study were compared and we found that SCoT had a higher marker efficiency than the other two markers. The 134 switchgrass samples could be divided into two sub-populations based on STRUCTURE, UPGMA clustering, and principal coordinate analyses (PCA), and upland and lowland ecotypes could be separated by UPGMA clustering and PCA analyses. Linkage disequilibrium analysis revealed an average r2 of 0.035 across all 51 markers, indicating a trend of higher LD in sub-population 2 than that in sub-population 1 (P &lt; 0.01). The population structure revealed in this study will guide the design of future association studies using these switchgrass samples.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-04-19">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="4">
<meta name="prism.copyright" content="2016 Zhang et al.">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0007-z">
<meta name="prism.doi" content="doi:10.1186/s41065-016-0007-z">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0007-z">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0007-z">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Genetic variation, population structure and linkage disequilibrium in Switchgrass with ISSR, SCoT and EST-SSR markers">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/04/19">
<meta name="citation_firstpage" content="4">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-016-0007-z">
<meta name="DOI" content="10.1186/s41065-016-0007-z">
<meta name="citation_doi" content="10.1186/s41065-016-0007-z">
<meta name="dc.creator" content="Yu Zhang">
<meta name="dc.creator" content="Haidong Yan">
<meta name="dc.creator" content="Xiaomei Jiang">
<meta name="dc.creator" content="Xiaoli Wang">
<meta name="dc.creator" content="Linkai Huang">
<meta name="dc.creator" content="Bin Xu">
<meta name="dc.creator" content="Xinquan Zhang">
<meta name="dc.creator" content="Lexin Zhang">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Plant Breed; citation_title=Genetic linkage map construction for white jute (Corchorus capsularis L.) using SRAP, ISSR and RAPD markers; citation_author=YP Chen, LW Zhang, JM Qi, H Chen, AF Tao, JT Xu; citation_volume=133; citation_issue=6; citation_publication_date=2014; citation_pages=777-81; citation_doi=10.1111/pbr.12205; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Assessment of genetic diversity in Azadirachta indica using AFLP markers; citation_author=A Singh, MS Negi, J Rajagopal, S Bhatia, UK Tomar, PS Srivastava; citation_volume=99; citation_issue=1-2; citation_publication_date=1999; citation_pages=272-9; citation_doi=10.1007/s001220051232; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=Biol Plant; citation_title=Assessment of genetic fidelity of micropropagated Swertia chirayita plantlets by ISSR marker assay; citation_author=P Joshi, V Dhawan; citation_volume=51; citation_issue=1; citation_publication_date=2007; citation_pages=22-6; citation_doi=10.1007/s10535-007-0005-0; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Inter simple sequence repeat (ISSR) polymorphism and its application in plant breeding; citation_author=MP Reddy, N Sarla, EA Siddiq; citation_volume=128; citation_issue=1; citation_publication_date=2002; citation_pages=9-17; citation_doi=10.1023/A:1020691618797; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=Plant Mol Biol Rep; citation_title=Start codon targeted (SCoT) polymorphism: a simple, novel DNA marker technique for generating gene-targeted markers in plants; citation_author=BCY Collard, DJ Mackill; citation_volume=27; citation_issue=1; citation_publication_date=2009; citation_pages=86-93; citation_doi=10.1007/s11105-008-0060-5; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=Mol Plant Breed; citation_title=SCoT: a novel gene targeted marker technique based on the translation start codon; citation_author=FQ Xiong, RH Tang, ZL Chen, LH Pan, WJ Zhuang; citation_volume=7; citation_publication_date=2009; citation_pages=635-8; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Analysis of diversity and relationships among mango cultivars using Start Codon Targeted (SCoT) markers; citation_author=C Luo, XH He, H Chen, SJ Ou, MP Gao; citation_volume=38; citation_issue=6; citation_publication_date=2010; citation_pages=1176-84; citation_doi=10.1016/j.bse.2010.11.004; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=Mol Genet Genomics; citation_title=Transferable EST-SSR markers for the study of polymorphism and genetic diversity in bread wheat; citation_author=PK Gupta, S Rustgi, S Sharma, R Singh, N Kumar, HS Balyan; citation_volume=270; citation_issue=4; citation_publication_date=2003; citation_pages=315-23; citation_doi=10.1007/s00438-003-0921-4; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Interspecific transferability and comparative mapping of barley EST-SSR markers in wheat, rye and rice; citation_author=RK Varshney, R Sigmund, A Börner, V Korzun, N Stein, ME Sorrells; citation_volume=168; citation_issue=1; citation_publication_date=2005; citation_pages=195-202; citation_doi=10.1016/j.plantsci.2004.08.001; citation_id=CR9">
<meta name="citation_reference" content="Qureshi SN, Saha S, Kantety RV, Jenkins JN, Saha S, editors. Molecular biology and physiology: EST-SSR: a new class of genetic markers in cotton. J Cotton Sci. 2004;8:112–23.">
<meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Comparative assessment of EST-SSR, EST-SNP and AFLP markers for evaluation of genetic diversity and conservation of genetic resources using wild, cultivated and elite barleys; citation_author=RK Varshney, K Chabane, PS Hendre, RK Aggarwal, A Graner; citation_volume=173; citation_issue=6; citation_publication_date=2007; citation_pages=638-49; citation_doi=10.1016/j.plantsci.2007.08.010; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Evol; citation_title=Genetic relationship and diversity of Mangifera indica L.: revealed through SCoT analysis; citation_author=C Luo, XH He, H Chen, Y Hu, SJ Ou; citation_volume=59; citation_issue=7; citation_publication_date=2012; citation_pages=1505-15; citation_doi=10.1007/s10722-011-9779-1; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=Development of EST-SSR markers for the study of population structure in lettuce (Lactuca sativa L.); citation_author=I Simko; citation_volume=100; citation_issue=2; citation_publication_date=2009; citation_pages=256-62; citation_doi=10.1093/jhered/esn072; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Am J Bot; citation_title=Population structure and genetic diversity of Botrychium pumicola (Ophioglossaceae) based on inter-simple sequence repeats (ISSR); citation_author=FJ Camacho, A Liston; citation_volume=88; citation_issue=6; citation_publication_date=2001; citation_pages=1065-70; citation_doi=10.2307/2657089; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Population structure and linkage disequilibrium unravelled in tetraploid potato; citation_author=B Björn, MJ Paulo, K Kowitwanich, M Sengers, RGF Visser, HJ Eck; citation_volume=121; citation_issue=6; citation_publication_date=2010; citation_pages=1151-70; citation_doi=10.1007/s00122-010-1379-5; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=A new citrus linkage map based on SRAP, SSR, ISSR, POGP, RGA and RAPD markers; citation_author=O Gulsen, A Uzun, I Canan, U Seday, E Canihos; citation_volume=173; citation_issue=2; citation_publication_date=2010; citation_pages=265-77; citation_doi=10.1007/s10681-010-0146-7; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Annu Rev Plant Biol; citation_title=Structure of linkage disequilibrium in plants; citation_author=SA Flint-Garcia, JM Thornsberry, B IV; citation_volume=54; citation_issue=1; citation_publication_date=2003; citation_pages=357-74; citation_doi=10.1146/annurev.arplant.54.031902.134907; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Dwarf8 polymorphisms associate with variation in flowering time; citation_author=JM Thornsberry, MM Goodman, J Doebley, S Kresovich, D Nielsen, ES Buckler; citation_volume=28; citation_issue=3; citation_publication_date=2001; citation_pages=286-9; citation_doi=10.1038/90135; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=Genet; citation_title=The extent of linkage disequilibrium in rice (Oryza sativa L.); citation_author=KA Mather, AL Caicedo, NR Polato, KM Olsen, S McCouch, MD Purugganan; citation_volume=177; citation_issue=4; citation_publication_date=2007; citation_pages=2223-32; citation_doi=10.1534/genetics.107.079616; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Genetic structure and linkage disequilibrium in landrace populations of barley in Sardinia; citation_author=M Rodriguez, D Rau, D O’Sullivan, AHD Brown, R Papa, G Attene; citation_volume=125; citation_issue=1; citation_publication_date=2012; citation_pages=171-84; citation_doi=10.1007/s00122-012-1824-8; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Characterization of genetic diversity and linkage disequilibrium of ZmLOX4 and ZmLOX5 loci in maize; citation_author=N Gerald, SC Murray, T Isakeit, Y-S Park, Y Yan, ML Warburton; citation_volume=8; citation_issue=1; citation_publication_date=2013; citation_doi=10.1371/journal.pone.0053973; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Natural allelic diversity, genetic structure and linkage disequilibrium pattern in wild Chickpea; citation_author=MS Saxena, D Bajaj, A Kujur, S Das, S Badoni, V Kumar; citation_volume=9; citation_issue=9; citation_publication_date=2014; citation_doi=10.1371/journal.pone.0107484; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Nucleotide diversity and linkage disequilibrium of nine genes with putative effects on flowering time in perennial ryegrass (Lolium perenne L.); citation_author=A Fiil, I Lenk, K Petersen, CS Jensen, KK Nielsen, B Schejbel; citation_volume=180; citation_issue=2; citation_publication_date=2011; citation_pages=228-37; citation_doi=10.1016/j.plantsci.2010.08.015; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Whole-genome nucleotide diversity, recombination, and linkage disequilibrium in the model legume Medicago truncatula; citation_author=A Branca, TD Paape, P Zhou, R Briskine, AD Farmer, J Mudge; citation_volume=108; citation_issue=42; citation_publication_date=2011; citation_pages=E864-70; citation_doi=10.1073/pnas.1104032108; citation_id=CR24">
<meta name="citation_reference" content="Abdurakhmonov IY, Abdukarimov A. Application of association mapping to understanding the genetic diversity of plant germplasm resources. Int J Plant Genomics. 2008;2008.">
<meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Use of unlinked genetic markers to detect population stratification in association studies; citation_author=JK Pritchard, NA Rosenberg; citation_volume=65; citation_issue=1; citation_publication_date=1999; citation_pages=220-8; citation_doi=10.1086/302449; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=Curr Opin Plant Biol; citation_title=Plant molecular diversity and applications to genomics; citation_author=ES Buckler, JM Thornsberry; citation_volume=5; citation_issue=2; citation_publication_date=2002; citation_pages=107-11; citation_doi=10.1016/S1369-5266(02)00238-8; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Genet; citation_title=Population structure and its effect on haplotype diversity and linkage disequilibrium surrounding the xa5 locus of rice (Oryza sativa L.); citation_author=AJ Garris, SR McCOUCH, S Kresovich; citation_volume=165; citation_issue=2; citation_publication_date=2003; citation_pages=759-69; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Markers for ornamental traits in Phalaenopsis orchids: population structure, linkage disequilibrium and association mapping; citation_author=I Gawenda, A Schröder-Lorenz, T Debener; citation_volume=30; citation_issue=1; citation_publication_date=2012; citation_pages=305-16; citation_doi=10.1007/s11032-011-9620-8; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=Bioresour Technol; citation_title=Switchgrass as a sustainable bioenergy crop; citation_author=MA Sanderson, RL Reed, SB McLaughlin, SD Wullschleger, BV Conger, DJ Parrish; citation_volume=56; citation_issue=1; citation_publication_date=1996; citation_pages=83-93; citation_doi=10.1016/0960-8524(95)00176-X; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Biomass Bioenerg; citation_title=
                              Miscanthus: European experience with a novel energy crop; citation_author=I Lewandowski, JC Clifton-Brown, JMO Scurlock, W Huisman; citation_volume=19; citation_issue=4; citation_publication_date=2000; citation_pages=209-27; citation_doi=10.1016/S0961-9534(00)00032-5; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Curr Opin Genet Dev; citation_title=Molecular breeding of switchgrass for use as a biofuel crop; citation_author=JH Bouton; citation_volume=17; citation_issue=6; citation_publication_date=2007; citation_pages=553-8; citation_doi=10.1016/j.gde.2007.08.012; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Curr Opin Plant Biol; citation_title=Directed molecular evolution in plant improvement; citation_author=M Lassner, J Bedbrook; citation_volume=4; citation_issue=2; citation_publication_date=2001; citation_pages=152-6; citation_doi=10.1016/S1369-5266(00)00152-7; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Plant Physiol Biochem; citation_title=Analysis of antioxidant enzyme activity during germination of alfalfa under salt and drought stresses; citation_author=W-B Wang, Y-H Kim, H-S Lee, K-Y Kim, X-P Deng, S-S Kwak; citation_volume=47; citation_issue=7; citation_publication_date=2009; citation_pages=570-7; citation_doi=10.1016/j.plaphy.2009.02.009; citation_id=CR34">
<meta name="citation_reference" content="Ersoz ES, Wright MH, Pangilinan JL, Sheehan MJ, Tobias C, Casler MD et al. SNP discovery with EST and NextGen sequencing in switchgrass (Panicum virgatum L.). 2012.">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Genetic diversity and relationship of global faba bean (Vicia faba L.) germplasm revealed by ISSR markers; citation_author=HF Wang, XX Zong, JP Guan, T Yang, XL Sun, Y Ma; citation_volume=124; citation_issue=5; citation_publication_date=2012; citation_pages=789-97; citation_doi=10.1007/s00122-011-1750-1; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Rep; citation_title=Genetic diversity in some grape varieties revealed by SCoT analyses; citation_author=DL Guo, JY Zhang, CH Liu; citation_volume=39; citation_issue=5; citation_publication_date=2012; citation_pages=5307-13; citation_doi=10.1007/s11033-011-1329-6; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Genetic diversity of Camellia japonica (Theaceae), a species endangered to East Asia, detected by inter-simple sequence repeat (ISSR); citation_author=L Lin, ZY Hu, S Ni, JY Li, YX Qiu; citation_volume=50; citation_publication_date=2013; citation_pages=199-206; citation_doi=10.1016/j.bse.2013.03.049; citation_id=CR38">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Assessment of genetic diversity in the sorghum reference set using EST-SSR markers; citation_author=P Ramu, C Billot, JF Rami, S Senthilvel, HD Upadhyaya, LA Reddy; citation_volume=126; citation_issue=8; citation_publication_date=2013; citation_pages=2051-64; citation_doi=10.1007/s00122-013-2117-6; citation_id=CR39">
<meta name="citation_reference" content="citation_journal_title=Bioenerg Res; citation_title=Genetic diversity in switchgrass collections assessed by EST-SSR markers; citation_author=B Narasimhamoorthy, MC Saha, T Swaller, JH Bouton; citation_volume=1; citation_issue=2; citation_publication_date=2008; citation_pages=136-46; citation_doi=10.1007/s12155-008-9011-0; citation_id=CR40">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Diversity among populations of switchgrass based on RAPD markers; citation_author=LE Gunter, GA Tuskan, SD Wullschleger; citation_volume=36; citation_issue=4; citation_publication_date=1996; citation_pages=1017-22; citation_doi=10.2135/cropsci1996.0011183X003600040034x; citation_id=CR41">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Incompatibility systems in switchgrass; citation_author=JM Martinez-Reyna, KP Vogel; citation_volume=42; citation_issue=6; citation_publication_date=2002; citation_pages=1800-5; citation_doi=10.2135/cropsci2002.1800; citation_id=CR42">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Genetic diversity, plant adaptation regions, and gene pools for switchgrass; citation_author=MD Casler, CA Stendal, L Kapich, KP Vogel; citation_volume=47; citation_issue=6; citation_publication_date=2007; citation_pages=2261-73; citation_doi=10.2135/cropsci2006.12.0797; citation_id=CR43">
<meta name="citation_reference" content="citation_journal_title=Agrofor Syst; citation_title=Genetic diversity analysis of switchgrass (Panicum virgatum L.) populations using microsatellites and chloroplast sequences; citation_author=M Nageswara-Rao, M Hanson, S Agarwal, CN Stewart, C Kwit; citation_volume=88; citation_issue=5; citation_publication_date=2014; citation_pages=823-34; citation_doi=10.1007/s10457-014-9728-z; citation_id=CR44">
<meta name="citation_reference" content="Hamrick JL, Godt MJW, Brown AHD, Clegg MT, Kahler AL, Weir BS. Allozyme diversity in plant species. Plant Popul Genet Breed Genet Res. 1990;43–63.">
<meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Comparison of PCR-based marker systems for the analysis of genetic relationships in cultivated potato; citation_author=D Milbourne, R Meyer, JE Bradshaw, E Baird, N Bonar, J Provan; citation_volume=3; citation_issue=2; citation_publication_date=1997; citation_pages=127-36; citation_doi=10.1023/A:1009633005390; citation_id=CR46">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Rep; citation_title=Start codon targeted polymorphism for evaluation of functional genetic variation and relationships in cultivated peanut (Arachis hypogaea L.) genotypes; citation_author=FQ Xiong, RC Zhong, ZQ Han, J Jiang, LQ He, WJ Zhuang; citation_volume=38; citation_issue=5; citation_publication_date=2011; citation_pages=3487-94; citation_doi=10.1007/s11033-010-0459-6; citation_id=CR47">
<meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Start Codon Targeted (SCoT) marker reveals genetic diversity of Dendrobium nobile Lindl., an endangered medicinal orchid species; citation_author=P Bhattacharyya, S Kumaria, S Kumar, P Tandon; citation_volume=529; citation_issue=1; citation_publication_date=2013; citation_pages=21-6; citation_doi=10.1016/j.gene.2013.07.096; citation_id=CR48">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Analysis of genetic diversity and relationships among genus Lycoris based on start codon targeted (SCoT) marker; citation_author=YH Gao, YQ Zhu, ZK Tong, ZY Xu, XF Jiang, CH Huang; citation_volume=57; citation_publication_date=2014; citation_pages=221-6; citation_doi=10.1016/j.bse.2014.08.002; citation_id=CR49">
<meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Genetic variability and structure of Quercus brantii assessed by ISSR, IRAP and SCoT markers; citation_author=L Alikhani, M-S Rahmani, N Shabanian, H Badakhshan, A Khadivi-Khub; citation_volume=552; citation_issue=1; citation_publication_date=2014; citation_pages=176-83; citation_doi=10.1016/j.gene.2014.09.034; citation_id=CR50">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Genetic diversity of mango cultivars estimated using SCoT and ISSR markers; citation_author=C Luo, XH He, H Chen, SJ Ou, MP Gao, JS Brown; citation_volume=39; citation_issue=4; citation_publication_date=2011; citation_pages=676-84; citation_doi=10.1016/j.bse.2011.05.023; citation_id=CR51">
<meta name="citation_reference" content="citation_journal_title=Mach Learn; citation_title=Being bayesian about network structure; citation_author=N Friedmain, D Koller; citation_volume=50; citation_publication_date=2003; citation_pages=95-126; citation_doi=10.1023/A:1020249912095; citation_id=CR52">
<meta name="citation_reference" content="citation_journal_title=Genet Res Crop Evol; citation_title=Molecular markers for the classification of switchgrass (Panicum virgatum L.) germplasm and to assess genetic diversity in three synthetic switchgrass populations; citation_author=AM Missaoui, AH Paterson, JH Bouton; citation_volume=53; citation_issue=6; citation_publication_date=2006; citation_pages=1291-302; citation_doi=10.1007/s10722-005-3878-9; citation_id=CR53">
<meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Evolution of switchgrass (Panicum virgatum L.) based on sequences of the nuclear gene encoding plastid acetyl-CoA carboxylase; citation_author=SX Huang, XJ Su, R Haselkorn, P Gornicki; citation_volume=164; citation_issue=1; citation_publication_date=2003; citation_pages=43-9; citation_doi=10.1016/S0168-9452(02)00327-8; citation_id=CR54">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Hierarchical classification of switchgrass genotypes using SSR and chloroplast sequences: ecotypes, ploidies, gene pools, and cultivars; citation_author=JE Zalapa, DL Price, SM Kaeppler, CM Tobias, M Okada, MD Casler; citation_volume=122; citation_issue=4; citation_publication_date=2011; citation_pages=805-17; citation_doi=10.1007/s00122-010-1488-1; citation_id=CR55">
<meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Genetic diversity in tetraploid switchgrass revealed by AFLP marker polymorphisms; citation_author=J Todd, YQ Wu, Z Wang, T Samuels; citation_volume=10; citation_issue=4; citation_publication_date=2011; citation_pages=2976-86; citation_doi=10.4238/2011.November.29.8; citation_id=CR56">
<meta name="citation_reference" content="Sneath PHA, Sokal RR. Numerical taxonomy. The principles and practice of numerical classification. San Francisco: WH Freeman; 1973.">
<meta name="citation_reference" content="citation_journal_title=Mol Plant Breed (China); citation_title=Progress on the trait-marker association analysis in plants; citation_author=L Jin, JS Bao; citation_volume=7; citation_issue=6; citation_publication_date=2009; citation_pages=1048-63; citation_id=CR58">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome; citation_author=DL Remington, JM Thornsberry, Y Matsuoka, LM Wilson, SR Whitt, J Doebley; citation_volume=98; citation_issue=20; citation_publication_date=2001; citation_pages=11479-84; citation_doi=10.1073/pnas.201394398; citation_id=CR59">
<meta name="citation_reference" content="citation_journal_title=Genet; citation_title=Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites; citation_author=K Liu, M Goodman, S Muse, JS Smith, E Buckler, J Doebley; citation_volume=165; citation_issue=4; citation_publication_date=2003; citation_pages=2117-28; citation_id=CR60">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Ribosomal DNA spacer-length polymorphisms in barley: Mendelian inheritance, chromosomal location, and population dynamics; citation_author=MA Saghai-Maroof, KM Soliman, RA Jorgensen, RW Allard; citation_volume=81; citation_issue=24; citation_publication_date=1984; citation_pages=8014-8; citation_doi=10.1073/pnas.81.24.8014; citation_id=CR61">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Evol; citation_title=Single-locus EST-SSR markers for characterization of population genetic diversity and structure across ploidy levels in switchgrass (Panicum virgatum L.); citation_author=M Okada, C Lanzatella, CM Tobias; citation_volume=58; citation_issue=6; citation_publication_date=2011; citation_pages=919-31; citation_doi=10.1007/s10722-010-9631-z; citation_id=CR62">
<meta name="citation_reference" content="citation_journal_title=Sci Hortic; citation_title=Genetic diversity of Chinese natural bermudagrass (Cynodon dactylon) germplasm using ISSR markers; citation_author=HY Li, L Liu, YH Lou, T Hu, JM Fu; citation_volume=127; citation_issue=4; citation_publication_date=2011; citation_pages=555-61; citation_doi=10.1016/j.scienta.2010.12.001; citation_id=CR63">
<meta name="citation_reference" content="citation_title=POPGENE-for the analysis of genetic variation among and within populations using co-dominant and dominant markers; citation_publication_date=2000; citation_id=CR64; citation_author=F Yeh; citation_author=R Yang">
<meta name="citation_reference" content="citation_journal_title=Genet; citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data; citation_author=L Excoffier, PE Smouse, JM Quattro; citation_volume=131; citation_issue=2; citation_publication_date=1992; citation_pages=479-91; citation_id=CR65">
<meta name="citation_reference" content="citation_journal_title=Chin Biodivers; citation_title=Data analysis in population genetics. I. Analysis of RAPD data with AMOVA; citation_author=F Zhang, S Ge; citation_volume=10; citation_issue=4; citation_publication_date=2001; citation_pages=438-44; citation_id=CR66">
<meta name="citation_reference" content="citation_journal_title=Sci Aric Sin; citation_title=Comparison of genetic diversity of twelve Elymus species using ISSR and SSR markers; citation_author=YX Li, SS Li, LH Li, XM Yang, XQ Li; citation_volume=38; citation_issue=8; citation_publication_date=2004; citation_pages=1522-7; citation_id=CR67">
<meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Comparative assessment of DNA fingerprinting techniques (RAPD, ISSR and AFLP) for genetic analysis of cashew (Anacardium occidentale L.) accessions of India; citation_author=S Archak, AB Gaikwad, D Gautam, EVVB Rao, KRM Swamy, JL Karihaloo; citation_volume=46; citation_issue=3; citation_publication_date=2003; citation_pages=362-9; citation_doi=10.1139/g03-016; citation_id=CR68">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=Inferring weak population structure with the assistance of sample group information; citation_author=MJ Hubisz, D Falush, M Stephens, JK Pritchard; citation_volume=9; citation_issue=5; citation_publication_date=2009; citation_pages=1322-32; citation_doi=10.1111/j.1755-0998.2009.02591.x; citation_id=CR69">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; citation_author=G Evanno, S Regnaut, J Goudet; citation_volume=14; citation_issue=8; citation_publication_date=2005; citation_pages=2611-20; citation_doi=10.1111/j.1365-294X.2005.02553.x; citation_id=CR70">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Notes; citation_title=DISTRUCT: a program for the graphical display of population structure; citation_author=NA Rosenberg; citation_volume=4; citation_issue=1; citation_publication_date=2004; citation_pages=137-8; citation_doi=10.1046/j.1471-8286.2003.00566.x; citation_id=CR71">
<meta name="citation_reference" content="citation_journal_title=Int J Syst Evol Microbiol; citation_title=Construction and bootstrap analysis of DNA fingerprinting-based phylogenetic trees with the freeware program FreeTree: application to trichomonad parasites; citation_author=V Hampl, A Pavlícek, J Flegr; citation_volume=51; citation_issue=3; citation_publication_date=2001; citation_pages=731-5; citation_doi=10.1099/00207713-51-3-731; citation_id=CR72">
<meta name="citation_reference" content="citation_title=NTSYS-PC, numerical taxonomy system for the PC ExeterSoftware, Version 2.1; citation_publication_date=2000; citation_id=CR73; citation_author=FJ Rohlf; citation_publisher=Applied Biostatistics Inc Setauket">
<meta name="citation_reference" content="citation_title=Evolution and the genetics of populations, volume 3: experimental results and evolutionary deductions; citation_publication_date=1984; citation_id=CR74; citation_author=S Wright; citation_publisher=University of Chicago Press">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=TASSEL: software for association mapping of complex traits in diverse samples; citation_author=PJ Bradbury, Z Zhang, DE Kroon, TM Casstevens, Y Ramdoss, ES Buckler; citation_volume=23; citation_issue=19; citation_publication_date=2007; citation_pages=2633-5; citation_doi=10.1093/bioinformatics/btm308; citation_id=CR75">
<meta name="citation_reference" content="Lewis PO, Zaykin D. Genetic data analysis: computer program for the analysis of allelic data. Available at 
                    http://lewis.eeb.uconn.edu/lewishome/software.html
                    
                  . 2001.">
<meta name="citation_author" content="Yu Zhang">
<meta name="citation_author_institution" content="Grassland Science Department, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author_institution" content="IRTA. Centre de Recerca en Agrigenòmica (CSIC-IRTA-UAB), Campus UAB – Edifici CRAG, Bellaterra - Cerdanyola del Vallès, Barcelona, Spain">
<meta name="citation_author" content="Haidong Yan">
<meta name="citation_author_institution" content="Grassland Science Department, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Xiaomei Jiang">
<meta name="citation_author_institution" content="Grassland Science Department, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Xiaoli Wang">
<meta name="citation_author_institution" content="Guizhou Institute of Prataculture, Guiyang, PR China">
<meta name="citation_author" content="Linkai Huang">
<meta name="citation_author_institution" content="Grassland Science Department, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Bin Xu">
<meta name="citation_author_institution" content="College of Grassland Science, Nanjing Agricultural University, Nanjing, China">
<meta name="citation_author" content="Xinquan Zhang">
<meta name="citation_author_institution" content="Grassland Science Department, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Lexin Zhang">
<meta name="citation_author_institution" content="Grassland Science Department, Sichuan Agricultural University, Chengdu, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0007-z',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0007-z","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Genetic variation;Linkage disequilibrium;Panicum virgatum L;Population structure"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic variation, population structure and linkage disequilibrium in Switchgrass with ISSR, SCoT and EST-SSR markers","publishedAt":1461024000000,"publishedAtDate":"2016-04-19","author":["Yu Zhang","Haidong Yan","Xiaomei Jiang","Xiaoli Wang","Linkai Huang","Bin Xu","Xinquan Zhang","Lexin Zhang"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0007-z">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0007-z">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Genetic variation, population structure and linkage disequilibrium in Switchgrass with ISSR, SCoT and EST-SSR markers">
<meta property="og:description" content="To evaluate genetic variation, population structure, and the extent of linkage disequilibrium (LD), 134 switchgrass (Panicum virgatum L.) samples were analyzed with 51 markers, including 16 ISSRs, 20 SCoTs, and 15 EST-SSRs. In this study, a high level of genetic variation was observed in the switchgrass samples and they had an average Neiâs gene diversity index (H) of 0.311. A total of 793 bands were obtained, of which 708 (89.28&amp;nbsp;%) were polymorphic. Using a parameter marker index (MI), the efficiency of the three types of markers (ISSR, SCoT, and EST-SSR) in the study were compared and we found that SCoT had a higher marker efficiency than the other two markers. The 134 switchgrass samples could be divided into two sub-populations based on STRUCTURE, UPGMA clustering, and principal coordinate analyses (PCA), and upland and lowland ecotypes could be separated by UPGMA clustering and PCA analyses. Linkage disequilibrium analysis revealed an average r2 of 0.035 across all 51 markers, indicating a trend of higher LD in sub-population 2 than that in sub-population 1 (Pâ&amp;lt;â0.01). The population structure revealed in this study will guide the design of future association studies using these switchgrass samples.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0007-z;kwrd=Genetic variation,Linkage disequilibrium,Panicum virgatum L,Population structure;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0007-z&amp;kwrd=Genetic%20variation,Linkage%20disequilibrium,Panicum%20virgatum%20L,Population%20structure&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0007-z&amp;kwrd=Genetic%20variation,Linkage%20disequilibrium,Panicum%20virgatum%20L,Population%20structure&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and discussion">Results and discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Method">Method</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic variation, population structure and linkage disequilibrium in Switchgrass with ISSR, SCoT and EST-SSR markers</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true">
<ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yu Zhang</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Haidong Yan</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaomei Jiang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaoli Wang</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Linkai Huang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:huanglinkai@sicau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Bin Xu</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xinquan Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:zhangxq@sicau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lexin Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>
</li>
</ul>
<div class="AffiliationLegend"><div class="AffiliationLegendLine">
<span class="EqualContributionSymbol">â </span>Contributed equally</div></div>
</div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:4</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0007-z">https://doi.org/10.1186/s41065-016-0007-z</a></p>
<p class="Copyright">Â©  Zhang et al. 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>23 December 2015</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>29 March 2016</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">19 April 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">To evaluate genetic variation, population structure, and the extent of linkage disequilibrium (LD), 134 switchgrass (<em xmlns="" class="EmphasisTypeItalic">Panicum virgatum</em> L.) samples were analyzed with 51 markers, including 16 ISSRs, 20 SCoTs, and 15 EST-SSRs.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">In this study, a high level of genetic variation was observed in the switchgrass samples and they had an average Neiâs gene diversity index (H) of 0.311. A total of 793 bands were obtained, of which 708 (89.28 %) were polymorphic. Using a parameter marker index (MI), the efficiency of the three types of markers (ISSR, SCoT, and EST-SSR) in the study were compared and we found that SCoT had a higher marker efficiency than the other two markers. The 134 switchgrass samples could be divided into two sub-populations based on STRUCTURE, UPGMA clustering, and principal coordinate analyses (PCA), and upland and lowland ecotypes could be separated by UPGMA clustering and PCA analyses. Linkage disequilibrium analysis revealed an average r<sup>2</sup> of 0.035 across all 51 markers, indicating a trend of higher LD in sub-population 2 than that in sub-population 1 (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.01).</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">The population structure revealed in this study will guide the design of future association studies using these switchgrass samples.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Genetic variation</li>
<li class="c-keywords__item" itemprop="about">Linkage disequilibrium</li>
<li class="c-keywords__item" itemprop="about">
                        <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Panicum virgatum</em> L</li>
<li class="c-keywords__item" itemprop="about">Population structure</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Genetic diverstiy is a significant factor that contributes to crop improvement. Evaluation of genetic variation in contemporary germplasm through breeding programs may be indirectly favorable for genetic progress in future cultivars [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Thus, estimation of plant diversity is crucial for the efficacious use of genetic resources in breeding programs. Molecular markers, as particular segments of DNA that represent different functional classes, play an essential role in all aspects of plant breeding, and have been widely used to estimate genetic variation.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Compared with conventional phenotyping methods, molecular markers have numerous advantages as they are easily detectable and stable in plant tissues regardless of environmental influences [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. The inter simple sequence repeat marker (ISSR) is highly polymorphic and is useful in studies of genetic diversity, genome mapping and evolutionary biology [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. This PCR-based technique is used in various types of plants and can overcome many defects of other marker methods, such as high-cost of amplified fragment length (AFLP) and the low reproducibility of random amplified polymorphic DNA (RAPD) [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. Start codon targeted marker (SCoT) is a reliable and simple gene-targeted marker located on the translational start codon [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. This technique involves designing single primers from the short conserved region flanking the ATG start codon [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] without knowing any further genomic sequence information. It has been used in peanut and mango crops for genetic diversity and cultivar relationship analysis [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Expressed sequence tag-simple sequence repeats marker (EST-SSR) detects variation based on the expressed portion of the genome from EST databases, thus explaining the low cost of development compared with the genomic simple sequence repeat marker (SSR) [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. These EST-SSR primers can be used across various species for comparative mapping and the construction of genetic linkage maps [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Each marker type has unique advantages and these three marker systems have found extensive application in the evaluation of genetic variation, population structure, and assisted selection for crop improvement [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>â<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Many studies have shown that these markers are mainly used to develop genetic linkage maps [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], however, fewer studies have focused on constructing linkage disequilibrium (LD) maps. Remarkably, LD and linkage are two different genetic terms, where LD refers to correlation between alleles in a population, while LD means the correlated inheritance of loci through physical connection on a chromosome [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Some factors can affect the LD level, including allele frequency and recombination. Unlike linkage analysis, LD mapping relies on a natural population which is used to identify the relationships between genetic and phenotypic variation. LD mapping, that is association analysis, represents a useful tool to identify trait-marker relationships, and the first LD mapping of a quantitative trait was the analysis of flowering time and the <em xmlns="" class="EmphasisTypeItalic">dwarf8</em> gene in maize [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Linkage disequilibrium (LD), referring to the nonrandom association of alleles between linked or unlinked loci, is the basis of association mapping to identify genetic regions associated with agronomic traits [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Recently, LD studies have been performed in various plants, such as rice (<em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em> L.) [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>], barley (<em xmlns="" class="EmphasisTypeItalic">Hordeum vulgare</em> L.) [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>], Maize (<em xmlns="" class="EmphasisTypeItalic">Zea mays</em> L.) [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], chickpea (<em xmlns="" class="EmphasisTypeItalic">Cicer arietinum</em> L.) [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>], perennial ryegrass (<em xmlns="" class="EmphasisTypeItalic">Lolium perenne</em> L.) [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], and the model legume, <em xmlns="" class="EmphasisTypeItalic">Medicago truncatula</em> [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. The level of LD is constantly regarded as a standard to reflect mapping resolution. Association mapping in populations with low LD requires a high number of markers, whereas a high LD means low mapping resolution [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. In addition, information about population structure within germplasm collections is also crucial for the interpretation and identification of associations between genetic and functional diversity, and to assess whether the inter-sample relatedness is suitable for association studies [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>â<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Therefore, population structure is also included as an effect in models used for association analysis. [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Switchgrass (<em xmlns="" class="EmphasisTypeItalic">Panicum virgatum</em> L.), as a warm season C<sub>4</sub> perennial grass that is native to North America [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>], is regarded as an important biofuel crop for its remarkable biomass yield and good adaptability on marginal lands thereby not competing with food crops on farmland [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>â<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. In this study, we explored two distinct forms of switchgrass, upland and lowland ecotypes. The upland accessions are distributed in northern cold areas with lower biomass than lowland varieties. Generally, upland switchgrass is shorter (â¤2.4 m, tall) than lowland types (â¥ 2.7 m) in favorable environments. However, lowland cultivars appear more sensitive to moisture stress than upland cultivars [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">Constructing association maps comparing the physiological and genetic basis of varying stresses can provide an available reference for the genetic improvement of switchgrass, and the evaluation of the level of LD and population structure can aid association analyses. To date, however, LD analysis across the switchgrass genome remains inadequate [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. In our study, we present 134 switchgrass accessions supplied by Plant Genetic Resources Conservation Unit, Griffin, Georgia USA to identify the levels of genetic variation, population structure, and extent of LD using 51 markers including 16 ISSRs, 20 SCoTs, and 15 EST-SSRs. These results will provide a valuable molecular basis for enriching switchgrass genetic variation, and the information on the level of LD and population structure may guide association mapping using this representative collection.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">Here we constructed a three-marker molecular dataset with important applications for diversity analysis, establishment of population structure and evaluation of linkage disequilibrium in switchgrass which is an allogamous species.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsanddiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsanddiscussion">Results and discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Resultsanddiscussion" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic variation analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">The ISSR, SCoT, and EST-SSR primers were screened using the selected four genotypes [PI421999 (AM-314/MS-155), PI422006 (Alamo), PI642190 (Falcon), and PI642207 (70SG 016)]. After the initial screening, the numbers of selected ISSR, SCoT, and EST-SSR primers used in further studies were reduced to 16, 20, and 15 pairs, respectively (Table <span class="InternalRef"><a href="#Tab1">1</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">The ISSR, SCoT, and EST-SSR primers used in this study and amplification results</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Primer</p></th>
<th><p xmlns="" class="SimplePara">Primer sequence (5â²âââ3â²)</p></th>
<th><p xmlns="" class="SimplePara">Annealing (Â°C)</p></th>
<th><p xmlns="" class="SimplePara">Total number of amplified bands (TNB)</p></th>
<th><p xmlns="" class="SimplePara">The number of polymorphic bands (NPB)</p></th>
<th><p xmlns="" class="SimplePara">Percentage of polymorphic bands (PPB) %)</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC812</p></td>
<td><p xmlns="" class="SimplePara">(GA)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>A</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">92.31</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC827</p></td>
<td><p xmlns="" class="SimplePara">(AC)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>G</p></td>
<td><p xmlns="" class="SimplePara">53.0</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC828</p></td>
<td><p xmlns="" class="SimplePara">(TG)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>A</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">88.89</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC829</p></td>
<td><p xmlns="" class="SimplePara">(TG)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>C</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">83.33</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC830</p></td>
<td><p xmlns="" class="SimplePara">(TG)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>G</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">92.86</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC835</p></td>
<td><p xmlns="" class="SimplePara">(AG)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>YC</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">86.67</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC836</p></td>
<td><p xmlns="" class="SimplePara">(AG)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>YT</p></td>
<td><p xmlns="" class="SimplePara">54.0</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">82.35</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC844</p></td>
<td><p xmlns="" class="SimplePara">(CT)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>RC</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">85.71</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC848</p></td>
<td><p xmlns="" class="SimplePara">(CA)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>RG</p></td>
<td><p xmlns="" class="SimplePara">53.0</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">92.86</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC854</p></td>
<td><p xmlns="" class="SimplePara">(TC)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub>RG</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC868</p></td>
<td><p xmlns="" class="SimplePara">(GAA)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">92.31</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC876</p></td>
<td><p xmlns="" class="SimplePara">(GATA)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub>(GACA)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub></p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">81.25</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC879</p></td>
<td><p xmlns="" class="SimplePara">(CTTCA)<sub xmlns="http://www.w3.org/1999/xhtml">3</sub></p></td>
<td><p xmlns="" class="SimplePara">53.0</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">82.35</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC887</p></td>
<td><p xmlns="" class="SimplePara">DVD(TC)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC890</p></td>
<td><p xmlns="" class="SimplePara">VHV(GT)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">84.62</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR-UBC891</p></td>
<td><p xmlns="" class="SimplePara">HVH(TG)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">85.71</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT2</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">82.35</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT3</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCG</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">34</p></td>
<td><p xmlns="" class="SimplePara">31</p></td>
<td><p xmlns="" class="SimplePara">91.18</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT4</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCT</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">90.48</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT5</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGA</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">90.48</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT6</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">91.91</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT7</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGG</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">91.91</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT9</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCAGCA</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">95.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT10</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCAGCC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">95.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT12</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCAACG</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">96.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT13</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCATCG</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">88.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT15</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCGCGA</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">88.89</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT16</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGAC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">26</p></td>
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">92.31</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT18</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">90.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT21</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCCACA</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">90.48</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT28</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCA</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">88.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT31</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCT</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">86.36</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT34</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCA</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">90.48</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT35</p></td>
<td><p xmlns="" class="SimplePara">CATGGCTACCACCGGCCC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">90.48</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT37</p></td>
<td><p xmlns="" class="SimplePara">CAATGGCTACCACTAGCC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">86.96</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCoT48</p></td>
<td><p xmlns="" class="SimplePara">ACAATGGCTACCACTGGC</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">90.00</p></td>
</tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl35</p></td>
<td><p xmlns="" class="SimplePara">f: AAGTGAGCACAACGACACGA</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">58.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">8</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">88.89</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:CGATCCAAAGAAGCAAAGATG</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl37</p></td>
<td><p xmlns="" class="SimplePara">f:CTGCCTCGCGTGAAAGATA</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">10</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">90.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:CCTCCTCGATCTGGATGGT</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl42</p></td>
<td><p xmlns="" class="SimplePara">f:GTTGGTCTGCTGCTCACTCG</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">8</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">88.89</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:CCGACGATGTTGAAGGAGAG</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl47</p></td>
<td><p xmlns="" class="SimplePara">f: GACTCGCACGATTTCTCCTC</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">57.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">8</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">88.89</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:GCCAGACAACCAATTCAGGT</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl51</p></td>
<td><p xmlns="" class="SimplePara">f:CTAGGGTTTCCCACCTCTCA</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">8</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">6</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">75.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:AATGTCCTTGGCGTTGCT</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl55</p></td>
<td><p xmlns="" class="SimplePara">f:GCTGATAGCGAGGTGGGTAG</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">58.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">14</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">11</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">78.57</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:CTGCCGGTTGATCTTGTTCT</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl61</p></td>
<td><p xmlns="" class="SimplePara">f:CACGAGTGCAGAGCTAGACG</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">60.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">5</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">4</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">80.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:ACAACAACCCGACTGCTACC</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl86</p></td>
<td><p xmlns="" class="SimplePara">f:CAACAACGTCAACGCCTTC</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">11</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">8</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">72.73</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:GCGTCTTGAACCTCTTGTCC</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl100</p></td>
<td><p xmlns="" class="SimplePara">f:CGTCGTCCTCTGCTGTGAG</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">58.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">5</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">4</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">80.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:AGGTCGTCCATCTGCTGCT</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl115</p></td>
<td><p xmlns="" class="SimplePara">f:CGAGAAGAAGGTGGTGTCGT</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">7</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">6</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">85.71</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:AGGTCGTGGAAGGTCTTGG</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl119</p></td>
<td><p xmlns="" class="SimplePara">f:ATCGTCTCCTCCTCCTCCA</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">57.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">6</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">6</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:ATGCCTCGGTGGACTGGTA</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl130</p></td>
<td><p xmlns="" class="SimplePara">f:AAATGTTGAGCAACGGGAGCT</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">7</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">6</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">85.71</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:ACTTCATAGGGCGGAGGTCT</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl144</p></td>
<td><p xmlns="" class="SimplePara">f:AGAAGGCGGCTCAGAAGAAG</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">58.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">10</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">10</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:GCTCCAACTCAGAATCAACAA</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl147</p></td>
<td><p xmlns="" class="SimplePara">f:GGCTAGGGTTTCGACTCCTC</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">60.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">7</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">77.78</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:AGATGGCGAACTCGACCTG</p></td></tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR-cnl158</p></td>
<td><p xmlns="" class="SimplePara">f:CTCATCCCACCACCACCAC</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">59.0</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">9</p></td>
<td rowspan="2"><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass"><td><p xmlns="" class="SimplePara">r:CCCTGAAGAAGTCGAACACG</p></td></tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">793</p></td>
<td><p xmlns="" class="SimplePara">708</p></td>
<td><p xmlns="" class="SimplePara">89.28</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">These three marker systems (ISSR, SCoT, and EST-SSR) have been used for cultivar identification and genetic variation assessment in many plant species [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>â<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. In this study, these markers were successfully used to differentiate switchgrass accessions. A total of 51 primer pairs were used and 793 bands were produced, with a mean of 15.5 bands per primer, among which 89.28 % were polymorphic. Our results suggested that ISSR, SCoT, and EST-SSR analyses could contribute to the detection of genetic variation. In addition, Neiâs (1973) gene diversity index (H) and Shannonâs information index (I) was 0.311 and 0.471, respectively, and the similarity coefficient, ranging from 0.162 to 0.857 with an average of 0.510 was similar to other studies on switchgrass, in which the similarity coefficients were estimated to be between 0.45 to 0.81 [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>] or 0.53 to 0.78 [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. This indicates that switchgrass has abundant genetic variation and is a highly heterogenous species [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. The AMOVA of the distance matrix for the genotypes permitted a partitioning of the overall variation into two levels: between upland and lowland ecotypes and within a population. The results revealed genetic differentiation between upland and lowland ecotypes (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001), with 31.42 % of genetic variation between ecotypes and 68.58 % of genetic variation within ecotypes. Similar results were obtained in other switchgrass germplasm collections [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>, <span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>, <span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>] and in other perennial, and cross-pollinated plants [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>].</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">Marker efficiency analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">In this study, we extracted genomic DNA from an individual so that we were able to obtain complete genetic information including allele numbers, gene frequency and observed heterozygosity for marker efficiency analysis. A parameter marker index (MI) was used to compare the efficiencies of the three assays in the collection of 134 switchgrass genotypes (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). There was almost no disparity between the average band informativeness (Ib<sub>av</sub>) indice for ISSRs, SCoTs, and EST-SSRs, which were 0.38, 0.43, and 0.36, respectively. However, the effective multiplex ratio (EMR) index for ScoT (20.10) was twice as high as that of the ISSRs (12.25) and three times as high as that of the EST-SSRs (7.33). The MI calculation indicated an efficient and distinctive nature of the SCoTs with the MI for these markers (8.64) higher than the other two assays examined here (4.66 for ISSRs and 2.64 for EST-SSRs).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Comparison of usefulness between ISSR, SCoT, and EST-SSR markers for 134 switchgrass accessions</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Items</p></th>
<th><p xmlns="" class="SimplePara">ISSR</p></th>
<th><p xmlns="" class="SimplePara">SCoT</p></th>
<th><p xmlns="" class="SimplePara">EST-SSR</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">No. of primers</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">No. of total bands</p></td>
<td><p xmlns="" class="SimplePara">220</p></td>
<td><p xmlns="" class="SimplePara">445</p></td>
<td><p xmlns="" class="SimplePara">128</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">No. of average bands per primers</p></td>
<td><p xmlns="" class="SimplePara">13.75</p></td>
<td><p xmlns="" class="SimplePara">22.25</p></td>
<td><p xmlns="" class="SimplePara">8.53</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Percentage of polymorphic bands (PPB)</p></td>
<td><p xmlns="" class="SimplePara">0.89</p></td>
<td><p xmlns="" class="SimplePara">0.90</p></td>
<td><p xmlns="" class="SimplePara">0.86</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average band informativeness (Ib<sub xmlns="http://www.w3.org/1999/xhtml">av</sub>)</p></td>
<td><p xmlns="" class="SimplePara">0.38</p></td>
<td><p xmlns="" class="SimplePara">0.43</p></td>
<td><p xmlns="" class="SimplePara">0.36</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Effective multiplex ratio (EMR)</p></td>
<td><p xmlns="" class="SimplePara">12.25</p></td>
<td><p xmlns="" class="SimplePara">20.10</p></td>
<td><p xmlns="" class="SimplePara">7.33</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Marker index (MI)</p></td>
<td><p xmlns="" class="SimplePara">4.66</p></td>
<td><p xmlns="" class="SimplePara">8.64</p></td>
<td><p xmlns="" class="SimplePara">2.64</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">A parameter MI, has been widely used to evaluate the overall utility of each marker system [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. The high MI in the SCoTs results from its high EMR, making these markers appropriate for fingerprinting [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>] or evaluating genetic variation in breeding populations [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>, <span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>]. In addition, the SCoTs performed well in other species. Compared with ISSR and inter-retrotransposon amplified polymorphism (IRAP), SCoT markers were more informative than IRAP and ISSR for the assessment of diversity among Persian oak (<em xmlns="" class="EmphasisTypeItalic">Quercus brantii</em> Lindl.) individuals [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>]. Results from the evaluation on the genetic variation of mango (<em xmlns="" class="EmphasisTypeItalic">Mangifera indica</em> L.) cultivars indicated that the SCoT analysis represents actual relationships better than the ISSR analysis [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>].</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Population structure analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">After removing low frequency bands (considering MAFââ¤â0.05), we analyzed the data from 51 pairs of ISSR, SCoT, and EST-SSR primers to understand the population structure of the entire switchgrass collection based on a Bayesian clustering approach using STRUCTURE [<span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>]. The number of subpopulations (K) was identified based on maximum likelihood and ÎK values. For the 134 switchgrass genotypes the maximum ÎK was observed at Kâ=â2 (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>), with genotypes falling into two subpopulations. Using a membership probability threshold of 0.75, 76 genotypes were assigned to subpopulation 1 (G1), out of which, 69 genotypes belonged to upland ecotypes, and the remaining 7 were lowland. Subpopulation 2 (G2) contained 42 genotypes, and all of them were upland ecotypes. The remaining 16 genotypes were classified into an admixed group as they had membership probabilities lower than 0.75 in any given subpopulation. With the maximum membership probability, 91 accessions were assigned to G1 and 43 accessions to G2 (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">STRUCTURE analysis of the number of populations for K. The number of subpopulations (K) was identified based on maximum likelihood and ÎK values. The most likely value of K identified by STRUCTURE was observed at Kâ=â2</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Two subgroups inferred from STRUCTURE analysis. The vertical coordinate of each subgroup means the membership coefficients for each accessions; the digits of the horizontal coordinate represent the 134 switchgrass accessions corresponding to Table <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab3">3</a></span>; Red zone: G1, Green zone: G2</p>
                      </div></figcaption></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">The UPGMA cluster analysis from 51 markers generated a dendrogram, demonstrating that the 134 genotypes could be clearly divided into two groups (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). The dendrogram clustered all of the lowland ecotypes (LL) into the first. The second group contained all of the upland ecotypes (UL). Other methods have also been used to cluster upland and lowland switchgrass ecotypes. Missaoui et al adopted restriction fragment length polymorphism (RFLP) markers to analyze the genetic relationships among 21 switchgrass genotypes, resulting in three upland and eighteen lowland genotypes clusteringinto two different groups [<span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>]. Huang et al identified differences between the coding sequences of a nuclear gene encoding plastid acetyl-CoA carboxylase in upland and lowland ecotypes genetic variation analysis at gene level, provided by Huang et al researching about a nuclear gene encoding plastid acetyl-CoA carboxylase [<span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>]. In this study, we preliminarily presented population structure analysis of 7 lowland and 127 upland genotypes using 51 ISSR, SCoT, and EST-SSR primer pairs, resulting in an apparently separate cluster among the two ecotypes, confirming the genetic differences between upland and lowland ecotypes. However, as we do not have as many lowland switchgrass samples as upland, we highly recommend more lowland ecotype or other nuclear markers should be used in conjunction with ISSR, SCoT and EST-SSR to more appropriately classify upland and lowland ecotypes.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig3_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig3_HTML.gif" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Radiation of genetic relationships for 134 switchgrass accessions based on UPGMA. G1 and G2 are the two subgroups identified by STRUCTURE with the maximum membership probability. The numbers at the branches are confidence values based on Felsensteinâs bootstrap produced by FreeTree software, as a general rule, the higher bootstrap value for a given interior branch indicates a closer relationship</p>
                      </div></figcaption></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">Based on modified Rogers distances (MRD), PCA separated the 134 genotypes into two major groups, which was consistent with assignments generated by STRUCTURE and the UPGMA dendrogram (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). Seven genotypes formed group 1 (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>, upper right), and the other 127 genotypes, belonging to group 2, were mainly distributed at the lower portion of the plot. The accessions belonging to G1 inferred by the STRUCTURE analysis were all distributed on the right portion of the resulting plot, while G2 was distributed on the left portion of the plot. The distribution of G1 accessions was less tightly clustered than G2, indicating accessions in G1 had higher diversity than G2 (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig4_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0007-z/MediaObjects/41065_2016_7_Fig4_HTML.gif" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Principal coordinate analysis of 134 switchgrass accessions based on ISSRs, SCoTs, and EST-SSRs. G1 and G2 are the two subgroups identified by STRUCTURE with the maximum membership probability</p>
                      </div></figcaption></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">Before analyzing LD and association mapping, the analysis of population structure emphasizes the need for the genetic analysis of different ecotypes [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. The UPGMA cluster and PCA analysis demonstrated that 134 genotypes could be clearly divided into two groups (Figs. <span class="InternalRef"><a href="#Fig1">1</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>), and the lowland and upland germplasm clusters were almost completely separated, which was consistent with the results of several other switchgrass studies [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>, <span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>, <span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. For the UPGMA cluster analysis, the first group only included lowland ecotypes, while the second group contained upland ecotypes and could be further classified into two subgroups. Subgroup 1 (G1) contained 83 genotypes, while the remaining 43 belonged to subgroup 2 (G2). The 46 accessions of the 70SG series and 42 accessions of the 71SG series dispersed into these two subgroups are from the same geographical distribution of North Dakota, United States. This indicates that most of the germplasm sub-clustered in accordance with different regions [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>, <span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>], and the assignment of 132 accessions (98.51 % of the total) by the UPGMA cluster analysis was consistent with their classification using PCA (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). Unexpectedly, in the STRUCTURE analysis, the 127 upland genotypes were assigned to two subpopulations, possibly because the UPGMA and STRUCTURE programs calculate parameters in different ways. Clusters are generated in STRUCTURE based on both transitory HardyâWeinberg disequilibrium and LD caused by admixture between populations [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>], while the UPGMA dendrogram generates clusters based on the genetic distance among populations [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>].</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Linkage disequilibrium estimation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">After the deletion of low frequency alleles (MAFââ¤â5 %), the 51 ISSRs, SCoTs, and EST-SSRs with unknown chromosome information were used to evaluate the extent of LD among the switchgrass samples. In the collection, interallelic r<sup>2</sup> values, the association between any pair of alleles from different loci, were calculated and ranged from 0.000 to 1.000 with an average r<sup>2</sup> of 0.035. Across all 51 loci, 247,456 locus pairs were detected in the 134 switchgrass samples. Among all of the locus pairs, 7107 of 135,718 (5.24 %) showed LD at the <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001 level for G1 and 5415 locus pairs (3.99 %) were found at r<sup>2</sup>â&gt;â0.1 at <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001. For G2, 84,154 locus pairs were detected, 4833 were significant pairs (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001, 5.74 %), while 4235 locus pairs (5.03 % of 84,154) were found at r<sup>2</sup>â&gt;â0.1 at <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001. The mean r<sup>2</sup> for all materials was 0.480 (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001), and the LD in G2 (0.668, ranging from 0.068 to 1.000) was significantly (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001) larger than that in G1 (0.291, ranging from 0.066 to 1.000) (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.01).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">Populations with high levels of outcrossing have relatively low LD [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>]. Among outcrossing maize (<em xmlns="" class="EmphasisTypeItalic">Zea mays</em> L.), Remington et al. [<span class="CitationRef"><a href="#CR59" data-toggle="citation">59</a></span>] found lower levels of LD among 47 SSR loci (9.7 % of SSR pairs performing LD at <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.01), compared to LD data from an SSR survey of inbred lines of maize, which showed high levels of LD [<span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>]. For switchgrass, LD data comparisons showed a trend towards higher LD in G2 (mean r<sup>2</sup>â=â0.668) including 42 genotypes all belonging to upland ecotypes, compared with G1 (mean r<sup>2</sup>â=â0.291), which contained 76 genotypes, including 7 lowland ecotypes.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Method" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Method">Method</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Method" data-test="collapse-section">
<section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Plant material</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">A total of 134 switchgrass genotypes, representing most of the natural geographical distribution areas of switchgrass supplied by the Plant Genetic Resources Conservation Unit, Griffin, Georgia USA were used in this study. These included 7 lowland genotypes originating from 5 US states and 127 upland genotypes originating from Belgium and 15 US states (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). The full accession data and information on switchgrass germplasm comes from ARS GRIN (<span class="ExternalRef"><a href="http://www.ars-grin.gov/" target="_blank" class="ext-link"><span class="RefSource">http://www.ars-grin.gov/</span></a></span>). The 134 genotypes, including one seedling from each accession, were grown and maintained in the experimental farm of the Sichuan Agricultural University during the 2012 growing season.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">The 134 switchgrass samples used for marker (ISSR, SCoT, and EST-SSR) genotyping</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Code</p></th>
<th><p xmlns="" class="SimplePara">Plant ID</p></th>
<th><p xmlns="" class="SimplePara">Plant name</p></th>
<th><p xmlns="" class="SimplePara">Ecotype</p></th>
<th><p xmlns="" class="SimplePara">Origin</p></th>
<th><p xmlns="" class="SimplePara">Code</p></th>
<th><p xmlns="" class="SimplePara">Plant ID</p></th>
<th><p xmlns="" class="SimplePara">Plant name</p></th>
<th><p xmlns="" class="SimplePara">Ecotype</p></th>
<th><p xmlns="" class="SimplePara">Origin</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">PI315723</p></td>
<td><p xmlns="" class="SimplePara">BN-8358-62</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">North Carolina, US</p></td>
<td><p xmlns="" class="SimplePara">68</p></td>
<td><p xmlns="" class="SimplePara">PI642244</p></td>
<td><p xmlns="" class="SimplePara">70SG 057</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">PI414065</p></td>
<td><p xmlns="" class="SimplePara">BN-14668-65</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">Arkansas, US</p></td>
<td><p xmlns="" class="SimplePara">69</p></td>
<td><p xmlns="" class="SimplePara">PI642245</p></td>
<td><p xmlns="" class="SimplePara">70SG 058</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">PI421521</p></td>
<td><p xmlns="" class="SimplePara">KANLOW</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
<td><p xmlns="" class="SimplePara">70</p></td>
<td><p xmlns="" class="SimplePara">PI642247</p></td>
<td><p xmlns="" class="SimplePara">70SG 060</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">PI421999</p></td>
<td><p xmlns="" class="SimplePara">AM-314/MS-155</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
<td><p xmlns="" class="SimplePara">71</p></td>
<td><p xmlns="" class="SimplePara">PI642248</p></td>
<td><p xmlns="" class="SimplePara">70SG 061</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">PI422006</p></td>
<td><p xmlns="" class="SimplePara">ALAMO</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">Texas, US</p></td>
<td><p xmlns="" class="SimplePara">72</p></td>
<td><p xmlns="" class="SimplePara">PI642249</p></td>
<td><p xmlns="" class="SimplePara">70SG 062</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">6</p></td>
<td><p xmlns="" class="SimplePara">PI607837</p></td>
<td><p xmlns="" class="SimplePara">TEM-SLC 01</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">Texas, US</p></td>
<td><p xmlns="" class="SimplePara">73</p></td>
<td><p xmlns="" class="SimplePara">PI642250</p></td>
<td><p xmlns="" class="SimplePara">70SG 063</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">PI607838</p></td>
<td><p xmlns="" class="SimplePara">TEM-SLC 02</p></td>
<td><p xmlns="" class="SimplePara">LL</p></td>
<td><p xmlns="" class="SimplePara">Texas, US</p></td>
<td><p xmlns="" class="SimplePara">74</p></td>
<td><p xmlns="" class="SimplePara">PI642251</p></td>
<td><p xmlns="" class="SimplePara">70SG 064</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">PI315724</p></td>
<td><p xmlns="" class="SimplePara">BN-10860-61</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
<td><p xmlns="" class="SimplePara">75</p></td>
<td><p xmlns="" class="SimplePara">PI642252</p></td>
<td><p xmlns="" class="SimplePara">70SG 065</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">PI315727</p></td>
<td><p xmlns="" class="SimplePara">BN-11357-63</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Carolina, US</p></td>
<td><p xmlns="" class="SimplePara">76</p></td>
<td><p xmlns="" class="SimplePara">PI642254</p></td>
<td><p xmlns="" class="SimplePara">70SG 067</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">PI414066</p></td>
<td><p xmlns="" class="SimplePara">GRENVILLE</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">New Mexico, US</p></td>
<td><p xmlns="" class="SimplePara">77</p></td>
<td><p xmlns="" class="SimplePara">PI642256</p></td>
<td><p xmlns="" class="SimplePara">70SG 069</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">PI414067</p></td>
<td><p xmlns="" class="SimplePara">BN-8624-67</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Carolina, US</p></td>
<td><p xmlns="" class="SimplePara">78</p></td>
<td><p xmlns="" class="SimplePara">PI642257</p></td>
<td><p xmlns="" class="SimplePara">70SG 071</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">PI414068</p></td>
<td><p xmlns="" class="SimplePara">BN-18758-67</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
<td><p xmlns="" class="SimplePara">79</p></td>
<td><p xmlns="" class="SimplePara">PI642259</p></td>
<td><p xmlns="" class="SimplePara">70SG 073</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">PI421138</p></td>
<td><p xmlns="" class="SimplePara">Carthage</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Carolina, US</p></td>
<td><p xmlns="" class="SimplePara">80</p></td>
<td><p xmlns="" class="SimplePara">PI642260</p></td>
<td><p xmlns="" class="SimplePara">70SG 074</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">PI421520</p></td>
<td><p xmlns="" class="SimplePara">Blackwell</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Oklahoma,US</p></td>
<td><p xmlns="" class="SimplePara">81</p></td>
<td><p xmlns="" class="SimplePara">PI642261</p></td>
<td><p xmlns="" class="SimplePara">70SG 075</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">PI421901</p></td>
<td><p xmlns="" class="SimplePara">MIAMI</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Florida, US</p></td>
<td><p xmlns="" class="SimplePara">82</p></td>
<td><p xmlns="" class="SimplePara">PI642262</p></td>
<td><p xmlns="" class="SimplePara">70SG 076</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">PI422001</p></td>
<td><p xmlns="" class="SimplePara">STUART</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Florida, US</p></td>
<td><p xmlns="" class="SimplePara">83</p></td>
<td><p xmlns="" class="SimplePara">PI642264</p></td>
<td><p xmlns="" class="SimplePara">70SG 078</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">PI422003</p></td>
<td><p xmlns="" class="SimplePara">PMT-785</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Florida, US</p></td>
<td><p xmlns="" class="SimplePara">84</p></td>
<td><p xmlns="" class="SimplePara">PI642265</p></td>
<td><p xmlns="" class="SimplePara">70SG 079</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">PI422016</p></td>
<td><p xmlns="" class="SimplePara">-</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Florida, US</p></td>
<td><p xmlns="" class="SimplePara">85</p></td>
<td><p xmlns="" class="SimplePara">PI642267</p></td>
<td><p xmlns="" class="SimplePara">70SG 081</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">PI431575</p></td>
<td><p xmlns="" class="SimplePara">KY1625</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Kentucky, US</p></td>
<td><p xmlns="" class="SimplePara">86</p></td>
<td><p xmlns="" class="SimplePara">PI642268</p></td>
<td><p xmlns="" class="SimplePara">70SG 082</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">PI442535</p></td>
<td><p xmlns="" class="SimplePara">156</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Belgium</p></td>
<td><p xmlns="" class="SimplePara">87</p></td>
<td><p xmlns="" class="SimplePara">PI642269</p></td>
<td><p xmlns="" class="SimplePara">71SG 001</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">PI469228</p></td>
<td><p xmlns="" class="SimplePara">Cave-in-Rock</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Illinois, US</p></td>
<td><p xmlns="" class="SimplePara">88</p></td>
<td><p xmlns="" class="SimplePara">PI642270</p></td>
<td><p xmlns="" class="SimplePara">71SG 002</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">PI476290</p></td>
<td><p xmlns="" class="SimplePara">T2086</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Carolina, US</p></td>
<td><p xmlns="" class="SimplePara">89</p></td>
<td><p xmlns="" class="SimplePara">PI642271</p></td>
<td><p xmlns="" class="SimplePara">71SG 004</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">PI476291</p></td>
<td><p xmlns="" class="SimplePara">T2099</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Maryland, US</p></td>
<td><p xmlns="" class="SimplePara">90</p></td>
<td><p xmlns="" class="SimplePara">PI642272</p></td>
<td><p xmlns="" class="SimplePara">71SG 005</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">PI414069</p></td>
<td><p xmlns="" class="SimplePara">BN-309-69</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">New York, US</p></td>
<td><p xmlns="" class="SimplePara">91</p></td>
<td><p xmlns="" class="SimplePara">PI642275</p></td>
<td><p xmlns="" class="SimplePara">71SG 008</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">PI414070</p></td>
<td><p xmlns="" class="SimplePara">BN-12323-69</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
<td><p xmlns="" class="SimplePara">92</p></td>
<td><p xmlns="" class="SimplePara">PI642276</p></td>
<td><p xmlns="" class="SimplePara">71SG 009</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">26</p></td>
<td><p xmlns="" class="SimplePara">PI476292</p></td>
<td><p xmlns="" class="SimplePara">T2100</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Arkansas, US</p></td>
<td><p xmlns="" class="SimplePara">93</p></td>
<td><p xmlns="" class="SimplePara">PI642277</p></td>
<td><p xmlns="" class="SimplePara">71SG 010</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">27</p></td>
<td><p xmlns="" class="SimplePara">PI476293</p></td>
<td><p xmlns="" class="SimplePara">T2101</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">New Jersey, US</p></td>
<td><p xmlns="" class="SimplePara">94</p></td>
<td><p xmlns="" class="SimplePara">PI642278</p></td>
<td><p xmlns="" class="SimplePara">71SG 011</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">28</p></td>
<td><p xmlns="" class="SimplePara">PI476294</p></td>
<td><p xmlns="" class="SimplePara">T4613</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Colorado, US</p></td>
<td><p xmlns="" class="SimplePara">95</p></td>
<td><p xmlns="" class="SimplePara">PI642279</p></td>
<td><p xmlns="" class="SimplePara">71SG 012</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">29</p></td>
<td><p xmlns="" class="SimplePara">PI476295</p></td>
<td><p xmlns="" class="SimplePara">T4614</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Colorado, US</p></td>
<td><p xmlns="" class="SimplePara">96</p></td>
<td><p xmlns="" class="SimplePara">PI642280</p></td>
<td><p xmlns="" class="SimplePara">71SG 013</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">30</p></td>
<td><p xmlns="" class="SimplePara">PI476296</p></td>
<td><p xmlns="" class="SimplePara">T16971</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Maryland, US</p></td>
<td><p xmlns="" class="SimplePara">97</p></td>
<td><p xmlns="" class="SimplePara">PI642281</p></td>
<td><p xmlns="" class="SimplePara">71SG 014</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">31</p></td>
<td><p xmlns="" class="SimplePara">PI476297</p></td>
<td><p xmlns="" class="SimplePara">Caddo</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Oklahoma,US</p></td>
<td><p xmlns="" class="SimplePara">98</p></td>
<td><p xmlns="" class="SimplePara">PI642282</p></td>
<td><p xmlns="" class="SimplePara">71SG 015</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">32</p></td>
<td><p xmlns="" class="SimplePara">PI477003</p></td>
<td><p xmlns="" class="SimplePara">Ncbraska 28</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Nebraska, US</p></td>
<td><p xmlns="" class="SimplePara">99</p></td>
<td><p xmlns="" class="SimplePara">PI642283</p></td>
<td><p xmlns="" class="SimplePara">71SG 016</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">33</p></td>
<td><p xmlns="" class="SimplePara">PI478002</p></td>
<td><p xmlns="" class="SimplePara">T6011</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">100</p></td>
<td><p xmlns="" class="SimplePara">PI642284</p></td>
<td><p xmlns="" class="SimplePara">71SG 017</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">34</p></td>
<td><p xmlns="" class="SimplePara">PI537588</p></td>
<td><p xmlns="" class="SimplePara">DACOTAH</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Oregon, US</p></td>
<td><p xmlns="" class="SimplePara">101</p></td>
<td><p xmlns="" class="SimplePara">PI642285</p></td>
<td><p xmlns="" class="SimplePara">71SG 018</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">35</p></td>
<td><p xmlns="" class="SimplePara">PI549094</p></td>
<td><p xmlns="" class="SimplePara">TRAILBLAZER</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Nebraska, US</p></td>
<td><p xmlns="" class="SimplePara">102</p></td>
<td><p xmlns="" class="SimplePara">PI642286</p></td>
<td><p xmlns="" class="SimplePara">71SG 019</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">36</p></td>
<td><p xmlns="" class="SimplePara">PI591824</p></td>
<td><p xmlns="" class="SimplePara">SHAWNEE</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Nebraska, US</p></td>
<td><p xmlns="" class="SimplePara">103</p></td>
<td><p xmlns="" class="SimplePara">PI642287</p></td>
<td><p xmlns="" class="SimplePara">71SG 020</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">37</p></td>
<td><p xmlns="" class="SimplePara">PI598136</p></td>
<td><p xmlns="" class="SimplePara">SUNBURST</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">South Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">104</p></td>
<td><p xmlns="" class="SimplePara">PI642288</p></td>
<td><p xmlns="" class="SimplePara">71SG 021</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">38</p></td>
<td><p xmlns="" class="SimplePara">PI642190</p></td>
<td><p xmlns="" class="SimplePara">FALCON</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">New Mexico, US</p></td>
<td><p xmlns="" class="SimplePara">105</p></td>
<td><p xmlns="" class="SimplePara">PI642289</p></td>
<td><p xmlns="" class="SimplePara">71SG 022</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">39</p></td>
<td><p xmlns="" class="SimplePara">PI642191</p></td>
<td><p xmlns="" class="SimplePara">SUMMER</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">South Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">106</p></td>
<td><p xmlns="" class="SimplePara">PI642290</p></td>
<td><p xmlns="" class="SimplePara">71SG 023</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">40</p></td>
<td><p xmlns="" class="SimplePara">PI642192</p></td>
<td><p xmlns="" class="SimplePara">PATHFINDER</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Nebraska, US</p></td>
<td><p xmlns="" class="SimplePara">107</p></td>
<td><p xmlns="" class="SimplePara">PI642291</p></td>
<td><p xmlns="" class="SimplePara">71SG 024</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">41</p></td>
<td><p xmlns="" class="SimplePara">PI642195</p></td>
<td><p xmlns="" class="SimplePara">70SG 003</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">108</p></td>
<td><p xmlns="" class="SimplePara">PI642292</p></td>
<td><p xmlns="" class="SimplePara">71SG 025</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">42</p></td>
<td><p xmlns="" class="SimplePara">PI642196</p></td>
<td><p xmlns="" class="SimplePara">70SG 004</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">109</p></td>
<td><p xmlns="" class="SimplePara">PI642293</p></td>
<td><p xmlns="" class="SimplePara">71SG 026</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">43</p></td>
<td><p xmlns="" class="SimplePara">PI642197</p></td>
<td><p xmlns="" class="SimplePara">70SG 005</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">110</p></td>
<td><p xmlns="" class="SimplePara">PI642294</p></td>
<td><p xmlns="" class="SimplePara">71SG 027</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">44</p></td>
<td><p xmlns="" class="SimplePara">PI642198</p></td>
<td><p xmlns="" class="SimplePara">70SG 006</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">111</p></td>
<td><p xmlns="" class="SimplePara">PI642295</p></td>
<td><p xmlns="" class="SimplePara">71SG 028</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">45</p></td>
<td><p xmlns="" class="SimplePara">PI642199</p></td>
<td><p xmlns="" class="SimplePara">70SG 007</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">112</p></td>
<td><p xmlns="" class="SimplePara">PI642296</p></td>
<td><p xmlns="" class="SimplePara">71SG 029</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">46</p></td>
<td><p xmlns="" class="SimplePara">PI642200</p></td>
<td><p xmlns="" class="SimplePara">70SG 008</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">113</p></td>
<td><p xmlns="" class="SimplePara">PI642297</p></td>
<td><p xmlns="" class="SimplePara">71SG 030</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">47</p></td>
<td><p xmlns="" class="SimplePara">PI642201</p></td>
<td><p xmlns="" class="SimplePara">70SG 010</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">114</p></td>
<td><p xmlns="" class="SimplePara">PI642298</p></td>
<td><p xmlns="" class="SimplePara">71SG 031</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">48</p></td>
<td><p xmlns="" class="SimplePara">PI642203</p></td>
<td><p xmlns="" class="SimplePara">70SG 012</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">115</p></td>
<td><p xmlns="" class="SimplePara">PI642299</p></td>
<td><p xmlns="" class="SimplePara">71SG 032</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">49</p></td>
<td><p xmlns="" class="SimplePara">PI642204</p></td>
<td><p xmlns="" class="SimplePara">70SG 013</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">116</p></td>
<td><p xmlns="" class="SimplePara">PI642301</p></td>
<td><p xmlns="" class="SimplePara">71SG 034</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">50</p></td>
<td><p xmlns="" class="SimplePara">PI642207</p></td>
<td><p xmlns="" class="SimplePara">70SG 016</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">117</p></td>
<td><p xmlns="" class="SimplePara">PI642302</p></td>
<td><p xmlns="" class="SimplePara">71SG 035</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">51</p></td>
<td><p xmlns="" class="SimplePara">PI642208</p></td>
<td><p xmlns="" class="SimplePara">70SG 017</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">118</p></td>
<td><p xmlns="" class="SimplePara">PI642303</p></td>
<td><p xmlns="" class="SimplePara">71SG 036</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">52</p></td>
<td><p xmlns="" class="SimplePara">PI642209</p></td>
<td><p xmlns="" class="SimplePara">70SG 018</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">119</p></td>
<td><p xmlns="" class="SimplePara">PI642304</p></td>
<td><p xmlns="" class="SimplePara">71SG 037</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">53</p></td>
<td><p xmlns="" class="SimplePara">PI642210</p></td>
<td><p xmlns="" class="SimplePara">70SG 019</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">120</p></td>
<td><p xmlns="" class="SimplePara">PI642305</p></td>
<td><p xmlns="" class="SimplePara">71SG 038</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">54</p></td>
<td><p xmlns="" class="SimplePara">PI642212</p></td>
<td><p xmlns="" class="SimplePara">70SG 021</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">121</p></td>
<td><p xmlns="" class="SimplePara">PI642306</p></td>
<td><p xmlns="" class="SimplePara">71SG 039</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">55</p></td>
<td><p xmlns="" class="SimplePara">PI642213</p></td>
<td><p xmlns="" class="SimplePara">70SG 022</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">122</p></td>
<td><p xmlns="" class="SimplePara">PI642307</p></td>
<td><p xmlns="" class="SimplePara">71SG 040</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">56</p></td>
<td><p xmlns="" class="SimplePara">PI642214</p></td>
<td><p xmlns="" class="SimplePara">70SG 023</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">123</p></td>
<td><p xmlns="" class="SimplePara">PI642309</p></td>
<td><p xmlns="" class="SimplePara">71SG 041B</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">57</p></td>
<td><p xmlns="" class="SimplePara">PI642217</p></td>
<td><p xmlns="" class="SimplePara">70SG 026</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">124</p></td>
<td><p xmlns="" class="SimplePara">PI642310</p></td>
<td><p xmlns="" class="SimplePara">71SG 042</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">58</p></td>
<td><p xmlns="" class="SimplePara">PI642218</p></td>
<td><p xmlns="" class="SimplePara">70SG 028</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">125</p></td>
<td><p xmlns="" class="SimplePara">PI642311</p></td>
<td><p xmlns="" class="SimplePara">71SG 043</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">59</p></td>
<td><p xmlns="" class="SimplePara">PI642229</p></td>
<td><p xmlns="" class="SimplePara">70SG 041</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">126</p></td>
<td><p xmlns="" class="SimplePara">PI642312</p></td>
<td><p xmlns="" class="SimplePara">71SG 044</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">60</p></td>
<td><p xmlns="" class="SimplePara">PI642232</p></td>
<td><p xmlns="" class="SimplePara">70SG 044</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">127</p></td>
<td><p xmlns="" class="SimplePara">PI648366</p></td>
<td><p xmlns="" class="SimplePara">70SG 053</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">61</p></td>
<td><p xmlns="" class="SimplePara">PI642233</p></td>
<td><p xmlns="" class="SimplePara">70SG 045</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">128</p></td>
<td><p xmlns="" class="SimplePara">PI648367</p></td>
<td><p xmlns="" class="SimplePara">70SG 070</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">62</p></td>
<td><p xmlns="" class="SimplePara">PI642234</p></td>
<td><p xmlns="" class="SimplePara">70SG 046</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">129</p></td>
<td><p xmlns="" class="SimplePara">PI657660</p></td>
<td><p xmlns="" class="SimplePara">Central lowa Germplasm</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Missouri, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">63</p></td>
<td><p xmlns="" class="SimplePara">PI642235</p></td>
<td><p xmlns="" class="SimplePara">70SG 047</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">130</p></td>
<td><p xmlns="" class="SimplePara">PI657661</p></td>
<td><p xmlns="" class="SimplePara">Blackwell</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">64</p></td>
<td><p xmlns="" class="SimplePara">PI642236</p></td>
<td><p xmlns="" class="SimplePara">70SG 048</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">131</p></td>
<td><p xmlns="" class="SimplePara">PI657662</p></td>
<td><p xmlns="" class="SimplePara">NEBRASKA28</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Nebraska, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">65</p></td>
<td><p xmlns="" class="SimplePara">PI642237</p></td>
<td><p xmlns="" class="SimplePara">70SG 049</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">132</p></td>
<td><p xmlns="" class="SimplePara">PI657663</p></td>
<td><p xmlns="" class="SimplePara">Blackwell</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">Kansas, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">66</p></td>
<td><p xmlns="" class="SimplePara">PI642242</p></td>
<td><p xmlns="" class="SimplePara">70SG 055</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">133</p></td>
<td><p xmlns="" class="SimplePara">PI657664</p></td>
<td><p xmlns="" class="SimplePara">GRENVILLE</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">New Mexico, US</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">67</p></td>
<td><p xmlns="" class="SimplePara">PI642243</p></td>
<td><p xmlns="" class="SimplePara">70SG 056</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">North Dakota, US</p></td>
<td><p xmlns="" class="SimplePara">134</p></td>
<td><p xmlns="" class="SimplePara">PI659345</p></td>
<td><p xmlns="" class="SimplePara">9086103</p></td>
<td><p xmlns="" class="SimplePara">UL</p></td>
<td><p xmlns="" class="SimplePara">New York, US</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Note: âULâ refers to upland ecotype switchgrass, âLLâ refers to lowland ecotype switchgrass</p>
                    </div></figure>
</div>
              </section><section id="Sec9" class="Section2 RenderAsSection2"><h3 class="Heading">DNA extraction and marker genotyping</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">Genomic DNA was extracted from tender leaves of each individual using a modified cetyltrimethylammonium bromide (CTAB) method [<span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>]. ISSR [designed by the University of British Columbia (UBC set No. 9)], EST-SSR [<span class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>], and SCoT primer [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>] sequences were aligned to the <em xmlns="" class="EmphasisTypeItalic">Panicum</em> reference genome using the bl2seq blast program in NCBI (<span class="ExternalRef"><a href="http://www.ncbi.nlm.nih.gov/BLAST/" target="_blank" class="ext-link"><span class="RefSource">www.ncbi.nlm.nih.gov/BLAST/</span></a></span>), which was designed to eliminate redundancies. Initially, four germplasms were used to screen marker primers [PI421999 (AM-314/MS-155), PI422006 (Alamo), PI642190 (Falcon), and PI642207 (70SG 016)]. The selected primers were synthesized by the Shanghai Sangon Biological Engineering Technology and Service Company (Shanghai, China) to genotype the collection.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">ISSR-PCR was carried out according to Li et al [<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>] as follows: the total reaction volume was 15 Î¼L and contained 20 ng template DNA, approximately 1.0 Î¼M primer, 7.5 Î¼L Mix (10âÃâPCR buffer, Mg<sup>2+</sup>, dNTPs; Tiangen Biotech, Beijing, China), and 1 U Taq polymerase. Amplifications were performed in a BioRad iCycle PCR machine (BIO-RAD Certified) under the following conditions: 95 Â°C for 5 min, followed by 35 cycles of the following: 95 Â°C for 45 s, 52â55 Â°C for 45 s, and 72 Â°C for 90 s. A final extension was conducted at 72 Â°C for 7 min. All PCR bands were visualized on 1 % polyacrylamide gel electrophoresis in 1âÃâTBE buffer. Silver staining was used to visualize the bands. The SCoT-PCR amplification reaction was conducted in a total volume of 15 Î¼L according to Collard and Mackill [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], and containing 10 ng template DNA, 0.8 mM primers, 1.2 mM MgCl<sub>2</sub>, 0.4 mM dNTPs, and 1 U Taq DNA polymerase (Tiangen Biotech, Beijing, China). PCR amplification had an initial denaturation step of 5 min at 95 Â°C, followed by 45 s at 95 Â°C, 45 s at 55 Â°C, 1.5 min at 72 Â°C for 30 cycles, and 7 min at 72 Â°C. PCR products were visualized following agarose gel (1.5 %) electrophoresis at 120Vfor 1.5 h in 1âÃâTBE buffer, followed by staining with GelRed (Tiangen Biotech, Beijing, China). The EST-SSR PCR consisted of a denaturation for 5 min at 94 Â°C then 35 cycles of 30 s at 94 Â°C, 30 s at 53â55 Â°C, and 2 min at 72 Â°C, with a final extension of 5 min at 72 Â°C [<span class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>] and products were visualized as described above.</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic variation and marker efficiency analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">For each marker, polymorphic alleles were scored as â1â for presence and â0â for absence at the same mobility, and this data was used to construct an original data matrix. Using Excel 2007 and POPGENE v.1.32 [<span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>], corresponding diversity parameters were estimated including: total number of bands (TNB), number of polymorphic bands (NPB), percentage of polymorphic bands (PPB), Neiâs (1973) gene diversity index (H), and Shannonâs information index (I). AMOVA v.1.55 was employed to reveal genetic variation among groups and within a population [<span class="CitationRef"><a href="#CR65" data-toggle="citation">65</a></span>]. The data input to POPGENE and AMOVA was produced by DCFA v.1.1 [<span class="CitationRef"><a href="#CR66" data-toggle="citation">66</a></span>].</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">The comparative efficiency of ISSRs, SCoTs, and EST-SSRs in these 134 switchgrass genotypes was assessed with MI. MI is the product of the EMR and the Ib<sub>av</sub> for the polymorphic markers [<span class="CitationRef"><a href="#CR67" data-toggle="citation">67</a></span>]. EMR is explained as the average number of polymorphic bands [<span class="CitationRef"><a href="#CR68" data-toggle="citation">68</a></span>]. Ib<sub>av</sub> is defined as:<div id="Equ1" class="Equation EquationMathjax">
<div class="EquationContent">$$ {\mathrm{Ib}}_{\mathrm{av}}=1/n{\displaystyle \sum }1-\left(2\mid 0.5-\mathrm{pi}\mid \right) $$</div> <div class="EquationNumber">(1)</div>
</div>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">pi is the proportion of the i-th amplification site, n represents the total number of amplification site.</p>
              </section><section id="Sec11" class="Section2 RenderAsSection2"><h3 class="Heading">Population structure analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">The model-based program STRUCTURE v.2.3.4 (<span class="ExternalRef"><a href="http://pritchardlab.stanford.edu/structure.html" target="_blank" class="ext-link"><span class="RefSource">http://pritchardlab.stanford.edu/structure.html</span></a></span>) [<span class="CitationRef"><a href="#CR69" data-toggle="citation">69</a></span>] was applied to assess the population structure of the 134 switchgrass genotypes with 51 ISSRs, SCoTs, and EST-SSRs. The number of subpopulations (K) was set from 1 to 10 based on admixture models and correlated band frequencies. With 5âÃâ10<sup>5</sup> Markov Chain Monte Carlo replications carried out for each run after a burn-in period of 10<sup>6</sup> iterations, 20 independent runs were performed per K. When there was a clear maximum value for posterior probability [LnP(D)] output in STRUCTURE, a K value was selected in the range of 1 to 10 subpopulations. The most probable K value was the ÎK, an ad hoc quantity related to the rate of change in LnP(D) between successive K inferred by STRUCTURE [<span class="CitationRef"><a href="#CR70" data-toggle="citation">70</a></span>]. The replication of K showing the maximum likelihood was applied to subdivide the genotypes into different groups with membership probabilitiesââ¥â0.75. Genotypes with less than 0.75 membership probabilities were assigned to an admixed group. Bar charts from the STRUCTURE data were displayed using Distruct 1.1 [<span class="CitationRef"><a href="#CR71" data-toggle="citation">71</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">A dendrogram was drawn using FreeTree and TreeView programs (<span class="ExternalRef"><a href="http://web.natur.cuni.cz/flegr/freetree.php" target="_blank" class="ext-link"><span class="RefSource">http://web.natur.cuni.cz/flegr/freetree.php</span></a></span>) [<span class="CitationRef"><a href="#CR72" data-toggle="citation">72</a></span>] based on Nei-Li genetic similarity coefficient with unweighted pair group method average (UPGMA) clustering.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par48" class="Para">To reveal relationships among the 134 switchgrass genotypes, a figure of two-dimensional scatterplots representing all of the genotypes was obtained for principal coordinate analysis (PCA) using NTsys-pc v.2.1 [<span class="CitationRef"><a href="#CR73" data-toggle="citation">73</a></span>]. All of the switchgrass individuals were analyzed to calculate MRD [<span class="CitationRef"><a href="#CR74" data-toggle="citation">74</a></span>]. The resulting genetic distance matrices were double-centered and used to obtain eigenvectors by the modules DCENTER and EIGEN using NTsys-pc.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Evaluation of linkage disequilibrium</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par49" class="Para">The significance of pairwise LD was evaluated using squared band-frequency correlations (r<sup>2</sup>) between all combinations of marker loci using the package TASSEL version 2.1 (<span class="ExternalRef"><a href="http://www.maizegenetics.net/bioinformatics" target="_blank" class="ext-link"><span class="RefSource">http://www.maizegenetics.net/bioinformatics</span></a></span>) [<span class="CitationRef"><a href="#CR75" data-toggle="citation">75</a></span>]. Rare bands with a band frequency of less than 5 % were removed to avoid biased evaluations of LD because of their large variances. Other pairs of bands were evaluated with a minor band frequency of at least 5 % (MAFââ¥â0.05) with the GDA 1.1 program [<span class="CitationRef"><a href="#CR76" data-toggle="citation">76</a></span>].</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par50" class="Para">The results of this study showed a great level of genetic variation among switchgrass germplasm. The switchgrass accessions were clearly divided into two groups containing upland and lowland ecotypes. For the first time, we revealed the extent of LD and population structure in switchgrass. The implications of these results in terms of utilizing association mapping for genes or QTL discovery in switchgrass were discussed. For further association mapping using a collection of switchgrass samples, we highly recommend the inclusion of more lowland ecotypes or the use of other nuclear markers in conjunction with ISSR, SCoT and EST-SSR.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry">
<dt class="Term"><dfn>CTAB: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">cetyltrimethylammonium bromide</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>EMR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">effective multiplex ratio</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>EST-SSR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">expressed sequence tag-simple sequence repeats marker</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>H: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Neiâs gene diversity index</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>I: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Shannonâs information index</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>Ib<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">av</sub>
                           : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">band informativeness</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>IRAP: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">inter-retrotransposon amplified polymorphism</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>ISSR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">inter simple sequence repeat marker</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>LD: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">linkage disequilibrium</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>LL: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">lowland ecotypes</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>MI: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">marker index</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>MRD: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">modified Rogers distances</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>NPB: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">number of polymorphic bands</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>PCA: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">principal coordinate analyses</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>PPB: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">percentage of polymorphic bands</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SCoT: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">start codon targeted marker</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SSR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">simple sequence repeat marker</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>TNB: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">total number of bands</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>UL: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">upland ecotypes</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>UPGMA: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">unweighted pair group method average</p>
                  </dd>
</dl>
</div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgments</h3>
<p xmlns="" class="SimplePara">This work was supported by the National High-Technology Research and Development Program (863 Program) of China (No. 2012AA101801-02), the National Natural Science Foundation of China (NSFC) (No. 31201845), the spring plan of Ministry of Education, and the Sichuan Agricultural University Students Innovation Plan (No. 121062603).</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3>
<p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authorsâ contributions</h3>
<p xmlns="" class="SimplePara">L-KH and X-QZ conceived the project and designed the experiments. YZ, X-MJ, and H-DY performed the experiment. H-DY, X-LW and YZ wrote the paper. BX and L-XZ revised the paper. All authors discussed the results and commented on the manuscript. All authors read and approved the final manuscript.</p></aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Grassland Science Department, Sichuan Agricultural University, Chengdu, 611130, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">IRTA. Centre de Recerca en AgrigenÃ²mica (CSIC-IRTA-UAB), Campus UAB â Edifici CRAG, Bellaterra - Cerdanyola del VallÃ¨s, Barcelona, 08193, Spain</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Guizhou Institute of Prataculture, Guiyang, 550006, PR China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4">
<span class="AffiliationNumber">(4)</span><div class="AffiliationText">College of Grassland Science, Nanjing Agricultural University, Nanjing, 210095, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen YP, Zhang LW, Qi JM, Chen H, Tao AF, Xu JT, et al. Genetic linkage map construction for white jute (<em class="EmphasisTypeItalic">Corchorus capsularis</em> L.) using SRAP, ISSR and RAPD markers. Plant Breed. 2014;133(6):777â81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/pbr.12205"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20construction%20for%20white%20jute%20%28Corchorus%20capsularis%20L.%29%20using%20SRAP%2C%20ISSR%20and%20RAPD%20markers&amp;author=YP.%20Chen&amp;author=LW.%20Zhang&amp;author=JM.%20Qi&amp;author=H.%20Chen&amp;author=AF.%20Tao&amp;author=JT.%20Xu&amp;journal=Plant%20Breed&amp;volume=133&amp;issue=6&amp;pages=777-81&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Singh A, Negi MS, Rajagopal J, Bhatia S, Tomar UK, Srivastava PS, et al. Assessment of genetic diversity in <em class="EmphasisTypeItalic">Azadirachta indica</em> using AFLP markers. Theor Appl Genet. 1999;99(1-2):272â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051232"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20in%20Azadirachta%20indica%20using%20AFLP%20markers&amp;author=A.%20Singh&amp;author=MS.%20Negi&amp;author=J.%20Rajagopal&amp;author=S.%20Bhatia&amp;author=UK.%20Tomar&amp;author=PS.%20Srivastava&amp;journal=Theor%20Appl%20Genet&amp;volume=99&amp;issue=1-2&amp;pages=272-9&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Joshi P, Dhawan V. Assessment of genetic fidelity of micropropagated <em class="EmphasisTypeItalic">Swertia chirayita</em> plantlets by ISSR marker assay. Biol Plant. 2007;51(1):22â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10535-007-0005-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20fidelity%20of%20micropropagated%20Swertia%20chirayita%20plantlets%20by%20ISSR%20marker%20assay&amp;author=P.%20Joshi&amp;author=V.%20Dhawan&amp;journal=Biol%20Plant&amp;volume=51&amp;issue=1&amp;pages=22-6&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Reddy MP, Sarla N, Siddiq EA. Inter simple sequence repeat (ISSR) polymorphism and its application in plant breeding. Euphytica. 2002;128(1):9â17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1020691618797"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inter%20simple%20sequence%20repeat%20%28ISSR%29%20polymorphism%20and%20its%20application%20in%20plant%20breeding&amp;author=MP.%20Reddy&amp;author=N.%20Sarla&amp;author=EA.%20Siddiq&amp;journal=Euphytica&amp;volume=128&amp;issue=1&amp;pages=9-17&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Collard BCY, Mackill DJ. Start codon targeted (SCoT) polymorphism: a simple, novel DNA marker technique for generating gene-targeted markers in plants. Plant Mol Biol Rep. 2009;27(1):86â93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11105-008-0060-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Start%20codon%20targeted%20%28SCoT%29%20polymorphism%3A%20a%20simple%2C%20novel%20DNA%20marker%20technique%20for%20generating%20gene-targeted%20markers%20in%20plants&amp;author=BCY.%20Collard&amp;author=DJ.%20Mackill&amp;journal=Plant%20Mol%20Biol%20Rep&amp;volume=27&amp;issue=1&amp;pages=86-93&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xiong FQ, Tang RH, Chen ZL, Pan LH, Zhuang WJ. SCoT: a novel gene targeted marker technique based on the translation start codon. Mol Plant Breed. 2009;7:635â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SCoT%3A%20a%20novel%20gene%20targeted%20marker%20technique%20based%20on%20the%20translation%20start%20codon&amp;author=FQ.%20Xiong&amp;author=RH.%20Tang&amp;author=ZL.%20Chen&amp;author=LH.%20Pan&amp;author=WJ.%20Zhuang&amp;journal=Mol%20Plant%20Breed&amp;volume=7&amp;pages=635-8&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luo C, He XH, Chen H, Ou SJ, Gao MP. Analysis of diversity and relationships among mango cultivars using Start Codon Targeted (SCoT) markers. Biochem Syst Ecol. 2010;38(6):1176â84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2010.11.004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20diversity%20and%20relationships%20among%20mango%20cultivars%20using%20Start%20Codon%20Targeted%20%28SCoT%29%20markers&amp;author=C.%20Luo&amp;author=XH.%20He&amp;author=H.%20Chen&amp;author=SJ.%20Ou&amp;author=MP.%20Gao&amp;journal=Biochem%20Syst%20Ecol&amp;volume=38&amp;issue=6&amp;pages=1176-84&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gupta PK, Rustgi S, Sharma S, Singh R, Kumar N, Balyan HS. Transferable EST-SSR markers for the study of polymorphism and genetic diversity in bread wheat. Mol Genet Genomics. 2003;270(4):315â23.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00438-003-0921-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14508680"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transferable%20EST-SSR%20markers%20for%20the%20study%20of%20polymorphism%20and%20genetic%20diversity%20in%20bread%20wheat&amp;author=PK.%20Gupta&amp;author=S.%20Rustgi&amp;author=S.%20Sharma&amp;author=R.%20Singh&amp;author=N.%20Kumar&amp;author=HS.%20Balyan&amp;journal=Mol%20Genet%20Genomics&amp;volume=270&amp;issue=4&amp;pages=315-23&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Varshney RK, Sigmund R, BÃ¶rner A, Korzun V, Stein N, Sorrells ME, et al. Interspecific transferability and comparative mapping of barley EST-SSR markers in wheat, rye and rice. Plant Sci. 2005;168(1):195â202.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plantsci.2004.08.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Interspecific%20transferability%20and%20comparative%20mapping%20of%20barley%20EST-SSR%20markers%20in%20wheat%2C%20rye%20and%20rice&amp;author=RK.%20Varshney&amp;author=R.%20Sigmund&amp;author=A.%20B%C3%B6rner&amp;author=V.%20Korzun&amp;author=N.%20Stein&amp;author=ME.%20Sorrells&amp;journal=Plant%20Sci&amp;volume=168&amp;issue=1&amp;pages=195-202&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qureshi SN, Saha S, Kantety RV, Jenkins JN, Saha S, editors. Molecular biology and physiology: EST-SSR: a new class of genetic markers in cotton. J Cotton Sci. 2004;8:112â23.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Qureshi%20SN%2C%20Saha%20S%2C%20Kantety%20RV%2C%20Jenkins%20JN%2C%20Saha%20S%2C%20editors.%20Molecular%20biology%20and%20physiology%3A%20EST-SSR%3A%20a%20new%20class%20of%20genetic%20markers%20in%20cotton.%20J%20Cotton%20Sci.%202004%3B8%3A112%E2%80%9323."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Varshney RK, Chabane K, Hendre PS, Aggarwal RK, Graner A. Comparative assessment of EST-SSR, EST-SNP and AFLP markers for evaluation of genetic diversity and conservation of genetic resources using wild, cultivated and elite barleys. Plant Sci. 2007;173(6):638â49.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plantsci.2007.08.010"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20assessment%20of%20EST-SSR%2C%20EST-SNP%20and%20AFLP%20markers%20for%20evaluation%20of%20genetic%20diversity%20and%20conservation%20of%20genetic%20resources%20using%20wild%2C%20cultivated%20and%20elite%20barleys&amp;author=RK.%20Varshney&amp;author=K.%20Chabane&amp;author=PS.%20Hendre&amp;author=RK.%20Aggarwal&amp;author=A.%20Graner&amp;journal=Plant%20Sci&amp;volume=173&amp;issue=6&amp;pages=638-49&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luo C, He XH, Chen H, Hu Y, Ou SJ. Genetic relationship and diversity of <em class="EmphasisTypeItalic">Mangifera indica</em> L.: revealed through SCoT analysis. Genet Resour Crop Evol. 2012;59(7):1505â15.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-011-9779-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20relationship%20and%20diversity%20of%20Mangifera%20indica%20L.%3A%20revealed%20through%20SCoT%20analysis&amp;author=C.%20Luo&amp;author=XH.%20He&amp;author=H.%20Chen&amp;author=Y.%20Hu&amp;author=SJ.%20Ou&amp;journal=Genet%20Resour%20Crop%20Evol&amp;volume=59&amp;issue=7&amp;pages=1505-15&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Simko I. Development of EST-SSR markers for the study of population structure in lettuce (<em class="EmphasisTypeItalic">Lactuca sativa</em> L.). J Hered. 2009;100(2):256â62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/esn072"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18796462"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20EST-SSR%20markers%20for%20the%20study%20of%20population%20structure%20in%20lettuce%20%28Lactuca%20sativa%20L.%29&amp;author=I.%20Simko&amp;journal=J%20Hered&amp;volume=100&amp;issue=2&amp;pages=256-62&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Camacho FJ, Liston A. Population structure and genetic diversity of <em class="EmphasisTypeItalic">Botrychium pumicola</em> (Ophioglossaceae) based on inter-simple sequence repeats (ISSR). Am J Bot. 2001;88(6):1065â70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2657089"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11410471"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20genetic%20diversity%20of%20Botrychium%20pumicola%20%28Ophioglossaceae%29%20based%20on%20inter-simple%20sequence%20repeats%20%28ISSR%29&amp;author=FJ.%20Camacho&amp;author=A.%20Liston&amp;journal=Am%20J%20Bot&amp;volume=88&amp;issue=6&amp;pages=1065-70&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">BjÃ¶rn B, Paulo MJ, Kowitwanich K, Sengers M, Visser RGF, van Eck HJ, et al. Population structure and linkage disequilibrium unravelled in tetraploid potato. Theor Appl Genet. 2010;121(6):1151â70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-010-1379-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20linkage%20disequilibrium%20unravelled%20in%20tetraploid%20potato&amp;author=B.%20Bj%C3%B6rn&amp;author=MJ.%20Paulo&amp;author=K.%20Kowitwanich&amp;author=M.%20Sengers&amp;author=RGF.%20Visser&amp;author=HJ.%20Eck&amp;journal=Theor%20Appl%20Genet&amp;volume=121&amp;issue=6&amp;pages=1151-70&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gulsen O, Uzun A, Canan I, Seday U, Canihos E. A new citrus linkage map based on SRAP, SSR, ISSR, POGP, RGA and RAPD markers. Euphytica. 2010;173(2):265â77.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-010-0146-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20new%20citrus%20linkage%20map%20based%20on%20SRAP%2C%20SSR%2C%20ISSR%2C%20POGP%2C%20RGA%20and%20RAPD%20markers&amp;author=O.%20Gulsen&amp;author=A.%20Uzun&amp;author=I.%20Canan&amp;author=U.%20Seday&amp;author=E.%20Canihos&amp;journal=Euphytica&amp;volume=173&amp;issue=2&amp;pages=265-77&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Flint-Garcia SA, Thornsberry JM, IV B. Structure of linkage disequilibrium in plants. Annu Rev Plant Biol. 2003;54(1):357â74.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.arplant.54.031902.134907"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14502995"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20in%20plants&amp;author=SA.%20Flint-Garcia&amp;author=JM.%20Thornsberry&amp;author=B.%20IV&amp;journal=Annu%20Rev%20Plant%20Biol&amp;volume=54&amp;issue=1&amp;pages=357-74&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thornsberry JM, Goodman MM, Doebley J, Kresovich S, Nielsen D, Buckler ES. Dwarf8 polymorphisms associate with variation in flowering time. Nat Genet. 2001;28(3):286â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/90135"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11431702"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Dwarf8%20polymorphisms%20associate%20with%20variation%20in%20flowering%20time&amp;author=JM.%20Thornsberry&amp;author=MM.%20Goodman&amp;author=J.%20Doebley&amp;author=S.%20Kresovich&amp;author=D.%20Nielsen&amp;author=ES.%20Buckler&amp;journal=Nat%20Genet&amp;volume=28&amp;issue=3&amp;pages=286-9&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mather KA, Caicedo AL, Polato NR, Olsen KM, McCouch S, Purugganan MD. The extent of linkage disequilibrium in rice (<em class="EmphasisTypeItalic">Oryza sativa</em> L.). Genet. 2007;177(4):2223â32.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.107.079616"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20extent%20of%20linkage%20disequilibrium%20in%20rice%20%28Oryza%20sativa%20L.%29&amp;author=KA.%20Mather&amp;author=AL.%20Caicedo&amp;author=NR.%20Polato&amp;author=KM.%20Olsen&amp;author=S.%20McCouch&amp;author=MD.%20Purugganan&amp;journal=Genet&amp;volume=177&amp;issue=4&amp;pages=2223-32&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rodriguez M, Rau D, OâSullivan D, Brown AHD, Papa R, Attene G. Genetic structure and linkage disequilibrium in landrace populations of barley in Sardinia. Theor Appl Genet. 2012;125(1):171â84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-012-1824-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22411093"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20linkage%20disequilibrium%20in%20landrace%20populations%20of%20barley%20in%20Sardinia&amp;author=M.%20Rodriguez&amp;author=D.%20Rau&amp;author=D.%20O%E2%80%99Sullivan&amp;author=AHD.%20Brown&amp;author=R.%20Papa&amp;author=G.%20Attene&amp;journal=Theor%20Appl%20Genet&amp;volume=125&amp;issue=1&amp;pages=171-84&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gerald N, Murray SC, Isakeit T, Park Y-S, Yan Y, Warburton ML, et al. Characterization of genetic diversity and linkage disequilibrium of <em class="EmphasisTypeItalic">ZmLOX4</em> and <em class="EmphasisTypeItalic">ZmLOX5</em> loci in maize. PLoS One. 2013;8(1), e53973.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0053973"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterization%20of%20genetic%20diversity%20and%20linkage%20disequilibrium%20of%20ZmLOX4%20and%20ZmLOX5%20loci%20in%20maize&amp;author=N.%20Gerald&amp;author=SC.%20Murray&amp;author=T.%20Isakeit&amp;author=Y-S.%20Park&amp;author=Y.%20Yan&amp;author=ML.%20Warburton&amp;journal=PLoS%20One&amp;volume=8&amp;issue=1&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saxena MS, Bajaj D, Kujur A, Das S, Badoni S, Kumar V, et al. Natural allelic diversity, genetic structure and linkage disequilibrium pattern in wild Chickpea. PLoS One. 2014;9(9), e107484.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0107484"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25222488"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4164632"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Natural%20allelic%20diversity%2C%20genetic%20structure%20and%20linkage%20disequilibrium%20pattern%20in%20wild%20Chickpea&amp;author=MS.%20Saxena&amp;author=D.%20Bajaj&amp;author=A.%20Kujur&amp;author=S.%20Das&amp;author=S.%20Badoni&amp;author=V.%20Kumar&amp;journal=PLoS%20One&amp;volume=9&amp;issue=9&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fiil A, Lenk I, Petersen K, Jensen CS, Nielsen KK, Schejbel B, et al. Nucleotide diversity and linkage disequilibrium of nine genes with putative effects on flowering time in perennial ryegrass (<em class="EmphasisTypeItalic">Lolium perenne</em> L.). Plant Sci. 2011;180(2):228â37.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plantsci.2010.08.015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21421365"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nucleotide%20diversity%20and%20linkage%20disequilibrium%20of%20nine%20genes%20with%20putative%20effects%20on%20flowering%20time%20in%20perennial%20ryegrass%20%28Lolium%20perenne%20L.%29&amp;author=A.%20Fiil&amp;author=I.%20Lenk&amp;author=K.%20Petersen&amp;author=CS.%20Jensen&amp;author=KK.%20Nielsen&amp;author=B.%20Schejbel&amp;journal=Plant%20Sci&amp;volume=180&amp;issue=2&amp;pages=228-37&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Branca A, Paape TD, Zhou P, Briskine R, Farmer AD, Mudge J, et al. Whole-genome nucleotide diversity, recombination, and linkage disequilibrium in the model legume Medicago truncatula. Proc Natl Acad Sci. 2011;108(42):E864â70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1104032108"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21949378"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3198318"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Whole-genome%20nucleotide%20diversity%2C%20recombination%2C%20and%20linkage%20disequilibrium%20in%20the%20model%20legume%20Medicago%20truncatula&amp;author=A.%20Branca&amp;author=TD.%20Paape&amp;author=P.%20Zhou&amp;author=R.%20Briskine&amp;author=AD.%20Farmer&amp;author=J.%20Mudge&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=108&amp;issue=42&amp;pages=E864-70&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Abdurakhmonov IY, Abdukarimov A. Application of association mapping to understanding the genetic diversity of plant germplasm resources. Int J Plant Genomics. 2008;2008.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Abdurakhmonov%20IY%2C%20Abdukarimov%20A.%20Application%20of%20association%20mapping%20to%20understanding%20the%20genetic%20diversity%20of%20plant%20germplasm%20resources.%20Int%20J%20Plant%20Genomics.%202008%3B2008."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard JK, Rosenberg NA. Use of unlinked genetic markers to detect population stratification in association studies. Am J Hum Genet. 1999;65(1):220â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/302449"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10364535"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1378093"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Use%20of%20unlinked%20genetic%20markers%20to%20detect%20population%20stratification%20in%20association%20studies&amp;author=JK.%20Pritchard&amp;author=NA.%20Rosenberg&amp;journal=Am%20J%20Hum%20Genet&amp;volume=65&amp;issue=1&amp;pages=220-8&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Buckler ES, Thornsberry JM. Plant molecular diversity and applications to genomics. Curr Opin Plant Biol. 2002;5(2):107â11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1369-5266(02)00238-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11856604"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Plant%20molecular%20diversity%20and%20applications%20to%20genomics&amp;author=ES.%20Buckler&amp;author=JM.%20Thornsberry&amp;journal=Curr%20Opin%20Plant%20Biol&amp;volume=5&amp;issue=2&amp;pages=107-11&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Garris AJ, McCOUCH SR, Kresovich S. Population structure and its effect on haplotype diversity and linkage disequilibrium surrounding the xa5 locus of rice (<em class="EmphasisTypeItalic">Oryza sativa</em> L.). Genet. 2003;165(2):759â69.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20its%20effect%20on%20haplotype%20diversity%20and%20linkage%20disequilibrium%20surrounding%20the%20xa5%20locus%20of%20rice%20%28Oryza%20sativa%20L.%29&amp;author=AJ.%20Garris&amp;author=SR.%20McCOUCH&amp;author=S.%20Kresovich&amp;journal=Genet&amp;volume=165&amp;issue=2&amp;pages=759-69&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gawenda I, SchrÃ¶der-Lorenz A, Debener T. Markers for ornamental traits in <em class="EmphasisTypeItalic">Phalaenopsis</em> orchids: population structure, linkage disequilibrium and association mapping. Mol Breed. 2012;30(1):305â16.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-011-9620-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Markers%20for%20ornamental%20traits%20in%20Phalaenopsis%20orchids%3A%20population%20structure%2C%20linkage%20disequilibrium%20and%20association%20mapping&amp;author=I.%20Gawenda&amp;author=A.%20Schr%C3%B6der-Lorenz&amp;author=T.%20Debener&amp;journal=Mol%20Breed&amp;volume=30&amp;issue=1&amp;pages=305-16&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sanderson MA, Reed RL, McLaughlin SB, Wullschleger SD, Conger BV, Parrish DJ, et al. Switchgrass as a sustainable bioenergy crop. Bioresour Technol. 1996;56(1):83â93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0960-8524(95)00176-X"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Switchgrass%20as%20a%20sustainable%20bioenergy%20crop&amp;author=MA.%20Sanderson&amp;author=RL.%20Reed&amp;author=SB.%20McLaughlin&amp;author=SD.%20Wullschleger&amp;author=BV.%20Conger&amp;author=DJ.%20Parrish&amp;journal=Bioresour%20Technol&amp;volume=56&amp;issue=1&amp;pages=83-93&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lewandowski I, Clifton-Brown JC, Scurlock JMO, Huisman W. <em class="EmphasisTypeItalic">Miscanthus</em>: European experience with a novel energy crop. Biomass Bioenerg. 2000;19(4):209â27.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0961-9534(00)00032-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Miscanthus%3A%20European%20experience%20with%20a%20novel%20energy%20crop&amp;author=I.%20Lewandowski&amp;author=JC.%20Clifton-Brown&amp;author=JMO.%20Scurlock&amp;author=W.%20Huisman&amp;journal=Biomass%20Bioenerg&amp;volume=19&amp;issue=4&amp;pages=209-27&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bouton JH. Molecular breeding of switchgrass for use as a biofuel crop. Curr Opin Genet Dev. 2007;17(6):553â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gde.2007.08.012"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17933511"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20breeding%20of%20switchgrass%20for%20use%20as%20a%20biofuel%20crop&amp;author=JH.%20Bouton&amp;journal=Curr%20Opin%20Genet%20Dev&amp;volume=17&amp;issue=6&amp;pages=553-8&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lassner M, Bedbrook J. Directed molecular evolution in plant improvement. Curr Opin Plant Biol. 2001;4(2):152â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1369-5266(00)00152-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11228439"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Directed%20molecular%20evolution%20in%20plant%20improvement&amp;author=M.%20Lassner&amp;author=J.%20Bedbrook&amp;journal=Curr%20Opin%20Plant%20Biol&amp;volume=4&amp;issue=2&amp;pages=152-6&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang W-B, Kim Y-H, Lee H-S, Kim K-Y, Deng X-P, Kwak S-S. Analysis of antioxidant enzyme activity during germination of alfalfa under salt and drought stresses. Plant Physiol Biochem. 2009;47(7):570â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plaphy.2009.02.009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19318268"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20antioxidant%20enzyme%20activity%20during%20germination%20of%20alfalfa%20under%20salt%20and%20drought%20stresses&amp;author=W-B.%20Wang&amp;author=Y-H.%20Kim&amp;author=H-S.%20Lee&amp;author=K-Y.%20Kim&amp;author=X-P.%20Deng&amp;author=S-S.%20Kwak&amp;journal=Plant%20Physiol%20Biochem&amp;volume=47&amp;issue=7&amp;pages=570-7&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ersoz ES, Wright MH, Pangilinan JL, Sheehan MJ, Tobias C, Casler MD et al. SNP discovery with EST and NextGen sequencing in switchgrass (Panicum virgatum L.). 2012.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Ersoz%20ES%2C%20Wright%20MH%2C%20Pangilinan%20JL%2C%20Sheehan%20MJ%2C%20Tobias%20C%2C%20Casler%20MD%20et%20al.%20SNP%20discovery%20with%20EST%20and%20NextGen%20sequencing%20in%20switchgrass%20%28Panicum%20virgatum%20L.%29.%202012."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang HF, Zong XX, Guan JP, Yang T, Sun XL, Ma Y, et al. Genetic diversity and relationship of global faba bean (<em class="EmphasisTypeItalic">Vicia faba</em> L.) germplasm revealed by ISSR markers. Theor Appl Genet. 2012;124(5):789â97.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-011-1750-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22204023"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20relationship%20of%20global%20faba%20bean%20%28Vicia%20faba%20L.%29%20germplasm%20revealed%20by%20ISSR%20markers&amp;author=HF.%20Wang&amp;author=XX.%20Zong&amp;author=JP.%20Guan&amp;author=T.%20Yang&amp;author=XL.%20Sun&amp;author=Y.%20Ma&amp;journal=Theor%20Appl%20Genet&amp;volume=124&amp;issue=5&amp;pages=789-97&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guo DL, Zhang JY, Liu CH. Genetic diversity in some grape varieties revealed by SCoT analyses. Mol Biol Rep. 2012;39(5):5307â13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11033-011-1329-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22170600"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20some%20grape%20varieties%20revealed%20by%20SCoT%20analyses&amp;author=DL.%20Guo&amp;author=JY.%20Zhang&amp;author=CH.%20Liu&amp;journal=Mol%20Biol%20Rep&amp;volume=39&amp;issue=5&amp;pages=5307-13&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lin L, Hu ZY, Ni S, Li JY, Qiu YX. Genetic diversity of <em class="EmphasisTypeItalic">Camellia japonica</em> (Theaceae), a species endangered to East Asia, detected by inter-simple sequence repeat (ISSR). Biochem Syst Ecol. 2013;50:199â206.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2013.03.049"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Camellia%20japonica%20%28Theaceae%29%2C%20a%20species%20endangered%20to%20East%20Asia%2C%20detected%20by%20inter-simple%20sequence%20repeat%20%28ISSR%29&amp;author=L.%20Lin&amp;author=ZY.%20Hu&amp;author=S.%20Ni&amp;author=JY.%20Li&amp;author=YX.%20Qiu&amp;journal=Biochem%20Syst%20Ecol&amp;volume=50&amp;pages=199-206&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ramu P, Billot C, Rami JF, Senthilvel S, Upadhyaya HD, Reddy LA, et al. Assessment of genetic diversity in the sorghum reference set using EST-SSR markers. Theor Appl Genet. 2013;126(8):2051â64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-013-2117-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23708149"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20in%20the%20sorghum%20reference%20set%20using%20EST-SSR%20markers&amp;author=P.%20Ramu&amp;author=C.%20Billot&amp;author=JF.%20Rami&amp;author=S.%20Senthilvel&amp;author=HD.%20Upadhyaya&amp;author=LA.%20Reddy&amp;journal=Theor%20Appl%20Genet&amp;volume=126&amp;issue=8&amp;pages=2051-64&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Narasimhamoorthy B, Saha MC, Swaller T, Bouton JH. Genetic diversity in switchgrass collections assessed by EST-SSR markers. Bioenerg Res. 2008;1(2):136â46.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12155-008-9011-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20switchgrass%20collections%20assessed%20by%20EST-SSR%20markers&amp;author=B.%20Narasimhamoorthy&amp;author=MC.%20Saha&amp;author=T.%20Swaller&amp;author=JH.%20Bouton&amp;journal=Bioenerg%20Res&amp;volume=1&amp;issue=2&amp;pages=136-46&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gunter LE, Tuskan GA, Wullschleger SD. Diversity among populations of switchgrass based on RAPD markers. Crop Sci. 1996;36(4):1017â22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1996.0011183X003600040034x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Diversity%20among%20populations%20of%20switchgrass%20based%20on%20RAPD%20markers&amp;author=LE.%20Gunter&amp;author=GA.%20Tuskan&amp;author=SD.%20Wullschleger&amp;journal=Crop%20Sci&amp;volume=36&amp;issue=4&amp;pages=1017-22&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Martinez-Reyna JM, Vogel KP. Incompatibility systems in switchgrass. Crop Sci. 2002;42(6):1800â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2002.1800"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Incompatibility%20systems%20in%20switchgrass&amp;author=JM.%20Martinez-Reyna&amp;author=KP.%20Vogel&amp;journal=Crop%20Sci&amp;volume=42&amp;issue=6&amp;pages=1800-5&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Casler MD, Stendal CA, Kapich L, Vogel KP. Genetic diversity, plant adaptation regions, and gene pools for switchgrass. Crop Sci. 2007;47(6):2261â73.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2006.12.0797"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%2C%20plant%20adaptation%20regions%2C%20and%20gene%20pools%20for%20switchgrass&amp;author=MD.%20Casler&amp;author=CA.%20Stendal&amp;author=L.%20Kapich&amp;author=KP.%20Vogel&amp;journal=Crop%20Sci&amp;volume=47&amp;issue=6&amp;pages=2261-73&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nageswara-Rao M, Hanson M, Agarwal S, Stewart Jr CN, Kwit C. Genetic diversity analysis of switchgrass (<em class="EmphasisTypeItalic">Panicum virgatum</em> L.) populations using microsatellites and chloroplast sequences. Agrofor Syst. 2014;88(5):823â34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10457-014-9728-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20analysis%20of%20switchgrass%20%28Panicum%20virgatum%20L.%29%20populations%20using%20microsatellites%20and%20chloroplast%20sequences&amp;author=M.%20Nageswara-Rao&amp;author=M.%20Hanson&amp;author=S.%20Agarwal&amp;author=CN.%20Stewart&amp;author=C.%20Kwit&amp;journal=Agrofor%20Syst&amp;volume=88&amp;issue=5&amp;pages=823-34&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hamrick JL, Godt MJW, Brown AHD, Clegg MT, Kahler AL, Weir BS. Allozyme diversity in plant species. Plant Popul Genet Breed Genet Res. 1990;43â63.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Hamrick%20JL%2C%20Godt%20MJW%2C%20Brown%20AHD%2C%20Clegg%20MT%2C%20Kahler%20AL%2C%20Weir%20BS.%20Allozyme%20diversity%20in%20plant%20species.%20Plant%20Popul%20Genet%20Breed%20Genet%20Res.%201990%3B43%E2%80%9363."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Milbourne D, Meyer R, Bradshaw JE, Baird E, Bonar N, Provan J, et al. Comparison of PCR-based marker systems for the analysis of genetic relationships in cultivated potato. Mol Breed. 1997;3(2):127â36.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1009633005390"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20PCR-based%20marker%20systems%20for%20the%20analysis%20of%20genetic%20relationships%20in%20cultivated%20potato&amp;author=D.%20Milbourne&amp;author=R.%20Meyer&amp;author=JE.%20Bradshaw&amp;author=E.%20Baird&amp;author=N.%20Bonar&amp;author=J.%20Provan&amp;journal=Mol%20Breed&amp;volume=3&amp;issue=2&amp;pages=127-36&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xiong FQ, Zhong RC, Han ZQ, Jiang J, He LQ, Zhuang WJ, et al. Start codon targeted polymorphism for evaluation of functional genetic variation and relationships in cultivated peanut (<em class="EmphasisTypeItalic">Arachis hypogaea</em> L.) genotypes. Mol Biol Rep. 2011;38(5):3487â94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11033-010-0459-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21104441"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Start%20codon%20targeted%20polymorphism%20for%20evaluation%20of%20functional%20genetic%20variation%20and%20relationships%20in%20cultivated%20peanut%20%28Arachis%20hypogaea%20L.%29%20genotypes&amp;author=FQ.%20Xiong&amp;author=RC.%20Zhong&amp;author=ZQ.%20Han&amp;author=J.%20Jiang&amp;author=LQ.%20He&amp;author=WJ.%20Zhuang&amp;journal=Mol%20Biol%20Rep&amp;volume=38&amp;issue=5&amp;pages=3487-94&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bhattacharyya P, Kumaria S, Kumar S, Tandon P. Start Codon Targeted (SCoT) marker reveals genetic diversity of <em class="EmphasisTypeItalic">Dendrobium nobile</em> Lindl., an endangered medicinal orchid species. Gene. 2013;529(1):21â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2013.07.096"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23939470"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Start%20Codon%20Targeted%20%28SCoT%29%20marker%20reveals%20genetic%20diversity%20of%20Dendrobium%20nobile%20Lindl.%2C%20an%20endangered%20medicinal%20orchid%20species&amp;author=P.%20Bhattacharyya&amp;author=S.%20Kumaria&amp;author=S.%20Kumar&amp;author=P.%20Tandon&amp;journal=Gene&amp;volume=529&amp;issue=1&amp;pages=21-6&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gao YH, Zhu YQ, Tong ZK, Xu ZY, Jiang XF, Huang CH. Analysis of genetic diversity and relationships among genus <em class="EmphasisTypeItalic">Lycoris</em> based on start codon targeted (SCoT) marker. Biochem Syst Ecol. 2014;57:221â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2014.08.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20and%20relationships%20among%20genus%20Lycoris%20based%20on%20start%20codon%20targeted%20%28SCoT%29%20marker&amp;author=YH.%20Gao&amp;author=YQ.%20Zhu&amp;author=ZK.%20Tong&amp;author=ZY.%20Xu&amp;author=XF.%20Jiang&amp;author=CH.%20Huang&amp;journal=Biochem%20Syst%20Ecol&amp;volume=57&amp;pages=221-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Alikhani L, Rahmani M-S, Shabanian N, Badakhshan H, Khadivi-Khub A. Genetic variability and structure of <em class="EmphasisTypeItalic">Quercus brantii</em> assessed by ISSR, IRAP and SCoT markers. Gene. 2014;552(1):176â83.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2014.09.034"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25241382"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variability%20and%20structure%20of%20Quercus%20brantii%20assessed%20by%20ISSR%2C%20IRAP%20and%20SCoT%20markers&amp;author=L.%20Alikhani&amp;author=M-S.%20Rahmani&amp;author=N.%20Shabanian&amp;author=H.%20Badakhshan&amp;author=A.%20Khadivi-Khub&amp;journal=Gene&amp;volume=552&amp;issue=1&amp;pages=176-83&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luo C, He XH, Chen H, Ou SJ, Gao MP, Brown JS, et al. Genetic diversity of mango cultivars estimated using SCoT and ISSR markers. Biochem Syst Ecol. 2011;39(4):676â84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2011.05.023"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20mango%20cultivars%20estimated%20using%20SCoT%20and%20ISSR%20markers&amp;author=C.%20Luo&amp;author=XH.%20He&amp;author=H.%20Chen&amp;author=SJ.%20Ou&amp;author=MP.%20Gao&amp;author=JS.%20Brown&amp;journal=Biochem%20Syst%20Ecol&amp;volume=39&amp;issue=4&amp;pages=676-84&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Friedmain N, Koller D. Being bayesian about network structure. Mach Learn. 2003;50:95â126.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1020249912095"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Being%20bayesian%20about%20network%20structure&amp;author=N.%20Friedmain&amp;author=D.%20Koller&amp;journal=Mach%20Learn&amp;volume=50&amp;pages=95-126&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Missaoui AM, Paterson AH, Bouton JH. Molecular markers for the classification of switchgrass (<em class="EmphasisTypeItalic">Panicum virgatum</em> L.) germplasm and to assess genetic diversity in three synthetic switchgrass populations. Genet Res Crop Evol. 2006;53(6):1291â302.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-005-3878-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20markers%20for%20the%20classification%20of%20switchgrass%20%28Panicum%20virgatum%20L.%29%20germplasm%20and%20to%20assess%20genetic%20diversity%20in%20three%20synthetic%20switchgrass%20populations&amp;author=AM.%20Missaoui&amp;author=AH.%20Paterson&amp;author=JH.%20Bouton&amp;journal=Genet%20Res%20Crop%20Evol&amp;volume=53&amp;issue=6&amp;pages=1291-302&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang SX, Su XJ, Haselkorn R, Gornicki P. Evolution of switchgrass (<em class="EmphasisTypeItalic">Panicum virgatum</em> L.) based on sequences of the nuclear gene encoding plastid acetyl-CoA carboxylase. Plant Sci. 2003;164(1):43â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0168-9452(02)00327-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%20of%20switchgrass%20%28Panicum%20virgatum%20L.%29%20based%20on%20sequences%20of%20the%20nuclear%20gene%20encoding%20plastid%20acetyl-CoA%20carboxylase&amp;author=SX.%20Huang&amp;author=XJ.%20Su&amp;author=R.%20Haselkorn&amp;author=P.%20Gornicki&amp;journal=Plant%20Sci&amp;volume=164&amp;issue=1&amp;pages=43-9&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zalapa JE, Price DL, Kaeppler SM, Tobias CM, Okada M, Casler MD. Hierarchical classification of switchgrass genotypes using SSR and chloroplast sequences: ecotypes, ploidies, gene pools, and cultivars. Theor Appl Genet. 2011;122(4):805â17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-010-1488-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21104398"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hierarchical%20classification%20of%20switchgrass%20genotypes%20using%20SSR%20and%20chloroplast%20sequences%3A%20ecotypes%2C%20ploidies%2C%20gene%20pools%2C%20and%20cultivars&amp;author=JE.%20Zalapa&amp;author=DL.%20Price&amp;author=SM.%20Kaeppler&amp;author=CM.%20Tobias&amp;author=M.%20Okada&amp;author=MD.%20Casler&amp;journal=Theor%20Appl%20Genet&amp;volume=122&amp;issue=4&amp;pages=805-17&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Todd J, Wu YQ, Wang Z, Samuels T. Genetic diversity in tetraploid switchgrass revealed by AFLP marker polymorphisms. Genet Mol Res. 2011;10(4):2976â86.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2011.November.29.8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22180031"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20tetraploid%20switchgrass%20revealed%20by%20AFLP%20marker%20polymorphisms&amp;author=J.%20Todd&amp;author=YQ.%20Wu&amp;author=Z.%20Wang&amp;author=T.%20Samuels&amp;journal=Genet%20Mol%20Res&amp;volume=10&amp;issue=4&amp;pages=2976-86&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sneath PHA, Sokal RR. Numerical taxonomy. The principles and practice of numerical classification. San Francisco: WH Freeman; 1973.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Sneath%20PHA%2C%20Sokal%20RR.%20Numerical%20taxonomy.%20The%20principles%20and%20practice%20of%20numerical%20classification.%20San%20Francisco%3A%20WH%20Freeman%3B%201973."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jin L, Bao JS. Progress on the trait-marker association analysis in plants. Mol Plant Breed (China). 2009;7(6):1048â63.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Progress%20on%20the%20trait-marker%20association%20analysis%20in%20plants&amp;author=L.%20Jin&amp;author=JS.%20Bao&amp;journal=Mol%20Plant%20Breed%20%28China%29&amp;volume=7&amp;issue=6&amp;pages=1048-63&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR59" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Remington DL, Thornsberry JM, Matsuoka Y, Wilson LM, Whitt SR, Doebley J, et al. Structure of linkage disequilibrium and phenotypic associations in the maize genome. Proc Natl Acad Sci. 2001;98(20):11479â84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.201394398"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11562485"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC58755"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20and%20phenotypic%20associations%20in%20the%20maize%20genome&amp;author=DL.%20Remington&amp;author=JM.%20Thornsberry&amp;author=Y.%20Matsuoka&amp;author=LM.%20Wilson&amp;author=SR.%20Whitt&amp;author=J.%20Doebley&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=98&amp;issue=20&amp;pages=11479-84&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR60" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu K, Goodman M, Muse S, Smith JS, Buckler E, Doebley J. Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites. Genet. 2003;165(4):2117â28.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20diversity%20among%20maize%20inbred%20lines%20as%20inferred%20from%20DNA%20microsatellites&amp;author=K.%20Liu&amp;author=M.%20Goodman&amp;author=S.%20Muse&amp;author=JS.%20Smith&amp;author=E.%20Buckler&amp;author=J.%20Doebley&amp;journal=Genet&amp;volume=165&amp;issue=4&amp;pages=2117-28&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR61" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saghai-Maroof MA, Soliman KM, Jorgensen RA, Allard RW. Ribosomal DNA spacer-length polymorphisms in barley: Mendelian inheritance, chromosomal location, and population dynamics. Proc Natl Acad Sci. 1984;81(24):8014â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.81.24.8014"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6096873"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC392284"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ribosomal%20DNA%20spacer-length%20polymorphisms%20in%20barley%3A%20Mendelian%20inheritance%2C%20chromosomal%20location%2C%20and%20population%20dynamics&amp;author=MA.%20Saghai-Maroof&amp;author=KM.%20Soliman&amp;author=RA.%20Jorgensen&amp;author=RW.%20Allard&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=81&amp;issue=24&amp;pages=8014-8&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR62" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Okada M, Lanzatella C, Tobias CM. Single-locus EST-SSR markers for characterization of population genetic diversity and structure across ploidy levels in switchgrass (<em class="EmphasisTypeItalic">Panicum virgatum</em> L.). Genet Resour Crop Evol. 2011;58(6):919â31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-010-9631-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Single-locus%20EST-SSR%20markers%20for%20characterization%20of%20population%20genetic%20diversity%20and%20structure%20across%20ploidy%20levels%20in%20switchgrass%20%28Panicum%20virgatum%20L.%29&amp;author=M.%20Okada&amp;author=C.%20Lanzatella&amp;author=CM.%20Tobias&amp;journal=Genet%20Resour%20Crop%20Evol&amp;volume=58&amp;issue=6&amp;pages=919-31&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR63" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li HY, Liu L, Lou YH, Hu T, Fu JM. Genetic diversity of Chinese natural bermudagrass (<em class="EmphasisTypeItalic">Cynodon dactylon</em>) germplasm using ISSR markers. Sci Hortic. 2011;127(4):555â61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.scienta.2010.12.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Chinese%20natural%20bermudagrass%20%28Cynodon%20dactylon%29%20germplasm%20using%20ISSR%20markers&amp;author=HY.%20Li&amp;author=L.%20Liu&amp;author=YH.%20Lou&amp;author=T.%20Hu&amp;author=JM.%20Fu&amp;journal=Sci%20Hortic&amp;volume=127&amp;issue=4&amp;pages=555-61&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR64" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yeh F, Yang R. POPGENE-for the analysis of genetic variation among and within populations using co-dominant and dominant markers. 2000.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=POPGENE-for%20the%20analysis%20of%20genetic%20variation%20among%20and%20within%20populations%20using%20co-dominant%20and%20dominant%20markers&amp;author=F.%20Yeh&amp;author=R.%20Yang&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR65" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Excoffier L, Smouse PE, Quattro JM. Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data. Genet. 1992;131(2):479â91.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20molecular%20variance%20inferred%20from%20metric%20distances%20among%20DNA%20haplotypes%3A%20application%20to%20human%20mitochondrial%20DNA%20restriction%20data&amp;author=L.%20Excoffier&amp;author=PE.%20Smouse&amp;author=JM.%20Quattro&amp;journal=Genet&amp;volume=131&amp;issue=2&amp;pages=479-91&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR66" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang F, Ge S. Data analysis in population genetics. I. Analysis of RAPD data with AMOVA. Chin Biodivers. 2001;10(4):438â44.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Data%20analysis%20in%20population%20genetics.%20I.%20Analysis%20of%20RAPD%20data%20with%20AMOVA&amp;author=F.%20Zhang&amp;author=S.%20Ge&amp;journal=Chin%20Biodivers&amp;volume=10&amp;issue=4&amp;pages=438-44&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR67" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li YX, Li SS, Li LH, Yang XM, Li XQ. Comparison of genetic diversity of twelve <em class="EmphasisTypeItalic">Elymus</em> species using ISSR and SSR markers. Sci Aric Sin. 2004;38(8):1522â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20genetic%20diversity%20of%20twelve%20Elymus%20species%20using%20ISSR%20and%20SSR%20markers&amp;author=YX.%20Li&amp;author=SS.%20Li&amp;author=LH.%20Li&amp;author=XM.%20Yang&amp;author=XQ.%20Li&amp;journal=Sci%20Aric%20Sin&amp;volume=38&amp;issue=8&amp;pages=1522-7&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR68" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Archak S, Gaikwad AB, Gautam D, Rao EVVB, Swamy KRM, Karihaloo JL. Comparative assessment of DNA fingerprinting techniques (RAPD, ISSR and AFLP) for genetic analysis of cashew (<em class="EmphasisTypeItalic">Anacardium occidentale</em> L.) accessions of India. Genome. 2003;46(3):362â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/g03-016"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12834051"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20assessment%20of%20DNA%20fingerprinting%20techniques%20%28RAPD%2C%20ISSR%20and%20AFLP%29%20for%20genetic%20analysis%20of%20cashew%20%28Anacardium%20occidentale%20L.%29%20accessions%20of%20India&amp;author=S.%20Archak&amp;author=AB.%20Gaikwad&amp;author=D.%20Gautam&amp;author=EVVB.%20Rao&amp;author=KRM.%20Swamy&amp;author=JL.%20Karihaloo&amp;journal=Genome&amp;volume=46&amp;issue=3&amp;pages=362-9&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR69" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hubisz MJ, Falush D, Stephens M, Pritchard JK. Inferring weak population structure with the assistance of sample group information. Mol Ecol Resour. 2009;9(5):1322â32.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1755-0998.2009.02591.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21564903"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518025"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inferring%20weak%20population%20structure%20with%20the%20assistance%20of%20sample%20group%20information&amp;author=MJ.%20Hubisz&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=JK.%20Pritchard&amp;journal=Mol%20Ecol%20Resour&amp;volume=9&amp;issue=5&amp;pages=1322-32&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR70" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14(8):2611â20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02553.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15969739"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;journal=Mol%20Ecol&amp;volume=14&amp;issue=8&amp;pages=2611-20&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR71" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rosenberg NA. DISTRUCT: a program for the graphical display of population structure. Mol Ecol Notes. 2004;4(1):137â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1471-8286.2003.00566.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DISTRUCT%3A%20a%20program%20for%20the%20graphical%20display%20of%20population%20structure&amp;author=NA.%20Rosenberg&amp;journal=Mol%20Ecol%20Notes&amp;volume=4&amp;issue=1&amp;pages=137-8&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR72" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hampl V, PavlÃ­cek A, Flegr J. Construction and bootstrap analysis of DNA fingerprinting-based phylogenetic trees with the freeware program FreeTree: application to trichomonad parasites. Int J Syst Evol Microbiol. 2001;51(3):731â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1099/00207713-51-3-731"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11411692"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20and%20bootstrap%20analysis%20of%20DNA%20fingerprinting-based%20phylogenetic%20trees%20with%20the%20freeware%20program%20FreeTree%3A%20application%20to%20trichomonad%20parasites&amp;author=V.%20Hampl&amp;author=A.%20Pavl%C3%ADcek&amp;author=J.%20Flegr&amp;journal=Int%20J%20Syst%20Evol%20Microbiol&amp;volume=51&amp;issue=3&amp;pages=731-5&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR73" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rohlf FJ. NTSYS-PC, numerical taxonomy system for the PC ExeterSoftware, Version 2.1. USA: Applied Biostatistics Inc Setauket; 2000.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NTSYS-PC%2C%20numerical%20taxonomy%20system%20for%20the%20PC%20ExeterSoftware%2C%20Version%202.1&amp;author=FJ.%20Rohlf&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR74" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wright S. Evolution and the genetics of populations, volume 3: experimental results and evolutionary deductions. Illinois: University of Chicago Press; 1984.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%20and%20the%20genetics%20of%20populations%2C%20volume%203%3A%20experimental%20results%20and%20evolutionary%20deductions&amp;author=S.%20Wright&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR75" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bradbury PJ, Zhang Z, Kroon DE, Casstevens TM, Ramdoss Y, Buckler ES. TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics. 2007;23(19):2633â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm308"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17586829"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=PJ.%20Bradbury&amp;author=Z.%20Zhang&amp;author=DE.%20Kroon&amp;author=TM.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=ES.%20Buckler&amp;journal=Bioinformatics&amp;volume=23&amp;issue=19&amp;pages=2633-5&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR76" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lewis PO, Zaykin D. Genetic data analysis: computer program for the analysis of allelic data. Available at <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://lewis.eeb.uconn.edu/lewishome/software.html" target="_blank" class="ext-link"><span class="RefSource">http://lewis.eeb.uconn.edu/lewishome/software.html</span></a></span>. 2001.</span><span class="Occurrences"></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Zhangetal" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Zhangetal">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Zhangetal"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© Zhang et al. 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0007-z" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-016-0007-z">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0007-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0007-z?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0007-z #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0007-z #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0007-z #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0007-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0007-z">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0007-z"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0007-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0007-z">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0007-z"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0007-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0007-z">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0007-z"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0007-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0007-z">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0007-z"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0007-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0007-z">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0007-z"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-016-0007-z/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-016-0007-z">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0007-z">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0007-z" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-016-0007-z&amp;volume=153&amp;issue=1&amp;title=Genetic+variation%2C+population+structure+and+linkage+disequilibrium+in+Switchgrass+with+ISSR%2C+SCoT+and+EST-SSR+markers&amp;author_name=Yu+Zhang&amp;start_page=1&amp;end_page=12">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-016-0007-z" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
