<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Genetic diversity and structure of Elymus tangutorum accessions from western China as unraveled by AFLP markers | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Understanding genetic diversity of wild plant germplasm and the relationships between ecogeographic and genetic characteristics may provide insights for better utilizing and conserving genetic resources. Elymus tangutorum (Nevski) Hand.-Mazz, a cool-season hexaploid perennial, is an important pasture bunchgrass species used for forages and grassland restoration in Qinghai-Tibet Plateau and northwest China. In this study, 27 E. tangutorum accessions from diverse origins of western China were evaluated using AFLP markers in an effort to delve into the genetic relationships among them. The effects of eco-environmental factors and geographical isolation on the genetic diversity and population structure were also elucidated. On account of 554 polymorphic fragments amplified with 14 primer combinations, the mean values of some marker parameters including polymorphic information content, resolving power and marker index were 0.2504, 14.10 and 23.07, respectively, validating the high efficiency and reliability of the markers selected. Genetic dissimilarity index values among accessions ranged from 0.1024 to 0.7137 with a mean of 0.2773. STRUCTURE, UPGMA clustering and PCoA analyses showed that all accessions could be divided into the three main clusters; however, this results do not exactly coincide with geographic groups. We found medium differentiation (FSTâ=â0.162) between Qinghai-Tibet Plateau (QTP) and Xinjiang (XJC), and high differentiation (FSTâ=â0.188) among three Bayesian subgroups. A significant correlation (râ=â0.312) between genetic and geographical distance was observed by Mantel test in the species level, while the weak correlation was detected between genetic and environmental distance for all accessions and most of geographical groups. In addition, a significant ecological influence of average annual precipitation on genetic distance was revealed in XJC group and the Bayesian subgroup A. This study indicates that AFLP technique are a useful tool to measure genetic diversity in E. tangutorum, showing that geographical and environmental factors (especially precipitation) together, play a crucial role in genetic differentiation patterns. These findings underline the importance of local adaptation in shaping patterns of genetic variability and population structure in E. tangutorum germplasm collected in Western China.">
<meta name="description" content="Understanding genetic diversity of wild plant germplasm and the relationships between ecogeographic and genetic characteristics may provide insights for better utilizing and conserving genetic resources. Elymus tangutorum (Nevski) Hand.-Mazz, a cool-season hexaploid perennial, is an important pasture bunchgrass species used for forages and grassland restoration in Qinghai-Tibet Plateau and northwest China. In this study, 27 E. tangutorum accessions from diverse origins of western China were evaluated using AFLP markers in an effort to delve into the genetic relationships among them. The effects of eco-environmental factors and geographical isolation on the genetic diversity and population structure were also elucidated. On account of 554 polymorphic fragments amplified with 14 primer combinations, the mean values of some marker parameters including polymorphic information content, resolving power and marker index were 0.2504, 14.10 and 23.07, respectively, validating the high efficiency and reliability of the markers selected. Genetic dissimilarity index values among accessions ranged from 0.1024 to 0.7137 with a mean of 0.2773. STRUCTURE, UPGMA clustering and PCoA analyses showed that all accessions could be divided into the three main clusters; however, this results do not exactly coincide with geographic groups. We found medium differentiation (FSTâ=â0.162) between Qinghai-Tibet Plateau (QTP) and Xinjiang (XJC), and high differentiation (FSTâ=â0.188) among three Bayesian subgroups. A significant correlation (râ=â0.312) between genetic and geographical distance was observed by Mantel test in the species level, while the weak correlation was detected between genetic and environmental distance for all accessions and most of geographical groups. In addition, a significant ecological influence of average annual precipitation on genetic distance was revealed in XJC group and the Bayesian subgroup A. This study indicates that AFLP technique are a useful tool to measure genetic diversity in E. tangutorum, showing that geographical and environmental factors (especially precipitation) together, play a crucial role in genetic differentiation patterns. These findings underline the importance of local adaptation in shaping patterns of genetic variability and population structure in E. tangutorum germplasm collected in Western China.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Genetic diversity and structure of Elymus tangutorum accessions from western China as unraveled by AFLP markers">
<meta name="dc.source" content="Hereditas 2019 156:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2019-01-29">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2019 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Understanding genetic diversity of wild plant germplasm and the relationships between ecogeographic and genetic characteristics may provide insights for better utilizing and conserving genetic resources. Elymus tangutorum (Nevski) Hand.-Mazz, a cool-season hexaploid perennial, is an important pasture bunchgrass species used for forages and grassland restoration in Qinghai-Tibet Plateau and northwest China. In this study, 27 E. tangutorum accessions from diverse origins of western China were evaluated using AFLP markers in an effort to delve into the genetic relationships among them. The effects of eco-environmental factors and geographical isolation on the genetic diversity and population structure were also elucidated. On account of 554 polymorphic fragments amplified with 14 primer combinations, the mean values of some marker parameters including polymorphic information content, resolving power and marker index were 0.2504, 14.10 and 23.07, respectively, validating the high efficiency and reliability of the markers selected. Genetic dissimilarity index values among accessions ranged from 0.1024 to 0.7137 with a mean of 0.2773. STRUCTURE, UPGMA clustering and PCoA analyses showed that all accessions could be divided into the three main clusters; however, this results do not exactly coincide with geographic groups. We found medium differentiation (FST = 0.162) between Qinghai-Tibet Plateau (QTP) and Xinjiang (XJC), and high differentiation (FST = 0.188) among three Bayesian subgroups. A significant correlation (r = 0.312) between genetic and geographical distance was observed by Mantel test in the species level, while the weak correlation was detected between genetic and environmental distance for all accessions and most of geographical groups. In addition, a significant ecological influence of average annual precipitation on genetic distance was revealed in XJC group and the Bayesian subgroup A. This study indicates that AFLP technique are a useful tool to measure genetic diversity in E. tangutorum, showing that geographical and environmental factors (especially precipitation) together, play a crucial role in genetic differentiation patterns. These findings underline the importance of local adaptation in shaping patterns of genetic variability and population structure in E. tangutorum germplasm collected in Western China.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2019-01-29">
<meta name="prism.volume" content="156">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="8">
<meta name="prism.copyright" content="2019 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0082-z">
<meta name="prism.doi" content="doi:10.1186/s41065-019-0082-z">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-019-0082-z">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0082-z">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Genetic diversity and structure of Elymus tangutorum accessions from western China as unraveled by AFLP markers">
<meta name="citation_volume" content="156">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/12">
<meta name="citation_online_date" content="2019/01/29">
<meta name="citation_firstpage" content="8">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-019-0082-z">
<meta name="DOI" content="10.1186/s41065-019-0082-z">
<meta name="citation_doi" content="10.1186/s41065-019-0082-z">
<meta name="dc.creator" content="Wen-Dan Wu">
<meta name="dc.creator" content="Wen-Hui Liu">
<meta name="dc.creator" content="Ming Sun">
<meta name="dc.creator" content="Ji-Qiong Zhou">
<meta name="dc.creator" content="Wei Liu">
<meta name="dc.creator" content="Cheng-Lin Zhang">
<meta name="dc.creator" content="Xing-Quan Zhang">
<meta name="dc.creator" content="Yan Peng">
<meta name="dc.creator" content="Lin-Kai Huang">
<meta name="dc.creator" content="Xiao Ma">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_title=The genomic system of classification as a guide to intergeneric hybridization with the perennial Triticeae; citation_inbook_title=Gene Manipulation in Plant Improvement; citation_publication_date=1984; citation_pages=209-279; citation_id=CR1; citation_author=DR Dewey; citation_publisher=Springer">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev.; citation_title=Levels and distribution of allozyme and RAPD variation in populations of Elymus fibrosus (Schrenk) Tzvel. (Poaceae); citation_author=O Díaz, GL Sun, B Salomon, RV Bothmer; citation_volume=47; citation_issue=1; citation_publication_date=2000; citation_pages=11-24; citation_doi=10.1023/A:1008719119756; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=Acta Phytotaxon Sin; citation_title=Studies on morphological evolution, classification and distribution of the Triticeae in China; citation_author=BZ Kuo, JS Wang, JH Li; citation_volume=23; citation_issue=3; citation_publication_date=1985; citation_pages=161-169; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=J Syst Evol; citation_title=Genome constitution of Elymus tangutorum (Poaceae: Triticeae) inferred from meiotic pairing behavior and genomic in situ hybridization; citation_author=CR Yang, HQ Zhang, FQ Zhao, XY Liu, X Fan, LN Sha, HY Kang, Y Wang, YH Zhou; citation_volume=53; citation_issue=6; citation_publication_date=2015; citation_pages=529-534; citation_doi=10.1111/jse.12155; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=Differentiation in the Elymus dahuricus complex (Poaceae): evidence from grain proteins, DNA, and crossability; citation_author=AV Agafonov, BR Baum, LG Bailey; citation_volume=135; citation_issue=2–3; citation_publication_date=2001; citation_pages=277-289; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=Chin Qinghai J Anim Vet Sci.; citation_title=Study on cultivation and domestication of Elymus tangutorum; citation_author=YL Wang; citation_volume=3; citation_issue=2; citation_publication_date=2012; citation_pages=1-2; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=Acta Phytotaxon Sin.; citation_title=Mating system evolution, resource allocation, and genetic diversity in plants; citation_author=DY Zhang, XH Jiang; citation_volume=25; citation_issue=2; citation_publication_date=2001; citation_pages=130-143; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Variation in agronomic and morphological traits among Russian wildrye accessions; citation_author=JD Berdahl, HF Mayland, KH Asay, PG Jefferson; citation_volume=39; citation_issue=6; citation_publication_date=1999; citation_pages=1890-1895; citation_doi=10.2135/cropsci1999.3961890x; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Pratacult Sci; citation_title=Identification of four Elymus sibiricus cultivar(strains)from northwest plateau of Sichuan using morphological traits and SSR markers; citation_author=ZH Guo, HY Yang, CL Zhang, M Sun, KX Fu, XQ Zhang, X Ma; citation_volume=33; citation_issue=9; citation_publication_date=2016; citation_pages=1718-1727; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Genetic diversity in Elymus caninus as revealed by isozyme, RAPD, and microsatellite markers; citation_author=GL Sun, O Díaz, BR Salomon BVon; citation_volume=42; citation_issue=3; citation_publication_date=1999; citation_pages=420; citation_doi=10.1139/g98-130; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Scientia Agri Sin; citation_title=Comparison of genetic diversity of twelvel Elymus species using ISSR and SSR markers; citation_author=Y Li, S Li, L Li, X Yang, X Li; citation_volume=38; citation_issue=08; citation_publication_date=2005; citation_pages=1522-1527; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Acta Pratacul Sin; citation_title=Genetic diversity of wild Elymus nutans germplasm detected by SRAP markers; citation_author=ZH Chen, JM Miao, JC Zhong, X Ma, SY Chen, XQ Zhang; citation_volume=30; citation_issue=1; citation_publication_date=2010; citation_pages=94-100; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Siberian wild rye ( Elymus sibiricus L.): Genetic diversity of germplasm determined using DNA fingerprinting and SCoT markers; citation_author=WG Xie, JC Zhang, XH Zhao, JQ Zhang, YR Wang; citation_volume=60; citation_publication_date=2015; citation_pages=186-192; citation_doi=10.1016/j.bse.2015.04.021; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Genes Genet Syst; citation_title=Genetic diversity of three Elymus species indigenous to Japan and East Asia (E. Tsukushiensis, E. Humidus and E. Dahuricus) detected by AFLP; citation_author=T Sasanuma, TR Endo, T Ban; citation_volume=77; citation_issue=6; citation_publication_date=2002; citation_pages=429-438; citation_doi=10.1266/ggs.77.429; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Cell Mol Biol Lett; citation_title=Morphological and AFLP variation of Elymus repens (L.) Gould (Poaceae); citation_author=M Szczepaniak, PT Cieślak EBednarek; citation_volume=7; citation_issue=2A; citation_publication_date=2002; citation_pages=547-558; citation_id=CR15">
<meta name="citation_reference" content="citation_title=Assessment of genetic variation in Elymus repens and E. hispidus using AFLP markers and its implications for intraspecific taxonomy; citation_inbook_title=Problems of grass biology; citation_publication_date=2003; citation_pages=271-286; citation_id=CR16; citation_author=M Szczepaniak; citation_author=E Cieślak; citation_publisher=W. Szafer Institute of Botany, Polish Academy of Sciences">
<meta name="citation_reference" content="citation_journal_title=Palaeogeogr Palaeocl; citation_title=Genetic structure of the Delphinium naviculare species group tracks Pleistocene climatic oscillations in the Tianshan Mountains, arid Central Asia; citation_author=HX Zhang, ML Zhang; citation_volume=353-355; citation_issue=5; citation_publication_date=2012; citation_pages=93-103; citation_doi=10.1016/j.palaeo.2012.07.013; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=Ecol Evol; citation_title=Influence of geography and environment on patterns of genetic differentiation in a widespread submerged macrophyte, Eurasian watermilfoil (Myriophyllum spicatum L., Haloragaceae); citation_author=ZG Wu, D Yu, X Li, XW Xu; citation_volume=6; citation_issue=2; citation_publication_date=2016; citation_pages=460-468; citation_doi=10.1002/ece3.1882; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Spatio-temporal genetic variation in populations of wild emmer wheat, Triticum turgidum ssp. dicoccoides , as revealed by AFLP analysis; citation_author=O Ozbek, E Millet, Y Anikster, M Arslan OFeldman; citation_volume=115; citation_issue=1; citation_publication_date=2007; citation_pages=19-26; citation_doi=10.1007/s00122-007-0536-y; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Plos One; citation_title=Highly structured genetic diversity of Bixa orellana var. urucurana, the wild ancestor of annatto, in Brazilian Amazonia; citation_author=G Dequigiovanni, SLF Ramos, A Alves-Pereira, EG Fabri, D Picanço-Rodrigues, CR Clement, EA Gepts PVeasey; citation_volume=13; citation_issue=6; citation_publication_date=2018; citation_pages=e0198593; citation_doi=10.1371/journal.pone.0198593; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Strong correlation of wild barley (Hordeum spontaneum) population structure with temperature and precipitation variation; citation_author=S Hübner, M HÖfken, E Oren, G Haseneyer, N Stein, A Graner, K Schmid, E Fridman; citation_volume=18; citation_issue=7; citation_publication_date=2009; citation_pages=1523-1536; citation_doi=10.1111/j.1365-294X.2009.04106.x; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Evolution of genome-phenome diversity under environmental stress; citation_author=E Nevo; citation_volume=98; citation_issue=11; citation_publication_date=2001; citation_pages=6233-6240; citation_doi=10.1073/pnas.101109298; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Genetic structure and ecogeographical adaptation in wild barley ( Hordeum chilense Roemer et Schultes) as revealed by microsatellite markers; citation_author=A Castillo, G Dorado, C Feuillet, P Sourdille PHernandez; citation_volume=10; citation_issue=1; citation_publication_date=2010; citation_pages=266; citation_doi=10.1186/1471-2229-10-266; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev.; citation_title=Genetic structure and eco-geographical adaptation of garlic landraces ( Allium sativum L.) in Iran; citation_author=S Shaaf, R Sharma, B Kilian, A Walther, H Özkan, B Karami EMohammadi; citation_volume=61; citation_issue=8; citation_publication_date=2014; citation_pages=1565-1580; citation_doi=10.1007/s10722-014-0131-4; citation_id=CR24">
<meta name="citation_reference" content="citation_journal_title=BioTechniques; citation_title=The CTAB-DNA precipitation method: a common mini-scale preparation of template DNA from phagemids, phages or plasmids suitable for sequencing; citation_author=SG Del, C Manfioletti GSchneider; citation_volume=7; citation_issue=5; citation_publication_date=1989; citation_pages=514-520; citation_id=CR25">
<meta name="citation_reference" content="citation_journal_title=Microbiology; citation_title=Evaluation of the DNA fingerprinting method AFLP as an new tool in bacterial taxonomy; citation_author=P Janssen, R Coopman, G Huys, J Swings, M Bleeker, P Vos, K Zabeau MKersters; citation_volume=142; citation_issue=7; citation_publication_date=1996; citation_pages=1881; citation_doi=10.1099/13500872-142-7-1881; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=AFLP: a new technique for DNA fingerprinting; citation_author=P Vos, R Hogers, M Bleeker, M Reijans, TVD Lee, M Hornes, A Friters, J Pot, M Paleman JKuiper; citation_volume=23; citation_issue=21; citation_publication_date=1995; citation_pages=4407; citation_doi=10.1093/nar/23.21.4407; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=AFLP markers reveal high polymorphic rates in ryegrasses (Lolium spp.); citation_author=I Roldanruiz, J Dendauw, B Evan, L Depicker AMde; citation_volume=6; citation_issue=2; citation_publication_date=2000; citation_pages=125-134; citation_doi=10.1023/A:1009680614564; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Mol Breeding.; citation_title=The comparison of RFLP, RAPD, AFLP and SSR (microsatellite) markers for germplasm analysis; citation_author=W Powell, M Morgante, C Andre, M Hanafey, J Vogel, A Tingey SRafalski; citation_volume=2; citation_issue=3; citation_publication_date=1996; citation_pages=225-238; citation_doi=10.1007/BF00564200; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=DNA fingerprinting in rice using oligonucleotide probes specific for simple repetitive DNA sequences; citation_author=W Ramakishana, MD Lagu, VS Gupta, PK Ranjekar; citation_volume=88; citation_issue=3–4; citation_publication_date=1994; citation_pages=402-406; citation_doi=10.1007/BF00223651; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Int J Syst Evol Microbiol; citation_title=Construction and bootstrap analysis of DNA fingerprinting-based phylogenetic trees with the freeware program FreeTree: application to trichomonad parasites; citation_author=V Hampl, A Pavlícek, J Flegr; citation_volume=51; citation_issue=Pt 3; citation_publication_date=2001; citation_pages=731; citation_doi=10.1099/00207713-51-3-731; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Farmers without borders genetic structuring in century old barley (Hordeum vulgare); citation_author=NEG Forsberg, J Russell, M Macaulay, MW Leino, J Hagenblad; citation_volume=114; citation_issue=2; citation_publication_date=2015; citation_pages=195; citation_doi=10.1038/hdy.2014.83; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; citation_author=G Evanno, S Regnaut, J Goudet; citation_volume=14; citation_issue=8; citation_publication_date=2005; citation_pages=2611-2620; citation_doi=10.1111/j.1365-294X.2005.02553.x; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure; citation_author=M Jakobsson, NA Rosenberg; citation_volume=23; citation_issue=14; citation_publication_date=2007; citation_pages=1801-1806; citation_doi=10.1093/bioinformatics/btm233; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=Ecol; citation_title=Distruct: a program for the graphical display of population structure. Mol; citation_author=NA Rosenberg; citation_volume=4; citation_issue=1; citation_publication_date=2004; citation_pages=137-138; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Notes; citation_title=Bayesian clustering algorithms ascertaining spatial population structure: a new computer program and a comparison study; citation_author=C Chibiao, D Eric, F Florence, F Olivier; citation_volume=7; citation_issue=5; citation_publication_date=2007; citation_pages=747-756; citation_doi=10.1111/j.1471-8286.2007.01769.x; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=Int J Mol Sci; citation_title=Comparison of Bayesian clustering and edge detection methods for inferring boundaries in landscape genetics; citation_author=T Safner, MP Miller, BH Mcrae, MJ Fortin, S Manel; citation_volume=12; citation_issue=2; citation_publication_date=2011; citation_pages=865-889; citation_doi=10.3390/ijms12020865; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=François. Forecasting changes in population genetic structure of alpine plants in response to global warming; citation_author=P Holderegger, O Taberlet; citation_volume=21; citation_issue=10; citation_publication_date=2012; citation_pages=2354-2368; citation_doi=10.1111/j.1365-294X.2012.05541.x; citation_id=CR38">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=GenAlEx 6.5: genetic analysis in excel. Population genetic software for teaching and research—an update; citation_author=R Peakall, PE Smouse; citation_volume=28; citation_issue=28; citation_publication_date=2012; citation_pages=2537-2539; citation_doi=10.1093/bioinformatics/bts460; citation_id=CR39">
<meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=IBD (isolation by distance): a program for analyses of isolation by distance; citation_author=AJ Bohonak; citation_volume=93; citation_issue=2; citation_publication_date=2002; citation_pages=153-154; citation_doi=10.1093/jhered/93.2.153; citation_id=CR40">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Genetic variation and geographical divergence in Elymus nutans Griseb. (Poaceae: Triticeae) from West China; citation_author=SY Chen, X Ma, XQ Zhang, ZH Chen; citation_volume=37; citation_issue=6; citation_publication_date=2009; citation_pages=716-722; citation_doi=10.1016/j.bse.2009.12.005; citation_id=CR41">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev; citation_title=Genetic diversity among Polish landraces of common oat (Avena sativa L.); citation_author=B Tarczyk; citation_volume=60; citation_issue=7; citation_publication_date=2013; citation_pages=2157-2169; citation_doi=10.1007/s10722-013-9984-1; citation_id=CR42">
<meta name="citation_reference" content="citation_journal_title=Genetica; citation_title=Genetic diversity and structure of Brazilian ginger germplasm (Zingiber officinale) revealed by AFLP markers; citation_author=EZ Blanco, MM Bajay, MV Siqueira, MI Zucchi, JB Pinheiro; citation_volume=144; citation_issue=6; citation_publication_date=2016; citation_pages=1-12; citation_doi=10.1007/s10709-016-9930-1; citation_id=CR43">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=The use of ISSR and RAPD markers for detecting DNA polymorphism, genotype identification and genetic diversity among barley cultivars with known origin; citation_author=M Fernández, A Figueiras, C Benito; citation_volume=104; citation_issue=5; citation_publication_date=2002; citation_pages=845-851; citation_doi=10.1007/s00122-001-0848-2; citation_id=CR44">
<meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Comparative assessment of EST-SSR, EST-SNP and AFLP markers for evaluation of genetic diversity and conservation of genetic resources using wild, cultivated and elite barleys; citation_author=RK Varshney, K Chabane, PS Hendre, RK Aggarwal, A Graner; citation_volume=173; citation_issue=6; citation_publication_date=2007; citation_pages=638-649; citation_doi=10.1016/j.plantsci.2007.08.010; citation_id=CR45">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev.; citation_title=AFLP fingerprinting of sesame (Sesamum indicum L.) cultivars: identification, genetic relationship and comparison of AFLP informativeness parameters; citation_author=H Laurentin, P Karlovsky; citation_volume=54; citation_issue=7; citation_publication_date=2007; citation_pages=1437-1446; citation_doi=10.1007/s10722-006-9128-y; citation_id=CR46">
<meta name="citation_reference" content="citation_journal_title=C R Biol; citation_title=AFLP assessment of genetic variability and relationships in an Asian wild germplasm collection of Dactylis glomerata L; citation_author=M Sun, CL Zhang, XQ Zhang, Y Fan, KX Fu, WD Wu, SQ Bai, JB Zhang, Y Peng, LK Huang; citation_volume=340; citation_issue=3; citation_publication_date=2017; citation_pages=145-155; citation_doi=10.1016/j.crvi.2016.12.003; citation_id=CR47">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=AFLP diversity within and among Hardinggrass populations; citation_author=MAR Mian, JC Zwonitzer, Y Chen, MC Saha, AA Hopkins; citation_volume=45; citation_issue=6; citation_publication_date=2005; citation_pages=2591-2597; citation_doi=10.2135/cropsci2005.04-0029; citation_id=CR48">
<meta name="citation_reference" content="citation_journal_title=Mol Biotechnol; citation_title=Application of ISSR markers to analyze molecular relationships in Iranian jasmine ( Jasminum spp.) accessions; citation_author=MG Ghehsareh, H Salehi, M Khosh-Khui, A Niazi; citation_volume=57; citation_issue=1; citation_publication_date=2015; citation_pages=65-74; citation_doi=10.1007/s12033-014-9802-9; citation_id=CR49">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Developing an appropriate strategy to assess genetic variability in plant germplasm collections; citation_author=JE Gilbert, RV Lewis, MJ Wilkinson, PDS Caligari; citation_volume=98; citation_issue=6–7; citation_publication_date=1999; citation_pages=1125-1131; citation_doi=10.1007/s001220051176; citation_id=CR50">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Assessment of genetic diversity of Siberian wild rye ( Elymus sibiricus L.) germplasms with variation of seed shattering and implication for future genetic improvement; citation_author=WG Xie, XH Zhao, JQ Zhang, YR Wang, WX Liu; citation_volume=58; citation_publication_date=2015; citation_pages=211-218; citation_doi=10.1016/j.bse.2014.12.006; citation_id=CR51">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Evaluating a germplasm collection of the cover crop hairy vetch for use in sustainable farming systems; citation_author=J Maul, S Mirsky, S Emche, T Devine; citation_volume=51; citation_issue=6; citation_publication_date=2011; citation_pages=2615; citation_doi=10.2135/cropsci2010.09.0561; citation_id=CR52">
<meta name="citation_reference" content="citation_journal_title=Molecules; citation_title=Potential of start codon targeted (SCoT) markers to estimate genetic diversity and relationships among Chinese Elymus sibiricus accessions; citation_author=JC Zhang, WG Xie, YR Wang, XH Zhao; citation_volume=20; citation_issue=4; citation_publication_date=2015; citation_pages=5987-6001; citation_doi=10.3390/molecules20045987; citation_id=CR53">
<meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Development and cross-species transferability of EST-SSR markers in Siberian wildrye (Elymus sibiricus L.) using Illumina sequencing; citation_author=Q Zhou, D Luo, L Ma, W Xie, Y Wang, Z Wang YLiu; citation_volume=6; citation_publication_date=2016; citation_pages=20549; citation_doi=10.1038/srep20549; citation_id=CR54">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev.; citation_title=Genetic diversity and relationships in Corchorus olitorius (Malvaceae s.L.) inferred from molecular and morphological data; citation_author=S Benor, FR Hammer KBlattner; citation_volume=59; citation_issue=6; citation_publication_date=2012; citation_pages=1125-1146; citation_doi=10.1007/s10722-011-9748-8; citation_id=CR55">
<meta name="citation_reference" content="citation_journal_title=Can J Plant Sci; citation_title=Assessment of genetic diversity and differentiation of Elymus nutans indigenous to Qinghai-Tibet plateau using simple sequence repeats markers; citation_author=SY Chen, XQ Zhang, X Ma, LK Huang; citation_volume=93; citation_issue=6; citation_publication_date=2013; citation_pages=1089-1096; citation_doi=10.4141/cjps2013-062; citation_id=CR56">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Assessing genetic diversity of Elymus sibiricus (Poaceae: Triticeae) populations from Qinghai-Tibet plateau by ISSR markers; citation_author=X Ma, XQ Zhang, YH Zhou, SQ Bai, W Liu; citation_volume=36; citation_issue=7; citation_publication_date=2008; citation_pages=514-522; citation_doi=10.1016/j.bse.2008.03.003; citation_id=CR57">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Comparison of different nuclear DNA markers for estimating intraspecific genetic diversity in plants; citation_author=H Nybom; citation_volume=13; citation_issue=5; citation_publication_date=2004; citation_pages=1143-1155; citation_doi=10.1111/j.1365-294X.2004.02141.x; citation_id=CR58">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev; citation_title=Relating morphologic and RAPD marker Varlation to collection site environment in wild populations of red clover (Trifolium Pratense L.); citation_author=SL Greene, M Gritsenko, G Vandemark; citation_volume=51; citation_issue=6; citation_publication_date=2004; citation_pages=643-653; citation_doi=10.1023/B:GRES.0000024655.48989.ab; citation_id=CR59">
<meta name="citation_reference" content="citation_journal_title=Chin Qinghai J Anim Vet Sci; citation_title=Effect of row spacing and seeding rate on the forage and seed yield of Elymus tagutorum; citation_author=YL Wang; citation_volume=45; citation_issue=3; citation_publication_date=2015; citation_pages=7-9; citation_id=CR60">
<meta name="citation_reference" content="citation_journal_title=Annu Rev Ecol Syst; citation_title=Ecological determinants of genetic structure in plant populations; citation_author=MD Loveless, JL Hamrick; citation_volume=15; citation_issue=1; citation_publication_date=1984; citation_pages=65-95; citation_doi=10.1146/annurev.es.15.110184.000433; citation_id=CR61">
<meta name="citation_reference" content="citation_journal_title=B; citation_title=Effects of life history traits on genetic diversity in plant species. Phil. Trans. Royal Soc. of London, Ser; citation_author=JL Hamrick, MJW Godt; citation_volume=351; citation_issue=1345; citation_publication_date=1996; citation_pages=1291-1298; citation_id=CR62">
<meta name="citation_reference" content="citation_title=Technical guidelines for the management of field and in vitro germplasm collections; citation_inbook_title=IPGRI Handbooks for Genebanks No. 7; citation_publication_date=2004; citation_pages=95-96; citation_id=CR63; citation_author=BM Reed; citation_author=F Engelmann; citation_author=ME Dulloo; citation_author=JMM Engels; citation_publisher=Bioversity International">
<meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Natural selection causes microscale allozyme diversity in wild barley and a lichen at 'Evolution Canyon', Mt. Carmel, Israel; citation_author=I Apelbaumelkaher; citation_volume=78; citation_issue=4; citation_publication_date=1997; citation_pages=373-382; citation_doi=10.1038/hdy.1997.60; citation_id=CR64">
<meta name="citation_author" content="Wen-Dan Wu">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Wen-Hui Liu">
<meta name="citation_author_institution" content="Key Laboratory of Superior Forage Germplasm in the Qinghai-Tibetan Plateau, Qinghai Academy of Animal Science and Veterinary Medicine, Xining, China">
<meta name="citation_author" content="Ming Sun">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Ji-Qiong Zhou">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Wei Liu">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Cheng-Lin Zhang">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Xing-Quan Zhang">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Yan Peng">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Lin-Kai Huang">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Xiao Ma">
<meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-019-0082-z',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-019-0082-z","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Elymus tangutorum;AFLPs;Genetic diversity;Geographic groups;Eco-environmental factors"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic diversity and structure of Elymus tangutorum accessions from western China as unraveled by AFLP markers","publishedAt":1548720000000,"publishedAtDate":"2019-01-29","author":["Wen-Dan Wu","Wen-Hui Liu","Ming Sun","Ji-Qiong Zhou","Wei Liu","Cheng-Lin Zhang","Xing-Quan Zhang","Yan Peng","Lin-Kai Huang","Xiao Ma"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"156","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0082-z">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0082-z">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Genetic diversity and structure of Elymus tangutorum accessions from western China as unraveled by AFLP markers">
<meta property="og:description" content="Understanding genetic diversity of wild plant germplasm and the relationships between ecogeographic and genetic characteristics may provide insights for better utilizing and conserving genetic resources. Elymus tangutorum (Nevski) Hand.-Mazz, a cool-season hexaploid perennial, is an important pasture bunchgrass species used for forages and grassland restoration in Qinghai-Tibet Plateau and northwest China. In this study, 27 E. tangutorum accessions from diverse origins of western China were evaluated using AFLP markers in an effort to delve into the genetic relationships among them. The effects of eco-environmental factors and geographical isolation on the genetic diversity and population structure were also elucidated. On account of 554 polymorphic fragments amplified with 14 primer combinations, the mean values of some marker parameters including polymorphic information content, resolving power and marker index were 0.2504, 14.10 and 23.07, respectively, validating the high efficiency and reliability of the markers selected. Genetic dissimilarity index values among accessions ranged from 0.1024 to 0.7137 with a mean of 0.2773. STRUCTURE, UPGMA clustering and PCoA analyses showed that all accessions could be divided into the three main clusters; however, this results do not exactly coincide with geographic groups. We found medium differentiation (FSTâ=â0.162) between Qinghai-Tibet Plateau (QTP) and Xinjiang (XJC), and high differentiation (FSTâ=â0.188) among three Bayesian subgroups. A significant correlation (râ=â0.312) between genetic and geographical distance was observed by Mantel test in the species level, while the weak correlation was detected between genetic and environmental distance for all accessions and most of geographical groups. In addition, a significant ecological influence of average annual precipitation on genetic distance was revealed in XJC group and the Bayesian subgroup A. This study indicates that AFLP technique are a useful tool to measure genetic diversity in E. tangutorum, showing that geographical and environmental factors (especially precipitation) together, play a crucial role in genetic differentiation patterns. These findings underline the importance of local adaptation in shaping patterns of genetic variability and population structure in E. tangutorum germplasm collected in Western China.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-019-0082-z;kwrd=Elymus tangutorum,AFLPs,Genetic diversity,Geographic groups,Eco-environmental factors;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-019-0082-z&amp;kwrd=Elymus%20tangutorum,AFLPs,Genetic%20diversity,Geographic%20groups,Eco-environmental%20factors&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-019-0082-z&amp;kwrd=Elymus%20tangutorum,AFLPs,Genetic%20diversity,Geographic%20groups,Eco-environmental%20factors&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec17" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic diversity and structure of <em xmlns="" class="EmphasisTypeItalic">Elymus tangutorum</em> accessions from western China as unraveled by AFLP markers</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true">
<ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wen-Dan Wu</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wen-Hui Liu</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ming Sun</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ji-Qiong Zhou</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wei Liu</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Cheng-Lin Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xing-Quan Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yan Peng</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lin-Kai Huang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiao Ma</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:maroar@126.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-4491-3528" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a>
</li>
</ul>
<div class="AffiliationLegend"><div class="AffiliationLegendLine">
<span class="EqualContributionSymbol">â </span>Contributed equally</div></div>
</div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2019</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">156</strong>:8</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-019-0082-z">https://doi.org/10.1186/s41065-019-0082-z</a></p>
<p class="Copyright">Â©  The Author(s) 2019</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>12 June 2018</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>17 January 2019</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">29 January 2019</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Understanding genetic diversity of wild plant germplasm and the relationships between ecogeographic and genetic characteristics may provide insights for better utilizing and conserving genetic resources. <em xmlns="" class="EmphasisTypeItalic">Elymus tangutorum</em> (Nevski) Hand.-Mazz, a cool-season hexaploid perennial, is an important pasture bunchgrass species used for forages and grassland restoration in Qinghai-Tibet Plateau and northwest China. In this study, 27 <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accessions from diverse origins of western China were evaluated using AFLP markers in an effort to delve into the genetic relationships among them. The effects of eco-environmental factors and geographical isolation on the genetic diversity and population structure were also elucidated.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">On account of 554 polymorphic fragments amplified with 14 primer combinations, the mean values of some marker parameters including polymorphic information content, resolving power and marker index were 0.2504, 14.10 and 23.07, respectively, validating the high efficiency and reliability of the markers selected. Genetic dissimilarity index values among accessions ranged from 0.1024 to 0.7137 with a mean of 0.2773. STRUCTURE, UPGMA clustering and PCoA analyses showed that all accessions could be divided into the three main clusters; however, this results do not exactly coincide with geographic groups. We found medium differentiation (<em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub>â=â0.162) between Qinghai-Tibet Plateau (QTP) and Xinjiang (XJC), and high differentiation (<em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub>â=â0.188) among three Bayesian subgroups. A significant correlation (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.312) between genetic and geographical distance was observed by Mantel test in the species level, while the weak correlation was detected between genetic and environmental distance for all accessions and most of geographical groups. In addition, a significant ecological influence of average annual precipitation on genetic distance was revealed in XJC group and the Bayesian subgroup A.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">This study indicates that AFLP technique are a useful tool to measure genetic diversity in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em>, showing that geographical and environmental factors (especially precipitation) together, play a crucial role in genetic differentiation patterns. These findings underline the importance of local adaptation in shaping patterns of genetic variability and population structure in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm collected in Western China.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Elymus tangutorum</em>
              </li>
<li class="c-keywords__item" itemprop="about">AFLPs</li>
<li class="c-keywords__item" itemprop="about">Genetic diversity</li>
<li class="c-keywords__item" itemprop="about">Geographic groups</li>
<li class="c-keywords__item" itemprop="about">Eco-environmental factors</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para"><em xmlns="" class="EmphasisTypeItalic">Elymus</em> Linn. is the largest and most widely distributed genus in the tribe Triticeae, with 150 species grown in temperate regions of the world [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. This genus has a closer phylogenetic relationship with some of the important cereal crops, such as wheat, barley, rye and triticale [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Therefore, it may serve as a valuable natural gene pool of desirable traits for the improvement of these crops [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Besides, many <em xmlns="" class="EmphasisTypeItalic">Elymus</em> species are also used as fodder grasses and ecological protection [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Elymus tangutorum</em> (Nevski) Hand.-Mazz, a perennial hexaploid species with the StYH genome (2nâ=â6xâ=â42), along with <em xmlns="" class="EmphasisTypeItalic">E. duhuricus</em> Turcz. ex Griseb., <em xmlns="" class="EmphasisTypeItalic">E. excelsus</em> Turcz. ex Griseb. and <em xmlns="" class="EmphasisTypeItalic">E. ivoroschilowii</em> Probat. constitute the <em xmlns="" class="EmphasisTypeItalic">E. dahuricus</em> complex [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> differs morphologically from <em xmlns="" class="EmphasisTypeItalic">E. dahuricus</em> Turcz. in the light of the short upward awns and its distribution in the Qinghai-Tibet Plateau (QTP), Xinjiang province of China and the Alpine regions of Central Asia. In QTP, <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> is widely used for the degraded grassland restoration, due to high productivity, drought resistance, cold tolerance, and adaptability [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. As one of the more important forage grasses in QTP, <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> has been widely studied. Most of studies mainly focused on the domestication, cultivation, phylogenesis and genome constitution [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Understanding genetic diversity of wild plant germplasm and the relationships between ecogeographic and genetic characteristics may provide insights for better utilizing and conserving genetic resources [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Investigation of germplasm diversity can be implemented via morphological and molecular means. Previous analyses based on agro-morphological characters and geographical origin indicated that a wide range of phenotypic divergence occurred among <em xmlns="" class="EmphasisTypeItalic">Elymus</em> accessions of ecotypes, and/or cultivars [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. DNA-based markers have been regarded as practical tools with high efficiency and wide genome coverage in illuminating the pattern of genetic diversity and phylogenetic relationships in plant germplasm resources, with various advantages over phenotypic traits such as being not subject to environmental influences, potential of unlimited numbers, and assay from any development stage. In <em xmlns="" class="EmphasisTypeItalic">Elymus</em> species, diverse PCR-based markers, employed in diversity and phylogeny studies, such as RAPD [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>], ISSR [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], SRAP [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>], ScoT [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], and AFLP [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>â<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] have become prevalent as any prior sequence information is not required. Among different marker systems available at present, AFLP method could explore variation throughout the entire genome, including both coding and non-coding DNA regions and may therefore genome-wide variation was allowed. Moreover, AFLP offers high reproducibility and multilocus polymorphisms simultaneously identified using a single assay. These characteristics make the AFLPs extremely appropriate for molecular characterization of germplasm collection. Although previous work has provided preliminary data to characterize genetic diversity between and within two <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> populations from Tibet province of China by AFLP analyses [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], until today there is no detailed study has been conducted for molecular characterization using a larger germplasm collection.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Genetic diversity in natural plant populations is held especially by the spatio-temporal environmental heterogeneity, i.e., genetic differentiation is strongly influenced through isolation by distance or local adaptation [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>â<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. To estimate the impact of environmental factors on the genetic diversity is indispensable to gain a deeper insight into such evolutionary forces [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>â<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Thus, the combination analysis of molecular markers and eco-geographical data can provide beneficial information for taking up suitable strategies for utilization and conservation of wild plant germplasms [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>].</p>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Here, AFLP markers were used to analyze, on a regional scale, genetic diversity and structure of 27 wild <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm accessions indigenous from two contrasting climatic zones of Western China, namely Qinghai-Tibet plateau and Xinjiang province (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). The goals of the current study were (1) to evaluate and compare the genetic diversity and population structure of <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm from different areas, and (2) to verify the potential influence of spatial and environmental factors on the patterns of detected population structure.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig1_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig1_HTML.png" alt="Fig. 1"></noscript></a>
                  </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">Geographical locations of the studied <em class="EmphasisTypeItalic">Elymus tangutorum</em> accessions in western China</p>
                    </div></figcaption></figure>
</div>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Plant materials and DNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Twenty seeds of each accession of <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> were germinated in vermiculite under identical conditions (25âÂ°C, 300 Î¼molsÂ·m<sup>2</sup>Â·s<sup>ââ1</sup>; 16-h photoperiod) in a plant growth chamber. Genomic DNA was extracted from bulked young leaves from five plants per accession employing a CTAB procedure [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. The quality and quantity of extracted DNA was assayed by 1% agarose gel electrophoresis and NanoDropÂ® spectrophotometer and was diluted with sterile distilled water to give a final concentration of 100âng/Î¼L.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">AFLP analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">AFLP fingerprinting was carried out following Vos et al. (1995) with minor modifications [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>], but the primers were labeled with 6-FAM fluorescent dye at the 5âend. Briefly, 300âng of total genomic DNA was digested using the two restrictive enzymes EcoRI and MseI, following ligation of proper adapters to the restricted DNA fragments. After ligation reactions, pre-selective amplification was performed with (EcoRI +â1)/(MseI +â1) primers. Then the products from the previous step were applied to selective amplification using (EcoRI +â3)/(MseI +â3) selective primers. Initially, 48 selective primer combinations were screened on six accessions, and 14 pairs were chosen based on total number of fragments detected, fragment robustness and polymorphism (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1). The automated separation and detection were fulfilled on an ABI 3730xl sequencer (Applied Biosystems) with the assistance of GeneScan 500 ROX as internal size standard. Genemarker 2.2 (SoftGenetics) was used to score fragments of 60â500âbp with a peak height above or equal to 100 reflective fluorescent units (RFUs). Then AFLP data from all primers was transferred into a binary matrix, scored asâ1â (presence of fragment) and â0â (absence of fragment).</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Data analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">The discriminatory power of each AFLP primer combination was evaluated by calculating polymorphic information content (PIC), marker index (MI), and resolving power (RP) with polymorphic bands only. The formula to calculate PIC value of each primer combination is: PIC<sub><em xmlns="" class="EmphasisTypeItalic">i</em></sub>â=â2 <em xmlns="" class="EmphasisTypeItalic">f</em><sub><em xmlns="" class="EmphasisTypeItalic">i</em></sub> (1âââ<em xmlns="" class="EmphasisTypeItalic">f</em><sub><em xmlns="" class="EmphasisTypeItalic">i</em></sub>), where PIC<sub><em xmlns="" class="EmphasisTypeItalic">i</em></sub> is the PIC of <em xmlns="" class="EmphasisTypeItalic">i</em>th marker, <em xmlns="" class="EmphasisTypeItalic">f</em><sub><em xmlns="" class="EmphasisTypeItalic">i</em></sub> stand for the frequency of the present fragments and 1âââ<em xmlns="" class="EmphasisTypeItalic">f</em><sub><em xmlns="" class="EmphasisTypeItalic">i</em></sub> represent the frequency of absent fragments [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Marker index (MI) was calculated as MIâ=âPIC Ã EMR, where EMR (effective multiple ratio) is defined as the product of the proportion of polymorphic loci (PP) and the number of polymorphic loci (NPF), namely the total number of polymorphic fragments per primer [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. The resolving power (RP) of each primer was estimated as <em xmlns="" class="EmphasisTypeItalic">Rp</em>â=ââIb, where Ib (fragment informativeness) defined as <em xmlns="" class="EmphasisTypeItalic">Ib</em>â=â1â2âÃâ|0.5âââp|, where, <em xmlns="" class="EmphasisTypeItalic">p</em> is the proportion of accessions in which the fragment is present [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">To understand the genetic relationship among all 27 accessions, pair-wise genetic distance (GD) were calculated on the basis of the Diceâs coefficient by NTSYS-pc v2.10 software. To obtain a more detailed view of the distribution of genetic variation within and between different geographical groups (XJ vs QTP), mean GD among accessions belonging to the same and/or different groups was also calculated. Based on GD matrix, the principal coordinate analysis (PCoA) and Unweighted Pair Group Method with Arithmetic (UPGMA) clustering analysis were performed. The robustness of the dendrograms was assessed with 1000 bootstrap iterations by Freetree software [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>]. The Bayesian model-based clustering method was applied to explore the population structure within the studied accessions or genetically diverse population (or groups) using the STRUCTURE software v.2.3.4 (<span class="ExternalRef"><a href="http://web.stanford.edu/group/pritchardlab/structure.html" target="_blank" class="ext-link"><span class="RefSource">http://web.stanford.edu/group/pritchardlab/structure.html</span></a></span>). Following the suggestion by Forsberg et al. [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>] for outcrossing species, the admixture model was adopted and the heterozygous loci were treated as missing data. Population structure analysis was performed on AFLP data of all 27 accessions without any prior classification based on the Bayesian model-based method. Evannoâs ad hoc ÎK statistic was used to determine the most probable clustering number of subgroups [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. We set the number of clusters (K) from 1 to 5 (10 times for each K) for accurate assignments of accessions. For each run, the burn-in period of 50,000 iterations and a run length of 500,000 MCMC replications were implemented. Considering that the indication of the most likely number of groups, namely LnP(D) usually plateaus or increases slightly after the âright Kâ is reached [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>], we used Structure Harvester online software (<span class="ExternalRef"><a href="http://taylor0.biology.ucla.edu/struct_harvest/" target="_blank" class="ext-link"><span class="RefSource">http://taylor0.biology.ucla.edu/struct_harvest/</span></a></span>) to detect optimum value of K based on ÎK method [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. The output files for the selected K based on ten independent runs were then processed in CLUMMP [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>] using the LargeKGreedy algorithm and the Gâpairwise similarities. Results were exported to DISTRUCT [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>] for better graphical presentation. We also executed spatial population analysis using TESS 2.3.1 [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>], which has been described as carrying high percentage of correct inference than other Bayesian clustering algorithm under certain conditions [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Geographical coordinates of each accession were contained in AFLP dataset as prior information, and the MCMC algorithm was run under BYM admixture model with 50,000 sweeps and10 000 burn-in. Fifty independent iteration analyses were performed at each value of clusters K (2~5).The value of K with the highest likelihood was determined according to the deviance information criterion (DIC), and the results of runs with 20% lowest DIC values were processed using CLUMMP and DISTRUCT, respectively, as did in the STRUCTURE analysis. Tess outputs was further visualized in a geographical perspective using an R script (POPSutilities) written by Flora Jay [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">To validate the genetic variation among and within inferred subgroups according to geographic origins or STRUCTURE results, analysis of molecular variance (AMOVA) based on PhiPT (analogue of <em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub>) value was using GenAlEx 6.5 was performed. The observed (Na) and effective number (Ne) of alleles, expected heterozygosity (H<sub>E</sub>), Shannonâs information index (I) [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>], and Neiâs genetic distance among geographic groups were also calculated by GenAlEx.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Based on geographical location of each accession, the corresponding environmental data (precipitation and temperature parameters) was obtained by DIVA-GIS (version 5.2.0.2, <span class="ExternalRef"><a href="http://swww.diva-gis.org/" target="_blank" class="ext-link"><span class="RefSource">http://swww.diva-gis.org/</span></a></span>). The geographic and environmental distance matrix was calculated based on the Euclidean distance between pairs of collection sites using NTSYS-pc v2.10. Correlation between geographical and genetic distances (Nei-Li coefficients) between accessions was measured by Mantel test with 9999 permutations using IBD (Isolation by Distance) software v1.53 [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. A similar analysis was conducted to compare the genetic distance and environmental distance (altitude, precipitation and temperature) for accessions in different geographic groups.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Marker informativeness</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">The AFLP fingerprinting was performed on 27 wild <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accessions using 14 primer combinations. A total of 554 scorable fragments were detected with a mean of 40 per assay unit, of which 509 (91.7%) were polymorphic and 70 (12.6%) were unique (Tables <span class="InternalRef"><a href="#Tab1">1</a></span> and <span class="InternalRef"><a href="#Tab2">2</a></span>). The size range of fragments scored across accessions was 60 to 500âbp. The total number of amplified fragments (TNF) per primer pair varied from 30 to 47 and the number of polymorphic fragments per primer pair (NPF) ranged from 26 (E46M51, E-ATT/M-CCA) to 41 (E43M55, E-ATA/M-CGA). The number of unique fragments per primer pair (NFU) ranged from one (for E46M60) (NUFâ=â1) to 10 (for E40M59 and E50M72). The percentage polymorphism varied from 80.95 to 97.56% with an average of 91.77% per primer combination. This result suggests that <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm from western China could be characterized well by above AFLP dataset retaining abundant variation. Polymorphism information content (PIC) of each primer pair being 0.5 for AFLP as the dominant marker, 38% of polymorphic loci (195) possessed PIC values above 0.25, implying high discriminatory power of AFLP markers for <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm. In addition, the marker index (MI) and resolving power (RP) values for individual primer pair were recorded. There were positive correlations between PIC, RP and MI (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.98, 0.90 and 0.87, respectively). The most discriminatory primer combinations among 14 primers were E-ATG/M-CTC (E45M60, PICâ=â0.3355, MIâ=â32.73, RPâ=â20.74) followed by E-AAC/M-GAC (E32M64, PICâ=â0.3099, MIâ=â29.32, RPâ=â17.63).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">The eco-geographic description on collection sites of <em class="EmphasisTypeItalic">Elymus tangutorum</em> accessions examined in AFLP analysis</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col char=".">
<col>
<col char=".">
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">No.</p></th>
<th><p xmlns="" class="SimplePara">Accessions</p></th>
<th><p xmlns="" class="SimplePara">Collecting regions</p></th>
<th><p xmlns="" class="SimplePara">Latitude (N)</p></th>
<th><p xmlns="" class="SimplePara">Longitude (E)</p></th>
<th char="."><p xmlns="" class="SimplePara">Altitude (m)</p></th>
<th><p xmlns="" class="SimplePara">Geographical group</p></th>
<th char="."><p xmlns="" class="SimplePara">Average annual precipitation (mm)</p></th>
<th char="."><p xmlns="" class="SimplePara">Average annual temperature (Â°C)</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et01</p></td>
<td><p xmlns="" class="SimplePara">PI 598528</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°49â²59â³</p></td>
<td><p xmlns="" class="SimplePara">86Â°16â²0â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1665</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">243</p></td>
<td char="."><p xmlns="" class="SimplePara">2.66</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 02</p></td>
<td><p xmlns="" class="SimplePara">PI 598530</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°55â²59â³N</p></td>
<td><p xmlns="" class="SimplePara">86Â°25â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1040</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">191</p></td>
<td char="."><p xmlns="" class="SimplePara">6.85</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 03</p></td>
<td><p xmlns="" class="SimplePara">PI 598537</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">44Â°8â²59â³N</p></td>
<td><p xmlns="" class="SimplePara">84Â°37â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1620</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">220</p></td>
<td char="."><p xmlns="" class="SimplePara">2.66</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 04</p></td>
<td><p xmlns="" class="SimplePara">PI 598552</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">44Â°47â²59â³N</p></td>
<td><p xmlns="" class="SimplePara">81Â°9â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1700</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">392</p></td>
<td char="."><p xmlns="" class="SimplePara">2.05</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 05</p></td>
<td><p xmlns="" class="SimplePara">PI 598554</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">45Â°1â²59â³N</p></td>
<td><p xmlns="" class="SimplePara">81Â°7â²0â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1300</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">343</p></td>
<td char="."><p xmlns="" class="SimplePara">3.73</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 06</p></td>
<td><p xmlns="" class="SimplePara">PI 598555</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">44Â°38â²59â³ N</p></td>
<td><p xmlns="" class="SimplePara">82Â°55â²0â³</p></td>
<td char="."><p xmlns="" class="SimplePara">300</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">105</p></td>
<td char="."><p xmlns="" class="SimplePara">8.21</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 07</p></td>
<td><p xmlns="" class="SimplePara">PI 598558</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">44Â°7â²0â³ N</p></td>
<td><p xmlns="" class="SimplePara">87Â°58â²0â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1680</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">199</p></td>
<td char="."><p xmlns="" class="SimplePara">7.24</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 08</p></td>
<td><p xmlns="" class="SimplePara">PI 598561</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°43â²59â³ N</p></td>
<td><p xmlns="" class="SimplePara">89Â°`27â²0â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1400</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">175</p></td>
<td char="."><p xmlns="" class="SimplePara">3.73</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 09</p></td>
<td><p xmlns="" class="SimplePara">PI 598564</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°40â²59â³ N</p></td>
<td><p xmlns="" class="SimplePara">89Â°`17â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1870</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">204</p></td>
<td char="."><p xmlns="" class="SimplePara">1.18</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 10</p></td>
<td><p xmlns="" class="SimplePara">PI 598566</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°47â²59â³ N</p></td>
<td><p xmlns="" class="SimplePara">87Â°`50â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1600</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">238</p></td>
<td char="."><p xmlns="" class="SimplePara">4.68</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 11</p></td>
<td><p xmlns="" class="SimplePara">PI 610896</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°49â²0â³ N</p></td>
<td><p xmlns="" class="SimplePara">86Â°20â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1510</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">234</p></td>
<td char="."><p xmlns="" class="SimplePara">3.60</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 12</p></td>
<td><p xmlns="" class="SimplePara">PI 619518</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">33Â°13â²59â³ N</p></td>
<td><p xmlns="" class="SimplePara">104Â°5â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2100</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">54</p></td>
<td char="."><p xmlns="" class="SimplePara">7.93</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 13</p></td>
<td><p xmlns="" class="SimplePara">PI 619524</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">42Â°37â²6â³ N</p></td>
<td><p xmlns="" class="SimplePara">94Â°26â²22â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1750</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">228</p></td>
<td char="."><p xmlns="" class="SimplePara">5.26</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 14</p></td>
<td><p xmlns="" class="SimplePara">PI 619570</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">30Â°28â²43â³ N</p></td>
<td><p xmlns="" class="SimplePara">101Â°29â²5â³</p></td>
<td char="."><p xmlns="" class="SimplePara">3500</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">698</p></td>
<td char="."><p xmlns="" class="SimplePara">6.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 15</p></td>
<td><p xmlns="" class="SimplePara">PI 619571</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">33Â°33â²37â³ N</p></td>
<td><p xmlns="" class="SimplePara">103Â°40â²10â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2480</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">773</p></td>
<td char="."><p xmlns="" class="SimplePara">4.73</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 16</p></td>
<td><p xmlns="" class="SimplePara">PI 619572</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">31Â°10â²0â³ N</p></td>
<td><p xmlns="" class="SimplePara">100Â°52â²59â³</p></td>
<td char="."><p xmlns="" class="SimplePara">3520</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">669</p></td>
<td char="."><p xmlns="" class="SimplePara">4.45</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 17</p></td>
<td><p xmlns="" class="SimplePara">PI 619588</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">31Â°34â²6â³ N</p></td>
<td><p xmlns="" class="SimplePara">98Â°32â²32â³</p></td>
<td char="."><p xmlns="" class="SimplePara">3600</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">652</p></td>
<td char="."><p xmlns="" class="SimplePara">6.40</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 18</p></td>
<td><p xmlns="" class="SimplePara">PI 636678</p></td>
<td><p xmlns="" class="SimplePara">Gansu, China</p></td>
<td><p xmlns="" class="SimplePara">34Â°33â²48â³ N</p></td>
<td><p xmlns="" class="SimplePara">102Â°33â²5â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2950</p></td>
<td><p xmlns="" class="SimplePara">GSC</p></td>
<td char="."><p xmlns="" class="SimplePara">560</p></td>
<td char="."><p xmlns="" class="SimplePara">5.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 19</p></td>
<td><p xmlns="" class="SimplePara">PI 639860</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">32Â°4â²42â³ N</p></td>
<td><p xmlns="" class="SimplePara">102Â°34â²22â³</p></td>
<td char="."><p xmlns="" class="SimplePara">3280</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">606</p></td>
<td char="."><p xmlns="" class="SimplePara">2.16</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 20</p></td>
<td><p xmlns="" class="SimplePara">PI 655139</p></td>
<td><p xmlns="" class="SimplePara">Xinjiang, China</p></td>
<td><p xmlns="" class="SimplePara">43Â°36â²55â³ N</p></td>
<td><p xmlns="" class="SimplePara">87Â°21â²11â³</p></td>
<td char="."><p xmlns="" class="SimplePara">1320</p></td>
<td><p xmlns="" class="SimplePara">XJC</p></td>
<td char="."><p xmlns="" class="SimplePara">841</p></td>
<td char="."><p xmlns="" class="SimplePara">2.78</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 21</p></td>
<td><p xmlns="" class="SimplePara">PI 655187</p></td>
<td><p xmlns="" class="SimplePara">Gansu, China</p></td>
<td><p xmlns="" class="SimplePara">35Â°11â²9â³ N</p></td>
<td><p xmlns="" class="SimplePara">102Â°29â²25â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2830</p></td>
<td><p xmlns="" class="SimplePara">GSC</p></td>
<td char="."><p xmlns="" class="SimplePara">563</p></td>
<td char="."><p xmlns="" class="SimplePara">1.95</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 22</p></td>
<td><p xmlns="" class="SimplePara">PI 655189</p></td>
<td><p xmlns="" class="SimplePara">Gansu, China</p></td>
<td><p xmlns="" class="SimplePara">34Â°54â²48â³ N</p></td>
<td><p xmlns="" class="SimplePara">102Â°51â²47â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2900</p></td>
<td><p xmlns="" class="SimplePara">GSC</p></td>
<td char="."><p xmlns="" class="SimplePara">586</p></td>
<td char="."><p xmlns="" class="SimplePara">2.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 23</p></td>
<td><p xmlns="" class="SimplePara">PI 655200</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">30Â°45â²6â³ N</p></td>
<td><p xmlns="" class="SimplePara">101Â°43â²31â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2320</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">694</p></td>
<td char="."><p xmlns="" class="SimplePara">8.54</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 24</p></td>
<td><p xmlns="" class="SimplePara">PI 655203</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">30Â°53â²12â³ N</p></td>
<td><p xmlns="" class="SimplePara">101Â°13â²40â³</p></td>
<td char="."><p xmlns="" class="SimplePara">3280</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">698</p></td>
<td char="."><p xmlns="" class="SimplePara">5.58</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 25</p></td>
<td><p xmlns="" class="SimplePara">PI 655204</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">31Â°11â²22â³ N</p></td>
<td><p xmlns="" class="SimplePara">100Â°51â²50â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2990</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">651</p></td>
<td char="."><p xmlns="" class="SimplePara">6.49</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 26</p></td>
<td><p xmlns="" class="SimplePara">PI 655212</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">29Â°59â²35â³ N</p></td>
<td><p xmlns="" class="SimplePara">101Â°53â²0â³</p></td>
<td char="."><p xmlns="" class="SimplePara">3110</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">930</p></td>
<td char="."><p xmlns="" class="SimplePara">4.09</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Et 27</p></td>
<td><p xmlns="" class="SimplePara">PI 655214</p></td>
<td><p xmlns="" class="SimplePara">Sichuan, China</p></td>
<td><p xmlns="" class="SimplePara">29Â°58â²59â³ N</p></td>
<td><p xmlns="" class="SimplePara">101Â°57â²24â³</p></td>
<td char="."><p xmlns="" class="SimplePara">2700</p></td>
<td><p xmlns="" class="SimplePara">SCC</p></td>
<td char="."><p xmlns="" class="SimplePara">905</p></td>
<td char="."><p xmlns="" class="SimplePara">5.26</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">XJC</em> Xinjiang province of China (XJC), <em class="EmphasisTypeItalic">SCC</em> Sichuan province of China, <em class="EmphasisTypeItalic">GSS</em> Gansu province of China</p>
                    </div></figure><figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Genetic diversity statistics of 14 polymorphic primer pairs used for AFLP fingerprinting of <em class="EmphasisTypeItalic">E. tangutorum</em> accessions</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col char=".">
<col char=".">
<col>
<col>
<col>
<col char=".">
<col char=".">
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Primer code</p></th>
<th char="."><p xmlns="" class="SimplePara">TNF</p></th>
<th char="."><p xmlns="" class="SimplePara">NPF</p></th>
<th char="."><p xmlns="" class="SimplePara">NUF</p></th>
<th><p xmlns="" class="SimplePara">PP(%)</p></th>
<th><p xmlns="" class="SimplePara">PIC</p></th>
<th><p xmlns="" class="SimplePara">MI</p></th>
<th char="."><p xmlns="" class="SimplePara">Rp</p></th>
<th char="."><p xmlns="" class="SimplePara">I</p></th>
<th char="."><p xmlns="" class="SimplePara">He</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E32M64</p></td>
<td char="."><p xmlns="" class="SimplePara">37</p></td>
<td char="."><p xmlns="" class="SimplePara">37</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">97.37</p></td>
<td><p xmlns="" class="SimplePara">0.3099</p></td>
<td><p xmlns="" class="SimplePara">29.32</p></td>
<td char="."><p xmlns="" class="SimplePara">17.63</p></td>
<td char="."><p xmlns="" class="SimplePara">0.427</p></td>
<td char="."><p xmlns="" class="SimplePara">0.277</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E40M59</p></td>
<td char="."><p xmlns="" class="SimplePara">42</p></td>
<td char="."><p xmlns="" class="SimplePara">39</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">92.86</p></td>
<td><p xmlns="" class="SimplePara">0.2120</p></td>
<td><p xmlns="" class="SimplePara">19.69</p></td>
<td char="."><p xmlns="" class="SimplePara">12.37</p></td>
<td char="."><p xmlns="" class="SimplePara">0.357</p></td>
<td char="."><p xmlns="" class="SimplePara">0.229</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E41M64</p></td>
<td char="."><p xmlns="" class="SimplePara">42</p></td>
<td char="."><p xmlns="" class="SimplePara">40</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">95.24</p></td>
<td><p xmlns="" class="SimplePara">0.2629</p></td>
<td><p xmlns="" class="SimplePara">25.04</p></td>
<td char="."><p xmlns="" class="SimplePara">14.59</p></td>
<td char="."><p xmlns="" class="SimplePara">0.409</p></td>
<td char="."><p xmlns="" class="SimplePara">0.267</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E42M54</p></td>
<td char="."><p xmlns="" class="SimplePara">30</p></td>
<td char="."><p xmlns="" class="SimplePara">28</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">93.33</p></td>
<td><p xmlns="" class="SimplePara">0.2253</p></td>
<td><p xmlns="" class="SimplePara">21.03</p></td>
<td char="."><p xmlns="" class="SimplePara">9.63</p></td>
<td char="."><p xmlns="" class="SimplePara">0.428</p></td>
<td char="."><p xmlns="" class="SimplePara">0.280</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E42M56</p></td>
<td char="."><p xmlns="" class="SimplePara">38</p></td>
<td char="."><p xmlns="" class="SimplePara">36</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">94.74</p></td>
<td><p xmlns="" class="SimplePara">0.2457</p></td>
<td><p xmlns="" class="SimplePara">23.28</p></td>
<td char="."><p xmlns="" class="SimplePara">13.11</p></td>
<td char="."><p xmlns="" class="SimplePara">0.346</p></td>
<td char="."><p xmlns="" class="SimplePara">0.222</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E43M55</p></td>
<td char="."><p xmlns="" class="SimplePara">47</p></td>
<td char="."><p xmlns="" class="SimplePara">41</p></td>
<td char="."><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">87.23</p></td>
<td><p xmlns="" class="SimplePara">0.2462</p></td>
<td><p xmlns="" class="SimplePara">21.48</p></td>
<td char="."><p xmlns="" class="SimplePara">16.15</p></td>
<td char="."><p xmlns="" class="SimplePara">0.391</p></td>
<td char="."><p xmlns="" class="SimplePara">0.261</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E45M60</p></td>
<td char="."><p xmlns="" class="SimplePara">41</p></td>
<td char="."><p xmlns="" class="SimplePara">40</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">97.56</p></td>
<td><p xmlns="" class="SimplePara">0.3355</p></td>
<td><p xmlns="" class="SimplePara">32.73</p></td>
<td char="."><p xmlns="" class="SimplePara">20.74</p></td>
<td char="."><p xmlns="" class="SimplePara">0.449</p></td>
<td char="."><p xmlns="" class="SimplePara">0.296</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E46M51</p></td>
<td char="."><p xmlns="" class="SimplePara">31</p></td>
<td char="."><p xmlns="" class="SimplePara">26</p></td>
<td char="."><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">83.87</p></td>
<td><p xmlns="" class="SimplePara">0.2189</p></td>
<td><p xmlns="" class="SimplePara">18.36</p></td>
<td char="."><p xmlns="" class="SimplePara">9.56</p></td>
<td char="."><p xmlns="" class="SimplePara">0.335</p></td>
<td char="."><p xmlns="" class="SimplePara">0.218</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E46M60</p></td>
<td char="."><p xmlns="" class="SimplePara">42</p></td>
<td char="."><p xmlns="" class="SimplePara">34</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">80.95</p></td>
<td><p xmlns="" class="SimplePara">0.1973</p></td>
<td><p xmlns="" class="SimplePara">15.97</p></td>
<td char="."><p xmlns="" class="SimplePara">11.04</p></td>
<td char="."><p xmlns="" class="SimplePara">0.364</p></td>
<td char="."><p xmlns="" class="SimplePara">0.238</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E46M72</p></td>
<td char="."><p xmlns="" class="SimplePara">40</p></td>
<td char="."><p xmlns="" class="SimplePara">39</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">97.50</p></td>
<td><p xmlns="" class="SimplePara">0.3037</p></td>
<td><p xmlns="" class="SimplePara">29.61</p></td>
<td char="."><p xmlns="" class="SimplePara">17.63</p></td>
<td char="."><p xmlns="" class="SimplePara">0.420</p></td>
<td char="."><p xmlns="" class="SimplePara">0.269</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E47M65</p></td>
<td char="."><p xmlns="" class="SimplePara">41</p></td>
<td char="."><p xmlns="" class="SimplePara">39</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">95.12</p></td>
<td><p xmlns="" class="SimplePara">0.2614</p></td>
<td><p xmlns="" class="SimplePara">24.86</p></td>
<td char="."><p xmlns="" class="SimplePara">15.04</p></td>
<td char="."><p xmlns="" class="SimplePara">0.376</p></td>
<td char="."><p xmlns="" class="SimplePara">0.239</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E48M54</p></td>
<td char="."><p xmlns="" class="SimplePara">45</p></td>
<td char="."><p xmlns="" class="SimplePara">39</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">86.67</p></td>
<td><p xmlns="" class="SimplePara">0.2198</p></td>
<td><p xmlns="" class="SimplePara">19.05</p></td>
<td char="."><p xmlns="" class="SimplePara">13.26</p></td>
<td char="."><p xmlns="" class="SimplePara">0.414</p></td>
<td char="."><p xmlns="" class="SimplePara">0.273</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E50M59</p></td>
<td char="."><p xmlns="" class="SimplePara">40</p></td>
<td char="."><p xmlns="" class="SimplePara">34</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">85.00</p></td>
<td><p xmlns="" class="SimplePara">0.2347</p></td>
<td><p xmlns="" class="SimplePara">19.95</p></td>
<td char="."><p xmlns="" class="SimplePara">13.63</p></td>
<td char="."><p xmlns="" class="SimplePara">0.436</p></td>
<td char="."><p xmlns="" class="SimplePara">0.295</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E50M72</p></td>
<td char="."><p xmlns="" class="SimplePara">38</p></td>
<td char="."><p xmlns="" class="SimplePara">37</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">97.391</p></td>
<td><p xmlns="" class="SimplePara">0.2325</p></td>
<td><p xmlns="" class="SimplePara">22.64</p></td>
<td char="."><p xmlns="" class="SimplePara">12.96</p></td>
<td char="."><p xmlns="" class="SimplePara">0.385</p></td>
<td char="."><p xmlns="" class="SimplePara">0.249</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">554</p></td>
<td char="."><p xmlns="" class="SimplePara">509</p></td>
<td char="."><p xmlns="" class="SimplePara">70</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td char="."> </td>
<td char="."><p xmlns="" class="SimplePara">0.400</p></td>
<td char="."><p xmlns="" class="SimplePara">0.260</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Minimum</p></td>
<td char="."><p xmlns="" class="SimplePara">30</p></td>
<td char="."><p xmlns="" class="SimplePara">26</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">80.95</p></td>
<td><p xmlns="" class="SimplePara">0.1973</p></td>
<td><p xmlns="" class="SimplePara">15.97</p></td>
<td char="."><p xmlns="" class="SimplePara">9.56</p></td>
<td char="."><p xmlns="" class="SimplePara">0.335</p></td>
<td char="."><p xmlns="" class="SimplePara">0.218</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Maximum</p></td>
<td char="."><p xmlns="" class="SimplePara">47</p></td>
<td char="."><p xmlns="" class="SimplePara">41</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">97.56</p></td>
<td><p xmlns="" class="SimplePara">0.3355</p></td>
<td><p xmlns="" class="SimplePara">32.73</p></td>
<td char="."><p xmlns="" class="SimplePara">20.74</p></td>
<td char="."><p xmlns="" class="SimplePara">0.449</p></td>
<td char="."><p xmlns="" class="SimplePara">0.296</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mean</p></td>
<td char="."><p xmlns="" class="SimplePara">39.64</p></td>
<td char="."><p xmlns="" class="SimplePara">36.36</p></td>
<td char="."><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">91.77</p></td>
<td><p xmlns="" class="SimplePara">0.2504</p></td>
<td><p xmlns="" class="SimplePara">23.07</p></td>
<td char="."><p xmlns="" class="SimplePara">14.10</p></td>
<td char="."><p xmlns="" class="SimplePara">0.396</p></td>
<td char="."><p xmlns="" class="SimplePara">0.258</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">TNF</em> total number of fragments, <em class="EmphasisTypeItalic">NPF</em> number of polymorphic fragments, <em class="EmphasisTypeItalic">PP</em> the fraction of polymorphic loci, <em class="EmphasisTypeItalic">PIC</em> polymorphic information content, <em class="EmphasisTypeItalic">MI</em> marker index, <em class="EmphasisTypeItalic">RP</em> resolving power, <em class="EmphasisTypeItalic">NUF</em> number of unique fragments, <em class="EmphasisTypeItalic">NRF</em> number of rare fragments</p>
                    </div></figure>
</div>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Population structure analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">The results of population using the STRUCTURE software showed a clear peak of the ÎK values when K was set at 2 (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>),which indicate that the sampled accessions belonged to two inferred genome fraction (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). Samples with a inferred genome fraction value (membership proportion) of 0.8 or more to a subgroups are considered as pure, and less than 0.80, as admixture [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. Considering individualâs membership proportion (Q<sub>i</sub>), all 27 accessions were assembled into three Bayesian subgroups designated as subgroup A (pure, blue fraction â¥0.8), subgroup B (admixture, 0.2ââ¤âblue fraction â¤0.8) and subgroup C (red fraction â¥0.2). The three subgroups included 10, 8 and 9 accessions, respectively. Geographically, most of accessions from Xingjiang (XJC) (10, 71.43%) belonged to subgroup A and the rest resided in subgroup B (Et20) and C (Et04 and Et13). Six accessions from Sichuan (SCC) were part of subgroup B and the five remaining accessions were part of subgroup C. For the three accessions from Gansu (GSC), Et18 consisted in subgroups B and the remaining two belonged to subgroup C.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig2_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig2_HTML.png" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Estimation of number of subpopulations (K) for the studied panel from STRUCTURE analysis</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig3_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig3_HTML.png" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig3_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig3_HTML.png" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Dendrogram of 27 <em class="EmphasisTypeItalic">Elymus tangutorum</em> accessions based on Diceâs coefficient using UPGMA method by NTSYS-pc v2.10 software and genetic structure using a Bayesian analysis by STRUCTURE v.2.3.4 and TESS 2.3.1 software</p>
                      </div></figcaption></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">The Bayesian clustering results by TESS 2.3.1 had lowest DIC values (highest probability) at K<sub>max</sub>â=â4, indicating that the sampled accessions belonged to four inferred genome fraction (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Figure S1). This pattern is slightly different from the result from Structure runs in which the optimal Kâ=â2. However, Considering the proportion of individualâs membership (Qi), the Bayesian subgroups based on TESS was almost exactly equivalent to the Structure result, both of which have the two dominant ancestry coefficients (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). On the other hand, the geographic map displaying the Q matrix spatially by R script [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>] also exhibited a general trend that the samples from XJC is obviously different from the samples from QTP, although the number of collection sites were limited and the spatial mapping for each accession is modeled over the whole Asia area rather than just considered at actual sampling sites (Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Figure S2).</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2"><h3 class="Heading">Cluster and principal coordinate analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">Based on the 509 polymorphic fragments, estimates of the genetic distances (Diceâs coefficient) ranged from 0.1024 to 0.7137 with an average of 0.2773, implying a largely genetic divergence in the germplasm collections studied. Accessions Et08 and Et11 were most related due to a minimum genetic distance of 0.1024, while Et01 and Et23 were most divergent with the maximum genetic distance of 0.7137. The rest of accessions have exhibited different middle levels of genetic distances. Compared with 0.2611 of inter-group dissimilarity, the accessions from Xinjiang were more homogenous with lower dissimilarity values (0.2214), whereas the QTP accessions were more variable genetically with higher dissimilarity values (0.2785). These results were also confirmed by STRUCTURE analysis that most of Xinjiang accessions had pure type of genome fraction. The genetic distance matrix was used to build a hierarchical dendogram of genetic relationships based on the UPGMA clustering method (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>), in which three major clades were clearly categorized with further internal subgroupings. These three clades were identified at the 0.259 dissimilarity level. Grouping of most of accessions was congruent with their respective geographical origins. However, the accession Et20 and Et13 from Xinjiang did not separate clearly from QTP accessions. The bootstrap support of all major branches were higher than 50%, and a majority of them ranged from 70 to 99% revealing reliability of data and clustering results. Despite some differences, there was coherence between Bayesian method and hierarchical clustering analysis (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). The UPGMA dendrogram could separate well three Bayesian subgroups, except for the accessions Et27, Et19 and Et23 were revealed as outliers.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">A similar clustering patter of the studied germplasm was demonstrated by the principal coordinate analysis (PCoA) (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>) illustrating that scatter-plot dipiction of PCoA is comparable to either of the hierarchical and Bayesian cluster analysis. That is, most of studied accessions were separated according to their geographical groups and/or the locations sites then put in the same cluster. PCoA was performed based on Nei-Li distances and confirmed division of 27 accessions into three major clusters: ClusterI, II and III. The first two principal coordinates represented the total of 69.1% of the total molecular variation (60.25, and 8.65%, respectively). The first axis (PC1), which contributed 60.25% of the variance, separated most of accessions, which have been assigned to three UPGMA clusters. The second axis (PC2),explained 8.65% of the variation, could further distinguish the accessions from cluster I and III.<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig4_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig4_HTML.png" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig4_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_Fig4_HTML.png" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Principal component analysis of 27 <em class="EmphasisTypeItalic">Elymus tangutorum</em> accessions based on AFLP data</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec10" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic diversity estimates</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">Geographically, the region of Gansu and Sichuan Province are both part of the QTP region. Hence, we merged the GSC and the SCC group into a new group named QTP group. The total gene diversity across different ecogeographical regions was estimated based on 509 polymorphic fragments (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S2). The percentage of polymorphic loci (PP) was found to be higher in accessions of QTP (73.29%) than those of Xinjiang (70.76%). The Xinjiang group (XJC) showed high Neiâs indices of gene diversity (He) of 0.24 and Shannonâs diversity (I) of 0.36, whereas QTP group displayed a moderate levels of diversity (Heâ=â0.21and Iâ=â0.32). For the three Bayesian subgroups, the P<em xmlns="" class="EmphasisTypeItalic">P</em> value was found to be highest among accessions from subgroup A (67.69%) followed by subgroup C (65.16%). The markers were least informative for assessment of genetic variability of accessions from subgroup B with PP restricted to 53.25%. Similarly, subgroup A showed the highest intra-group diversity (Iâ=â0.36 and Heâ=â0.24), whereas subgroup B had the lowest intra-group diversity (Iâ=â0.01 and Heâ=â0.0.01).</p>
              </section><section id="Sec11" class="Section2 RenderAsSection2"><h3 class="Heading">AMOVA analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">The two independent AMOVA analyses implemented relying on geographic origins and STRUCTURE clustering results and a highly significant variation (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.01) was detected (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). For regional groups, 16.24% of the total variation was due to differences among populations and 73.90% variation was due to divergence within groups. Both of the pairwise <em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub> values of two regional groups were significant (<em xmlns="" class="EmphasisTypeItalic">P</em>ââ¤â0.01) which were tested by GenAlex 6.5. The total <em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub>â=â0.162 indicated that the genetic variability is moderate between QTP and XJC groups. The three subgroups inferred from STRUCTURE showed 81 and 19% of total variance within and among the groups. This means that the total <em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub> was 0.188 (<em xmlns="" class="EmphasisTypeItalic">P</em>ââ¤â0.01) and the derived three subgroups are highly structured by Bayesian inference panel. Significant differences in the F<sub>ST</sub> were observed for all the pairwise comparisons of three subgroups inferred from STRUCTURE analysis (Table <span class="InternalRef"><a href="#Tab4">4</a></span>). The maximum pairwise <em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub> (0.274) found between subgroup A and subgroup C was confirmed by their great difference of inferred genome fraction (Table <span class="InternalRef"><a href="#Tab4">4</a></span>). In addition, AMOVA analysis indicate that the level of gene flow either between geographic areas or between Bayesian clusters was low and insufficient to make homogenization in the <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> collections of western China.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Analysis of molecular variance (AMOVA) among and within geographical groups and subgroups derived from STRUCTURE</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Groups</p></th>
<th><p xmlns="" class="SimplePara">Source of variation</p></th>
<th char="."><p xmlns="" class="SimplePara">d.f.</p></th>
<th char="."><p xmlns="" class="SimplePara">Sum of squares</p></th>
<th char="."><p xmlns="" class="SimplePara">Variance components</p></th>
<th char="."><p xmlns="" class="SimplePara">Percentage of variation (%)</p></th>
<th><p xmlns="" class="SimplePara">Fst</p></th>
<th><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">P</em>-value</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">Two Geographical Groups (XJC, QTP)</p></td>
<td><p xmlns="" class="SimplePara">Among groups</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">235.476</p></td>
<td char="."><p xmlns="" class="SimplePara">12.636</p></td>
<td char="."><p xmlns="" class="SimplePara">16.24%</p></td>
<td char="."><p xmlns="" class="SimplePara">0.162</p></td>
<td char="."><p xmlns="" class="SimplePara">â¤0.01</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Within groups</p></td>
<td char="."><p xmlns="" class="SimplePara">25</p></td>
<td char="."><p xmlns="" class="SimplePara">1628.154</p></td>
<td char="."><p xmlns="" class="SimplePara">65.126</p></td>
<td char="."><p xmlns="" class="SimplePara">83.75%</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">Three Bayesian subgroups (A, B, C)</p></td>
<td><p xmlns="" class="SimplePara">Among groups</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">380.382</p></td>
<td char="."><p xmlns="" class="SimplePara">14.324</p></td>
<td char="."><p xmlns="" class="SimplePara">19%</p></td>
<td char="."><p xmlns="" class="SimplePara">0.188</p></td>
<td char="."><p xmlns="" class="SimplePara">â¤0.01</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Within groups</p></td>
<td char="."><p xmlns="" class="SimplePara">24</p></td>
<td char="."><p xmlns="" class="SimplePara">1483.247</p></td>
<td char="."><p xmlns="" class="SimplePara">61.802</p></td>
<td char="."><p xmlns="" class="SimplePara">81%</p></td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table></div></figure><figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Pairwise estimates of Fst values between Bayesian subgroups of <em class="EmphasisTypeItalic">E. tangutorum</em> accessions as inferred by STRUCTURE software</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th> </th>
<th><p xmlns="" class="SimplePara">Subgroup A</p></th>
<th><p xmlns="" class="SimplePara">Subgroup B</p></th>
<th><p xmlns="" class="SimplePara">Subgroup C</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Subgroup A</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Subgroup B</p></td>
<td><p xmlns="" class="SimplePara">0.130*</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Subgroup C</p></td>
<td><p xmlns="" class="SimplePara">0.274**</p></td>
<td><p xmlns="" class="SimplePara">0.110**</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The P value for estimated Fst was calculated using 10,000 permutations (*<em class="EmphasisTypeItalic">P</em>â&lt;â0.05; **<em class="EmphasisTypeItalic">P</em>â&lt;â0.01)</p>
                    </div></figure>
</div>
              </section><section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Mantel test</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">To verify the eco-geographical factors influencing genetic structuring, Mantelâs test was used to estimate matrix correlations between genetic distance and geographical, elevation, average annual temperature, and average annual precipitation distances. A weak but significant pattern of isolation by geographic distance was detected within all accessions (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.312; <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.001). A similar result was found in each group except GSC (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). This pattern derives largely from measurable differentiation among geographic groups seen in STRUCTURE and UPGMA clustering analysis. Furthermore, there was a faint or non-significant correlation between distance matrices of genetic and environmental factors for all accessions. However, a significantly positive correlation was demonstrated between genetic distance and the average annual precipitation distance in XJC (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.685; <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.01) and Subgroup A (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.594; <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.05) (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). In addition, relatively high Mantel r-value were observed between environmental factors and genetic distance for Gansu accessions, all of which were non-significant due to a small group size (only 3 accessions).<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">Correlation between genetic distance and eco-environmental distance based on pairwise Mantel test for all accessions, geographical groups and Bayesian subgroups</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col char=".">
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Groups</p></th>
<th><p xmlns="" class="SimplePara">Factors</p></th>
<th char="."><p xmlns="" class="SimplePara">r</p></th>
<th char="."><p xmlns="" class="SimplePara">p</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">All</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">0.312</p></td>
<td char="."><p xmlns="" class="SimplePara">0.001</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.118</p></td>
<td char="."><p xmlns="" class="SimplePara">0.13</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.285</p></td>
<td char="."><p xmlns="" class="SimplePara">0.01</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.187</p></td>
<td char="."><p xmlns="" class="SimplePara">0.13</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">XJC</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">0.313</p></td>
<td char="."><p xmlns="" class="SimplePara">0.076</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.052</p></td>
<td char="."><p xmlns="" class="SimplePara">0.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.685</p></td>
<td char="."><p xmlns="" class="SimplePara">0.01</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.051</p></td>
<td char="."><p xmlns="" class="SimplePara">0.36</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">SCC</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">0.113</p></td>
<td char="."><p xmlns="" class="SimplePara">0.187</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.3</p></td>
<td char="."><p xmlns="" class="SimplePara">0.09</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.005</p></td>
<td char="."><p xmlns="" class="SimplePara">0.69</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.439</p></td>
<td char="."><p xmlns="" class="SimplePara">0.11</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">GSC</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">0.809</p></td>
<td char="."><p xmlns="" class="SimplePara">0.32</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.971</p></td>
<td char="."><p xmlns="" class="SimplePara">0.24</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.92</p></td>
<td char="."><p xmlns="" class="SimplePara">0.16</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.309</p></td>
<td char="."><p xmlns="" class="SimplePara">0.49</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">QTP</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">0.131</p></td>
<td char="."><p xmlns="" class="SimplePara">0.19</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.325</p></td>
<td char="."><p xmlns="" class="SimplePara">0.1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.05</p></td>
<td char="."><p xmlns="" class="SimplePara">0.32</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.365</p></td>
<td char="."><p xmlns="" class="SimplePara">0.05</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">Subgroup A</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">0.399</p></td>
<td char="."><p xmlns="" class="SimplePara">0.07</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.226</p></td>
<td char="."><p xmlns="" class="SimplePara">0.21</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.594</p></td>
<td char="."><p xmlns="" class="SimplePara">0.05</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.014</p></td>
<td char="."><p xmlns="" class="SimplePara">0.51</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">Subgroup B</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.125</p></td>
<td char="."><p xmlns="" class="SimplePara">0.5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.064</p></td>
<td char="."><p xmlns="" class="SimplePara">0.41</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.208</p></td>
<td char="."><p xmlns="" class="SimplePara">0.24</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.024</p></td>
<td char="."><p xmlns="" class="SimplePara">0.37</p></td>
</tr>
<tr class="noclass">
<td rowspan="4"><p xmlns="" class="SimplePara">Subgroup C</p></td>
<td><p xmlns="" class="SimplePara">Geographical distance</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.032</p></td>
<td char="."><p xmlns="" class="SimplePara">0.53</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Elevation</p></td>
<td char="."><p xmlns="" class="SimplePara">0.054</p></td>
<td char="."><p xmlns="" class="SimplePara">0.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual precipitation</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.186</p></td>
<td char="."><p xmlns="" class="SimplePara">0.31</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average annual temperature</p></td>
<td char="."><p xmlns="" class="SimplePara">0.079</p></td>
<td char="."><p xmlns="" class="SimplePara">0.28</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">XJC</em> Xinjiang province of China (XJC), <em class="EmphasisTypeItalic">SCC</em> Sichuan province of China, <em class="EmphasisTypeItalic">GSS</em> Gansu province of China</p>
                    </div></figure>
</div>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<section id="Sec14" class="Section2 RenderAsSection2"><h3 class="Heading">AFLP polymorphism and discriminating capacity of the assays</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">The practical availability of molecular markers in plant germplasm characterization depends on their power to discriminate the genotypically different accessions analyzed. In the present paper, the efficiency of AFLPs were assessed by recording different marker parameters concerning polymorphism, PIC (Polymorphic Information Content), MI (marker index) and RP (resolving power). In total, 509 AFLP polymorphic fragments with 91.77% of polymorphism were observed in the current work, which was obviously higher than previous works by AFLP in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> and other related <em xmlns="" class="EmphasisTypeItalic">Elymus</em> species. In the studies of <em xmlns="" class="EmphasisTypeItalic">E. dahuricus</em> complex diversity, 69.35, 58.79, 29.65 and 27.14% of polymorphic fragments for <em xmlns="" class="EmphasisTypeItalic">E.excelsus</em>, <em xmlns="" class="EmphasisTypeItalic">E.dahuricus</em>, <em xmlns="" class="EmphasisTypeItalic">E.woroschilowii</em> and <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> was obtained respectively [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. In three <em xmlns="" class="EmphasisTypeItalic">Elymus</em> species indigenous to East Asia, 34.2, 30.1 and 51.5% of polymorphism was detected for <em xmlns="" class="EmphasisTypeItalic">E. tsukushiensis</em>, <em xmlns="" class="EmphasisTypeItalic">E. humidus</em> and <em xmlns="" class="EmphasisTypeItalic">E. dahuricus</em> respectively [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. So far, the primer combinations used in this study successfully established high levels of genetic distinctness among the studied <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accessions using AFLP fingerprints. The level of polymorphism obtained here with AFLP markers is also higher than that reported for other <em xmlns="" class="EmphasisTypeItalic">Elymus</em> germplasm in western China by other types of dominant markers such as <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> by ScoT (89%) [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] and <em xmlns="" class="EmphasisTypeItalic">E. nutans</em> by ISSR (91.4%) [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. In all, the participation of polymorphic fragments largely relies on the species, origin ecogeographic area and the differentiation degree of sampled accessions, amount of primers and even type of molecular marker used [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>, <span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. Moreover, as all primer combinations produced unique fragments (on average, 5 per primer), those primers could be used to differentiate particular accessions [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">The polymorphic information content (PIC) values of 0.25 in current study demonstrated good marker discriminatory power in view of PIC value starting from 0 to 0.5 for dominant marker like AFLPs [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>, <span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. This values also agree with AFLP studies done with other forage grass such as <em xmlns="" class="EmphasisTypeItalic">Dactylis glomerata</em> [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>] and <em xmlns="" class="EmphasisTypeItalic">Phalaris aquatica</em> [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>].The Marker index (MI) and resolving power (RP), two importantly alternative parameters in selecting informative markers for diversity studies [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>, <span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>], had also displayed high efficiency of AFLP markers in unveiling polymorphisms in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm. A very strong and positive correlation observed among PIC, RP and MI value in present work indicates the use of any of three parameters to screen the informative primer combinations [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]; although the relationship between the informativeness of molecular markers for identifying germplasms and these marker parameters is not totally clear [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. Primer combinations E-ATG/M-CTC and E-AAC/M-GAC should be the most informative due to the highest records on PIC, RP and MI. Therefore, they are recommended for use in germplasm diversity analyses of other <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> collections. The Shannon index is an accurate alternative measure of diversity due to no need for estimate of allele frequencies under Hardy-Weinberg equilibrium. The Shannon diversity detected in this study (0.34) was higher than those of <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> germplasm from western China by SCoT (<em xmlns="" class="EmphasisTypeItalic">I</em>â=â0.285) and EST-SSR (<em xmlns="" class="EmphasisTypeItalic">I</em>â=â0.237) markers [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>], also indicating a high level of genetic diversity in the accessions of <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> (Table <span class="InternalRef"><a href="#Tab2">2</a></span>).</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2"><h3 class="Heading">Clustering pattern and genetic structure</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">It is attractive for breeders and germplasm curators to know the width of genetic diversity within a plant species of great importance [<span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>]. The spatial genetic diversity analysis was conducted to verify the relationship between <em xmlns="" class="EmphasisTypeItalic">Elymus tangutorum</em> accessions and biogeographical patterns by different clustering approaches. The UPGMA tree, the PCoA scatter plot, and the STRUCTURE analyses revealed a similar or identical membership among 27 wild <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accessions, with a general grouping pattern including three genetically distinct and consistent groups. When these accessions are grouped according to clusters distinguished(Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>), a greater range of Nei-Li distances is observed, which is comparable to previous studies for other <em xmlns="" class="EmphasisTypeItalic">Elymus</em> germplasm with similar collection areas [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>, <span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>]. The first cluster derived from UPGMA tree consisted of ten accessions from Xinjiang Province in southwest of China, having a typical temperate continental climate. Meanwhile, Cluster I had the distinctly pure membership based on Bayesian clustering. The second cluster was primarily formed by accessions from southeast of Qinghai-Tibet Plateau (QTP), a region greatly far from Xinjiang, having a typical plateau mountain climate. Cluster II was corresponding well to the admixture subgroup STRUCTUE by analysis. The third cluster was also chiefly comprised by the accessions from southeast of QTP, with another distinctly pure membership. In short, the above results were in agreement with the geographical proximity of studied accessions and their genetic relationships.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">Exceptions to this pattern of clustering were three accessions (Et04, Et13 and Et20) from Xinjiang,which were grouped with QTP accessions in Clade II and Clade III respectively. Since <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> is used in natural grassland restoration projects [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], it is possible that these three accessions are not native accessions from QTP and might have been introduced through domestic trade of seeds from Xinjiang. By contrasting the figures of UPGMA, PCoA and STRUCTURE analyses, two accessions (Et19 and Et27) assigned to Bayesian Subgroup II, fall into outgroups of UPGMA analyses. The possible reason causing such relationships may be STRUCTURE assumes that the loci within a population are under HardyâWeinberg equilibrium (HWE) and it is susceptible to many factors such as non-random mating, genetic drift, mutations, gene flow and selection [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>]. Nonetheless, in different clustering analyses, it was largely found that separate clusters were often obtained in accordance with the respective geographic origin of accessions. Therefore the cluster pattern in present work might be due, in part, to differences in the adaptation of ecotypes to various selection forces from local conditions [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. These results are in agreement with earlier studies which showed that geographical separation of other <em xmlns="" class="EmphasisTypeItalic">Elymus</em> species germplasm did not always result in greater genetic differentiation [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">Indeed, the genetic diversity and population structure are determined by joint effects of many factors including geographic distribution, life cycle, mating system, selection and adaptation [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>]. Considering the genetic differences among studied regions, the accessions from Xinjiang province showed higher diversity (Iâ=â0.24, Heâ=â0.36) than those from QTP (Iâ=â0.21, Heâ=â0.32). With regard to the Bayesian subgroups, higher diversity was found among accessions in subgroup A, which correspond to accessions from Xinjiang province. Based on a membership probability threshold of 0.80, most accessions (66.67%) such as subgroup A and C appeared to have a pure population ancestry in their genome component, which might result from limited gene flow among wild accessions with distantly geographical origin [<span class="CitationRef"><a href="#CR59" data-toggle="citation">59</a></span>]. This explains why we have observed high diversity within geographical groups. However, more than half of the accessions from QTP were composed of admixture of genome fraction. Such case could be supported by the fact that <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> was frequently used to grassland restoration in Qinghai-Tibetan Plateau as mentioned above [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>]. Therefore, human activities may lead to frequently natural crossing by wind pollination between adjacent stands.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">Results of AMOVA analysis showed low genetic variation among regions accounted for only 16.2% of the germplasm collection, whereas within-region variation accounts for 83.8% of the total variability. Similarly, only 19% of variation was detected for Bayesian groups,. These results on variation among accessions within geographical groups were confirmed with previous studies for <em xmlns="" class="EmphasisTypeItalic">E. nutans</em> and <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> collections with similar origins [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]. However, our results are more extreme. This could be a consequence of high gene flow resulting from grassland restoration project or the limited number of studied accessions for each region. The low among-group variance components indicated that the genetic background attributable to the geographical origin contributes slightly to the observed genetic diversity. Besides, the high gene flow and differentiation within <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em>, supported by clustering and AMOVA, was also probably driven by its mating system, which is one of the important life-history feature that strongly influence genetic diversity and population structure [<span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>]. As reported in genetic diversity study for <em xmlns="" class="EmphasisTypeItalic">Elymus dahuricus</em> complex [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> is probably a predominantly self-pollinated species with 0.9 of <em xmlns="" class="EmphasisTypeItalic">F</em><sub><em xmlns="" class="EmphasisTypeItalic">ST</em></sub> value. In their review, Hamrick and Godt [<span class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>] claimed that short-lived, self-pollinating species have most of their genetic diversity partitioned among populations rather than within populations. Similarly, Nybom [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>] assumed that short-lived self-pollinating species allocate most of their genetic variability within populations. Each <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accession in present investigation could be treated as a population consisting of seeds of neighboring individual plants collected from differently distant locations [<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>]. This situation may limit gene flow between accessions by geographic isolation barriers (i.e. mountains, rivers), thus the variation will maintain its characteristics within each accession. Although the within-accession variability is not determined by individual-based analysis, we have indeed observed the high degree of differentiation within this species according to UPGMA and PCoA analysis based on bulked DNA samples. Hence it is not surprising that the vast proportion of AFLP variation detected in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> was present rather within than among groups, since the large-scale geographical group contained numerous accessions with distinctly genetic architecture. These results could be confirmed by that the totally average genetic distance (0.277) between all of accessions was higher than that (0.261) between accessions from the two main geographical groups (QTP and Xinjiang).</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2"><h3 class="Heading">Correlation of genetic diversity and environmental variables</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">In virtue of the evolutionary forces, the decisive factor to influence the maintenance of genetic variability is the spatial variation of environment and the ecological difference between habitats [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. Remarkably, the Mantel test between genetic distance and geographical distance in all <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accessions and most groups was weakly correlated (Table <span class="InternalRef"><a href="#Tab5">5</a></span>) and similar results were reported in previous studies of <em xmlns="" class="EmphasisTypeItalic">Elymus</em> species [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]. Within accessions from QTP group, we found lower <em xmlns="" class="EmphasisTypeItalic">r</em>-values of geographical distance (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.131) compared with the XJC group (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.313), which was not significantly different from zero and might be related to the highly heterogeneous topography nature of the QTP collection. The complex topographic features and climate will work together to form varieties niches to influence the genetic variability [<span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>]. Thus, spatial distribution cannot be solely accounted for by a simple isolation-by-distance model and requires additional factors that influence the observed genetic population structure, such as environmental factors [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. In the Mantel test between genetic distance and environmental distance, we found a significantly positive correlation between genetic distance and average annual precipitation (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.685; <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.01 and <em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.594; <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.05) in the XJC group and Bayesian subgroup A. A possible explanation for this results is that in Xinjiang with arid and/or semi-arid temperate climate, the lower precipitation hinders seed germination and lead to decrease plantsâ genetic diversity [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. In the diversity study of <em xmlns="" class="EmphasisTypeItalic">E. nutans</em> germplasm from western China, the environmental divergence like elevation is also related to genetic distance (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.695; <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.01) [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. On the other hand, the relationship between environmental factor and genetic distance is difficult to be fully resolved in complex environment [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. This could be responsible for the weak and non-significant correlation between genetic distance and environmental distance found in QTP group. Besides, although the accessions of GSC group had high Mantel r-value (Table <span class="InternalRef"><a href="#Tab5">5</a></span>) between genetic distance and environmental distance except the average annual temperature (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.309), the correlation between the genetic diversity and environmental factor was still unraveled due to the small population size.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec17" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">This study indicates that AFLP markers are a powerful tool to measure genetic diversity in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> and geographical and environmental factors (especially precipitation) together plays a crucial role in genetic differentiation patterns. These findings highlight the importance of local adaptation in shaping patterns of genetic structure inferred in <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> accessions fromWestern China. Therefore, collecting and assessing <em xmlns="" class="EmphasisTypeItalic">E. tangutorum</em> germplasm from major geographic regions and special ecogeographic environment such as Qinghai-Tibet Plateau, will help to expand the genetic base and sample the full extent of available variation.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Notes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Notes">Notes</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Notes"><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteAuthorContribution"><p xmlns="" class="SimplePara">Wen-Dan Wu and Wen-Hui Liu contributed equally to this work.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry">
<dt class="Term"><dfn>AFLP: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Amplified fragment length polymorphism</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>AMOVA: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Analysis of molecular variance</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>CTAB: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Hexadecyl trimethyl ammonium Bromid</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>GD: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Genetic distance</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>I: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Shannon information index</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>ISSR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Inter-simple sequence repeat</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>MI: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Marker index</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>NPF: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Number of polymorphic loci</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>PCR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Polymerase chain reaction</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>PIC: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Polymorphism information content</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>PcoA: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Principal coordinate analysis</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>RAPD: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Random Amplified Polymorphic DNA</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>RP: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Resolving power</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SCoT: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Started codon targeted polymorphism</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SRAP: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Sequenceârelated amplified polymorphism</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SSR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Simple sequence repeat</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>TNF: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Total number of amplified fragments</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>UPGMA: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Unweighted pair-group method with arithmetic means</p>
                  </dd>
</dl>
</div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledge</h3>
<p xmlns="" class="SimplePara">Not applicable.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par48" class="Para">This study was funded by the Key Laboratory of Superior Forage Germplasm in the Qinghai-Tibetan plateau (2017-ZJ-Y12) and the earmarked fund for Modern Agro-industry Technology Research System (No.CARS-34). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par49" class="Para">The datasets generated during the current study are available from the corresponding author on reasonable request.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1">
<h3 xmlns="" class="Heading">Authorsâ contributions</h3>
<p xmlns="" class="SimplePara">XM and W-HL conceived the project and designed the experiments. W-DW and MS performed the experiment. W-DW, W-HL, and XM wrote the paper. J-QZ, WL, C-LZ, X-QZ, YP and L-KH gave comments on the manuscript. XM revised the paper. All authors discussed the results and commented on the manuscript. All authors read and approved the final manuscript.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par50" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par51" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par52" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisherâs Note</h3>
                <p id="Par53" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside class="Appendix" id="App1"><h3 class="Heading">Additional files</h3>
<div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par54" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_MOESM1_ESM.docx"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S1.</strong> Adaptor and primer sequences used for AFLP analysis. (DOCX 17 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_MOESM2_ESM.tif"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S1.</strong> Bayesian clustering results for 27 accessions of <em class="EmphasisTypeItalic">Elymus tangutorum</em> using the program TESS from Kâ=â2 to Kâ=â5. (TIF 686 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_MOESM3_ESM.tif"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S2.</strong> Maps representing the mean membership proportions for K clusters of <em class="EmphasisTypeItalic">Elymus tangutorum. (TIF 80 kb)</em></span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-019-0082-z/MediaObjects/41065_2019_82_MOESM4_ESM.docx"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S2.</strong> Different genetic diversity estimates for geographical groups and Bayesian subgroups of <em class="EmphasisTypeItalic">E. tangutorum</em> based on AFLP results. (DOCX 18 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, 611130, Sichuan, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Key Laboratory of Superior Forage Germplasm in the Qinghai-Tibetan Plateau, Qinghai Academy of Animal Science and Veterinary Medicine, Xining, 81108, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dewey DR. The genomic system of classification as a guide to intergeneric hybridization with the perennial Triticeae. In: Gustafson JP, editor. Gene Manipulation in Plant Improvement. Boston: Springer; 1984. p. 209â79.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-1-4613-2429-4_9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20genomic%20system%20of%20classification%20as%20a%20guide%20to%20intergeneric%20hybridization%20with%20the%20perennial%20Triticeae&amp;author=DR.%20Dewey&amp;pages=209-279&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">DÃ­az O, Sun GL, Salomon B, Bothmer RV. Levels and distribution of allozyme and RAPD variation in populations of Elymus fibrosus (Schrenk) Tzvel. (Poaceae). Genet Resour Crop Ev. 2000;47(1):11â24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1008719119756"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Levels%20and%20distribution%20of%20allozyme%20and%20RAPD%20variation%20in%20populations%20of%20Elymus%20fibrosus%20%28Schrenk%29%20Tzvel.%20%28Poaceae%29&amp;author=O.%20D%C3%ADaz&amp;author=GL.%20Sun&amp;author=B.%20Salomon&amp;author=RV.%20Bothmer&amp;journal=Genet%20Resour%20Crop%20Ev.&amp;volume=47&amp;issue=1&amp;pages=11-24&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kuo BZ, Wang JS, Li JH. Studies on morphological evolution, classification and distribution of the Triticeae in China. Acta Phytotaxon Sin. 1985;23(3):161â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Studies%20on%20morphological%20evolution%2C%20classification%20and%20distribution%20of%20the%20Triticeae%20in%20China&amp;author=BZ.%20Kuo&amp;author=JS.%20Wang&amp;author=JH.%20Li&amp;journal=Acta%20Phytotaxon%20Sin&amp;volume=23&amp;issue=3&amp;pages=161-169&amp;publication_year=1985"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang CR, Zhang HQ, Zhao FQ, Liu XY, Fan X, Sha LN, Kang HY, Wang Y, Zhou YH. Genome constitution of Elymus tangutorum (Poaceae: Triticeae) inferred from meiotic pairing behavior and genomic in situ hybridization. J Syst Evol. 2015;53(6):529â34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/jse.12155"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20constitution%20of%20Elymus%20tangutorum%20%28Poaceae%3A%20Triticeae%29%20inferred%20from%20meiotic%20pairing%20behavior%20and%20genomic%20in%20situ%20hybridization&amp;author=CR.%20Yang&amp;author=HQ.%20Zhang&amp;author=FQ.%20Zhao&amp;author=XY.%20Liu&amp;author=X.%20Fan&amp;author=LN.%20Sha&amp;author=HY.%20Kang&amp;author=Y.%20Wang&amp;author=YH.%20Zhou&amp;journal=J%20Syst%20Evol&amp;volume=53&amp;issue=6&amp;pages=529-534&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Agafonov AV, Baum BR, Bailey LG. Differentiation in the Elymus dahuricus complex (Poaceae): evidence from grain proteins, DNA, and crossability. Hereditas. 2001;135(2â3):277â89.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12152347"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differentiation%20in%20the%20Elymus%20dahuricus%20complex%20%28Poaceae%29%3A%20evidence%20from%20grain%20proteins%2C%20DNA%2C%20and%20crossability&amp;author=AV.%20Agafonov&amp;author=BR.%20Baum&amp;author=LG.%20Bailey&amp;journal=Hereditas&amp;volume=135&amp;issue=2%E2%80%933&amp;pages=277-289&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang YL. Study on cultivation and domestication of Elymus tangutorum. Chin Qinghai J Anim Vet Sci. 2012;3(2):1â2.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Study%20on%20cultivation%20and%20domestication%20of%20Elymus%20tangutorum&amp;author=YL.%20Wang&amp;journal=Chin%20Qinghai%20J%20Anim%20Vet%20Sci.&amp;volume=3&amp;issue=2&amp;pages=1-2&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang DY, Jiang XH. Mating system evolution, resource allocation, and genetic diversity in plants. Acta Phytotaxon Sin. 2001;25(2):130â43.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mating%20system%20evolution%2C%20resource%20allocation%2C%20and%20genetic%20diversity%20in%20plants&amp;author=DY.%20Zhang&amp;author=XH.%20Jiang&amp;journal=Acta%20Phytotaxon%20Sin.&amp;volume=25&amp;issue=2&amp;pages=130-143&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Berdahl JD, Mayland HF, Asay KH, Jefferson PG. Variation in agronomic and morphological traits among Russian wildrye accessions. Crop Sci. 1999;39(6):1890â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1999.3961890x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Variation%20in%20agronomic%20and%20morphological%20traits%20among%20Russian%20wildrye%20accessions&amp;author=JD.%20Berdahl&amp;author=HF.%20Mayland&amp;author=KH.%20Asay&amp;author=PG.%20Jefferson&amp;journal=Crop%20Sci&amp;volume=39&amp;issue=6&amp;pages=1890-1895&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guo ZH, Yang HY, Zhang CL, Sun M, Fu KX, Zhang XQ, Ma X. Identification of four Elymus sibiricus cultivar(strains)from northwest plateau of Sichuan using morphological traits and SSR markers. Pratacult Sci. 2016;33(9):1718â27.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20four%20Elymus%20sibiricus%20cultivar%28strains%29from%20northwest%20plateau%20of%20Sichuan%20using%20morphological%20traits%20and%20SSR%20markers&amp;author=ZH.%20Guo&amp;author=HY.%20Yang&amp;author=CL.%20Zhang&amp;author=M.%20Sun&amp;author=KX.%20Fu&amp;author=XQ.%20Zhang&amp;author=X.%20Ma&amp;journal=Pratacult%20Sci&amp;volume=33&amp;issue=9&amp;pages=1718-1727&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun GL, DÃ­az O, Salomon BVon BR. Genetic diversity in Elymus caninus as revealed by isozyme, RAPD, and microsatellite markers. Genome. 1999;42(3):420.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/g98-130"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20Elymus%20caninus%20as%20revealed%20by%20isozyme%2C%20RAPD%2C%20and%20microsatellite%20markers&amp;author=GL.%20Sun&amp;author=O.%20D%C3%ADaz&amp;author=BR.%20Salomon%20BVon&amp;journal=Genome&amp;volume=42&amp;issue=3&amp;pages=420&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Li S, Li L, Yang X, Li X. Comparison of genetic diversity of twelvel Elymus species using ISSR and SSR markers. Scientia Agri Sin. 2005;38(08):1522â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20genetic%20diversity%20of%20twelvel%20Elymus%20species%20using%20ISSR%20and%20SSR%20markers&amp;author=Y.%20Li&amp;author=S.%20Li&amp;author=L.%20Li&amp;author=X.%20Yang&amp;author=X.%20Li&amp;journal=Scientia%20Agri%20Sin&amp;volume=38&amp;issue=08&amp;pages=1522-1527&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen ZH, Miao JM, Zhong JC, Ma X, Chen SY, Zhang XQ. Genetic diversity of wild Elymus nutans germplasm detected by SRAP markers. Acta Pratacul Sin. 2010;30(1):94â100.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20wild%20Elymus%20nutans%20germplasm%20detected%20by%20SRAP%20markers&amp;author=ZH.%20Chen&amp;author=JM.%20Miao&amp;author=JC.%20Zhong&amp;author=X.%20Ma&amp;author=SY.%20Chen&amp;author=XQ.%20Zhang&amp;journal=Acta%20Pratacul%20Sin&amp;volume=30&amp;issue=1&amp;pages=94-100&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie WG, Zhang JC, Zhao XH, Zhang JQ, Wang YR. Siberian wild rye ( <em class="EmphasisTypeItalic">Elymus sibiricus</em> L.): Genetic diversity of germplasm determined using DNA fingerprinting and SCoT markers. Biochem Syst Ecol. 2015;60:186â92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2015.04.021"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Siberian%20wild%20rye%20%28%20Elymus%20sibiricus%20L.%29%3A%20Genetic%20diversity%20of%20germplasm%20determined%20using%20DNA%20fingerprinting%20and%20SCoT%20markers&amp;author=WG.%20Xie&amp;author=JC.%20Zhang&amp;author=XH.%20Zhao&amp;author=JQ.%20Zhang&amp;author=YR.%20Wang&amp;journal=Biochem%20Syst%20Ecol&amp;volume=60&amp;pages=186-192&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sasanuma T, Endo TR, Ban T. Genetic diversity of three Elymus species indigenous to Japan and East Asia (E. Tsukushiensis, E. Humidus and E. Dahuricus) detected by AFLP. Genes Genet Syst. 2002;77(6):429â38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1266/ggs.77.429"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20three%20Elymus%20species%20indigenous%20to%20Japan%20and%20East%20Asia%20%28E.%20Tsukushiensis%2C%20E.%20Humidus%20and%20E.%20Dahuricus%29%20detected%20by%20AFLP&amp;author=T.%20Sasanuma&amp;author=TR.%20Endo&amp;author=T.%20Ban&amp;journal=Genes%20Genet%20Syst&amp;volume=77&amp;issue=6&amp;pages=429-438&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Szczepaniak M, CieÅlak EBednarek PT. Morphological and AFLP variation of Elymus repens (L.) Gould (Poaceae). Cell Mol Biol Lett. 2002;7(2A):547â58.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12378260"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Morphological%20and%20AFLP%20variation%20of%20Elymus%20repens%20%28L.%29%20Gould%20%28Poaceae%29&amp;author=M.%20Szczepaniak&amp;author=PT.%20Cie%C5%9Blak%20EBednarek&amp;journal=Cell%20Mol%20Biol%20Lett&amp;volume=7&amp;issue=2A&amp;pages=547-558&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Szczepaniak M, CieÅlak E. Assessment of genetic variation in <em class="EmphasisTypeItalic">Elymus repens</em> and <em class="EmphasisTypeItalic">E. hispidus</em> using AFLP markers and its implications for intraspecific taxonomy. In: Frey L, editor. Problems of grass biology. KrakÃ³w: W. Szafer Institute of Botany, Polish Academy of Sciences; 2003. p. 271â86.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20variation%20in%20Elymus%20repens%20and%20E.%20hispidus%20using%20AFLP%20markers%20and%20its%20implications%20for%20intraspecific%20taxonomy&amp;author=M.%20Szczepaniak&amp;author=E.%20Cie%C5%9Blak&amp;pages=271-286&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang HX, Zhang ML. Genetic structure of the Delphinium naviculare species group tracks Pleistocene climatic oscillations in the Tianshan Mountains, arid Central Asia. Palaeogeogr Palaeocl. 2012;353-355(5):93â103.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.palaeo.2012.07.013"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20of%20the%20Delphinium%20naviculare%20species%20group%20tracks%20Pleistocene%20climatic%20oscillations%20in%20the%20Tianshan%20Mountains%2C%20arid%20Central%20Asia&amp;author=HX.%20Zhang&amp;author=ML.%20Zhang&amp;journal=Palaeogeogr%20Palaeocl&amp;volume=353-355&amp;issue=5&amp;pages=93-103&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu ZG, Yu D, Li X, Xu XW. Influence of geography and environment on patterns of genetic differentiation in a widespread submerged macrophyte, Eurasian watermilfoil (Myriophyllum spicatum L., Haloragaceae). Ecol Evol. 2016;6(2):460â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/ece3.1882"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Influence%20of%20geography%20and%20environment%20on%20patterns%20of%20genetic%20differentiation%20in%20a%20widespread%20submerged%20macrophyte%2C%20Eurasian%20watermilfoil%20%28Myriophyllum%20spicatum%20L.%2C%20Haloragaceae%29&amp;author=ZG.%20Wu&amp;author=D.%20Yu&amp;author=X.%20Li&amp;author=XW.%20Xu&amp;journal=Ecol%20Evol&amp;volume=6&amp;issue=2&amp;pages=460-468&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ozbek O, Millet E, Anikster Y, Arslan OFeldman M. Spatio-temporal genetic variation in populations of wild emmer wheat, <em class="EmphasisTypeItalic">Triticum turgidum ssp. dicoccoides</em> , as revealed by AFLP analysis. Theor Appl Genet. 2007;115(1):19â26.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-007-0536-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Spatio-temporal%20genetic%20variation%20in%20populations%20of%20wild%20emmer%20wheat%2C%20Triticum%20turgidum%20ssp.%20dicoccoides%20%2C%20as%20revealed%20by%20AFLP%20analysis&amp;author=O.%20Ozbek&amp;author=E.%20Millet&amp;author=Y.%20Anikster&amp;author=M.%20Arslan%20OFeldman&amp;journal=Theor%20Appl%20Genet&amp;volume=115&amp;issue=1&amp;pages=19-26&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dequigiovanni G, Ramos SLF, Alves-Pereira A, Fabri EG, PicanÃ§o-Rodrigues D, Clement CR, Gepts PVeasey EA. Highly structured genetic diversity of <em class="EmphasisTypeItalic">Bixa orellana</em> var. urucurana, the wild ancestor of annatto, in Brazilian Amazonia. Plos One. 2018;13(6):e0198593.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0198593"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Highly%20structured%20genetic%20diversity%20of%20Bixa%20orellana%20var.%20urucurana%2C%20the%20wild%20ancestor%20of%20annatto%2C%20in%20Brazilian%20Amazonia&amp;author=G.%20Dequigiovanni&amp;author=SLF.%20Ramos&amp;author=A.%20Alves-Pereira&amp;author=EG.%20Fabri&amp;author=D.%20Pican%C3%A7o-Rodrigues&amp;author=CR.%20Clement&amp;author=EA.%20Gepts%20PVeasey&amp;journal=Plos%20One&amp;volume=13&amp;issue=6&amp;pages=e0198593&amp;publication_year=2018"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">HÃ¼bner S, HÃfken M, Oren E, Haseneyer G, Stein N, Graner A, Schmid K, Fridman E. Strong correlation of wild barley (Hordeum spontaneum) population structure with temperature and precipitation variation. Mol Ecol. 2009;18(7):1523â36.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2009.04106.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Strong%20correlation%20of%20wild%20barley%20%28Hordeum%20spontaneum%29%20population%20structure%20with%20temperature%20and%20precipitation%20variation&amp;author=S.%20H%C3%BCbner&amp;author=M.%20H%C3%96fken&amp;author=E.%20Oren&amp;author=G.%20Haseneyer&amp;author=N.%20Stein&amp;author=A.%20Graner&amp;author=K.%20Schmid&amp;author=E.%20Fridman&amp;journal=Mol%20Ecol&amp;volume=18&amp;issue=7&amp;pages=1523-1536&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nevo E. Evolution of genome-phenome diversity under environmental stress. Proc Natl Acad Sci U S A. 2001;98(11):6233â40.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.101109298"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%20of%20genome-phenome%20diversity%20under%20environmental%20stress&amp;author=E.%20Nevo&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=98&amp;issue=11&amp;pages=6233-6240&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Castillo A, Dorado G, Feuillet C, Sourdille PHernandez P. Genetic structure and ecogeographical adaptation in wild barley ( Hordeum chilense Roemer et Schultes) as revealed by microsatellite markers. BMC Plant Biol. 2010;10(1):266.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-10-266"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20ecogeographical%20adaptation%20in%20wild%20barley%20%28%20Hordeum%20chilense%20Roemer%20et%20Schultes%29%20as%20revealed%20by%20microsatellite%20markers&amp;author=A.%20Castillo&amp;author=G.%20Dorado&amp;author=C.%20Feuillet&amp;author=P.%20Sourdille%20PHernandez&amp;journal=BMC%20Plant%20Biol&amp;volume=10&amp;issue=1&amp;pages=266&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shaaf S, Sharma R, Kilian B, Walther A, Ãzkan H, Karami EMohammadi B. Genetic structure and eco-geographical adaptation of garlic landraces ( Allium sativum L.) in Iran. Genet Resour Crop Ev. 2014;61(8):1565â80.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-014-0131-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20eco-geographical%20adaptation%20of%20garlic%20landraces%20%28%20Allium%20sativum%20L.%29%20in%20Iran&amp;author=S.%20Shaaf&amp;author=R.%20Sharma&amp;author=B.%20Kilian&amp;author=A.%20Walther&amp;author=H.%20%C3%96zkan&amp;author=B.%20Karami%20EMohammadi&amp;journal=Genet%20Resour%20Crop%20Ev.&amp;volume=61&amp;issue=8&amp;pages=1565-1580&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Del SG, Manfioletti GSchneider C. The CTAB-DNA precipitation method: a common mini-scale preparation of template DNA from phagemids, phages or plasmids suitable for sequencing. BioTechniques. 1989;7(5):514â20.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20CTAB-DNA%20precipitation%20method%3A%20a%20common%20mini-scale%20preparation%20of%20template%20DNA%20from%20phagemids%2C%20phages%20or%20plasmids%20suitable%20for%20sequencing&amp;author=SG.%20Del&amp;author=C.%20Manfioletti%20GSchneider&amp;journal=BioTechniques&amp;volume=7&amp;issue=5&amp;pages=514-520&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Janssen P, Coopman R, Huys G, Swings J, Bleeker M, Vos P, Zabeau MKersters K. Evaluation of the DNA fingerprinting method AFLP as an new tool in bacterial taxonomy. Microbiology. 1996;142(7):1881.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1099/13500872-142-7-1881"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluation%20of%20the%20DNA%20fingerprinting%20method%20AFLP%20as%20an%20new%20tool%20in%20bacterial%20taxonomy&amp;author=P.%20Janssen&amp;author=R.%20Coopman&amp;author=G.%20Huys&amp;author=J.%20Swings&amp;author=M.%20Bleeker&amp;author=P.%20Vos&amp;author=K.%20Zabeau%20MKersters&amp;journal=Microbiology&amp;volume=142&amp;issue=7&amp;pages=1881&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vos P, Hogers R, Bleeker M, Reijans M, Lee TVD, Hornes M, Friters A, Pot J, Paleman JKuiper M. AFLP: a new technique for DNA fingerprinting. Nucleic Acids Res. 1995;23(21):4407.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/23.21.4407"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AFLP%3A%20a%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;author=M.%20Reijans&amp;author=TVD.%20Lee&amp;author=M.%20Hornes&amp;author=A.%20Friters&amp;author=J.%20Pot&amp;author=M.%20Paleman%20JKuiper&amp;journal=Nucleic%20Acids%20Res&amp;volume=23&amp;issue=21&amp;pages=4407&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roldanruiz I, Dendauw J, Evan B, Depicker AMde L. AFLP markers reveal high polymorphic rates in ryegrasses (Lolium spp.). Mol Breeding. 2000;6(2):125â34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1009680614564"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AFLP%20markers%20reveal%20high%20polymorphic%20rates%20in%20ryegrasses%20%28Lolium%20spp.%29&amp;author=I.%20Roldanruiz&amp;author=J.%20Dendauw&amp;author=B.%20Evan&amp;author=L.%20Depicker%20AMde&amp;journal=Mol%20Breeding&amp;volume=6&amp;issue=2&amp;pages=125-134&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Powell W, Morgante M, Andre C, Hanafey M, Vogel J, Tingey SRafalski A. The comparison of RFLP, RAPD, AFLP and SSR (microsatellite) markers for germplasm analysis. Mol Breeding. 1996;2(3):225â38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00564200"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20comparison%20of%20RFLP%2C%20RAPD%2C%20AFLP%20and%20SSR%20%28microsatellite%29%20markers%20for%20germplasm%20analysis&amp;author=W.%20Powell&amp;author=M.%20Morgante&amp;author=C.%20Andre&amp;author=M.%20Hanafey&amp;author=J.%20Vogel&amp;author=A.%20Tingey%20SRafalski&amp;journal=Mol%20Breeding.&amp;volume=2&amp;issue=3&amp;pages=225-238&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ramakishana W, Lagu MD, Gupta VS, Ranjekar PK. DNA fingerprinting in rice using oligonucleotide probes specific for simple repetitive DNA sequences. Theor Appl Genet. 1994;88(3â4):402â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00223651"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20fingerprinting%20in%20rice%20using%20oligonucleotide%20probes%20specific%20for%20simple%20repetitive%20DNA%20sequences&amp;author=W.%20Ramakishana&amp;author=MD.%20Lagu&amp;author=VS.%20Gupta&amp;author=PK.%20Ranjekar&amp;journal=Theor%20Appl%20Genet&amp;volume=88&amp;issue=3%E2%80%934&amp;pages=402-406&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hampl V, PavlÃ­cek A, Flegr J. Construction and bootstrap analysis of DNA fingerprinting-based phylogenetic trees with the freeware program FreeTree: application to trichomonad parasites. Int J Syst Evol Microbiol. 2001;51(Pt 3):731.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1099/00207713-51-3-731"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20and%20bootstrap%20analysis%20of%20DNA%20fingerprinting-based%20phylogenetic%20trees%20with%20the%20freeware%20program%20FreeTree%3A%20application%20to%20trichomonad%20parasites&amp;author=V.%20Hampl&amp;author=A.%20Pavl%C3%ADcek&amp;author=J.%20Flegr&amp;journal=Int%20J%20Syst%20Evol%20Microbiol&amp;volume=51&amp;issue=Pt%203&amp;pages=731&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Forsberg NEG, Russell J, Macaulay M, Leino MW, Hagenblad J. Farmers without borders genetic structuring in century old barley (Hordeum vulgare). Heredity. 2015;114(2):195.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/hdy.2014.83"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Farmers%20without%20borders%20genetic%20structuring%20in%20century%20old%20barley%20%28Hordeum%20vulgare%29&amp;author=NEG.%20Forsberg&amp;author=J.%20Russell&amp;author=M.%20Macaulay&amp;author=MW.%20Leino&amp;author=J.%20Hagenblad&amp;journal=Heredity&amp;volume=114&amp;issue=2&amp;pages=195&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14(8):2611â20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02553.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;journal=Mol%20Ecol&amp;volume=14&amp;issue=8&amp;pages=2611-2620&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jakobsson M, Rosenberg NA. CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure. Bioinformatics. 2007;23(14):1801â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm233"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=CLUMPP%3A%20a%20cluster%20matching%20and%20permutation%20program%20for%20dealing%20with%20label%20switching%20and%20multimodality%20in%20analysis%20of%20population%20structure&amp;author=M.%20Jakobsson&amp;author=NA.%20Rosenberg&amp;journal=Bioinformatics&amp;volume=23&amp;issue=14&amp;pages=1801-1806&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rosenberg NA. Distruct: a program for the graphical display of population structure. Mol. Ecol. 2004;4(1):137â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Distruct%3A%20a%20program%20for%20the%20graphical%20display%20of%20population%20structure.%20Mol&amp;author=NA.%20Rosenberg&amp;journal=Ecol&amp;volume=4&amp;issue=1&amp;pages=137-138&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chibiao C, Eric D, Florence F, Olivier F. Bayesian clustering algorithms ascertaining spatial population structure: a new computer program and a comparison study. Mol Ecol Notes. 2007;7(5):747â56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2007.01769.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bayesian%20clustering%20algorithms%20ascertaining%20spatial%20population%20structure%3A%20a%20new%20computer%20program%20and%20a%20comparison%20study&amp;author=C.%20Chibiao&amp;author=D.%20Eric&amp;author=F.%20Florence&amp;author=F.%20Olivier&amp;journal=Mol%20Ecol%20Notes&amp;volume=7&amp;issue=5&amp;pages=747-756&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Safner T, Miller MP, Mcrae BH, Fortin MJ, Manel S. Comparison of Bayesian clustering and edge detection methods for inferring boundaries in landscape genetics. Int J Mol Sci. 2011;12(2):865â89.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/ijms12020865"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20Bayesian%20clustering%20and%20edge%20detection%20methods%20for%20inferring%20boundaries%20in%20landscape%20genetics&amp;author=T.%20Safner&amp;author=MP.%20Miller&amp;author=BH.%20Mcrae&amp;author=MJ.%20Fortin&amp;author=S.%20Manel&amp;journal=Int%20J%20Mol%20Sci&amp;volume=12&amp;issue=2&amp;pages=865-889&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Holderegger P, Taberlet O. FranÃ§ois. Forecasting changes in population genetic structure of alpine plants in response to global warming. Mol Ecol. 2012;21(10):2354â68.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2012.05541.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fran%C3%A7ois.%20Forecasting%20changes%20in%20population%20genetic%20structure%20of%20alpine%20plants%20in%20response%20to%20global%20warming&amp;author=P.%20Holderegger&amp;author=O.%20Taberlet&amp;journal=Mol%20Ecol&amp;volume=21&amp;issue=10&amp;pages=2354-2368&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peakall R, Smouse PE. GenAlEx 6.5: genetic analysis in excel. Population genetic software for teaching and researchâan update. Bioinformatics. 2012;28(28):2537â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts460"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GenAlEx%206.5%3A%20genetic%20analysis%20in%20excel.%20Population%20genetic%20software%20for%20teaching%20and%20research%E2%80%94an%20update&amp;author=R.%20Peakall&amp;author=PE.%20Smouse&amp;journal=Bioinformatics&amp;volume=28&amp;issue=28&amp;pages=2537-2539&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bohonak AJ. IBD (isolation by distance): a program for analyses of isolation by distance. J Hered. 2002;93(2):153â4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/93.2.153"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=IBD%20%28isolation%20by%20distance%29%3A%20a%20program%20for%20analyses%20of%20isolation%20by%20distance&amp;author=AJ.%20Bohonak&amp;journal=J%20Hered&amp;volume=93&amp;issue=2&amp;pages=153-154&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen SY, Ma X, Zhang XQ, Chen ZH. Genetic variation and geographical divergence in Elymus nutans Griseb. (Poaceae: Triticeae) from West China. Biochem Syst Ecol. 2009;37(6):716â22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2009.12.005"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variation%20and%20geographical%20divergence%20in%20Elymus%20nutans%20Griseb.%20%28Poaceae%3A%20Triticeae%29%20from%20West%20China&amp;author=SY.%20Chen&amp;author=X.%20Ma&amp;author=XQ.%20Zhang&amp;author=ZH.%20Chen&amp;journal=Biochem%20Syst%20Ecol&amp;volume=37&amp;issue=6&amp;pages=716-722&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tarczyk B. Genetic diversity among Polish landraces of common oat (<em class="EmphasisTypeItalic">Avena sativa</em> L.). Genet Resour Crop Ev. 2013;60(7):2157â69.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-013-9984-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20among%20Polish%20landraces%20of%20common%20oat%20%28Avena%20sativa%20L.%29&amp;author=B.%20Tarczyk&amp;journal=Genet%20Resour%20Crop%20Ev&amp;volume=60&amp;issue=7&amp;pages=2157-2169&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Blanco EZ, Bajay MM, Siqueira MV, Zucchi MI, Pinheiro JB. Genetic diversity and structure of Brazilian ginger germplasm (Zingiber officinale) revealed by AFLP markers. Genetica. 2016;144(6):1â12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10709-016-9930-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20structure%20of%20Brazilian%20ginger%20germplasm%20%28Zingiber%20officinale%29%20revealed%20by%20AFLP%20markers&amp;author=EZ.%20Blanco&amp;author=MM.%20Bajay&amp;author=MV.%20Siqueira&amp;author=MI.%20Zucchi&amp;author=JB.%20Pinheiro&amp;journal=Genetica&amp;volume=144&amp;issue=6&amp;pages=1-12&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">FernÃ¡ndez M, Figueiras A, Benito C. The use of ISSR and RAPD markers for detecting DNA polymorphism, genotype identification and genetic diversity among barley cultivars with known origin. Theor Appl Genet. 2002;104(5):845â51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-001-0848-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20use%20of%20ISSR%20and%20RAPD%20markers%20for%20detecting%20DNA%20polymorphism%2C%20genotype%20identification%20and%20genetic%20diversity%20among%20barley%20cultivars%20with%20known%20origin&amp;author=M.%20Fern%C3%A1ndez&amp;author=A.%20Figueiras&amp;author=C.%20Benito&amp;journal=Theor%20Appl%20Genet&amp;volume=104&amp;issue=5&amp;pages=845-851&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Varshney RK, Chabane K, Hendre PS, Aggarwal RK, Graner A. Comparative assessment of EST-SSR, EST-SNP and AFLP markers for evaluation of genetic diversity and conservation of genetic resources using wild, cultivated and elite barleys. Plant Sci. 2007;173(6):638â49.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plantsci.2007.08.010"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20assessment%20of%20EST-SSR%2C%20EST-SNP%20and%20AFLP%20markers%20for%20evaluation%20of%20genetic%20diversity%20and%20conservation%20of%20genetic%20resources%20using%20wild%2C%20cultivated%20and%20elite%20barleys&amp;author=RK.%20Varshney&amp;author=K.%20Chabane&amp;author=PS.%20Hendre&amp;author=RK.%20Aggarwal&amp;author=A.%20Graner&amp;journal=Plant%20Sci&amp;volume=173&amp;issue=6&amp;pages=638-649&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Laurentin H, Karlovsky P. AFLP fingerprinting of sesame (Sesamum indicum L.) cultivars: identification, genetic relationship and comparison of AFLP informativeness parameters. Genet Resour Crop Ev. 2007;54(7):1437â46.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-006-9128-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AFLP%20fingerprinting%20of%20sesame%20%28Sesamum%20indicum%20L.%29%20cultivars%3A%20identification%2C%20genetic%20relationship%20and%20comparison%20of%20AFLP%20informativeness%20parameters&amp;author=H.%20Laurentin&amp;author=P.%20Karlovsky&amp;journal=Genet%20Resour%20Crop%20Ev.&amp;volume=54&amp;issue=7&amp;pages=1437-1446&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun M, Zhang CL, Zhang XQ, Fan Y, Fu KX, Wu WD, Bai SQ, Zhang JB, Peng Y, Huang LK. AFLP assessment of genetic variability and relationships in an Asian wild germplasm collection of Dactylis glomerata L. C R Biol. 2017;340(3):145â55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.crvi.2016.12.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AFLP%20assessment%20of%20genetic%20variability%20and%20relationships%20in%20an%20Asian%20wild%20germplasm%20collection%20of%20Dactylis%20glomerata%20L&amp;author=M.%20Sun&amp;author=CL.%20Zhang&amp;author=XQ.%20Zhang&amp;author=Y.%20Fan&amp;author=KX.%20Fu&amp;author=WD.%20Wu&amp;author=SQ.%20Bai&amp;author=JB.%20Zhang&amp;author=Y.%20Peng&amp;author=LK.%20Huang&amp;journal=C%20R%20Biol&amp;volume=340&amp;issue=3&amp;pages=145-155&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mian MAR, Zwonitzer JC, Chen Y, Saha MC, Hopkins AA. AFLP diversity within and among Hardinggrass populations. Crop Sci. 2005;45(6):2591â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2005.04-0029"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AFLP%20diversity%20within%20and%20among%20Hardinggrass%20populations&amp;author=MAR.%20Mian&amp;author=JC.%20Zwonitzer&amp;author=Y.%20Chen&amp;author=MC.%20Saha&amp;author=AA.%20Hopkins&amp;journal=Crop%20Sci&amp;volume=45&amp;issue=6&amp;pages=2591-2597&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ghehsareh MG, Salehi H, Khosh-Khui M, Niazi A. Application of ISSR markers to analyze molecular relationships in Iranian jasmine ( Jasminum spp.) accessions. Mol Biotechnol. 2015;57(1):65â74.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12033-014-9802-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Application%20of%20ISSR%20markers%20to%20analyze%20molecular%20relationships%20in%20Iranian%20jasmine%20%28%20Jasminum%20spp.%29%20accessions&amp;author=MG.%20Ghehsareh&amp;author=H.%20Salehi&amp;author=M.%20Khosh-Khui&amp;author=A.%20Niazi&amp;journal=Mol%20Biotechnol&amp;volume=57&amp;issue=1&amp;pages=65-74&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gilbert JE, Lewis RV, Wilkinson MJ, Caligari PDS. Developing an appropriate strategy to assess genetic variability in plant germplasm collections. Theor Appl Genet. 1999;98(6â7):1125â31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051176"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Developing%20an%20appropriate%20strategy%20to%20assess%20genetic%20variability%20in%20plant%20germplasm%20collections&amp;author=JE.%20Gilbert&amp;author=RV.%20Lewis&amp;author=MJ.%20Wilkinson&amp;author=PDS.%20Caligari&amp;journal=Theor%20Appl%20Genet&amp;volume=98&amp;issue=6%E2%80%937&amp;pages=1125-1131&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie WG, Zhao XH, Zhang JQ, Wang YR, Liu WX. Assessment of genetic diversity of Siberian wild rye ( <em class="EmphasisTypeItalic">Elymus sibiricus</em> L.) germplasms with variation of seed shattering and implication for future genetic improvement. Biochem Syst Ecol. 2015;58:211â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2014.12.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20of%20Siberian%20wild%20rye%20%28%20Elymus%20sibiricus%20L.%29%20germplasms%20with%20variation%20of%20seed%20shattering%20and%20implication%20for%20future%20genetic%20improvement&amp;author=WG.%20Xie&amp;author=XH.%20Zhao&amp;author=JQ.%20Zhang&amp;author=YR.%20Wang&amp;author=WX.%20Liu&amp;journal=Biochem%20Syst%20Ecol&amp;volume=58&amp;pages=211-218&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maul J, Mirsky S, Emche S, Devine T. Evaluating a germplasm collection of the cover crop hairy vetch for use in sustainable farming systems. Crop Sci. 2011;51(6):2615.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2010.09.0561"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluating%20a%20germplasm%20collection%20of%20the%20cover%20crop%20hairy%20vetch%20for%20use%20in%20sustainable%20farming%20systems&amp;author=J.%20Maul&amp;author=S.%20Mirsky&amp;author=S.%20Emche&amp;author=T.%20Devine&amp;journal=Crop%20Sci&amp;volume=51&amp;issue=6&amp;pages=2615&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang JC, Xie WG, Wang YR, Zhao XH. Potential of start codon targeted (SCoT) markers to estimate genetic diversity and relationships among Chinese Elymus sibiricus accessions. Molecules. 2015;20(4):5987â6001.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/molecules20045987"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Potential%20of%20start%20codon%20targeted%20%28SCoT%29%20markers%20to%20estimate%20genetic%20diversity%20and%20relationships%20among%20Chinese%20Elymus%20sibiricus%20accessions&amp;author=JC.%20Zhang&amp;author=WG.%20Xie&amp;author=YR.%20Wang&amp;author=XH.%20Zhao&amp;journal=Molecules&amp;volume=20&amp;issue=4&amp;pages=5987-6001&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhou Q, Luo D, Ma L, Xie W, Wang Y, Wang YLiu Z. Development and cross-species transferability of EST-SSR markers in Siberian wildrye (<em class="EmphasisTypeItalic">Elymus sibiricus</em> L.) using Illumina sequencing. Sci Rep. 2016;6:20549.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/srep20549"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20and%20cross-species%20transferability%20of%20EST-SSR%20markers%20in%20Siberian%20wildrye%20%28Elymus%20sibiricus%20L.%29%20using%20Illumina%20sequencing&amp;author=Q.%20Zhou&amp;author=D.%20Luo&amp;author=L.%20Ma&amp;author=W.%20Xie&amp;author=Y.%20Wang&amp;author=Z.%20Wang%20YLiu&amp;journal=Sci%20Rep&amp;volume=6&amp;pages=20549&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Benor S, Hammer KBlattner FR. Genetic diversity and relationships in Corchorus olitorius (Malvaceae s.L.) inferred from molecular and morphological data. Genet Resour Crop Ev. 2012;59(6):1125â46.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-011-9748-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20relationships%20in%20Corchorus%20olitorius%20%28Malvaceae%20s.L.%29%20inferred%20from%20molecular%20and%20morphological%20data&amp;author=S.%20Benor&amp;author=FR.%20Hammer%20KBlattner&amp;journal=Genet%20Resour%20Crop%20Ev.&amp;volume=59&amp;issue=6&amp;pages=1125-1146&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen SY, Zhang XQ, Ma X, Huang LK. Assessment of genetic diversity and differentiation of Elymus nutans indigenous to Qinghai-Tibet plateau using simple sequence repeats markers. Can J Plant Sci. 2013;93(6):1089â96.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4141/cjps2013-062"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20and%20differentiation%20of%20Elymus%20nutans%20indigenous%20to%20Qinghai-Tibet%20plateau%20using%20simple%20sequence%20repeats%20markers&amp;author=SY.%20Chen&amp;author=XQ.%20Zhang&amp;author=X.%20Ma&amp;author=LK.%20Huang&amp;journal=Can%20J%20Plant%20Sci&amp;volume=93&amp;issue=6&amp;pages=1089-1096&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ma X, Zhang XQ, Zhou YH, Bai SQ, Liu W. Assessing genetic diversity of Elymus sibiricus (Poaceae: Triticeae) populations from Qinghai-Tibet plateau by ISSR markers. Biochem Syst Ecol. 2008;36(7):514â22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2008.03.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessing%20genetic%20diversity%20of%20Elymus%20sibiricus%20%28Poaceae%3A%20Triticeae%29%20populations%20from%20Qinghai-Tibet%20plateau%20by%20ISSR%20markers&amp;author=X.%20Ma&amp;author=XQ.%20Zhang&amp;author=YH.%20Zhou&amp;author=SQ.%20Bai&amp;author=W.%20Liu&amp;journal=Biochem%20Syst%20Ecol&amp;volume=36&amp;issue=7&amp;pages=514-522&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nybom H. Comparison of different nuclear DNA markers for estimating intraspecific genetic diversity in plants. Mol Ecol. 2004;13(5):1143â55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2004.02141.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20different%20nuclear%20DNA%20markers%20for%20estimating%20intraspecific%20genetic%20diversity%20in%20plants&amp;author=H.%20Nybom&amp;journal=Mol%20Ecol&amp;volume=13&amp;issue=5&amp;pages=1143-1155&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR59" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Greene SL, Gritsenko M, Vandemark G. Relating morphologic and RAPD marker Varlation to collection site environment in wild populations of red clover (Trifolium Pratense L.). Genet Resour Crop Ev. 2004;51(6):643â53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/B%3AGRES.0000024655.48989.ab"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Relating%20morphologic%20and%20RAPD%20marker%20Varlation%20to%20collection%20site%20environment%20in%20wild%20populations%20of%20red%20clover%20%28Trifolium%20Pratense%20L.%29&amp;author=SL.%20Greene&amp;author=M.%20Gritsenko&amp;author=G.%20Vandemark&amp;journal=Genet%20Resour%20Crop%20Ev&amp;volume=51&amp;issue=6&amp;pages=643-653&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR60" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang YL. Effect of row spacing and seeding rate on the forage and seed yield of Elymus tagutorum. Chin Qinghai J Anim Vet Sci. 2015;45(3):7â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20row%20spacing%20and%20seeding%20rate%20on%20the%20forage%20and%20seed%20yield%20of%20Elymus%20tagutorum&amp;author=YL.%20Wang&amp;journal=Chin%20Qinghai%20J%20Anim%20Vet%20Sci&amp;volume=45&amp;issue=3&amp;pages=7-9&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR61" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Loveless MD, Hamrick JL. Ecological determinants of genetic structure in plant populations. Annu Rev Ecol Syst. 1984;15(1):65â95.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.es.15.110184.000433"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ecological%20determinants%20of%20genetic%20structure%20in%20plant%20populations&amp;author=MD.%20Loveless&amp;author=JL.%20Hamrick&amp;journal=Annu%20Rev%20Ecol%20Syst&amp;volume=15&amp;issue=1&amp;pages=65-95&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR62" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hamrick JL, Godt MJW. Effects of life history traits on genetic diversity in plant species. Phil. Trans. Royal Soc. of London, Ser. B. 1996;351(1345):1291â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effects%20of%20life%20history%20traits%20on%20genetic%20diversity%20in%20plant%20species.%20Phil.%20Trans.%20Royal%20Soc.%20of%20London%2C%20Ser&amp;author=JL.%20Hamrick&amp;author=MJW.%20Godt&amp;journal=B&amp;volume=351&amp;issue=1345&amp;pages=1291-1298&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR63" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Reed BM, Engelmann F, Dulloo ME, Engels JMM. Technical guidelines for the management of field and in vitro germplasm collections. In: IPGRI Handbooks for Genebanks No. 7. Rome: Bioversity International; 2004. p. 95â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Technical%20guidelines%20for%20the%20management%20of%20field%20and%20in%20vitro%20germplasm%20collections&amp;author=BM.%20Reed&amp;author=F.%20Engelmann&amp;author=ME.%20Dulloo&amp;author=JMM.%20Engels&amp;pages=95-96&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR64" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Apelbaumelkaher I. Natural selection causes microscale allozyme diversity in wild barley and a lichen at 'Evolution Canyon', Mt. Carmel, Israel. Heredity. 1997;78(4):373â82.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/hdy.1997.60"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Natural%20selection%20causes%20microscale%20allozyme%20diversity%20in%20wild%20barley%20and%20a%20lichen%20at%20%27Evolution%20Canyon%27%2C%20Mt.%20Carmel%2C%20Israel&amp;author=I.%20Apelbaumelkaher&amp;journal=Heredity&amp;volume=78&amp;issue=4&amp;pages=373-382&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© The Author(s) 2019</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-019-0082-z" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-019-0082-z">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-019-0082-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0082-z?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-019-0082-z #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-019-0082-z #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-019-0082-z #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-019-0082-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-019-0082-z">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-019-0082-z"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-019-0082-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-019-0082-z">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-019-0082-z"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-019-0082-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-019-0082-z">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-019-0082-z"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-019-0082-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-019-0082-z">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-019-0082-z"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-019-0082-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-019-0082-z">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-019-0082-z"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-019-0082-z/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-019-0082-z">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-019-0082-z">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-019-0082-z" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-019-0082-z&amp;volume=156&amp;issue=1&amp;title=Genetic+diversity+and+structure+of+Elymus+tangutorum+accessions+from+western+China+as+unraveled+by+AFLP+markers&amp;author_name=Wen-Dan+Wu&amp;start_page=1&amp;end_page=13">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-019-0082-z" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
