<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Construction of a high-density genetic map: genotyping by sequencing (GBS) to map purple seed coat color (Psc) in hulless barley | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Colored hulless barley are more suitable in food processing compared to normal (yellow) varieties because it is rich in bioactive compounds and produces higher extraction pearling fractions. Therefore, seed coat color is an important agronomic trait for the breeding and study of hulless barley. Genotyping-by-sequencing single-nucleotide polymorphism (GBS-SNP) analysis of a doubled haploid (DH) mapping population (Nierumuzha Ã Kunlun10) was conducted to map the purple seed coat color genes (Psc). A high-density genetic map of hulless barley was constructed, which contains 3662 efficient SNP markers with 1129 bin markers. Seven linkage groups were resolved, which had a total length of 645.56Â cM. Chromosome length ranged from 60.21Â cM to 127.21Â cM, with average marker density of 0.57Â cM. A total of five loci accounting for 3.79% to 23.86% of the observed phenotypic variation for Psc were detected using this high-density map. Five structural candidate genes (F3âM, HID, UF3GT, UFGT and 5MAT) and one regulatory factor (Ant1) related to flavonoid or anthocyanin biosynthesis were identified.. Five structural candidate genes and one regulatory factor related to flavonoid or anthocyanin biosynthesis have been identified using a high-density genetic map of hulless barley. This study lays the foundation for map-based cloning of Psc but provides a valuable tool for studying marker-trait associations and its application to marker-assisted breeding of hulless barley.">
<meta name="description" content="Colored hulless barley are more suitable in food processing compared to normal (yellow) varieties because it is rich in bioactive compounds and produces higher extraction pearling fractions. Therefore, seed coat color is an important agronomic trait for the breeding and study of hulless barley. Genotyping-by-sequencing single-nucleotide polymorphism (GBS-SNP) analysis of a doubled haploid (DH) mapping population (Nierumuzha Ã Kunlun10) was conducted to map the purple seed coat color genes (Psc). A high-density genetic map of hulless barley was constructed, which contains 3662 efficient SNP markers with 1129 bin markers. Seven linkage groups were resolved, which had a total length of 645.56Â cM. Chromosome length ranged from 60.21Â cM to 127.21Â cM, with average marker density of 0.57Â cM. A total of five loci accounting for 3.79% to 23.86% of the observed phenotypic variation for Psc were detected using this high-density map. Five structural candidate genes (F3âM, HID, UF3GT, UFGT and 5MAT) and one regulatory factor (Ant1) related to flavonoid or anthocyanin biosynthesis were identified.. Five structural candidate genes and one regulatory factor related to flavonoid or anthocyanin biosynthesis have been identified using a high-density genetic map of hulless barley. This study lays the foundation for map-based cloning of Psc but provides a valuable tool for studying marker-trait associations and its application to marker-assisted breeding of hulless barley.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Construction of a high-density genetic map: genotyping by sequencing (GBS) to map purple seed coat color ( Psc ) in hulless barley">
<meta name="dc.source" content="Hereditas 2018 155:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2018-11-17">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2018 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Colored hulless barley are more suitable in food processing compared to normal (yellow) varieties because it is rich in bioactive compounds and produces higher extraction pearling fractions. Therefore, seed coat color is an important agronomic trait for the breeding and study of hulless barley. Genotyping-by-sequencing single-nucleotide polymorphism (GBS-SNP) analysis of a doubled haploid (DH) mapping population (Nierumuzha × Kunlun10) was conducted to map the purple seed coat color genes (Psc). A high-density genetic map of hulless barley was constructed, which contains 3662 efficient SNP markers with 1129 bin markers. Seven linkage groups were resolved, which had a total length of 645.56 cM. Chromosome length ranged from 60.21 cM to 127.21 cM, with average marker density of 0.57 cM. A total of five loci accounting for 3.79% to 23.86% of the observed phenotypic variation for Psc were detected using this high-density map. Five structural candidate genes (F3’M, HID, UF3GT, UFGT and 5MAT) and one regulatory factor (Ant1) related to flavonoid or anthocyanin biosynthesis were identified.. Five structural candidate genes and one regulatory factor related to flavonoid or anthocyanin biosynthesis have been identified using a high-density genetic map of hulless barley. This study lays the foundation for map-based cloning of Psc but provides a valuable tool for studying marker-trait associations and its application to marker-assisted breeding of hulless barley.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2018-11-17">
<meta name="prism.volume" content="155">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="37">
<meta name="prism.copyright" content="2018 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0072-6">
<meta name="prism.doi" content="doi:10.1186/s41065-018-0072-6">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0072-6">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0072-6">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Construction of a high-density genetic map: genotyping by sequencing (GBS) to map purple seed coat color ( Psc ) in hulless barley">
<meta name="citation_volume" content="155">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2018/12">
<meta name="citation_online_date" content="2018/11/17">
<meta name="citation_firstpage" content="37">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-018-0072-6">
<meta name="DOI" content="10.1186/s41065-018-0072-6">
<meta name="citation_doi" content="10.1186/s41065-018-0072-6">
<meta name="dc.creator" content="Xiaohua Yao">
<meta name="dc.creator" content="Kunlun Wu">
<meta name="dc.creator" content="Youhua Yao">
<meta name="dc.creator" content="Yixiong Bai">
<meta name="dc.creator" content="Jingxiu Ye">
<meta name="dc.creator" content="Dezhao Chi">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=Virus-induced silencing of genes encoding LEA protein in Tibetan hulless barley (Hordeumvulgare ssp. vulgare) and their relationship to drought tolerance; citation_author=JJ Liang, GD Deng, H Long, ZF Pan, CP Wang, P Cai, DL Xu, ZX Nimaand, MQ Yu; citation_volume=30; citation_issue=1; citation_publication_date=2012; citation_pages=441-451; citation_doi=10.1007/s11032-011-9633-3; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Co-location of seed oil content, seed hull content and seed coat color QTL in three different environments in Brassicanapus L; citation_author=XY Yan, JN Li, FY Fu, MY Jin, L Chen, LZ Liu; citation_volume=170; citation_issue=3; citation_publication_date=2009; citation_pages=355-364; citation_doi=10.1007/s10681-009-0006-5; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Regulation of the flavonoid biosynthesis pathway genesin purple and black grains of Hordeumvulgare; citation_author=OY Shoeva, HP Mock, V Kukoeva Tatjana, A Borner, EK Khlestkina; citation_volume=11; citation_issue=10; citation_publication_date=2016; citation_pages=e0163782; citation_doi=10.1371/journal.pone.0163782; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=J Agric Food Chem; citation_title=Anthocyanin composition in black, blue, pink, purple and red cereal grains; citation_author=ES Abdel-Aal, JC Young, I Rabalski; citation_volume=54; citation_issue=13; citation_publication_date=2006; citation_pages=4696-4704; citation_doi=10.1021/jf0606609; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=J Agric Food Chem; citation_title=Anthocyanin composition and oxygen radical scavenging capacity (ORAC) of milled and pearled purple, black, and common barley; citation_author=GG Bellido, B Trust; citation_volume=57; citation_issue=3; citation_publication_date=2009; citation_pages=1022-1028; citation_doi=10.1021/jf802846x; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=J Agric Food Chem; citation_title=Antioxidant properties of pearled barley fractions; citation_author=T Madhujith, M Izydorczyk, F Shahidi; citation_volume=54; citation_issue=9; citation_publication_date=2006; citation_pages=3283-3289; citation_doi=10.1021/jf0527504; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=J Agric Food Chem; citation_title=Relationship between phenolic compounds, anthocyanins content and antioxidant activity in colored barley germplasm; citation_author=MJ Kim, NH Jong, J Aekim, JC P ark, MY Kim, JG Kim, SJ Lee, SC Chun, AM Chung; citation_volume=55; citation_issue=12; citation_publication_date=2007; citation_pages=4802-4809; citation_doi=10.1021/jf0701943; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=Carlsb Res Commun; citation_title=Location of the proanthocyanidins in the barley grain; citation_author=S Aastrup, H Outtrup, K Erdal; citation_volume=49; citation_publication_date=1984; citation_pages=105-109; citation_doi=10.1007/BF02913969; citation_id=CR8">
<meta name="citation_reference" content="Harlan HV. Some distinctions in our cultivated barleys with reference to their use in plant breeding. U S Dept Agr Bull. 1914;137:38.">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Ant28 gene for proanthocyanidin synthesis encoding the R2R3 MYB domain protein (Hvmyb10) highly affects grain dormancy in barley; citation_author=E Himi, Y Yamashita, N Haruyama, T Yanagisawa, M Maekawa, S Taketa; citation_volume=188; citation_issue=1; citation_publication_date=2012; citation_pages=141-151; citation_doi=10.1007/s10681-011-0552-5; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=Genetic control of flavonoid biosynthesis in barley; citation_author=JS Badbro; citation_volume=119; citation_issue=2; citation_publication_date=1993; citation_pages=187-204; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Barley Ant17, encoding flavanone 3-hydroxylase (F3H), is a promising target locus for attaining anthocyanin/ proanthocyanidin-free plants without pleiotropic reduction of grain dormancy; citation_author=E Himi, S Taketa; citation_volume=58; citation_issue=1; citation_publication_date=2015; citation_pages=43-53; citation_doi=10.1139/gen-2014-0189; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome; citation_author=J Cockram, J White, DL Zuluaga, D Smith, J Comadran, M Macaulay, ZW Luo, MJ Kearsey, P Werner, DF Marshall; citation_volume=107; citation_issue=50; citation_publication_date=2010; citation_pages=21611-21616; citation_doi=10.1073/pnas.1010179107; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=BMC Genet; citation_title=Development of SNP markers for genes of the phenylpropanoid pathway and their association to kernel and malting traits in barley; citation_author=M Peukert, S Weise, MS Roder, IE Matthies; citation_volume=14; citation_issue=1; citation_publication_date=2013; citation_pages=97-112; citation_doi=10.1186/1471-2156-14-97; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Mol Gen Genet; citation_title=Structure of the Hordeumvulgare gene encoding dihydroflavonol-4-reductase and molecular analysis of ant18 mutants blocked in flavonoid synthesis; citation_author=K Kristiansen, W Rohde; citation_volume=230; citation_issue=1–2; citation_publication_date=1991; citation_pages=49-59; citation_doi=10.1007/BF00290650; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model rganism; citation_author=SW Baxter, JW Davey, JS Johnston; citation_volume=6; citation_issue=4; citation_publication_date=2011; citation_pages=e19315; citation_doi=10.1371/journal.pone.0019315; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=The draft genome of Tibetan hulless barley revealsadaptive patterns to the high stressful Tibetan plateau; citation_author=XQ Zeng, H Long, Z Wang, SC Zhao, YW Tang, ZY Huang, YL Wang, QJ Xu, LK Mao, GB Deng; citation_volume=112; citation_issue=4; citation_publication_date=2015; citation_pages=1095-1100; citation_doi=10.1073/pnas.1423628112; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Construction of high density sweet cherry (PrunusaviumL.) linkage maps using microsatellite markers and SNPs detected by genotyping-by-sequencing(GBS); citation_author=V Guajardo, S Solis, B Sagredo, F Gainza, C Muñoz, K Gasic, P Hinrichsen; citation_volume=10; citation_issue=5; citation_publication_date=2015; citation_pages=e0127750; citation_doi=10.1371/journal.pone.0127750; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=High-density genetic map construction and gene mapping of pericarp color in wax gourd using specific-locus amplified fragment (SLAF) sequencing; citation_author=B Jiang, W Liu, D Xie, Q Peng, X He, Y Lin, ZJ Liang; citation_volume=16; citation_issue=1; citation_publication_date=2015; citation_pages=1035; citation_doi=10.1186/s12864-015-2220-y; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Construction of a high-density genetic map using genotyping by sequencing (GBS) for quantitative trait loci (QTL) analysis of three plant morphological traits in upland cotton (GossypiumhirsutumL.); citation_author=HK Qi, N Wang, WQ Qiao, QH Xu, H Zhou, JB Shi, GT Yan, Q Huang; citation_volume=213; citation_issue=4; citation_publication_date=2017; citation_pages=83-99; citation_doi=10.1007/s10681-017-1867-7; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=G3; citation_title=A high-density SNP genetic linkage map and QTL analysis of growth-related traits in a hybrid family of oysters (Crassostreagigas× Crassostreaangulata) using genotyping-by-sequencing; citation_author=JP Wang, L Li, GF Zhang; citation_volume=6; citation_issue=5; citation_publication_date=2016; citation_pages=1417-1426; citation_doi=10.1534/g3.116.026971; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Genetic dissection of maize plantarchitecture with an ultra-high density binmap based on recombinant inbred lines; citation_author=ZQ Zhou, CS Zhang, Y Zhou, ZF Hao, ZH Wang, X Zeng; citation_volume=7; citation_publication_date=2016; citation_pages=178-192; citation_doi=10.1186/s12864-016-2555-z; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Construction of a high-density genetic map and quantitative trait locus mapping in the manila clam Ruditapesphilippinarum; citation_author=HT Nie, XW Yan, ZM Huo, LW Jiang, P Chen, H Liu, JF Ding, F Yang; citation_volume=7; citation_issue=1; citation_publication_date=2017; citation_pages=229-237; citation_doi=10.1038/s41598-017-00246-0; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate short read alignment with burrows-wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_issue=14; citation_publication_date=2009; citation_pages=1754-1760; citation_doi=10.1093/bioinformatics/btp324; citation_id=CR24">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=1000 genome project data processing subgroup. The sequence alignment/map format and SAMtools; citation_author=H Li, B Handsaker, A Wysoker, T Fennell, J Ruan, N Homer, G Marth, G Abecasis, R Durbin; citation_volume=25; citation_issue=16; citation_publication_date=2009; citation_pages=2078-2079; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR25">
<meta name="citation_reference" content="citation_title=JoinMap® 4.0: software for the calculation of genetic linkage maps in experimental population. Wageningen; citation_inbook_title=The Netherlands; citation_publication_date=2006; citation_id=CR26; citation_author=JW Van Ooijen">
<meta name="citation_reference" content="citation_journal_title=Ann Eugenics; citation_title=The estimation of map distance from recombination values; citation_author=DD Kosambi; citation_volume=12; citation_issue=1; citation_publication_date=1943; citation_pages=172-175; citation_doi=10.1111/j.1469-1809.1943.tb02321.x; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Genetic analysis and molecular mapping of genes controlling seed coat colour in adzuki bean (Vigna angularis); citation_author=HC Yuki, Y Hiroki, R Ogura, S Naomi, S Hitoshi, K Kiyoaki; citation_volume=206; citation_issue=3; citation_publication_date=2015; citation_pages=609-617; citation_doi=10.1007/s10681-015-1461-9; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Agronomie; citation_title=RFLP analysis of resistance to barley yellow mosaic virus complex; citation_author=A Graner, E Bauer, A Kellermann, G Proeseler, G Wenzel, F Ordon; citation_volume=15; citation_publication_date=1995; citation_pages=475-479; citation_doi=10.1051/agro:19950716; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=Mol Breeding.; citation_title=Molecular marker development and linkage analysis in three low phytic acid barley (Hordeum vulgare) mutant lines; citation_author=V Roslinsky, PE Eckstein, V Raboy, BG Rossnagel, GJ Scoles; citation_volume=20; citation_issue=4; citation_publication_date=2007; citation_pages=323-330; citation_doi=10.1007/s11032-007-9094-x; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Mapping resistance to powdery mildew in barley reveals a large-effect nonhost resistance QTL; citation_author=CTR Cynara, PV Jasper, V Anton, H Axel, M Martin, EN Rients; citation_volume=131; citation_issue=5; citation_publication_date=2018; citation_pages=1031-1045; citation_doi=10.1007/s00122-018-3055-0; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Planta; citation_title=Meta-analysis of major QTL for abiotic stress tolerance in barley and implications for barley breeding; citation_author=XC Zhang, S Shabala, A Koutoulis, L Shabala, MX Zhou; citation_volume=245; citation_issue=2; citation_publication_date=2017; citation_pages=283-295; citation_doi=10.1007/s00425-016-2605-4; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Identification of QTLs controlling grain protein concentration using a high-density SNP and SSR linkage map in barley (Hordeum vulgare L.); citation_author=CF Fan, HJ Zhai, HF Wang, YF Yue, MH Zhang, JH Li, SZ Wen, GG Guo, YW Zeng, ZF Ni, MS You; citation_volume=17; citation_publication_date=2017; citation_pages=122-145; citation_doi=10.1186/s12870-017-1067-6; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Advanced-backcross QTL analysis in hulless barley: I. detection of exotic alleles for yield and yield components introgressed from Hordeum vulgare ssp. spontaneum. Intl Agri; citation_author=R Eshghi, S Salayeva, F Ebrahimpour, M Rahimi, M Baraty, J Ojaghi; citation_volume=5; citation_issue=2; citation_publication_date=2013; citation_pages=95-100; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=An evaluation of genotyping by sequencing (GBS) to map the Breviaristatum-e (ari-e) locus in cultivated barley; citation_author=H Liu, M Bayer, A Druka, JR Russell, CA Hackett, J Poland, L Ramsay, PE Hedley, R Waugh; citation_volume=15; citation_issue=1; citation_publication_date=2014; citation_pages=104-115; citation_doi=10.1186/1471-2164-15-104; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=G3; citation_title=A high-density linkage map for Astyanaxmexicanus using genotyping-by-sequencing technology; citation_author=BM Carlson, SW Onusko, JB Gross; citation_volume=5; citation_issue=2; citation_publication_date=2015; citation_pages=241-251; citation_doi=10.1534/g3.114.015438; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=Agron J; citation_title=A genetic study of complementary genes for purple lemma, Palea and pericarp in barley (Hordeum vulgare L.); citation_author=RW Woodward, JW Thieret; citation_volume=45; citation_publication_date=1953; citation_pages=182-185; citation_doi=10.2134/agronj1953.00021962004500050002x; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Barley Genet Newsl; citation_title=New and revised descriptions of barley genes; citation_author=JD Franckowiak, U Lundqvist; citation_volume=26; citation_publication_date=1997; citation_pages=209; citation_id=CR38">
<meta name="citation_reference" content="citation_journal_title=Mol Gen Genomics; citation_title=De novo transcriptome characterization of Lilium ‘Sorbonne’ and key enzymes related to the flavonoid biosynthesis; citation_author=MF Zhang, LM Jiang, DM Zhang, GX Jia; citation_volume=290; citation_issue=1; citation_publication_date=2015; citation_pages=399-412; citation_doi=10.1007/s00438-014-0919-0; citation_id=CR39">
<meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Microsomal flavonoid 3′- monooxygenase from maize seedlings; citation_author=RL Larson, JB Bussard; citation_volume=80; citation_issue=2; citation_publication_date=1986; citation_pages=483-486; citation_doi=10.1104/pp.80.2.483; citation_id=CR40">
<meta name="citation_reference" content="citation_journal_title=Int J Mol Sci; citation_title=The creation and physiological relevance of divergent hydroxylation patterns in the flavonoid pathway; citation_author=H Halbwirth; citation_volume=11; citation_issue=2; citation_publication_date=2010; citation_pages=595-621; citation_doi=10.3390/ijms11020595; citation_id=CR41">
<meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=The Arabidopsis UDP-glycosyl transferases UGT79B2 and UGT79B3, contribute to cold, salt and drought stress tolerance via modulating anthocyanin accumulation; citation_author=P Li, YJ Li, FJ Zhang, GZ Zhang, XY Jiang, HM Yu, BK Hou; citation_volume=89; citation_issue=1; citation_publication_date=2017; citation_pages=85-103; citation_doi=10.1111/tpj.13324; citation_id=CR42">
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol; citation_title=Molecular and biochemical characterization of a novel hydroxycinnamoyl-CoA: anthocyanin 3-O-glucoside-6-O-acyltransferase from Perillafrutescens; citation_author=K Yonekura-Sakakibara, Y Tanaka, M Fukuchi-Mizutani, H Fujiwara, Y Fukui, T Ashikari, Y Murakami, M Yamaguchi, T Kusumi; citation_volume=41; citation_issue=4; citation_publication_date=2000; citation_pages=495-502; citation_doi=10.1093/pcp/41.4.495; citation_id=CR43">
<meta name="citation_reference" content="citation_journal_title=Springer Plus; citation_title=Identification of differentially expressed genes implicated in peel color (red and green) of Dimocarpusconfinis; citation_author=F Jiang, XP Chen, WS Hu, SQ Zheng; citation_volume=5; citation_issue=1; citation_publication_date=2016; citation_pages=1088-1101; citation_doi=10.1186/s40064-016-2743-y; citation_id=CR44">
<meta name="citation_reference" content="citation_journal_title=Funct Integr Genomics; citation_title=The R2R3-MYB, bHLH, WD40, and related transcription factors in flavonoid biosynthesis; citation_author=L Zhao, LP Gao, HX Wang, XT Chen, YS Wang, Y Hua, CL Wei, T Wan XC Xia; citation_volume=13; citation_issue=1; citation_publication_date=2013; citation_pages=75-98; citation_doi=10.1007/s10142-012-0301-4; citation_id=CR45">
<meta name="citation_author" content="Xiaohua Yao">
<meta name="citation_author_institution" content="State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Qinghai Key Laboratory of Hulless Barley Genetics and Breeding, Xining, China">
<meta name="citation_author_institution" content="Qinghai Subcenter of National Hulless Barley Improvement, Xining, China">
<meta name="citation_author" content="Kunlun Wu">
<meta name="citation_author_institution" content="State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Qinghai Key Laboratory of Hulless Barley Genetics and Breeding, Xining, China">
<meta name="citation_author_institution" content="Qinghai Subcenter of National Hulless Barley Improvement, Xining, China">
<meta name="citation_author" content="Youhua Yao">
<meta name="citation_author_institution" content="State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Qinghai Key Laboratory of Hulless Barley Genetics and Breeding, Xining, China">
<meta name="citation_author_institution" content="Qinghai Subcenter of National Hulless Barley Improvement, Xining, China">
<meta name="citation_author" content="Yixiong Bai">
<meta name="citation_author_institution" content="State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Qinghai Key Laboratory of Hulless Barley Genetics and Breeding, Xining, China">
<meta name="citation_author_institution" content="Qinghai Subcenter of National Hulless Barley Improvement, Xining, China">
<meta name="citation_author" content="Jingxiu Ye">
<meta name="citation_author_institution" content="State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, China">
<meta name="citation_author" content="Dezhao Chi">
<meta name="citation_author_institution" content="State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, China">
<meta name="citation_author_institution" content="Qinghai Key Laboratory of Hulless Barley Genetics and Breeding, Xining, China">
<meta name="citation_author_institution" content="Qinghai Subcenter of National Hulless Barley Improvement, Xining, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0072-6',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0072-6","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Hulless barleyÂ·genotyping-by-sequencing (GBS);Purple seed coat color (Psc);Linkage analysis"},"contentInfo":{"imprint":"BioMed Central","title":"Construction of a high-density genetic map: genotyping by sequencing (GBS) to map purple seed coat color ( Psc ) in hulless barley","publishedAt":1542412800000,"publishedAtDate":"2018-11-17","author":["Xiaohua Yao","Kunlun Wu","Youhua Yao","Yixiong Bai","Jingxiu Ye","Dezhao Chi"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0072-6">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0072-6">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Construction of a high-density genetic map: genotyping by sequencing (GBS) to map purple seed coat color ( Psc ) in hulless barley">
<meta property="og:description" content="Colored hulless barley are more suitable in food processing compared to normal (yellow) varieties because it is rich in bioactive compounds and produces higher extraction pearling fractions. Therefore, seed coat color is an important agronomic trait for the breeding and study of hulless barley. Genotyping-by-sequencing single-nucleotide polymorphism (GBS-SNP) analysis of a doubled haploid (DH) mapping population (Nierumuzha Ã Kunlun10) was conducted to map the purple seed coat color genes (Psc). A high-density genetic map of hulless barley was constructed, which contains 3662 efficient SNP markers with 1129 bin markers. Seven linkage groups were resolved, which had a total length of 645.56&amp;nbsp;cM. Chromosome length ranged from 60.21&amp;nbsp;cM to 127.21&amp;nbsp;cM, with average marker density of 0.57&amp;nbsp;cM. A total of five loci accounting for 3.79% to 23.86% of the observed phenotypic variation for Psc were detected using this high-density map. Five structural candidate genes (F3âM, HID, UF3GT, UFGT and 5MAT) and one regulatory factor (Ant1) related to flavonoid or anthocyanin biosynthesis were identified.. Five structural candidate genes and one regulatory factor related to flavonoid or anthocyanin biosynthesis have been identified using a high-density genetic map of hulless barley. This study lays the foundation for map-based cloning of Psc but provides a valuable tool for studying marker-trait associations and its application to marker-assisted breeding of hulless barley.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0072-6;kwrd=Hulless barleyÂ·genotyping-by-sequencing (GBS),Purple seed coat color (Psc),Linkage analysis;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0072-6&amp;kwrd=Hulless%20barley%C3%82%C2%B7genotyping-by-sequencing%20(GBS),Purple%20seed%20coat%20color%20(Psc),Linkage%20analysis&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0072-6&amp;kwrd=Hulless%20barley%C3%82%C2%B7genotyping-by-sequencing%20(GBS),Purple%20seed%20coat%20color%20(Psc),Linkage%20analysis&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Materials and methods">Materials and methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec17" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec18" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Construction of a high-density genetic map: genotyping by sequencing (GBS) to map purple seed coat color (<em xmlns="" class="EmphasisTypeItalic">Psc</em>) in hulless barley</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaohua Yao</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Kunlun Wu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Youhua Yao</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yixiong Bai</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jingxiu Ye</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Dezhao Chi</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup><a href="mailto:qhcdz@163.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-8264-0305" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a>
</li>
</ul></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:37</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0072-6">https://doi.org/10.1186/s41065-018-0072-6</a></p>
<p class="Copyright">Â©  The Author(s) 2018</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>2 July 2018</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>18 October 2018</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">17 November 2018</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Colored hulless barley are more suitable in food processing compared to normal (yellow) varieties because it is rich in bioactive compounds and produces higher extraction pearling fractions. Therefore, seed coat color is an important agronomic trait for the breeding and study of hulless barley.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par100" class="Para">Genotyping-by-sequencing single-nucleotide polymorphism (GBS-SNP) analysis of a doubled haploid (DH) mapping population (Nierumuzha Ã Kunlun10) was conducted to map the purple seed coat color genes (<em xmlns="" class="EmphasisTypeItalic">Psc</em>). A high-density genetic map of hulless barley was constructed, which contains 3662 efficient SNP markers with 1129 bin markers. Seven linkage groups were resolved, which had a total length of 645.56 cM. Chromosome length ranged from 60.21 cM to 127.21 cM, with average marker density of 0.57 cM. A total of five loci accounting for 3.79% to 23.86% of the observed phenotypic variation for <em xmlns="" class="EmphasisTypeItalic">Psc</em> were detected using this high-density map. Five structural candidate genes (<em xmlns="" class="EmphasisTypeItalic">F3âM</em>, <em xmlns="" class="EmphasisTypeItalic">HID</em>, <em xmlns="" class="EmphasisTypeItalic">UF3GT</em>, <em xmlns="" class="EmphasisTypeItalic">UFGT</em> and <em xmlns="" class="EmphasisTypeItalic">5MAT</em>) and one regulatory factor (<em xmlns="" class="EmphasisTypeItalic">Ant1</em>) related to flavonoid or anthocyanin biosynthesis were identified..</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par2" class="Para">Five structural candidate genes and one regulatory factor related to flavonoid or anthocyanin biosynthesis have been identified using a high-density genetic map of hulless barley. This study lays the foundation for map-based cloning of <em xmlns="" class="EmphasisTypeItalic">Psc</em> but provides a valuable tool for studying marker-trait associations and its application to marker-assisted breeding of hulless barley.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Hulless barleyÂ·genotyping-by-sequencing (GBS)</li>
<li class="c-keywords__item" itemprop="about">Purple seed coat color (<em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Psc</em>)</li>
<li class="c-keywords__item" itemprop="about">Linkage analysis</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Hulless barley (<em xmlns="" class="EmphasisTypeItalic">Hordeum vulgare</em> L. <em xmlns="" class="EmphasisTypeItalic">var</em>. <em xmlns="" class="EmphasisTypeItalic">nudum</em> Hook. f.) is a self-reproducing annual species that produces naked grains. Hulless barley is widely grown on the Qinghai-Tibet Plateau. It has served as a staple food for the Tibetan people since the fifth century CE [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. The seed coat color is an important agronomic trait in crops due to its association with unique biological activity and function in healthcare [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Upon maturity, barley grains may display different pigmentations. Barley has a variety of seed coat colors, including yellow, blue, purple, and black [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. The use of colored grains in cereal-based functional foods has been considered based on their high levels of natural antioxidants such as phenolic compounds, anthocyanins and essential amino acids [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. Colored barley varieties have been found to be more suitable for health than the standard (yellow) variety because these produce pearling fractions that are rich in bioactive compounds [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. The development of barley flour, which itself has antioxidant properties, is used in various mainstream foods such as breads, muffins, noodles, and pasta [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. The breeding of colored barley varieties has become an increasingly pertinent issue in agriculture in recent years.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Seed coat color is thought to be associated with the synthesis of anthocyanins, which are flavonoids present in plants [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. The yellow color of barley is attributed to proanthocyanidins synthesized in the seed coat (testa layer) [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. The purple color is associated with anthocyanins synthesized in the pericarp and glumes [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. In barley, a number of mutants that lack anthocyanins or proanthocyanidins (designated <em xmlns="" class="EmphasisTypeItalic">ant</em> mutants) have been documented. The <em xmlns="" class="EmphasisTypeItalic">Ant</em> loci, known as anthocyanin or proanthocyanidin synthesis genes, are classified as <em xmlns="" class="EmphasisTypeItalic">Ant1</em> to <em xmlns="" class="EmphasisTypeItalic">Ant30</em> [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Flavonoid biosynthesis and the associated metabolic pathways have been studied in barley [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Also some candidate genes have been identified. <em xmlns="" class="EmphasisTypeItalic">Ant1</em> (<em xmlns="" class="EmphasisTypeItalic">HvC1</em>), which is located on chromosome 7H, reduces stem anthocyanin content [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Ant2</em> (<em xmlns="" class="EmphasisTypeItalic">HvbHLH1</em>), which is located on chromosome 2HL, regulates anthocyanin pigmentation in the auricles, awns, and lemmata. It is not involved, however, in grain proanthocyanidin pigmentation in grains [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Ant17</em> (flavanone 3-hydroxylase; <em xmlns="" class="EmphasisTypeItalic">F3H</em>), which is located on chromosome 2HL, has been observed in pigmented tissues. It is not found in nonpigmented roots and stems [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Ant18</em> (dihydroflavonol 4-reductase; <em xmlns="" class="EmphasisTypeItalic">DFR</em>), which is located on chromosome 3HL, is involved in both proanthocyanidin and anthocyanin synthesis [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Ant28</em> (<em xmlns="" class="EmphasisTypeItalic">Hvmyb10</em>), which is located on chromosome 3HL, specifically regulates proanthocyanidin synthesis for grain color and dormancy [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Although these genes that are involved in the biosynthesis of anthocyanins in barley have been identified, their association with seed color in hulless barley remains unclear.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Major obstacles to traditional methods for marker development include low efficiency and high cost of generating high-density genetic linkage maps [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. High-throughput sequencing technology has enabled the determination of hulless barley genome sequences [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Therefore, accelerating the process and efficiency of molecular marker-assisted breeding in this plant species is important. A high-density genetic map is a valuable tool in genomic and genetic applications and especially in fine mapping [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. Genotyping-by-sequencing (GBS), a simple and relatively inexpensive procedure, has reduced the complexity of mapping because it is particularly suitable for a large number of samples in genetic map construction [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>â<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Recent advances in GBS technology have allowed the identification of numerous genetic molecular markers at a reasonable cost. This has promoted the development of several high-throughput single-nucleotide polymorphism (SNP) genotyping methods [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">The present study generated a large double haploid (DH) population from a cross between the Nierumuzha and Kunlun10. The seed coat of Kunlun10 is yellow and that of Nierumuzha is deep purple. Dense marker data was obtained for 298 DH individuals using the GBS technology. We identified 3662 efficient SNP markers, which were ultimately refined into 1129 bin markers after screening. The seed coat color of individual plants was determined. This information was used to map the <em xmlns="" class="EmphasisTypeItalic">Psc</em> of hulless barley. The candidate genes and significant loci detected in this study suggested that our approach is cost-effective for fine mapping and can identify rapidly other key phenotypic genes in hulless barley.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Materialsandmethods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Materialsandmethods">Materials and methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Materialsandmethods" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Plant materials and phenotyping</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">A DH mapping population that consisted of 298 individuals was drawn from hulless barley Kunlun10 (yellow seed coat color) as males and Nierumuzha individuals (deep purple seed coat color) were drawn as females (Fig. <span class="InternalRef"><a href="#Fig1">1a</a></span>). Seed color was assessed using the Wanseen seed test system SC-G (Wseen, Hangzhou, China) combined with visual inspection. Mean trait values were averaged from up to three repeats from each genotype in the DH population. Linkage analysis of the seed color was derived from average color estimates.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig1_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig1_HTML.png" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">The phenotype pictures of female, male and four types of seed coat color. <strong class="EmphasisTypeBold">a</strong> The seed coat color of Nierumuzha and Kunlun10; <strong class="EmphasisTypeBold">b</strong> The phenotype of deep purple, purple, light purple, and yellow color seeds</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">DNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Young leaves were collected from seven-week-old plants of parents and progeny after germination in April 2016. The leaves were immediately placed in liquid nitrogen immediately and then stored at ââ80 Â°C. A plant genomic DNA kit (TIANGEN, Beijing, China) was used for genomic DNA isolation in accordance with the manufacturerâs protocol. DNA concentration and purity were assessed using a Nano PhotometerÂ® spectrophotometer (Implen, CA, US). DNA contamination and degradation were assessed on 1.0% agarose gels.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Library construction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">A genotyping-by-sequencing (GBS) (Novogene, Beijing, China) was used in this study to develop the SNP markers. A GBS pre-design experiment was performed. The GBS library was constructed using a double digest of genomic DNA with a combination of <em xmlns="" class="EmphasisTypeItalic">HaeIII, MseI</em>, and <em xmlns="" class="EmphasisTypeItalic">EcoRI</em> enzymes. This was subsequent to adapters with barcodes, after which each sample was amplified in multiplex with selected fragments for library construction. A tight length range was selected (about 50 bp) to maintain a uniform sequence depth among different fragments.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Illumina sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">First, standard analysis of the raw data was conducted. The Illumina HiSeqâ¢ sequencing platform (Illumina, San Diego, CA, USA) was employed for double-stranded (paired-end) 150-bp sequencing. Next, we conducted advanced analyses based on the assessment of the original data and DNA library assembly followed by HiSeq sequencing with removal of reads with low-quality base calls or uncalled bases. Second, we determined the number of reads digested by <em xmlns="" class="EmphasisTypeItalic">Mse</em>I at both ends of each screened read in the progeny subjected to GBS-Seq analysis. We then discarded reads that did not contain these restriction sites. The specific reads were recorded as well as the ratio of the total number of reads to the number of enzyme captured reads. Finally, using an Illumina high-throughput sequencing platform, pair-end sequencing was performed on the selected tags, followed by SNP genotyping and evaluation followed.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Reference genome mapping</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">For GBS, each sample was sequenced and then compared to the reference genome. Paired-end (PE) reads of the clean data from both the parent and offspring were compared to the reference genome by Burrows-Wheeler Aligner (BWA) software. SAM tools software [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] was used to create SAM/BAM format files, which were used to detect mutations, and coverage was determined using Perl scripts. A linkage map was constructed that was based on the hulless barley genome database (reference genome, <span class="ExternalRef"><a href="ftp://ftp.ensemblgenomes.org/pub/release-29/plants/fasta/hordeum_vulgare/dna/" target="_blank" class="ext-link"><span class="RefSource">ftp://ftp.ensemblgenomes.org/pub/release-29/plants/fasta/hordeum_vulgare/dna/</span></a></span>).</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Sequence data analysis and SNP identification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">The barcodes were used to sort the sequences of each sample. To ensure that the reads were without artificial bias and were reliable (low-quality paired reads, which are mainly caused by base-calling duplicates and adapter contamination), raw data (raw reads) in FASTQ format were first processed in the subsequent analyses in a series of quality control (QC) procedures using in-house C scripts. The QC standards were as follows: (1) removal of reads with &gt;â50% bases having Phred quality &lt;â5; (2) removal of reads with â¥10% unidentified nucleotides (N); (3) removal of reads containing the enzyme <em xmlns="" class="EmphasisTypeItalic">HaeIII</em> and <em xmlns="" class="EmphasisTypeItalic">EcoRI</em> enzyme restriction sites; and (4) Removal of reads with &gt;â10 nt aligned to the adapter, allowing â¤10% mismatches.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">SNP calling in parents and progeny was performed using the SAMtools software [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. The SNPs and types of transversions or transitions were counted. A Perl script was then employed to filter the SNPs that had more than two genotypes. The parental polymorphic markers were arranged into eight segregation patterns (hkâÃâhk, aaÃbb, nnâÃânp, ccâÃâab, abÃcc, lmâÃâll, abÃcd and efâÃâeg) according to the cross pollination (CP) model employed by JoinMap 4.0 software [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. Segregation patterns were selected for the genetic mapping of offspring.</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2"><h3 class="Heading">Linkage map construction and linkage analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">Prior to map construction, the markers with integrity &gt;â75%, segregation distortion (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001), or that were in possession of abnormal bases were filtered. The segregation pattern aa Ã bb was used for map construction using JoinMap 4.0. To calculate the marker distances, a regression algorithm, three times circulation sequence, and Kosambi [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] mapping function were used. LOD values were within the range of 2.0â10.0. The integrated map was computed for the male and female parents using the combined group for map integration function in the MergeMap software. A Perl script SVG was used to visualize the exported maps. Heat maps were constructed to evaluate those maps. Linkage analysis was conducted using an LOD threshold of 1000 permutations and a <em xmlns="" class="EmphasisTypeItalic">P</em>ââ¤â0.05. A LOD score of 3.2 was set as the minimum for 1000 permutations to declare that any particular genomic region contained a locus. All candidate genes were categorized according to gene annotations by Swiss-Prot, TrEMBL, the Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<section id="Sec11" class="Section2 RenderAsSection2"><h3 class="Heading">Resequencing of parental lines and GBS of the DH lines</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">The Nierumuzha and Kunlun10 parents were sequenced at effective sequencing depths of about 24.55-fold and 24.19-fold, respectively. 13,682,278 reads of Nierumuzha and 13,395,851 reads of Kunlun10 were mapped to the barley genome. The mapping rates were 96.81% and 96.72%, respectively (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Finally, 224,008 polymorphic loci were identified. The marker genotype data are summarized in Table <span class="InternalRef"><a href="#Tab2">2</a></span>. Only the genotype aa Ã bb, consisting of 20,615 markers between Nierumuzha and Kunlun10, was used in the subsequent for further analysis (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Sequence depth and coverage statistics</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Sample</p></th>
<th char="."><p xmlns="" class="SimplePara">Clean reads<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Mapping reads<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Mapping rate <sup xmlns="http://www.w3.org/1999/xhtml">c</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Average depth<sup xmlns="http://www.w3.org/1999/xhtml">d</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Coverage1Ã<sup xmlns="http://www.w3.org/1999/xhtml">e</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Coverage4Ã<sup xmlns="http://www.w3.org/1999/xhtml">f</sup></p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kunlun10</p></td>
<td char="."><p xmlns="" class="SimplePara">13,850,762</p></td>
<td char="."><p xmlns="" class="SimplePara">13,395,851</p></td>
<td char="."><p xmlns="" class="SimplePara">96.72</p></td>
<td char="."><p xmlns="" class="SimplePara">24.19</p></td>
<td char="."><p xmlns="" class="SimplePara">8.33</p></td>
<td char="."><p xmlns="" class="SimplePara">4.33</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Nierumuzha</p></td>
<td char="."><p xmlns="" class="SimplePara">14,133,242</p></td>
<td char="."><p xmlns="" class="SimplePara">13,682,278</p></td>
<td char="."><p xmlns="" class="SimplePara">96.81</p></td>
<td char="."><p xmlns="" class="SimplePara">24.55</p></td>
<td char="."><p xmlns="" class="SimplePara">8.39</p></td>
<td char="."><p xmlns="" class="SimplePara">4.37</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Number of reads used for the alignment</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Number of clean reads that mapped to the reference genome</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">c</sup>The percentage of reads that mapped to the genome</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">d</sup>Average sequencing depth</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">e</sup>Percentage of the reference genome with at least 19 coverage</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">f</sup>Percentage of the reference genome with at least 49 coverage</p>
                    </div></figure><figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Marker types</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Marker type<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Numbers<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">hkâÃâhk</p></td>
<td char="."><p xmlns="" class="SimplePara">141,005</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">aaÃbb</p></td>
<td char="."><p xmlns="" class="SimplePara">20,615</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">nnâÃânp</p></td>
<td char="."><p xmlns="" class="SimplePara">30,054</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ccâÃâab</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">abÃcc</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">lmâÃâll</p></td>
<td char="."><p xmlns="" class="SimplePara">30,310</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">abÃcd</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">efâÃâeg</p></td>
<td char="."><p xmlns="" class="SimplePara">2010</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total markers:</p></td>
<td char="."><p xmlns="" class="SimplePara">224,008</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Parental genotypes (i.e., ab Ã cc, where ab and cc genotypes represent the male and female parents)</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Total number of markers of each type</p>
                    </div></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig2_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig2_HTML.png" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Distribution of SNPs and the aa Ã bb genotype throughout the Nierumuzha and Kunlun10 genome-wide. The outermost box that includes a scale represents the seven chromosomes of hulless barley. The orange histogram represents the density of SNPs that are polymorphic between Kunlun10 and Nierumuzha. The green histogram represents the density of aa Ã bb genotype SNPs that are polymorphic between Kunlun10 and Nierumuzha</p>
                      </div></figcaption></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">The DH lines were then genotyped using the GBS technology. The average number of raw individual reads of the 298 hulless barley lines was about 1.6 Gb. This yielded 490.07 Gb of high-quality sequence reads (Q20ââ¥â95%, Q30%ââ¥â88%) and a normal GC content among the reads. The average <em xmlns="" class="EmphasisTypeItalic">Mse</em>I enzyme capture rate was 93.14% across the GBS of the 298 DH lines, indicating that enzyme digestion was high quality. The average coverage of the 298 offspring was 96.93% of the whole genome, with 21.11 read depth at 8.38% (coverage at least 1Ã) and 3.76% (coverage at least 4Ã) of sites. The screened genotypes contained all of the markers in more than 75% of individual lines. Therefore, at least 227 of the 298 progeny lines contained all the markers.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Genotyping of the progeny and selection of genetic markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">The low-coverage sequences of the DH lines (coverage under 75%) were filtered out. This left 7028 markers out of the original 20,615. Markers with significant distortion (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001) were filtered, and a total of 1549 markers were retained in total for identification of bin markers.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic linkage map with bin markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">Unlinked markers were filtered out, and 3662 SNPs (1129 bin markers) were mapped to seven linkage maps using Joinmap 4.0. A high-density genetic map was constructed after these 1129 bin markers were mapped onto the seven chromosomes of hulless barley. The genetic maps were 645.56 cM in length with an average distance of 0.57 cM between the markers. 3H was the largest group among the seven linkage groups, consisting of 182 markers, and a genetic length of 127.21 cM. 1H was the smallest group, with 103 markers, and a genetic length of 60.21 cM (Table <span class="InternalRef"><a href="#Tab3">3</a></span>, Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). A total of 1124 gaps were detected between markers. Among these, 11,102 gaps (98.04%) wereâ&lt;â5 cM, 15 gaps were between 5 and 10 cM in size, and only seven gaps were between 10 and 20 cM in size. However, there were no gaps over 20 cM in size were observed in any of the chromosomes (Table <span class="InternalRef"><a href="#Tab4">4</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Genetic linkage group statistics</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Chr<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">PD(bp)<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Bin markers<sup xmlns="http://www.w3.org/1999/xhtml">c</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">SNP<sup xmlns="http://www.w3.org/1999/xhtml">d</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">GD(cM)<sup xmlns="http://www.w3.org/1999/xhtml">e</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Average distance(cM)<sup xmlns="http://www.w3.org/1999/xhtml">f</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">Max. gap (cM)<sup xmlns="http://www.w3.org/1999/xhtml">g</sup></p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">1H</p></td>
<td char="."><p xmlns="" class="SimplePara">464,124,043</p></td>
<td char="."><p xmlns="" class="SimplePara">103</p></td>
<td char="."><p xmlns="" class="SimplePara">230</p></td>
<td char="."><p xmlns="" class="SimplePara">60.21</p></td>
<td char="."><p xmlns="" class="SimplePara">0.58</p></td>
<td char="."><p xmlns="" class="SimplePara">4.67</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">2H</p></td>
<td char="."><p xmlns="" class="SimplePara">628,342,783</p></td>
<td char="."><p xmlns="" class="SimplePara">301</p></td>
<td char="."><p xmlns="" class="SimplePara">1340</p></td>
<td char="."><p xmlns="" class="SimplePara">81.43</p></td>
<td char="."><p xmlns="" class="SimplePara">0.27</p></td>
<td char="."><p xmlns="" class="SimplePara">10.82</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">3H</p></td>
<td char="."><p xmlns="" class="SimplePara">564,427,874</p></td>
<td char="."><p xmlns="" class="SimplePara">182</p></td>
<td char="."><p xmlns="" class="SimplePara">406</p></td>
<td char="."><p xmlns="" class="SimplePara">127.21</p></td>
<td char="."><p xmlns="" class="SimplePara">0.70</p></td>
<td char="."><p xmlns="" class="SimplePara">16.02</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">4H</p></td>
<td char="."><p xmlns="" class="SimplePara">544,168,226</p></td>
<td char="."><p xmlns="" class="SimplePara">117</p></td>
<td char="."><p xmlns="" class="SimplePara">312</p></td>
<td char="."><p xmlns="" class="SimplePara">81.36</p></td>
<td char="."><p xmlns="" class="SimplePara">0.70</p></td>
<td char="."><p xmlns="" class="SimplePara">11.05</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">5H</p></td>
<td char="."><p xmlns="" class="SimplePara">561,411,686</p></td>
<td char="."><p xmlns="" class="SimplePara">127</p></td>
<td char="."><p xmlns="" class="SimplePara">240</p></td>
<td char="."><p xmlns="" class="SimplePara">101.94</p></td>
<td char="."><p xmlns="" class="SimplePara">0.80</p></td>
<td char="."><p xmlns="" class="SimplePara">12.86</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">6H</p></td>
<td char="."><p xmlns="" class="SimplePara">538,755,036</p></td>
<td char="."><p xmlns="" class="SimplePara">111</p></td>
<td char="."><p xmlns="" class="SimplePara">541</p></td>
<td char="."><p xmlns="" class="SimplePara">98.76</p></td>
<td char="."><p xmlns="" class="SimplePara">0.89</p></td>
<td char="."><p xmlns="" class="SimplePara">9.36</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">7H</p></td>
<td char="."><p xmlns="" class="SimplePara">601,597,413</p></td>
<td char="."><p xmlns="" class="SimplePara">188</p></td>
<td char="."><p xmlns="" class="SimplePara">593</p></td>
<td char="."><p xmlns="" class="SimplePara">94.63</p></td>
<td char="."><p xmlns="" class="SimplePara">0.50</p></td>
<td char="."><p xmlns="" class="SimplePara">14.84</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">3,902,827,061</p></td>
<td char="."><p xmlns="" class="SimplePara">1129</p></td>
<td char="."><p xmlns="" class="SimplePara">3662</p></td>
<td char="."><p xmlns="" class="SimplePara">645.56</p></td>
<td char="."><p xmlns="" class="SimplePara">0.57</p></td>
<td char="."><p xmlns="" class="SimplePara">16.02</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Chromosome number</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Total physical length of the chromosomes (bp)</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">c</sup>Number of bin markers</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">d</sup>Single Nucleotide Polymorphism</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">e</sup>Total genetic distance of chromosomes (cM)</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">f</sup>Average genetic distance between markers (cM)</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">g</sup>Maximum gap between markers (cM)</p>
                    </div></figure><figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig3_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig3_HTML.png" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig3_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig3_HTML.png" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Distribution map of the linkage groups. Note, the x-axis is the chromosome number, the y-axis is the genetic distance (in cM), and the black line is the bin marker</p>
                      </div></figcaption></figure><figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Linkage map gap statistics</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Chr<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">&lt;â5 cM</p></th>
<th char="."><p xmlns="" class="SimplePara">5to10 cM</p></th>
<th char="."><p xmlns="" class="SimplePara">10to20 cM</p></th>
<th char="."><p xmlns="" class="SimplePara">&gt;â20 cM</p></th>
<th char="."><p xmlns="" class="SimplePara">ratio</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">1H</p></td>
<td char="."><p xmlns="" class="SimplePara">103</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">2H</p></td>
<td char="."><p xmlns="" class="SimplePara">297</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">99.00</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">3H</p></td>
<td char="."><p xmlns="" class="SimplePara">176</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">97.24</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">4H</p></td>
<td char="."><p xmlns="" class="SimplePara">113</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">97.41</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">5H</p></td>
<td char="."><p xmlns="" class="SimplePara">123</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">97.62</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">6H</p></td>
<td char="."><p xmlns="" class="SimplePara">106</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">96.36</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">7H</p></td>
<td char="."><p xmlns="" class="SimplePara">184</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">98.40</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">1102</p></td>
<td char="."><p xmlns="" class="SimplePara">15</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">98.00</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Chromosome number</p>
                    </div></figure>
</div>
              </section><section id="Sec14" class="Section2 RenderAsSection2"><h3 class="Heading">Map quality validation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">A co-linearity analysis indicated that most of the markers in the linkage groups were consistent with the hulless barley reference genome, indicating that our estimation of genetic recombination rate was highly accurate (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). The genetic map that was constructed with the SNP markers discovered in the GBS-Seq analysis had sufficient coverage throughout the barley genome. Most of the SNP loci on the linkage map were in the same order as those on the corresponding chromosomes of the physical map of the hulless barley genome.<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig4_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig4_HTML.png" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig4_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig4_HTML.png" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">The physical map and genetic linkage map. Note, the x-axis is the chromosome number and the y-axis is the genetic distance (in cM). The genetic map is shown in red, the physical map is in blue, and the green line shows the position of each marker on the genetic map and the physical map</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig5_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig5_HTML.png" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig5_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_Fig5_HTML.png" alt="Fig. 5"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Linkage analysis of <em class="EmphasisTypeItalic">Psc</em> in hulless barley among all the linkage groups. Note: the x-axis is the genetic distance of each linkage group, the y-axis is the physical position of bin markers against LOD scores and the red line is the threshold LOD value as determined by a permutation test; 1~â7, seven linkage groups</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec15" class="Section2 RenderAsSection2"><h3 class="Heading">Phenotypic features of DH lines and linkage analysis of <em class="EmphasisTypeItalic">Psc</em>
                        </h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">A DH population of 298 offspring from Nierumuzha Ã Kunlun10 was used in this study. The seed coat color of offspring ranged from deep purple to yellow gradually (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>). We divided them artificially into four categories according to the result of Wanseen seed test system SC-G (27 deep purple seeds, 68 purple seeds, 133 light purple seeds, and 70 yellow seeds) (Fig. <span class="InternalRef"><a href="#Fig1">1b</a></span>) [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Linkage analysis was based on the phenotypic data for <em xmlns="" class="EmphasisTypeItalic">Psc</em> in DH lines at a LOD of 3.2. A total of 5 loci for <em xmlns="" class="EmphasisTypeItalic">Psc</em> were identified: one locus on 4H, explaining 3.79% and four loci on 7H (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>), explaining 13.64%, 17.56%, 23.86%, and 15.26% of phenotypic variation, respectively. The locus peak positions were located on 10.91 cM (4H), 71.81 cM (7H), 77.41 cM (7H), 84.61 cM (7H), and 90.61 cM (7H), respectively, and were located within the marker intervals of 4H-bin857 to 4H- bin863 (275 genes), 7H- bin1533 to 7H- bin1532 (57 genes), 7H- bin1383 to 7H- bin1384 (60 genes), 7H- bin1377 to 7H- bin1372 (248 genes), and 7H- bin1370 to 7H- bin1369 (189 genes) (Table <span class="InternalRef"><a href="#Tab5">5</a></span>).<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">Linkage analysis of <em class="EmphasisTypeItalic">Psc</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col char="Ã">
<col>
<col char=".">
<col char=".">
<col char="â">
<col char=".">
<col char=".">
<col>
<col>
<col char=".">
</colgroup>
<thead><tr class="header">
<th char="Ã"><p xmlns="" class="SimplePara">Site name</p></th>
<th><p xmlns="" class="SimplePara">Chr<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">LOD peak</p></th>
<th char=".">
<p xmlns="" class="SimplePara">Position</p>
<p xmlns="" class="SimplePara">(cM)</p>
</th>
<th char="â"><p xmlns="" class="SimplePara">99% CI (cM)<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></th>
<th char="."><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup>(%)<sup xmlns="http://www.w3.org/1999/xhtml">c</sup></p></th>
<th char=".">
<p xmlns="" class="SimplePara">Additive</p>
<p xmlns="" class="SimplePara">effect</p>
</th>
<th>
<p xmlns="" class="SimplePara">Left</p>
<p xmlns="" class="SimplePara">marker<sup xmlns="http://www.w3.org/1999/xhtml">d</sup></p>
</th>
<th>
<p xmlns="" class="SimplePara">Right</p>
<p xmlns="" class="SimplePara">marker<sup xmlns="http://www.w3.org/1999/xhtml">e</sup></p>
</th>
<th char=".">
<p xmlns="" class="SimplePara">Gene</p>
<p xmlns="" class="SimplePara">number</p>
</th>
</tr></thead>
<tbody>
<tr class="noclass">
<td char="Ã"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Psc-1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">4H</p></td>
<td char="."><p xmlns="" class="SimplePara">4.81</p></td>
<td char="."><p xmlns="" class="SimplePara">10.91</p></td>
<td char="â"><p xmlns="" class="SimplePara">10.5â13.8</p></td>
<td char="."><p xmlns="" class="SimplePara">3.79</p></td>
<td char="."><p xmlns="" class="SimplePara">0.2779</p></td>
<td><p xmlns="" class="SimplePara">bin857</p></td>
<td><p xmlns="" class="SimplePara">bin863</p></td>
<td char="."><p xmlns="" class="SimplePara">275</p></td>
</tr>
<tr class="noclass">
<td char="Ã"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Psc-2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">7H</p></td>
<td char="."><p xmlns="" class="SimplePara">10.45</p></td>
<td char="."><p xmlns="" class="SimplePara">71.81</p></td>
<td char="â"><p xmlns="" class="SimplePara">71â73.1</p></td>
<td char="."><p xmlns="" class="SimplePara">13.64</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.376</p></td>
<td><p xmlns="" class="SimplePara">bin1533</p></td>
<td><p xmlns="" class="SimplePara">bin1532</p></td>
<td char="."><p xmlns="" class="SimplePara">57</p></td>
</tr>
<tr class="noclass">
<td char="Ã"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Psc-3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">7H</p></td>
<td char="."><p xmlns="" class="SimplePara">17.57</p></td>
<td char="."><p xmlns="" class="SimplePara">77.41</p></td>
<td char="â"><p xmlns="" class="SimplePara">77.3â78.5</p></td>
<td char="."><p xmlns="" class="SimplePara">17.56</p></td>
<td char="."><p xmlns="" class="SimplePara">0.443</p></td>
<td><p xmlns="" class="SimplePara">bin1383</p></td>
<td><p xmlns="" class="SimplePara">bin1384</p></td>
<td char="."><p xmlns="" class="SimplePara">60</p></td>
</tr>
<tr class="noclass">
<td char="Ã"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Psc-4</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">7H</p></td>
<td char="."><p xmlns="" class="SimplePara">25.37</p></td>
<td char="."><p xmlns="" class="SimplePara">84.61</p></td>
<td char="â"><p xmlns="" class="SimplePara">84.5â86</p></td>
<td char="."><p xmlns="" class="SimplePara">23.86</p></td>
<td char="."><p xmlns="" class="SimplePara">0.5598</p></td>
<td><p xmlns="" class="SimplePara">bin1377</p></td>
<td><p xmlns="" class="SimplePara">bin1372</p></td>
<td char="."><p xmlns="" class="SimplePara">248</p></td>
</tr>
<tr class="noclass">
<td char="Ã"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Psc-5</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">7H</p></td>
<td char="."><p xmlns="" class="SimplePara">14.34</p></td>
<td char="."><p xmlns="" class="SimplePara">90.61</p></td>
<td char="â"><p xmlns="" class="SimplePara">90.5â90.7</p></td>
<td char="."><p xmlns="" class="SimplePara">15.26</p></td>
<td char="."><p xmlns="" class="SimplePara">â0.3707</p></td>
<td><p xmlns="" class="SimplePara">bin1370</p></td>
<td><p xmlns="" class="SimplePara">bin1369</p></td>
<td char="."><p xmlns="" class="SimplePara">189</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Chromosome number</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">b</sup>99% confidence interval for site length</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">c</sup>Proportion of phenotypic variation explained by each site</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">d</sup>Left boundary markers for each site</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">e</sup>Right boundary markers for each site</p>
                    </div></figure>
</div>
              </section><section id="Sec16" class="Section2 RenderAsSection2"><h3 class="Heading">Prediction of candidate genes</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">Five structural candidate genes and one regulatory factor related to flavonoid or anthocyanin biosynthesis were identified in the <em xmlns="" class="EmphasisTypeItalic">Psc</em> regions within the loci (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>) according to the barley gene annotation database accessible at TrEMBL, NCBI, KEGG and Swiss-Prot. The candidate gene MLOC_6177, located on chromosome 7H (71.00â73.10 cM, see Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>), was annotated to an uncharacterized protein in barley in TrEMBL and flavonoid 3â²-monooxygenase (<em xmlns="" class="EmphasisTypeItalic">F3</em>â<em xmlns="" class="EmphasisTypeItalic">M</em>) in Aegilops (GenBank Accession Number XM_020332250.1) in KEGG and Swiss-Prot. MLOC_71630 located on chromosome 7H (77.30â78.50 cM, see Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>), was annotated to an uncharacterized protein in barley at TrEMBL and 2-hydroxyisoflavanone dehydratase (<em xmlns="" class="EmphasisTypeItalic">HID</em>) in Aegilops in Swissprot. MLOC_62096 located on chromosome 7H (84.50â86.00 cM, see Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>), was annotated to anthocyanidin 3-O-glucoside 2â²â²-O-xylosyltransferase (<em xmlns="" class="EmphasisTypeItalic">UF3GT</em>) in barley in Swiss-Prot. MLOC_38343 located on chromosome 7H (84.50â86.00 cM, see Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>), was annotated anthocyanidin/flavonoid 3-O-glucosyltransferase (<em xmlns="" class="EmphasisTypeItalic">UFGT</em>) in barley in Swiss-Prot. MLOC_32012 located on chromosome 7H (90.50â90.70 cM, see Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>), was annotated to an uncharacterized protein in barley in TrEMBL and malonyl-coenzyme: anthocyanin 5-O-glucoside-6â²â²â²-O-malonyltransferase (<em xmlns="" class="EmphasisTypeItalic">5MAT</em>) in Aegilops in Swiss-Prot. MLOC_6171 located on chromosome 7H (84.50â86.00 cM, see Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>), was annotated to an uncharacterized protein in barley in TrEMBL, but anthocyanin regulatory C1 protein (<em xmlns="" class="EmphasisTypeItalic">Ant1</em>) in barley in NCBI.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec17" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">Linkage analysis is an efficient way to analyze some important traits in barley molecular breeding [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. The quality of genetic maps, however, can significantly affect the accuracy of gene mapping. This increases the density of markers distributed around the entire genome, which can improve the resolution of genetic maps [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>â<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. SSR, one of the traditional gene mapping methods, was used in hulless barley, but the number of SSR markers is limited. Meanwhile, no markers were sufficiently close to key traits to ensure reliable detection after exhaustively testing [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. GBS is a fast, efficient, informative, and cost-effective strategy for SNP discovery, genetic linkage map construction, and genotyping [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. In this study we used a combination of three restriction enzymes (<em xmlns="" class="EmphasisTypeItalic">HaeIII, MseI</em>, and <em xmlns="" class="EmphasisTypeItalic">EcoRI</em>) for GBS library construction. <em xmlns="" class="EmphasisTypeItalic">MseI</em> recognizes a 4-bp restriction site (TTAA). It also has a higher distribution frequency in the Chinese jujube (<em xmlns="" class="EmphasisTypeItalic">Ziziphusjujuba</em>) genome. The use of two additional enzymes, namely, <em xmlns="" class="EmphasisTypeItalic">EcoRI</em> and <em xmlns="" class="EmphasisTypeItalic">HaeIII</em>, was performed to further digest the fragments after <em xmlns="" class="EmphasisTypeItalic">MseI</em>. The three restriction enzyme combination improved the efficiency of GBS by increasing sequencing depth, number of tags, and genome coverage. This combination also allowed the detection of suitable regions for targeted fragments [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. The present study generated the first high-density genetic map of hulless barely using the GBS technology. A total of 490.07 Gb raw sequencing data and 96.93% of clean data were mapped to unique positions on the reference genome. We were able to cost-effectively genotype 3662 SNPs with 1129 bin markers with an average distance of 0.57 cM in 300 samples (2 parents and 298 offspring) of hulless barely using the GBS technology. Such a high-density linkage map will likely be a valuable resource for genomic analyses and fine-scale gene mapping in hulless barely.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">The inheritance of the purple pericarp and lemma has been studied with inconsistent results up to date. Woodward and Thieret [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] obtained 3:1 ratios of purple and nonpurple seeds from 28 crosses of âpurpleââÃâânonpurpleâ barley genotypes, and 9:7 ratios from the crosses in which both parents were non-purple seeds, indicating a two-factor inheritance. To simplify the nomenclature, previously reported symbols were dropped and <em xmlns="" class="EmphasisTypeItalic">Pre1</em> and <em xmlns="" class="EmphasisTypeItalic">Pre2</em> symbols were used for C, c, and P, p, located on 1H and 2H, respectively [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. In this study we found 298 offspring from Nierumuzha Ã Kunlun10 seed coat colors range from deep purple to yellow gradually. In order to shorten the distinguish time of breeding in future, we divided them artificially into four categories: deep purple, purple, light purple and yellow. We thus mapped <em xmlns="" class="EmphasisTypeItalic">Psc</em> as a qualitative trait using the GBS linkage map. The major <em xmlns="" class="EmphasisTypeItalic">Psc</em> loci were located on chromosome 4H (1 locus) and 7H (4 loci). Previous studies have indicated that seed coat color of barley was located on chromosomes 2H [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>] and 4H [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] at a genetic distance greater than 10 cM. In this study, five loci were detected on chromosomes 4H and 7H that one explained 3.79% (length 3.3 cM) of the phenotypic variation, whereas the other four (71.00 cMâ90.70 cM) accounted for 13.64% (length 2.1 cM), 17.56% (length 1.2 cM), 23.86% (length 1.5 cM), and15.26% (length 0.2 cM) of the phenotypic variation according to the high-resolution map (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). There are 23 bin markers between 70.944 cM and 92.072 cM (bin1533-bin1369) on 7H, and the average marker density is 0.92 cM. The average gap is 0.746 cM except the largest one (5.455 cM) between qPSC-2 and qPSC-3 (bin1532-bin1383) (Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>). In order to avoid the wrong judgment in this region, we re-tested candidate genes according to gene annotations by Swissprot, TrEMBL, and the Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis, but didnât found new ones.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">Seed coat color is thought to be associated with the synthesis of anthocyanins. Anthocyanin biosynthesis is well characterized at the enzymatic, genetic, and production levels. These structural genes can be separated into two categories: those of the late flavonoid biosynthetic pathway, including anthyocyanidin synthase (<em xmlns="" class="EmphasisTypeItalic">ANS</em>), dihydroflavonol 4-reductase (<em xmlns="" class="EmphasisTypeItalic">DFR</em>), UDP-sugar: flavonoid-3-O-glucosyltransferase (<em xmlns="" class="EmphasisTypeItalic">UFGT</em>), and flavonoid-5-O-glucosyltransferase (<em xmlns="" class="EmphasisTypeItalic">UF5GT</em>) [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>], and those of the early flavonoid biosynthetic pathway, including chalcone synthase (<em xmlns="" class="EmphasisTypeItalic">CHS</em>), chalcone isomerase (<em xmlns="" class="EmphasisTypeItalic">CHI</em>), phenylalanine ammonia-lyase (<em xmlns="" class="EmphasisTypeItalic">PAL</em>), flavonoid 3-hydroxylase (<em xmlns="" class="EmphasisTypeItalic">F3H</em>), and flavonoid 3â²-hydroxylase (<em xmlns="" class="EmphasisTypeItalic">F3</em>â²<em xmlns="" class="EmphasisTypeItalic">H</em>). Shoeva et al. identified the <em xmlns="" class="EmphasisTypeItalic">Ant2</em> gene and showed that the mRNA levels of flavonoid biosynthesis structural genes <em xmlns="" class="EmphasisTypeItalic">CHS</em>, <em xmlns="" class="EmphasisTypeItalic">CHI</em>, <em xmlns="" class="EmphasisTypeItalic">F3H</em>, <em xmlns="" class="EmphasisTypeItalic">DFR</em>, <em xmlns="" class="EmphasisTypeItalic">F3</em>â²<em xmlns="" class="EmphasisTypeItalic">H</em>, <em xmlns="" class="EmphasisTypeItalic">ANS</em>, and the regulatory gene <em xmlns="" class="EmphasisTypeItalic">ANT2</em> were higher in purple barley than in yellow barley, as indicated by qRT-PCR [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Even though the genes involved in anthocyanin biosynthesis have been identified, their relation to <em xmlns="" class="EmphasisTypeItalic">Psc</em> in hulless barley is currently unknown. In this study, some structural candidate genes and regulatory factors related to flavonoid or anthocyanin biosynthesis were identified in the <em xmlns="" class="EmphasisTypeItalic">Psc</em> regions by gene annotation. Sequence analysis showed the nucleotide sequence of MLOC_6177 from hulless barley to be 99% homologous with a predicted protein gene NIASHv2048J10 (GenBank Accession Number AK366933.1) of barley and 95% homologous with <em xmlns="" class="EmphasisTypeItalic">F3</em>â²<em xmlns="" class="EmphasisTypeItalic">M</em> of Arabidopsis (GenBank Accession Number AK366933.1), respectively. <em xmlns="" class="EmphasisTypeItalic">F3</em>â²<em xmlns="" class="EmphasisTypeItalic">M</em> belongs to the cytochrome P450 family and is related to flavonoid biosynthesis [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. The nucleotide sequence of MLOC_71630 from hulless barley was 99% homologous with a predicted protein gene NIASHv1124H04 of barley (GenBank Accession Number AK360742.1) and 92% homologous with 2-hydroxyisoflavanone dehydratase gene of Aegilops (GenBank Accession Number XM_020315035.1). The co-action of a <em xmlns="" class="EmphasisTypeItalic">HID</em> and 2-hydroxyisoflavanone synthase (<em xmlns="" class="EmphasisTypeItalic">IFS</em>) produces flavone from flavanone, which is related to isoflavone biosynthesis [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. The nucleotide sequence of MLOC_62096 from hulless barley was 99% homologous with <em xmlns="" class="EmphasisTypeItalic">UF3GT</em> of barley (GenBank Accession Number AK358154.1). UF3GT contributes to the last few steps in anthocyanin biosynthesis by converting cyanidin 3-O-xylosyl (1-â&gt;â2) glucoside. It can use 3-O-glucosylated anthocyanidins and uridine diphosphate (UDP)-xylose as substrates [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. The nucleotide sequence of MLOC_38343 from hulless barley was 99% homologous with <em xmlns="" class="EmphasisTypeItalic">UFGT</em> of barley (GenBank Accession Number X15694.1). <em xmlns="" class="EmphasisTypeItalic">UFGT</em> is involved in the anthocyanin biosynthesis pathway, which forms part of pigment biosynthesis [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. The nucleotide sequence of MLOC_32012 from hulless barley was 92% homologous with <em xmlns="" class="EmphasisTypeItalic">5MAT</em> of Aegilops (GenBank Accession Number XM_020296576.1). <em xmlns="" class="EmphasisTypeItalic">5MAT</em> is involved in later reactions in anthocyanin modification [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. The nucleotide sequence of MLOC_6171 from hulless barley was 99% homologous with <em xmlns="" class="EmphasisTypeItalic">Ant1</em> of barley (GenBank Accession Number KP265979.1). <em xmlns="" class="EmphasisTypeItalic">Ant1</em> can act as a trans-acting factor (MYB related family protein) that controls the expression of genes involved in anthocyanin biosynthesis and regulates the expression of at least three structural genes: dihydroflavonol reductase, chalcone synthase, and flavonol O<sub>3</sub> glucosyltransferase [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. Regulatory factors such as MYB, bHLH and WD40 proteins [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>] primarily control anthocyanin biosynthesis, as well as a series of structural genes (enzymes). In addition to <em xmlns="" class="EmphasisTypeItalic">Ant1</em>, we also discovered some other transcription factors including bHLH69, bHLH82, bHLH96, MYB39, WDR, TRAB1 and IND, etc. The relationship between transcription factors and anthocyanin biosynthesis still requires further validation. Also, the function of the above-mentioned annotation genes need further study.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec18" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">A high-density genetic linkage map was constructed using the GBS method. The linkage map contained seven linkage groups with a low inter-marker distance. This high-density linkage map can serve as a foundation for obtaining additional genetic knowledge of hulless barley. Five loci for <em xmlns="" class="EmphasisTypeItalic">Psc</em> were identified and will be useful in marker-assisted selection studies for this important agronomic trait. Using linkage analysis and gene annotation, five structural candidate genes and one regulatory factor related to flavonoid and anthocyanin biosynthesis were identified. These genomic resources may also play an important role in genetic breeding studies and future whole-genome sequencing projects in hulless barley.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">5MAT</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par3" class="Para">Malonyl-coenzyme: -O-glucoside-6â³â-O- malonyltransferase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">ANS</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Anthyocyanidin synthase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>BWA: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Burrows-wheeler aligner</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>CHI: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Chalcone isomerase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>CHS: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Chalcone synthase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>CP: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Cross pollination</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">DFR</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Dihydroflavonol 4-reductase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>DH: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Doubled haploid lines</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                              <em class="EmphasisTypeItalic">F3</em>â²<em class="EmphasisTypeItalic">H</em>
                           : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Flavonoid 3â²-hydroxylase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                              <em class="EmphasisTypeItalic">F3</em>â²<em class="EmphasisTypeItalic">M</em>
                           : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Flavonoid 3â²-monooxygenase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">F3H</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Flavanone 3-hydroxylase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>GBS: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Genotyping by sequencing</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">IFS</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">2-hydroxyisoflavanone synthase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>LOD: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Logarithm of odds</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">PAL</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Phenylalanine ammonia-lyase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>PE: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Paired-end</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">Psc</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Purple seed coat color</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>QC: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Quality control</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SNP: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Single nucleotide polymorphism</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">UF5GT</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Flavonoid-5-O-glucosyltransferase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">UFGT</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Anthocyanidin/ flavonoid 3-O-glucosyltransferase</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">UFGT</em>
                  : </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">UDP-sugar: flavonoid-3-O-glucosyltransferase</p>
                  </dd>
</dl>
</div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgements</h3>
<p xmlns="" class="SimplePara">We would like to thank Jianghua Huang and Ruiju Lu for DH of the Shanghai Academy of Agricultural Science, Novogene (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://en.novogene.com" target="_blank" class="ext-link"><span class="RefSource">https://en.novogene.com</span></a></span>) for the practical conduction of the GBS analysis and Accdon (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.accdon.com" target="_blank" class="ext-link"><span class="RefSource">www.accdon.com</span></a></span>) for editing this manuscript. We are grateful to the reviewers for their valuable comments.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par48" class="Para">This research was funded by the Natural Science Foundation of China (31660388), China Agriculture Research System (CARS-05), the Project of Qinghai Science and Technology Department (2016-ZJ-Y01and 2016-NK-131), and the Open Project of State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University (2017-ZZ-14).</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par49" class="Para">The data supporting the findings of this study are available within the article.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1">
<h3 xmlns="" class="Heading">Authorsâ contributions</h3>
<p xmlns="" class="SimplePara">XHY designed the research protocol, analyzed the data, carried out the experiment, and drafted the manuscript. YHY, JXY, and YXB assisted in the data analysis, supervised the protocol and prepared the manuscript. KLW and DZC finally approved the version to be published. All authors read and approved the final manuscript.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par50" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par51" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par52" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisherâs note</h3>
                <p id="Par53" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in institutional affiliations and published maps.</p>
              </div>
            </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside class="Appendix" id="App1"><h3 class="Heading">Additional files</h3>
<div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par54" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM1_ESM.jpg"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">The phenotype pictures of female, male and four types of seed coat color. (JPG 267 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM2_ESM.docx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">All candidate genes sequences. (DOCX 18 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM3_ESM.xlsx"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">Gene annotations of 7H within the region encompassing 71.00â73.10 cM. (XLSX 15 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM4_ESM.xlsx"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara">Gene annotations of 7H within the region encompassing 77.30â78.50 cM. (XLSX 15 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM5_ESM.xlsx"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara">Gene annotations of 7H within the region encompassing of 84.50â86.00 cM. (XLSX 30 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM6_ESM.xlsx"><span class="caption-number">Additional file 6:</span></a>
                          <span xmlns="" class="SimplePara">Gene annotations of 7H within the region encompassing 90.50â90.70 cM. (XLSX 25 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0072-6/MediaObjects/41065_2018_72_MOESM7_ESM.docx"><span class="caption-number">Additional file 7:</span></a>
                          <span xmlns="" class="SimplePara">The marker information between bin1533 to bin1531of 7H. (DOCX 16 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, 810016, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Academy of Agricultural and Forestry Sciences, Qinghai University, Xining, 810016, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Qinghai Key Laboratory of Hulless Barley Genetics and Breeding, Xining, 810016, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4">
<span class="AffiliationNumber">(4)</span><div class="AffiliationText">Qinghai Subcenter of National Hulless Barley Improvement, Xining, 810016, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liang JJ, Deng GD, Long H, Pan ZF, Wang CP, Cai P, Xu DL, Nimaand ZX, Yu MQ. Virus-induced silencing of genes encoding LEA protein in Tibetan hulless barley (Hordeumvulgare ssp. vulgare) and their relationship to drought tolerance. Mol Breeding. 2012;30(1):441â51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-011-9633-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Virus-induced%20silencing%20of%20genes%20encoding%20LEA%20protein%20in%20Tibetan%20hulless%20barley%20%28Hordeumvulgare%20ssp.%20vulgare%29%20and%20their%20relationship%20to%20drought%20tolerance&amp;author=JJ.%20Liang&amp;author=GD.%20Deng&amp;author=H.%20Long&amp;author=ZF.%20Pan&amp;author=CP.%20Wang&amp;author=P.%20Cai&amp;author=DL.%20Xu&amp;author=ZX.%20Nimaand&amp;author=MQ.%20Yu&amp;journal=Mol%20Breeding&amp;volume=30&amp;issue=1&amp;pages=441-451&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan XY, Li JN, Fu FY, Jin MY, Chen L, Liu LZ. Co-location of seed oil content, seed hull content and seed coat color QTL in three different environments in Brassicanapus L. Euphytica. 2009;170(3):355â64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-009-0006-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Co-location%20of%20seed%20oil%20content%2C%20seed%20hull%20content%20and%20seed%20coat%20color%20QTL%20in%20three%20different%20environments%20in%20Brassicanapus%20L&amp;author=XY.%20Yan&amp;author=JN.%20Li&amp;author=FY.%20Fu&amp;author=MY.%20Jin&amp;author=L.%20Chen&amp;author=LZ.%20Liu&amp;journal=Euphytica&amp;volume=170&amp;issue=3&amp;pages=355-364&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shoeva OY, Mock HP, Kukoeva Tatjana V, Borner A, Khlestkina EK. Regulation of the flavonoid biosynthesis pathway genesin purple and black grains of Hordeumvulgare. PLoS One. 2016;11(10):e0163782.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0163782"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Regulation%20of%20the%20flavonoid%20biosynthesis%20pathway%20genesin%20purple%20and%20black%20grains%20of%20Hordeumvulgare&amp;author=OY.%20Shoeva&amp;author=HP.%20Mock&amp;author=V.%20Kukoeva%20Tatjana&amp;author=A.%20Borner&amp;author=EK.%20Khlestkina&amp;journal=PLoS%20One&amp;volume=11&amp;issue=10&amp;pages=e0163782&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Abdel-Aal ES, Young JC, Rabalski I. Anthocyanin composition in black, blue, pink, purple and red cereal grains. J Agric Food Chem. 2006;54(13):4696â704.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/jf0606609"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Anthocyanin%20composition%20in%20black%2C%20blue%2C%20pink%2C%20purple%20and%20red%20cereal%20grains&amp;author=ES.%20Abdel-Aal&amp;author=JC.%20Young&amp;author=I.%20Rabalski&amp;journal=J%20Agric%20Food%20Chem&amp;volume=54&amp;issue=13&amp;pages=4696-4704&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bellido GG, Trust B. Anthocyanin composition and oxygen radical scavenging capacity (ORAC) of milled and pearled purple, black, and common barley. J Agric Food Chem. 2009;57(3):1022â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/jf802846x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Anthocyanin%20composition%20and%20oxygen%20radical%20scavenging%20capacity%20%28ORAC%29%20of%20milled%20and%20pearled%20purple%2C%20black%2C%20and%20common%20barley&amp;author=GG.%20Bellido&amp;author=B.%20Trust&amp;journal=J%20Agric%20Food%20Chem&amp;volume=57&amp;issue=3&amp;pages=1022-1028&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Madhujith T, Izydorczyk M, Shahidi F. Antioxidant properties of pearled barley fractions. J Agric Food Chem. 2006;54(9):3283â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/jf0527504"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Antioxidant%20properties%20of%20pearled%20barley%20fractions&amp;author=T.%20Madhujith&amp;author=M.%20Izydorczyk&amp;author=F.%20Shahidi&amp;journal=J%20Agric%20Food%20Chem&amp;volume=54&amp;issue=9&amp;pages=3283-3289&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kim MJ, Jong NH, Aekim J, P ark JC, Kim MY, Kim JG, Lee SJ, Chun SC, Chung AM. Relationship between phenolic compounds, anthocyanins content and antioxidant activity in colored barley germplasm. J Agric Food Chem. 2007;55(12):4802â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/jf0701943"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Relationship%20between%20phenolic%20compounds%2C%20anthocyanins%20content%20and%20antioxidant%20activity%20in%20colored%20barley%20germplasm&amp;author=MJ.%20Kim&amp;author=NH.%20Jong&amp;author=J.%20Aekim&amp;author=JC.%20P%20ark&amp;author=MY.%20Kim&amp;author=JG.%20Kim&amp;author=SJ.%20Lee&amp;author=SC.%20Chun&amp;author=AM.%20Chung&amp;journal=J%20Agric%20Food%20Chem&amp;volume=55&amp;issue=12&amp;pages=4802-4809&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Aastrup S, Outtrup H, Erdal K. Location of the proanthocyanidins in the barley grain. Carlsb Res Commun. 1984;49:105â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02913969"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Location%20of%20the%20proanthocyanidins%20in%20the%20barley%20grain&amp;author=S.%20Aastrup&amp;author=H.%20Outtrup&amp;author=K.%20Erdal&amp;journal=Carlsb%20Res%20Commun&amp;volume=49&amp;pages=105-109&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Harlan HV. Some distinctions in our cultivated barleys with reference to their use in plant breeding. U S Dept Agr Bull. 1914;137:38.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Harlan%20HV.%20Some%20distinctions%20in%20our%20cultivated%20barleys%20with%20reference%20to%20their%20use%20in%20plant%20breeding.%20U%20S%20Dept%20Agr%20Bull.%201914%3B137%3A38."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Himi E, Yamashita Y, Haruyama N, Yanagisawa T, Maekawa M, Taketa S. Ant28 gene for proanthocyanidin synthesis encoding the R2R3 MYB domain protein (Hvmyb10) highly affects grain dormancy in barley. Euphytica. 2012;188(1):141â51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-011-0552-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ant28%20gene%20for%20proanthocyanidin%20synthesis%20encoding%20the%20R2R3%20MYB%20domain%20protein%20%28Hvmyb10%29%20highly%20affects%20grain%20dormancy%20in%20barley&amp;author=E.%20Himi&amp;author=Y.%20Yamashita&amp;author=N.%20Haruyama&amp;author=T.%20Yanagisawa&amp;author=M.%20Maekawa&amp;author=S.%20Taketa&amp;journal=Euphytica&amp;volume=188&amp;issue=1&amp;pages=141-151&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Badbro JS. Genetic control of flavonoid biosynthesis in barley. Hereditas. 1993;119(2):187â204.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20flavonoid%20biosynthesis%20in%20barley&amp;author=JS.%20Badbro&amp;journal=Hereditas&amp;volume=119&amp;issue=2&amp;pages=187-204&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Himi E, Taketa S. Barley Ant17, encoding flavanone 3-hydroxylase (F3H), is a promising target locus for attaining anthocyanin/ proanthocyanidin-free plants without pleiotropic reduction of grain dormancy. Genome. 2015;58(1):43â53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/gen-2014-0189"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Barley%20Ant17%2C%20encoding%20flavanone%203-hydroxylase%20%28F3H%29%2C%20is%20a%20promising%20target%20locus%20for%20attaining%20anthocyanin%2F%20proanthocyanidin-free%20plants%20without%20pleiotropic%20reduction%20of%20grain%20dormancy&amp;author=E.%20Himi&amp;author=S.%20Taketa&amp;journal=Genome&amp;volume=58&amp;issue=1&amp;pages=43-53&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cockram J, White J, Zuluaga DL, Smith D, Comadran J, Macaulay M, Luo ZW, Kearsey MJ, Werner P, Marshall DF, et al. Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome. Proc Natl Acad Sci U S A. 2010;107(50):21611â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1010179107"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20to%20candidate%20polymorphism%20resolution%20in%20the%20unsequenced%20barley%20genome&amp;author=J.%20Cockram&amp;author=J.%20White&amp;author=DL.%20Zuluaga&amp;author=D.%20Smith&amp;author=J.%20Comadran&amp;author=M.%20Macaulay&amp;author=ZW.%20Luo&amp;author=MJ.%20Kearsey&amp;author=P.%20Werner&amp;author=DF.%20Marshall&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=107&amp;issue=50&amp;pages=21611-21616&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peukert M, Weise S, Roder MS, Matthies IE. Development of SNP markers for genes of the phenylpropanoid pathway and their association to kernel and malting traits in barley. BMC Genet. 2013;14(1):97â112.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2156-14-97"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20SNP%20markers%20for%20genes%20of%20the%20phenylpropanoid%20pathway%20and%20their%20association%20to%20kernel%20and%20malting%20traits%20in%20barley&amp;author=M.%20Peukert&amp;author=S.%20Weise&amp;author=MS.%20Roder&amp;author=IE.%20Matthies&amp;journal=BMC%20Genet&amp;volume=14&amp;issue=1&amp;pages=97-112&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kristiansen K, Rohde W. Structure of the Hordeumvulgare gene encoding dihydroflavonol-4-reductase and molecular analysis of ant18 mutants blocked in flavonoid synthesis. Mol Gen Genet. 1991;230(1â2):49â59.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00290650"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20Hordeumvulgare%20gene%20encoding%20dihydroflavonol-4-reductase%20and%20molecular%20analysis%20of%20ant18%20mutants%20blocked%20in%20flavonoid%20synthesis&amp;author=K.%20Kristiansen&amp;author=W.%20Rohde&amp;journal=Mol%20Gen%20Genet&amp;volume=230&amp;issue=1%E2%80%932&amp;pages=49-59&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Baxter SW, Davey JW, Johnston JS. Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model rganism. PLoS One. 2011;6(4):e19315.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0019315"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Linkage%20mapping%20and%20comparative%20genomics%20using%20next-generation%20RAD%20sequencing%20of%20a%20non-model%20rganism&amp;author=SW.%20Baxter&amp;author=JW.%20Davey&amp;author=JS.%20Johnston&amp;journal=PLoS%20One&amp;volume=6&amp;issue=4&amp;pages=e19315&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zeng XQ, Long H, Wang Z, Zhao SC, Tang YW, Huang ZY, Wang YL, Xu QJ, Mao LK, Deng GB, et al. The draft genome of Tibetan hulless barley revealsadaptive patterns to the high stressful Tibetan plateau. Proc Natl Acad Sci U S A. 2015;112(4):1095â100.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1423628112"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20Tibetan%20hulless%20barley%20revealsadaptive%20patterns%20to%20the%20high%20stressful%20Tibetan%20plateau&amp;author=XQ.%20Zeng&amp;author=H.%20Long&amp;author=Z.%20Wang&amp;author=SC.%20Zhao&amp;author=YW.%20Tang&amp;author=ZY.%20Huang&amp;author=YL.%20Wang&amp;author=QJ.%20Xu&amp;author=LK.%20Mao&amp;author=GB.%20Deng&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=112&amp;issue=4&amp;pages=1095-1100&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guajardo V, Solis S, Sagredo B, Gainza F, MuÃ±oz C, Gasic K, Hinrichsen P. Construction of high density sweet cherry (PrunusaviumL.) linkage maps using microsatellite markers and SNPs detected by genotyping-by-sequencing(GBS). PLoS One. 2015;10(5):e0127750.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0127750"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20high%20density%20sweet%20cherry%20%28PrunusaviumL.%29%20linkage%20maps%20using%20microsatellite%20markers%20and%20SNPs%20detected%20by%20genotyping-by-sequencing%28GBS%29&amp;author=V.%20Guajardo&amp;author=S.%20Solis&amp;author=B.%20Sagredo&amp;author=F.%20Gainza&amp;author=C.%20Mu%C3%B1oz&amp;author=K.%20Gasic&amp;author=P.%20Hinrichsen&amp;journal=PLoS%20One&amp;volume=10&amp;issue=5&amp;pages=e0127750&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang B, Liu W, Xie D, Peng Q, He X, Lin Y, Liang ZJ. High-density genetic map construction and gene mapping of pericarp color in wax gourd using specific-locus amplified fragment (SLAF) sequencing. BMC Genomics. 2015;16(1):1035.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-015-2220-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-density%20genetic%20map%20construction%20and%20gene%20mapping%20of%20pericarp%20color%20in%20wax%20gourd%20using%20specific-locus%20amplified%20fragment%20%28SLAF%29%20sequencing&amp;author=B.%20Jiang&amp;author=W.%20Liu&amp;author=D.%20Xie&amp;author=Q.%20Peng&amp;author=X.%20He&amp;author=Y.%20Lin&amp;author=ZJ.%20Liang&amp;journal=BMC%20Genomics&amp;volume=16&amp;issue=1&amp;pages=1035&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qi HK, Wang N, Qiao WQ, Xu QH, Zhou H, Shi JB, Yan GT, Huang Q. Construction of a high-density genetic map using genotyping by sequencing (GBS) for quantitative trait loci (QTL) analysis of three plant morphological traits in upland cotton (GossypiumhirsutumL.). Euphytica. 2017;213(4):83â99.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-017-1867-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20using%20genotyping%20by%20sequencing%20%28GBS%29%20for%20quantitative%20trait%20loci%20%28QTL%29%20analysis%20of%20three%20plant%20morphological%20traits%20in%20upland%20cotton%20%28GossypiumhirsutumL.%29&amp;author=HK.%20Qi&amp;author=N.%20Wang&amp;author=WQ.%20Qiao&amp;author=QH.%20Xu&amp;author=H.%20Zhou&amp;author=JB.%20Shi&amp;author=GT.%20Yan&amp;author=Q.%20Huang&amp;journal=Euphytica&amp;volume=213&amp;issue=4&amp;pages=83-99&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang JP, Li L, Zhang GF. A high-density SNP genetic linkage map and QTL analysis of growth-related traits in a hybrid family of oysters (CrassostreagigasÃ Crassostreaangulata) using genotyping-by-sequencing. G3. 2016;6(5):1417â26.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/g3.116.026971"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20high-density%20SNP%20genetic%20linkage%20map%20and%20QTL%20analysis%20of%20growth-related%20traits%20in%20a%20hybrid%20family%20of%20oysters%20%28Crassostreagigas%C3%97%20Crassostreaangulata%29%20using%20genotyping-by-sequencing&amp;author=JP.%20Wang&amp;author=L.%20Li&amp;author=GF.%20Zhang&amp;journal=G3&amp;volume=6&amp;issue=5&amp;pages=1417-1426&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhou ZQ, Zhang CS, Zhou Y, Hao ZF, Wang ZH, Zeng X, et al. Genetic dissection of maize plantarchitecture with an ultra-high density binmap based on recombinant inbred lines. BMC Genomics. 2016;7:178â92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-016-2555-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20maize%20plantarchitecture%20with%20an%20ultra-high%20density%20binmap%20based%20on%20recombinant%20inbred%20lines&amp;author=ZQ.%20Zhou&amp;author=CS.%20Zhang&amp;author=Y.%20Zhou&amp;author=ZF.%20Hao&amp;author=ZH.%20Wang&amp;author=X.%20Zeng&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=178-192&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nie HT, Yan XW, Huo ZM, Jiang LW, Chen P, Liu H, Ding JF, Yang F, et al. Construction of a high-density genetic map and quantitative trait locus mapping in the manila clam Ruditapesphilippinarum. Sci Rep. 2017;7(1):229â37.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41598-017-00246-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20and%20quantitative%20trait%20locus%20mapping%20in%20the%20manila%20clam%20Ruditapesphilippinarum&amp;author=HT.%20Nie&amp;author=XW.%20Yan&amp;author=ZM.%20Huo&amp;author=LW.%20Jiang&amp;author=P.%20Chen&amp;author=H.%20Liu&amp;author=JF.%20Ding&amp;author=F.%20Yang&amp;journal=Sci%20Rep&amp;volume=7&amp;issue=1&amp;pages=229-237&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Durbin R. Fast and accurate short read alignment with burrows-wheeler transform. Bioinformatics. 2009;25(14):1754â60.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp324"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20burrows-wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;journal=Bioinformatics&amp;volume=25&amp;issue=14&amp;pages=1754-1760&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R. 1000 genome project data processing subgroup. The sequence alignment/map format and SAMtools. Bioinformatics. 2009;25(16):2078â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp352"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=1000%20genome%20project%20data%20processing%20subgroup.%20The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;author=T.%20Fennell&amp;author=J.%20Ruan&amp;author=N.%20Homer&amp;author=G.%20Marth&amp;author=G.%20Abecasis&amp;author=R.%20Durbin&amp;journal=Bioinformatics&amp;volume=25&amp;issue=16&amp;pages=2078-2079&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van Ooijen JW. JoinMapÂ® 4.0: software for the calculation of genetic linkage maps in experimental population. Wageningen. In: The Netherlands; 2006.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=JoinMap%C2%AE%204.0%3A%20software%20for%20the%20calculation%20of%20genetic%20linkage%20maps%20in%20experimental%20population.%20Wageningen&amp;author=JW.%20Van%20Ooijen&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kosambi DD. The estimation of map distance from recombination values. Ann Eugenics. 1943;12(1):172â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1469-1809.1943.tb02321.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=DD.%20Kosambi&amp;journal=Ann%20Eugenics&amp;volume=12&amp;issue=1&amp;pages=172-175&amp;publication_year=1943"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yuki HC, Hiroki Y, Ogura R, Naomi S, Hitoshi S, Kiyoaki K. Genetic analysis and molecular mapping of genes controlling seed coat colour in adzuki bean (Vigna angularis). Euphytica. 2015;206(3):609â17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-015-1461-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20and%20molecular%20mapping%20of%20genes%20controlling%20seed%20coat%20colour%20in%20adzuki%20bean%20%28Vigna%20angularis%29&amp;author=HC.%20Yuki&amp;author=Y.%20Hiroki&amp;author=R.%20Ogura&amp;author=S.%20Naomi&amp;author=S.%20Hitoshi&amp;author=K.%20Kiyoaki&amp;journal=Euphytica&amp;volume=206&amp;issue=3&amp;pages=609-617&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Graner A, Bauer E, Kellermann A, Proeseler G, Wenzel G, Ordon F. RFLP analysis of resistance to barley yellow mosaic virus complex. Agronomie. 1995;15:475â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1051/agro%3A19950716"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RFLP%20analysis%20of%20resistance%20to%20barley%20yellow%20mosaic%20virus%20complex&amp;author=A.%20Graner&amp;author=E.%20Bauer&amp;author=A.%20Kellermann&amp;author=G.%20Proeseler&amp;author=G.%20Wenzel&amp;author=F.%20Ordon&amp;journal=Agronomie&amp;volume=15&amp;pages=475-479&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roslinsky V, Eckstein PE, Raboy V, Rossnagel BG, Scoles GJ. Molecular marker development and linkage analysis in three low phytic acid barley (Hordeum vulgare) mutant lines. Mol Breeding. 2007;20(4):323â30.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-007-9094-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20marker%20development%20and%20linkage%20analysis%20in%20three%20low%20phytic%20acid%20barley%20%28Hordeum%20vulgare%29%20mutant%20lines&amp;author=V.%20Roslinsky&amp;author=PE.%20Eckstein&amp;author=V.%20Raboy&amp;author=BG.%20Rossnagel&amp;author=GJ.%20Scoles&amp;journal=Mol%20Breeding.&amp;volume=20&amp;issue=4&amp;pages=323-330&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cynara CTR, Jasper PV, Anton V, Axel H, Martin M, Rients EN. Mapping resistance to powdery mildew in barley reveals a large-effect nonhost resistance QTL. Theor Appl Genet. 2018;131(5):1031â45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-018-3055-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mapping%20resistance%20to%20powdery%20mildew%20in%20barley%20reveals%20a%20large-effect%20nonhost%20resistance%20QTL&amp;author=CTR.%20Cynara&amp;author=PV.%20Jasper&amp;author=V.%20Anton&amp;author=H.%20Axel&amp;author=M.%20Martin&amp;author=EN.%20Rients&amp;journal=Theor%20Appl%20Genet&amp;volume=131&amp;issue=5&amp;pages=1031-1045&amp;publication_year=2018"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang XC, Shabala S, Koutoulis A, Shabala L, Zhou MX. Meta-analysis of major QTL for abiotic stress tolerance in barley and implications for barley breeding. Planta. 2017;245(2):283â95.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00425-016-2605-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Meta-analysis%20of%20major%20QTL%20for%20abiotic%20stress%20tolerance%20in%20barley%20and%20implications%20for%20barley%20breeding&amp;author=XC.%20Zhang&amp;author=S.%20Shabala&amp;author=A.%20Koutoulis&amp;author=L.%20Shabala&amp;author=MX.%20Zhou&amp;journal=Planta&amp;volume=245&amp;issue=2&amp;pages=283-295&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fan CF, Zhai HJ, Wang HF, Yue YF, Zhang MH, Li JH, Wen SZ, Guo GG, Zeng YW, Ni ZF, You MS. Identification of QTLs controlling grain protein concentration using a high-density SNP and SSR linkage map in barley (Hordeum vulgare L.). BMC Plant Biol. 2017;17:122â45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12870-017-1067-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20QTLs%20controlling%20grain%20protein%20concentration%20using%20a%20high-density%20SNP%20and%20SSR%20linkage%20map%20in%20barley%20%28Hordeum%20vulgare%20L.%29&amp;author=CF.%20Fan&amp;author=HJ.%20Zhai&amp;author=HF.%20Wang&amp;author=YF.%20Yue&amp;author=MH.%20Zhang&amp;author=JH.%20Li&amp;author=SZ.%20Wen&amp;author=GG.%20Guo&amp;author=YW.%20Zeng&amp;author=ZF.%20Ni&amp;author=MS.%20You&amp;journal=BMC%20Plant%20Biol&amp;volume=17&amp;pages=122-145&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Eshghi R, Salayeva S, Ebrahimpour F, Rahimi M, Baraty M, Ojaghi J. Advanced-backcross QTL analysis in hulless barley: I. detection of exotic alleles for yield and yield components introgressed from <em class="EmphasisTypeItalic">Hordeum vulgare</em> ssp. spontaneum. Intl Agri. Crop Sci. 2013;5(2):95â100.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Advanced-backcross%20QTL%20analysis%20in%20hulless%20barley%3A%20I.%20detection%20of%20exotic%20alleles%20for%20yield%20and%20yield%20components%20introgressed%20from%20Hordeum%20vulgare%20ssp.%20spontaneum.%20Intl%20Agri&amp;author=R.%20Eshghi&amp;author=S.%20Salayeva&amp;author=F.%20Ebrahimpour&amp;author=M.%20Rahimi&amp;author=M.%20Baraty&amp;author=J.%20Ojaghi&amp;journal=Crop%20Sci&amp;volume=5&amp;issue=2&amp;pages=95-100&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu H, Bayer M, Druka A, Russell JR, Hackett CA, Poland J, Ramsay L, Hedley PE, Waugh R. An evaluation of genotyping by sequencing (GBS) to map the Breviaristatum-e (ari-e) locus in cultivated barley. BMC Genomics. 2014;15(1):104â15.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-15-104"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20evaluation%20of%20genotyping%20by%20sequencing%20%28GBS%29%20to%20map%20the%20Breviaristatum-e%20%28ari-e%29%20locus%20in%20cultivated%20barley&amp;author=H.%20Liu&amp;author=M.%20Bayer&amp;author=A.%20Druka&amp;author=JR.%20Russell&amp;author=CA.%20Hackett&amp;author=J.%20Poland&amp;author=L.%20Ramsay&amp;author=PE.%20Hedley&amp;author=R.%20Waugh&amp;journal=BMC%20Genomics&amp;volume=15&amp;issue=1&amp;pages=104-115&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Carlson BM, Onusko SW, Gross JB. A high-density linkage map for Astyanaxmexicanus using genotyping-by-sequencing technology. G3. 2015;5(2):241â51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/g3.114.015438"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20high-density%20linkage%20map%20for%20Astyanaxmexicanus%20using%20genotyping-by-sequencing%20technology&amp;author=BM.%20Carlson&amp;author=SW.%20Onusko&amp;author=JB.%20Gross&amp;journal=G3&amp;volume=5&amp;issue=2&amp;pages=241-251&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Woodward RW, Thieret JW. A genetic study of complementary genes for purple lemma, Palea and pericarp in barley (Hordeum vulgare L.). Agron J. 1953;45:182â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2134/agronj1953.00021962004500050002x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20genetic%20study%20of%20complementary%20genes%20for%20purple%20lemma%2C%20Palea%20and%20pericarp%20in%20barley%20%28Hordeum%20vulgare%20L.%29&amp;author=RW.%20Woodward&amp;author=JW.%20Thieret&amp;journal=Agron%20J&amp;volume=45&amp;pages=182-185&amp;publication_year=1953"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Franckowiak JD, Lundqvist U. New and revised descriptions of barley genes. Barley Genet Newsl. 1997;26:209.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=New%20and%20revised%20descriptions%20of%20barley%20genes&amp;author=JD.%20Franckowiak&amp;author=U.%20Lundqvist&amp;journal=Barley%20Genet%20Newsl&amp;volume=26&amp;pages=209&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang MF, Jiang LM, Zhang DM, Jia GX. De novo transcriptome characterization of Lilium âSorbonneâ and key enzymes related to the flavonoid biosynthesis. Mol Gen Genomics. 2015;290(1):399â412.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00438-014-0919-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20characterization%20of%20Lilium%20%E2%80%98Sorbonne%E2%80%99%20and%20key%20enzymes%20related%20to%20the%20flavonoid%20biosynthesis&amp;author=MF.%20Zhang&amp;author=LM.%20Jiang&amp;author=DM.%20Zhang&amp;author=GX.%20Jia&amp;journal=Mol%20Gen%20Genomics&amp;volume=290&amp;issue=1&amp;pages=399-412&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Larson RL, Bussard JB. Microsomal flavonoid 3â²- monooxygenase from maize seedlings. Plant Physiol. 1986;80(2):483â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.80.2.483"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsomal%20flavonoid%203%E2%80%B2-%20monooxygenase%20from%20maize%20seedlings&amp;author=RL.%20Larson&amp;author=JB.%20Bussard&amp;journal=Plant%20Physiol&amp;volume=80&amp;issue=2&amp;pages=483-486&amp;publication_year=1986"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Halbwirth H. The creation and physiological relevance of divergent hydroxylation patterns in the flavonoid pathway. Int J Mol Sci. 2010;11(2):595â621.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/ijms11020595"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20creation%20and%20physiological%20relevance%20of%20divergent%20hydroxylation%20patterns%20in%20the%20flavonoid%20pathway&amp;author=H.%20Halbwirth&amp;journal=Int%20J%20Mol%20Sci&amp;volume=11&amp;issue=2&amp;pages=595-621&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li P, Li YJ, Zhang FJ, Zhang GZ, Jiang XY, Yu HM, Hou BK. The Arabidopsis UDP-glycosyl transferases UGT79B2 and UGT79B3, contribute to cold, salt and drought stress tolerance via modulating anthocyanin accumulation. Plant J. 2017;89(1):85â103.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/tpj.13324"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20UDP-glycosyl%20transferases%20UGT79B2%20and%20UGT79B3%2C%20contribute%20to%20cold%2C%20salt%20and%20drought%20stress%20tolerance%20via%20modulating%20anthocyanin%20accumulation&amp;author=P.%20Li&amp;author=YJ.%20Li&amp;author=FJ.%20Zhang&amp;author=GZ.%20Zhang&amp;author=XY.%20Jiang&amp;author=HM.%20Yu&amp;author=BK.%20Hou&amp;journal=Plant%20J&amp;volume=89&amp;issue=1&amp;pages=85-103&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yonekura-Sakakibara K, Tanaka Y, Fukuchi-Mizutani M, Fujiwara H, Fukui Y, Ashikari T, Murakami Y, Yamaguchi M, Kusumi T. Molecular and biochemical characterization of a novel hydroxycinnamoyl-CoA: anthocyanin 3-O-glucoside-6-O-acyltransferase from Perillafrutescens. Plant Cell Physiol. 2000;41(4):495â502.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/pcp/41.4.495"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20and%20biochemical%20characterization%20of%20a%20novel%20hydroxycinnamoyl-CoA%3A%20anthocyanin%203-O-glucoside-6-O-acyltransferase%20from%20Perillafrutescens&amp;author=K.%20Yonekura-Sakakibara&amp;author=Y.%20Tanaka&amp;author=M.%20Fukuchi-Mizutani&amp;author=H.%20Fujiwara&amp;author=Y.%20Fukui&amp;author=T.%20Ashikari&amp;author=Y.%20Murakami&amp;author=M.%20Yamaguchi&amp;author=T.%20Kusumi&amp;journal=Plant%20Cell%20Physiol&amp;volume=41&amp;issue=4&amp;pages=495-502&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang F, Chen XP, Hu WS, Zheng SQ. Identification of differentially expressed genes implicated in peel color (red and green) of Dimocarpusconfinis. Springer Plus. 2016;5(1):1088â101.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s40064-016-2743-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20differentially%20expressed%20genes%20implicated%20in%20peel%20color%20%28red%20and%20green%29%20of%20Dimocarpusconfinis&amp;author=F.%20Jiang&amp;author=XP.%20Chen&amp;author=WS.%20Hu&amp;author=SQ.%20Zheng&amp;journal=Springer%20Plus&amp;volume=5&amp;issue=1&amp;pages=1088-1101&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao L, Gao LP, Wang HX, Chen XT, Wang YS, Hua Y, Wei CL, Wan XC Xia T. The R2R3-MYB, bHLH, WD40, and related transcription factors in flavonoid biosynthesis. Funct Integr Genomics. 2013;13(1):75â98.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10142-012-0301-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20R2R3-MYB%2C%20bHLH%2C%20WD40%2C%20and%20related%20transcription%20factors%20in%20flavonoid%20biosynthesis&amp;author=L.%20Zhao&amp;author=LP.%20Gao&amp;author=HX.%20Wang&amp;author=XT.%20Chen&amp;author=YS.%20Wang&amp;author=Y.%20Hua&amp;author=CL.%20Wei&amp;author=T.%20Wan%20XC%20Xia&amp;journal=Funct%20Integr%20Genomics&amp;volume=13&amp;issue=1&amp;pages=75-98&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© The Author(s) 2018</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0072-6" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-018-0072-6">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0072-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0072-6?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0072-6 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0072-6 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0072-6 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0072-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0072-6">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0072-6"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0072-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0072-6">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0072-6"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0072-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0072-6">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0072-6"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0072-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0072-6">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0072-6"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0072-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0072-6">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0072-6"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-018-0072-6/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-018-0072-6">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0072-6">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0072-6" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-018-0072-6&amp;volume=155&amp;issue=1&amp;title=Construction+of+a+high-density+genetic+map%3A+genotyping+by+sequencing+%28GBS%29+to+map+purple+seed+coat+color+%28+Psc+%29+in+hulless+barley&amp;author_name=Xiaohua+Yao&amp;start_page=1&amp;end_page=11">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-018-0072-6" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
