<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Genetic structure of Mount Huang honey bee (Apis cerana) populations: evidence from microsatellite polymorphism | Hereditas | Full Text</title>
<meta name="citation_abstract" content="The Mount Huang eastern honey bees (Apis cerana) are an endemic population, which is well adapted to the local agricultural and ecological environment. In this study, the genetic structure of seven eastern honey bees (A. cerana) populations from Mount Huang in China were analyzed by SSR (simple sequence repeat) markers. The results revealed that 16 pairs of primers used amplified a total of 143 alleles. The number of alleles per locus ranged from 6 to 13, with a mean value of 8.94 alleles per locus. Observed and expected heterozygosities showed mean values of 0.446 and 0.831 respectively. UPGMA cluster analysis grouped seven eastern honey bees in three groups. The results obtained show a high genetic diversity in the honey bee populations studied in Mount Huang, and high differentiation among all the populations, suggesting that scarce exchange of honey bee species happened in Mount Huang. Our study demonstrated that the Mount Huang honey bee populations still have a natural genome worth being protected for conservation.">
<meta name="description" content="The Mount Huang eastern honey bees (Apis cerana) are an endemic population, which is well adapted to the local agricultural and ecological environment. In this study, the genetic structure of seven eastern honey bees (A. cerana) populations from Mount Huang in China were analyzed by SSR (simple sequence repeat) markers. The results revealed that 16 pairs of primers used amplified a total of 143 alleles. The number of alleles per locus ranged from 6 to 13, with a mean value of 8.94 alleles per locus. Observed and expected heterozygosities showed mean values of 0.446 and 0.831 respectively. UPGMA cluster analysis grouped seven eastern honey bees in three groups. The results obtained show a high genetic diversity in the honey bee populations studied in Mount Huang, and high differentiation among all the populations, suggesting that scarce exchange of honey bee species happened in Mount Huang. Our study demonstrated that the Mount Huang honey bee populations still have a natural genome worth being protected for conservation.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Genetic structure of Mount Huang honey bee ( Apis cerana ) populations: evidence from microsatellite polymorphism">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-07-06">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="The Mount Huang eastern honey bees (Apis cerana) are an endemic population, which is well adapted to the local agricultural and ecological environment. In this study, the genetic structure of seven eastern honey bees (A. cerana) populations from Mount Huang in China were analyzed by SSR (simple sequence repeat) markers. The results revealed that 16 pairs of primers used amplified a total of 143 alleles. The number of alleles per locus ranged from 6 to 13, with a mean value of 8.94 alleles per locus. Observed and expected heterozygosities showed mean values of 0.446 and 0.831 respectively. UPGMA cluster analysis grouped seven eastern honey bees in three groups. The results obtained show a high genetic diversity in the honey bee populations studied in Mount Huang, and high differentiation among all the populations, suggesting that scarce exchange of honey bee species happened in Mount Huang. Our study demonstrated that the Mount Huang honey bee populations still have a natural genome worth being protected for conservation.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-07-06">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="8">
<meta name="prism.copyright" content="2016 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0010-4">
<meta name="prism.doi" content="doi:10.1186/s41065-016-0010-4">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0010-4">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0010-4">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Genetic structure of Mount Huang honey bee ( Apis cerana ) populations: evidence from microsatellite polymorphism">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/07/06">
<meta name="citation_firstpage" content="8">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-016-0010-4">
<meta name="DOI" content="10.1186/s41065-016-0010-4">
<meta name="citation_doi" content="10.1186/s41065-016-0010-4">
<meta name="dc.creator" content="Fang Liu">
<meta name="dc.creator" content="Tengfei Shi">
<meta name="dc.creator" content="Sisi Huang">
<meta name="dc.creator" content="Linsheng Yu">
<meta name="dc.creator" content="Shoudong Bi">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Chin J Appl Ecol; citation_title=Relation between population dynamics and distribution of honey bees and ecological environment; citation_author=LS Yu, YD Zou, S Bi SD; citation_volume=17; citation_publication_date=2006; citation_pages=1465-1468; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Nature; citation_title=High resolution of human evolutionary trees with polymorphic microsatellites; citation_author=AM Bowcock, RA Linares, J Tomforhde, E Minch, JR Kidd, CLL Sforza; citation_volume=368; citation_publication_date=1994; citation_pages=455-457; citation_doi=10.1038/368455a0; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Genetic variation of microsatellite loci in a bottlenecked species: the northern hairy-nosed wombat Lasiorhinus krefftii
                           ; citation_author=AC Taylor, WB Sherwin, RK Wayne; citation_volume=3; citation_publication_date=1994; citation_pages=277-299; citation_doi=10.1111/j.1365-294X.1994.tb00068.x; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=J Apicult Res; citation_title=Genetic structure of honeybee, Apis mellifera L. (Hymenoptera:Apidae) populations of Turkey inferred from microsatellite analysis; citation_author=ÇM Bodur, KA Kence; citation_volume=46; citation_publication_date=2007; citation_pages=50-56; citation_doi=10.1080/00218839.2007.11101366; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=J Fujian Agric Forestry Univ (National Science Edition); citation_title=Genetic diversity of Apis cerana cerana in Fujian based on microsatellite markers; citation_author=XJ Zhu, BF Zhou, XD Wu, JX Xu, XC Xia; citation_volume=40; citation_issue=4; citation_publication_date=2011; citation_pages=407-411; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=Fujian J Agric Sci; citation_title=Genetic analysis of Apis cerana cerana in Wuyi mountain nature reserve based on microsatellite DNA; citation_author=XJ Zhu, XJ Xu, MQ Zhou, XD Wu, BF Zhou; citation_volume=26; citation_publication_date=2011; citation_pages=935-940; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=China Anim Husbandry Vet Med; citation_title=Study on microsatellite genetic polymorphism of wild and domestic Apis cerana cerana populations from Jiangsu province; citation_author=T Ji, J Chen, L Yin, WB Bao, GH Chen; citation_volume=34; citation_publication_date=2007; citation_pages=39-41; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=Apicult Chin; citation_title=Genetic Diversity of Microsatellite DNA for Apis cerana cerana in Zhejiang; citation_author=XL Su, L Cao, DX Zhao, Q Hua, FL Hu; citation_volume=64; citation_publication_date=2013; citation_pages=10-11; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Acta Entomol Sin; citation_title=Population genetic analysis of Apis cerana cerana from the Qinling-Daba mountain areas based on microsatellite DNA; citation_author=HP Guo, SJ Zhou, XJ Xu, JX Xu, YL Yu, KJ Yang, DY Chen, BF Zhou; citation_volume=59; citation_publication_date=2016; citation_pages=337-345; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Afr J Biotechnol; citation_title=Genetic diversity and population structure of Chinese honeybees (Apis cerana) under microsatellite markers; citation_author=T Ji, L Yin, G Chen; citation_volume=10; citation_publication_date=2011; citation_pages=1712-1720; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Apiculture of China; citation_title=Study on genetic diversity of Apis cerana cerana populations from Lushui by microsatellite markers; citation_author=D Lin, L Yin, T Jing, M Liu, HG Chen; citation_volume=60; citation_publication_date=2009; citation_pages=9-11; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Southwest China J Agric Sci; citation_title=Study on genetic diversity and genetic differentiation of six geographic populations of Apis cerana (Hymenoptera: Apidae) in Yunnan by microsatellite markers; citation_author=L Yin, T Ji, GH Chen; citation_volume=24; citation_publication_date=2011; citation_pages=772-779; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=China J Apic Sci; citation_title=Morphmetric analysis of Apis cerana populations in Huangshan; citation_author=LS Yu, F Liu, SS Huang, SD Bi, C Zong, TS Wang; citation_volume=57; citation_publication_date=2013; citation_pages=117-124; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=J Kansas EntomolSoc; citation_title=The biogeography of Apis cerana as revealed by mitochondrial DNA sequence data; citation_author=DR Smith, RH Hagen; citation_volume=69; citation_publication_date=1996; citation_pages=294-310; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=A microsatellite-based linkage Map of the honeybee, Apis mellifera L; citation_author=M Solignac, D Vautrin, E Baudry, F Mougel, A Loiseau, JM Cornuet; citation_volume=167; citation_issue=1; citation_publication_date=2004; citation_pages=253-262; citation_doi=10.1534/genetics.167.1.253; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=P Natl Acad Sci USA; citation_title=Analysis of gene diversity in subdivided populations; citation_author=M Nei; citation_volume=70; citation_publication_date=1973; citation_pages=3321-3323; citation_doi=10.1073/pnas.70.12.3321; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=GENEPOP’ 007: a complete reimplementation of the GENEPOP software for windows and linux; citation_author=F Rousset; citation_volume=8; citation_publication_date=2008; citation_pages=103-106; citation_doi=10.1111/j.1471-8286.2007.01931.x; citation_id=CR17">
<meta name="citation_reference" content="Goudet J. FSTAT, a program to estimate and test gene diversities and fixation indices (version 2.9.3). Available from 
                    http://www2.unil.ch/popgen/softwares/fstat.htm
                    
                  . 2001. Updated from Goudet (1995).">
<meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Analyzing tables of statistical tests; citation_author=WS Rice; citation_volume=43; citation_publication_date=1989; citation_pages=223-225; citation_doi=10.2307/2409177; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Accuracy of estimated phylogenetic trees from molecular data; citation_author=MF Nei, TY Tajima; citation_volume=19; citation_publication_date=1983; citation_pages=153-170; citation_doi=10.1007/BF02300753; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=J Fish Biol; citation_title=The evolving technology of DNA fingerprinting and its application to fisheries and aquaculture; citation_author=P O’Reilly, JM Wright; citation_volume=47; citation_publication_date=1995; citation_pages=29-55; citation_doi=10.1111/j.1095-8649.1995.tb06042.x; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Incidence and origin of ‘null’ alleles in the (AC) n microsatellite markers; citation_author=DF Callen, AD Thompson, Y Shen, HA Phillips, RI Richards, JC Mulley, GR Suthereland; citation_volume=52; citation_publication_date=1993; citation_pages=922-927; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=J Apicult Res; citation_title=Genetic differentiation of the honey bee (Apis cerana) in Thailand: evidence from microsatellite polymorphism; citation_author=S Sittipraneed, S Laoaroon, S Klinbunga, S Wongsiri; citation_volume=40; citation_publication_date=2001; citation_pages=9-16; citation_doi=10.1080/00218839.2001.11101043; citation_id=CR23">
<meta name="citation_reference" content="citation_title=Mount Huangshan; citation_publication_date=1994; citation_id=CR24; citation_author=CR Jing; citation_publisher=Science Press">
<meta name="citation_author" content="Fang Liu">
<meta name="citation_author_institution" content="Honey bee Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, China">
<meta name="citation_author" content="Tengfei Shi">
<meta name="citation_author_institution" content="Honey bee Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, China">
<meta name="citation_author" content="Sisi Huang">
<meta name="citation_author_institution" content="Honey bee Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, China">
<meta name="citation_author" content="Linsheng Yu">
<meta name="citation_author_institution" content="Honey bee Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, China">
<meta name="citation_author_institution" content="Apiculture Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, China">
<meta name="citation_author" content="Shoudong Bi">
<meta name="citation_author_institution" content="College of Forest and Garden, Anhui Agricultural University, Hefei, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0010-4',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0010-4","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Apis cerana;Honey bee;Microsatellite;Mount huang;SSRs"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic structure of Mount Huang honey bee ( Apis cerana ) populations: evidence from microsatellite polymorphism","publishedAt":1467763200000,"publishedAtDate":"2016-07-06","author":["Fang Liu","Tengfei Shi","Sisi Huang","Linsheng Yu","Shoudong Bi"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0010-4">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0010-4">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Genetic structure of Mount Huang honey bee ( Apis cerana ) populations: evidence from microsatellite polymorphism">
<meta property="og:description" content="The Mount Huang eastern honey bees (Apis cerana) are an endemic population, which is well adapted to the local agricultural and ecological environment. In this study, the genetic structure of seven eastern honey bees (A. cerana) populations from Mount Huang in China were analyzed by SSR (simple sequence repeat) markers. The results revealed that 16 pairs of primers used amplified a total of 143 alleles. The number of alleles per locus ranged from 6 to 13, with a mean value of 8.94 alleles per locus. Observed and expected heterozygosities showed mean values of 0.446 and 0.831 respectively. UPGMA cluster analysis grouped seven eastern honey bees in three groups. The results obtained show a high genetic diversity in the honey bee populations studied in Mount Huang, and high differentiation among all the populations, suggesting that scarce exchange of honey bee species happened in Mount Huang. Our study demonstrated that the Mount Huang honey bee populations still have a natural genome worth being protected for conservation.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0010-4;kwrd=Apis cerana,Honey bee,Microsatellite,Mount huang,SSRs;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0010-4&amp;kwrd=Apis%20cerana,Honey%20bee,Microsatellite,Mount%20huang,SSRs&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0010-4&amp;kwrd=Apis%20cerana,Honey%20bee,Microsatellite,Mount%20huang,SSRs&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec8" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic structure of Mount Huang honey bee (<em xmlns="" class="EmphasisTypeItalic">Apis cerana</em>) populations: evidence from microsatellite polymorphism</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true">
<ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Fang Liu</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Tengfei Shi</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Sisi Huang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Linsheng Yu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:yulinsheng@ahau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Shoudong Bi</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>
</li>
</ul>
<div class="AffiliationLegend"><div class="AffiliationLegendLine">
<span class="EqualContributionSymbol">†</span>Contributed equally</div></div>
</div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:8</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0010-4">https://doi.org/10.1186/s41065-016-0010-4</a></p>
<p class="Copyright">©  The Author(s) 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>4 February 2016</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>13 June 2016</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">6 July 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">The Mount Huang eastern honey bees (<em xmlns="" class="EmphasisTypeItalic">Apis cerana</em>) are an endemic population, which is well adapted to the local agricultural and ecological environment. In this study, the genetic structure of seven eastern honey bees (<em xmlns="" class="EmphasisTypeItalic">A. cerana</em>) populations from Mount Huang in China were analyzed by SSR (simple sequence repeat) markers.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">The results revealed that 16 pairs of primers used amplified a total of 143 alleles. The number of alleles per locus ranged from 6 to 13, with a mean value of 8.94 alleles per locus. Observed and expected heterozygosities showed mean values of 0.446 and 0.831 respectively. UPGMA cluster analysis grouped seven eastern honey bees in three groups.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">The results obtained show a high genetic diversity in the honey bee populations studied in Mount Huang, and high differentiation among all the populations, suggesting that scarce exchange of honey bee species happened in Mount Huang. Our study demonstrated that the Mount Huang honey bee populations still have a natural genome worth being protected for conservation.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Apis cerana</em>
              </li>
<li class="c-keywords__item" itemprop="about">Honey bee</li>
<li class="c-keywords__item" itemprop="about">Microsatellite</li>
<li class="c-keywords__item" itemprop="about">Mount huang</li>
<li class="c-keywords__item" itemprop="about">SSRs</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 SectionTypeIntroduction c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Mount Huang lies in the southern end of the Anhui province of China (Wannan), between 117°02′–118°55′ east longitude and 29°24′–30°24′ north latitude. It has a typical humid subtropical monsoon climate, luxuriantly diverse flora and fauna, including honey bees. The Mount Huang eastern honey bees (<em xmlns="" class="EmphasisTypeItalic">A. cerana</em>) are an endemic population, which is well adapted to the local agricultural and ecological environment. They have a particularly strong foraging instinct and resistance to parasitic mites as well as American foulbrood. This endemic population was listed in the first directory of potential breeding stock for conservation in 2001. The <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> conservation document Wannan (Wan D/XM01-19-87) was published according to ecological standards in Anhui province. However, since then, a massive replacement of the endemic honey bees has occurred, especially after the importation of <em xmlns="" class="EmphasisTypeItalic">Apis mellifera</em> [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. In some regions, this has resulted in almost complete replacement of endemic <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations with imported stocks.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Microsatellites are suitable nuclear markers used in population genetic studies in various taxonomic groups [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. This is due to their mass polymorphic loci, codominance, abundance throughout the genome, and relative ease of scoring [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The genetic structure of eastern honey bee populations from many regions of China have been analyzed using microsatellite markers. These regions mostly include areas in the middle east (Fujian, Jiangsu and Zhejiang provinces) and the south west (Tibei, Yunnan province and Hainan province). Zhu et al. [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] used five microsatellite loci to analyze the genetic structure of six <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations from Fujian province, their results showed that the mean expected heterozygosities (<em xmlns="" class="EmphasisTypeItalic">He</em>) of populations from most of regions in Fujian was 0.5171 . In another study, a significant difference in genetic structures was found among honey bees from six regions in Wuyi Mountain located in Fujian province, the mean <em xmlns="" class="EmphasisTypeItalic">He</em> of the six different region population was range from 0.4525–0.5613 [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Using six microsatellite loci, Ji et al. [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>] found higher population heterozygosity and abundant genetic diversity in <em xmlns="" class="EmphasisTypeItalic">Apis cerana</em> as well as in <em xmlns="" class="EmphasisTypeItalic">Apis mellifera</em> (western European honey bee) populations in Jiangsu province, they found the mean <em xmlns="" class="EmphasisTypeItalic">He</em> of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> population in this province was 0.4424. Cao et al. [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] analyzed the genetic structure of more than ten <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations from different regions in Zhejiang province, their results showed that the genetic diversity of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> in Zhejiang was relatively low, with the mean <em xmlns="" class="EmphasisTypeItalic">He</em> was 0.3179. Guo et al. [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] used 8 microsatellite markers found the <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations in Qinling-Data mountain areas has rich genetic polymorphism, with the mean <em xmlns="" class="EmphasisTypeItalic">He</em>, number of alleles, polymorphism information content (<em xmlns="" class="EmphasisTypeItalic">PIC</em>) were 0.6877, 7.71, 3.7488 respectively. Ji et al. [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] found significant differentiation among 20 <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations from geographically different regions in China using 21 microsatellite loci. 502 alleles were found, and the average value of <em xmlns="" class="EmphasisTypeItalic">He</em> of 21 microsatellites was 0.8689. Additionally, the mean <em xmlns="" class="EmphasisTypeItalic">He</em> of the populations from Wuding (Yunnan province), Hainan, Bomi Tibet was 0.695, 0.6641 and 0.562 respectively, revealing their relatively high genetic diversity. Genetic diversity of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations from Lushui in Yunnan province was evaluated with 21 microsatellite markers. 121 alleles were found, and the mean number of effective alleles was 5.76, the average <em xmlns="" class="EmphasisTypeItalic">He</em> and <em xmlns="" class="EmphasisTypeItalic">PIC</em> of all loci were 0.6752 and 0.6192, respectively, these results indicated the high level of genetic diversity in Lushui [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Six <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations distributed in different geographic areas in the south west Yunnan province were also surveyed using 21 microsatellite loci. All populations showed high levels of heterozygosity except the population from Diqing, the average <em xmlns="" class="EmphasisTypeItalic">He</em> of the total populations was 0.739, and a significant genetic differentiation was found among all six populations [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. Taken together, the genetic variation of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations is overall higher in the southwest of China than in the middle east of China.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">However, information about genetic variation of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> in Mount Huang of Anhui province, in south east China, is presently not available. Therefore, the objectives of this study were to examine the genetic structure of Mount Huang honey bee populations using microsatellite analyses. Our findings provide important information that can be used for effective management and conservation of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> in Wannan.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Samples</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Adult workers from 190 colonies were collected from seven locations covering the main bee keeping areas in Huangshan [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], south of Anhui province in China (Wannan). Over 25 colonies were obtained from Jing County (JC), Huangshan District (HSD), Qimen County (QMC), and Yi County (YC), whereas 30 colonies were collected from Jixi County (JXC), She County (SC) and Huizhou District (HZD). The distances between the locations are more than 10 km. Ten bees from each colony were subjected to genetic analysis. Honey bee samples were preserved in 95 % ethanol and kept at −20 °C until needed.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">DNA extraction and microsatellites amplification</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Total DNA was extracted from the thorax of each bee, following the method of Smith and Hagen [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], and the concentration was then estimated using ultraviolet spectrophotometer MV-2201. DNA was stored at −20 °C until needed. The primer sequences of these loci were as previously described by Solignac et al. [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. The annealing temperature and the concentration of MgCl<sub>2</sub> are listed in Table <span class="InternalRef"><a href="#Tab1">1</a></span>. The PCR conditions were as follow: one cycle at 95 °C for 5 min; 35 cycles of 95 °C for 45 s, 50–60 °C for 45 s, 72 °C for 60 s; and one cycle at 72 °C for 8 min. The PCR products were electrophoretically fractionated in 6% denaturating polyacrylamide gels and visualized by silver staining. The size of amplified microsatellites was measured using EDA20 Kodak imaging system (Eastman Kodak, USA).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Location of 16 microsatellite loci in chromosome or linkage group and PCR conditions</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Locus</p></th>
<th><p xmlns="" class="SimplePara">Chromosome or linkage group</p></th>
<th><p xmlns="" class="SimplePara">GenBank Accession number</p></th>
<th><p xmlns="" class="SimplePara">M<sub xmlns="http://www.w3.org/1999/xhtml">g</sub><sup xmlns="http://www.w3.org/1999/xhtml">2+</sup> (mmol/L)</p></th>
<th><p xmlns="" class="SimplePara">Annealing temperature (°C)</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP243</p></td>
<td><p xmlns="" class="SimplePara">ChrLG1</p></td>
<td><p xmlns="" class="SimplePara">AJ509466</p></td>
<td><p xmlns="" class="SimplePara">2.2</p></td>
<td><p xmlns="" class="SimplePara">57.5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A043</p></td>
<td><p xmlns="" class="SimplePara">ChrLG1</p></td>
<td><p xmlns="" class="SimplePara">AJ509256</p></td>
<td><p xmlns="" class="SimplePara">1.5</p></td>
<td><p xmlns="" class="SimplePara">55</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP274</p></td>
<td><p xmlns="" class="SimplePara">ChrLG3</p></td>
<td><p xmlns="" class="SimplePara">AJ509486</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AT105</p></td>
<td><p xmlns="" class="SimplePara">ChrLG5</p></td>
<td><p xmlns="" class="SimplePara">AJ509553</p></td>
<td><p xmlns="" class="SimplePara">2.2</p></td>
<td><p xmlns="" class="SimplePara">56.5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A113</p></td>
<td><p xmlns="" class="SimplePara">ChrLG6</p></td>
<td><p xmlns="" class="SimplePara">AJ509290</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">58.2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A107</p></td>
<td><p xmlns="" class="SimplePara">ChrLG7</p></td>
<td><p xmlns="" class="SimplePara">AJ509287</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">57.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A024d</p></td>
<td><p xmlns="" class="SimplePara">ChrLG7</p></td>
<td><p xmlns="" class="SimplePara">AJ509241</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">57.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A014</p></td>
<td><p xmlns="" class="SimplePara">ChrLG8</p></td>
<td><p xmlns="" class="SimplePara">AJ509239</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">55.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A088</p></td>
<td><p xmlns="" class="SimplePara">ChrLG8</p></td>
<td><p xmlns="" class="SimplePara">AJ509283</p></td>
<td><p xmlns="" class="SimplePara">1.6</p></td>
<td><p xmlns="" class="SimplePara">58.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A007</p></td>
<td><p xmlns="" class="SimplePara">ChrLG8</p></td>
<td><p xmlns="" class="SimplePara">AJ509236</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">55</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP033</p></td>
<td><p xmlns="" class="SimplePara">ChrLG10</p></td>
<td><p xmlns="" class="SimplePara">AJ509319</p></td>
<td><p xmlns="" class="SimplePara">2.2</p></td>
<td><p xmlns="" class="SimplePara">56.5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP297</p></td>
<td><p xmlns="" class="SimplePara">ChrLG12</p></td>
<td><p xmlns="" class="SimplePara">AJ509499</p></td>
<td><p xmlns="" class="SimplePara">1.0</p></td>
<td><p xmlns="" class="SimplePara">54</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A029</p></td>
<td><p xmlns="" class="SimplePara">ChrLG12</p></td>
<td><p xmlns="" class="SimplePara">AJ509245</p></td>
<td><p xmlns="" class="SimplePara">1.2</p></td>
<td><p xmlns="" class="SimplePara">58</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A035</p></td>
<td><p xmlns="" class="SimplePara">ChrLG14</p></td>
<td><p xmlns="" class="SimplePara">AJ509251</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">53.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A028</p></td>
<td><p xmlns="" class="SimplePara">ChrLG14</p></td>
<td><p xmlns="" class="SimplePara">AJ509244</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AG005C</p></td>
<td><p xmlns="" class="SimplePara">ChrLG16</p></td>
<td><p xmlns="" class="SimplePara">AJ509723</p></td>
<td><p xmlns="" class="SimplePara">2.0</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Statistical analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">The number of alleles, allele frequencies, observed and expected heterozygosity (<em xmlns="" class="EmphasisTypeItalic">Ho</em> and <em xmlns="" class="EmphasisTypeItalic">He</em>) of each microsatellite locus were computed with Microsatellite-Toolkit software [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Genetic and genotypic differentiation tests were conducted among the 7 populations using GENEPOP 4.1. The exact test for Hardy-Weinberg equilibrium (HWE), genetic structure and gene flow (Nm) were calculated with GENEPOP 4.1 [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. F-statistics between populations were calculated using the software FSTAT (v2.9.3) [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. The significance level of multiple comparisons was calculated using the sequential Bonferroni procedure [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">The number and frequency of alleles were used to obtain the Da genetic distance between each population. Population tree based on Da values was constructed using the neighbor-joining (NJ) algorithms and unweighted pair group method with arithmetic mean (UPGMA) using PHYLIP software package (version 3.69) [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>].</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 SectionTypeResults c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">The locations where honey bee samples were collected were mentioned in Yu et al. [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. The number and size of alleles, the value of <em xmlns="" class="EmphasisTypeItalic">Ho</em> and <em xmlns="" class="EmphasisTypeItalic">He</em> are listed in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1. A total of 143 alleles were detected at 16 microsatellite loci. The overall parameters per population are shown in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1. The gene diversity (the mean <em xmlns="" class="EmphasisTypeItalic">He</em>) varied between 0.597 ± 0.109 in the HSD population and 0.71 ± 0.056 in the SC population, and mean observed heterozygosities between 0.398 ± 0.186 in HSD population and 0.495 ± 0.235 in JC population. The observed heterozygosity (<em xmlns="" class="EmphasisTypeItalic">Ho</em>) was relatively high in all populations except that of JC at loci A035, and HSD population at loci A107. The mean number of alleles for each population ranged from 2.936 ± 0.853 in HSD to 6.438 ± 1.632 in SC population.</p>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">We detected significant deviations (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.05) from Hardy-Weinberg equilibrium in 86 out of 112 population-locus combination and all deviations were in favor of homozygotes (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Both genic and genotypic differentiation tests showed highly significant (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.0001) differentiation among all populations. Multilocus F<sub>IS</sub> values varied between 0.181 (A029) and 0.618 (A088), while the F<sub>IT</sub> values varied from 0.314 (A028) to 0.701 (AP297). A high heterozygote deficiency within populations and the total population were revealed by positive F<sub>IS</sub> and F<sub>IT</sub> (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). The value of F<sub>ST</sub> for the total populations was 0.171, all loci contributed significantly to the differentiation (<em xmlns="" class="EmphasisTypeItalic">P &lt;</em> 0.001).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                      <p xmlns="" class="SimplePara">The <em class="EmphasisTypeItalic">P</em> -value of every population and every locus in Hardy-Weinberg test</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead>
<tr class="header">
<th rowspan="2"><p xmlns="" class="SimplePara">locus</p></th>
<th colspan="7"><p xmlns="" class="SimplePara">
                                          <em class="EmphasisTypeItalic">P</em> value</p></th>
</tr>
<tr class="header">
<th><p xmlns="" class="SimplePara">JC</p></th>
<th><p xmlns="" class="SimplePara">HSD</p></th>
<th><p xmlns="" class="SimplePara">JXC</p></th>
<th><p xmlns="" class="SimplePara">YC</p></th>
<th><p xmlns="" class="SimplePara">QMC</p></th>
<th><p xmlns="" class="SimplePara">SC</p></th>
<th><p xmlns="" class="SimplePara">HZD</p></th>
</tr>
</thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP243</p></td>
<td><p xmlns="" class="SimplePara">0.4447</p></td>
<td><p xmlns="" class="SimplePara">0.2713</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0087</p></td>
<td><p xmlns="" class="SimplePara">0.2283</p></td>
<td><p xmlns="" class="SimplePara">0.0084</p></td>
<td><p xmlns="" class="SimplePara">0.0034</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A043</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0.0046</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.3076</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0008</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP274</p></td>
<td><p xmlns="" class="SimplePara">0.0003</p></td>
<td><p xmlns="" class="SimplePara">0.0995</p></td>
<td><p xmlns="" class="SimplePara">0.001</p></td>
<td><p xmlns="" class="SimplePara">0.0058</p></td>
<td><p xmlns="" class="SimplePara">0.2845</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0017</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AT105</p></td>
<td><p xmlns="" class="SimplePara">0.003</p></td>
<td><p xmlns="" class="SimplePara">0.3695</p></td>
<td><p xmlns="" class="SimplePara">0.0002</p></td>
<td><p xmlns="" class="SimplePara">0.071</p></td>
<td><p xmlns="" class="SimplePara">0.0716</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0813</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A113</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0023</p></td>
<td><p xmlns="" class="SimplePara">0.0725</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0025</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A107</p></td>
<td><p xmlns="" class="SimplePara">0.1234</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.1747</p></td>
<td><p xmlns="" class="SimplePara">0.322</p></td>
<td><p xmlns="" class="SimplePara">0.0402</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.7541</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A024d</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0019</p></td>
<td><p xmlns="" class="SimplePara">0.0027</p></td>
<td><p xmlns="" class="SimplePara">0.0106</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0009</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A014</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0.0092</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0117</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0004</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A088</p></td>
<td><p xmlns="" class="SimplePara">0.0115</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP033</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">0.0025</p></td>
<td><p xmlns="" class="SimplePara">0.0029</p></td>
<td><p xmlns="" class="SimplePara">0.0034</p></td>
<td><p xmlns="" class="SimplePara">0.0057</p></td>
<td><p xmlns="" class="SimplePara">0.007</p></td>
<td><p xmlns="" class="SimplePara">0.2327</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP297</p></td>
<td><p xmlns="" class="SimplePara">0.0052</p></td>
<td><p xmlns="" class="SimplePara">0.7253</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0018</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A029</p></td>
<td><p xmlns="" class="SimplePara">0.0243</p></td>
<td><p xmlns="" class="SimplePara">0.9913</p></td>
<td><p xmlns="" class="SimplePara">0.0195</p></td>
<td><p xmlns="" class="SimplePara">0.8768</p></td>
<td><p xmlns="" class="SimplePara">0.9816</p></td>
<td><p xmlns="" class="SimplePara">0.1932</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A007</p></td>
<td><p xmlns="" class="SimplePara">0.0148</p></td>
<td><p xmlns="" class="SimplePara">0.0001</p></td>
<td><p xmlns="" class="SimplePara">0.0033</p></td>
<td><p xmlns="" class="SimplePara">0.001</p></td>
<td><p xmlns="" class="SimplePara">0.0027</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A035</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.9728</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.101</p></td>
<td><p xmlns="" class="SimplePara">0.0111</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0026</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A028</p></td>
<td><p xmlns="" class="SimplePara">0.3279</p></td>
<td><p xmlns="" class="SimplePara">0.0358</p></td>
<td><p xmlns="" class="SimplePara">0.309</p></td>
<td><p xmlns="" class="SimplePara">0.0004</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0008</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AG005C</p></td>
<td><p xmlns="" class="SimplePara">0.0009</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0014</p></td>
<td><p xmlns="" class="SimplePara">0.0023</p></td>
<td><p xmlns="" class="SimplePara">0.2447</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td><p xmlns="" class="SimplePara">0.0012</p></td>
</tr>
</tbody>
</table></div></figure><figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                      <p xmlns="" class="SimplePara">F. coefficients of total of populations including 7 <em class="EmphasisTypeItalic">A. cerana</em> in Mount Huangshan of Wannan</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Locus</p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">F</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">IT</em>
                            </sub></p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">F</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">ST</em>
                            </sub></p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">F</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">IS</em>
                            </sub></p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP243</p></td>
<td><p xmlns="" class="SimplePara">0.361***</p></td>
<td><p xmlns="" class="SimplePara">0.108***</p></td>
<td><p xmlns="" class="SimplePara">0.283***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A043</p></td>
<td><p xmlns="" class="SimplePara">0.631***</p></td>
<td><p xmlns="" class="SimplePara">0.203***</p></td>
<td><p xmlns="" class="SimplePara">0.541***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP274</p></td>
<td><p xmlns="" class="SimplePara">0.453***</p></td>
<td><p xmlns="" class="SimplePara">0.131***</p></td>
<td><p xmlns="" class="SimplePara">0.372***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AT105</p></td>
<td><p xmlns="" class="SimplePara">0.390***</p></td>
<td><p xmlns="" class="SimplePara">0.134***</p></td>
<td><p xmlns="" class="SimplePara">0.295***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A113</p></td>
<td><p xmlns="" class="SimplePara">0.361***</p></td>
<td><p xmlns="" class="SimplePara">0.126***</p></td>
<td><p xmlns="" class="SimplePara">0.268***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A107</p></td>
<td><p xmlns="" class="SimplePara">0.360***</p></td>
<td><p xmlns="" class="SimplePara">0.198***</p></td>
<td><p xmlns="" class="SimplePara">0.200***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A024d</p></td>
<td><p xmlns="" class="SimplePara">0.527***</p></td>
<td><p xmlns="" class="SimplePara">0.110***</p></td>
<td><p xmlns="" class="SimplePara">0.470***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A014</p></td>
<td><p xmlns="" class="SimplePara">0.568***</p></td>
<td><p xmlns="" class="SimplePara">0.170***</p></td>
<td><p xmlns="" class="SimplePara">0.479***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A088</p></td>
<td><p xmlns="" class="SimplePara">0.681***</p></td>
<td><p xmlns="" class="SimplePara">0.166***</p></td>
<td><p xmlns="" class="SimplePara">0.618***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP033</p></td>
<td><p xmlns="" class="SimplePara">0.349***</p></td>
<td><p xmlns="" class="SimplePara">0.181***</p></td>
<td><p xmlns="" class="SimplePara">0.208***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AP297</p></td>
<td><p xmlns="" class="SimplePara">0.701***</p></td>
<td><p xmlns="" class="SimplePara">0.268***</p></td>
<td><p xmlns="" class="SimplePara">0.589***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A029</p></td>
<td><p xmlns="" class="SimplePara">0.404***</p></td>
<td><p xmlns="" class="SimplePara">0.272***</p></td>
<td><p xmlns="" class="SimplePara">0.181***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A007</p></td>
<td><p xmlns="" class="SimplePara">0.444***</p></td>
<td><p xmlns="" class="SimplePara">0.077***</p></td>
<td><p xmlns="" class="SimplePara">0.398***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A035</p></td>
<td><p xmlns="" class="SimplePara">0.604***</p></td>
<td><p xmlns="" class="SimplePara">0.277***</p></td>
<td><p xmlns="" class="SimplePara">0.458***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">A028</p></td>
<td><p xmlns="" class="SimplePara">0.314***</p></td>
<td><p xmlns="" class="SimplePara">0.116***</p></td>
<td><p xmlns="" class="SimplePara">0.225***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AG005C</p></td>
<td><p xmlns="" class="SimplePara">0.481***</p></td>
<td><p xmlns="" class="SimplePara">0.150***</p></td>
<td><p xmlns="" class="SimplePara">0.390***</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mean ± SD</p></td>
<td><p xmlns="" class="SimplePara">0.47 ± 0.032***</p></td>
<td><p xmlns="" class="SimplePara">0.171 ± 0.016***</p></td>
<td><p xmlns="" class="SimplePara">0.369 ± 0.035***</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Note: Mean estimates from jack-knife over loci, standard deviations (SD) are given in parentheses; ***<em class="EmphasisTypeItalic">p</em> &lt; 0.001</p>
                  </div></figure>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Genetic distances between pairs of Mount Huang <em xmlns="" class="EmphasisTypeItalic">A. cerana,</em> in the present study, were 0.2104-0.5027 (Table <span class="InternalRef"><a href="#Tab4">4</a></span>, lower triangle). The pairwise Nm value (Table <span class="InternalRef"><a href="#Tab4">4</a></span>, upper triangle) between populations varied between 0.6049 (SC and JXC) and 0.8345 (YC and HSD). The neighbor-joining tree (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>) was constructed according to the Da genetic distance between each pair of populations. The SC population formed a group separating from other populations. JXC and HZD formed a cluster, JC, HSD, YC and QMC grouped together.<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                      <p xmlns="" class="SimplePara">Nei’s genetic distance, Da (lower triangle) and the gene flow, Nm (up triangle) between populations</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Population</p></th>
<th><p xmlns="" class="SimplePara">JX</p></th>
<th><p xmlns="" class="SimplePara">HSQ</p></th>
<th><p xmlns="" class="SimplePara">JXX</p></th>
<th><p xmlns="" class="SimplePara">YX</p></th>
<th><p xmlns="" class="SimplePara">QMX</p></th>
<th><p xmlns="" class="SimplePara">SX</p></th>
<th><p xmlns="" class="SimplePara">HZQ</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">JX</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
<td><p xmlns="" class="SimplePara">0.7812</p></td>
<td><p xmlns="" class="SimplePara">0.7326</p></td>
<td><p xmlns="" class="SimplePara">0.8103</p></td>
<td><p xmlns="" class="SimplePara">0.7714</p></td>
<td><p xmlns="" class="SimplePara">0.6623</p></td>
<td><p xmlns="" class="SimplePara">0.7503</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">HSQ</p></td>
<td><p xmlns="" class="SimplePara">0.2469</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
<td><p xmlns="" class="SimplePara">0.6594</p></td>
<td><p xmlns="" class="SimplePara">0.8345</p></td>
<td><p xmlns="" class="SimplePara">0.6827</p></td>
<td><p xmlns="" class="SimplePara">0.6053</p></td>
<td><p xmlns="" class="SimplePara">0.6148</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">JXX</p></td>
<td><p xmlns="" class="SimplePara">0.3112</p></td>
<td><p xmlns="" class="SimplePara">0.4164</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
<td><p xmlns="" class="SimplePara">0.7201</p></td>
<td><p xmlns="" class="SimplePara">0.6678</p></td>
<td><p xmlns="" class="SimplePara">0.6049</p></td>
<td><p xmlns="" class="SimplePara">0.7016</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">YX</p></td>
<td><p xmlns="" class="SimplePara">0.2104</p></td>
<td><p xmlns="" class="SimplePara">0.1809</p></td>
<td><p xmlns="" class="SimplePara">0.3284</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
<td><p xmlns="" class="SimplePara">0.8050</p></td>
<td><p xmlns="" class="SimplePara">0.6749</p></td>
<td><p xmlns="" class="SimplePara">0.7239</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">QMX</p></td>
<td><p xmlns="" class="SimplePara">0.2595</p></td>
<td><p xmlns="" class="SimplePara">0.3817</p></td>
<td><p xmlns="" class="SimplePara">0.4038</p></td>
<td><p xmlns="" class="SimplePara">0.2169</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
<td><p xmlns="" class="SimplePara">0.6335</p></td>
<td><p xmlns="" class="SimplePara">0.6568</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SX</p></td>
<td><p xmlns="" class="SimplePara">0.4120</p></td>
<td><p xmlns="" class="SimplePara">0.5020</p></td>
<td><p xmlns="" class="SimplePara">0.5027</p></td>
<td><p xmlns="" class="SimplePara">0.3932</p></td>
<td><p xmlns="" class="SimplePara">0.4565</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
<td><p xmlns="" class="SimplePara">0.6456</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">HZQ</p></td>
<td><p xmlns="" class="SimplePara">0.2872</p></td>
<td><p xmlns="" class="SimplePara">0.4864</p></td>
<td><p xmlns="" class="SimplePara">0.3544</p></td>
<td><p xmlns="" class="SimplePara">0.3231</p></td>
<td><p xmlns="" class="SimplePara">0.4204</p></td>
<td><p xmlns="" class="SimplePara">0.4375</p></td>
<td><p xmlns="" class="SimplePara">–––––</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">The line means blank. There is no genetic distance between the same populations</p>
                  </div></figure><figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0010-4/MediaObjects/41065_2016_10_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0010-4/MediaObjects/41065_2016_10_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0010-4/MediaObjects/41065_2016_10_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0010-4/MediaObjects/41065_2016_10_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                  </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">Dendogram of phylogenetic relationships among 7 <em class="EmphasisTypeItalic">Apis cerana</em> populations based on Nei’s genetic distance using neighbor-joining method</p>
                    </div></figcaption></figure>
</div>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Population differentiation was detected in Mount Huangshan <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> through analysis of microsatellite polymorphism. There are limited population genetic studies of Mount Huangshan <em xmlns="" class="EmphasisTypeItalic">A. cerana</em>, using microsatellite analysis. A previous study by Yu et al. [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], using morphometric analysis reported similar finding. Therefore, our study adds to the knowledge of the genetic changes that may be occurring in this important endemic honey bee population. This knowledge can be used for genetic management and conservation efforts, for the preservation of this population, which was first listed in the directory of potential breeding stock for conservation in 2001. The pure <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> population has several desirable advantages biological features, including disease resistance and the ability to get rid of bee mites from colonies. To date, other studies of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> by microsatellite loci have focused on the eastern and southern populations of China. According to those reports, the expected heterozygosity levels were highest among honey bee populations in the south west region, which ranged between 0.5625 and 0.738 [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>–<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>], and were lowest among honey bees in the middle east reported as 0.3179-0.5613 [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>–<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. In the present study, the microsatellite mean expected heterozygosity values for ranged between 0.597 and 0.781, which is similar to those of honey bees in Yunnan. This similarity may be attributable to similar geographic environments in Yunnan and Mount Huang, which are both mountainous with high altitude.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The microsatellite data also showed that, the Mount Huang populations seem to be more variable compared to those in the eastern Chinese region. Average allele numbers and heterozygosities were higher in Mount Huang populations than in Fujian [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], Jiangsu [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], Zhejiang [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. This difference may, in part, be attributable to differences in the evaluated population sizes.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Significant deviations from Hardy-Weinberg equilibrium were observed at loci A024d, A014, A088, and A007 in all investigated populations even with correction by sequential Bonferroni method. Such deviation may be due to miscoring of alleles resulting from the existence of shatter bands [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. However, in this case it is unlikely this is the cause, because the band intensity among a group of stutter bands was very carefully examined in order to obtain consistent allele designation. Due to the use of heterospecific primers in this species, the other factor may be a result of the presence of null alleles at these microsatellite loci [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. It will be premature to conclude which of these factors contribute heterozygote deficiencies of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> in Mount Huang until these bees are examined using microsatellite primers developed from <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> rather than those from <em xmlns="" class="EmphasisTypeItalic">A. mellifera</em> [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Considering the F<sub>ST</sub> estimates, all 16 microsatellite loci contributed mostly to differentiate the populations (Table <span class="InternalRef"><a href="#Tab3">3</a></span>), corresponding to the low Nm value (Nm &lt; 1) between populations, which indicated. The F<sub>ST</sub> value of AP033 loci here is similar to Ji et al. [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. The bee samples we collected in present study included wild honey bees and domestic honey bees, which also may contribute to the large differentiation among honey bees from different locations. A total of seven <em xmlns="" class="EmphasisTypeItalic">Apis cerana</em> populations in Huangshan were isolated by high mountains [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], it is unlikely gene flow for these honey bees.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Large genetic distances were observed among all populations, especially those between <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> from SC population and other populations (<em xmlns="" class="EmphasisTypeItalic">d</em> = 0.3932-0.5027). The seven <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> populations in this study could then be genetically divided into three different groups, which included one group with JC, HSD, YC, and QMC populations, another with JXC and HZD populations, while the SC population was a separate group (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). YC was firstly linked to HSD, which was also indicated by the relative high Nm values (0.7812) between them. While SC formed one group alone, it is located between Mount Huang and Mount Tianmu, this may have been an important barrier to gene flow for SC <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> and other populations.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec8" class="Section1 RenderAsSection1 SectionTypeConclusion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Microsatellite data obtained in this study demonstrated that the Mount Huanghoney bee populations still have a natural genome worth being protected for conservation. However, further screening of <em xmlns="" class="EmphasisTypeItalic">A. cerana</em> on a larger geographic scale should be carried out to fully elucidate the patterns of conspecific and intraspecific population structure in Wannan <em xmlns="" class="EmphasisTypeItalic">A. cerana</em>.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgements</h3>
<p xmlns="" class="SimplePara">This work was supported by grants earmarked for Modern Agro-industry Technology Research System (CARS-45-KXJ9), and National Natural Science Foundation of China (No, 31272511; No. 31302039). We are grateful to Chengwu Wu, the director of Bureau of Animal husbandry of Huangshan in Anhui Province, and the Five Bee Home Limited Company for their help in bee-collecting. And we thank Ting Ji and Ling Yin from Yangzhou University for their kindly help in the research.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par20" class="Para">FL and LSY designed research; FL and STF performed research; FL, SSH and SDB analyzed data and wrote the paper. All authors read and approved the final manuscript.</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par21" class="Para">The authors declare that they have no competing interests.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside class="Appendix" id="App1"><h3 class="Heading">Additional files</h3>
<div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-016-0010-4/MediaObjects/41065_2016_10_MOESM1_ESM.doc"><span class="caption-number">Additional file 1: Table S1.</span></a>
                          <span xmlns="" class="SimplePara">Allele frequencies, proportion of observed (<em class="EmphasisTypeItalic">Ho</em>) and expected (<em class="EmphasisTypeItalic">He</em>) heterozygosity of 16 microsatellites loci across seven populations of Mount Huangshan <em class="EmphasisTypeItalic">A. cerana</em>, n is the number of individual bees examined. (DOC 218 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Honey bee Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, 230036, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">College of Forest and Garden, Anhui Agricultural University, Hefei, 230000, 230036, Anhui, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Apiculture Research Institute, College of Animal Science &amp; Technology, Anhui Agricultural University, Hefei, 230036, Anhui, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu LS, Zou YD, Bi SD S. Relation between population dynamics and distribution of honey bees and ecological environment. Chin J Appl Ecol. 2006;17:1465–8. In Chinese.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Relation%20between%20population%20dynamics%20and%20distribution%20of%20honey%20bees%20and%20ecological%20environment&amp;author=LS.%20Yu&amp;author=YD.%20Zou&amp;author=S.%20Bi%20SD&amp;journal=Chin%20J%20Appl%20Ecol&amp;volume=17&amp;pages=1465-1468&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bowcock AM, Linares RA, Tomforhde J, Minch E, Kidd JR, Sforza CLL. High resolution of human evolutionary trees with polymorphic microsatellites. Nature. 1994;368:455–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/368455a0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7510853"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High%20resolution%20of%20human%20evolutionary%20trees%20with%20polymorphic%20microsatellites&amp;author=AM.%20Bowcock&amp;author=RA.%20Linares&amp;author=J.%20Tomforhde&amp;author=E.%20Minch&amp;author=JR.%20Kidd&amp;author=CLL.%20Sforza&amp;journal=Nature&amp;volume=368&amp;pages=455-457&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Taylor AC, Sherwin WB, Wayne RK. Genetic variation of microsatellite loci in a bottlenecked species: the northern hairy-nosed wombat <em class="EmphasisTypeItalic">Lasiorhinus krefftii</em>. Mol Ecol. 1994;3:277–99.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.1994.tb00068.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7921355"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variation%20of%20microsatellite%20loci%20in%20a%20bottlenecked%20species%3A%20the%20northern%20hairy-nosed%20wombat%20Lasiorhinus%20krefftii%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=AC.%20Taylor&amp;author=WB.%20Sherwin&amp;author=RK.%20Wayne&amp;journal=Mol%20Ecol&amp;volume=3&amp;pages=277-299&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bodur ÇM, Kence KA. Genetic structure of honeybee, <em class="EmphasisTypeItalic">Apis mellifera</em> L. (Hymenoptera:Apidae) populations of Turkey inferred from microsatellite analysis. J Apicult Res. 2007;46:50–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/00218839.2007.11101366"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20of%20honeybee%2C%20Apis%20mellifera%20L.%20%28Hymenoptera%3AApidae%29%20populations%20of%20Turkey%20inferred%20from%20microsatellite%20analysis&amp;author=%C3%87M.%20Bodur&amp;author=KA.%20Kence&amp;journal=J%20Apicult%20Res&amp;volume=46&amp;pages=50-56&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhu XJ, Zhou BF, Wu XD, Xu JX, Xia XC. Genetic diversity of <em class="EmphasisTypeItalic">Apis cerana cerana</em> in Fujian based on microsatellite markers. J Fujian Agric Forestry Univ (National Science Edition). 2011;40(4):407–11. In Chinese.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Apis%20cerana%20cerana%20in%20Fujian%20based%20on%20microsatellite%20markers&amp;author=XJ.%20Zhu&amp;author=BF.%20Zhou&amp;author=XD.%20Wu&amp;author=JX.%20Xu&amp;author=XC.%20Xia&amp;journal=J%20Fujian%20Agric%20Forestry%20Univ%20%28National%20Science%20Edition%29&amp;volume=40&amp;issue=4&amp;pages=407-411&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhu XJ, Xu XJ, Zhou MQ, Wu XD, Zhou BF. Genetic analysis of <em class="EmphasisTypeItalic">Apis cerana cerana</em> in Wuyi mountain nature reserve based on microsatellite DNA. Fujian J Agric Sci. 2011;26:935–40 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20Apis%20cerana%20cerana%20in%20Wuyi%20mountain%20nature%20reserve%20based%20on%20microsatellite%20DNA&amp;author=XJ.%20Zhu&amp;author=XJ.%20Xu&amp;author=MQ.%20Zhou&amp;author=XD.%20Wu&amp;author=BF.%20Zhou&amp;journal=Fujian%20J%20Agric%20Sci&amp;volume=26&amp;pages=935-940&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ji T, Chen J, Yin L, Bao WB, Chen GH. Study on microsatellite genetic polymorphism of wild and domestic <em class="EmphasisTypeItalic">Apis cerana cerana</em> populations from Jiangsu province. China Anim Husbandry Vet Med. 2007;34:39–41 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Study%20on%20microsatellite%20genetic%20polymorphism%20of%20wild%20and%20domestic%20Apis%20cerana%20cerana%20populations%20from%20Jiangsu%20province&amp;author=T.%20Ji&amp;author=J.%20Chen&amp;author=L.%20Yin&amp;author=WB.%20Bao&amp;author=GH.%20Chen&amp;journal=China%20Anim%20Husbandry%20Vet%20Med&amp;volume=34&amp;pages=39-41&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Su XL, Cao L, Zhao DX, Hua Q, Hu FL. Genetic Diversity of Microsatellite DNA for <em class="EmphasisTypeItalic">Apis cerana cerana</em> in Zhejiang. Apicult Chin. 2013;64:10–1 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20Diversity%20of%20Microsatellite%20DNA%20for%20Apis%20cerana%20cerana%20in%20Zhejiang&amp;author=XL.%20Su&amp;author=L.%20Cao&amp;author=DX.%20Zhao&amp;author=Q.%20Hua&amp;author=FL.%20Hu&amp;journal=Apicult%20Chin&amp;volume=64&amp;pages=10-11&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guo HP, Zhou SJ, Xu XJ, Xu JX, Yu YL, Yang KJ, et al. Population genetic analysis of <em class="EmphasisTypeItalic">Apis cerana cerana</em> from the Qinling-Daba mountain areas based on microsatellite DNA. Acta Entomol Sin. 2016;59:337–45 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20genetic%20analysis%20of%20Apis%20cerana%20cerana%20from%20the%20Qinling-Daba%20mountain%20areas%20based%20on%20microsatellite%20DNA&amp;author=HP.%20Guo&amp;author=SJ.%20Zhou&amp;author=XJ.%20Xu&amp;author=JX.%20Xu&amp;author=YL.%20Yu&amp;author=KJ.%20Yang&amp;author=DY.%20Chen&amp;author=BF.%20Zhou&amp;journal=Acta%20Entomol%20Sin&amp;volume=59&amp;pages=337-345&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ji T, Yin L, Chen G. Genetic diversity and population structure of Chinese honeybees (<em class="EmphasisTypeItalic">Apis cerana</em>) under microsatellite markers. Afr J Biotechnol. 2011;10:1712–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20population%20structure%20of%20Chinese%20honeybees%20%28Apis%20cerana%29%20under%20microsatellite%20markers&amp;author=T.%20Ji&amp;author=L.%20Yin&amp;author=G.%20Chen&amp;journal=Afr%20J%20Biotechnol&amp;volume=10&amp;pages=1712-1720&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lin D, Yin L, Jing T, Liu M, Chen HG. Study on genetic diversity of Apis cerana cerana populations from Lushui by microsatellite markers. Apiculture of China. 2009;60:9–11 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Study%20on%20genetic%20diversity%20of%20Apis%20cerana%20cerana%20populations%20from%20Lushui%20by%20microsatellite%20markers&amp;author=D.%20Lin&amp;author=L.%20Yin&amp;author=T.%20Jing&amp;author=M.%20Liu&amp;author=HG.%20Chen&amp;journal=Apiculture%20of%20China&amp;volume=60&amp;pages=9-11&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yin L, Ji T, Chen GH. Study on genetic diversity and genetic differentiation of six geographic populations of <em class="EmphasisTypeItalic">Apis cerana</em> (Hymenoptera: Apidae) in Yunnan by microsatellite markers. Southwest China J Agric Sci. 2011;24:772–9 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Study%20on%20genetic%20diversity%20and%20genetic%20differentiation%20of%20six%20geographic%20populations%20of%20Apis%20cerana%20%28Hymenoptera%3A%20Apidae%29%20in%20Yunnan%20by%20microsatellite%20markers&amp;author=L.%20Yin&amp;author=T.%20Ji&amp;author=GH.%20Chen&amp;journal=Southwest%20China%20J%20Agric%20Sci&amp;volume=24&amp;pages=772-779&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu LS, Liu F, Huang SS, Bi SD, Zong C, Wang TS. Morphmetric analysis of <em class="EmphasisTypeItalic">Apis cerana</em> populations in Huangshan. China J Apic Sci. 2013;57:117–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Morphmetric%20analysis%20of%20Apis%20cerana%20populations%20in%20Huangshan&amp;author=LS.%20Yu&amp;author=F.%20Liu&amp;author=SS.%20Huang&amp;author=SD.%20Bi&amp;author=C.%20Zong&amp;author=TS.%20Wang&amp;journal=China%20J%20Apic%20Sci&amp;volume=57&amp;pages=117-124&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith DR, Hagen RH. The biogeography of <em class="EmphasisTypeItalic">Apis cerana</em> as revealed by mitochondrial DNA sequence data. J Kansas EntomolSoc. 1996;69:294–310.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20biogeography%20of%20Apis%20cerana%20as%20revealed%20by%20mitochondrial%20DNA%20sequence%20data&amp;author=DR.%20Smith&amp;author=RH.%20Hagen&amp;journal=J%20Kansas%20EntomolSoc&amp;volume=69&amp;pages=294-310&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Solignac M, Vautrin D, Baudry E, Mougel F, Loiseau A, Cornuet JM. A microsatellite-based linkage Map of the honeybee, Apis mellifera L. Genetics. 2004;167(1):253–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.167.1.253"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15166152"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1470837"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20microsatellite-based%20linkage%20Map%20of%20the%20honeybee%2C%20Apis%20mellifera%20L&amp;author=M.%20Solignac&amp;author=D.%20Vautrin&amp;author=E.%20Baudry&amp;author=F.%20Mougel&amp;author=A.%20Loiseau&amp;author=JM.%20Cornuet&amp;journal=Genetics&amp;volume=167&amp;issue=1&amp;pages=253-262&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei M. Analysis of gene diversity in subdivided populations. P Natl Acad Sci USA. 1973;70:3321–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.70.12.3321"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20gene%20diversity%20in%20subdivided%20populations&amp;author=M.%20Nei&amp;journal=P%20Natl%20Acad%20Sci%20USA&amp;volume=70&amp;pages=3321-3323&amp;publication_year=1973"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rousset F. GENEPOP’ 007: a complete reimplementation of the GENEPOP software for windows and linux. Mol Ecol Resour. 2008;8:103–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2007.01931.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21585727"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GENEPOP%E2%80%99%20007%3A%20a%20complete%20reimplementation%20of%20the%20GENEPOP%20software%20for%20windows%20and%20linux&amp;author=F.%20Rousset&amp;journal=Mol%20Ecol%20Resour&amp;volume=8&amp;pages=103-106&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goudet J. FSTAT, a program to estimate and test gene diversities and fixation indices (version 2.9.3). Available from <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www2.unil.ch/popgen/softwares/fstat.htm" target="_blank" class="ext-link"><span class="RefSource">http://www2.unil.ch/popgen/softwares/fstat.htm</span></a></span>. 2001. Updated from Goudet (1995).</span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rice WS. Analyzing tables of statistical tests. Evolution. 1989;43:223–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2409177"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analyzing%20tables%20of%20statistical%20tests&amp;author=WS.%20Rice&amp;journal=Evolution&amp;volume=43&amp;pages=223-225&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei MF, Tajima TY. Accuracy of estimated phylogenetic trees from molecular data. J Mol Evol. 1983;19:153–70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02300753"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6571220"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Accuracy%20of%20estimated%20phylogenetic%20trees%20from%20molecular%20data&amp;author=MF.%20Nei&amp;author=TY.%20Tajima&amp;journal=J%20Mol%20Evol&amp;volume=19&amp;pages=153-170&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">O’Reilly P, Wright JM. The evolving technology of DNA fingerprinting and its application to fisheries and aquaculture. J Fish Biol. 1995;47:29–55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1095-8649.1995.tb06042.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20evolving%20technology%20of%20DNA%20fingerprinting%20and%20its%20application%20to%20fisheries%20and%20aquaculture&amp;author=P.%20O%E2%80%99Reilly&amp;author=JM.%20Wright&amp;journal=J%20Fish%20Biol&amp;volume=47&amp;pages=29-55&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Callen DF, Thompson AD, Shen Y, Phillips HA, Richards RI, Mulley JC, et al. Incidence and origin of ‘null’ alleles in the (AC) n microsatellite markers. Am J Hum Genet. 1993;52:922–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8488841"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1682051"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Incidence%20and%20origin%20of%20%E2%80%98null%E2%80%99%20alleles%20in%20the%20%28AC%29%20n%20microsatellite%20markers&amp;author=DF.%20Callen&amp;author=AD.%20Thompson&amp;author=Y.%20Shen&amp;author=HA.%20Phillips&amp;author=RI.%20Richards&amp;author=JC.%20Mulley&amp;author=GR.%20Suthereland&amp;journal=Am%20J%20Hum%20Genet&amp;volume=52&amp;pages=922-927&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sittipraneed S, Laoaroon S, Klinbunga S, Wongsiri S. Genetic differentiation of the honey bee (<em class="EmphasisTypeItalic">Apis cerana</em>) in Thailand: evidence from microsatellite polymorphism. J Apicult Res. 2001;40:9–16.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/00218839.2001.11101043"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20differentiation%20of%20the%20honey%20bee%20%28Apis%20cerana%29%20in%20Thailand%3A%20evidence%20from%20microsatellite%20polymorphism&amp;author=S.%20Sittipraneed&amp;author=S.%20Laoaroon&amp;author=S.%20Klinbunga&amp;author=S.%20Wongsiri&amp;journal=J%20Apicult%20Res&amp;volume=40&amp;pages=9-16&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jing CR. Mount Huangshan. Beijing: Science Press; 1994.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mount%20Huangshan&amp;author=CR.%20Jing&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">© The Author(s) 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0010-4" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-016-0010-4">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0010-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0010-4?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0010-4 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0010-4 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0010-4 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0010-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0010-4">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0010-4"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0010-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0010-4">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0010-4"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0010-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0010-4">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0010-4"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0010-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0010-4">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0010-4"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0010-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0010-4">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0010-4"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-016-0010-4/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-016-0010-4">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0010-4">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0010-4" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-016-0010-4&amp;volume=153&amp;issue=1&amp;title=Genetic+structure+of+Mount+Huang+honey+bee+%28+Apis+cerana+%29+populations%3A+evidence+from+microsatellite+polymorphism&amp;author_name=Fang+Liu&amp;start_page=1&amp;end_page=6">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-016-0010-4" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
