<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Characterization of the Tibet plateau Jerusalem artichoke (Helianthus tuberosus L.) transcriptome by de novo assembly to discover genes associated with fructan synthesis and SSR analysis | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Jerusalem artichoke (Helianthus tuberosus L.) is a characteristic crop in the Qinghai-Tibet Plateau which has rapidly developed and gained socioeconomic importance in recent years. Fructans are abundant in tubers and represent the foundation for their formation, processing and utilization of yield; and are also widely used in new sugar-based materials, bioenergy processing, ecological management, and functional feed. To identify key genes in the metabolic pathway of fructans in Jerusalem artichoke, high-throughput sequencing was performed using Illumina Hi Seqâ¢ 2500 equipment to construct a transcriptome library. Qinghai-Tibet Plateau Jerusalem artichoke âQingyu No.1â was used as the material; roots, stems, leaves, flowers and tubers of Jerusalem artichoke in its flowering stage were mixed into a mosaic of the Jerusalem artichoke transcriptome library, obtaining 63,089 unigenes with an average length of 713.6âbp. Gene annotation through the Nr, Swiss Prot, GO, KOG and KEGG databases revealed 34.95 and 46.91% of these unigenes had similar sequences in the Nr and Swiss Prot databases. The GO classification showed the Jerusalem artichoke unigenes were divided into three ontologies, with a total of 49 functional groups encompassing biological processes, cellular components, and molecular functions. Among them, there were more unigenes involved in the functional groups for cellular processes, metabolic processes, and single-organism processes. 38,999 unigenes were annotated by KOG and divided into 25 categories according to their functions; the most common annotation being general function prediction. A total of 13,878 unigenes (22%) were annotated in the KEGG database, with the largest proportion corresponding to pathways related to carbohydrate metabolism. A total of 12 unigenes were involved in the synthesis and degradation of fructan. Cluster analysis revealed the candidate 12 unigene proteins were dispersed in the 5 major families of proteins involved in fructan synthesis and degradation. The synergistic effect of INV gene is necessary during fructose synthesis and degradation in Jerusalem artichoke tuber development. The sequencing data from the transcriptome of this species can provide a reliable data basis for the identification and assessment of the expression of the members of the INV gene family.A simple sequence repeat (SSR) loci search was performed on the transcriptome data of Jerusalem artichoke, identifying 6635 eligible SSR loci with a large proportion of dinucleotide and trinucleotide repeats, and the most different motifs were repeated 5 times and 6 times. Dinucleotide and trinucleotide repeat motifs were the most frequent, with AG/CT and ACC/GGT repeat motifs accounting for the highest proportion. In this study, a database search of the transcriptome of the Jerusalem artichoke from the Qinghai Tibet Plateau was conducted by high throughput sequencing technology to obtain important transcriptional and SSR loci information. This allowed characterization of the overall expression features of the Jerusalem artichoke transcriptome, identifying the key genes involved in metabolism in this species. In turn, this offers a foundation for further research on the regulatory mechanisms of fructan metabolism in Jerusalem artichoke.">
<meta name="description" content="Jerusalem artichoke (Helianthus tuberosus L.) is a characteristic crop in the Qinghai-Tibet Plateau which has rapidly developed and gained socioeconomic importance in recent years. Fructans are abundant in tubers and represent the foundation for their formation, processing and utilization of yield; and are also widely used in new sugar-based materials, bioenergy processing, ecological management, and functional feed. To identify key genes in the metabolic pathway of fructans in Jerusalem artichoke, high-throughput sequencing was performed using Illumina Hi Seqâ¢ 2500 equipment to construct a transcriptome library. Qinghai-Tibet Plateau Jerusalem artichoke âQingyu No.1â was used as the material; roots, stems, leaves, flowers and tubers of Jerusalem artichoke in its flowering stage were mixed into a mosaic of the Jerusalem artichoke transcriptome library, obtaining 63,089 unigenes with an average length of 713.6âbp. Gene annotation through the Nr, Swiss Prot, GO, KOG and KEGG databases revealed 34.95 and 46.91% of these unigenes had similar sequences in the Nr and Swiss Prot databases. The GO classification showed the Jerusalem artichoke unigenes were divided into three ontologies, with a total of 49 functional groups encompassing biological processes, cellular components, and molecular functions. Among them, there were more unigenes involved in the functional groups for cellular processes, metabolic processes, and single-organism processes. 38,999 unigenes were annotated by KOG and divided into 25 categories according to their functions; the most common annotation being general function prediction. A total of 13,878 unigenes (22%) were annotated in the KEGG database, with the largest proportion corresponding to pathways related to carbohydrate metabolism. A total of 12 unigenes were involved in the synthesis and degradation of fructan. Cluster analysis revealed the candidate 12 unigene proteins were dispersed in the 5 major families of proteins involved in fructan synthesis and degradation. The synergistic effect of INV gene is necessary during fructose synthesis and degradation in Jerusalem artichoke tuber development. The sequencing data from the transcriptome of this species can provide a reliable data basis for the identification and assessment of the expression of the members of the INV gene family.A simple sequence repeat (SSR) loci search was performed on the transcriptome data of Jerusalem artichoke, identifying 6635 eligible SSR loci with a large proportion of dinucleotide and trinucleotide repeats, and the most different motifs were repeated 5 times and 6 times. Dinucleotide and trinucleotide repeat motifs were the most frequent, with AG/CT and ACC/GGT repeat motifs accounting for the highest proportion. In this study, a database search of the transcriptome of the Jerusalem artichoke from the Qinghai Tibet Plateau was conducted by high throughput sequencing technology to obtain important transcriptional and SSR loci information. This allowed characterization of the overall expression features of the Jerusalem artichoke transcriptome, identifying the key genes involved in metabolism in this species. In turn, this offers a foundation for further research on the regulatory mechanisms of fructan metabolism in Jerusalem artichoke.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Characterization of the Tibet plateau Jerusalem artichoke ( Helianthus tuberosus L.) transcriptome by de novo assembly to discover genes associated with fructan synthesis and SSR analysis">
<meta name="dc.source" content="Hereditas 2019 156:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2019-02-06">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2019 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Jerusalem artichoke (Helianthus tuberosus L.) is a characteristic crop in the Qinghai-Tibet Plateau which has rapidly developed and gained socioeconomic importance in recent years. Fructans are abundant in tubers and represent the foundation for their formation, processing and utilization of yield; and are also widely used in new sugar-based materials, bioenergy processing, ecological management, and functional feed. To identify key genes in the metabolic pathway of fructans in Jerusalem artichoke, high-throughput sequencing was performed using Illumina Hi Seq™ 2500 equipment to construct a transcriptome library. Qinghai-Tibet Plateau Jerusalem artichoke “Qingyu No.1” was used as the material; roots, stems, leaves, flowers and tubers of Jerusalem artichoke in its flowering stage were mixed into a mosaic of the Jerusalem artichoke transcriptome library, obtaining 63,089 unigenes with an average length of 713.6 bp. Gene annotation through the Nr, Swiss Prot, GO, KOG and KEGG databases revealed 34.95 and 46.91% of these unigenes had similar sequences in the Nr and Swiss Prot databases. The GO classification showed the Jerusalem artichoke unigenes were divided into three ontologies, with a total of 49 functional groups encompassing biological processes, cellular components, and molecular functions. Among them, there were more unigenes involved in the functional groups for cellular processes, metabolic processes, and single-organism processes. 38,999 unigenes were annotated by KOG and divided into 25 categories according to their functions; the most common annotation being general function prediction. A total of 13,878 unigenes (22%) were annotated in the KEGG database, with the largest proportion corresponding to pathways related to carbohydrate metabolism. A total of 12 unigenes were involved in the synthesis and degradation of fructan. Cluster analysis revealed the candidate 12 unigene proteins were dispersed in the 5 major families of proteins involved in fructan synthesis and degradation. The synergistic effect of INV gene is necessary during fructose synthesis and degradation in Jerusalem artichoke tuber development. The sequencing data from the transcriptome of this species can provide a reliable data basis for the identification and assessment of the expression of the members of the INV gene family.A simple sequence repeat (SSR) loci search was performed on the transcriptome data of Jerusalem artichoke, identifying 6635 eligible SSR loci with a large proportion of dinucleotide and trinucleotide repeats, and the most different motifs were repeated 5 times and 6 times. Dinucleotide and trinucleotide repeat motifs were the most frequent, with AG/CT and ACC/GGT repeat motifs accounting for the highest proportion. In this study, a database search of the transcriptome of the Jerusalem artichoke from the Qinghai Tibet Plateau was conducted by high throughput sequencing technology to obtain important transcriptional and SSR loci information. This allowed characterization of the overall expression features of the Jerusalem artichoke transcriptome, identifying the key genes involved in metabolism in this species. In turn, this offers a foundation for further research on the regulatory mechanisms of fructan metabolism in Jerusalem artichoke.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2019-02-06">
<meta name="prism.volume" content="156">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="9">
<meta name="prism.copyright" content="2019 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0086-8">
<meta name="prism.doi" content="doi:10.1186/s41065-019-0086-8">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-019-0086-8">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0086-8">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Characterization of the Tibet plateau Jerusalem artichoke ( Helianthus tuberosus L.) transcriptome by de novo assembly to discover genes associated with fructan synthesis and SSR analysis">
<meta name="citation_volume" content="156">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/12">
<meta name="citation_online_date" content="2019/02/06">
<meta name="citation_firstpage" content="9">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-019-0086-8">
<meta name="DOI" content="10.1186/s41065-019-0086-8">
<meta name="citation_doi" content="10.1186/s41065-019-0086-8">
<meta name="dc.creator" content="Shipeng Yang">
<meta name="dc.creator" content="Xuemei Sun">
<meta name="dc.creator" content="Xiaoting Jiang">
<meta name="dc.creator" content="Lihui Wang">
<meta name="dc.creator" content="Jie Tian">
<meta name="dc.creator" content="Li Li">
<meta name="dc.creator" content="Mengliang Zhao">
<meta name="dc.creator" content="Qiwen Zhong">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Eur Food Res Technol; citation_title=A simplified spectrophotometric method for the determination of inulin in Jerusalem artichoke (Helianthus tuberosus L.) tubers; citation_author=A Saengkanuk, S Nuchadomrong, S Jogloy, A Patanothai, S Srijaranai; citation_volume=233; citation_publication_date=2011; citation_pages=609; citation_doi=10.1007/s00217-011-1552-3; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=J Ind Microbiol Biotechnol; citation_title=Bioethanol production from the dry powder of Jerusalem artichoke tubers by recombinant Saccharomyces cerevisiae in simultaneous saccharification and fermentation; citation_author=Y-Z Wang, S-M Zou, M-L He, C-H Wang; citation_volume=42; citation_publication_date=2015; citation_pages=543-551; citation_doi=10.1007/s10295-014-1572-7; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=Food Sci Biotechnol; citation_title=Changes in physicochemical characteristics and antioxidant activities of Jerusalem artichoke tea infusions resulting from different production processes; citation_author=Y-J Lee, L M-G, S-Y Yu, W-B Yoon, O-H Lee; citation_volume=23; citation_publication_date=2014; citation_pages=1885-1892; citation_doi=10.1007/s10068-014-0257-3; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=Waste Biomass Valorization; citation_title=Production of fermentation feedstock from Jerusalem artichoke tubers and its potential for Polyhydroxybutyrate synthesis; citation_author=AA Koutinas, IL Garcia, N Kopsahelis, S Papanikolaou, C Webb, MA Villar, JA López; citation_volume=4; citation_publication_date=2013; citation_pages=359-370; citation_doi=10.1007/s12649-012-9154-2; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=J Biol Eng; citation_title=Facing the challenge of sustainable bioenergy production: could halophytes be part of the solution?; citation_author=A Debez, I Belghith, J Friesen, C Montzka, S Elleuche; citation_volume=11; citation_publication_date=2017; citation_pages=27; citation_doi=10.1186/s13036-017-0069-0; citation_id=CR5">
<meta name="citation_reference" content="citation_title=Microbially supported phytoremediation of heavy metal contaminated soils: strategies and applications; citation_inbook_title=Geobiotechnology I: Metal-related Issues; citation_publication_date=2014; citation_pages=211-235; citation_id=CR6; citation_author=R Phieler; citation_author=A Voit; citation_author=E Kothe; citation_publisher=Springer Berlin Heidelberg">
<meta name="citation_reference" content="citation_title=Organisation und Ausgestaltung des Sprosses. — organization and development of the shoot; citation_inbook_title=Differentiation and Development / Differenzierung und Entwicklung: Part 1 / Teil 1; citation_publication_date=1965; citation_pages=966-1255; citation_id=CR7; citation_author=CW Wardlaw; citation_author=CA Reinders-Gouwentak; citation_author=P Champagnat; citation_author=J Doorenbos; citation_author=A Allsopp; citation_publisher=Springer Berlin Heidelberg">
<meta name="citation_reference" content="citation_journal_title=Carbohydr Polym; citation_title=Inulin-type fructans: a review on different aspects of biochemical and pharmaceutical technology; citation_author=AC Apolinário, BPG Lima Damasceno, NE Macêdo Beltrão, A Pessoa, A Converti, JA Silva; citation_volume=101; citation_publication_date=2014; citation_pages=368-378; citation_doi=10.1016/j.carbpol.2013.09.081; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=World J Microbiol Biotechnol; citation_title=High-yield ethanol production from Jerusalem artichoke tubers; citation_author=C Barthomeuf, F Regerat, H Pourrat; citation_volume=7; citation_publication_date=1991; citation_pages=490-493; citation_doi=10.1007/BF00303375; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Acta Physiol Plant; citation_title=Physiological responses of two Jerusalem artichoke cultivars to drought stress induced by polyethylene glycol; citation_author=M Zhang, Q Chen, S Shen; citation_volume=33; citation_publication_date=2011; citation_pages=313-318; citation_doi=10.1007/s11738-010-0549-z; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Planta; citation_title=Wheat stem reserves and salinity tolerance: molecular dissection of fructan biosynthesis and remobilization to grains; citation_author=M Sharbatkhari, Z-S Shobbar, S Galeshi, B Nakhoda; citation_volume=244; citation_publication_date=2016; citation_pages=191-202; citation_doi=10.1007/s00425-016-2497-3; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Nat Rev Gastroenterol Hepatol; citation_title=Inulin regulates endothelial function: a prebiotic smoking gun?; citation_author=F Fava, KM Tuohy; citation_volume=14; citation_publication_date=2017; citation_pages=392; citation_doi=10.1038/nrgastro.2017.68; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Fructans and other water soluble carbohydrates in vegetative organs and fruits of different Musa spp. accessions; citation_author=CI Cruz-Cárdenas, ML Miranda-Ham, LA Castro-Concha, JR Ku-Cauich, R Vergauwen, T Reijnders, W Ende, RM Escobedo-GraciaMedrano; citation_volume=6; citation_publication_date=2015; citation_pages=395; citation_doi=10.3389/fpls.2015.00395; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Advanced Pharmaceutical Bulletin; citation_title=A combination of prebiotic inulin and Oligofructose improve some of cardiovascular disease risk factors in women with type 2 diabetes: a randomized controlled clinical trial; citation_author=A Aliasgharzadeh, M Khalili, E Mirtaheri, B Pourghassem Gargari, F Tavakoli, M Abbasalizad Farhangi, H Babaei, P Dehghan; citation_volume=5; citation_publication_date=2015; citation_pages=507-514; citation_doi=10.15171/apb.2015.069; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Curr Opin Plant Biol; citation_title=Fructans: beneficial for plants and humans; citation_author=T Ritsema, S Smeekens; citation_volume=6; citation_publication_date=2003; citation_pages=223; citation_doi=10.1016/S1369-5266(03)00034-7; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Fructan: more than a reserve carbohydrate?; citation_author=I Vijn, S Smeekens; citation_volume=120; citation_publication_date=1999; citation_pages=351-360; citation_doi=10.1104/pp.120.2.351; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=The mechanisim of fructosan metabolism in higher plants as exemplified in helianthus tuberosus; citation_author=J Edelman, TG Jefford; citation_volume=67; citation_publication_date=1968; citation_pages=517-531; citation_doi=10.1111/j.1469-8137.1968.tb05480.x; citation_id=CR17">
<meta name="citation_reference" content="Higashiya H, Kobayashi K, Terada S. Fructan as a novel effective factor for mammalian cell. Culture. 2010:679–82.">
<meta name="citation_reference" content="citation_journal_title=Carbohydr Res; citation_title=Purification and characterisation of 1F-fructosyltransferase from the roots of asparagus (asparagus officinalis L.); citation_author=N Shiomi; citation_volume=99; citation_publication_date=1982; citation_pages=157-169; citation_doi=10.1016/S0008-6215(00)81905-9; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Chem Cent J; citation_title=Cloning and functional characterization of a fructan 1-exohydrolase (1-FEH) in edible burdock (Arctium lappa L.); citation_author=K Ueno, Y Ishiguro, M Yoshida, S Onodera, N Shiomi; citation_volume=5; citation_publication_date=2011; citation_pages=16; citation_doi=10.1186/1752-153X-5-16; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=Shedding light on an extremophile lifestyle through transcriptomics; citation_author=M Dassanayake, JS Haas, HJ Bohnert, JM Cheeseman; citation_volume=183; citation_publication_date=2009; citation_pages=764-775; citation_doi=10.1111/j.1469-8137.2009.02913.x; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Function annotation of the rice transcriptome at single-nucleotide resolution by RNA-seq; citation_author=T Lu, G Lu, D Fan, C Zhu, W Li, Q Zhao, Q Feng, Y Zhao, Y Guo, W Li; citation_volume=20; citation_publication_date=2010; citation_pages=1238-1249; citation_doi=10.1101/gr.106120.110; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Gene; citation_title=De novo sequencing and assembly of Centella asiatica leaf transcriptome for mapping of structural, functional and regulatory genes with special reference to secondary metabolism; citation_author=RS Sangwan, S Tripathi, J Singh, LK Narnoliya, NS Sangwan; citation_volume=525; citation_publication_date=2013; citation_pages=58-76; citation_doi=10.1016/j.gene.2013.04.057; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=Genome skimming reveals the origin of the Jerusalem artichoke tuber crop species: neither from Jerusalem nor an artichoke; citation_author=DG Bock, NC Kane, DP Ebert, LH Rieseberg; citation_volume=201; citation_publication_date=2014; citation_pages=1021-1030; citation_doi=10.1111/nph.12560; citation_id=CR24">
<meta name="citation_reference" content="citation_journal_title=Gene; citation_title=The okra (Abelmoschus esculentus) transcriptome as a source for gene sequence information and molecular markers for diversity analysis; citation_author=R Schafleitner, S Kumar, L C-y, SG Hegde, A Ebert; citation_volume=517; citation_publication_date=2013; citation_pages=27-36; citation_doi=10.1016/j.gene.2012.12.098; citation_id=CR25">
<meta name="citation_reference" content="citation_journal_title=Biol Plant; citation_title=De novo transcriptome analysis of Fraxinus velutina using Illumina platform and development of EST-SSR markers; citation_author=LP Yan, CL Liu, DJ Wu, L Li, J Shu, C Sun, Y Xia, LJ Zhao; citation_volume=61; citation_publication_date=2016; citation_pages=210-218; citation_doi=10.1007/s10535-016-0681-8; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=Russ J Plant Physiol; citation_title=De novo assembly of mulberry (Morus alba L.) transcriptome and identification of candidate unigenes related to salt stress responses; citation_author=CY Liu, XQ Liu, DP Long, BN Cao, ZH Xiang, AC Zhao; citation_volume=64; citation_publication_date=2017; citation_pages=738-748; citation_doi=10.1134/S1021443717050053; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Mar Biotechnol; citation_title=Identification of genes potentially related to biomineralization and immunity by transcriptome analysis of pearl sac in pearl oyster Pinctada martensii; citation_author=X Zhao, Q Wang, Y Jiao, R Huang, Y Deng, H Wang, X Du; citation_volume=14; citation_publication_date=2012; citation_pages=730-739; citation_doi=10.1007/s10126-012-9438-3; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Planta; citation_title=Fructan:fructan 1-fructosyltransferase, a key enzyme for biosynthesis of graminan oligomers in hardened wheat; citation_author=A Kawakami, M Yoshida; citation_volume=223; citation_publication_date=2005; citation_pages=90-104; citation_doi=10.1007/s00425-005-0054-6; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Development of rubber-enriched dandelion varieties by metabolic engineering of the inulin pathway; citation_author=A Stolze, A Wanke, N Deenen, R Geyer, D Prufer, C Schulze Gronover; citation_volume=15; citation_publication_date=2017; citation_pages=740-753; citation_doi=10.1111/pbi.12672; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=Cloning, characterization and functional analysis of novel 6-kestose exohydrolases (6-KEHs) from wheat (Triticum aestivum); citation_author=VE Wim, Y Midori, C Stefan, V Rudy, K Akira; citation_volume=166; citation_publication_date=2005; citation_pages=917-932; citation_doi=10.1111/j.1469-8137.2005.01394.x; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=Cloning and functional characterization of two abiotic stress-responsive Jerusalem artichoke (Helianthus tuberosus) fructan 1-exohydrolases (1-FEHs); citation_author=H Xu, M Liang, L Xu, H Li, X Zhang, J Kang, Q Zhao, H Zhao; citation_volume=87; citation_publication_date=2015; citation_pages=81-98; citation_doi=10.1007/s11103-014-0262-1; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Cloning of the fructan biosynthesis pathway of Jerusalem artichoke; citation_author=IM Meer, AJ Koops, JC Hakkert, AJ Tunen; citation_volume=15; citation_publication_date=1998; citation_pages=489-500; citation_doi=10.1046/j.1365-313X.1998.00230.x; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol Rep; citation_title=Comparative transcriptome profiling and SSR marker identification in three Jerusalem artichoke (Helianthus tuberosus L.) cultivars exhibiting phenotypic variation; citation_author=WY Jung, SS Lee, HJ Park, CW Kim, S-Y Kwon, J-H Jeon, H-S Kim, HS Cho; citation_volume=10; citation_publication_date=2016; citation_pages=447-461; citation_doi=10.1007/s11816-016-0421-8; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Development of genic-SSR markers by deep transcriptome sequencing in pigeonpea [Cajanus cajan (L.) Millspaugh]; citation_author=S Dutta, G Kumawat, BP Singh, DK Gupta, S Singh, V Dogra, K Gaikwad, TR Sharma, RS Raje, TK Bandhopadhya; citation_volume=11; citation_publication_date=2011; citation_pages=17; citation_doi=10.1186/1471-2229-11-17; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=Plant Mol Biol Report; citation_title=De novo characterization of leaf transcriptome using 454 sequencing and development of EST-SSR markers in tea (Camellia sinensis); citation_author=H Wu, D Chen, J Li, B Yu, X Qiao, H Huang, Y He; citation_volume=31; citation_publication_date=2013; citation_pages=524-538; citation_doi=10.1007/s11105-012-0519-2; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Utility of EST-derived SSR in cultivated peanut (Arachis hypogaea L.) and Arachiswild species; citation_author=X Liang, X Chen, Y Hong, H Liu, G Zhou, S Li, B Guo; citation_volume=9; citation_publication_date=2009; citation_pages=35; citation_doi=10.1186/1471-2229-9-35; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Development, characterization and cross-species/genera transferability of EST-SSR markers for rubber tree (Hevea brasiliensis); citation_author=SP Feng, WG Li, HS Huang, JY Wang, YT Wu; citation_volume=23; citation_publication_date=2009; citation_pages=85-97; citation_doi=10.1007/s11032-008-9216-0; citation_id=CR38">
<meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Mining and validating grape (Vitis L.) ESTs to develop EST-SSR markers for genotyping and mapping; citation_author=H Huang, J Lu, Z Ren, W Hunter, SE Dowd, P Dang; citation_volume=28; citation_publication_date=2011; citation_pages=241-254; citation_doi=10.1007/s11032-010-9477-2; citation_id=CR39">
<meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes; citation_author=M Morgante, M Hanafey, W Powell; citation_volume=30; citation_publication_date=2002; citation_pages=194; citation_doi=10.1038/ng822; citation_id=CR40">
<meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome; citation_author=MG Grabherr, BJ Haas, M Yassour, JZ Levin, DA Thompson, I Amit, X Adiconis, L Fan, R Raychowdhury, Q Zeng; citation_volume=29; citation_publication_date=2011; citation_pages=644; citation_doi=10.1038/nbt.1883; citation_id=CR41">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=TIGR gene indices clustering tools (TGICL): a software system for fast clustering of large EST datasets; citation_author=G Pertea, X Huang, F Liang, V Antonescu, R Sultana, S Karamycheva, Y Lee, J White, F Cheung, B Parvizi; citation_volume=19; citation_publication_date=2003; citation_pages=651-652; citation_doi=10.1093/bioinformatics/btg034; citation_id=CR42">
<meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Base-calling of automated sequencer traces UsingPhred. I. Accuracy assessment; citation_author=B Ewing, LD Hillier, MC Wendl, P Green, B Ewing, LD Hillier, MC Wendl, P Green; citation_volume=8; citation_issue=3; citation_publication_date=1998; citation_pages=175-185; citation_doi=10.1101/gr.8.3.175; citation_id=CR43">
<meta name="citation_reference" content="citation_journal_title=Methods; citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2−ΔΔCT method; citation_author=KJ Livak, TD Schmittgen; citation_volume=25; citation_publication_date=2001; citation_pages=402-408; citation_doi=10.1006/meth.2001.1262; citation_id=CR44">
<meta name="citation_author" content="Shipeng Yang">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author" content="Xuemei Sun">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author" content="Xiaoting Jiang">
<meta name="citation_author_institution" content="Qinghai Higher Vocational &amp; Technical College Institute, Ledu, China">
<meta name="citation_author" content="Lihui Wang">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author" content="Jie Tian">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author" content="Li Li">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author" content="Mengliang Zhao">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta name="citation_author" content="Qiwen Zhong">
<meta name="citation_author_institution" content="Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-019-0086-8',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-019-0086-8","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Helianthus tuberosus L.;Transcriptome;Gene analysis;Simple sequence repeats"},"contentInfo":{"imprint":"BioMed Central","title":"Characterization of the Tibet plateau Jerusalem artichoke ( Helianthus tuberosus L.) transcriptome by de novo assembly to discover genes associated with fructan synthesis and SSR analysis","publishedAt":1549411200000,"publishedAtDate":"2019-02-06","author":["Shipeng Yang","Xuemei Sun","Xiaoting Jiang","Lihui Wang","Jie Tian","Li Li","Mengliang Zhao","Qiwen Zhong"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"156","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0086-8">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-019-0086-8">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Characterization of the Tibet plateau Jerusalem artichoke ( Helianthus tuberosus L.) transcriptome by de novo assembly to discover genes associated with fructan synthesis and SSR analysis">
<meta property="og:description" content="Jerusalem artichoke (Helianthus tuberosus L.) is a characteristic crop in the Qinghai-Tibet Plateau which has rapidly developed and gained socioeconomic importance in recent years. Fructans are abundant in tubers and represent the foundation for their formation, processing and utilization of yield; and are also widely used in new sugar-based materials, bioenergy processing, ecological management, and functional feed. To identify key genes in the metabolic pathway of fructans in Jerusalem artichoke, high-throughput sequencing was performed using Illumina Hi Seqâ¢ 2500 equipment to construct a transcriptome library. Qinghai-Tibet Plateau Jerusalem artichoke âQingyu No.1â was used as the material; roots, stems, leaves, flowers and tubers of Jerusalem artichoke in its flowering stage were mixed into a mosaic of the Jerusalem artichoke transcriptome library, obtaining 63,089 unigenes with an average length of 713.6âbp. Gene annotation through the Nr, Swiss Prot, GO, KOG and KEGG databases revealed 34.95 and 46.91% of these unigenes had similar sequences in the Nr and Swiss Prot databases. The GO classification showed the Jerusalem artichoke unigenes were divided into three ontologies, with a total of 49 functional groups encompassing biological processes, cellular components, and molecular functions. Among them, there were more unigenes involved in the functional groups for cellular processes, metabolic processes, and single-organism processes. 38,999 unigenes were annotated by KOG and divided into 25 categories according to their functions; the most common annotation being general function prediction. A total of 13,878 unigenes (22%) were annotated in the KEGG database, with the largest proportion corresponding to pathways related to carbohydrate metabolism. A total of 12 unigenes were involved in the synthesis and degradation of fructan. Cluster analysis revealed the candidate 12 unigene proteins were dispersed in the 5 major families of proteins involved in fructan synthesis and degradation. The synergistic effect of INV gene is necessary during fructose synthesis and degradation in Jerusalem artichoke tuber development. The sequencing data from the transcriptome of this species can provide a reliable data basis for the identification and assessment of the expression of the members of the INV gene family.A simple sequence repeat (SSR) loci search was performed on the transcriptome data of Jerusalem artichoke, identifying 6635 eligible SSR loci with a large proportion of dinucleotide and trinucleotide repeats, and the most different motifs were repeated 5 times and 6 times. Dinucleotide and trinucleotide repeat motifs were the most frequent, with AG/CT and ACC/GGT repeat motifs accounting for the highest proportion. In this study, a database search of the transcriptome of the Jerusalem artichoke from the Qinghai Tibet Plateau was conducted by high throughput sequencing technology to obtain important transcriptional and SSR loci information. This allowed characterization of the overall expression features of the Jerusalem artichoke transcriptome, identifying the key genes involved in metabolism in this species. In turn, this offers a foundation for further research on the regulatory mechanisms of fructan metabolism in Jerusalem artichoke.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-019-0086-8;kwrd=Helianthus tuberosus L.,Transcriptome,Gene analysis,Simple sequence repeats;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-019-0086-8&amp;kwrd=Helianthus%20tuberosus%20L.,Transcriptome,Gene%20analysis,Simple%20sequence%20repeats&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-019-0086-8&amp;kwrd=Helianthus%20tuberosus%20L.,Transcriptome,Gene%20analysis,Simple%20sequence%20repeats&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Materials and methods">Materials and methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Characterization of the Tibet plateau Jerusalem artichoke (<em xmlns="" class="EmphasisTypeItalic">Helianthus tuberosus</em> L.) transcriptome by de novo assembly to discover genes associated with fructan synthesis and SSR analysis</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true">
<ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Shipeng Yang</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xuemei Sun</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaoting Jiang</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lihui Wang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jie Tian</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Li Li</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Mengliang Zhao</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qiwen Zhong</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:jerusalemyys@aliyun.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>
</li>
</ul>
<div class="AffiliationLegend"><div class="AffiliationLegendLine">
<span class="EqualContributionSymbol">â </span>Contributed equally</div></div>
</div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2019</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">156</strong>:9</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-019-0086-8">https://doi.org/10.1186/s41065-019-0086-8</a></p>
<p class="Copyright">Â©  The Author(s) 2019</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>23 August 2018</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>27 January 2019</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">6 February 2019</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Jerusalem artichoke (<em xmlns="" class="EmphasisTypeItalic">Helianthus tuberosus</em> L.) is a characteristic crop in the Qinghai-Tibet Plateau which has rapidly developed and gained socioeconomic importance in recent years. Fructans are abundant in tubers and represent the foundation for their formation, processing and utilization of yield; and are also widely used in new sugar-based materials, bioenergy processing, ecological management, and functional feed. To identify key genes in the metabolic pathway of fructans in Jerusalem artichoke, high-throughput sequencing was performed using Illumina Hi Seqâ¢ 2500 equipment to construct a transcriptome library.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Qinghai-Tibet Plateau Jerusalem artichoke âQingyu No.1â was used as the material; roots, stems, leaves, flowers and tubers of Jerusalem artichoke in its flowering stage were mixed into a mosaic of the Jerusalem artichoke transcriptome library, obtaining 63,089 unigenes with an average length of 713.6âbp. Gene annotation through the Nr, Swiss Prot, GO, KOG and KEGG databases revealed 34.95 and 46.91% of these unigenes had similar sequences in the Nr and Swiss Prot databases. The GO classification showed the Jerusalem artichoke unigenes were divided into three ontologies, with a total of 49 functional groups encompassing biological processes, cellular components, and molecular functions. Among them, there were more unigenes involved in the functional groups for cellular processes, metabolic processes, and single-organism processes. 38,999 unigenes were annotated by KOG and divided into 25 categories according to their functions; the most common annotation being general function prediction. A total of 13,878 unigenes (22%) were annotated in the KEGG database, with the largest proportion corresponding to pathways related to carbohydrate metabolism. A total of 12 unigenes were involved in the synthesis and degradation of fructan. Cluster analysis revealed the candidate 12 unigene proteins were dispersed in the 5 major families of proteins involved in fructan synthesis and degradation. The synergistic effect of INV gene is necessary during fructose synthesis and degradation in Jerusalem artichoke tuber development. The sequencing data from the transcriptome of this species can provide a reliable data basis for the identification and assessment of the expression of the members of the INV gene family.A simple sequence repeat (SSR) loci search was performed on the transcriptome data of Jerusalem artichoke, identifying 6635 eligible SSR loci with a large proportion of dinucleotide and trinucleotide repeats, and the most different motifs were repeated 5 times and 6 times. Dinucleotide and trinucleotide repeat motifs were the most frequent, with AG/CT and ACC/GGT repeat motifs accounting for the highest proportion.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">In this study, a database search of the transcriptome of the Jerusalem artichoke from the Qinghai Tibet Plateau was conducted by high throughput sequencing technology to obtain important transcriptional and SSR loci information. This allowed characterization of the overall expression features of the Jerusalem artichoke transcriptome, identifying the key genes involved in metabolism in this species. In turn, this offers a foundation for further research on the regulatory mechanisms of fructan metabolism in Jerusalem artichoke.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">
                        <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Helianthus tuberosus</em> L.</li>
<li class="c-keywords__item" itemprop="about">Transcriptome</li>
<li class="c-keywords__item" itemprop="about">Gene analysis</li>
<li class="c-keywords__item" itemprop="about">Simple sequence repeats</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Jerusalem artichoke (<em xmlns="" class="EmphasisTypeItalic">Helianthus tuberosus</em> L.) belongs to the genus <em xmlns="" class="EmphasisTypeItalic">Helianthus L</em>. of Asteraceae. Although it is native to temperate zones in North America, it was introduced into China over 300âyears ago, where it has been planted sporadically as a pickled vegetable for a long time. The twenty-first Century has seen a gradual decrease of the economic value of Jerusalem artichoke, with a rapid expansion in its planting and processing. The Jerusalem artichoke tuber contains fructose, which can be used to process inulin [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], bioethanol [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>] and various foods [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. In addition, because of its strong stress resistance, it can be cultivated in dry and saline alkaline environments, and has also been utilized in the ecological treatment of desert soil [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], seashore saline soil [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] and heavy metal-contaminated soil [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. The Qinghai-Tibet Plateau is one of the earliest areas in China to conduct large-scale planting and processing of Jerusalem artichoke. The climate is cold, with large differences in temperature between day and night. The yield of the Jerusalem artichoke tuber is high, due its elevated fructan content. Moreover, it has natural and pollution-free environmental advantages, being very beneficial to the processing of organic fructan products. The industry of Jerusalem artichoke planting and processing has achieved rich development potential in the Qinghai-Tibet Plateau.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Fructose is the third largest storage carbohydrate after starch and sucrose. About 15% of angiosperms contain fructan, most prominently in the Compositae, Liliaceae and Gramineae families; which mostly grow in cold temperate zones [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Plant fructan has been studied for over 200âyears, with early research focused on the biochemical properties of fructans [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. In recent years, abundant studies have shown fructan metabolism is closely related to plant resistance to cold [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>], drought [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] and saline alkali [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], as well as other stressors. With the discovery of the enzymes involved in fructan metabolism, research focus has shifted to the understanding of the metabolic processes and functions of fructosan [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. In addition to the important role of fructan in plants, processed food derived from this carbohydrate also helps maintain human health. The properties and processing techniques of fructan have been extensively studied [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Fructan is an optimal nutrient and sweetener for patients with diabetes and hypertension, as it has a caloric value of only 1âkcal/g, and it is not easily digested and decomposed into monosaccharides and therefore does not sharply increase blood glucose [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Highly polymerized fructans are often used as substitutes for fats in foods such as yogurt, ice cream and jelly pudding [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Fructan structure is mainly linear or branched in plants and 5 different types of derivatives. The earliest model of fructan metabolism was proposed by research on Jerusalem artichoke [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Jerusalem artichoke is a typical crop that accumulates inulin-type linear fructan. This type is composed of D-fructose residues linked by Î²-2, 1 bonds with molecular weights ranging from 3 to 5âkDa [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. Current models of inulin-type fructan metabolism suggest its synthesis is based on sucrose as a substrate: First, the synthesis of kestose (1-Kestose) is catalyzed by sucrose 1-fructosyltransferase (1-SST, EC 2.4.1.99), followed by elongation of the fructan chain [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] catalyzed by fructan 1-fructosyltransferase (1-FFT, EC 2.4.1.100). Then, the degradation of fructan is catalyzed by 1-Fructan exohydrolase (1-FEH, EC 3.2.1.80), with the fructosyl being cleaved one by one from the end of the sugar chain. At present, the functions of known key enzymes in the metabolism of Jerusalem artichoke fructan are unclear, and which genes are involved in their metabolic regulation remains to be discovered.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">In recent years, high-throughput transcriptome sequencing has been widely used to study gene expression in different biological problems. This method can comprehensively and quickly acquire all mRNA sequences [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>â<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] generated by gene expression in a certain state and identify important functional genes from it in order to reveal the molecular mechanisms underlying different biological traits. In this study, the transcriptome of Jerusalem artichoke from the Qinghai Tibet Plateau was sequenced using high-throughput sequencing technology with Illumina Hi Seq 2500 equipment. Functional annotation of genes, and functional classification and metabolic pathway analysis of unigene were carried out with bioinformatics methods to lay a foundation for further research on the key metabolic genes in Jerusalem artichoke, as well as for interpretation of the molecular mechanisms involved in the regulation of fructan metabolism, and the development of related molecular markers.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Analysis of RNA-Seq datasets</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Illumina sequencing obtained a total of 123,169,632 reads, containing 18.68âG of nucleotide sequence information. The percentages of Q20 (sequencing error rate less than 1%), Q30 (sequencing error rate less than 0.1%), and GC content were 98.28, 95.40, and 44.92%, respectively (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). The data from the transcriptome sequencing had high volume and quality, providing good raw data for subsequent data assembly. Trinity was assembled to obtain 63,098 spliced transcript sequences, with an average length of 713ânt and a N50 of 1106ânt. The longest transcript in each gene was taken as an unigene, with an average length of 713.6âbp. The length distribution of the unigenes revealed a total of 12,962 unigenes with a length greater than 1000ânt, accounting for 20.55% of all unigenes. Among them, 34,765 unigenes had a length of 200â499ânt, accounting for 55.10% of the total; whereas 14,004 unigenes had a length of 500â999ânt, corresponding to 22.20% of the total; and 14,320 unigenes had a length of â¥1000ânt, accounting for 22.70% of the total (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). This indicated the transcriptome library had good sequencing and assembly results, and adequate for subsequent bioinformatics analysis.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Transcriptome Sequence data quality for the Jerusalem artichoke</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Raw reads</p></th>
<th><p xmlns="" class="SimplePara">Clean reads</p></th>
<th><p xmlns="" class="SimplePara">Clean bases</p></th>
<th><p xmlns="" class="SimplePara">Q20(%)<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></th>
<th><p xmlns="" class="SimplePara">Q30(%)<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></th>
<th><p xmlns="" class="SimplePara">GC(%)</p></th>
</tr></thead>
<tbody><tr class="noclass">
<td><p xmlns="" class="SimplePara">124,500,360</p></td>
<td char="."><p xmlns="" class="SimplePara">123,169,632</p></td>
<td><p xmlns="" class="SimplePara">18.68G</p></td>
<td char="."><p xmlns="" class="SimplePara">98.28%</p></td>
<td char="."><p xmlns="" class="SimplePara">95.40%</p></td>
<td char="."><p xmlns="" class="SimplePara">44.92%</p></td>
</tr></tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Q20: percentage is the proportion of nucleotides with a quality &gt;â20 in reads</p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Q30: percentage is the proportion of nucleotides with a quality &gt;â30 in reads</p>
                    </div></figure><figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig1_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig1_HTML.png" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Size range of unigenes from transcriptome data of Jerusalem artichoke</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">Functional annotation and classification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The assembled unigene sequences were separately aligned according to the Nr and Swissprot databases by Blast software. Results recognized 22,048 unigenes aligned to similar sequences in the Nr database, with E values ranging from 1E-50 to 1E-20, with a total of 9873 (44.78%). To determine the species to which homologous sequences belong, the sequence in which each unigene yielded the best results (the lowest E value) in the Nr library was selected (if there was a parallel, the first one was selected). The number of homologous sequences statistically aligned to each species was determined in the Nr database. The highest matches with the unigenes from Jerusalem artichoke were <em xmlns="" class="EmphasisTypeItalic">Cynara cardunculus</em> (19002), followed by <em xmlns="" class="EmphasisTypeItalic">Daucus carota</em> (1346), <em xmlns="" class="EmphasisTypeItalic">Helianthus annuus</em> (1126), <em xmlns="" class="EmphasisTypeItalic">Vitis vinifera</em> (1042), <em xmlns="" class="EmphasisTypeItalic">Theobroma cacao</em> (934), <em xmlns="" class="EmphasisTypeItalic">Sesamum indicum</em> (810), and others. 29,595 (46.91%) unigenes had similar sequences in the Swiss Prot database, where the E values of most ranged between 1E-50 to 1E-20, corresponding to 8135 unigenes (27.49%). Compared with Nr, SwissProt function annotation found a large reduction in unigenea with high similarity sequences, consistent with database characteristics. The SwissProt data source is very strict, and all sequence entries are carefully checked by experienced molecular biologists and protein chemists through computer tools and related literature. In contrast, the Nr database integration standard is looser, and any two sequences any differences are regarded as two distinct records.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">The GO database was used to classify the unigenes according to biologic function. Results showed the 63,089 Jerusalem artichoke unigenes were divided into three ontologies (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>): Biological processes, Cellular components, and Molecular functions, with a total of 49 functional groups. Further analysis showed 46,360 GO entries belonged to 19 functional groups related to biological processes; 24,082 corresponded to 17 functional groups linked to cellular components; and 374,83 GO entries belonged to 13 functional groups related to molecular functions. Most unigenes were allocated to the functional groups of cellular processes (10,358 unigenes), metabolic processes (11,147 unigenes), the single-organism processes (8223 unigenes), the cell (8716 unigenes), cell parts (8716 unigenes), binding (10,443); while the lowest amount of unigenes were involved in the functional groups of cell killing (3 unigenes), locomotion (4 unigenes), metallochaperone activity (1 unigenes), translation regulator activity (1 unigenes), nucleoid (1 unigenes), extracellular matrix components (4 unigenes), extracellular region parts (5 unigenes) and the extracellular matrix (6 unigenes).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig2_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig2_HTML.png" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Gene ontology (GO) classification of assembled unigenes in Jerusalem artichoke</p>
                      </div></figcaption></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Jerusalem artichoke unigenes were compared with the KOG database and classified according to their function. 38,999 unigenes were divided into 25 categories, with the most annotated unigenes corresponding to General function prediction (7016 unigenes), followed by those related to posttranslational modification, protein turnover and chaperones (4507 unigenes), and signal transduction mechanisms (4740 unigenes). Meanwhile, the groups with the least annotated unigenes were those relate to nuclear structure (147 unigenes), extracellular structures (114 unigenes), and cell motility (21 unigenes). The identified unigenes of Jerusalem artichoke were involved in most biological activities typical of growth and development (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig3_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig3_HTML.png" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig3_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig3_HTML.png" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Clusters of orthologous groups for eukaryotic complete genomes (KOG) classification of assembled unigenes</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Functional classification by the KEGG pathway</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Jerusalem artichoke unigenes were compared to the KEGG database, and the metabolic pathways were analyzed based on the annotation information to understand the metabolic pathways and functions of their products in cells. A total of 13,878 unigenes (22.00%) were annotated for analysis of metabolic pathways. Among the 7 most statistically significant pathways, carbohydrate metabolism-related pathways accounted for the largest proportion, followed by translation, global and overview, folding, sorting and degradation, amino acid metabolism, transport and catabolism, and energy metabolism. Most unigenes corresponded to metabolic pathways, indicating Jerusalem artichoke had strong metabolism during this period. On the other hand, more than 400 other unigenes were linked to other subclasses related to ribosomes, carbon metabolism, biosynthesis of amino acids, protein processing in the endoplasmic reticulum, and plant hormone signal transduction pathways (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig4_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig4_HTML.png" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig4_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig4_HTML.png" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">KEGG analysis of Jerusalem artichoke unigenes for metabolic pathways. A: Carbohydrate metabolism. B: Translation. C: Global and Overview. D: Folding, sorting and degradation. E: Amino acid metabolism. F: Transport and catabolism. G: Energy metabolism</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Phylogenetic analysis of fructan synthesis and degradation unigenes</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Results indicated 12 unigenes were involved in the synthesis and degradation of fructan (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The protein phylogenetic tree of Jerusalem artichoke and other fructan crops was constructed by MEGA 5.0 software. According to the topology of the phylogenetic tree (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>), fructan synthesis and degrading enzyme gene proteins were clustered and analyzed into three categories: nigenes, fructosyltransferase gene and fructosyl synthase gene, obtained by sequencing. The 12 candidate unigenesâ proteins were dispersed in five major protein families of fructan synthesis and degradation enzymes; especially in the fructosyltransferase gene family, accounting for 91.67%. This may be because the fructosyltransferase gene is similar in structure to the vacuolar invertase and cell wall invertase genes, displaying the widest distribution in the phylogenetic tree. The fructosyltransferase gene were clustered with Unigene37416, Unigene49657, Unigene7092 and Unigene23036. These fructosyltransferase genes awere basically INV, CW-INV and Î²-fructofuranosidase. However, Unigene32177 was the only candidate unigene that was combined with the fructan synthase protein. In addition, Unigene37416 and Unigene49657 were on the same evolutionary branch as <em xmlns="" class="EmphasisTypeItalic">Helianthus annuus</em> and <em xmlns="" class="EmphasisTypeItalic">Cichorium intybus</em>, respectively. Unigene36264, Unigene46471, Unigene20477, and Unigene21582 were adjacent and on the same sub-branches, and Unigene11775, Unigene45359, and Unigene39046 shared the same distribution, suggesting they may have similar functions.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Information of candidate 12 fructan pathways genes in the transcriptome of Jerusalem artichoke</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col>
<col>
<col>
</colgroup>
<thead>
<tr class="header">
<th rowspan="2"><p xmlns="" class="SimplePara">Gene ID</p></th>
<th rowspan="2" char="."><p xmlns="" class="SimplePara">Length(bp)</p></th>
<th colspan="3"><p xmlns="" class="SimplePara">Blast result</p></th>
</tr>
<tr class="header">
<th><p xmlns="" class="SimplePara">Species</p></th>
<th><p xmlns="" class="SimplePara">E value</p></th>
<th><p xmlns="" class="SimplePara">Annotation</p></th>
</tr>
</thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0007092</p></td>
<td char="."><p xmlns="" class="SimplePara">1084</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Solanum tuberosum</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">5.00E-162</p></td>
<td><p xmlns="" class="SimplePara">Apoplastic invertase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0011775</p></td>
<td char="."><p xmlns="" class="SimplePara">1982</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Solanum tuberosum</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">2.00E-60</p></td>
<td><p xmlns="" class="SimplePara">Vacuolar invertase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0020477</p></td>
<td char="."><p xmlns="" class="SimplePara">862</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Daucus carota</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">1.00E-123</p></td>
<td><p xmlns="" class="SimplePara">Cell wall beta-fructosidase(Inv1)</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0021582</p></td>
<td char="."><p xmlns="" class="SimplePara">445</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Rhus chinensis</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">4.00E-62</p></td>
<td><p xmlns="" class="SimplePara">Beta-fructofuranosidase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0023036</p></td>
<td char="."><p xmlns="" class="SimplePara">726</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Populus grandidentata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">9.00E-116</p></td>
<td><p xmlns="" class="SimplePara">Cell-wall invertase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0032177</p></td>
<td char="."><p xmlns="" class="SimplePara">2094</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Cichorium intybus</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">3.00E-32</p></td>
<td><p xmlns="" class="SimplePara">Sucrose:sucrose 1-fructosyltransferase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0036264</p></td>
<td char="."><p xmlns="" class="SimplePara">1987</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Cichorium intybus</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">1.00E-10</p></td>
<td><p xmlns="" class="SimplePara">Beta-fructofuranosidase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0037416</p></td>
<td char="."><p xmlns="" class="SimplePara">1838</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Helianthus annuus</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">6.00E-36</p></td>
<td><p xmlns="" class="SimplePara">Cell wall invertase (Cw-inv1)</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0039046</p></td>
<td char="."><p xmlns="" class="SimplePara">2047</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Cynara cardunculus</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">1.00E-126</p></td>
<td><p xmlns="" class="SimplePara">Cell wall beta-fructosidase(Inv2)</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0045359</p></td>
<td char="."><p xmlns="" class="SimplePara">2175</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Lycium barbarum</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">2.00E-25</p></td>
<td><p xmlns="" class="SimplePara">Acid invertase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0046471</p></td>
<td char="."><p xmlns="" class="SimplePara">1841</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Cichorium intybus</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">2.00E-11</p></td>
<td><p xmlns="" class="SimplePara">Invertase</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Unigene0049657</p></td>
<td char="."><p xmlns="" class="SimplePara">1709</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Manihot esculenta</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">4.00E-62</p></td>
<td><p xmlns="" class="SimplePara">Cell wall invertase</p></td>
</tr>
</tbody>
</table></div></figure><figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig5_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig5_HTML.png" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig5_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig5_HTML.png" alt="Fig. 5"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Phylogenetic analysis of 32 families of glycosyl hydrolase from Jerusalem artichoke and different species</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Candidate genes involved in fructan metabolism and expression analysis of 12 candidate genes by quantitative polymerase chain reaction</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Assessment with quantitative polymerase chain reaction (qPCR) was performed in Jerusalem artichoke samples at the tuber formation stage to evaluate the expression of 12 unigenes related to the fructan metabolism. Changes in the expression of Cell wall-INV related Unigene23036 and Unigene39046 were similar Jerusalem artichoke had increased height and tubers had matured. The highest value was reached in the third stage and then gradually decreased, showing fluctuation (Fig. <span class="InternalRef"><a href="#Fig6">6</a></span>). The decomposition of sucrose is mainly catalyzed by INV in higher plants. INV is a key enzyme gene regulating sucrose degradation in Jerusalem artichoke, and can hydrolyze sucrose to glucose and fructose, which are the substrates for fructan synthesis. Interestingly, the INV-related Unigene36264, Unigene45359, Unigene20477, Unigene37416 and Unigene07092 were expressed at higher levels in the early stage of tuber development. With gradual tuber expansion, expression gradually decreased, with a slight drop after the initial second stage of tuber expansion, and remaining relatively stable in the late stage of tuber development. This difference in expression patterns indicates each type of INV may have different biological functions. In addition, Unigene21582, Unigene11775, Unigene49657 and Unigene32177 showed up-regulated expression throughout the whole development and expansion process of the Jerusalem artichoke tuber. The expression level continued to rise, especially from September 1st to the harvest season,. These unigenes may be associated with fructan accumulation in the late stage of tuber development of this species. The distinct expression characteristics of these INV-related unigenes indicate there is functional differentiation during the evolution process. In general, these unigenes are highly expressed in the tuber development stage; suggesting they may play an important role in the synthesis and degradation of fructan. However, further research is required to elucidate specific molecular mechanisms.<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig6_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig6_HTML.png" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig6_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig6_HTML.png" alt="Fig. 6"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Quantitative analysis of gene expression during the formation of Jerusalem artichoke tubers: 1(Date: 7.15). 2(Date: 8.1). 3(Date: 8.15). 4(Date: 9.1). 5(Date: 9.15)</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Expressed sequence tag-simple sequence repeat discovery: distribution and frequencies</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">A total of 63,089 unigenes (with a total length of about 45,020,635âbp) were searched in the transcriptome of Jerusalem artichoke with MISA software, identifying 6635 eligible SSR loci in 5452 unigenes, with a 10.52% occurrence frequency (SSR locus/total unigene number) and 8.64% incidence frequency (the number of unigenes containing the SSR loci/the total number of unigenes). Among these unigenes containing SSR loci, 4521 (82.92%) contained 1 SSR locus, whereas the remaining 931 (17.08%) contained â¥2 SSR loci. Regarding distribution frequency, the ratio of dinucleotides and trinucleotides was relatively large, accounting for 45.24 and 42.53% of the total, respectively. Meanwhile, concerning the repetition of different primitives, the frequency of 4â9 times repeats was higher, accounting for 95.31% of the total. Among them, they were mainly 5- and 6-time repeats, which corresponded to 1887 and 2059 unigenes, respectively, and accounted for 28.44 and 31.03% of the total (Table <span class="InternalRef"><a href="#Tab3">3</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Distribution of SSR types in the transcriptome of <em class="EmphasisTypeItalic">Helianthus tuberosus</em> L</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
<col char=".">
</colgroup>
<thead>
<tr class="header">
<th rowspan="2"><p xmlns="" class="SimplePara">Repeat number</p></th>
<th colspan="5" char="."><p xmlns="" class="SimplePara">Motif type</p></th>
<th rowspan="2" char="."><p xmlns="" class="SimplePara">Total</p></th>
<th rowspan="2" char="."><p xmlns="" class="SimplePara">Ratio (%)</p></th>
</tr>
<tr class="header">
<th char="."><p xmlns="" class="SimplePara">Di-</p></th>
<th char="."><p xmlns="" class="SimplePara">Tri-</p></th>
<th char="."><p xmlns="" class="SimplePara">Tetra-</p></th>
<th char="."><p xmlns="" class="SimplePara">Penta-</p></th>
<th char="."><p xmlns="" class="SimplePara">Hexa-</p></th>
</tr>
</thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">4</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">360</p></td>
<td char="."><p xmlns="" class="SimplePara">135</p></td>
<td char="."><p xmlns="" class="SimplePara">111</p></td>
<td char="."><p xmlns="" class="SimplePara">606</p></td>
<td char="."><p xmlns="" class="SimplePara">9.13</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">5</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">1736</p></td>
<td char="."><p xmlns="" class="SimplePara">92</p></td>
<td char="."><p xmlns="" class="SimplePara">27</p></td>
<td char="."><p xmlns="" class="SimplePara">32</p></td>
<td char="."><p xmlns="" class="SimplePara">1887</p></td>
<td char="."><p xmlns="" class="SimplePara">28.44</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">1301</p></td>
<td char="."><p xmlns="" class="SimplePara">732</p></td>
<td char="."><p xmlns="" class="SimplePara">22</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">2059</p></td>
<td char="."><p xmlns="" class="SimplePara">31.03</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">733</p></td>
<td char="."><p xmlns="" class="SimplePara">247</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">995</p></td>
<td char="."><p xmlns="" class="SimplePara">14.99</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">8</p></td>
<td char="."><p xmlns="" class="SimplePara">456</p></td>
<td char="."><p xmlns="" class="SimplePara">41</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td char="."><p xmlns="" class="SimplePara">504</p></td>
<td char="."><p xmlns="" class="SimplePara">7.59</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">256</p></td>
<td char="."><p xmlns="" class="SimplePara">16</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">273</p></td>
<td char="."><p xmlns="" class="SimplePara">4.11</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">&gt;â9</p></td>
<td char="."><p xmlns="" class="SimplePara">256</p></td>
<td char="."><p xmlns="" class="SimplePara">50</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td char="."><p xmlns="" class="SimplePara">0</p></td>
<td char="."><p xmlns="" class="SimplePara">2</p></td>
<td char="."><p xmlns="" class="SimplePara">311</p></td>
<td char="."><p xmlns="" class="SimplePara">4.69</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">3002</p></td>
<td char="."><p xmlns="" class="SimplePara">2822</p></td>
<td char="."><p xmlns="" class="SimplePara">489</p></td>
<td char="."><p xmlns="" class="SimplePara">169</p></td>
<td char="."><p xmlns="" class="SimplePara">153</p></td>
<td char="."><p xmlns="" class="SimplePara">6635</p></td>
<td char="."> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Proportion%</p></td>
<td char="."><p xmlns="" class="SimplePara">45.24</p></td>
<td char="."><p xmlns="" class="SimplePara">42.53</p></td>
<td char="."><p xmlns="" class="SimplePara">7.37</p></td>
<td char="."><p xmlns="" class="SimplePara">2.55</p></td>
<td char="."><p xmlns="" class="SimplePara">2.31</p></td>
<td char="."> </td>
<td char="."><p xmlns="" class="SimplePara">100.00</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">We found the Jerusalem artichoke transcriptome to be rich in SSR types, with 6635 SSR loci containing 180 repeat motifs, corresponding to 4 dinucleotides, 10 trinucleotides, 30 tetranucleotides, 52 petanucleotides and 83 hexanucleotides. Among the dinucleotides, AG/CT repeat motifs were dominant, accounting for 32.9% of all SSR; while the repeating motifs ACC/GGT, ATC/ATG, and AAG/CTT in the trinucleotides accounted for 10, 8.4 and 8.4%, respectively. The higher proportion of tetranucleotides corresponded to AAAAC/GTTTT, AAAAG/CTTTT, AAAAT/ATTTT and AAACC/GGTTT, while the remaining tetranucleotide, pentanucleotide and hexanucleotide repeating motifs were more dispersed, with low occurrence frequencies, and account only for 9.4% (Fig. <span class="InternalRef"><a href="#Fig7">7</a></span>).<figure xmlns="" class="Figure" id="Fig7"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO7">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig7_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig7_HTML.png" alt="Fig. 7" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig7_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-019-0086-8/MediaObjects/41065_2019_86_Fig7_HTML.png" alt="Fig. 7"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 7</span>
                        <p xmlns="" class="SimplePara">Proportions of SSR among the total tandem repeats in Jerusalem artichoke</p>
                      </div></figcaption></figure>
</div>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Jerusalem artichoke is a hexaploid (2nâ=â6xâ=â102) outcrossing plant with a large genome [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], which utilizes tubers for asexual reproduction and has self-incompatibility, low hybridization rate, and a complex genetic structure. Compared with other large crops, bioinformatics resources on Jerusalem artichoke are relatively scarce. With the widespread use of transcriptome sequencing technology, mining for genetic resources is possible with low cost and high throughput. In this study, RNA-Seq technology was used to assemble and analyze the transcriptomes of different tissues at flowering stage, obtaining 63,089 unigenes. After analysis of the Nr and Swiss Prot protein databases in NCBI, 41,041 (65.05%) and 33,494 (53.09%) unigenes were not annotated as relevant information in each database, respectively. This phenomenon has been reported in the results of transcriptome sequencing in many species, such as <em xmlns="" class="EmphasisTypeItalic">Abelmoschu sesculentus</em> [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>], <em xmlns="" class="EmphasisTypeItalic">Fraxinus velutina</em> [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>], <em xmlns="" class="EmphasisTypeItalic">Morus alba</em> L. [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>], and other. This has been related to the short length of some Unigene fragments, the lack of information on the relevant database genes, and the possible presence of new genes in Jerusalem artichoke [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. The GO database was used to analyze the functional classification and biological characteristics of the annotated genes. However, due to defects such as incomplete functional classification in the GO database, 41,738 unigenes have not been annotated to possible GO entries. Therefore, other bioinformatics methods are needed to supplement the functional classification of Jerusalem artichoke unigenes.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Through the analysis of amino acid sequence alignment, we found the unigenes related to fructan metabolism in Jerusalem artichoke had some sequence-conserved functional domains, all belonging to the 32 family of glycosyl hydrolase (Glycosylhydrolysase 32, GH-32). The phylogenetic tree was constructed to show fructosyltransferases (FBE) and vacuolar invertase constituted a branch of the Jerusalem artichoke phylogenetic tree, while extracellular hydrolase and cell wall transformase were clustered into one species, indicating FBE in plants may have evolved from vacuolar invertase [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. These similar evolutionary origins are supported by studies where synthetase 1-FFT has been observed to display degradative enzymatic function, while the degrading enzyme 1-FEH sometimes has partial synthesis ability [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>, <span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. Further research is need to clarify the functions of currently known fructan metabolism-related enzymes. Indeed, although some genes involved in fructan metabolism in Jerusalem artichoke fructan have been cloned [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>, <span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>], other genes in this metabolic pathway need further exploration.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Analysis of the expression of key unigenes in the synthesis and degradation of fructan in Jerusalem artichoke revealed 12 unigenes were expressed during tuber development. The metabolism of fructan is a complex processes. Fructan is a carbohydrate composed of multiple fructose groups linked by glycosidic bonds. Sucrose is a precursor of fructan synthesis, and different combinations of fructosyltransferases catalyze the synthesis of multiple types of fructans, without the involvement of phosphorylation or nucleotide cofactors [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Results indicate Unigene32177, which is related to fructan synthesis in Jerusalem artichoke, has similar expression patterns to sucrose invertases Unigene7092 and Unigene46471. Because the substrates are fructose and glucose produced by sucrose hydrolysis during inulin synthesis, fructan synthesis and sucrose conversion are thought to occur simultaneously. Current research on inulin-type fructan have focused on 1-SST, 1-FFT (two important genes involved in fructan synthesis) and FEH (which is ultimately responsible for the degradation of fructan). However, invertase INV is required for the complete degradation of fructan [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. Because one the final product of FEH hydrolysis is sucrose, this reaction is inhibited by sucrose [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. The outcomes of the present study indicate different INV have diverse expression levels in various stages of Jerusalem artichoke tuber development. It is hypothesized these functional differences among INV are expressed at a variety of stages, showing functional pleiotropy. More importantly, the synergistic effect of INV gene is necessary during fructose synthesis and degradation in Jerusalem artichoke tuber development. The sequencing data from the transcriptome of this species can provide a reliable data basis for the identification and assessment of the expression of the members of the INV gene family. Therefore, further study is required regarding the functional analysis of INV in fructose synthesis and degradation, and its involvement in signal transduction pathways.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">There is currently no SSR marker available in Jerusalem artichoke, and only 6 pairs of SSR-labeled primers have been identified in the same sunflower of Jerusalem artichoke [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>], which does not meet research needs. However, the direct use of Jerusalem artichoke genome information to develop SSR markers is an important prerequisite for systematic research on the inheritance and evolution of this species. In this study, the transcriptome sequence information of Jerusalem artichoke was used to conduct a SSR locus search, identifying 6635 SSR loci with a frequency of 10.52%, which was higher than 7.6% reported for <em xmlns="" class="EmphasisTypeItalic">Cajanus cajan</em> L. [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>], but lower than the 14.70% documented for tea [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. Dinucleotides (45.24%) and trinucleotides (42.53%) appeared most frequently as repeat motifs in Jerusalem artichoke, which was consistent with most species [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. The most frequent dinucleotide repeat motif was AG/CT, echoing studies in dicots such as <em xmlns="" class="EmphasisTypeItalic">Hevea brasiliensis</em> [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>] and <em xmlns="" class="EmphasisTypeItalic">Vitis</em> L. [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. The most abundant trinucleotide repeat was ACC/GGT, which is inconsistent with the previous reports where the main repeat type of trinucleotide in dicots was AAG/CTT [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>].</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">In this study, a database evaluation of the transcriptome of Jerusalem artichoke from the Qinghai Tibet Plateau was conducted with high throughput sequencing technology to obtain a great amount of data on transcripts and SSR loci, depicting the general expression characteristics of this speciesâ transcriptome, and identifying its key metabolism-related genes. Moreover, this study also provides a reference for subsequent investigation of the metabolism of fructan and its regulation in Jerusalem artichoke; along with abundant data for molecular biology research on Asteraceae Helianthus plants.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Materialsandmethods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Materialsandmethods">Materials and methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Materialsandmethods" data-test="collapse-section">
<section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Plant material and RNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">The transcriptome data was derived from the results of high-throughput sequencing conducted with Illumina equipment in âQingyu No. 1â Jerusalem artichoke in 2016. In March 2016, this variety was planted in the experimental site of the College of Agriculture and Forestry of Qinghai University (N36Â°43â²45.21â²â²E101Â°44â²57.69â²â²). After the roots (1âcm of thick root tip, 1âcm of young root tip), stems (1âcm at the base of the stem, 0.5âcm at the tip of the stem), leaves (0.1âg of the first euphylla, 0.1âg of the tender leaf), flowers (0.1âg of flower petals, 0.1âg of pollen) and tubers (0.1âg of the longitudinal epidermis, 0.1âg of the tuber center) were taken from samples in the flowering stage, RNA was extracted, and the mixed sample was formed in equal amount. Then poly (A) was added, and the sequencing linker was ligated to prepare a sequencing library for PCR amplification and subsequent sequencing.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2"><h3 class="Heading">Illumina sequencing and de novo assembly</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">The PE125 sequencing method was used to construct the Jerusalem artichoke transcriptome library using the Illumina HiSeqâ¢ 2500 sequencing platform. The original image data obtained by order were converted to raw reads via base recognition (base calling), and raw reads were filtered to get clean reads, followed by statistical analysis of the amount of raw data and its length. The original data was subjected to removal of sequencing joints, repeated redundant sequences, and low-quality sequence data. Then, the number of clean reads, their total length, Q20, Q30%, N%, and GC% were counted. Q20 represents the ratio of bases having a mass of not less than 20 after filtration, Q30 represents the ratio of bases having a mass of not less than 30 after filtration, and N represents the ratio of bases which were not determined after filtration. The assembly was performed using Trinity [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>] software, the sequence was extended to contigs by overlap between sequences, and then transcripts were obtained by local assembly. Finally, Transcripts were homogenously clustered and spliced using TGICL [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>] and Phrap software [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>] to obtain a single gene cluster (unigene). Analytical projects included analysis of the sequencing assembly results, FPKM statistical analysis, and SSR analysis.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2"><h3 class="Heading">Sequence annotation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">By comparing the protein database of Jerusalem artichoke to the protein databases (E value &lt;1e-5) with the Blastx alignment tool, functional annotation was performed according to the similarity of the genes, and the protein with the highest sequence similarity with the given unigene was selected, thus the relevant functional annotation information. All annotation details were merged to summarize the information retrieved in this process. The protein databases utilized include Nr (Non-redundant protein database), Swiss Prot (Swiss Prot protein database), KOG (Clusters of orthologous groups for eukaryotic complete genomes), Cluster), GO (Gene Ontology) and KEGG (Kyoto Encyclopedia of Genes and Genomes). In addition, gene annotation was performed in the latter to identify key genes involved in the biosynthetic pathway of fructan in Jerusalem artichoke.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2"><h3 class="Heading">Expressed sequence tag-simple sequence repeat identification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">Unigenes in the Jerusalem artichoke transcriptome data was investigated using microsatellite identification tool (MISA), Available online: <span class="ExternalRef"><a href="http://pgrc.ipk-gatersleben.de/misa" target="_blank" class="ext-link"><span class="RefSource">http://pgrc.ipk-gatersleben.de/misa</span></a></span>. The search criteria were: minimum repeat number of dinucleotide 6 times, trinucleotide 5 times, 4 nucleotides 4 times, 5 nucleotides 4 times, and hexanucleotide 4 times, and the SSR locus interval was set to a minimum of 100âbp. The text file generated was imported into Excel for basic statistical analysis. SSR occurrence frequencyâ=ânumber of searched SSR-containing unigene sequences / total number of unigene sequence; SSR distribution frequencyâ=âSSR number / total number of unigene sequences; average distance of SSR distributionâ=âtotal unigene length / number of searched SSR.</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2"><h3 class="Heading">Quantitative polymerase chain reaction analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">The test material was âQingyu No.1â, the main cultivar of the Qinghai Tibet Plateau. Planting occurred in the middle of April 2016 in the experimental field of the Academy of agriculture and Forestry Sciences, Qinghai University, Xining, Qinghai. The normal cultivation method was implemented, with a row spacing of 1âm, and a plant spacing of 80âcm. Samples were taken during the expansion of the tubers, and a total of 4 samples were taken, until the plants withered. The tuber was set to repeat 3 times. Immediately after sampling, it was placed in liquid nitrogen for quick freezing, and then stored at ââ80âÂ°C in a refrigerator. RNA was extracted using the TIANGEN RNA prep Pure Plant Total RNA Extraction Kit (DP432), and then gelled with 1.2% agarose gel, as well as freshly prepared 0.5*TBE for 15âmin with 150âV to assess RNA integrity. In addition, the TaKaRa reverse transcription kit PrimeScriptâ¢ RTMaster Mix (Perfect Real Time) (RR036B) was used to realize reverse transcription according to manufacturerâs instructions. The RNA sample amount was 1âÎ¼g, and the reaction system was 20âÎ¼L. Furthermore, the reverse-transcribed cDNA sample was then stored at ââ20âÂ°C for later use. We performed real-time qPCR on a 96-well reaction plate with Applied Biosystems SteponePlus using TaKaRaâs SYBR Premix Ex TaqTM (TliRNaseH Plus) kit (RR420B). The internal reference primer was selected from the 25S ribosomal RNA primer F of Jerusalem artichoke: CTGTCTACTATCCAGCGAAACCA, R: AGGGCTCCCACTTATCCTACAC. The total volume of the quantitative PCR reaction was 20âÎ¼L, containing 10âÎ¼L of SYBR Premix, 0.6âÎ¼L of the upstream and downstream primers, 3.4âÎ¼L of ddH2O, and 5âÎ¼L of 0.1* cDNA. The reaction conditions of qPCR were: Pre-denaturation at 95âÂ°C for 30âs; 95âÂ°C for 5âs, 60âÂ°C for 30âs, 40âcycles. The fluorescence signal was collected during the extension phase at 60âÂ°C to obtain the cyclic Ct threshold of different genes. At the end of the cycle, the dissolution curve was measured to evaluate the specificity of the PCR product. The dissolution curve program was 95âÂ°C for 15âs, 60âÂ°C for 1âmin, and 95âÂ°C for 15âs. Each sample contained 3 biological replicates during the quantification process, 3 replicates of the same sample, and 3 primer-free negative controls in each 96-well reaction plate. The relative expression results of the genes were calculated using 2<sup>-ÎÎCT</sup> [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. The HeatMap Tools at <span class="ExternalRef"><a href="http://www.omicshare.com/tools/home/soft/heatmap" target="_blank" class="ext-link"><span class="RefSource">http://www.omicshare.com/tools/home/soft/heatmap</span></a></span>/ were implemented to draw a heatmap with parameters set to: cluster_rows: TRUE, cluster_cols: TRUE, colors: green, black, red.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Notes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Notes">Notes</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Notes"><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteAuthorContribution"><p xmlns="" class="SimplePara">Shipeng Yang and Xuemei Sun contributed equally to this work.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry">
<dt class="Term"><dfn>GO: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Gene ontology</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>KEGG: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Kyoto encyclopedia for genes and genomes</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>KOG: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Clusters of orthologous groups for eukaryotic complete genomes</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>SSR: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Simple sequence repeat</p>
                  </dd>
</dl>
</div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgments</h3>
<p xmlns="" class="SimplePara">We are grateful to the Qinghai Academy of Agriculture and Forestry Sciences for providing the cultivar. This work was supported by The Open Project of State Key Laboratory of Plateau Ecology and Agriculture of Qinghai University, Key Laboratory of Vegetable Genetics and Physiology of Qinghai Province.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par31" class="Para">This work was funded by National Natural Science Foundation of China (31460523), (31660588), (31660569), (31760600), and The Fundamental Research Program of Qinghai (2016-ZJ-751), (2017-ZJ-Y18).</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par32" class="Para">Data sharing not applicable to this article as no data-sets were generated or analyzed during the current study.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1">
<h3 xmlns="" class="Heading">Authorsâ contributions</h3>
<p xmlns="" class="SimplePara">Conceived and designed the experiments: ZQW. Performed the experiments: WLH, TJ, ZML, and LL, Analyzed the data: YSP, SXM and JXT. Wrote the paper: YSP, SXM, WLH, TJ and ZQW. Revised and approved the final version of the paper: YSP. All authors read and approved the final manuscript.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par33" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par34" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisherâs Note</h3>
                <p id="Par35" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Academy of Agriculture and Forestry Sciences of Qinghai University (Qinghai Academy of Agriculture and Forestry Sciences), Qinghai Key Laboratory of Vegetable Genetics and Physiology, State Key Laboratory of Plateau Ecology and Agriculture, Qinghai University, Xining, 810016, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Qinghai Higher Vocational &amp; Technical College Institute, Ledu, 810799, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saengkanuk A, Nuchadomrong S, Jogloy S, Patanothai A, Srijaranai S. A simplified spectrophotometric method for the determination of inulin in Jerusalem artichoke (Helianthus tuberosus L.) tubers. Eur Food Res Technol. 2011;233:609.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00217-011-1552-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20simplified%20spectrophotometric%20method%20for%20the%20determination%20of%20inulin%20in%20Jerusalem%20artichoke%20%28Helianthus%20tuberosus%20L.%29%20tubers&amp;author=A.%20Saengkanuk&amp;author=S.%20Nuchadomrong&amp;author=S.%20Jogloy&amp;author=A.%20Patanothai&amp;author=S.%20Srijaranai&amp;journal=Eur%20Food%20Res%20Technol&amp;volume=233&amp;pages=609&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang Y-Z, Zou S-M, He M-L, Wang C-H. Bioethanol production from the dry powder of Jerusalem artichoke tubers by recombinant Saccharomyces cerevisiae in simultaneous saccharification and fermentation. J Ind Microbiol Biotechnol. 2015;42:543â51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10295-014-1572-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bioethanol%20production%20from%20the%20dry%20powder%20of%20Jerusalem%20artichoke%20tubers%20by%20recombinant%20Saccharomyces%20cerevisiae%20in%20simultaneous%20saccharification%20and%20fermentation&amp;author=Y-Z.%20Wang&amp;author=S-M.%20Zou&amp;author=M-L.%20He&amp;author=C-H.%20Wang&amp;journal=J%20Ind%20Microbiol%20Biotechnol&amp;volume=42&amp;pages=543-551&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lee Y-J, M-G L, Yu S-Y, Yoon W-B, Lee O-H. Changes in physicochemical characteristics and antioxidant activities of Jerusalem artichoke tea infusions resulting from different production processes. Food Sci Biotechnol. 2014;23:1885â92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10068-014-0257-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Changes%20in%20physicochemical%20characteristics%20and%20antioxidant%20activities%20of%20Jerusalem%20artichoke%20tea%20infusions%20resulting%20from%20different%20production%20processes&amp;author=Y-J.%20Lee&amp;author=L.%20M-G&amp;author=S-Y.%20Yu&amp;author=W-B.%20Yoon&amp;author=O-H.%20Lee&amp;journal=Food%20Sci%20Biotechnol&amp;volume=23&amp;pages=1885-1892&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koutinas AA, Garcia IL, Kopsahelis N, Papanikolaou S, Webb C, Villar MA, LÃ³pez JA. Production of fermentation feedstock from Jerusalem artichoke tubers and its potential for Polyhydroxybutyrate synthesis. Waste Biomass Valorization. 2013;4:359â70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12649-012-9154-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Production%20of%20fermentation%20feedstock%20from%20Jerusalem%20artichoke%20tubers%20and%20its%20potential%20for%20Polyhydroxybutyrate%20synthesis&amp;author=AA.%20Koutinas&amp;author=IL.%20Garcia&amp;author=N.%20Kopsahelis&amp;author=S.%20Papanikolaou&amp;author=C.%20Webb&amp;author=MA.%20Villar&amp;author=JA.%20L%C3%B3pez&amp;journal=Waste%20Biomass%20Valorization&amp;volume=4&amp;pages=359-370&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Debez A, Belghith I, Friesen J, Montzka C, Elleuche S. Facing the challenge of sustainable bioenergy production: could halophytes be part of the solution? J Biol Eng. 2017;11:27.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s13036-017-0069-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Facing%20the%20challenge%20of%20sustainable%20bioenergy%20production%3A%20could%20halophytes%20be%20part%20of%20the%20solution%3F&amp;author=A.%20Debez&amp;author=I.%20Belghith&amp;author=J.%20Friesen&amp;author=C.%20Montzka&amp;author=S.%20Elleuche&amp;journal=J%20Biol%20Eng&amp;volume=11&amp;pages=27&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Phieler R, Voit A, Kothe E. Microbially supported phytoremediation of heavy metal contaminated soils: strategies and applications. In: Schippers A, Glombitza F, Sand W, editors. Geobiotechnology I: Metal-related Issues. Berlin, Heidelberg: Springer Berlin Heidelberg; 2014. p. 211â35.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microbially%20supported%20phytoremediation%20of%20heavy%20metal%20contaminated%20soils%3A%20strategies%20and%20applications&amp;author=R.%20Phieler&amp;author=A.%20Voit&amp;author=E.%20Kothe&amp;pages=211-235&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wardlaw CW, Reinders-Gouwentak CA, Champagnat P, Doorenbos J, Allsopp A. Organisation und Ausgestaltung des Sprosses. â organization and development of the shoot. In: Allsopp A, Bloch R, Bonner JT, Bopp M, Brachet J, Brown R, BÃ¼nning E, Buvat R, Cantino EC, Champagnat P, et al., editors. Differentiation and Development / Differenzierung und Entwicklung: Part 1 / Teil 1. Berlin, Heidelberg: Springer Berlin Heidelberg; 1965. p. 966â1255.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Organisation%20und%20Ausgestaltung%20des%20Sprosses.%20%E2%80%94%20organization%20and%20development%20of%20the%20shoot&amp;author=CW.%20Wardlaw&amp;author=CA.%20Reinders-Gouwentak&amp;author=P.%20Champagnat&amp;author=J.%20Doorenbos&amp;author=A.%20Allsopp&amp;pages=966-1255&amp;publication_year=1965"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">ApolinÃ¡rio AC, de Lima Damasceno BPG, de MacÃªdo BeltrÃ£o NE, Pessoa A, Converti A, da Silva JA. Inulin-type fructans: a review on different aspects of biochemical and pharmaceutical technology. Carbohydr Polym. 2014;101:368â78.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.carbpol.2013.09.081"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inulin-type%20fructans%3A%20a%20review%20on%20different%20aspects%20of%20biochemical%20and%20pharmaceutical%20technology&amp;author=AC.%20Apolin%C3%A1rio&amp;author=BPG.%20Lima%20Damasceno&amp;author=NE.%20Mac%C3%AAdo%20Beltr%C3%A3o&amp;author=A.%20Pessoa&amp;author=A.%20Converti&amp;author=JA.%20Silva&amp;journal=Carbohydr%20Polym&amp;volume=101&amp;pages=368-378&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Barthomeuf C, Regerat F, Pourrat H. High-yield ethanol production from Jerusalem artichoke tubers. World J Microbiol Biotechnol. 1991;7:490â3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00303375"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-yield%20ethanol%20production%20from%20Jerusalem%20artichoke%20tubers&amp;author=C.%20Barthomeuf&amp;author=F.%20Regerat&amp;author=H.%20Pourrat&amp;journal=World%20J%20Microbiol%20Biotechnol&amp;volume=7&amp;pages=490-493&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang M, Chen Q, Shen S. Physiological responses of two Jerusalem artichoke cultivars to drought stress induced by polyethylene glycol. Acta Physiol Plant. 2011;33:313â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11738-010-0549-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Physiological%20responses%20of%20two%20Jerusalem%20artichoke%20cultivars%20to%20drought%20stress%20induced%20by%20polyethylene%20glycol&amp;author=M.%20Zhang&amp;author=Q.%20Chen&amp;author=S.%20Shen&amp;journal=Acta%20Physiol%20Plant&amp;volume=33&amp;pages=313-318&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sharbatkhari M, Shobbar Z-S, Galeshi S, Nakhoda B. Wheat stem reserves and salinity tolerance: molecular dissection of fructan biosynthesis and remobilization to grains. Planta. 2016;244:191â202.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00425-016-2497-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Wheat%20stem%20reserves%20and%20salinity%20tolerance%3A%20molecular%20dissection%20of%20fructan%20biosynthesis%20and%20remobilization%20to%20grains&amp;author=M.%20Sharbatkhari&amp;author=Z-S.%20Shobbar&amp;author=S.%20Galeshi&amp;author=B.%20Nakhoda&amp;journal=Planta&amp;volume=244&amp;pages=191-202&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fava F, Tuohy KM. Inulin regulates endothelial function: a prebiotic smoking gun? Nat Rev Gastroenterol Hepatol. 2017;14:392.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrgastro.2017.68"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inulin%20regulates%20endothelial%20function%3A%20a%20prebiotic%20smoking%20gun%3F&amp;author=F.%20Fava&amp;author=KM.%20Tuohy&amp;journal=Nat%20Rev%20Gastroenterol%20Hepatol&amp;volume=14&amp;pages=392&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cruz-CÃ¡rdenas CI, Miranda-Ham ML, Castro-Concha LA, Ku-Cauich JR, Vergauwen R, Reijnders T, Van den Ende W, Escobedo-GraciaMedrano RM. Fructans and other water soluble carbohydrates in vegetative organs and fruits of different Musa spp. accessions. Front Plant Sci. 2015;6:395.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3389/fpls.2015.00395"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fructans%20and%20other%20water%20soluble%20carbohydrates%20in%20vegetative%20organs%20and%20fruits%20of%20different%20Musa%20spp.%20accessions&amp;author=CI.%20Cruz-C%C3%A1rdenas&amp;author=ML.%20Miranda-Ham&amp;author=LA.%20Castro-Concha&amp;author=JR.%20Ku-Cauich&amp;author=R.%20Vergauwen&amp;author=T.%20Reijnders&amp;author=W.%20Ende&amp;author=RM.%20Escobedo-GraciaMedrano&amp;journal=Front%20Plant%20Sci&amp;volume=6&amp;pages=395&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Aliasgharzadeh A, Khalili M, Mirtaheri E, Pourghassem Gargari B, Tavakoli F, Abbasalizad Farhangi M, Babaei H, Dehghan P. A combination of prebiotic inulin and Oligofructose improve some of cardiovascular disease risk factors in women with type 2 diabetes: a randomized controlled clinical trial. Advanced Pharmaceutical Bulletin. 2015;5:507â14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.15171/apb.2015.069"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20combination%20of%20prebiotic%20inulin%20and%20Oligofructose%20improve%20some%20of%20cardiovascular%20disease%20risk%20factors%20in%20women%20with%20type%202%20diabetes%3A%20a%20randomized%20controlled%20clinical%20trial&amp;author=A.%20Aliasgharzadeh&amp;author=M.%20Khalili&amp;author=E.%20Mirtaheri&amp;author=B.%20Pourghassem%20Gargari&amp;author=F.%20Tavakoli&amp;author=M.%20Abbasalizad%20Farhangi&amp;author=H.%20Babaei&amp;author=P.%20Dehghan&amp;journal=Advanced%20Pharmaceutical%20Bulletin&amp;volume=5&amp;pages=507-514&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ritsema T, Smeekens S. Fructans: beneficial for plants and humans. Curr Opin Plant Biol. 2003;6:223.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1369-5266(03)00034-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fructans%3A%20beneficial%20for%20plants%20and%20humans&amp;author=T.%20Ritsema&amp;author=S.%20Smeekens&amp;journal=Curr%20Opin%20Plant%20Biol&amp;volume=6&amp;pages=223&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vijn I, Smeekens S. Fructan: more than a reserve carbohydrate? Plant Physiol. 1999;120:351â60.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.120.2.351"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fructan%3A%20more%20than%20a%20reserve%20carbohydrate%3F&amp;author=I.%20Vijn&amp;author=S.%20Smeekens&amp;journal=Plant%20Physiol&amp;volume=120&amp;pages=351-360&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edelman J, Jefford TG. The mechanisim of fructosan metabolism in higher plants as exemplified in helianthus tuberosus. New Phytol. 1968;67:517â31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1469-8137.1968.tb05480.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20mechanisim%20of%20fructosan%20metabolism%20in%20higher%20plants%20as%20exemplified%20in%20helianthus%20tuberosus&amp;author=J.%20Edelman&amp;author=TG.%20Jefford&amp;journal=New%20Phytol&amp;volume=67&amp;pages=517-531&amp;publication_year=1968"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Higashiya H, Kobayashi K, Terada S. Fructan as a novel effective factor for mammalian cell. Culture. 2010:679â82.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Higashiya%20H%2C%20Kobayashi%20K%2C%20Terada%20S.%20Fructan%20as%20a%20novel%20effective%20factor%20for%20mammalian%20cell.%20Culture.%202010%3A679%E2%80%9382."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shiomi N. Purification and characterisation of 1F-fructosyltransferase from the roots of asparagus (asparagus officinalis L.). Carbohydr Res. 1982;99:157â69.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0008-6215(00)81905-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Purification%20and%20characterisation%20of%201F-fructosyltransferase%20from%20the%20roots%20of%20asparagus%20%28asparagus%20officinalis%20L.%29&amp;author=N.%20Shiomi&amp;journal=Carbohydr%20Res&amp;volume=99&amp;pages=157-169&amp;publication_year=1982"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ueno K, Ishiguro Y, Yoshida M, Onodera S, Shiomi N. Cloning and functional characterization of a fructan 1-exohydrolase (1-FEH) in edible burdock (Arctium lappa L.). Chem Cent J. 2011;5:16.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1752-153X-5-16"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cloning%20and%20functional%20characterization%20of%20a%20fructan%201-exohydrolase%20%281-FEH%29%20in%20edible%20burdock%20%28Arctium%20lappa%20L.%29&amp;author=K.%20Ueno&amp;author=Y.%20Ishiguro&amp;author=M.%20Yoshida&amp;author=S.%20Onodera&amp;author=N.%20Shiomi&amp;journal=Chem%20Cent%20J&amp;volume=5&amp;pages=16&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dassanayake M, Haas JS, Bohnert HJ, Cheeseman JM. Shedding light on an extremophile lifestyle through transcriptomics. New Phytol. 2009;183:764â75.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1469-8137.2009.02913.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Shedding%20light%20on%20an%20extremophile%20lifestyle%20through%20transcriptomics&amp;author=M.%20Dassanayake&amp;author=JS.%20Haas&amp;author=HJ.%20Bohnert&amp;author=JM.%20Cheeseman&amp;journal=New%20Phytol&amp;volume=183&amp;pages=764-775&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lu T, Lu G, Fan D, Zhu C, Li W, Zhao Q, Feng Q, Zhao Y, Guo Y, Li W, et al. Function annotation of the rice transcriptome at single-nucleotide resolution by RNA-seq. Genome Res. 2010;20:1238â49.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.106120.110"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Function%20annotation%20of%20the%20rice%20transcriptome%20at%20single-nucleotide%20resolution%20by%20RNA-seq&amp;author=T.%20Lu&amp;author=G.%20Lu&amp;author=D.%20Fan&amp;author=C.%20Zhu&amp;author=W.%20Li&amp;author=Q.%20Zhao&amp;author=Q.%20Feng&amp;author=Y.%20Zhao&amp;author=Y.%20Guo&amp;author=W.%20Li&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1238-1249&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sangwan RS, Tripathi S, Singh J, Narnoliya LK, Sangwan NS. De novo sequencing and assembly of Centella asiatica leaf transcriptome for mapping of structural, functional and regulatory genes with special reference to secondary metabolism. Gene. 2013;525:58â76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2013.04.057"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20sequencing%20and%20assembly%20of%20Centella%20asiatica%20leaf%20transcriptome%20for%20mapping%20of%20structural%2C%20functional%20and%20regulatory%20genes%20with%20special%20reference%20to%20secondary%20metabolism&amp;author=RS.%20Sangwan&amp;author=S.%20Tripathi&amp;author=J.%20Singh&amp;author=LK.%20Narnoliya&amp;author=NS.%20Sangwan&amp;journal=Gene&amp;volume=525&amp;pages=58-76&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bock DG, Kane NC, Ebert DP, Rieseberg LH. Genome skimming reveals the origin of the Jerusalem artichoke tuber crop species: neither from Jerusalem nor an artichoke. New Phytol. 2014;201:1021â30.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/nph.12560"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20skimming%20reveals%20the%20origin%20of%20the%20Jerusalem%20artichoke%20tuber%20crop%20species%3A%20neither%20from%20Jerusalem%20nor%20an%20artichoke&amp;author=DG.%20Bock&amp;author=NC.%20Kane&amp;author=DP.%20Ebert&amp;author=LH.%20Rieseberg&amp;journal=New%20Phytol&amp;volume=201&amp;pages=1021-1030&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schafleitner R, Kumar S, C-y L, Hegde SG, Ebert A. The okra (Abelmoschus esculentus) transcriptome as a source for gene sequence information and molecular markers for diversity analysis. Gene. 2013;517:27â36.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2012.12.098"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20okra%20%28Abelmoschus%20esculentus%29%20transcriptome%20as%20a%20source%20for%20gene%20sequence%20information%20and%20molecular%20markers%20for%20diversity%20analysis&amp;author=R.%20Schafleitner&amp;author=S.%20Kumar&amp;author=L.%20C-y&amp;author=SG.%20Hegde&amp;author=A.%20Ebert&amp;journal=Gene&amp;volume=517&amp;pages=27-36&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan LP, Liu CL, Wu DJ, Li L, Shu J, Sun C, Xia Y, Zhao LJ. De novo transcriptome analysis of Fraxinus velutina using Illumina platform and development of EST-SSR markers. Biol Plant. 2016;61:210â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10535-016-0681-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20analysis%20of%20Fraxinus%20velutina%20using%20Illumina%20platform%20and%20development%20of%20EST-SSR%20markers&amp;author=LP.%20Yan&amp;author=CL.%20Liu&amp;author=DJ.%20Wu&amp;author=L.%20Li&amp;author=J.%20Shu&amp;author=C.%20Sun&amp;author=Y.%20Xia&amp;author=LJ.%20Zhao&amp;journal=Biol%20Plant&amp;volume=61&amp;pages=210-218&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu CY, Liu XQ, Long DP, Cao BN, Xiang ZH, Zhao AC. De novo assembly of mulberry (Morus alba L.) transcriptome and identification of candidate unigenes related to salt stress responses. Russ J Plant Physiol. 2017;64:738â48.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1134/S1021443717050053"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20mulberry%20%28Morus%20alba%20L.%29%20transcriptome%20and%20identification%20of%20candidate%20unigenes%20related%20to%20salt%20stress%20responses&amp;author=CY.%20Liu&amp;author=XQ.%20Liu&amp;author=DP.%20Long&amp;author=BN.%20Cao&amp;author=ZH.%20Xiang&amp;author=AC.%20Zhao&amp;journal=Russ%20J%20Plant%20Physiol&amp;volume=64&amp;pages=738-748&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao X, Wang Q, Jiao Y, Huang R, Deng Y, Wang H, Du X. Identification of genes potentially related to biomineralization and immunity by transcriptome analysis of pearl sac in pearl oyster Pinctada martensii. Mar Biotechnol. 2012;14:730â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10126-012-9438-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20potentially%20related%20to%20biomineralization%20and%20immunity%20by%20transcriptome%20analysis%20of%20pearl%20sac%20in%20pearl%20oyster%20Pinctada%20martensii&amp;author=X.%20Zhao&amp;author=Q.%20Wang&amp;author=Y.%20Jiao&amp;author=R.%20Huang&amp;author=Y.%20Deng&amp;author=H.%20Wang&amp;author=X.%20Du&amp;journal=Mar%20Biotechnol&amp;volume=14&amp;pages=730-739&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kawakami A, Yoshida M. Fructan:fructan 1-fructosyltransferase, a key enzyme for biosynthesis of graminan oligomers in hardened wheat. Planta. 2005;223:90â104.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00425-005-0054-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fructan%3Afructan%201-fructosyltransferase%2C%20a%20key%20enzyme%20for%20biosynthesis%20of%20graminan%20oligomers%20in%20hardened%20wheat&amp;author=A.%20Kawakami&amp;author=M.%20Yoshida&amp;journal=Planta&amp;volume=223&amp;pages=90-104&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stolze A, Wanke A, van Deenen N, Geyer R, Prufer D, Schulze Gronover C. Development of rubber-enriched dandelion varieties by metabolic engineering of the inulin pathway. Plant Biotechnol J. 2017;15:740â53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/pbi.12672"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20rubber-enriched%20dandelion%20varieties%20by%20metabolic%20engineering%20of%20the%20inulin%20pathway&amp;author=A.%20Stolze&amp;author=A.%20Wanke&amp;author=N.%20Deenen&amp;author=R.%20Geyer&amp;author=D.%20Prufer&amp;author=C.%20Schulze%20Gronover&amp;journal=Plant%20Biotechnol%20J&amp;volume=15&amp;pages=740-753&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wim VE, Midori Y, Stefan C, Rudy V, Akira K. Cloning, characterization and functional analysis of novel 6-kestose exohydrolases (6-KEHs) from wheat (Triticum aestivum). New Phytol. 2005;166:917â32.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1469-8137.2005.01394.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cloning%2C%20characterization%20and%20functional%20analysis%20of%20novel%206-kestose%20exohydrolases%20%286-KEHs%29%20from%20wheat%20%28Triticum%20aestivum%29&amp;author=VE.%20Wim&amp;author=Y.%20Midori&amp;author=C.%20Stefan&amp;author=V.%20Rudy&amp;author=K.%20Akira&amp;journal=New%20Phytol&amp;volume=166&amp;pages=917-932&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu H, Liang M, Xu L, Li H, Zhang X, Kang J, Zhao Q, Zhao H. Cloning and functional characterization of two abiotic stress-responsive Jerusalem artichoke (Helianthus tuberosus) fructan 1-exohydrolases (1-FEHs). Plant Mol Biol. 2015;87:81â98.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11103-014-0262-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cloning%20and%20functional%20characterization%20of%20two%20abiotic%20stress-responsive%20Jerusalem%20artichoke%20%28Helianthus%20tuberosus%29%20fructan%201-exohydrolases%20%281-FEHs%29&amp;author=H.%20Xu&amp;author=M.%20Liang&amp;author=L.%20Xu&amp;author=H.%20Li&amp;author=X.%20Zhang&amp;author=J.%20Kang&amp;author=Q.%20Zhao&amp;author=H.%20Zhao&amp;journal=Plant%20Mol%20Biol&amp;volume=87&amp;pages=81-98&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van Der Meer IM, Koops AJ, Hakkert JC, Van Tunen AJ. Cloning of the fructan biosynthesis pathway of Jerusalem artichoke. Plant J. 1998;15:489â500.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-313X.1998.00230.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cloning%20of%20the%20fructan%20biosynthesis%20pathway%20of%20Jerusalem%20artichoke&amp;author=IM.%20Meer&amp;author=AJ.%20Koops&amp;author=JC.%20Hakkert&amp;author=AJ.%20Tunen&amp;journal=Plant%20J&amp;volume=15&amp;pages=489-500&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jung WY, Lee SS, Park HJ, Kim CW, Kwon S-Y, Jeon J-H, Kim H-S, Cho HS. Comparative transcriptome profiling and SSR marker identification in three Jerusalem artichoke (Helianthus tuberosus L.) cultivars exhibiting phenotypic variation. Plant Biotechnol Rep. 2016;10:447â61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11816-016-0421-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20transcriptome%20profiling%20and%20SSR%20marker%20identification%20in%20three%20Jerusalem%20artichoke%20%28Helianthus%20tuberosus%20L.%29%20cultivars%20exhibiting%20phenotypic%20variation&amp;author=WY.%20Jung&amp;author=SS.%20Lee&amp;author=HJ.%20Park&amp;author=CW.%20Kim&amp;author=S-Y.%20Kwon&amp;author=J-H.%20Jeon&amp;author=H-S.%20Kim&amp;author=HS.%20Cho&amp;journal=Plant%20Biotechnol%20Rep&amp;volume=10&amp;pages=447-461&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dutta S, Kumawat G, Singh BP, Gupta DK, Singh S, Dogra V, Gaikwad K, Sharma TR, Raje RS, Bandhopadhya TK, et al. Development of genic-SSR markers by deep transcriptome sequencing in pigeonpea [Cajanus cajan (L.) Millspaugh]. BMC Plant Biol. 2011;11:17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-11-17"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20genic-SSR%20markers%20by%20deep%20transcriptome%20sequencing%20in%20pigeonpea%20%5BCajanus%20cajan%20%28L.%29%20Millspaugh%5D&amp;author=S.%20Dutta&amp;author=G.%20Kumawat&amp;author=BP.%20Singh&amp;author=DK.%20Gupta&amp;author=S.%20Singh&amp;author=V.%20Dogra&amp;author=K.%20Gaikwad&amp;author=TR.%20Sharma&amp;author=RS.%20Raje&amp;author=TK.%20Bandhopadhya&amp;journal=BMC%20Plant%20Biol&amp;volume=11&amp;pages=17&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu H, Chen D, Li J, Yu B, Qiao X, Huang H, He Y. De novo characterization of leaf transcriptome using 454 sequencing and development of EST-SSR markers in tea (Camellia sinensis). Plant Mol Biol Report. 2013;31:524â38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11105-012-0519-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20characterization%20of%20leaf%20transcriptome%20using%20454%20sequencing%20and%20development%20of%20EST-SSR%20markers%20in%20tea%20%28Camellia%20sinensis%29&amp;author=H.%20Wu&amp;author=D.%20Chen&amp;author=J.%20Li&amp;author=B.%20Yu&amp;author=X.%20Qiao&amp;author=H.%20Huang&amp;author=Y.%20He&amp;journal=Plant%20Mol%20Biol%20Report&amp;volume=31&amp;pages=524-538&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liang X, Chen X, Hong Y, Liu H, Zhou G, Li S, Guo B. Utility of EST-derived SSR in cultivated peanut (Arachis hypogaea L.) and Arachiswild species. BMC Plant Biol. 2009;9:35.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-9-35"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Utility%20of%20EST-derived%20SSR%20in%20cultivated%20peanut%20%28Arachis%20hypogaea%20L.%29%20and%20Arachiswild%20species&amp;author=X.%20Liang&amp;author=X.%20Chen&amp;author=Y.%20Hong&amp;author=H.%20Liu&amp;author=G.%20Zhou&amp;author=S.%20Li&amp;author=B.%20Guo&amp;journal=BMC%20Plant%20Biol&amp;volume=9&amp;pages=35&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Feng SP, Li WG, Huang HS, Wang JY, Wu YT. Development, characterization and cross-species/genera transferability of EST-SSR markers for rubber tree (Hevea brasiliensis). Mol Breed. 2009;23:85â97.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-008-9216-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%2C%20characterization%20and%20cross-species%2Fgenera%20transferability%20of%20EST-SSR%20markers%20for%20rubber%20tree%20%28Hevea%20brasiliensis%29&amp;author=SP.%20Feng&amp;author=WG.%20Li&amp;author=HS.%20Huang&amp;author=JY.%20Wang&amp;author=YT.%20Wu&amp;journal=Mol%20Breed&amp;volume=23&amp;pages=85-97&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang H, Lu J, Ren Z, Hunter W, Dowd SE, Dang P. Mining and validating grape (Vitis L.) ESTs to develop EST-SSR markers for genotyping and mapping. Mol Breed. 2011;28:241â54.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-010-9477-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mining%20and%20validating%20grape%20%28Vitis%20L.%29%20ESTs%20to%20develop%20EST-SSR%20markers%20for%20genotyping%20and%20mapping&amp;author=H.%20Huang&amp;author=J.%20Lu&amp;author=Z.%20Ren&amp;author=W.%20Hunter&amp;author=SE.%20Dowd&amp;author=P.%20Dang&amp;journal=Mol%20Breed&amp;volume=28&amp;pages=241-254&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Morgante M, Hanafey M, Powell W. Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes. Nat Genet. 2002;30:194.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng822"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellites%20are%20preferentially%20associated%20with%20nonrepetitive%20DNA%20in%20plant%20genomes&amp;author=M.%20Morgante&amp;author=M.%20Hanafey&amp;author=W.%20Powell&amp;journal=Nat%20Genet&amp;volume=30&amp;pages=194&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, Adiconis X, Fan L, Raychowdhury R, Zeng Q, et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat Biotechnol. 2011;29:644.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1883"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=MG.%20Grabherr&amp;author=BJ.%20Haas&amp;author=M.%20Yassour&amp;author=JZ.%20Levin&amp;author=DA.%20Thompson&amp;author=I.%20Amit&amp;author=X.%20Adiconis&amp;author=L.%20Fan&amp;author=R.%20Raychowdhury&amp;author=Q.%20Zeng&amp;journal=Nat%20Biotechnol&amp;volume=29&amp;pages=644&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pertea G, Huang X, Liang F, Antonescu V, Sultana R, Karamycheva S, Lee Y, White J, Cheung F, Parvizi B, et al. TIGR gene indices clustering tools (TGICL): a software system for fast clustering of large EST datasets. Bioinformatics. 2003;19:651â2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg034"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TIGR%20gene%20indices%20clustering%20tools%20%28TGICL%29%3A%20a%20software%20system%20for%20fast%20clustering%20of%20large%20EST%20datasets&amp;author=G.%20Pertea&amp;author=X.%20Huang&amp;author=F.%20Liang&amp;author=V.%20Antonescu&amp;author=R.%20Sultana&amp;author=S.%20Karamycheva&amp;author=Y.%20Lee&amp;author=J.%20White&amp;author=F.%20Cheung&amp;author=B.%20Parvizi&amp;journal=Bioinformatics&amp;volume=19&amp;pages=651-652&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ewing B, Hillier LD, Wendl MC, Green P, Ewing B, Hillier LD, Wendl MC, Green P. Base-calling of automated sequencer traces UsingPhred. I. Accuracyâassessment. Genome Res. 1998;8(3):175â85.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.8.3.175"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20UsingPhred.%20I.%20Accuracy%E2%80%89assessment&amp;author=B.%20Ewing&amp;author=LD.%20Hillier&amp;author=MC.%20Wendl&amp;author=P.%20Green&amp;author=B.%20Ewing&amp;author=LD.%20Hillier&amp;author=MC.%20Wendl&amp;author=P.%20Green&amp;journal=Genome%20Res&amp;volume=8&amp;issue=3&amp;pages=175-185&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Livak KJ, Schmittgen TD. Analysis of relative gene expression data using real-time quantitative PCR and the 2âÎÎCT method. Methods. 2001;25:402â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/meth.2001.1262"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202%E2%88%92%CE%94%CE%94CT%20method&amp;author=KJ.%20Livak&amp;author=TD.%20Schmittgen&amp;journal=Methods&amp;volume=25&amp;pages=402-408&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© The Author(s) 2019</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-019-0086-8" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-019-0086-8">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-019-0086-8" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-019-0086-8?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-019-0086-8 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-019-0086-8 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-019-0086-8 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-019-0086-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-019-0086-8">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-019-0086-8"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-019-0086-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-019-0086-8">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-019-0086-8"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-019-0086-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-019-0086-8">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-019-0086-8"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-019-0086-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-019-0086-8">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-019-0086-8"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-019-0086-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-019-0086-8">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-019-0086-8"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-019-0086-8/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-019-0086-8">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-019-0086-8">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-019-0086-8" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-019-0086-8&amp;volume=156&amp;issue=1&amp;title=Characterization+of+the+Tibet+plateau+Jerusalem+artichoke+%28+Helianthus+tuberosus+L.%29+transcriptome+by+de+novo+assembly+to+discover+genes+associated+with+fructan+synthesis+and+SSR+analysis&amp;author_name=Shipeng+Yang&amp;start_page=1&amp;end_page=13">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-019-0086-8" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
