<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Insight into the genetic variability analysis and cultivar identification of tall fescue by using SSR markers | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Genetic diversity of 19 forage-type and 2 turf-type cultivars of tall fescue (Festuca arundinacea Schreb.) was revealed using SSR markers in an attempt to explore the genetic relationships among them, and examine potential use of SSR markers to identify cultivars by bulked samples. A total of 227 clear band was scored with 14 SSR primers and out of which 201 (88.6Â %) were found polymorphic. The percentage of polymorphic bands (PPB) per primer pair varied from 62.5 to 100Â % with an average of 86.9Â %. The polymorphism information content (PIC) value ranged from 0.116 to 0.347 with an average of 0.257 and the highest PIC value (0.347) was noticed for primer NFA040 followed by NFA113 (0.346) whereas the highest discriminating power (D) of 1 was shown in NFA037 and LMgSSR02-01C. A Neighbor-joining dendrogram and the principal component analysis identified six major clusters and grouped the cultivars in agreement with their breeding histories. STRUCTURE analysis divided these cultivars into 3 sub-clades which correspond to distance based groupings. These findings indicates that SSR markers by bulking strategy are a useful tool to measure genetic diversity among tall fescue cultivars and could be used to supplement morphological data for plant variety protection.">
<meta name="description" content="Genetic diversity of 19 forage-type and 2 turf-type cultivars of tall fescue (Festuca arundinacea Schreb.) was revealed using SSR markers in an attempt to explore the genetic relationships among them, and examine potential use of SSR markers to identify cultivars by bulked samples. A total of 227 clear band was scored with 14 SSR primers and out of which 201 (88.6Â %) were found polymorphic. The percentage of polymorphic bands (PPB) per primer pair varied from 62.5 to 100Â % with an average of 86.9Â %. The polymorphism information content (PIC) value ranged from 0.116 to 0.347 with an average of 0.257 and the highest PIC value (0.347) was noticed for primer NFA040 followed by NFA113 (0.346) whereas the highest discriminating power (D) of 1 was shown in NFA037 and LMgSSR02-01C. A Neighbor-joining dendrogram and the principal component analysis identified six major clusters and grouped the cultivars in agreement with their breeding histories. STRUCTURE analysis divided these cultivars into 3 sub-clades which correspond to distance based groupings. These findings indicates that SSR markers by bulking strategy are a useful tool to measure genetic diversity among tall fescue cultivars and could be used to supplement morphological data for plant variety protection.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Insight into the genetic variability analysis and cultivar identification of tall fescue by using SSR markers">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-08-11">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Genetic diversity of 19 forage-type and 2 turf-type cultivars of tall fescue (Festuca arundinacea Schreb.) was revealed using SSR markers in an attempt to explore the genetic relationships among them, and examine potential use of SSR markers to identify cultivars by bulked samples. A total of 227 clear band was scored with 14 SSR primers and out of which 201 (88.6 %) were found polymorphic. The percentage of polymorphic bands (PPB) per primer pair varied from 62.5 to 100 % with an average of 86.9 %. The polymorphism information content (PIC) value ranged from 0.116 to 0.347 with an average of 0.257 and the highest PIC value (0.347) was noticed for primer NFA040 followed by NFA113 (0.346) whereas the highest discriminating power (D) of 1 was shown in NFA037 and LMgSSR02-01C. A Neighbor-joining dendrogram and the principal component analysis identified six major clusters and grouped the cultivars in agreement with their breeding histories. STRUCTURE analysis divided these cultivars into 3 sub-clades which correspond to distance based groupings. These findings indicates that SSR markers by bulking strategy are a useful tool to measure genetic diversity among tall fescue cultivars and could be used to supplement morphological data for plant variety protection.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-08-11">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="9">
<meta name="prism.copyright" content="2016 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0013-1">
<meta name="prism.doi" content="doi:10.1186/s41065-016-0013-1">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0013-1">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0013-1">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Insight into the genetic variability analysis and cultivar identification of tall fescue by using SSR markers">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/08/11">
<meta name="citation_firstpage" content="9">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-016-0013-1">
<meta name="DOI" content="10.1186/s41065-016-0013-1">
<meta name="citation_doi" content="10.1186/s41065-016-0013-1">
<meta name="dc.creator" content="Kaixin Fu">
<meta name="dc.creator" content="Zhihui Guo">
<meta name="dc.creator" content="Xinquan Zhang">
<meta name="dc.creator" content="Yan Fan">
<meta name="dc.creator" content="Wendan Wu">
<meta name="dc.creator" content="Daxu Li">
<meta name="dc.creator" content="Yan Peng">
<meta name="dc.creator" content="Linkai Huang">
<meta name="dc.creator" content="Ming Sun">
<meta name="dc.creator" content="Shiqie Bai">
<meta name="dc.creator" content="Xiao Ma">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Plant Breeding Rev; citation_title=Breeding tall fescue; citation_author=DA Sleper; citation_volume=3; citation_publication_date=1985; citation_pages=313-42; citation_id=CR1">
<meta name="citation_reference" content="citation_title=The fescues; citation_inbook_title=Forages; citation_publication_date=1995; citation_pages=345-56; citation_id=CR2; citation_author=DA Sleper; citation_author=RC Buckner; citation_publisher=Iowa State University Press">
<meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Discriminating the ancestral progenitors of hexaploid Festuca arundinacea using genomic in situ hybridization; citation_author=MW Humphreys, HM Thomas, WG Morgan, MR Meredith, JA Harper, H Thomas, Z Zwierzkowski, M Ghesquiere; citation_volume=75; citation_issue=2; citation_publication_date=1995; citation_pages=171-4; citation_doi=10.1038/hdy.1995.120; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Genetic diversity of tall fescue germplasm based on RFLPs; citation_author=WW Xu, DA Sleper, GF Krause; citation_volume=34; citation_issue=1; citation_publication_date=1994; citation_pages=246-52; citation_doi=10.2135/cropsci1994.0011183X003400010045x; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Genetic variation, heritability and correlations of agro-morphological traits in tall fescue (Festuca arundinacea Schreb.); citation_author=MM Majidi, A Mirlohi, F Amini; citation_volume=167; citation_issue=3; citation_publication_date=2009; citation_pages=323-31; citation_doi=10.1007/s10681-009-9887-6; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Potential of RAPD markers for use in the identification of creeping bentgrass cultivars; citation_author=RC Golembiewski, TK Danneberger, PM Sweeney; citation_volume=37; citation_issue=1; citation_publication_date=1997; citation_pages=212-4; citation_doi=10.2135/cropsci1997.0011183X003700010036x; citation_id=CR6">
<meta name="citation_reference" content="citation_title=Fescues; citation_inbook_title=Fodder crops and amenity grasses, handbook of plant breeding 5; citation_publication_date=2010; citation_pages=261-92; citation_id=CR7; citation_author=OA Rognli; citation_author=MC Saha; citation_author=S Bhamidimarri; citation_author=S Heijden; citation_publisher=Springer">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev; citation_title=RFLP markers for cultivar identification in tall fescue (Festuca arundinacea Schreb.); citation_author=A Busti, ME Caceres, O Calderini, S Arcioni, F Pupilli; citation_volume=51; citation_issue=4; citation_publication_date=2004; citation_pages=443-8; citation_doi=10.1023/B:GRES.0000023459.62679.e8; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=RAPD variation within and among natural populations of outcrossing buffalograss [Buchloe dactyloides (Nutt.) Engelm.]; citation_author=DR Huff, R Peakall, PE Smouse; citation_volume=86; citation_issue=8; citation_publication_date=1993; citation_pages=927-34; citation_doi=10.1007/BF00211043; citation_id=CR9">
<meta name="citation_reference" content="citation_title=Analysis and optimization of bulk DNA sampling with binary scoring for germplasm characterization; citation_publication_date=2013; citation_id=CR10; citation_author=MH Reyes-Valdés; citation_author=A Santacruz-Varela; citation_author=O Martínez; citation_author=J Simpson; citation_author=C Hayano-Kanashiro; citation_author=C Cortés-Romero">
<meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Identification of orchardgrass (Dactylis glomerata L.) cultivars by using simple sequence repeat markers; citation_author=LF Jiang, XQ Zhang, X Ma, LK Huang, WG Xie, YM Ma, YF Zhao; citation_volume=12; citation_issue=4; citation_publication_date=2013; citation_pages=5111-23; citation_doi=10.4238/2013.October.29.5; citation_id=CR11">
<meta name="citation_reference" content="citation_title=Application of DNA profiling to an outbreeding forage species; citation_publication_date=2001; citation_id=CR12; citation_author=JW Forster; citation_author=ES Jones; citation_author=R Kölliker; citation_author=MC Drayton; citation_author=MP Dupal; citation_author=KM Guthridge; citation_author=KF Smith; citation_author=RJ Henry">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Determination of genetic diversity in tall fescue with AFLP markers; citation_author=MA Mian, AA Hopkins, JC Zwonitzer; citation_volume=42; citation_issue=3; citation_publication_date=2002; citation_pages=944-50; citation_doi=10.2135/cropsci2002.0944; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Plant Breeding; citation_title=Molecular markers and their applications in wheat breeding; citation_author=PK Gupta, RK Varshney, PC Sharma, B Ramesh; citation_volume=118; citation_issue=5; citation_publication_date=1999; citation_pages=369-90; citation_doi=10.1046/j.1439-0523.1999.00401.x; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Tall fescue EST-SSR markers with transferability across several grass species; citation_author=MC Saha, MR Mian, I Eujayl, JC Zwonitzer, L Wang, GD May; citation_volume=109; citation_issue=4; citation_publication_date=2004; citation_pages=783-91; citation_doi=10.1007/s00122-004-1681-1; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Tall fescue genomic SSR markers: development and transferability across multiple grass species; citation_author=MC Saha, JD Cooper, MR Mian, K Chekhovskiy, GD May; citation_volume=113; citation_issue=8; citation_publication_date=2006; citation_pages=1449-58; citation_doi=10.1007/s00122-006-0391-2; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Development of simple sequence repeat (SSR) markers and construction of an SSR-based linkage map in Italian ryegrass (Lolium multiflorum Lam.); citation_author=M Hirata, H Cai, M Inoue, N Yuyama, Y Miura, T Komatsu, T Takamizo, M Fujimori; citation_volume=113; citation_issue=2; citation_publication_date=2006; citation_pages=270-9; citation_doi=10.1007/s00122-006-0292-4; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=SSRs transferability and genetic diversity of Tunisian Festuca arundinacea and Lolium perenne
                           ; citation_author=H Elazreg, S Ghariani, N Chtourou-Ghorbel, M Chakroun, N Trifi-Farah; citation_volume=39; citation_issue=2; citation_publication_date=2011; citation_pages=79-87; citation_doi=10.1016/j.bse.2011.01.002; citation_id=CR18">
<meta name="citation_reference" content="citation_title=Breeding and cultivars; citation_publication_date=1979; citation_id=CR19; citation_author=KH Asay; citation_author=RV Frakes; citation_author=RC Buckner">
<meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=A modified protocol for rapid DNA isolation from plant tissues using cetyltrimethylammonium bromide; citation_author=GC Allen, MA Flores-Vergara, S Krasynanski, S Kumar, WF Thompson; citation_volume=1; citation_issue=5; citation_publication_date=2006; citation_pages=2320-5; citation_doi=10.1038/nprot.2006.384; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=J Bell Syst Tech; citation_title=A mathematical theory of communication; citation_author=CE Shannon; citation_volume=27; citation_publication_date=1948; citation_pages=379-423; citation_doi=10.1002/j.1538-7305.1948.tb01338.x; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Optimizing parental selection for genetic linkage maps; citation_author=JA Anderson, GA Churchill, JE Autrique, SD Tanksley, ME Sorrells; citation_volume=36; citation_issue=1; citation_publication_date=1993; citation_pages=181-6; citation_doi=10.1139/g93-024; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Optimization of the choice of molecular markers for varietal identification in Vitis vinifera L; citation_author=C Tessier, J David, P This, J Boursiquot, A Charrier; citation_volume=98; citation_issue=1; citation_publication_date=1999; citation_pages=171-7; citation_doi=10.1007/s001220051054; citation_id=CR23">
<meta name="citation_reference" content="citation_title=NTSYS-PC, numerical taxonomy system for the PC; citation_publication_date=2000; citation_id=CR24; citation_author=FJ Rohlf">
<meta name="citation_reference" content="citation_journal_title=Int J Syst Evol Micr; citation_title=Construction and bootstrap analysis of DNA fingerprinting-based phylogenetic trees with the freeware program FreeTree: application to trichomonad parasites; citation_author=V Hampl, A Pavlícek, J Flegr; citation_volume=51; citation_issue=3; citation_publication_date=2001; citation_pages=731-5; citation_doi=10.1099/00207713-51-3-731; citation_id=CR25">
<meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Inference of population structure using multilocus genotype data; citation_author=JK Pritchard, M Stephens, P Donnelly; citation_volume=155; citation_issue=2; citation_publication_date=2000; citation_pages=945-59; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; citation_author=G Evanno, S Regnaut, J Goudet; citation_volume=14; citation_issue=8; citation_publication_date=2005; citation_pages=2611-20; citation_doi=10.1111/j.1365-294X.2005.02553.x; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Conserv Genet Resour; citation_title=STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method; citation_author=DA Earl; citation_volume=4; citation_issue=2; citation_publication_date=2012; citation_pages=359-61; citation_doi=10.1007/s12686-011-9548-7; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure; citation_author=M Jakobsson, NA Rosenberg; citation_volume=23; citation_issue=14; citation_publication_date=2007; citation_pages=1801-6; citation_doi=10.1093/bioinformatics/btm233; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=SpringerPlus; citation_title=STRUCTURE PLOT: a program for drawing elegant STRUCTURE bar plots in user friendly interface; citation_author=RK Ramasamy, S Ramasamy, BB Bindroo, VG Naik; citation_volume=3; citation_issue=1; citation_publication_date=2014; citation_pages=431; citation_doi=10.1186/2193-1801-3-431; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Biomass Bioenerg; citation_title=Assessment of fatty acid content and genetic diversity in Eruca sativa (L.)(Taramira) using ISSR markers; citation_author=G Garg, V Sharma; citation_volume=76; citation_publication_date=2015; citation_pages=118-29; citation_doi=10.1016/j.biombioe.2015.03.010; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Genetic diversity in seven perennial ryegrass (L.) cultivars based on SSR markers; citation_author=C Kubik, M Sawkins, WA Meyer, BS Gaut; citation_volume=41; citation_issue=5; citation_publication_date=2001; citation_pages=1565-72; citation_doi=10.2135/cropsci2001.4151565x; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Molecular characterisation and interpretation of genetic diversity within globally distributed germplasm collections of tall fescue (Festuca arundinacea Schreb.) and meadow fescue (F. pratensis Huds.); citation_author=ML Hand, NO Cogan, JW Forster; citation_volume=124; citation_issue=6; citation_publication_date=2012; citation_pages=1127-37; citation_doi=10.1007/s00122-011-1774-6; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Am J Bot; citation_title=Population structure in Pseudoroegneria spicata (Poaceae: Triticeae) modeled by Bayesian clustering of AFLP genotypes; citation_author=SR Larson, TA Jones, KB Jensen; citation_volume=91; citation_issue=11; citation_publication_date=2004; citation_pages=1789-801; citation_doi=10.3732/ajb.91.11.1789; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=Plant Syst Evol; citation_title=Genetic diversity and structure among Iranian tall fescue populations based on genomic-SSR and EST-SSR marker analysis; citation_author=MS Tehrani, M Mardi, J Sahebi, P Catalán, A Díaz-Pérez; citation_volume=282; citation_issue=1-2; citation_publication_date=2009; citation_pages=57-70; citation_doi=10.1007/s00606-009-0207-3; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Johnstone tall fescue (Reg. No. 23); citation_author=RE Buckner, JA Boling, PB Burrus, LP Bush, RA Hemken; citation_volume=23; citation_issue=2; citation_publication_date=1983; citation_pages=399-400; citation_doi=10.2135/cropsci1983.0011183X002300020057x; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Classification and pedigree verification of tall fescue cultivars utilizing the prolamin seed protein fraction; citation_author=RH Abernethy, JJ Steiner, DS Wofford, DS Thiel; citation_volume=29; citation_issue=3; citation_publication_date=1989; citation_pages=791-7; citation_doi=10.2135/cropsci1989.0011183X002900030051x; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Kenhy tall fescue (Reg. No. 12); citation_author=RC Buckner, PB Burrus, LP Bush; citation_volume=17; citation_issue=4; citation_publication_date=1977; citation_pages=672-3; citation_doi=10.2135/cropsci1977.0011183X001700040059x; citation_id=CR38">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Kenwell tall fescue (Reg. No. 5); citation_author=RC Buckner, PB Burrus; citation_volume=8; citation_issue=3; citation_publication_date=1968; citation_pages=398; citation_doi=10.2135/cropsci1968.0011183X000800030051x; citation_id=CR39">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Kentucky 31 tall fescue (Reg. No. 7); citation_author=EN Fergus, RC Buckner; citation_volume=12; citation_issue=5; citation_publication_date=1972; citation_pages=714; citation_doi=10.2135/cropsci1972.0011183X001200050061x; citation_id=CR40">
<meta name="citation_reference" content="Hoveland CS. Origin and history of tall fescue. In: Fribourg HA, Hannaway DB, West CP, editors. Tall fescue for the twenty-first century. Madison, WI: American Society of Agronomy, Crop Science Society of America, Soil Science Society of America; 2009. p 3–10.">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Missouri-96 tall fescue (Reg. No. 15); citation_author=KH Asay, DA Sleper; citation_volume=19; citation_issue=4; citation_publication_date=1979; citation_pages=563; citation_doi=10.2135/cropsci1979.0011183X001900040035x; citation_id=CR42">
<meta name="citation_reference" content="citation_journal_title=Agron J; citation_title=Registration of varieties and strains of grasses, I; citation_author=EA Hollowell; citation_volume=37; citation_issue=8; citation_publication_date=1945; citation_pages=653-4; citation_doi=10.2134/agronj1945.00021962003700080012x; citation_id=CR43">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Maximize tall fescue; citation_author=CA Rose-Fricker, WA Meyer, JP Wuk; citation_volume=33; citation_publication_date=1993; citation_pages=203; citation_id=CR44">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Au Triumph tall fescue (Reg. No. 21); citation_author=JF Pedersen, RL Haaland, CS Hoveland, CD Berry, SP Schmidt, RR Harris; citation_volume=23; citation_issue=1; citation_publication_date=1983; citation_pages=182; citation_doi=10.2135/cropsci1983.0011183X002300010062x; citation_id=CR45">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Cajun tall fescue; citation_author=JF Pedersen, KJ McVeigh, CW Edminster; citation_volume=29; citation_issue=1; citation_publication_date=1989; citation_pages=236-7; citation_doi=10.2135/cropsci1989.0011183X002900010057x; citation_id=CR46">
<meta name="citation_reference" content="citation_journal_title=Agron J; citation_title=Animal preference of tall fescue during reproductive growth in the spring; citation_author=E Santen; citation_volume=84; citation_issue=6; citation_publication_date=1992; citation_pages=979-82; citation_doi=10.2134/agronj1992.00021962008400060014x; citation_id=CR47">
<meta name="citation_reference" content="citation_journal_title=Int Turfgrass Soc Res J; citation_title=Genetic diversity of creeping bentgrass cultivars using SSR markers; citation_author=C Kubik, JA Honig, WA Meyer, SA Bonos; citation_volume=11; citation_publication_date=2009; citation_pages=533-47; citation_id=CR48">
<meta name="citation_reference" content="Baluch SJ, Stratton SD, Buker RJ. Registration of Forager tall fescue (Reg. No. 17). Crop Sci. 1980;20(5):670.">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of Nanryo tall fescue; citation_author=B Kindiger, M Gau, H Hasegawa, M Katsura, Y Ueyama, K Goto, S Sato, T Yoshiyama, Y Terada, Y Tsurumi; citation_volume=46; citation_issue=4; citation_publication_date=2006; citation_pages=1815-6; citation_doi=10.2135/cropsci2005.09-0344; citation_id=CR50">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of HiMag tall fescue germplasm; citation_author=DA Sleper, HF Mayland, RJ Crawford, GE Shewmaker, MD Massie; citation_volume=42; citation_publication_date=2002; citation_pages=318-9; citation_doi=10.2135/cropsci2002.0318; citation_id=CR51">
<meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev; citation_title=Genetic variation and relationships of pedigree-known oat, wheat, and barley cultivars releaved by bulking and single-plant sampling; citation_author=Y Fu, WG Legge, JM Clarke, KW Richards; citation_volume=53; citation_issue=6; citation_publication_date=2006; citation_pages=1153-64; citation_doi=10.1007/s10722-005-1306-9; citation_id=CR52">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations; citation_author=RW Michelmore, I Paran, RV Kesseli; citation_volume=88; citation_issue=21; citation_publication_date=1991; citation_pages=9828-32; citation_doi=10.1073/pnas.88.21.9828; citation_id=CR53">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Rapid estimation of genetic relatedness among heterogeneous populations of alfalfa by random amplification of bulked genomic DNA samples; citation_author=K Yu, KP Pauls; citation_volume=86; citation_issue=6; citation_publication_date=1993; citation_pages=788-94; citation_doi=10.1007/BF00222671; citation_id=CR54">
<meta name="citation_reference" content="citation_journal_title=Hortscience; citation_title=RAPD markers from perennial ryegrass DNA extracted from seeds; citation_author=PM Sweeney, TK Danneberger; citation_volume=32; citation_issue=7; citation_publication_date=1997; citation_pages=1212-5; citation_id=CR55">
<meta name="citation_reference" content="citation_journal_title=Sci Agr Sin; citation_title=Comparation of SSR Molecular Markers Analysis of Annual Ryegrass Varieties in DUS Testing; citation_author=T Huang, X Ma, XQ Zhang, XY Zhang, FUKX Zhang RZ; citation_volume=48; citation_issue=2; citation_publication_date=2015; citation_pages=381-9; citation_id=CR56">
<meta name="citation_author" content="Kaixin Fu">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Zhihui Guo">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Xinquan Zhang">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Yan Fan">
<meta name="citation_author_institution" content="Chongqing Municipal Institute of Animal Husbandry, Chongqing, China">
<meta name="citation_author" content="Wendan Wu">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Daxu Li">
<meta name="citation_author_institution" content="Sichuan Academy of Grassland Science, Chengdu, China">
<meta name="citation_author" content="Yan Peng">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Linkai Huang">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Ming Sun">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta name="citation_author" content="Shiqie Bai">
<meta name="citation_author_institution" content="Sichuan Academy of Grassland Science, Chengdu, China">
<meta name="citation_author" content="Xiao Ma">
<meta name="citation_author_institution" content="Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0013-1',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0013-1","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Tall fescue;SSR markers;Genetic diversity;Cultivar identification;Principal component analysis;Neighbor joining;Population structure"},"contentInfo":{"imprint":"BioMed Central","title":"Insight into the genetic variability analysis and cultivar identification of tall fescue by using SSR markers","publishedAt":1470873600000,"publishedAtDate":"2016-08-11","author":["Kaixin Fu","Zhihui Guo","Xinquan Zhang","Yan Fan","Wendan Wu","Daxu Li","Yan Peng","Linkai Huang","Ming Sun","Shiqie Bai","Xiao Ma"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0013-1">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0013-1">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Insight into the genetic variability analysis and cultivar identification of tall fescue by using SSR markers">
<meta property="og:description" content="Genetic diversity of 19 forage-type and 2 turf-type cultivars of tall fescue (Festuca arundinacea Schreb.) was revealed using SSR markers in an attempt to explore the genetic relationships among them, and examine potential use of SSR markers to identify cultivars by bulked samples. A total of 227 clear band was scored with 14 SSR primers and out of which 201 (88.6&amp;nbsp;%) were found polymorphic. The percentage of polymorphic bands (PPB) per primer pair varied from 62.5 to 100&amp;nbsp;% with an average of 86.9&amp;nbsp;%. The polymorphism information content (PIC) value ranged from 0.116 to 0.347 with an average of 0.257 and the highest PIC value (0.347) was noticed for primer NFA040 followed by NFA113 (0.346) whereas the highest discriminating power (D) of 1 was shown in NFA037 and LMgSSR02-01C. A Neighbor-joining dendrogram and the principal component analysis identified six major clusters and grouped the cultivars in agreement with their breeding histories. STRUCTURE analysis divided these cultivars into 3 sub-clades which correspond to distance based groupings. These findings indicates that SSR markers by bulking strategy are a useful tool to measure genetic diversity among tall fescue cultivars and could be used to supplement morphological data for plant variety protection.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0013-1;kwrd=Tall fescue,SSR markers,Genetic diversity,Cultivar identification,Principal component analysis,Neighbor joining,Population structure;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0013-1&amp;kwrd=Tall%20fescue,SSR%20markers,Genetic%20diversity,Cultivar%20identification,Principal%20component%20analysis,Neighbor%20joining,Population%20structure&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0013-1&amp;kwrd=Tall%20fescue,SSR%20markers,Genetic%20diversity,Cultivar%20identification,Principal%20component%20analysis,Neighbor%20joining,Population%20structure&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and discussion">Results and discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abbreviations">Abbreviations</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Insight into the genetic variability analysis and cultivar identification of tall fescue by using SSR markers</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true">
<ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Kaixin Fu</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zhihui Guo</span><span class="EqualContributionSymbol">â </span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xinquan Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yan Fan</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wendan Wu</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Daxu Li</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yan Peng</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Linkai Huang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ming Sun</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Shiqie Bai</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:baiturf@qq.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiao Ma</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:maroar@126.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>
</li>
</ul>
<div class="AffiliationLegend"><div class="AffiliationLegendLine">
<span class="EqualContributionSymbol">â </span>Contributed equally</div></div>
</div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:9</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0013-1">https://doi.org/10.1186/s41065-016-0013-1</a></p>
<p class="Copyright">Â©  The Author(s) 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>1 June 2016</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>27 July 2016</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">11 August 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Genetic diversity of 19 forage-type and 2 turf-type cultivars of tall fescue (<em xmlns="" class="EmphasisTypeItalic">Festuca arundinacea</em> Schreb.) was revealed using SSR markers in an attempt to explore the genetic relationships among them, and examine potential use of SSR markers to identify cultivars by bulked samples.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">A total of 227 clear band was scored with 14 SSR primers and out of which 201 (88.6 %) were found polymorphic. The percentage of polymorphic bands (PPB) per primer pair varied from 62.5 to 100 % with an average of 86.9 %. The polymorphism information content (PIC) value ranged from 0.116 to 0.347 with an average of 0.257 and the highest PIC value (0.347) was noticed for primer NFA040 followed by NFA113 (0.346) whereas the highest discriminating power (D) of 1 was shown in NFA037 and LMgSSR02-01C. A Neighbor-joining dendrogram and the principal component analysis identified six major clusters and grouped the cultivars in agreement with their breeding histories. STRUCTURE analysis divided these cultivars into 3 sub-clades which correspond to distance based groupings.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">These findings indicates that SSR markers by bulking strategy are a useful tool to measure genetic diversity among tall fescue cultivars and could be used to supplement morphological data for plant variety protection.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Tall fescue</li>
<li class="c-keywords__item" itemprop="about">SSR markers</li>
<li class="c-keywords__item" itemprop="about">Genetic diversity</li>
<li class="c-keywords__item" itemprop="about">Cultivar identification</li>
<li class="c-keywords__item" itemprop="about">Principal component analysis</li>
<li class="c-keywords__item" itemprop="about">Neighbor joining</li>
<li class="c-keywords__item" itemprop="about">Population structure</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">
                        <em xmlns="" class="EmphasisTypeItalic">Festuca arundinacea</em> Schreb., commonly known as tall fescue is the predominant cool-season perennial grass that is widely grown throughout the temperate regions of the world [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. It forms the forage basis for beef cow and calf production and also is widely used as turf in lawns, parks, football fields, highway medians, and roadsides [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. It is a cross-pollinated allohexaploid (2n =6Ãâ=â42) with the genomic constitution PPG<sub>1</sub>G<sub>1</sub>G<sub>2</sub>G<sub>2</sub> and a high degree of self-incompatibility [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. These features make breeding efforts generally focus on the development of superior synthetic cultivars or improved heterogeneous populations, both of which consist of genetically unique individuals that share many characters [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Simultaneously, the relative drought tolerance of tall fescue and climatic changes are leading to increased cultivation in the transitional climates between temperate and subtropical zones [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. The demand for forage type cultivars of tall fescue has resulted in the development and release of hundreds of cultivars since 1940s [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">The accurate description of genetic diversity in natural and artificial populations and identification of elite tall fescue cultivars rely on molecular techniques since morphological traits are easily vulnerable to environmental conditions and often showed the limited efficiency of detection of inter-varietal and intra-varietal polymorphisms on account of environmental plasticity [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Molecular marker analysis offers an efficient approach for assessing genetic diversity and germplasm characterization in cross-pollinated grass species [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. In view of cheapness and effectiveness, bulking strategy has been successfully employed to investigate genetic variation between cultivars of allogamous species [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Additionally, a value of 20 genotypes per population seems appropriate for most foragegrass species [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. RFLP and AFLP have been used to distinguish several cultivars of tall fescue by pooled DNA samples [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], but these methods require more technically demanding skills than conventional PCR markers [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Simple sequence repeats (SSR) have become one of the most widely used molecular marker systems in plant genetics due to the co-dominant inheritance, relative abundance, multi-allelic nature, extensive genome coverage, high reproducibility, and simple detection [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. A large number of SSR markers have been developed for the <em xmlns="" class="EmphasisTypeItalic">Lolium</em>/<em xmlns="" class="EmphasisTypeItalic">Festuca</em> complex species, such as tall fescue [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] and <em xmlns="" class="EmphasisTypeItalic">L. multiflorum</em> [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Moreover, it has been reported that SSR markers has showed a high transferability across closely related species [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">To date, the old cultivar Kentucky-31 and its derived cultivars bred in North America have been widely introduced to other parts of the world as a forage base for beef, wool and dairy production [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. However, there is little information about the genetic relationships between these popular cultivars. The objective of the present work, therefore, was (i) to reveal patterns of molecular polymorphism and (ii) to survey the extent of genetic variability and relationships among some elite cultivars of tall fescue from North America and Europe by using SSR markers and bulked samples. Here we wanted to confirm known genetic relationships among these cultivars based on recorded pedigrees and to evaluate the usefulness of SSR markers for germplasm identification.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Plant material and DNA extraction</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Nineteen forage- and two turf-type cultivars of tall fescue (<em xmlns="" class="EmphasisTypeItalic">F. arundinacea</em> Schreb.) were included in this experiment, all of whose seeds were provided by National Plant Germplasm System of USDA. Pedigree or breeding history information of 21 cultivars could be traced, listed in the Table <span class="InternalRef"><a href="#Tab1">1</a></span>. Two of all cultivars, viz., Willamette and Carefree belong to turf-type, while the rest are forage-type. Seed was germinated at 26 Â°C on blotter paper in a lighted growth chamber. Bulked samples were composed by DNA extraction from pooled tissue samples consisting of single leaves from each of the 20 randomly selected plants, for each cultivar. Total genomic DNA was isolated from bulked leaves according to the protocol of using a modified CTAB method [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. The quality and concentration of the extracted DNA were determined by Nano-Drop ND 2000 spectrophotometer (NanoDrop Technologies, Inc.) and 1 % (w/v) agarose gels electrophoresis. The isolated genomic DNA was diluted to 10 ng/Î¼L and stored at â20 Â°C for use.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Description of the plant material analyzed</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Cultivar</p></th>
<th><p xmlns="" class="SimplePara">Country</p></th>
<th><p xmlns="" class="SimplePara">Origin/Pedigree</p></th>
<th><p xmlns="" class="SimplePara">Registration date</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kenhy</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">11 42-chromosome <em class="EmphasisTypeItalic">Lolium multiflorum</em>âÃâ<em class="EmphasisTypeItalic">F. arundinacea</em> clones</p></td>
<td><p xmlns="" class="SimplePara">1977</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Cajun</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">cultivar AuTriumph</p></td>
<td><p xmlns="" class="SimplePara">1989</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Maximize</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">ecotypes from southeast France</p></td>
<td><p xmlns="" class="SimplePara">1993</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kentucky 31</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">ecotype from temperate pasture in Kentucky</p></td>
<td><p xmlns="" class="SimplePara">1972</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kenwell</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">three inbred lines</p></td>
<td><p xmlns="" class="SimplePara">1968</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Alta</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">a 4-year-old plant selection in Oregon.</p></td>
<td><p xmlns="" class="SimplePara">1945</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Fawn</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">temperate 8-clone synthetic</p></td>
<td><p xmlns="" class="SimplePara">1974</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Martin</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">2 clones from broad based population.</p></td>
<td><p xmlns="" class="SimplePara">1987</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Missouri-96</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">13 clones from France germplasm</p></td>
<td><p xmlns="" class="SimplePara">1979</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Forager</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">Kenwell, Fawn, Kentucky 31, etc.</p></td>
<td><p xmlns="" class="SimplePara">1980</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Barcel</p></td>
<td><p xmlns="" class="SimplePara">Netherland</p></td>
<td><p xmlns="" class="SimplePara">13 temperate clones from Netherlands</p></td>
<td><p xmlns="" class="SimplePara">1981</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Johnstone</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">blend of two strains of Kenhy derivatives and 42-chromosome <em class="EmphasisTypeItalic">Lolium</em> sp. Ã <em class="EmphasisTypeItalic">F. arundinacea</em> hybrid clones</p></td>
<td><p xmlns="" class="SimplePara">1983</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Au triumph</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">an open pollinated population AF-5 comprised of 12 genotypess</p></td>
<td><p xmlns="" class="SimplePara">1983</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Willamette</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">an open pollinated progeny of 5 elite parental clones</p></td>
<td><p xmlns="" class="SimplePara">1985</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Safe</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">Kenhy and two germplasm with diseases and drought resistance</p></td>
<td><p xmlns="" class="SimplePara">1985</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Barvetia</p></td>
<td><p xmlns="" class="SimplePara">Netherland</p></td>
<td><p xmlns="" class="SimplePara">temperate 13-clone synthetic</p></td>
<td><p xmlns="" class="SimplePara">unavailable</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mozark</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">Kenmont, Kentucky-31, and introductions from Algeria and France</p></td>
<td><p xmlns="" class="SimplePara">1987</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Penngrazer</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">Kentucky 31</p></td>
<td><p xmlns="" class="SimplePara">1988</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Cattleclub</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">Kentucky 31</p></td>
<td><p xmlns="" class="SimplePara">1988</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Carefree</p></td>
<td><p xmlns="" class="SimplePara">U.S.</p></td>
<td><p xmlns="" class="SimplePara">Houndog, Rutgers, and GPTF accessions.</p></td>
<td><p xmlns="" class="SimplePara">1989</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Nanryo</p></td>
<td><p xmlns="" class="SimplePara">Japan</p></td>
<td><p xmlns="" class="SimplePara">Fawn, Kentucky 31, Rozelle, Electa, Krasnadorskaja and Yamanami</p></td>
<td><p xmlns="" class="SimplePara">2006</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">SSR maker primers and SSR-PCR amplification</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">A total of 15 SSR markers (Table <span class="InternalRef"><a href="#Tab2">2</a></span>) were used to characterize 21 cultivars tall fescue, of which primer LMgSSR02-01C is genic-SSR markers from annual ryegrass [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>], the remaining 14 were EST-SSR markers from tall fescue [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Each 15 Î¼L amplification reaction consisted of 3.0 Î¼L of template DNA (10 ng/Î¼L), 0.6 Î¼L primer (5 pM), 0.3 Î¼L of Taq polymerase (2.5 U/Î¼L), 3 Î¼L of sterile distilled water and 7.5 Î¼L of 2Ã Taq PCR Master Mix (Tiangen Biotech, Beijing, China). Touchdown PCR amplification was carried out under the following conditions: 5 min at 95 Â°C; 16 cycles of 95 Â°C for 50 s, 68 Â°C with 0.5 Â°C decrease per cycle for 50 s until the annealing temperature reached 60 Â°C, and 72 Â°C for 1 min; 19 cycles of 95 Â°C for 50 s, 60 Â°C for 50 s, 72 Â°C for 1 min; followed by a final extension at 72 Â°C for 10 min, and in the end 4 Â°C as a holding temperature. The PCR amplification products were electrophoresed in 8 % non-denaturing polyacrylamide gel at 150 V for 30 min, followed by 2 h at 350 V. Then the gels were visualized by silver staining f and photographed by a gel scanner.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Characteristics of 15 SSR markers selected for use in the study</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Primes</p></th>
<th><p xmlns="" class="SimplePara">motif</p></th>
<th><p xmlns="" class="SimplePara">Size range (bp)</p></th>
<th><p xmlns="" class="SimplePara">NTB</p></th>
<th><p xmlns="" class="SimplePara">NPB</p></th>
<th><p xmlns="" class="SimplePara">PPB (%)</p></th>
<th><p xmlns="" class="SimplePara">PIC</p></th>
<th><p xmlns="" class="SimplePara">I</p></th>
<th><p xmlns="" class="SimplePara">CSM</p></th>
<th><p xmlns="" class="SimplePara">DV</p></th>
<th><p xmlns="" class="SimplePara">D</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">LMgSSR02-01C</p></td>
<td><p xmlns="" class="SimplePara">(aac)<sub xmlns="http://www.w3.org/1999/xhtml">26</sub></p></td>
<td><p xmlns="" class="SimplePara">86â193</p></td>
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">95.8</p></td>
<td><p xmlns="" class="SimplePara">0.252</p></td>
<td><p xmlns="" class="SimplePara">0.403</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">1.000</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA018</p></td>
<td><p xmlns="" class="SimplePara">(cgg)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">136â198</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">90</p></td>
<td><p xmlns="" class="SimplePara">0.267</p></td>
<td><p xmlns="" class="SimplePara">0.411</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">0.952</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA021</p></td>
<td><p xmlns="" class="SimplePara">(cct)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">191â358</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">85.7</p></td>
<td><p xmlns="" class="SimplePara">0.259</p></td>
<td><p xmlns="" class="SimplePara">0.402</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">0.971</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA022</p></td>
<td><p xmlns="" class="SimplePara">(agg)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">180â391</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">86.7</p></td>
<td><p xmlns="" class="SimplePara">0.269</p></td>
<td><p xmlns="" class="SimplePara">0.410</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">0.967</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA035</p></td>
<td><p xmlns="" class="SimplePara">(ggc)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">177â371</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">62.5</p></td>
<td><p xmlns="" class="SimplePara">0.193</p></td>
<td><p xmlns="" class="SimplePara">0.293</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">0.914</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA037</p></td>
<td><p xmlns="" class="SimplePara">(acc)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">177â384</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">96</p></td>
<td><p xmlns="" class="SimplePara">0.279</p></td>
<td><p xmlns="" class="SimplePara">0.434</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">1.000</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA040</p></td>
<td><p xmlns="" class="SimplePara">(cgg)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">233â404</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">100</p></td>
<td><p xmlns="" class="SimplePara">0.347</p></td>
<td><p xmlns="" class="SimplePara">0.518</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">0.986</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA050</p></td>
<td><p xmlns="" class="SimplePara">(cag)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">187â361</p></td>
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">62.5</p></td>
<td><p xmlns="" class="SimplePara">0.116</p></td>
<td><p xmlns="" class="SimplePara">0.205</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">0.267</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA060</p></td>
<td><p xmlns="" class="SimplePara">(tag)<sub xmlns="http://www.w3.org/1999/xhtml">8</sub></p></td>
<td><p xmlns="" class="SimplePara">205â350</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">92.3</p></td>
<td><p xmlns="" class="SimplePara">0.237</p></td>
<td><p xmlns="" class="SimplePara">0.383</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">0.967</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA065</p></td>
<td><p xmlns="" class="SimplePara">(agc)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">200â344</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">100</p></td>
<td><p xmlns="" class="SimplePara">0.298</p></td>
<td><p xmlns="" class="SimplePara">0.461</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">0.995</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA066</p></td>
<td><p xmlns="" class="SimplePara">(cct)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">165â267</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">81.8</p></td>
<td><p xmlns="" class="SimplePara">0.250</p></td>
<td><p xmlns="" class="SimplePara">0.380</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">0.962</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA113</p></td>
<td><p xmlns="" class="SimplePara">(cgg)<sub xmlns="http://www.w3.org/1999/xhtml">6</sub></p></td>
<td><p xmlns="" class="SimplePara">161â357</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">94.7</p></td>
<td><p xmlns="" class="SimplePara">0.346</p></td>
<td><p xmlns="" class="SimplePara">0.514</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">0.995</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA133</p></td>
<td><p xmlns="" class="SimplePara">(cgg)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">162â268</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">93.8</p></td>
<td><p xmlns="" class="SimplePara">0.274</p></td>
<td><p xmlns="" class="SimplePara">0.424</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">0.967</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA150</p></td>
<td><p xmlns="" class="SimplePara">(ctg)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">173â297</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">76.9</p></td>
<td><p xmlns="" class="SimplePara">0.306</p></td>
<td><p xmlns="" class="SimplePara">0.443</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">0.971</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NFA153</p></td>
<td><p xmlns="" class="SimplePara">(gca)<sub xmlns="http://www.w3.org/1999/xhtml">7</sub></p></td>
<td><p xmlns="" class="SimplePara">187â294</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">84.6</p></td>
<td><p xmlns="" class="SimplePara">0.170</p></td>
<td><p xmlns="" class="SimplePara">0.286</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">0.900</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">227</p></td>
<td><p xmlns="" class="SimplePara">201</p></td>
<td><p xmlns="" class="SimplePara">88.6</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td><p xmlns="" class="SimplePara">35</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mean</p></td>
<td><p xmlns="" class="SimplePara">â</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">15.1</p></td>
<td><p xmlns="" class="SimplePara">13.4</p></td>
<td><p xmlns="" class="SimplePara">86.9</p></td>
<td><p xmlns="" class="SimplePara">0.257</p></td>
<td><p xmlns="" class="SimplePara">0.398</p></td>
<td><p xmlns="" class="SimplePara">2.3</p></td>
<td><p xmlns="" class="SimplePara">13.4</p></td>
<td><p xmlns="" class="SimplePara">0.920</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">
                                    <em class="EmphasisTypeItalic">NTB</em> number of total bands, <em class="EmphasisTypeItalic">NPB</em> number of polymorphic bands, <em class="EmphasisTypeItalic">PPB</em> percentage of polymorphic bands, <em class="EmphasisTypeItalic">PIC</em> polymorphic information content, <em class="EmphasisTypeItalic">I</em> Shannonâs information index, <em class="EmphasisTypeItalic">CSM</em> Cultivar-specific markers (presence and absence), <em class="EmphasisTypeItalic">DV</em> Distinguished varieties, <em class="EmphasisTypeItalic">D</em> discriminating power</p>
                    </div></figure>
</div>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Data analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">In spite of co-dominant nature of SSR, there is great difficulty in allele calling by amplified pattern because of the allohexaploid nature of tall fescue and the bulking strategy used in present study. Therefore, unequivocally scorable bands were scored manually as either present (1) and absent (0) to create the binary raw data matrix for further analysis. The total number of bands (TB) and polymorphic bands (PB), polymorphic rate (P), Shannon information index (I) [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], and polymorphism information content (PIC) [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] were calculated as MS Excel 2010, using the formula: PICâ=â1-âP<sup>2</sup><sub>ij</sub>, where P<sub>ij</sub> is the frequency of the <em xmlns="" class="EmphasisTypeItalic">j</em>th allele (marker) for the <em xmlns="" class="EmphasisTypeItalic">i</em>th SSR locus. The distinguished varieties (DV), which were calculated by the number of unique sequences required to identify a particular cultivar from the clustering dendrogram of all 21 cultivars. To compare the efficiency of the markers in varietal identification, the discrimination power (D) [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] was estimated for each primer. This parameter was calculated in accordance with the formula as follows: <span class="InlineEquation" id="IEq1">\( D=1-{\displaystyle {\sum}_{i=1}^I pi}\frac{\left(Npi-1\right)}{N-1} \)</span>, where D is the probability that two randomly selected samples have different and distinct banding patterns, <em xmlns="" class="EmphasisTypeItalic">p</em>
                           <sub><em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub> is the frequency of the <em xmlns="" class="EmphasisTypeItalic">i</em>th pattern revealed by each primer, N is the number of samples analyzed and <em xmlns="" class="EmphasisTypeItalic">I</em> is the total number of patterns generated by each primer [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. The presence/absence data matrix was further used to calculate genetic similarity (GS) between pairs of cultivars according to Diceâs similarity coefficient using NTSYSpc v2.2 [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. Then a genetic distance (GD) matrix developed as GDâ=â1-GS was used to construct dendrograms based on the neighbour-joining (NJ) clustering method with a bootstrapping value of 10,000 replications by the FreeTree software [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. To verify the adjustment between genetic distance matrices and respective dendrogram-derived matrices (cophenetic matrix), the cophenetic correlation coefficient (r) was estimated by NTSYSpc v2.2. As well as the principal coordinate analysis (PCoA) was obtained by the similarity matrix, which was computed from the same program. Further, Bayesian model-based cluster analysis was performed to infer genetic structure and to define the number of clusters in the data set using the software STRUCTURE version 2.3.4 [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. The membership of each cultivars was tested for the range of genetic clusters from Kâ=â1 to 10 with admixture model and without prior information on their origin. Twenty independent runs were assessed for each fixed K and each run consisted of 20,000 burn-in period and 50,000 MCMC iterations. The most likely value of K was determined by examination of the ÎK statistic and L(K) [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] using Structure Harvester [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. A consensus STRUCTURE plot was obtained from the admixture repeats using the greedy algorithm in CLUMPP version 1.1 [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>], and final plots were produced in STRUCTURE PLOT [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Within a subgroup, cultivars with inferred ancestry based on probability values â¥60 % were assigned to a different group, and those with &lt;60 % were treated as âadmixtureâ, i.e., these cultivars seem to have a mixed ancestry from parents belonging to different origins or gene pools.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsanddiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsanddiscussion">Results and discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Resultsanddiscussion" data-test="collapse-section">
<section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Statistical analysis of SSR markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Due to the outbreeding nature of tall fescue, we used bulked samples that represent the mixture of genotypes with a cultivar. In the present work, genetic variability among the 21 tall fescue cultivars was analyzed using a set of SSR primer pairs (PPs), most of which originated from ESTs sequence of tall fescue [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Out of 100 original primer pairs (PPs), 15 selected PPs could generate reproducible and polymorphic patterns among the cultivars, and hence these 15 PPs were retained for further statistical analyses. Totally 227 bands were produced, of which 201 (88.6 %) were polymorphic with an average of 13.4 polymorphic bands per primer (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The number of scorable bands produced per PPs ranged from 8 (NFA050) to 25 (NFA037) with an average of 15.1, and amplicon size varied from 80 to 400 bp. The percentage of polymorphism varied between 62.5 (NFA035 and NFA50) to 100 (NFA40 and NFA65) with an average value of 86.9 %.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">The polymorphism information content (PIC) was in the range from 0.116 to 0.347 with an average value of 0.258 and PIC value was found to be highest with the primer NFA40 (0.347), followed by NFA113 (0.346) and NFA150 (0.306). Nine SSR primers showed more PIC than the average value (0.257). The Shannon index ranged from 0.205 to 0.518 and NFA40 showed highest diversity (0.518), followed by NFA113 (0.514); and average Shannon index was 0.398. Furthermore, the number of the evaluated cultivars distinguished by any SSR marker ranged from 1 to 21, with an average of 13.4 per PPs. The discriminating power (D), a measure of the efficiency of a primer or a locus is an effective method to know the ability of a primer in distinguishing the crop genotypes [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>]. In this study, D value ranged between 0.267 (NFA50) to 1 (NFA37 and LMgSSR02-01C) with an average value of 0.921 (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Here, 14 out of 15 SSR markers showed high discriminatory power (&gt;0.90), indicating that these 15 PPs used in this study were highly effective in tall fescue cultivar characterization. Primer NFA37 and LMgSSR02-01C, which showed the highest D value being one, was found to be the best SSR markers for detecting polymorphism in tall fescue. Although only 35 cultivar-specific markers were observed for 15 primers, all of 21 cultivars had their own unique SSR banding patterns individually, which could be used to distinguish cultivars by characteristic sets of bands.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Diceâs distance coefficient and hierarchical clustering</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Two hundred and twenty-seven fragments from 15 SSR loci were used to estimate pairwise Diceâs distances among 21 tall fescue cultivars. The Diceâs coefficient was moderately high, varying from 0.163 (Forager vs. Fawn) to 0.475 (AuTriumph vs. Barvetia), with an average of 0.320 (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). These low values were similar to those obtained from studies about analyzing tall fescue cultivars and/or populations by RFLP [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] and AFLP [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], which were less than 0.50 among cultivars. The less genetic distance indicated a relatively low genetic diversity or a closer relationship among studied cultivars although some visible differences obviously exist between them. This is not surprising considering the breeding histories of these cultivars (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Some popular cultivars such as Kentucky 31, often served as common parental germplasm to intercross with other limited germplasm in breeding program of many tall fescue cultivars [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Therefore, the similar pedigree or germplasm sources could account for the high genetic similarity and close relationship among cultivars evaluated. Likewise, seven perennial ryegrass cultivars representing a broad germplasm sample also showed high levels of genetic similarity using SSR markers [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. Besides that, this low level of genetic diversity could be due in part, to the tall fescueâs allogamy and self-incompatibility [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>], as well as to its hexaploid composition [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. Typically, the cross-pollinated species maintain relatively high intra-population variability as compared to its inter-population variability [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The Analysis of molecular variance (AMOVA) indicated that most (92.3 %) of the molecular variation in wild Iranian tall fescue populations exists among individuals within populations, with lesser amounts among populations (7.7 %) [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. As a result, the enormous intra-cultivar genetic variability led to the high genetic similarity among tall fescue cultivars.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Diceâs distance matrix of 21 cultivars of tall fescue based on SSR profile</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th> </th>
<th><p xmlns="" class="SimplePara">1</p></th>
<th><p xmlns="" class="SimplePara">2</p></th>
<th><p xmlns="" class="SimplePara">3</p></th>
<th><p xmlns="" class="SimplePara">4</p></th>
<th><p xmlns="" class="SimplePara">5</p></th>
<th><p xmlns="" class="SimplePara">6</p></th>
<th><p xmlns="" class="SimplePara">7</p></th>
<th><p xmlns="" class="SimplePara">8</p></th>
<th><p xmlns="" class="SimplePara">9</p></th>
<th><p xmlns="" class="SimplePara">10</p></th>
<th><p xmlns="" class="SimplePara">11</p></th>
<th><p xmlns="" class="SimplePara">12</p></th>
<th><p xmlns="" class="SimplePara">13</p></th>
<th><p xmlns="" class="SimplePara">14</p></th>
<th><p xmlns="" class="SimplePara">15</p></th>
<th><p xmlns="" class="SimplePara">16</p></th>
<th><p xmlns="" class="SimplePara">17</p></th>
<th><p xmlns="" class="SimplePara">18</p></th>
<th><p xmlns="" class="SimplePara">19</p></th>
<th><p xmlns="" class="SimplePara">20</p></th>
<th><p xmlns="" class="SimplePara">21</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kenhy</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Cajun</p></td>
<td><p xmlns="" class="SimplePara">0.336</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Maximize</p></td>
<td><p xmlns="" class="SimplePara">0.313</p></td>
<td><p xmlns="" class="SimplePara">0.295</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kentucky31</p></td>
<td><p xmlns="" class="SimplePara">0.291</p></td>
<td><p xmlns="" class="SimplePara">0.302</p></td>
<td><p xmlns="" class="SimplePara">0.352</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kenwell</p></td>
<td><p xmlns="" class="SimplePara">0.289</p></td>
<td><p xmlns="" class="SimplePara">0.292</p></td>
<td><p xmlns="" class="SimplePara">0.304</p></td>
<td><p xmlns="" class="SimplePara">0.237</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Alta</p></td>
<td><p xmlns="" class="SimplePara">0.294</p></td>
<td><p xmlns="" class="SimplePara">0.245</p></td>
<td><p xmlns="" class="SimplePara">0.243</p></td>
<td><p xmlns="" class="SimplePara">0.306</p></td>
<td><p xmlns="" class="SimplePara">0.268</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Fawn</p></td>
<td><p xmlns="" class="SimplePara">0.345</p></td>
<td><p xmlns="" class="SimplePara">0.291</p></td>
<td><p xmlns="" class="SimplePara">0.326</p></td>
<td><p xmlns="" class="SimplePara">0.354</p></td>
<td><p xmlns="" class="SimplePara">0.311</p></td>
<td><p xmlns="" class="SimplePara">0.253</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Martin</p></td>
<td><p xmlns="" class="SimplePara">0.415</p></td>
<td><p xmlns="" class="SimplePara">0.418</p></td>
<td><p xmlns="" class="SimplePara">0.322</p></td>
<td><p xmlns="" class="SimplePara">0.384</p></td>
<td><p xmlns="" class="SimplePara">0.364</p></td>
<td><p xmlns="" class="SimplePara">0.322</p></td>
<td><p xmlns="" class="SimplePara">0.307</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Missouri96</p></td>
<td><p xmlns="" class="SimplePara">0.313</p></td>
<td><p xmlns="" class="SimplePara">0.295</p></td>
<td><p xmlns="" class="SimplePara">0.327</p></td>
<td><p xmlns="" class="SimplePara">0.192</p></td>
<td><p xmlns="" class="SimplePara">0.294</p></td>
<td><p xmlns="" class="SimplePara">0.289</p></td>
<td><p xmlns="" class="SimplePara">0.348</p></td>
<td><p xmlns="" class="SimplePara">0.379</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Forager</p></td>
<td><p xmlns="" class="SimplePara">0.366</p></td>
<td><p xmlns="" class="SimplePara">0.293</p></td>
<td><p xmlns="" class="SimplePara">0.316</p></td>
<td><p xmlns="" class="SimplePara">0.343</p></td>
<td><p xmlns="" class="SimplePara">0.313</p></td>
<td><p xmlns="" class="SimplePara">0.257</p></td>
<td><p xmlns="" class="SimplePara">0.163</p></td>
<td><p xmlns="" class="SimplePara">0.285</p></td>
<td><p xmlns="" class="SimplePara">0.306</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Barcel</p></td>
<td><p xmlns="" class="SimplePara">0.372</p></td>
<td><p xmlns="" class="SimplePara">0.382</p></td>
<td><p xmlns="" class="SimplePara">0.330</p></td>
<td><p xmlns="" class="SimplePara">0.347</p></td>
<td><p xmlns="" class="SimplePara">0.359</p></td>
<td><p xmlns="" class="SimplePara">0.298</p></td>
<td><p xmlns="" class="SimplePara">0.317</p></td>
<td><p xmlns="" class="SimplePara">0.364</p></td>
<td><p xmlns="" class="SimplePara">0.286</p></td>
<td><p xmlns="" class="SimplePara">0.329</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Johnstone</p></td>
<td><p xmlns="" class="SimplePara">0.219</p></td>
<td><p xmlns="" class="SimplePara">0.370</p></td>
<td><p xmlns="" class="SimplePara">0.340</p></td>
<td><p xmlns="" class="SimplePara">0.288</p></td>
<td><p xmlns="" class="SimplePara">0.256</p></td>
<td><p xmlns="" class="SimplePara">0.272</p></td>
<td><p xmlns="" class="SimplePara">0.341</p></td>
<td><p xmlns="" class="SimplePara">0.361</p></td>
<td><p xmlns="" class="SimplePara">0.310</p></td>
<td><p xmlns="" class="SimplePara">0.340</p></td>
<td><p xmlns="" class="SimplePara">0.322</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">AuTriumph</p></td>
<td><p xmlns="" class="SimplePara">0.429</p></td>
<td><p xmlns="" class="SimplePara">0.254</p></td>
<td><p xmlns="" class="SimplePara">0.292</p></td>
<td><p xmlns="" class="SimplePara">0.355</p></td>
<td><p xmlns="" class="SimplePara">0.356</p></td>
<td><p xmlns="" class="SimplePara">0.293</p></td>
<td><p xmlns="" class="SimplePara">0.297</p></td>
<td><p xmlns="" class="SimplePara">0.348</p></td>
<td><p xmlns="" class="SimplePara">0.379</p></td>
<td><p xmlns="" class="SimplePara">0.218</p></td>
<td><p xmlns="" class="SimplePara">0.301</p></td>
<td><p xmlns="" class="SimplePara">0.403</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Willamette</p></td>
<td><p xmlns="" class="SimplePara">0.280</p></td>
<td><p xmlns="" class="SimplePara">0.312</p></td>
<td><p xmlns="" class="SimplePara">0.296</p></td>
<td><p xmlns="" class="SimplePara">0.275</p></td>
<td><p xmlns="" class="SimplePara">0.303</p></td>
<td><p xmlns="" class="SimplePara">0.268</p></td>
<td><p xmlns="" class="SimplePara">0.326</p></td>
<td><p xmlns="" class="SimplePara">0.393</p></td>
<td><p xmlns="" class="SimplePara">0.265</p></td>
<td><p xmlns="" class="SimplePara">0.326</p></td>
<td><p xmlns="" class="SimplePara">0.273</p></td>
<td><p xmlns="" class="SimplePara">0.288</p></td>
<td><p xmlns="" class="SimplePara">0.370</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Safe</p></td>
<td><p xmlns="" class="SimplePara">0.262</p></td>
<td><p xmlns="" class="SimplePara">0.330</p></td>
<td><p xmlns="" class="SimplePara">0.385</p></td>
<td><p xmlns="" class="SimplePara">0.194</p></td>
<td><p xmlns="" class="SimplePara">0.217</p></td>
<td><p xmlns="" class="SimplePara">0.327</p></td>
<td><p xmlns="" class="SimplePara">0.348</p></td>
<td><p xmlns="" class="SimplePara">0.367</p></td>
<td><p xmlns="" class="SimplePara">0.268</p></td>
<td><p xmlns="" class="SimplePara">0.337</p></td>
<td><p xmlns="" class="SimplePara">0.373</p></td>
<td><p xmlns="" class="SimplePara">0.230</p></td>
<td><p xmlns="" class="SimplePara">0.388</p></td>
<td><p xmlns="" class="SimplePara">0.317</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Barvetia</p></td>
<td><p xmlns="" class="SimplePara">0.345</p></td>
<td><p xmlns="" class="SimplePara">0.431</p></td>
<td><p xmlns="" class="SimplePara">0.449</p></td>
<td><p xmlns="" class="SimplePara">0.387</p></td>
<td><p xmlns="" class="SimplePara">0.367</p></td>
<td><p xmlns="" class="SimplePara">0.380</p></td>
<td><p xmlns="" class="SimplePara">0.377</p></td>
<td><p xmlns="" class="SimplePara">0.361</p></td>
<td><p xmlns="" class="SimplePara">0.440</p></td>
<td><p xmlns="" class="SimplePara">0.398</p></td>
<td><p xmlns="" class="SimplePara">0.406</p></td>
<td><p xmlns="" class="SimplePara">0.365</p></td>
<td><p xmlns="" class="SimplePara">0.475</p></td>
<td><p xmlns="" class="SimplePara">0.408</p></td>
<td><p xmlns="" class="SimplePara">0.360</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mozark</p></td>
<td><p xmlns="" class="SimplePara">0.337</p></td>
<td><p xmlns="" class="SimplePara">0.248</p></td>
<td><p xmlns="" class="SimplePara">0.340</p></td>
<td><p xmlns="" class="SimplePara">0.265</p></td>
<td><p xmlns="" class="SimplePara">0.271</p></td>
<td><p xmlns="" class="SimplePara">0.294</p></td>
<td><p xmlns="" class="SimplePara">0.319</p></td>
<td><p xmlns="" class="SimplePara">0.326</p></td>
<td><p xmlns="" class="SimplePara">0.321</p></td>
<td><p xmlns="" class="SimplePara">0.271</p></td>
<td><p xmlns="" class="SimplePara">0.354</p></td>
<td><p xmlns="" class="SimplePara">0.333</p></td>
<td><p xmlns="" class="SimplePara">0.306</p></td>
<td><p xmlns="" class="SimplePara">0.359</p></td>
<td><p xmlns="" class="SimplePara">0.312</p></td>
<td><p xmlns="" class="SimplePara">0.405</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Penngrazer</p></td>
<td><p xmlns="" class="SimplePara">0.320</p></td>
<td><p xmlns="" class="SimplePara">0.257</p></td>
<td><p xmlns="" class="SimplePara">0.324</p></td>
<td><p xmlns="" class="SimplePara">0.265</p></td>
<td><p xmlns="" class="SimplePara">0.272</p></td>
<td><p xmlns="" class="SimplePara">0.305</p></td>
<td><p xmlns="" class="SimplePara">0.290</p></td>
<td><p xmlns="" class="SimplePara">0.341</p></td>
<td><p xmlns="" class="SimplePara">0.294</p></td>
<td><p xmlns="" class="SimplePara">0.282</p></td>
<td><p xmlns="" class="SimplePara">0.380</p></td>
<td><p xmlns="" class="SimplePara">0.327</p></td>
<td><p xmlns="" class="SimplePara">0.317</p></td>
<td><p xmlns="" class="SimplePara">0.333</p></td>
<td><p xmlns="" class="SimplePara">0.285</p></td>
<td><p xmlns="" class="SimplePara">0.390</p></td>
<td><p xmlns="" class="SimplePara">0.243</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Cattleclub</p></td>
<td><p xmlns="" class="SimplePara">0.337</p></td>
<td><p xmlns="" class="SimplePara">0.350</p></td>
<td><p xmlns="" class="SimplePara">0.330</p></td>
<td><p xmlns="" class="SimplePara">0.188</p></td>
<td><p xmlns="" class="SimplePara">0.245</p></td>
<td><p xmlns="" class="SimplePara">0.260</p></td>
<td><p xmlns="" class="SimplePara">0.360</p></td>
<td><p xmlns="" class="SimplePara">0.358</p></td>
<td><p xmlns="" class="SimplePara">0.248</p></td>
<td><p xmlns="" class="SimplePara">0.330</p></td>
<td><p xmlns="" class="SimplePara">0.303</p></td>
<td><p xmlns="" class="SimplePara">0.277</p></td>
<td><p xmlns="" class="SimplePara">0.360</p></td>
<td><p xmlns="" class="SimplePara">0.292</p></td>
<td><p xmlns="" class="SimplePara">0.240</p></td>
<td><p xmlns="" class="SimplePara">0.445</p></td>
<td><p xmlns="" class="SimplePara">0.281</p></td>
<td><p xmlns="" class="SimplePara">0.273</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Carefree</p></td>
<td><p xmlns="" class="SimplePara">0.356</p></td>
<td><p xmlns="" class="SimplePara">0.316</p></td>
<td><p xmlns="" class="SimplePara">0.339</p></td>
<td><p xmlns="" class="SimplePara">0.265</p></td>
<td><p xmlns="" class="SimplePara">0.327</p></td>
<td><p xmlns="" class="SimplePara">0.313</p></td>
<td><p xmlns="" class="SimplePara">0.320</p></td>
<td><p xmlns="" class="SimplePara">0.422</p></td>
<td><p xmlns="" class="SimplePara">0.274</p></td>
<td><p xmlns="" class="SimplePara">0.340</p></td>
<td><p xmlns="" class="SimplePara">0.395</p></td>
<td><p xmlns="" class="SimplePara">0.371</p></td>
<td><p xmlns="" class="SimplePara">0.370</p></td>
<td><p xmlns="" class="SimplePara">0.321</p></td>
<td><p xmlns="" class="SimplePara">0.376</p></td>
<td><p xmlns="" class="SimplePara">0.447</p></td>
<td><p xmlns="" class="SimplePara">0.307</p></td>
<td><p xmlns="" class="SimplePara">0.300</p></td>
<td><p xmlns="" class="SimplePara">0.333</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Nanryo</p></td>
<td><p xmlns="" class="SimplePara">0.345</p></td>
<td><p xmlns="" class="SimplePara">0.330</p></td>
<td><p xmlns="" class="SimplePara">0.294</p></td>
<td><p xmlns="" class="SimplePara">0.344</p></td>
<td><p xmlns="" class="SimplePara">0.344</p></td>
<td><p xmlns="" class="SimplePara">0.295</p></td>
<td><p xmlns="" class="SimplePara">0.190</p></td>
<td><p xmlns="" class="SimplePara">0.256</p></td>
<td><p xmlns="" class="SimplePara">0.326</p></td>
<td><p xmlns="" class="SimplePara">0.200</p></td>
<td><p xmlns="" class="SimplePara">0.317</p></td>
<td><p xmlns="" class="SimplePara">0.352</p></td>
<td><p xmlns="" class="SimplePara">0.277</p></td>
<td><p xmlns="" class="SimplePara">0.326</p></td>
<td><p xmlns="" class="SimplePara">0.358</p></td>
<td><p xmlns="" class="SimplePara">0.389</p></td>
<td><p xmlns="" class="SimplePara">0.289</p></td>
<td><p xmlns="" class="SimplePara">0.301</p></td>
<td><p xmlns="" class="SimplePara">0.330</p></td>
<td><p xmlns="" class="SimplePara">0.289</p></td>
<td><p xmlns="" class="SimplePara">0</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">A dendrogram was generated from the NJ cluster analysis of SSR data (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). This dendrogram divided the 21 culitvars into six major clades in general agreement with their breeding origins with a few exceptions. A high correlation (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.79, <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.01) between the cophenetic matrix and the original matrix revealed a high goodness of fit for the cultivars clustering. Clade 1 consisted of three cultivars, Kenhy, Johnstone and Kenwell. Here, both Kenhy and Johnstone origined from derivatives of 11 42-chromosome <em xmlns="" class="EmphasisTypeItalic">Lolium</em>âÃâ<em xmlns="" class="EmphasisTypeItalic">Festuca</em> hybrid clones at Kentucky Agricultural Experiment Station (KAES) of USA [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>â<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. Kenwell was also developed from three inbred lines at KAES and may provide share some common pollen sources with Kenhy and Johnstone [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>], therefore it is not surprising that it was also included in Clade 1. Clade 2 was composed of Kentucky 31, Safe, Cattleclub, Missouri 96 and Carefree, whose clustering placement could be chiefly attributed to their pedigree origin. Kentucky 31 originated from a population that had undergone approximately 43 years of natural selection on a farm in Kentucky State of USA [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>, <span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. And Cattleclub originated from several old Kentucky 31 seed fields by phenotypic selection [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. However, Missouri-96 was derived from plant introductions from France [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. It was used as a pollen source in the development of Houndog, one of the first improved turf-type cultivar [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Moreover, six parental clones from Houndog was used to ploycross with other germplasm in the development of Carefree, a turf-type synthetic [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>], which is the most dissimilar from the others found in Clade 2. Yet it is hard to explain the low distance of 0.194 between Safe and Kentucky 31, since Safe was derived from Kenhy existing in Clade 1 [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. Clade 3 was randomly made up of Barcel and Willamette supported by a bootstrap value of 47 %, indicating that these two cultivars are distinctly different from the other cultivars in the study. Barcel derived from old pastures in the Netherlands [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>], but there is little information available concerning the parentage of Willamette other than it may have some common germplasm with cultivar Houndog [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Clade 4 contained two cultivars Alta and Maximize with a high pairwise similarity of 0.76, while it is hard to give a clear elucidation by their pedigree history. Alta was developed from a plant selection in Oregon [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>], and Maximize was resulted from French ecotypes [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. Clade 5 included four cultivars, viz., AuTriumph, Cajun, Mozark and Penngrazer. By and large, this clade came together as predicted by the breeding histories for these cultivars. AuTriumph was developed from 12 introduction genotypes and Cajun derived from AuTriumph [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>, <span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>], while Penngrazer could trace to Kentucky 31, one of the four parent clones of Mozark was also Kentucky 31 [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>]. However, Penngrazer and Mozark failed to group tighter with the Kentucky 31 in the Clade 2. The reason for that probably is that the nature of outcrossing and multiple mass selections resulted in an unequal contribution of parent plants in the final genetic makeup of a cultivar [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>]. Clade 6 including five cultivars could be divided into sub-clades with three and one cultivar respectively. Here Forager and Nanryo sharing one of the common parent germplasm Fawn resulted to a firm genetic relationship with the high bootstrap values [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>, <span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>]. Additionally, Barvetia and Martin formed a sub-clade with a middle bootstrap value of 42 %, but this could not correspond to their pedigree information, since Barvetia was developed in Netherland and Martin was developed in Missouri university of USA [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>, <span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>]. Yet it was worth noting that most of NJ clusters showed lower bootstrap values below the 50 %, which suggested that more primers may be needed to identify genetic relationship between cultivars surveyed. In particular, the detailed pedigree information of some commercial or patented cultivars is desperately needed to explain the genetic relationship among different cultivars. In short, these results illustrated that SSR markers were effective in surveying the affinities among tall fescue cultivars studied.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">NJ dendrogram of 21 tall fescue cultivars based on Diceâs coefficient and Model-based Bayesian clustering performed in STRUCTURE for Kâ=â3 populations. Each color represents one population and length of colored segment shows estimated membership proportion of each cultivars. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article)</p>
                      </div></figcaption></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">The principal coordinate analysis (PCoA) was done to see the displacement of the cultivars and to further confirm the clustering pattern obtained from the dendrogram (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>). The first three eigenvectors accounted for 32.3 % of total variation among all the cultivars. The first and second axes represented 13.7 and 10.1 % of the variation respectively. The affinities produced by PCoA are generally in agreement with the NJ cluster and Diceâs dissimilarity coefficients analyses. Some cultivars shared common parental resources were closely dispersed in the PCoA plot, as Nanryo, Forager and Fawn showed closely related each other on NJ dendrogram.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0013-1/MediaObjects/41065_2016_13_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Principal coordinates analysis (PCoA) scatterplot of 21 tall fescue cultivars based on the first two principal coordinates</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec9" class="Section2 RenderAsSection2"><h3 class="Heading">Population structure</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The pattern of population structure was further analyzed with a Model based Bayesian approach implemented in Structure program. We investigated the range from Kâ=â1 to Kâ=â10 and calculated the posterior probability for each value of K using the estimated log likelihood of K. The number of clusters best fitting the data was Kâ=â3 as indicated by the modal value of ÎK. As a result, the studied cultivars were successfully assigned to three subgroups with slight mixing (Kâ=â3, ÎKâ=â65.37) as represented in Fig. <span class="InternalRef"><a href="#Fig1">1</a></span> and Table <span class="InternalRef"><a href="#Tab3">3</a></span>; inferred clusters were calculated with more than 60 % probability intervals. In general, the clustering based on Bayesian statistics was found to be in consistency with distance based NJ clustering, whereas there were deviations and fragmentation of genetic clusters because of admixed cultivars such as Alta, Barvetia and Barcel. A second analysis of each cultivar assignment probabilities for the model Kâ=â3 shows that cluster 1 (red) is corresponding to the Clade 6 and Barcel in Clade 3 in NJ dendrogram. The cluster 2 (blue) includes Clade 4 and 5, and Carefree in Clade 2 of NJ tree. Finally, cluster 3 (green) consists of Clade 1, remaining entries except Carefree and Willamette in Clade 3. Moreover, the affinities produced by PCoA are generally in agreement with the results of the STRUCTURE analysis, since the PCoA grouped the cultivars into three clusters strongly differentiated, which correspond substantially to three inferred sub-clusters from STRUCTURE analysis (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). Itâs worth noting that cultivars Alta, Barvetia and Barcel showed the admixed membership based on Q-matrix values, nor did they reflected clear membership in any of the groups identified in the PCoA. This might be due to the fact that extensively used poly-intercross among genotypes from registered varieties or exchanged germplasm in breeding programs led to disturbed Hardy-Weinberg equilibrium (HWE) and genetic admixture in the synthetic cultivar populations [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">In this study, due to using a DNA bulking strategy, it was possible to survey the genetic variation among the 21 tall fescue cultivars regardless of intra-cultivar variability. Assaying bulked samples not only drastically reduces the number of individual samples that need to be processed [<span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>], but also results in a preferential elimination of rare alleles by dilution in larger bulk samples [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>] and therefore simplifies the marker profile of an individual cultivar or accession. In previous studies, various molecular markers have been used to determine diversity among heterogeneous cultivars populations of outcrossing fodder species based on bulked DNA samples [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>â<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. Likewise, the 21 tall fescue cultivars used in this study were easily characterized using SSR markers. The large number of molecular markers produced in this study showed that SSR patterns based on bulked DNA samples were found to be a fast, reliable, and highly efficient method to analyze genetic relationships among heterogeneous cultivars of tall fescue. These results will help breeders choose cultivars more genetically distant to be used in crosses in order to obtain transgressive segregation of some agronomic trait in the hybrid descendant populations.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">In conclusion, SSR patterns based on bulked DNA samples were found to be a good method of choice to evaluate the extent of genetic variability and relationships among 21 heterogeneous tall fescue cultivars. Even though low levels of diversity were detected here, most of SSR markers showed extremely high discriminatory power for evaluated cultivars. SSR analysis is therefore a powerful tool for distinguishing synthetic cultivars as well as assessing genetic relationships among cultivars in outcrossing grass species.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Abbreviations" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">AFLP, amplified fragment length polymorphism; AMOVA, analysis of molecular variance; CTAB, hexadecyl trimethyl ammonium bromide; D, discriminating power; DV, distinguished varieties; GD, genetic distance; GS, genetic similarity; I, Shannon information index; NJ, neighbour-joining; P, polymorphic rate; PB, polymorphic bands; PCoA, principal coordinate analysis; PCR, polymerase chain reaction; PIC, polymorphism information content; PIC, the polymorphism information content; PPB, the percentage of polymorphic bands; PPs, primer pairs; RFLP, restriction fragment length polymorphism; SSR, simple sequence repeats; TB, total number of bands</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgments</h3>
<p xmlns="" class="SimplePara">We thank all the participants from the Department of Grassland Science at Sichuan Agricultural University for their contributions to this study. We are also grateful to the U.S. National Plant Germplasm System (NPGS) for providing most of seed materials. This work was funded by National Key Basic Research Program of China (#2014CB138705), the National Science and Technology Supporting Project (#2012BAD13B06) and the Earmarked Fund for the Modern Agro-Industry Technology Research System (#CARS-35-05). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authorsâ contributions</h3>
                <p id="Par19" class="Para">XM and S-QB conceived the project and designed the experiments. K-XF, Z-HG, W-DW and MS performed the experiment. K-XF, Z-HG, and XM wrote the paper. X-QZ, YF, D-XL, YP and L-KH gave comments on the manuscript. XM and S-QB revised the paper. All authors discussed the results and commented on the manuscript. All authors read and approved the final manuscript.</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par20" class="Para">The authors declare that they have no competing interests.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Animal Science and Technology College, Sichuan Agricultural University, Chengdu, 611130, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Chongqing Municipal Institute of Animal Husbandry, Chongqing, 400039, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Sichuan Academy of Grassland Science, Chengdu, 611731, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sleper DA. Breeding tall fescue. Plant Breeding Rev. 1985;3:313â42.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Breeding%20tall%20fescue&amp;author=DA.%20Sleper&amp;journal=Plant%20Breeding%20Rev&amp;volume=3&amp;pages=313-42&amp;publication_year=1985"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sleper DA, Buckner RC. The fescues. In: Barnes RF, Miller DA, Nelson CJ, Heath ME, editors. Forages, vol. 1. Ames: Iowa State University Press; 1995. p. 345â56.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20fescues&amp;author=DA.%20Sleper&amp;author=RC.%20Buckner&amp;pages=345-56&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Humphreys MW, Thomas HM, Morgan WG, Meredith MR, Harper JA, Thomas H, Zwierzkowski Z, Ghesquiere M. Discriminating the ancestral progenitors of hexaploid <em class="EmphasisTypeItalic">Festuca arundinacea</em> using genomic in situ hybridization. Heredity. 1995;75(2):171â4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/hdy.1995.120"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Discriminating%20the%20ancestral%20progenitors%20of%20hexaploid%20Festuca%20arundinacea%20using%20genomic%20in%20situ%20hybridization&amp;author=MW.%20Humphreys&amp;author=HM.%20Thomas&amp;author=WG.%20Morgan&amp;author=MR.%20Meredith&amp;author=JA.%20Harper&amp;author=H.%20Thomas&amp;author=Z.%20Zwierzkowski&amp;author=M.%20Ghesquiere&amp;journal=Heredity&amp;volume=75&amp;issue=2&amp;pages=171-4&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu WW, Sleper DA, Krause GF. Genetic diversity of tall fescue germplasm based on RFLPs. Crop Sci. 1994;34(1):246â52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1994.0011183X003400010045x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20tall%20fescue%20germplasm%20based%20on%20RFLPs&amp;author=WW.%20Xu&amp;author=DA.%20Sleper&amp;author=GF.%20Krause&amp;journal=Crop%20Sci&amp;volume=34&amp;issue=1&amp;pages=246-52&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Majidi MM, Mirlohi A, Amini F. Genetic variation, heritability and correlations of agro-morphological traits in tall fescue (<em class="EmphasisTypeItalic">Festuca arundinacea</em> Schreb.). Euphytica. 2009;167(3):323â31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-009-9887-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variation%2C%20heritability%20and%20correlations%20of%20agro-morphological%20traits%20in%20tall%20fescue%20%28Festuca%20arundinacea%20Schreb.%29&amp;author=MM.%20Majidi&amp;author=A.%20Mirlohi&amp;author=F.%20Amini&amp;journal=Euphytica&amp;volume=167&amp;issue=3&amp;pages=323-31&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Golembiewski RC, Danneberger TK, Sweeney PM. Potential of RAPD markers for use in the identification of creeping bentgrass cultivars. Crop Sci. 1997;37(1):212â4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1997.0011183X003700010036x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Potential%20of%20RAPD%20markers%20for%20use%20in%20the%20identification%20of%20creeping%20bentgrass%20cultivars&amp;author=RC.%20Golembiewski&amp;author=TK.%20Danneberger&amp;author=PM.%20Sweeney&amp;journal=Crop%20Sci&amp;volume=37&amp;issue=1&amp;pages=212-4&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rognli OA, Saha MC, Bhamidimarri S, van der Heijden S. Fescues. In: Boller B, Posselt UK, Veronesi F, editors. Fodder crops and amenity grasses, handbook of plant breeding 5. New York: Springer; 2010. p. 261â92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-1-4419-0760-8_11"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fescues&amp;author=OA.%20Rognli&amp;author=MC.%20Saha&amp;author=S.%20Bhamidimarri&amp;author=S.%20Heijden&amp;pages=261-92&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Busti A, Caceres ME, Calderini O, Arcioni S, Pupilli F. RFLP markers for cultivar identification in tall fescue (<em class="EmphasisTypeItalic">Festuca arundinacea</em> Schreb.). Genet Resour Crop Ev. 2004;51(4):443â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/B%3AGRES.0000023459.62679.e8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RFLP%20markers%20for%20cultivar%20identification%20in%20tall%20fescue%20%28Festuca%20arundinacea%20Schreb.%29&amp;author=A.%20Busti&amp;author=ME.%20Caceres&amp;author=O.%20Calderini&amp;author=S.%20Arcioni&amp;author=F.%20Pupilli&amp;journal=Genet%20Resour%20Crop%20Ev&amp;volume=51&amp;issue=4&amp;pages=443-8&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huff DR, Peakall R, Smouse PE. RAPD variation within and among natural populations of outcrossing buffalograss [Buchloe dactyloides (Nutt.) Engelm.]. Theor Appl Genet. 1993;86(8):927â34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00211043"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24193999"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RAPD%20variation%20within%20and%20among%20natural%20populations%20of%20outcrossing%20buffalograss%20%5BBuchloe%20dactyloides%20%28Nutt.%29%20Engelm.%5D&amp;author=DR.%20Huff&amp;author=R.%20Peakall&amp;author=PE.%20Smouse&amp;journal=Theor%20Appl%20Genet&amp;volume=86&amp;issue=8&amp;pages=927-34&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Reyes-ValdÃ©s MH, Santacruz-Varela A, MartÃ­nez O, Simpson J, Hayano-Kanashiro C, CortÃ©s-Romero C. Analysis and optimization of bulk DNA sampling with binary scoring for germplasm characterization. 2013.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20and%20optimization%20of%20bulk%20DNA%20sampling%20with%20binary%20scoring%20for%20germplasm%20characterization&amp;author=MH.%20Reyes-Vald%C3%A9s&amp;author=A.%20Santacruz-Varela&amp;author=O.%20Mart%C3%ADnez&amp;author=J.%20Simpson&amp;author=C.%20Hayano-Kanashiro&amp;author=C.%20Cort%C3%A9s-Romero&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang LF, Zhang XQ, Ma X, Huang LK, Xie WG, Ma YM, Zhao YF. Identification of orchardgrass (<em class="EmphasisTypeItalic">Dactylis glomerata</em> L.) cultivars by using simple sequence repeat markers. Genet Mol Res. 2013;12(4):5111â23.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2013.October.29.5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24301771"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20orchardgrass%20%28Dactylis%20glomerata%20L.%29%20cultivars%20by%20using%20simple%20sequence%20repeat%20markers&amp;author=LF.%20Jiang&amp;author=XQ.%20Zhang&amp;author=X.%20Ma&amp;author=LK.%20Huang&amp;author=WG.%20Xie&amp;author=YM.%20Ma&amp;author=YF.%20Zhao&amp;journal=Genet%20Mol%20Res&amp;volume=12&amp;issue=4&amp;pages=5111-23&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Forster JW, Jones ES, KÃ¶lliker R, Drayton MC, Dupal MP, Guthridge KM, Smith KF, Henry RJ. Application of DNA profiling to an outbreeding forage species, Plant genotyping: the DNA fingerprinting of plants. 2001. p. 299â320.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Application%20of%20DNA%20profiling%20to%20an%20outbreeding%20forage%20species&amp;author=JW.%20Forster&amp;author=ES.%20Jones&amp;author=R.%20K%C3%B6lliker&amp;author=MC.%20Drayton&amp;author=MP.%20Dupal&amp;author=KM.%20Guthridge&amp;author=KF.%20Smith&amp;author=RJ.%20Henry&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mian MA, Hopkins AA, Zwonitzer JC. Determination of genetic diversity in tall fescue with AFLP markers. Crop Sci. 2002;42(3):944â50.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2002.0944"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Determination%20of%20genetic%20diversity%20in%20tall%20fescue%20with%20AFLP%20markers&amp;author=MA.%20Mian&amp;author=AA.%20Hopkins&amp;author=JC.%20Zwonitzer&amp;journal=Crop%20Sci&amp;volume=42&amp;issue=3&amp;pages=944-50&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gupta PK, Varshney RK, Sharma PC, Ramesh B. Molecular markers and their applications in wheat breeding. Plant Breeding. 1999;118(5):369â90.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1439-0523.1999.00401.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20markers%20and%20their%20applications%20in%20wheat%20breeding&amp;author=PK.%20Gupta&amp;author=RK.%20Varshney&amp;author=PC.%20Sharma&amp;author=B.%20Ramesh&amp;journal=Plant%20Breeding&amp;volume=118&amp;issue=5&amp;pages=369-90&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saha MC, Mian MR, Eujayl I, Zwonitzer JC, Wang L, May GD. Tall fescue EST-SSR markers with transferability across several grass species. Theor Appl Genet. 2004;109(4):783â91.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-004-1681-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15205737"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tall%20fescue%20EST-SSR%20markers%20with%20transferability%20across%20several%20grass%20species&amp;author=MC.%20Saha&amp;author=MR.%20Mian&amp;author=I.%20Eujayl&amp;author=JC.%20Zwonitzer&amp;author=L.%20Wang&amp;author=GD.%20May&amp;journal=Theor%20Appl%20Genet&amp;volume=109&amp;issue=4&amp;pages=783-91&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saha MC, Cooper JD, Mian MR, Chekhovskiy K, May GD. Tall fescue genomic SSR markers: development and transferability across multiple grass species. Theor Appl Genet. 2006;113(8):1449â58.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-006-0391-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16947059"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tall%20fescue%20genomic%20SSR%20markers%3A%20development%20and%20transferability%20across%20multiple%20grass%20species&amp;author=MC.%20Saha&amp;author=JD.%20Cooper&amp;author=MR.%20Mian&amp;author=K.%20Chekhovskiy&amp;author=GD.%20May&amp;journal=Theor%20Appl%20Genet&amp;volume=113&amp;issue=8&amp;pages=1449-58&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hirata M, Cai H, Inoue M, Yuyama N, Miura Y, Komatsu T, Takamizo T, Fujimori M. Development of simple sequence repeat (SSR) markers and construction of an SSR-based linkage map in Italian ryegrass (<em class="EmphasisTypeItalic">Lolium multiflorum</em> Lam.). Theor Appl Genet. 2006;113(2):270â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-006-0292-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16791693"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20simple%20sequence%20repeat%20%28SSR%29%20markers%20and%20construction%20of%20an%20SSR-based%20linkage%20map%20in%20Italian%20ryegrass%20%28Lolium%20multiflorum%20Lam.%29&amp;author=M.%20Hirata&amp;author=H.%20Cai&amp;author=M.%20Inoue&amp;author=N.%20Yuyama&amp;author=Y.%20Miura&amp;author=T.%20Komatsu&amp;author=T.%20Takamizo&amp;author=M.%20Fujimori&amp;journal=Theor%20Appl%20Genet&amp;volume=113&amp;issue=2&amp;pages=270-9&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Elazreg H, Ghariani S, Chtourou-Ghorbel N, Chakroun M, Trifi-Farah N. SSRs transferability and genetic diversity of Tunisian <em class="EmphasisTypeItalic">Festuca arundinacea</em> and <em class="EmphasisTypeItalic">Lolium perenne</em>. Biochem Syst Ecol. 2011;39(2):79â87.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2011.01.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SSRs%20transferability%20and%20genetic%20diversity%20of%20Tunisian%20Festuca%20arundinacea%20and%20Lolium%20perenne%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=H.%20Elazreg&amp;author=S.%20Ghariani&amp;author=N.%20Chtourou-Ghorbel&amp;author=M.%20Chakroun&amp;author=N.%20Trifi-Farah&amp;journal=Biochem%20Syst%20Ecol&amp;volume=39&amp;issue=2&amp;pages=79-87&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Asay KH, Frakes RV, Buckner RC. Breeding and cultivars, Tall fescue. 1979. p. 111â39.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Breeding%20and%20cultivars&amp;author=KH.%20Asay&amp;author=RV.%20Frakes&amp;author=RC.%20Buckner&amp;publication_year=1979"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Allen GC, Flores-Vergara MA, Krasynanski S, Kumar S, Thompson WF. A modified protocol for rapid DNA isolation from plant tissues using cetyltrimethylammonium bromide. Nat Protoc. 2006;1(5):2320â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2006.384"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17406474"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20modified%20protocol%20for%20rapid%20DNA%20isolation%20from%20plant%20tissues%20using%20cetyltrimethylammonium%20bromide&amp;author=GC.%20Allen&amp;author=MA.%20Flores-Vergara&amp;author=S.%20Krasynanski&amp;author=S.%20Kumar&amp;author=WF.%20Thompson&amp;journal=Nat%20Protoc&amp;volume=1&amp;issue=5&amp;pages=2320-5&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shannon CE. A mathematical theory of communication. J Bell Syst Tech. 1948;27:379â423. 623-56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/j.1538-7305.1948.tb01338.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20mathematical%20theory%20of%20communication&amp;author=CE.%20Shannon&amp;journal=J%20Bell%20Syst%20Tech&amp;volume=27&amp;pages=379-423&amp;publication_year=1948"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anderson JA, Churchill GA, Autrique JE, Tanksley SD, Sorrells ME. Optimizing parental selection for genetic linkage maps. Genome. 1993;36(1):181â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/g93-024"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18469981"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Optimizing%20parental%20selection%20for%20genetic%20linkage%20maps&amp;author=JA.%20Anderson&amp;author=GA.%20Churchill&amp;author=JE.%20Autrique&amp;author=SD.%20Tanksley&amp;author=ME.%20Sorrells&amp;journal=Genome&amp;volume=36&amp;issue=1&amp;pages=181-6&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tessier C, David J, This P, Boursiquot J, Charrier A. Optimization of the choice of molecular markers for varietal identification in <em class="EmphasisTypeItalic">Vitis vinifera</em> L. Theor Appl Genet. 1999;98(1):171â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051054"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Optimization%20of%20the%20choice%20of%20molecular%20markers%20for%20varietal%20identification%20in%20Vitis%20vinifera%20L&amp;author=C.%20Tessier&amp;author=J.%20David&amp;author=P.%20This&amp;author=J.%20Boursiquot&amp;author=A.%20Charrier&amp;journal=Theor%20Appl%20Genet&amp;volume=98&amp;issue=1&amp;pages=171-7&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rohlf FJ. NTSYS-PC, numerical taxonomy system for the PC, Exeter Software, Version 2.1. 2000. Applied Biostatistics Inc Setauket, USA.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NTSYS-PC%2C%20numerical%20taxonomy%20system%20for%20the%20PC&amp;author=FJ.%20Rohlf&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hampl V, PavlÃ­cek A, Flegr J. Construction and bootstrap analysis of DNA fingerprinting-based phylogenetic trees with the freeware program FreeTree: application to trichomonad parasites. Int J Syst Evol Micr. 2001;51(3):731â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1099/00207713-51-3-731"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20and%20bootstrap%20analysis%20of%20DNA%20fingerprinting-based%20phylogenetic%20trees%20with%20the%20freeware%20program%20FreeTree%3A%20application%20to%20trichomonad%20parasites&amp;author=V.%20Hampl&amp;author=A.%20Pavl%C3%ADcek&amp;author=J.%20Flegr&amp;journal=Int%20J%20Syst%20Evol%20Micr&amp;volume=51&amp;issue=3&amp;pages=731-5&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155(2):945â59.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10835412"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461096"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=JK.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;journal=Genetics&amp;volume=155&amp;issue=2&amp;pages=945-59&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14(8):2611â20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02553.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15969739"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;journal=Mol%20Ecol&amp;volume=14&amp;issue=8&amp;pages=2611-20&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Earl DA. STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conserv Genet Resour. 2012;4(2):359â61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12686-011-9548-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=STRUCTURE%20HARVESTER%3A%20a%20website%20and%20program%20for%20visualizing%20STRUCTURE%20output%20and%20implementing%20the%20Evanno%20method&amp;author=DA.%20Earl&amp;journal=Conserv%20Genet%20Resour&amp;volume=4&amp;issue=2&amp;pages=359-61&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jakobsson M, Rosenberg NA. CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure. Bioinformatics. 2007;23(14):1801â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm233"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17485429"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=CLUMPP%3A%20a%20cluster%20matching%20and%20permutation%20program%20for%20dealing%20with%20label%20switching%20and%20multimodality%20in%20analysis%20of%20population%20structure&amp;author=M.%20Jakobsson&amp;author=NA.%20Rosenberg&amp;journal=Bioinformatics&amp;volume=23&amp;issue=14&amp;pages=1801-6&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ramasamy RK, Ramasamy S, Bindroo BB, Naik VG. STRUCTURE PLOT: a program for drawing elegant STRUCTURE bar plots in user friendly interface. SpringerPlus. 2014;3(1):431.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/2193-1801-3-431"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25152854"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4141070"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=STRUCTURE%20PLOT%3A%20a%20program%20for%20drawing%20elegant%20STRUCTURE%20bar%20plots%20in%20user%20friendly%20interface&amp;author=RK.%20Ramasamy&amp;author=S.%20Ramasamy&amp;author=BB.%20Bindroo&amp;author=VG.%20Naik&amp;journal=SpringerPlus&amp;volume=3&amp;issue=1&amp;pages=431&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Garg G, Sharma V. Assessment of fatty acid content and genetic diversity in <em class="EmphasisTypeItalic">Eruca sativa</em> (L.)(Taramira) using ISSR markers. Biomass Bioenerg. 2015;76:118â29.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.biombioe.2015.03.010"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20fatty%20acid%20content%20and%20genetic%20diversity%20in%20Eruca%20sativa%20%28L.%29%28Taramira%29%20using%20ISSR%20markers&amp;author=G.%20Garg&amp;author=V.%20Sharma&amp;journal=Biomass%20Bioenerg&amp;volume=76&amp;pages=118-29&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kubik C, Sawkins M, Meyer WA, Gaut BS. Genetic diversity in seven perennial ryegrass (L.) cultivars based on SSR markers. Crop Sci. 2001;41(5):1565â72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2001.4151565x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20seven%20perennial%20ryegrass%20%28L.%29%20cultivars%20based%20on%20SSR%20markers&amp;author=C.%20Kubik&amp;author=M.%20Sawkins&amp;author=WA.%20Meyer&amp;author=BS.%20Gaut&amp;journal=Crop%20Sci&amp;volume=41&amp;issue=5&amp;pages=1565-72&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hand ML, Cogan NO, Forster JW. Molecular characterisation and interpretation of genetic diversity within globally distributed germplasm collections of tall fescue (<em class="EmphasisTypeItalic">Festuca arundinacea</em> Schreb.) and meadow fescue (<em class="EmphasisTypeItalic">F. pratensis</em> Huds.). Theor Appl Genet. 2012;124(6):1127â37.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-011-1774-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22222441"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20characterisation%20and%20interpretation%20of%20genetic%20diversity%20within%20globally%20distributed%20germplasm%20collections%20of%20tall%20fescue%20%28Festuca%20arundinacea%20Schreb.%29%20and%20meadow%20fescue%20%28F.%20pratensis%20Huds.%29&amp;author=ML.%20Hand&amp;author=NO.%20Cogan&amp;author=JW.%20Forster&amp;journal=Theor%20Appl%20Genet&amp;volume=124&amp;issue=6&amp;pages=1127-37&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Larson SR, Jones TA, Jensen KB. Population structure in <em class="EmphasisTypeItalic">Pseudoroegneria spicata</em> (Poaceae: Triticeae) modeled by Bayesian clustering of AFLP genotypes. Am J Bot. 2004;91(11):1789â801.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3732/ajb.91.11.1789"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21652326"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20structure%20in%20Pseudoroegneria%20spicata%20%28Poaceae%3A%20Triticeae%29%20modeled%20by%20Bayesian%20clustering%20of%20AFLP%20genotypes&amp;author=SR.%20Larson&amp;author=TA.%20Jones&amp;author=KB.%20Jensen&amp;journal=Am%20J%20Bot&amp;volume=91&amp;issue=11&amp;pages=1789-801&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tehrani MS, Mardi M, Sahebi J, CatalÃ¡n P, DÃ­az-PÃ©rez A. Genetic diversity and structure among Iranian tall fescue populations based on genomic-SSR and EST-SSR marker analysis. Plant Syst Evol. 2009;282(1-2):57â70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00606-009-0207-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20structure%20among%20Iranian%20tall%20fescue%20populations%20based%20on%20genomic-SSR%20and%20EST-SSR%20marker%20analysis&amp;author=MS.%20Tehrani&amp;author=M.%20Mardi&amp;author=J.%20Sahebi&amp;author=P.%20Catal%C3%A1n&amp;author=A.%20D%C3%ADaz-P%C3%A9rez&amp;journal=Plant%20Syst%20Evol&amp;volume=282&amp;issue=1-2&amp;pages=57-70&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Buckner RE, Boling JA, Burrus PB, Bush LP, Hemken RA. Registration of Johnstone tall fescue (Reg. No. 23). Crop Sci. 1983;23(2):399â400.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1983.0011183X002300020057x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Johnstone%20tall%20fescue%20%28Reg.%20No.%2023%29&amp;author=RE.%20Buckner&amp;author=JA.%20Boling&amp;author=PB.%20Burrus&amp;author=LP.%20Bush&amp;author=RA.%20Hemken&amp;journal=Crop%20Sci&amp;volume=23&amp;issue=2&amp;pages=399-400&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Abernethy RH, Steiner JJ, Wofford DS, Thiel DS. Classification and pedigree verification of tall fescue cultivars utilizing the prolamin seed protein fraction. Crop Sci. 1989;29(3):791â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1989.0011183X002900030051x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Classification%20and%20pedigree%20verification%20of%20tall%20fescue%20cultivars%20utilizing%20the%20prolamin%20seed%20protein%20fraction&amp;author=RH.%20Abernethy&amp;author=JJ.%20Steiner&amp;author=DS.%20Wofford&amp;author=DS.%20Thiel&amp;journal=Crop%20Sci&amp;volume=29&amp;issue=3&amp;pages=791-7&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Buckner RC, Burrus PB, Bush LP. Registration of Kenhy tall fescue (Reg. No. 12). Crop Sci. 1977;17(4):672â3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1977.0011183X001700040059x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Kenhy%20tall%20fescue%20%28Reg.%20No.%2012%29&amp;author=RC.%20Buckner&amp;author=PB.%20Burrus&amp;author=LP.%20Bush&amp;journal=Crop%20Sci&amp;volume=17&amp;issue=4&amp;pages=672-3&amp;publication_year=1977"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Buckner RC, Burrus PB. Registration of Kenwell tall fescue (Reg. No. 5). Crop Sci. 1968;8(3):398.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1968.0011183X000800030051x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Kenwell%20tall%20fescue%20%28Reg.%20No.%205%29&amp;author=RC.%20Buckner&amp;author=PB.%20Burrus&amp;journal=Crop%20Sci&amp;volume=8&amp;issue=3&amp;pages=398&amp;publication_year=1968"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fergus EN, Buckner RC. Registration of Kentucky 31 tall fescue (Reg. No. 7). Crop Sci. 1972;12(5):714.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1972.0011183X001200050061x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Kentucky%2031%20tall%20fescue%20%28Reg.%20No.%207%29&amp;author=EN.%20Fergus&amp;author=RC.%20Buckner&amp;journal=Crop%20Sci&amp;volume=12&amp;issue=5&amp;pages=714&amp;publication_year=1972"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hoveland CS. Origin and history of tall fescue. In: Fribourg HA, Hannaway DB, West CP, editors. Tall fescue for the twenty-first century. Madison, WI: American Society of Agronomy, Crop Science Society of America, Soil Science Society of America; 2009. p 3â10.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Hoveland%20CS.%20Origin%20and%20history%20of%20tall%20fescue.%20In%3A%20Fribourg%20HA%2C%20Hannaway%20DB%2C%20West%20CP%2C%20editors.%20Tall%20fescue%20for%20the%20twenty-first%20century.%20Madison%2C%20WI%3A%20American%20Society%20of%20Agronomy%2C%20Crop%20Science%20Society%20of%20America%2C%20Soil%20Science%20Society%20of%20America%3B%202009.%20p%203%E2%80%9310."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Asay KH, Sleper DA. Registration of Missouri-96 tall fescue (Reg. No. 15). Crop Sci. 1979;19(4):563.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1979.0011183X001900040035x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Missouri-96%20tall%20fescue%20%28Reg.%20No.%2015%29&amp;author=KH.%20Asay&amp;author=DA.%20Sleper&amp;journal=Crop%20Sci&amp;volume=19&amp;issue=4&amp;pages=563&amp;publication_year=1979"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hollowell EA. Registration of varieties and strains of grasses, I. Agron J. 1945;37(8):653â4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2134/agronj1945.00021962003700080012x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20varieties%20and%20strains%20of%20grasses%2C%20I&amp;author=EA.%20Hollowell&amp;journal=Agron%20J&amp;volume=37&amp;issue=8&amp;pages=653-4&amp;publication_year=1945"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rose-Fricker CA, Meyer WA, Van Wuk JP. Registration of Maximize tall fescue. Crop Sci. 1993;33:203.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Maximize%20tall%20fescue&amp;author=CA.%20Rose-Fricker&amp;author=WA.%20Meyer&amp;author=JP.%20Wuk&amp;journal=Crop%20Sci&amp;volume=33&amp;pages=203&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pedersen JF, Haaland RL, Hoveland CS, Berry CD, Schmidt SP, Harris RR. Registration of Au Triumph tall fescue (Reg. No. 21). Crop Sci. 1983;23(1):182.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1983.0011183X002300010062x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Au%20Triumph%20tall%20fescue%20%28Reg.%20No.%2021%29&amp;author=JF.%20Pedersen&amp;author=RL.%20Haaland&amp;author=CS.%20Hoveland&amp;author=CD.%20Berry&amp;author=SP.%20Schmidt&amp;author=RR.%20Harris&amp;journal=Crop%20Sci&amp;volume=23&amp;issue=1&amp;pages=182&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pedersen JF, McVeigh KJ, Edminster CW. Registration of Cajun tall fescue. Crop Sci. 1989;29(1):236â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1989.0011183X002900010057x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Cajun%20tall%20fescue&amp;author=JF.%20Pedersen&amp;author=KJ.%20McVeigh&amp;author=CW.%20Edminster&amp;journal=Crop%20Sci&amp;volume=29&amp;issue=1&amp;pages=236-7&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">van Santen E. Animal preference of tall fescue during reproductive growth in the spring. Agron J. 1992;84(6):979â82.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2134/agronj1992.00021962008400060014x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Animal%20preference%20of%20tall%20fescue%20during%20reproductive%20growth%20in%20the%20spring&amp;author=E.%20Santen&amp;journal=Agron%20J&amp;volume=84&amp;issue=6&amp;pages=979-82&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kubik C, Honig JA, Meyer WA, Bonos SA. Genetic diversity of creeping bentgrass cultivars using SSR markers. Int Turfgrass Soc Res J. 2009;11:533â47.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20creeping%20bentgrass%20cultivars%20using%20SSR%20markers&amp;author=C.%20Kubik&amp;author=JA.%20Honig&amp;author=WA.%20Meyer&amp;author=SA.%20Bonos&amp;journal=Int%20Turfgrass%20Soc%20Res%20J&amp;volume=11&amp;pages=533-47&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Baluch SJ, Stratton SD, Buker RJ. Registration of Forager tall fescue (Reg. No. 17). Crop Sci. 1980;20(5):670.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Baluch%20SJ%2C%20Stratton%20SD%2C%20Buker%20RJ.%20Registration%20of%20Forager%20tall%20fescue%20%28Reg.%20No.%2017%29.%20Crop%20Sci.%201980%3B20%285%29%3A670."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kindiger B, Gau M, Hasegawa H, Katsura M, Ueyama Y, Goto K, Sato S, Yoshiyama T, Terada Y, Tsurumi Y. Registration of Nanryo tall fescue. Crop Sci. 2006;46(4):1815â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2005.09-0344"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20Nanryo%20tall%20fescue&amp;author=B.%20Kindiger&amp;author=M.%20Gau&amp;author=H.%20Hasegawa&amp;author=M.%20Katsura&amp;author=Y.%20Ueyama&amp;author=K.%20Goto&amp;author=S.%20Sato&amp;author=T.%20Yoshiyama&amp;author=Y.%20Terada&amp;author=Y.%20Tsurumi&amp;journal=Crop%20Sci&amp;volume=46&amp;issue=4&amp;pages=1815-6&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sleper DA, Mayland HF, Crawford RJ, Shewmaker GE, Massie MD. Registration of HiMag tall fescue germplasm. Crop Sci. 2002;42:318â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2002.0318"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11756311"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20HiMag%20tall%20fescue%20germplasm&amp;author=DA.%20Sleper&amp;author=HF.%20Mayland&amp;author=RJ.%20Crawford&amp;author=GE.%20Shewmaker&amp;author=MD.%20Massie&amp;journal=Crop%20Sci&amp;volume=42&amp;pages=318-9&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fu Y, Legge WG, Clarke JM, Richards KW. Genetic variation and relationships of pedigree-known oat, wheat, and barley cultivars releaved by bulking and single-plant sampling. Genet Resour Crop Ev. 2006;53(6):1153â64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-005-1306-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variation%20and%20relationships%20of%20pedigree-known%20oat%2C%20wheat%2C%20and%20barley%20cultivars%20releaved%20by%20bulking%20and%20single-plant%20sampling&amp;author=Y.%20Fu&amp;author=WG.%20Legge&amp;author=JM.%20Clarke&amp;author=KW.%20Richards&amp;journal=Genet%20Resour%20Crop%20Ev&amp;volume=53&amp;issue=6&amp;pages=1153-64&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Michelmore RW, Paran I, Kesseli RV. Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations. Proc Natl Acad Sci U S A. 1991;88(21):9828â32.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.88.21.9828"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1682921"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC52814"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20markers%20linked%20to%20disease-resistance%20genes%20by%20bulked%20segregant%20analysis%3A%20a%20rapid%20method%20to%20detect%20markers%20in%20specific%20genomic%20regions%20by%20using%20segregating%20populations&amp;author=RW.%20Michelmore&amp;author=I.%20Paran&amp;author=RV.%20Kesseli&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=88&amp;issue=21&amp;pages=9828-32&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu K, Pauls KP. Rapid estimation of genetic relatedness among heterogeneous populations of alfalfa by random amplification of bulked genomic DNA samples. Theor Appl Genet. 1993;86(6):788â94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00222671"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24193791"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20estimation%20of%20genetic%20relatedness%20among%20heterogeneous%20populations%20of%20alfalfa%20by%20random%20amplification%20of%20bulked%20genomic%20DNA%20samples&amp;author=K.%20Yu&amp;author=KP.%20Pauls&amp;journal=Theor%20Appl%20Genet&amp;volume=86&amp;issue=6&amp;pages=788-94&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sweeney PM, Danneberger TK. RAPD markers from perennial ryegrass DNA extracted from seeds. Hortscience. 1997;32(7):1212â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RAPD%20markers%20from%20perennial%20ryegrass%20DNA%20extracted%20from%20seeds&amp;author=PM.%20Sweeney&amp;author=TK.%20Danneberger&amp;journal=Hortscience&amp;volume=32&amp;issue=7&amp;pages=1212-5&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang T, Ma X, Zhang XQ, Zhang XY, Zhang RZ FUKX. Comparation of SSR Molecular Markers Analysis of Annual Ryegrass Varieties in DUS Testing. Sci Agr Sin. 2015;48(2):381â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparation%20of%20SSR%20Molecular%20Markers%20Analysis%20of%20Annual%20Ryegrass%20Varieties%20in%20DUS%20Testing&amp;author=T.%20Huang&amp;author=X.%20Ma&amp;author=XQ.%20Zhang&amp;author=XY.%20Zhang&amp;author=FUKX.%20Zhang%20RZ&amp;journal=Sci%20Agr%20Sin&amp;volume=48&amp;issue=2&amp;pages=381-9&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© The Author(s) 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0013-1" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-016-0013-1">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0013-1" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0013-1?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0013-1 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0013-1 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0013-1 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0013-1">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0013-1">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0013-1"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0013-1">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0013-1">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0013-1"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0013-1">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0013-1">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0013-1"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0013-1">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0013-1">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0013-1"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0013-1">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0013-1">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0013-1"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-016-0013-1/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-016-0013-1">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0013-1">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0013-1" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-016-0013-1&amp;volume=153&amp;issue=1&amp;title=Insight+into+the+genetic+variability+analysis+and+cultivar+identification+of+tall+fescue+by+using+SSR+markers&amp;author_name=Kaixin+Fu&amp;start_page=1&amp;end_page=9">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-016-0013-1" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
