<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Assessment of diversity and genetic relationships of Neonectria ditissima: the causal agent of fruit tree canker | Hereditas | Full Text</title>
<meta name="citation_abstract" content="
                           Neonectria ditissima is one of the most important fungal pathogens of apple trees, where it causes fruit tree canker. Information about the amount and partitioning of genetic variation of this fungus could be helpful for improving orchard management strategies and for breeding apple cultivars with high levels of genetically determined resistance. In this study single-spore Neonectria isolates originating from both the same and from different perithecia, apple cultivars and apple orchards in Sweden and Belgium, were evaluated for AFLP- and SSR-based genetic similarity and for mating system. Seven SSR loci produced a total of 31 alleles with an average of 4 alleles per locus, while 11 AFLP primer combinations produced an average of 35 fragments per primer combination and 71Â % polymorphic fragments. An AFLP-based analysis of molecular variance (AMOVA) revealed that 89Â % of the variation was found within orchards and 11Â % between orchards. Genetic similarity among the studied isolates was illustrated with a principal coordinate analyseis (PCoA) and a dendrogram. AFLP-based Jaccardâs similarity coefficients were the highest when single-ascospore isolates obtained from the same perithecium were compared, medium-high for isolates from different perithecia on the same tree, and lowest when isolates from different trees were compared. Based on the results of PCoA and AMOVA analysis, isolates from the same or geographically close orchards did not group together. Since AFLP profiles differed also when single-ascospore isolates from the same perithecium were compared, the mating system of N. ditissima is most likely heterothallic.">
<meta name="description" content="
                           Neonectria ditissima is one of the most important fungal pathogens of apple trees, where it causes fruit tree canker. Information about the amount and partitioning of genetic variation of this fungus could be helpful for improving orchard management strategies and for breeding apple cultivars with high levels of genetically determined resistance. In this study single-spore Neonectria isolates originating from both the same and from different perithecia, apple cultivars and apple orchards in Sweden and Belgium, were evaluated for AFLP- and SSR-based genetic similarity and for mating system. Seven SSR loci produced a total of 31 alleles with an average of 4 alleles per locus, while 11 AFLP primer combinations produced an average of 35 fragments per primer combination and 71Â % polymorphic fragments. An AFLP-based analysis of molecular variance (AMOVA) revealed that 89Â % of the variation was found within orchards and 11Â % between orchards. Genetic similarity among the studied isolates was illustrated with a principal coordinate analyseis (PCoA) and a dendrogram. AFLP-based Jaccardâs similarity coefficients were the highest when single-ascospore isolates obtained from the same perithecium were compared, medium-high for isolates from different perithecia on the same tree, and lowest when isolates from different trees were compared. Based on the results of PCoA and AMOVA analysis, isolates from the same or geographically close orchards did not group together. Since AFLP profiles differed also when single-ascospore isolates from the same perithecium were compared, the mating system of N. ditissima is most likely heterothallic.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Assessment of diversity and genetic relationships of Neonectria ditissima : the causal agent of fruit tree canker">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-07-01">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="
                           Neonectria ditissima is one of the most important fungal pathogens of apple trees, where it causes fruit tree canker. Information about the amount and partitioning of genetic variation of this fungus could be helpful for improving orchard management strategies and for breeding apple cultivars with high levels of genetically determined resistance. In this study single-spore Neonectria isolates originating from both the same and from different perithecia, apple cultivars and apple orchards in Sweden and Belgium, were evaluated for AFLP- and SSR-based genetic similarity and for mating system. Seven SSR loci produced a total of 31 alleles with an average of 4 alleles per locus, while 11 AFLP primer combinations produced an average of 35 fragments per primer combination and 71 % polymorphic fragments. An AFLP-based analysis of molecular variance (AMOVA) revealed that 89 % of the variation was found within orchards and 11 % between orchards. Genetic similarity among the studied isolates was illustrated with a principal coordinate analyseis (PCoA) and a dendrogram. AFLP-based Jaccard’s similarity coefficients were the highest when single-ascospore isolates obtained from the same perithecium were compared, medium-high for isolates from different perithecia on the same tree, and lowest when isolates from different trees were compared. Based on the results of PCoA and AMOVA analysis, isolates from the same or geographically close orchards did not group together. Since AFLP profiles differed also when single-ascospore isolates from the same perithecium were compared, the mating system of N. ditissima is most likely heterothallic.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-07-01">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="7">
<meta name="prism.copyright" content="2016 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0011-3">
<meta name="prism.doi" content="doi:10.1186/s41065-016-0011-3">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0011-3">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0011-3">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Assessment of diversity and genetic relationships of Neonectria ditissima : the causal agent of fruit tree canker">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/07/01">
<meta name="citation_firstpage" content="7">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-016-0011-3">
<meta name="DOI" content="10.1186/s41065-016-0011-3">
<meta name="citation_doi" content="10.1186/s41065-016-0011-3">
<meta name="dc.creator" content="Marjan Ghasemkhani">
<meta name="dc.creator" content="Larisa Garkava-Gustavsson">
<meta name="dc.creator" content="Erland Liljeroth">
<meta name="dc.creator" content="Hilde Nybom">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Can J Bot; citation_title=Mutations in Ophiostoma ulmi induced by N-methyl-N'-nitro-N-nitroso-guanidine; citation_author=L Bernier, M Hubbes; citation_volume=68; citation_publication_date=1990; citation_pages=225-31; citation_doi=10.1139/b90-031; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=J Evol Biol; citation_title=Sex, outcrossing and mating types: unsolved questions in fungi and beyond; citation_author=S Billiard, M López‐Villavicencio, M Hood, T Giraud; citation_volume=25; citation_issue=6; citation_publication_date=2012; citation_pages=1020-38; citation_doi=10.1111/j.1420-9101.2012.02495.x; citation_id=CR2">
<meta name="citation_reference" content="citation_title=The Fungi; citation_publication_date=2001; citation_id=CR3; citation_author=MJ Carlile; citation_author=SC Watkinson; citation_author=GW Gooday; citation_publisher=Gulf Professional Publishing">
<meta name="citation_reference" content="citation_journal_title=Can J Bot; citation_title=Phylogenetic relationships of Neonectria/Cylindrocarpon on Fagus in North America; citation_author=LA Castlebury, AY Rossman, AS Hyten; citation_volume=84; citation_publication_date=2006; citation_pages=1417-33; citation_doi=10.1139/b06-105; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=Ann Bot-London; citation_title=Xylem structure of four grape varieties and 12 alternative hosts to the xylem-limited bacterium Xylella fastidious
                           ; citation_author=DS Chatelet, CM Wistrom, AH Purcell, TL Rost, MA Matthews; citation_volume=108; citation_publication_date=2011; citation_pages=73-85; citation_doi=10.1093/aob/mcr106; citation_id=CR5">
<meta name="citation_reference" content="citation_title=Crop Protection Compendium on Internet; citation_publication_date=2014; citation_id=CR6; citation_publisher=CAB International">
<meta name="citation_reference" content="citation_journal_title=Mycol Res; citation_title=Segregation and recombination of PCR based markers in sexual progeny of Phaeosphaeria species; citation_author=PC Czembor, E Arseniuk; citation_volume=104; citation_publication_date=2000; citation_pages=919-26; citation_doi=10.1017/S095375620000246X; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=How did host domestication modify life history traits of its pathogens?; citation_author=M Gracia, M Cascales, P Expert, M-N Bellanger, B Cam, C Lemaire; citation_volume=10; citation_publication_date=2015; citation_pages=e0122909; citation_doi=10.1371/journal.pone.0122909; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Can J Bot; citation_title=Homothallism in Nectria galligena
                           ; citation_author=N El-Gholl, E Barnard, R Schroeder; citation_volume=64; citation_publication_date=1986; citation_pages=902-3; citation_doi=10.1139/b86-119; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Evol Bioinform Online; citation_title=Arlequin (version 3.0): an integrated software package for population genetics data analysis; citation_author=L Excoffier, G Laval, S Schneider; citation_volume=1; citation_publication_date=2005; citation_pages=47; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Int J Adv Agric Res; citation_title=Comparative analyses of amplified fragment length polymorphism (AFLP) and simple sequence repeat (SSR) in genetic diversity of Teak (Tectona grandis Lf); citation_author=IJ Fofana, S Silue, N Diarrassouba, AA Kadio, A Sangare; citation_volume=1; citation_publication_date=2013; citation_pages=114-23; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Tunis J Plant Prot; citation_title=AFLP analysis of the genetic variability and population structure of the wheat crown rot fungus Fusarium pseudograminearum in Tunisia; citation_author=S Gargouri, S Hamza, M Hajlaoui; citation_volume=1; citation_publication_date=2006; citation_pages=93-104; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Trees; citation_title=Real-time PCR for detection and quantification, and histological characterization of Neonectria ditissima in apple trees; citation_author=M Ghasemkhani, A Holefors, S Marttila, K Dalman, A Zborowska, M Rur, J Rees-George, H Nybom, KR Everett, RWA Scheper, L Garkava-Gustavsson; citation_publication_date=2016; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=J Phytopathol; citation_title=Zur Genetik von Nectria galligena Bres; citation_author=J Krüger; citation_volume=79; citation_publication_date=1974; citation_pages=320-42; citation_doi=10.1111/j.1439-0434.1974.tb02715.x; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Trends Ecol Evol; citation_title=Gene flow and the limits to natural selection; citation_author=T Lenormand; citation_volume=17; citation_publication_date=2002; citation_pages=183-9; citation_doi=10.1016/S0169-5347(02)02497-7; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=J Agric Sci; citation_title=Comparison of AFLP and SSR for genetic diversity analysis of Brassica napus hybrids; citation_author=L Li, C Wanapu, X Huang, T Huang, Q Li, Y Peng, G Huang; citation_volume=3; citation_publication_date=2011; citation_pages=101-10; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Mycologia; citation_title=Origin, genetic diversity and population structure of Nectria coccinea var. faginata in North America; citation_author=EM Mahoney, MG Milgroom, WA Sinclair; citation_volume=91; citation_publication_date=1999; citation_pages=583-92; citation_doi=10.2307/3761243; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=Cancer Res; citation_title=The detection of disease clustering and a generalized regression approach; citation_author=N Mantel; citation_volume=27; citation_publication_date=1967; citation_pages=209-20; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=Isolation and characterization of codominant markers for the perennial canker fungal pathogen Neonectria ditissima
                           ; citation_author=RE Marra, JA Corwin; citation_volume=9; citation_publication_date=2009; citation_pages=906-9; citation_doi=10.1111/j.1755-0998.2008.02438.x; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Plant Pathol; citation_title=Relative significance of nursery infections and orchard inoculum in the development and spread of apple canker (Nectria galligena) in young orchards; citation_author=AR McCracken, A Berrie, DJ Barbara, T Locke, LR Cooke, K Phelps, TR Swinburne, AE Brown, B Ellerker, SRH Langrell; citation_volume=52; citation_publication_date=2003; citation_pages=553-66; citation_doi=10.1046/j.1365-3059.2003.00924.x; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=Phytopathology; citation_title=The population genetics of fungi: tools and techniques; citation_author=BA McDonald; citation_volume=87; citation_publication_date=1997; citation_pages=448-53; citation_doi=10.1094/PHYTO.1997.87.4.448; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Annu Rev Phytopathol; citation_title=Pathogen population genetics, evolutionary potential, and durable resistance; citation_author=BA McDonald, C Linde; citation_volume=40; citation_publication_date=2002; citation_pages=349-79; citation_doi=10.1146/annurev.phyto.40.120501.101443; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Bioscience; citation_title=Population Genetics of Plant Pathogenic Fungi; citation_author=BA McDonald, JM McDermott; citation_volume=43; citation_publication_date=1993; citation_pages=311-9; citation_doi=10.2307/1312063; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=3 Biotech; citation_title=Genetic diversity of Phytophthora colocasiae isolates in India based on AFLP analysis; citation_author=VS Nath, M Senthil, VM Hegde, ML Jeeva, RS Misra, SS Veena, M Raj; citation_volume=3; citation_publication_date=2013; citation_pages=297-305; citation_doi=10.1007/s13205-012-0101-5; citation_id=CR24">
<meta name="citation_reference" content="citation_journal_title=Mycol Res; citation_title=A comparative study of genetic diversity of populations of Nectria galligena and N. coccinea var. faginata in North America; citation_author=F Plante, RC Hamelin, L Bernier; citation_volume=106; citation_publication_date=2002; citation_pages=183-93; citation_doi=10.1017/S0953756201005329; citation_id=CR25">
<meta name="citation_reference" content="citation_journal_title=Trends Microbiol; citation_title=The fitness of filamentous fungi; citation_author=A Pringle, JW Taylor; citation_volume=10; citation_publication_date=2002; citation_pages=474-81; citation_doi=10.1016/S0966-842X(02)02447-2; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=Fungal Genet Biol; citation_title=Breeding systems in the lichen-forming fungal genus Cladonia
                           ; citation_author=FA Seymour, PD Crittenden, MJ Dickinson, M Paoletti, D Montiel, L Cho, PS Dyer; citation_volume=42; citation_publication_date=2005; citation_pages=554-63; citation_doi=10.1016/j.fgb.2005.03.006; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Phytopathology; citation_title=Effect of nitrogen-fertilization on expression of slow-mildewing resistance in Knox wheat; citation_author=G Shaner, RE Finney; citation_volume=67; citation_publication_date=1977; citation_pages=1051-6; citation_doi=10.1094/Phyto-67-1051; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Genetic diversity and population structure of rice pathogen Ustilaginoidea virens in China; citation_author=X Sun, S Kang, Y Zhang, X Tan, Y Yu, H He, X Zhang, Y Liu, S Wang, W Sun; citation_volume=8; citation_publication_date=2013; citation_pages=e76879; citation_doi=10.1371/journal.pone.0076879; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0; citation_author=K Tamura, J Dudley, M Nei, S Kumar; citation_volume=24; citation_publication_date=2007; citation_pages=1596-9; citation_doi=10.1093/molbev/msm092; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Evol Appl; citation_title=Evolution of pathogenicity traits in the apple scab fungal pathogen in response to the domestication of its host; citation_author=A Lê, P Gladieux, C Lemaire, A Cornille, T Giraud, CE Durel, V Caffier, B Cam; citation_volume=5; citation_publication_date=2012; citation_pages=694-704; citation_doi=10.1111/j.1752-4571.2012.00246.x; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Recombination and clonal propagation in different populations of the lichen Lobaria pulmonaria
                           ; citation_author=J Walser, F Gugerli, R Holderegger, D Kuonen, C Scheidegger; citation_volume=93; citation_publication_date=2004; citation_pages=322-9; citation_doi=10.1038/sj.hdy.6800505; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Erwerbs-Obstbau; citation_title=Biology and control of the apple canker fungus Neonectria ditissima (syn. N. galligena) from a Northwestern European perspective; citation_author=RWS Weber; citation_volume=56; citation_publication_date=2014; citation_pages=95-107; citation_doi=10.1007/s10341-014-0210-x; citation_id=CR33">
<meta name="citation_reference" content="citation_title=DNA fingerprinting in plants: principles, methods, and applications; citation_publication_date=2005; citation_id=CR34; citation_author=K Weising; citation_author=H Nybom; citation_author=M Pfenninger; citation_author=K Wolff; citation_author=G Kahl; citation_publisher=CRC press">
<meta name="citation_reference" content="citation_journal_title=Eur J Plant Pathol; citation_title=Population variation of apple scab (Venturia inaequalis) within mixed orchards in the UK; citation_author=X Xu, N Harvey, A Roberts, D Barbara; citation_volume=135; citation_publication_date=2013; citation_pages=97-104; citation_doi=10.1007/s10658-012-0068-4; citation_id=CR35">
<meta name="citation_reference" content="citation_title=POPGENE, the user-friendly shareware for population genetic analysis; citation_publication_date=1997; citation_id=CR36; citation_author=FC Yeh; citation_author=R Yang; citation_author=TB Boyle; citation_author=Z Ye; citation_author=JX Mao; citation_publisher=Molecular Biology and Biotechnology Centre, University of Alberta">
<meta name="citation_author" content="Marjan Ghasemkhani">
<meta name="citation_author_institution" content="Department of Plant Breeding, Swedish University of Agricultural Sciences, Alnarp, Sweden">
<meta name="citation_author" content="Larisa Garkava-Gustavsson">
<meta name="citation_author_institution" content="Department of Plant Breeding, Swedish University of Agricultural Sciences, Alnarp, Sweden">
<meta name="citation_author" content="Erland Liljeroth">
<meta name="citation_author_institution" content="Department of Plant Protection Biology, Swedish University of Agricultural Sciences, Alnarp, Sweden">
<meta name="citation_author" content="Hilde Nybom">
<meta name="citation_author_institution" content="Department of Plant Breeding, Swedish University of Agricultural Sciences, Kristianstad, Sweden">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0011-3',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0011-3","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"AFLP;Apple canker;Genetic variability;Heterothallism;Mating system;SSR"},"contentInfo":{"imprint":"BioMed Central","title":"Assessment of diversity and genetic relationships of Neonectria ditissima : the causal agent of fruit tree canker","publishedAt":1467331200000,"publishedAtDate":"2016-07-01","author":["Marjan Ghasemkhani","Larisa Garkava-Gustavsson","Erland Liljeroth","Hilde Nybom"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0011-3">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0011-3">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Assessment of diversity and genetic relationships of Neonectria ditissima : the causal agent of fruit tree canker">
<meta property="og:description" content="Neonectria ditissima is one of the most important fungal pathogens of apple trees, where it causes fruit tree canker. Information about the amount and partitioning of genetic variation of this fungus could be helpful for improving orchard management strategies and for breeding apple cultivars with high levels of genetically determined resistance. In this study single-spore Neonectria isolates originating from both the same and from different perithecia, apple cultivars and apple orchards in Sweden and Belgium, were evaluated for AFLP- and SSR-based genetic similarity and for mating system. Seven SSR loci produced a total of 31 alleles with an average of 4 alleles per locus, while 11 AFLP primer combinations produced an average of 35 fragments per primer combination and 71&amp;nbsp;% polymorphic fragments. An AFLP-based analysis of molecular variance (AMOVA) revealed that 89&amp;nbsp;% of the variation was found within orchards and 11&amp;nbsp;% between orchards. Genetic similarity among the studied isolates was illustrated with a principal coordinate analyseis (PCoA) and a dendrogram. AFLP-based Jaccardâs similarity coefficients were the highest when single-ascospore isolates obtained from the same perithecium were compared, medium-high for isolates from different perithecia on the same tree, and lowest when isolates from different trees were compared. Based on the results of PCoA and AMOVA analysis, isolates from the same or geographically close orchards did not group together. Since AFLP profiles differed also when single-ascospore isolates from the same perithecium were compared, the mating system of N. ditissima is most likely heterothallic.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0011-3;kwrd=AFLP,Apple canker,Genetic variability,Heterothallism,Mating system,SSR;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0011-3&amp;kwrd=AFLP,Apple%20canker,Genetic%20variability,Heterothallism,Mating%20system,SSR&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0011-3&amp;kwrd=AFLP,Apple%20canker,Genetic%20variability,Heterothallism,Mating%20system,SSR&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and discussion">Results and discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec14" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec15" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abbreviations">Abbreviations</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Assessment of diversity and genetic relationships of <em xmlns="" class="EmphasisTypeItalic">Neonectria ditissima</em>: the causal agent of fruit tree canker</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Marjan Ghasemkhani</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:marjan.ghasemkhani@slu.se" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-0600-2475" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Larisa Garkava-Gustavsson</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Erland Liljeroth</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Hilde Nybom</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>
</li>
</ul></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:7</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0011-3">https://doi.org/10.1186/s41065-016-0011-3</a></p>
<p class="Copyright">Â©  The Author(s) 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>27 February 2016</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>15 June 2016</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">1 July 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">
                           <em xmlns="" class="EmphasisTypeItalic">Neonectria ditissima</em> is one of the most important fungal pathogens of apple trees, where it causes fruit tree canker. Information about the amount and partitioning of genetic variation of this fungus could be helpful for improving orchard management strategies and for breeding apple cultivars with high levels of genetically determined resistance. In this study single-spore <em xmlns="" class="EmphasisTypeItalic">Neonectria</em> isolates originating from both the same and from different perithecia, apple cultivars and apple orchards in Sweden and Belgium, were evaluated for AFLP- and SSR-based genetic similarity and for mating system.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Seven SSR loci produced a total of 31 alleles with an average of 4 alleles per locus, while 11 AFLP primer combinations produced an average of 35 fragments per primer combination and 71 % polymorphic fragments. An AFLP-based analysis of molecular variance (AMOVA) revealed that 89 % of the variation was found within orchards and 11 % between orchards. Genetic similarity among the studied isolates was illustrated with a principal coordinate analyseis (PCoA) and a dendrogram. AFLP-based Jaccardâs similarity coefficients were the highest when single-ascospore isolates obtained from the same perithecium were compared, medium-high for isolates from different perithecia on the same tree, and lowest when isolates from different trees were compared.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Based on the results of PCoA and AMOVA analysis, isolates from the same or geographically close orchards did not group together. Since AFLP profiles differed also when single-ascospore isolates from the same perithecium were compared, the mating system of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> is most likely heterothallic.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">AFLP</li>
<li class="c-keywords__item" itemprop="about">Apple canker</li>
<li class="c-keywords__item" itemprop="about">Genetic variability</li>
<li class="c-keywords__item" itemprop="about">Heterothallism</li>
<li class="c-keywords__item" itemprop="about">Mating system</li>
<li class="c-keywords__item" itemprop="about">SSR</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">The haploid fungus <em xmlns="" class="EmphasisTypeItalic">Neonectria ditissima</em> (Tul. &amp; C. Tul.) Samuels &amp; Rossman<em xmlns="" class="EmphasisTypeItalic">,</em> previously known as <em xmlns="" class="EmphasisTypeItalic">Neonectria galligena</em> (Bres.) Rossman &amp; Samuels<em xmlns="" class="EmphasisTypeItalic">,</em> causes cankers on a wide range of trees and shrubs including apple (<em xmlns="" class="EmphasisTypeItalic">Malus</em>) and pear (<em xmlns="" class="EmphasisTypeItalic">Pyrus</em>). This pathogen can damage the woody tissue of infected trees substantially, and render fruit production unprofitable in certain areas [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Apple canker is mainly associated with mild and wet conditions, and climate therefore has an important impact on the geographic distribution of the pathogen [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. This pathogen produces both conidia and ascospores under favorable conditions over prolonged periods [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. The spores usually enter the trees through natural and artificial wound sites e.g., pruning wounds, leaf scars, fruit scars due to chemical thinning or natural abscission, twig stubs, in the crotches of limbs, and even through lesions caused by apple scab. The spores can also be dispersed within and between orchards with infected equipment used by growers [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Since the pathogen can cause systemic infections that spread inside the plant, the use of infected rootstocks can probably lead to canker symptoms also in the grafted part (i.e. the proper cultivar) of young apple trees.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">In spite of the economic impacts of this pathogen, very little is known about the genetics of the causal agent. Most studies of <em xmlns="" class="EmphasisTypeItalic">Neonectria</em> have focused on analyses of material from North America. In a study of Mahoney et al. [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>], Restriction Fragment Length Polymorphism (RFLP) of ribosomal, mitochondrial, and anonymous nuclear DNA was used to determine the origin of <em xmlns="" class="EmphasisTypeItalic">N. coccinea</em> var. <em xmlns="" class="EmphasisTypeItalic">faginata</em> in North America, and also to estimate haplotype and nucleotide diversity in this species as well as in <em xmlns="" class="EmphasisTypeItalic">N. ditissima.</em> Plante et al. [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>] investigated genetic diversity of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> and <em xmlns="" class="EmphasisTypeItalic">N. coccinea</em> var. <em xmlns="" class="EmphasisTypeItalic">faginata</em> using Random Amplified Polymorphic DNA (RAPD) and ribosomal DNA polymorphisms, and found that diversity of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> on various hardwood hosts in North America was higher than that of <em xmlns="" class="EmphasisTypeItalic">N. faginata</em> (<em xmlns="" class="EmphasisTypeItalic">Nectria coccinea var. faginata</em>). Based on a combined analysis of elongation factor 1-alpha (EF1-Î±), RNA polymerase II second largest subunit (RPB2), and Î²-tubulin gene regions, Castlebury et al. [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>] report that <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> appears to consist of two major groups; one with primarily North American isolates, and another with European isolates from <em xmlns="" class="EmphasisTypeItalic">Fagus</em>, <em xmlns="" class="EmphasisTypeItalic">Malus</em> and <em xmlns="" class="EmphasisTypeItalic">Salix</em>.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Fungi can be self-fertile and undergo sexual reproduction (haploid selfing) on a single colony or be self-sterile and require two colonies of different mating types for sexual reproduction (out-crossing) [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. However, many fungi can revert to haploid selfing if a suitable partner is not available.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Studies of fungal population biology can assist in understanding the evolution of disease in agriculture and how it is affected by, e.g., the life cycle of the pathogen [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. Population genetic analyses can also provide information about the amount of sexual recombination in fungal populations [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. Sexual pathogen populations can produce new combinations of alleles including fungicide resistance alleles, which allow the pathogen to rapidly adapt to changes in the environment [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Analysis of sibling ascospore offspring can be used to more closely define the nature of mating systems [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Out-crossing was thus concluded when RAPD- and AFLP-markers detected genetic variation among sibling single-ascospore progeny from apothecia of <em xmlns="" class="EmphasisTypeItalic">Cladonia xoerkeana</em>, <em xmlns="" class="EmphasisTypeItalic">C. galindezii</em>, and <em xmlns="" class="EmphasisTypeItalic">C. portentosa</em> [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. However, previously collected data for <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> are conflicting; based on the morphology of macroconidia and ascospores, El-Gholl et al. [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] reported this species to be homothallic while out-crossing was reported in an older study of ascospore morphology [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Fungal isolates</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Ascospore samples were collected from vegetatively propagated apple cultivars grown in seven different apple orchards in southern Sweden and from one orchard in Belgium (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span> and Table <span class="InternalRef"><a href="#Tab1">1</a></span>). In each orchard, 2â10 trees, located in different places in the orchard, with visible canker lesions were chosen, and pieces of bark and wood bearing perithecia were collected. Individual perithecia were surface-sterilized by immersion for 30 s in 50 % ethanol and for 30 s in 1.5 % sodium hypochloride, and then washed in sterile distilled water. Each perithecium was crushed in 100 Î¼l of sterile distilled water. About 50 Î¼l of the spore suspension was plated on agar (Difco) plates containing tetracycline (100 ppm) and incubated at 21 Â°C. One day later, 1â3 germinated ascospores were recovered individually using a tiny needle under a stereo-microscope, and transferred onto fresh 2 % malt extract agar (MEA, Merck) plates and left to grow for 14â18 days to produce single-ascospore cultures [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Correct species determination was ascertained by observation of colony morphology and by PCR amplification of DNA samples using species-specific primers previously developed for <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>].<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Map showing origin of isolates in Sweden and Belgium, A (black circle): Jonstorp, B (purple circle): BjÃ¤rred, C (yellow circle): Kivik, D (pink circle): BalsgÃ¥rd, E (green circle): JÃ¶nkÃ¶ping, F (blue circle): Julita, G (brown circle): Stockholm, H (orange circle): Gembloux</p>
                      </div></figcaption></figure><figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Geographic origin of the 44 isolates of <em class="EmphasisTypeItalic">N. ditissima</em> used in this study. Number 1â38 were isolated in 2013 and the remainder in 2014</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Number</p></th>
<th><p xmlns="" class="SimplePara">CBS</p></th>
<th><p xmlns="" class="SimplePara">Apple cultivar</p></th>
<th><p xmlns="" class="SimplePara">Origin</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139272</p></td>
<td><p xmlns="" class="SimplePara">Aroma</p></td>
<td><p xmlns="" class="SimplePara">Jonstorp, Sweden (N 56Â°13', E 12Â°40')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139273</p></td>
<td><p xmlns="" class="SimplePara">Ingrid Marie</p></td>
<td><p xmlns="" class="SimplePara">Jonstorp, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139271</p></td>
<td><p xmlns="" class="SimplePara">Rubinstar</p></td>
<td><p xmlns="" class="SimplePara">Jonstorp, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 4</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139269</p></td>
<td><p xmlns="" class="SimplePara">ÃkerÃ¶</p></td>
<td><p xmlns="" class="SimplePara">Julita, Sweden (N 59Â°11', E 16Â°1')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 5</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139270</p></td>
<td><p xmlns="" class="SimplePara">Norrstack</p></td>
<td><p xmlns="" class="SimplePara">Julita, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 6</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139388</p></td>
<td><p xmlns="" class="SimplePara">Oranie</p></td>
<td><p xmlns="" class="SimplePara">JÃ¶nkÃ¶ping, Sweden (N 57Â°46' N, E 14Â°9')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 7</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139268</p></td>
<td><p xmlns="" class="SimplePara">James Grieve</p></td>
<td><p xmlns="" class="SimplePara">JÃ¶nkÃ¶ping, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 8</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139390</p></td>
<td><p xmlns="" class="SimplePara">Discovery</p></td>
<td><p xmlns="" class="SimplePara">Kivik, Sweden (N 55Â°41', E 14Â°13')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 9</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139389</p></td>
<td><p xmlns="" class="SimplePara">Ingrid Marie</p></td>
<td><p xmlns="" class="SimplePara">Kivik, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 10</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139265</p></td>
<td><p xmlns="" class="SimplePara">John Standish</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden (N 56Â°6', E 14Â°9')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 11</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139267</p></td>
<td><p xmlns="" class="SimplePara">Brite Spur</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 12</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139327</p></td>
<td><p xmlns="" class="SimplePara">Pigeon</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 13</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139266</p></td>
<td><p xmlns="" class="SimplePara">McIntosh</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 14</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139255</p></td>
<td><p xmlns="" class="SimplePara">Tompkinâs King</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 15- RP1A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139260</p></td>
<td><p xmlns="" class="SimplePara">Red Delicious</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 16- RP2A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Red Delicious</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 17- RP3A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Red Delicious</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 18- RP3A2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Red Delicious</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 19- RP3A3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Red Delicious</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 20</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139262</p></td>
<td><p xmlns="" class="SimplePara">Beacon</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 21- HP1A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139263</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 22- HP1A2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 23- HP1A3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 24- HP3A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 25- HP3A2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 26- HP2A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 27- HP2A2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 28- HP2A3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Holsteiner Cox</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 29</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139264</p></td>
<td><p xmlns="" class="SimplePara">Freiherr von Berlepsch</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 30- GP1A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139261</p></td>
<td><p xmlns="" class="SimplePara">Gravensteiner</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 31- GP1A2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Gravensteiner</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 32- GP1A3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Gravensteiner</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 33- GP2A1</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Gravensteiner</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 34- GP2A2</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Gravensteiner</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 35- GP2A3</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">---------</p></td>
<td><p xmlns="" class="SimplePara">Gravensteiner</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 36</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139275</p></td>
<td><p xmlns="" class="SimplePara">Gloster</p></td>
<td><p xmlns="" class="SimplePara">BjÃ¤rred, Sweden (N 55Â°43', E 13Â°1')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 37</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139274</p></td>
<td><p xmlns="" class="SimplePara">Ingrid Marie</p></td>
<td><p xmlns="" class="SimplePara">BjÃ¤rred, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 38</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139503</p></td>
<td><p xmlns="" class="SimplePara">Elise</p></td>
<td><p xmlns="" class="SimplePara">BjÃ¤rred, Sweden</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 39</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139259</p></td>
<td><p xmlns="" class="SimplePara">Discovery</p></td>
<td><p xmlns="" class="SimplePara">Gembloux, Belgium N 54Â°34', E 4Â°42')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 40</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139257</p></td>
<td><p xmlns="" class="SimplePara">Api Etoile</p></td>
<td><p xmlns="" class="SimplePara">Gembloux, Belgium</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 41</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139256</p></td>
<td><p xmlns="" class="SimplePara">Baron</p></td>
<td><p xmlns="" class="SimplePara">Gembloux, Belgium</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 42</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139258</p></td>
<td><p xmlns="" class="SimplePara">Reinette</p></td>
<td><p xmlns="" class="SimplePara">Gembloux, Belgium</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 43</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139277</p></td>
<td><p xmlns="" class="SimplePara">Eva-Lotta</p></td>
<td><p xmlns="" class="SimplePara">Stockholm, Sweden (N 59Â°19', E 18Â°4')</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Isolate 44</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">139276</p></td>
<td><p xmlns="" class="SimplePara">SÃ¶dermanlandsÃ¤pple</p></td>
<td><p xmlns="" class="SimplePara">Stockholm, Sweden</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Isolator: Marjan Ghasemkhani</p>
                      <p xmlns="" class="SimplePara">CBS-KNAW Fungal Biodiversity Center, Utrecht, The Netherlands</p>
                      <p xmlns="" class="SimplePara">R: Red Delicious, G: Gravensteiner, H: Holsteiner Cox, P1-3: different perithecia on the same tree, A1-3: different single-ascospore isolates from the same perithecium</p>
                    </div></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">DNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">For DNA extraction of the isolates of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em>, an agar plug of each fungal culture was transferred into liquid medium, malt extract/glucose/peptone (MGP, 20 g malt extract, 20 g glucose and 1 g peptone in 1 L water at pH 5.5) [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>], and grown at room temperature on a shaker platform for 10 days. The mycelium was harvested by filtration through Whatman No. 1 filter paper, washed two times with distilled water, freeze-dried and stored at â20 Â°C until processing. DNA was extracted with DNeasy Plant Mini Kit (Qiagen GmbH, Hilden, Germany) according to manufacturerâs instructions. DNA quantification was performed using electrophoresis in 1.5 % agarose gel containing ethidium bromide and visualized by UV fluorescence, and concentrations and purity (260/280 nm &gt; 1.8) were estimated using a ND-1000 spectrophotometer (NanoDrop, Wilmington, USA).</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Amplification with SSR markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Seven primer pairs, previously developed from isolates of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> sampled on black birch, <em xmlns="" class="EmphasisTypeItalic">Betula nigra</em> [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>], were used for SSR analysis. Polymerase chain reaction (PCR) was performed in a total volume of 25 Î¼L in 2.5 Î¼l of 10X PCR buffer, 0.2 mM dNTP mix, 0.5 Î¼M of each primer, 1.5 mM MgCl<sub>2</sub>, 1 Unit of <em xmlns="" class="EmphasisTypeItalic">Taq</em> DNA polymerase (Thermo Scientific, San Jose, USA) and 10 ng genomic DNA. Amplifications were performed in a S1000 thermal cycler (BIO-RAD, San Francisco, USA) with the following cycling profile: 94 Â°C for 5 min, 10 cycles of 94 Â°C for 30 s, 58 Â°C for 45 s, and 72 Â°C for 45 s, followed by 30 cycles of 94 Â°C for 30 s, 53 Â°C for 45 s, and 72 Â°C for 45 s, a final extension step for 7 min at 72 Â°C.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Amplification with AFLP markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">AFLP analysis of DNA was carried out with the AFLP Microbial Fingerprinting kit (Applied Biosystems, Foster City, CA, USA) according to the recommendations of the manufacturer. Restriction-ligation reaction and preselective amplification of the AFLP procedure were carried out using the AFLPÂ® Ligation and Preselective Amplification Module of the Microbial Fingerprinting kit. Briefly, in the restriction-ligation reactions 10 ng of genomic DNA was simultaneously digested with <em xmlns="" class="EmphasisTypeItalic">Eco</em>RI and <em xmlns="" class="EmphasisTypeItalic">Mse</em>I endonucleases (New England Biolabs, Beverly, Mass., USA) and ligated to <em xmlns="" class="EmphasisTypeItalic">Eco</em>RI and <em xmlns="" class="EmphasisTypeItalic">Mse</em>I adaptors in a final volume of 11 Î¼l containing 10X T4 DNA ligase buffer with ATP, <em xmlns="" class="EmphasisTypeItalic">Mse</em>I, <em xmlns="" class="EmphasisTypeItalic">Eco</em>RI, T4 DNA ligase, BSA (all: New England Biolabs, Mass., USA), 0.5 M NaCl, and <em xmlns="" class="EmphasisTypeItalic">Eco</em>RI-adaptors and <em xmlns="" class="EmphasisTypeItalic">Mse</em>I-adaptors (Applied Biosystems). Restriction-ligation reactions were carried out for 2 h at 37 Â°C in a Bio-Rad S1000 thermal cycler. After dilution of DNA with TE<sub>0.1</sub> buffer (20 mM TrisâHCl, 0.1 mM EDTA, pH 8.0), for the preselective amplification, 4 Î¼l diluted DNA was amplified with the <em xmlns="" class="EmphasisTypeItalic">Eco</em>RI and <em xmlns="" class="EmphasisTypeItalic">Mse</em>I core primer sequences (Applied Biosystems) in a final volume of 20 Î¼l. Selective amplification was performed with 11 primer combinations of fluorescently labelled primers binding to the <em xmlns="" class="EmphasisTypeItalic">Eco</em>RI-adaptor and unlabeled primers binding to the <em xmlns="" class="EmphasisTypeItalic">Mse</em>I-adaptor: E+AA/M+CA, E+A/M+CT, E+T/M+CT, E+AC/M+CT, E+AC/M+CA, E+AG/M+CT, E+AT/M+CT, E+AA/M+CT, E+AG/M+CG, E+AC/M+CC, E+AC/M+CG, by using 3 Î¼l of the diluted preselective amplification reaction mixture in a final volume of 20 Î¼l. All PCR amplifications were carried out in a BioRad S1000 thermal cycler.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">In order to check the reproducibility of AFLP amplification, 5â8 randomly selected samples were added as technical replicates to each 96-well plate before running the PCR.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Fragment detection and analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">The reaction products were mixed with 0.1 x TE buffer and a ROXâ¢ dye-labeled size standard with fragments in the range 58â362 bp for SSR and 50â500 bp for AFLP markers and electrophoresed on an ABI 3130xl DNA analyzer (Applied Biosystems) using a 36 cm capillary array and POP-7â¢ polymer. Trace files were analyzed with GeneMarkerÂ® software (Version 1.85, SoftGenetics LLC, State College, PA, USA). AFLP markers producing weak or ambiguous signals, relative fluorescence unit (RFU) &gt;100, were not considered.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Data analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Distinct and well-resolved SSR and AFLP fragments were scored using the GeneMarkerÂ® program, and converted to binary data (one separate file for SSR data and one for AFLP data) based on the presence or absence of the discriminatory bands (1 for presence and 0 for absence). Genetic diversity was evaluated using the program POPGENE v 1.32 [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>], and the following parameters: number of alleles (NA, for SSR), number of polymorphic loci (NPL, for AFLP), percentage of polymorphic loci (PPL, for AFLP), Neiâs gene diversity (H, both marker types), and Shannonâs information index (I, both marker types). Pair-wise comparisons of single-ascospore isolates were conducted using Jaccardâs similarity coefficient (SJ), according to the formula <em xmlns="" class="EmphasisTypeItalic">SJ</em>â=â(<em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub><em xmlns="" class="EmphasisTypeItalic">ab</em>
                  </sub>)/(<em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub><em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub>â+ <em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub><em xmlns="" class="EmphasisTypeItalic">b</em>
                  </sub>ââ <em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub><em xmlns="" class="EmphasisTypeItalic">ab</em>
                  </sub>), where N<sub>ab</sub> is number of shared bands; N<sub>a</sub> and N<sub>b</sub> are number of bands in isolates a and b, respectively [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Genetic relationships among the studied single-ascospore isolates were also assessed by two multivariate principal coordinate analyses (PCoA) performed with NTSYS 2.02 (Applied Biostatistics, Setauket, NY, USA) for SSR and AFLP data, respectively. A dendrogram was constructed by binary band matching using the unweighted pairwise group method with arithmetic mean (UPGMA) with NTSYS 2.02 [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>] for single ascospore isolates sampled from perithecia on three trees at BalsgÃ¥rd and investigated with AFLP. A cophenetic correlation coefficient was calculated using a Mantel test [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>] in order to check the goodness-of-fit of the cluster analysis with the matrix on which it was based. A Mantel test was also performed to investigate possible correlation between two genetic distance matrices based on SSR and AFLP markers using NTSYS 2.02. An analysis of molecular variation (AMOVA) was calculated for partitioning of AFLP marker variation between and within orchards, using Arlequin 3.11 [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsanddiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsanddiscussion">Results and discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Resultsanddiscussion" data-test="collapse-section">
<section id="Sec10" class="Section2 RenderAsSection2"><h3 class="Heading">Evaluation of primers used</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Seven different SSR primer pairs were used (Table <span class="InternalRef"><a href="#Tab2">2</a></span>), with number of alleles varying from 2 to 7 per locus. Since all single-ascospore isolates collected from a single tree produced identical profiles, only one isolate for each tree (a total of 28) was used in subsequent analyses. Shannonâs information index varied from 0.15 for primer NdGGT39 to 0.51 for primer NdGGT3. NdGGT39 also yielded the highest Neiâs gene diversity (0.69) while NdGGT23 produced the lowest (0.13). The average gene diversity for the seven primer pairs was 0.28. When evaluated on <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> isolates collected from a single black birch population, gene diversity obtained with the same seven primer pairs was considerably higher, ranging from 0.11 to 0.86, and with an average of 0.48 [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. These results suggest that the fungal populations occurring on apple were less variable than the single population occurring on black birch.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Number of alleles (NA), Neiâs gene diversity (H) and Shannonâs information index (I) for seven SSR primer pairs used to study variation among 28 isolates of <em class="EmphasisTypeItalic">N. ditissima</em>
                                    </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Primer name</p></th>
<th><p xmlns="" class="SimplePara">NA</p></th>
<th><p xmlns="" class="SimplePara">H</p></th>
<th><p xmlns="" class="SimplePara">I</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdCAA3</p></td>
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">0.24</p></td>
<td><p xmlns="" class="SimplePara">0.38</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdCAA11</p></td>
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">0.17</p></td>
<td><p xmlns="" class="SimplePara">0.30</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdGGT2</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">0.22</p></td>
<td><p xmlns="" class="SimplePara">0.36</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdGGT3</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">0.35</p></td>
<td><p xmlns="" class="SimplePara">0.51</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdGGT23</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
<td><p xmlns="" class="SimplePara">0.26</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdGGT39</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">0.69</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NdGGT44</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
<td><p xmlns="" class="SimplePara">0.26</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td><p xmlns="" class="SimplePara">31</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">0.28</p></td>
<td><p xmlns="" class="SimplePara">0.32</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Eleven different AFLP primer combinations (Table <span class="InternalRef"><a href="#Tab3">3</a></span>) were used in this study to generate AFLP profiles for 44 isolates. A total of 391 fragments (ranging from 50 to 500 bp) were identified, of which 272 were polymorphic with reliable and clear reading. The lowest number of fragments was detected for the combination E+AC/M+CG and the highest for the combination E+T/M+CT, with 21 and 58 fragments, respectively. The percentage of polymorphism ranged from 55 % (in the combination E+A/M+CT) to 96 % (in the combination E+AC/M+CC). The combination E+AC/M+CC also produced the highest values for Neiâs gene diversity and Shannonâs information index while the combination E+AA/M+CT produced the lowest values. Evaluation of the technical replicates, which were included in each 96-well plate, demonstrated high reproducibility of produced band patterns (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Number of loci (NL), number of polymorphic loci (NPL), percentage of polymorphic loci (PPL), Neiâs gene diversity (H) and Shannonâs information index (I) for 11 AFLP primer combinations used to study variation among 44 isolates of <em class="EmphasisTypeItalic">N. ditissima</em>
                                    </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Primer combinations</p></th>
<th><p xmlns="" class="SimplePara">NL</p></th>
<th><p xmlns="" class="SimplePara">NPL</p></th>
<th><p xmlns="" class="SimplePara">PPL</p></th>
<th><p xmlns="" class="SimplePara">H</p></th>
<th><p xmlns="" class="SimplePara">I</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AA/M+CA</p></td>
<td><p xmlns="" class="SimplePara">40</p></td>
<td><p xmlns="" class="SimplePara">35</p></td>
<td><p xmlns="" class="SimplePara">87</p></td>
<td><p xmlns="" class="SimplePara">0.20</p></td>
<td><p xmlns="" class="SimplePara">0.33</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+A/M+CT</p></td>
<td><p xmlns="" class="SimplePara">56</p></td>
<td><p xmlns="" class="SimplePara">31</p></td>
<td><p xmlns="" class="SimplePara">55</p></td>
<td><p xmlns="" class="SimplePara">0.10</p></td>
<td><p xmlns="" class="SimplePara">0.17</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+T/M+CT</p></td>
<td><p xmlns="" class="SimplePara">58</p></td>
<td><p xmlns="" class="SimplePara">37</p></td>
<td><p xmlns="" class="SimplePara">64</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
<td><p xmlns="" class="SimplePara">0.22</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AC/M+CT</p></td>
<td><p xmlns="" class="SimplePara">29</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">62</p></td>
<td><p xmlns="" class="SimplePara">0.18</p></td>
<td><p xmlns="" class="SimplePara">0.28</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AC/M+CA</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">72</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
<td><p xmlns="" class="SimplePara">0.23</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AG/M+CT</p></td>
<td><p xmlns="" class="SimplePara">33</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">76</p></td>
<td><p xmlns="" class="SimplePara">0.17</p></td>
<td><p xmlns="" class="SimplePara">0.26</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AT/M+CT</p></td>
<td><p xmlns="" class="SimplePara">38</p></td>
<td><p xmlns="" class="SimplePara">27</p></td>
<td><p xmlns="" class="SimplePara">71</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
<td><p xmlns="" class="SimplePara">0.21</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AA/M+CT</p></td>
<td><p xmlns="" class="SimplePara">42</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">59</p></td>
<td><p xmlns="" class="SimplePara">0.09</p></td>
<td><p xmlns="" class="SimplePara">0.16</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AG/M+CG</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">73</p></td>
<td><p xmlns="" class="SimplePara">0.17</p></td>
<td><p xmlns="" class="SimplePara">0.27</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AC/M+CC</p></td>
<td><p xmlns="" class="SimplePara">27</p></td>
<td><p xmlns="" class="SimplePara">26</p></td>
<td><p xmlns="" class="SimplePara">96</p></td>
<td><p xmlns="" class="SimplePara">0.21</p></td>
<td><p xmlns="" class="SimplePara">0.34</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">E+AC/M+CG</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">67</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
<td><p xmlns="" class="SimplePara">0.22</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td><p xmlns="" class="SimplePara">391</p></td>
<td><p xmlns="" class="SimplePara">272</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average</p></td>
<td><p xmlns="" class="SimplePara">35</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">71</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
<td><p xmlns="" class="SimplePara">0.24</p></td>
</tr>
</tbody>
</table></div></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Electropherograms generated from 3 runs on an ABI 3130xl of the same sample that obtained with the primer combination E+T/M+CT. They indicated high reproducibility of AFLP marker</p>
                      </div></figcaption></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Overall, SSR markers showed a higher level of polymorphism (100 %) than the AFLP markers (71 %). These findings are consistent with other studies comparing the level of polymorphism detected with SSR and AFLP markers [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], and can be explained by the fact that SSR markers detect multiple alleles at a given locus, usually pre-selected for its high variability, whereas AFLP assays mainly detect single alleles at multiple loci randomly distributed in the genome.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">A Mantel test of the two genetic distance matrices derived from SSR and AFLP data, respectively, yielded a correlation coefficient of only 0.12 (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.89), which indicates that the two marker systems have not produced concordant results. Genetic distances are often closely correlated when different multi-locus marker methods (like AFLP, RAPD and ISSR) are compared [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. By contrast, comparison of a multi-locus method with SSR markers can result in poor correlation, especially when number of SSR bands evaluated is too low. The present study is a first report to assess genetic variation among and within populations of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> in European countries using both SSR and AFLP markers.</p>
              </section><section id="Sec11" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic variation between and within orchards</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Analysis of molecular variation (AMOVA) among 44 AFLP-screened isolates, obtained from eight different orchards, revealed only 11 % genetic differentiation among orchards, while the remaining 89 % of the variation resided within orchards. As expected, number of polymorphic bands detected in each orchard was closely associated with the number of isolates investigated. The highest number of polymorphic AFLP bands was observed in the orchard at BalsgÃ¥rd (201 bands among 26 isolates from 10 trees) followed by Gembloux (102 bands among four isolates) while JÃ¶nkÃ¶ping had the lowest number (42 bands among 2 isolates). The highest number of SSR bands was also found at BalsgÃ¥rd (14) and Gembloux (12) while the lowest number (3) was identified in Stockholm (two isolates sampled) (Table <span class="InternalRef"><a href="#Tab4">4</a></span>).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Number of polymorphic loci (NPL) and percentage of polymorphic loci (PPL) for AFLP, number of polymorphic alleles (NPA) and percentage of polymorphic alleles (PPA) for SSR, Neiâs gene diversity (H) and Shannonâs information index (I) for populations of <em class="EmphasisTypeItalic">Neonectria ditissima</em> from different orchards based on AFLP with 44 isolates and SSR markers with 28 isolates</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<tbody>
<tr class="noclass">
<td colspan="2"> </td>
<td><p xmlns="" class="SimplePara">NPL</p></td>
<td><p xmlns="" class="SimplePara">PPL</p></td>
<td><p xmlns="" class="SimplePara">H</p></td>
<td><p xmlns="" class="SimplePara">I</p></td>
</tr>
<tr class="noclass">
<td colspan="2"><p xmlns="" class="SimplePara">AFLP Marker</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass"><td colspan="6"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Genetic diversity of the eight populations</em>
                            </p></td></tr>
<tr class="noclass">
<td rowspan="8"><p xmlns="" class="SimplePara">Orchards</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
<td><p xmlns="" class="SimplePara">201</p></td>
<td><p xmlns="" class="SimplePara">51</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
<td><p xmlns="" class="SimplePara">0.21</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">BjÃ¤rred, Sweden</p></td>
<td><p xmlns="" class="SimplePara">88</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">0.10</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kivik, Sweden</p></td>
<td><p xmlns="" class="SimplePara">64</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">0.08</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Julita, Sweden</p></td>
<td><p xmlns="" class="SimplePara">60</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">0.07</p></td>
<td><p xmlns="" class="SimplePara">0.10</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Jonstorp, Sweden</p></td>
<td><p xmlns="" class="SimplePara">80</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">0.09</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">JÃ¶nkÃ¶ping, Sweden</p></td>
<td><p xmlns="" class="SimplePara">42</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">0.05</p></td>
<td><p xmlns="" class="SimplePara">0.07</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Stockholm, Sweden</p></td>
<td><p xmlns="" class="SimplePara">67</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">0.08</p></td>
<td><p xmlns="" class="SimplePara">0.12</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Gembloux, Belgium</p></td>
<td><p xmlns="" class="SimplePara">102</p></td>
<td><p xmlns="" class="SimplePara">26</p></td>
<td><p xmlns="" class="SimplePara">0.10</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
</tr>
<tr class="noclass">
<td> </td>
<td><p xmlns="" class="SimplePara">Mean</p></td>
<td><p xmlns="" class="SimplePara">88</p></td>
<td><p xmlns="" class="SimplePara">22.2</p></td>
<td><p xmlns="" class="SimplePara">0.09</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
</tr>
<tr class="noclass"><td colspan="6"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Genetic diversity among 44 isolates</em>
                            </p></td></tr>
<tr class="noclass">
<td colspan="2"><p xmlns="" class="SimplePara">44 isolates</p></td>
<td><p xmlns="" class="SimplePara">272</p></td>
<td><p xmlns="" class="SimplePara">69</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
<td><p xmlns="" class="SimplePara">0.23</p></td>
</tr>
<tr class="noclass">
<td colspan="2"> </td>
<td><p xmlns="" class="SimplePara">NPA</p></td>
<td><p xmlns="" class="SimplePara">PPA</p></td>
<td><p xmlns="" class="SimplePara">H</p></td>
<td><p xmlns="" class="SimplePara">I</p></td>
</tr>
<tr class="noclass">
<td colspan="2"><p xmlns="" class="SimplePara">SSR Marker</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass"><td colspan="6"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Genetic diversity of the eight populations</em>
                            </p></td></tr>
<tr class="noclass">
<td rowspan="8"><p xmlns="" class="SimplePara">Orchards</p></td>
<td><p xmlns="" class="SimplePara">BalsgÃ¥rd, Sweden</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">45</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
<td><p xmlns="" class="SimplePara">0.22</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">BjÃ¤rred, Sweden</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">29</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
<td><p xmlns="" class="SimplePara">0.18</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Kivik, Sweden</p></td>
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
<td><p xmlns="" class="SimplePara">0.16</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Julita, Sweden</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">0.08</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Jonstorp, Sweden</p></td>
<td><p xmlns="" class="SimplePara">6</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">0.09</p></td>
<td><p xmlns="" class="SimplePara">0.12</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">JÃ¶nkÃ¶ping, Sweden</p></td>
<td><p xmlns="" class="SimplePara">6</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">0.10</p></td>
<td><p xmlns="" class="SimplePara">0.13</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Stockholm, Sweden</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">0.05</p></td>
<td><p xmlns="" class="SimplePara">0.07</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Gembloux, Belgium</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">39</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
<td><p xmlns="" class="SimplePara">0.22</p></td>
</tr>
<tr class="noclass">
<td> </td>
<td><p xmlns="" class="SimplePara">Mean</p></td>
<td><p xmlns="" class="SimplePara">7.7</p></td>
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
</tr>
<tr class="noclass"><td colspan="6"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Genetic diversity among 28 isolates</em>
                            </p></td></tr>
<tr class="noclass">
<td colspan="2"><p xmlns="" class="SimplePara">28 isolates</p></td>
<td><p xmlns="" class="SimplePara">31</p></td>
<td><p xmlns="" class="SimplePara">100</p></td>
<td><p xmlns="" class="SimplePara">0.19</p></td>
<td><p xmlns="" class="SimplePara">0.32</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">When calculated at the species level, Neiâs gene diversity and Shannonâs information index were 0.14 and 0.23 based on AFLP data (44 isolates in total), respectively, and 0.19 and 0.32 based on SSR data (28 isolates), respectively (Table <span class="InternalRef"><a href="#Tab4">4</a></span>). At the population (i.e. orchard) level, BalsgÃ¥rd and Gembloux had the highest values for Neiâs gene diversity and Shannonâs information index for both AFLP and SSR markers. When based on AFLP data, Neiâs gene diversity and Shannonâs information index was 0.13 and 0.21, respectively, at BalsgÃ¥rd, and 0.05 and 0.07, respectively, at JÃ¶nkÃ¶ping where the lowest values were encountered. For SSR marker data, Gembloux and BalsgÃ¥rd had a Neiâs gene diversity of 0.15 and 0.14, respectively, and a Shannonâs information index of 0.22 for both orchards. The lowest values were instead encountered at Stockholm (0.05 and 0.07, respectively). It should, however, be noted that all the estimates of intra-population variation are based on very few isolates and are therefore unlikely to be representative of the species at large. In the previous study where these SSR primer pairs were developed and evaluated on a single <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> population sampled on black birch, Neiâs gene diversity was considerably higher (0.48) [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. The latter population was, however, also much larger (38 isolates) which may explain the difference in diversity.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Only 11 % of the variation resided between populations in the present study as assessed by the AFLP markers. Rather low values for population differentiation have been reported also in other studies on molecular variation in fungi. AFLP-based studies have thus shown that 7 % of the total genetic variability occurred between populations of <em xmlns="" class="EmphasisTypeItalic">Fusarium pseudograminearum</em> [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] and 14 % between populations of <em xmlns="" class="EmphasisTypeItalic">Phytophthora colocasiae</em> [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] while 11 % of SNP-based variability occurred between populations of <em xmlns="" class="EmphasisTypeItalic">Ustilaginoidea virens</em> [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. Low genetic differentiation is often hypothesized to result from high levels of gene flow thus preventing geographic subdivision [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Since the apple canker fungus occurs on cultivated crops, gene flow is probably augmented by movement of plant material from one area to another, e.g. from commercial plant nurseries in Belgium to orchards in Sweden as well as between orchards. Another possible explanation for the lack of strong inter-populational differentiation among isolates is a high degree of spontaneous mutations. A large number of spores is produced by the fungus in a short period of time, and the diversity could therefore be influenced by mutations [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>].</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Mating system of <em class="EmphasisTypeItalic">N. ditissima</em>
                        </h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">In order to determine the most likely mode of reproductive system in <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> and to quantify genetic variation among isolates at different levels of genetic relatedness, single-ascospore isolates derived from the same or from different perithecia on a single tree, were analyzed with both AFLP and SSR markers (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). A set of 45 single-ascospore isolates representing five different trees from different orchards (nine single-ascospore isolates from three perithecia on each tree) was analyzed with SSR, yielding identical profiles when isolates from the same tree were compared (data not shown).</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Subsequently, a set of 19 single-ascospore isolates sampled in the orchard at BalsgÃ¥rd (five from three perithecia on a âRed Deliciousâ tree, eight from three perithecia on a âHolsteiner Coxâ tree and six from two perithecia on a âGravensteinerâ tree) was analyzed with AFLP. Average pairwise similarity was analyzed at three levels; 1) among different single-ascospore isolates derived from the same perithecium, b) among single-ascospore isolates derived from different perithecia on the same tree, and c) among single-ascospore isolates derived from different trees (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). For level 1, pairwise comparisons of isolates using Jaccard's similarity coefficient ranged from 0.73 to 1.00 with a mean of 0.85. For level 2, Jaccard's similarity coefficients ranged from 0.60 to 0.87 with a mean of 0.76, suggesting that genetic relatedness is lower among isolates from different perithecia compared to isolates originating from the same perithecium. Finally, for level 3, Jaccard's similarity coefficients calculated on comparisons between isolates from different trees in the same orchard ranged from 0.44 to 0.70, with a mean of 0.56.<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">AFLP-based variation estimated with Jaccardâs similarity coefficients among single-ascospore isolates originating from within the same or from different perithecia of <em class="EmphasisTypeItalic">N. ditissima</em>
                                    </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th colspan="2"><p xmlns="" class="SimplePara">Within the same perithecium</p></th>
<th colspan="2"><p xmlns="" class="SimplePara">Between different perithecia on the same tree</p></th>
<th colspan="2"><p xmlns="" class="SimplePara">Between different perithecia on different trees in the same orchard</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">RP3 :</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">RP1ÃRP2ÃRP3:</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">RPÃGP:</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.96</p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.75</p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.56</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.85</p></td>
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.68</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.94</p></td>
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.60</p></td>
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.45</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">GP1:</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">GP1ÃGP2:</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">RPÃHP:</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.76</p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.77</p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.56</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.84</p></td>
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.87</p></td>
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.69</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.73</p></td>
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.61</p></td>
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.44</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">GP2 :</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">HP1ÃHP2ÃHP3:</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">GPÃHP:</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.85</p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.76</p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.58</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.90</p></td>
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.84</p></td>
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.70</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.80</p></td>
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.63</p></td>
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.45</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">HP1:</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">0.76</strong>
                            </p></td>
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">0.56</strong>
                            </p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.86</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.90</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.84</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">HP2:</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">0.86</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMax</p></td>
<td><p xmlns="" class="SimplePara">0.90</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMin</p></td>
<td><p xmlns="" class="SimplePara">0.83</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âMean</p></td>
<td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">0.85</strong>
                            </p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">R, G and H are different apple cultivars; R: âRed Deliciousâ, G: âGravensteinerâ, H: âHolsteiner Coxâ, P1-3: different perithecia of each cultivar</p>
                      <p xmlns="" class="SimplePara">Bold data indicate the total average pairwise similarity at three levels</p>
                    </div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Since fingerprint profiles evidently vary amongst sibling progeny, outcrossing seems to occur in <em xmlns="" class="EmphasisTypeItalic">N. ditissima.</em> Still, occasional selfing cannot be ruled out since two of the isolates from perithecium number 3 on âRed Deliciousâ were highly identical and the third differed only to a very minor extent from the other two (Jaccard's similarity coefficient = 0.94).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Both haploid selfing [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] and outcrossing [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>] has previously been reported in <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> according to morphological characterization of ascospores. In general, sexual recombination has the potential to increase genotype diversity since novel recombinants are created, and genetic recombination through sexual crosses have been shown to be correlated with genetic diversity [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. In the current study, genetic variation among single-ascospore isolates from different perithecia was higher than single-ascospore isolates from the same perithecium, which suggests that each tree has been colonized by several genetically different spores. Moreover, perithecia on different trees in the same orchard produced even more genetically different spores. Obviously, colonization of the fungus within an orchard involves several different, sexually derived ascospores. Management of diseases caused by sexually reproducing pathogens becomes more difficult due to the constant appearance of new genotypes, which increase the variation of features like fungicide resistance. In addition, <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> produces asexual spores during a large part of the year. Pathogens that reproduce both sexually and asexually are especially difficult to control, since aggressive isolates selected after sexual reproduction can increase asexually and then disperse widely within a short time-span [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>].</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2"><h3 class="Heading">Phenetic analyses</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Both SSR and AFLP data were used for estimating the distribution of genetic variation among and within orchards. An SSR-based principal coordinates analysis (PCoA) of similarities among 28 isolates did not produce any distinct groupings of isolates from different trees growing in the same orchard (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). Neither was an impact from geographical distances among the eight sampled orchards detected. In this analysis, the first two dimensions (dim-1 and dim-2) explained 65.7 and 6.9 % of the genetic variation, respectively. A PCoA based on AFLP data for the same 28 isolates similarly failed to produce any groupings of the material (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>), and 85.7 and 1.4 % of the genetic variation were explained by dim-1 and dim-2, respectively. Obviously, isolates from the same or geographically close orchards did not group together in the PCoA analysis with either marker type.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig3_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig3_HTML.gif" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Principal coordinate analysis (PCoA) based on SSR data with 28 isolates. (black diamond): Jonstorp, (blue diamond): Julita, (green diamond): JÃ¶nkÃ¶ping, (yellow diamond): Kivik, (pink diamond): BalsgÃ¥rd, (purple diamond): BjÃ¤rred, (orange diamond): Gembloux, (brown diamond): Stockholm</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig4_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig4_HTML.gif" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Principal coordinate analysis (PCoA) based on AFLP data with 28 isolates. (black diamond): Jonstorp, (blue diamond): Julita, (green diamond): JÃ¶nkÃ¶ping, (yellow diamond): Kivik, (pink diamond): BalsgÃ¥rd, (purple diamond): BjÃ¤rred, (orange diamond): Gembloux, (brown diamond): Stockholm</p>
                      </div></figcaption></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">When the AFLP data for single-ascospore isolates arising from different perithecia on three trees at BalsgÃ¥rd was subjected to a cluster analysis, three distinct groups were formed, each with isolates originating from the same tree, namely groups A (isolates from a âRed Deliciousâ tree at BalsgÃ¥rd), B (isolates from a âGravensteinerâ tree at BalsgÃ¥rd) and C (isolates from a âHolsteiner Coxâ tree at BalsgÃ¥rd) (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>). The cophenetic correlation coefficient was 0.94, indicating that the dendrogram was a good fit of the original data [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig5_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0011-3/MediaObjects/41065_2016_11_Fig5_HTML.gif" alt="Fig. 5"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Dendrogram obtained with UPGMA based on AFLP data of single-ascospore isolates arising from different perithecia of <em class="EmphasisTypeItalic">N. ditissima</em>. R: Red Delicious, G: Gravensteiner, H: Holsteiner Cox, P1-3: different perithecia, A1-3: different single-ascospores. Vertical bars show all isolates arising from different perithecia of <strong class="EmphasisTypeBold">a</strong> âGravensteinerâ, <strong class="EmphasisTypeBold">b</strong> âRed Deliciousâ and <strong class="EmphasisTypeBold">c</strong> âHolsteiner Coxâ</p>
                      </div></figcaption></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The identity of the cultivar from which the perithecia were sampled, did not appear to influence distribution of genetic variation of the resulting isolates. A possible explanation might be that there is random mating in the <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> population. By contrast, significant differences among fungal populations of <em xmlns="" class="EmphasisTypeItalic">Venturia inaequalis,</em> the causal agent of apple scab, collected from different apple cultivars indicated non-random mating [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Pathogenicity traits of fungal pathogens can be augmented by so-called pestification processes, which affect mainly growth and dispersal. Previous studies have thus shown that apple domestication has had a considerable impact on the modification of pathogenicity of <em xmlns="" class="EmphasisTypeItalic">V. inaequalis</em> [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] and that the aggressiveness of this fungus has increased during the domestication process [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. The extent to which these findings, obtained for the hemibiotrophic, ephemeral (trees are re-colonized each year) and multi-race fungus <em xmlns="" class="EmphasisTypeItalic">Venturia</em>, can be expected also in a necrotrophic and polyphagous fungus like <em xmlns="" class="EmphasisTypeItalic">Neonectria</em> is not known. Evolution of fitness-related traits is extremely dependent on the different life histories such as predominance of sexual over asexual reproduction [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Information on the distribution of genetic diversity of fungal populations may indicate how spread of the pathogen is accomplished, and this can be used to predict how long a control measure is likely to be effective [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. A previous study has shown that fungal populations with regular sexual reproduction recombine their genes into new combinations while populations with asexual reproduction have a more limited number of different gene combinations [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. In the present study, sexually produced spores were used to evaluate genetic variation. The intra-population variation detected is probably largely due to sexual recombination. Gene flow, mutation and selection are hypothesized as the most important mechanisms of variation in <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> populations. Strategies for improved control should therefore include restricted movement of infected plant material, or contaminated equipment.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">To the best of our knowledge, studies of genetic variation in <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> on <em xmlns="" class="EmphasisTypeItalic">Malus domestica</em> have not been reported before in Sweden or Belgium. In this study, genetic diversity of <em xmlns="" class="EmphasisTypeItalic">N. ditissima</em> was estimated at an inter-population level using isolates from different apple orchards in Sweden and from one orchard in Belgium since trees grown in commercial Swedish orchards are often imported from Belgium. In addition, the mating system of the pathogen was investigated at an intra-population level by comparing sibling single-ascospore progeny either from a single perithecium or from different perithecia of the same tree. First, Simple Sequence Repeat (SSR) was used, which is a single-locus method with co-dominantly inherited bands. Each primer pair produces one band in each isolate (the studied species is haploid). These bands can have different size in different isolates. Since total number of polymorphic SSR alleles turned out to be rather low, the Amplified Fragment Length Polymorphism (AFLP) method was applied to complement the SSR data. AFLP is a multi-locus method with dominantly inherited bands, with each primer pair producing multiple bands in each isolate. When several isolates are compared, individual AFLP bands are either present or absent [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec15" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Abbreviations" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">AFLP, amplified fragment length polymorphism; AMOVA, analysis of molecular variance; EF1-Î±, elongation factor 1-alpha; ISSR, application of inter simple sequence repeat; PCoA, principal coordinates analysis; RAPD, random amplified polymorphic DNA; RPB2, RNA polymerase II second largest subunit; SSR, simple sequence repeat; UPGMA, unweighted pairwise group method with arithmetic mean</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgements</h3>
<p xmlns="" class="SimplePara">We are grateful to Eva Eneblad, Dr. Marc Lateur and BjÃ¶rn Kalin for providing fungal material.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par33" class="Para">Financial support was obtained from the Nordic Ministries of Food and Agriculture through the Nordic collaboration on Public-Private Partnership for pre-breeding, PPP, administered by NordGen.</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authorsâ contributions</h3>
                <p id="Par34" class="Para">MG performed the experiments, analyzed the data and wrote the manuscript. HN designed the experiments and HN and LG-G participated in the sampling of the isolates. EL helped with the isolation of <em xmlns="" class="EmphasisTypeItalic">Neonectria ditissima</em>. EL, HN and LG-G helped to develop the concept of the paper and assisted in manuscript writing. All authors read and approved the final manuscript.</p>
              </div>
<div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par35" class="Para">The authors declare that they have no competing interests.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Plant Breeding, Swedish University of Agricultural Sciences, Box 101, 230 53 Alnarp, Sweden</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Department of Plant Protection Biology, Swedish University of Agricultural Sciences, Box 102, 230 53 Alnarp, Sweden</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Department of Plant Breeding, Swedish University of Agricultural Sciences, BalsgÃ¥rd, FjÃ¤lkestadsvÃ¤gen 459, 29194 Kristianstad, Sweden</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bernier L, Hubbes M. Mutations in <em class="EmphasisTypeItalic">Ophiostoma ulmi</em> induced by N-methyl-N'-nitro-N-nitroso-guanidine. Can J Bot. 1990;68:225â31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/b90-031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutations%20in%20Ophiostoma%20ulmi%20induced%20by%20N-methyl-N%27-nitro-N-nitroso-guanidine&amp;author=L.%20Bernier&amp;author=M.%20Hubbes&amp;journal=Can%20J%20Bot&amp;volume=68&amp;pages=225-31&amp;publication_year=1990"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Billiard S, LÃ³pezâVillavicencio M, Hood M, Giraud T. Sex, outcrossing and mating types: unsolved questions in fungi and beyond. J Evol Biol. 2012;25(6):1020â38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1420-9101.2012.02495.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22515640"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sex%2C%20outcrossing%20and%20mating%20types%3A%20unsolved%20questions%20in%20fungi%20and%20beyond&amp;author=S.%20Billiard&amp;author=M.%20L%C3%B3pez%E2%80%90Villavicencio&amp;author=M.%20Hood&amp;author=T.%20Giraud&amp;journal=J%20Evol%20Biol&amp;volume=25&amp;issue=6&amp;pages=1020-38&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Carlile MJ, Watkinson SC, Gooday GW. The Fungi. Houston: Gulf Professional Publishing; 2001.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Fungi&amp;author=MJ.%20Carlile&amp;author=SC.%20Watkinson&amp;author=GW.%20Gooday&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Castlebury LA, Rossman AY, Hyten AS. Phylogenetic relationships of <em class="EmphasisTypeItalic">Neonectria/Cylindrocarpon</em> on <em class="EmphasisTypeItalic">Fagus</em> in North America. Can J Bot. 2006;84:1417â33.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/b06-105"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20of%20Neonectria%2FCylindrocarpon%20on%20Fagus%20in%20North%20America&amp;author=LA.%20Castlebury&amp;author=AY.%20Rossman&amp;author=AS.%20Hyten&amp;journal=Can%20J%20Bot&amp;volume=84&amp;pages=1417-33&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chatelet DS, Wistrom CM, Purcell AH, Rost TL, Matthews MA. Xylem structure of four grape varieties and 12 alternative hosts to the xylem-limited bacterium <em class="EmphasisTypeItalic">Xylella fastidious</em>. Ann Bot-London. 2011;108:73â85.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/aob/mcr106"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Xylem%20structure%20of%20four%20grape%20varieties%20and%2012%20alternative%20hosts%20to%20the%20xylem-limited%20bacterium%20Xylella%20fastidious%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=DS.%20Chatelet&amp;author=CM.%20Wistrom&amp;author=AH.%20Purcell&amp;author=TL.%20Rost&amp;author=MA.%20Matthews&amp;journal=Ann%20Bot-London&amp;volume=108&amp;pages=73-85&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">CPCI. Crop Protection Compendium on Internet. Wallingford: CAB International; 2014. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.cabi.org/cpc" target="_blank" class="ext-link"><span class="RefSource">www.cabi.org/cpc</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Crop%20Protection%20Compendium%20on%20Internet&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Czembor PC, Arseniuk E. Segregation and recombination of PCR based markers in sexual progeny of <em class="EmphasisTypeItalic">Phaeosphaeria</em> species. Mycol Res. 2000;104:919â26.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/S095375620000246X"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Segregation%20and%20recombination%20of%20PCR%20based%20markers%20in%20sexual%20progeny%20of%20Phaeosphaeria%20species&amp;author=PC.%20Czembor&amp;author=E.%20Arseniuk&amp;journal=Mycol%20Res&amp;volume=104&amp;pages=919-26&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">De Gracia M, Cascales M, Expert P, Bellanger M-N, Le Cam B, Lemaire C. How did host domestication modify life history traits of its pathogens? PLoS One. 2015;10:e0122909.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0122909"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26091067"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4475019"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=How%20did%20host%20domestication%20modify%20life%20history%20traits%20of%20its%20pathogens%3F&amp;author=M.%20Gracia&amp;author=M.%20Cascales&amp;author=P.%20Expert&amp;author=M-N.%20Bellanger&amp;author=B.%20Cam&amp;author=C.%20Lemaire&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0122909&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">El-Gholl N, Barnard E, Schroeder R. Homothallism in <em class="EmphasisTypeItalic">Nectria galligena</em>. Can J Bot. 1986;64:902â3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/b86-119"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Homothallism%20in%20Nectria%20galligena%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=N.%20El-Gholl&amp;author=E.%20Barnard&amp;author=R.%20Schroeder&amp;journal=Can%20J%20Bot&amp;volume=64&amp;pages=902-3&amp;publication_year=1986"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Excoffier L, Laval G, Schneider S. Arlequin (version 3.0): an integrated software package for population genetics data analysis. Evol Bioinform Online. 2005;1:47.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Arlequin%20%28version%203.0%29%3A%20an%20integrated%20software%20package%20for%20population%20genetics%20data%20analysis&amp;author=L.%20Excoffier&amp;author=G.%20Laval&amp;author=S.%20Schneider&amp;journal=Evol%20Bioinform%20Online&amp;volume=1&amp;pages=47&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fofana IJ, Silue S, Diarrassouba N, Kadio AA, Sangare A. Comparative analyses of amplified fragment length polymorphism (AFLP) and simple sequence repeat (SSR) in genetic diversity of Teak (<em class="EmphasisTypeItalic">Tectona grandis</em> Lf). Int J Adv Agric Res. 2013;1:114â23.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20of%20amplified%20fragment%20length%20polymorphism%20%28AFLP%29%20and%20simple%20sequence%20repeat%20%28SSR%29%20in%20genetic%20diversity%20of%20Teak%20%28Tectona%20grandis%20Lf%29&amp;author=IJ.%20Fofana&amp;author=S.%20Silue&amp;author=N.%20Diarrassouba&amp;author=AA.%20Kadio&amp;author=A.%20Sangare&amp;journal=Int%20J%20Adv%20Agric%20Res&amp;volume=1&amp;pages=114-23&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gargouri S, Hamza S, Hajlaoui M. AFLP analysis of the genetic variability and population structure of the wheat crown rot fungus <em class="EmphasisTypeItalic">Fusarium pseudograminearum</em> in Tunisia. Tunis J Plant Prot. 2006;1:93â104.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AFLP%20analysis%20of%20the%20genetic%20variability%20and%20population%20structure%20of%20the%20wheat%20crown%20rot%20fungus%20Fusarium%20pseudograminearum%20in%20Tunisia&amp;author=S.%20Gargouri&amp;author=S.%20Hamza&amp;author=M.%20Hajlaoui&amp;journal=Tunis%20J%20Plant%20Prot&amp;volume=1&amp;pages=93-104&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ghasemkhani M, Holefors A, Marttila S, Dalman K, Zborowska A, Rur M, Rees-George J, Nybom H, Everett KR, Scheper RWA, Garkava-Gustavsson L. Real-time PCR for detection and quantification, and histological characterization of <em class="EmphasisTypeItalic">Neonectria ditissima</em> in apple trees. Trees. 2016. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1007/s00468-015-1350-9" target="_blank" class="ext-link"><span class="RefSource">10.1007/s00468-015-1350-9</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Real-time%20PCR%20for%20detection%20and%20quantification%2C%20and%20histological%20characterization%20of%20Neonectria%20ditissima%20in%20apple%20trees&amp;author=M.%20Ghasemkhani&amp;author=A.%20Holefors&amp;author=S.%20Marttila&amp;author=K.%20Dalman&amp;author=A.%20Zborowska&amp;author=M.%20Rur&amp;author=J.%20Rees-George&amp;author=H.%20Nybom&amp;author=KR.%20Everett&amp;author=RWA.%20Scheper&amp;author=L.%20Garkava-Gustavsson&amp;journal=Trees&amp;publication_year=2016&amp;doi=10.1007%2Fs00468-015-1350-9"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">KrÃ¼ger J. Zur Genetik von <em class="EmphasisTypeItalic">Nectria galligena</em> Bres. J Phytopathol. 1974;79:320â42.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1439-0434.1974.tb02715.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Zur%20Genetik%20von%20Nectria%20galligena%20Bres&amp;author=J.%20Kr%C3%BCger&amp;journal=J%20Phytopathol&amp;volume=79&amp;pages=320-42&amp;publication_year=1974"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lenormand T. Gene flow and the limits to natural selection. Trends Ecol Evol. 2002;17:183â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0169-5347(02)02497-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gene%20flow%20and%20the%20limits%20to%20natural%20selection&amp;author=T.%20Lenormand&amp;journal=Trends%20Ecol%20Evol&amp;volume=17&amp;pages=183-9&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li L, Wanapu C, Huang X, Huang T, Li Q, Peng Y, Huang G. Comparison of AFLP and SSR for genetic diversity analysis of <em class="EmphasisTypeItalic">Brassica napus</em> hybrids. J Agric Sci. 2011;3:101â10.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20AFLP%20and%20SSR%20for%20genetic%20diversity%20analysis%20of%20Brassica%20napus%20hybrids&amp;author=L.%20Li&amp;author=C.%20Wanapu&amp;author=X.%20Huang&amp;author=T.%20Huang&amp;author=Q.%20Li&amp;author=Y.%20Peng&amp;author=G.%20Huang&amp;journal=J%20Agric%20Sci&amp;volume=3&amp;pages=101-10&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mahoney EM, Milgroom MG, Sinclair WA. Origin, genetic diversity and population structure of <em class="EmphasisTypeItalic">Nectria coccinea</em> var. <em class="EmphasisTypeItalic">faginata</em> in North America. Mycologia. 1999;91:583â92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/3761243"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Origin%2C%20genetic%20diversity%20and%20population%20structure%20of%20Nectria%20coccinea%20var.%20faginata%20in%20North%20America&amp;author=EM.%20Mahoney&amp;author=MG.%20Milgroom&amp;author=WA.%20Sinclair&amp;journal=Mycologia&amp;volume=91&amp;pages=583-92&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mantel N. The detection of disease clustering and a generalized regression approach. Cancer Res. 1967;27:209â20.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6018555"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20detection%20of%20disease%20clustering%20and%20a%20generalized%20regression%20approach&amp;author=N.%20Mantel&amp;journal=Cancer%20Res&amp;volume=27&amp;pages=209-20&amp;publication_year=1967"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Marra RE, Corwin JA. Isolation and characterization of codominant markers for the perennial canker fungal pathogen <em class="EmphasisTypeItalic">Neonectria ditissima</em>. Mol Ecol Resour. 2009;9:906â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1755-0998.2008.02438.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21564786"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20codominant%20markers%20for%20the%20perennial%20canker%20fungal%20pathogen%20Neonectria%20ditissima%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=RE.%20Marra&amp;author=JA.%20Corwin&amp;journal=Mol%20Ecol%20Resour&amp;volume=9&amp;pages=906-9&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McCracken AR, Berrie A, Barbara DJ, Locke T, Cooke LR, Phelps K, Swinburne TR, Brown AE, Ellerker B, Langrell SRH. Relative significance of nursery infections and orchard inoculum in the development and spread of apple canker (<em class="EmphasisTypeItalic">Nectria galligena</em>) in young orchards. Plant Pathol. 2003;52:553â66.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-3059.2003.00924.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Relative%20significance%20of%20nursery%20infections%20and%20orchard%20inoculum%20in%20the%20development%20and%20spread%20of%20apple%20canker%20%28Nectria%20galligena%29%20in%20young%20orchards&amp;author=AR.%20McCracken&amp;author=A.%20Berrie&amp;author=DJ.%20Barbara&amp;author=T.%20Locke&amp;author=LR.%20Cooke&amp;author=K.%20Phelps&amp;author=TR.%20Swinburne&amp;author=AE.%20Brown&amp;author=B.%20Ellerker&amp;author=SRH.%20Langrell&amp;journal=Plant%20Pathol&amp;volume=52&amp;pages=553-66&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McDonald BA. The population genetics of fungi: tools and techniques. Phytopathology. 1997;87:448â53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1094/PHYTO.1997.87.4.448"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18945126"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20population%20genetics%20of%20fungi%3A%20tools%20and%20techniques&amp;author=BA.%20McDonald&amp;journal=Phytopathology&amp;volume=87&amp;pages=448-53&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McDonald BA, Linde C. Pathogen population genetics, evolutionary potential, and durable resistance. Annu Rev Phytopathol. 2002;40:349â79.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.phyto.40.120501.101443"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12147764"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Pathogen%20population%20genetics%2C%20evolutionary%20potential%2C%20and%20durable%20resistance&amp;author=BA.%20McDonald&amp;author=C.%20Linde&amp;journal=Annu%20Rev%20Phytopathol&amp;volume=40&amp;pages=349-79&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McDonald BA, McDermott JM. Population Genetics of Plant Pathogenic Fungi. Bioscience. 1993;43:311â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/1312063"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20Genetics%20of%20Plant%20Pathogenic%20Fungi&amp;author=BA.%20McDonald&amp;author=JM.%20McDermott&amp;journal=Bioscience&amp;volume=43&amp;pages=311-9&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nath VS, Senthil M, Hegde VM, Jeeva ML, Misra RS, Veena SS, Raj M. Genetic diversity of <em class="EmphasisTypeItalic">Phytophthora colocasiae</em> isolates in India based on AFLP analysis. 3 Biotech. 2013;3:297â305.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s13205-012-0101-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Phytophthora%20colocasiae%20isolates%20in%20India%20based%20on%20AFLP%20analysis&amp;author=VS.%20Nath&amp;author=M.%20Senthil&amp;author=VM.%20Hegde&amp;author=ML.%20Jeeva&amp;author=RS.%20Misra&amp;author=SS.%20Veena&amp;author=M.%20Raj&amp;journal=3%20Biotech&amp;volume=3&amp;pages=297-305&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Plante F, Hamelin RC, Bernier L. A comparative study of genetic diversity of populations of <em class="EmphasisTypeItalic">Nectria galligena</em> and <em class="EmphasisTypeItalic">N. coccinea</em> var. <em class="EmphasisTypeItalic">faginata</em> in North America. Mycol Res. 2002;106:183â93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/S0953756201005329"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comparative%20study%20of%20genetic%20diversity%20of%20populations%20of%20Nectria%20galligena%20and%20N.%20coccinea%20var.%20faginata%20in%20North%20America&amp;author=F.%20Plante&amp;author=RC.%20Hamelin&amp;author=L.%20Bernier&amp;journal=Mycol%20Res&amp;volume=106&amp;pages=183-93&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pringle A, Taylor JW. The fitness of filamentous fungi. Trends Microbiol. 2002;10:474â81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0966-842X(02)02447-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12377558"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20fitness%20of%20filamentous%20fungi&amp;author=A.%20Pringle&amp;author=JW.%20Taylor&amp;journal=Trends%20Microbiol&amp;volume=10&amp;pages=474-81&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seymour FA, Crittenden PD, Dickinson MJ, Paoletti M, Montiel D, Cho L, Dyer PS. Breeding systems in the lichen-forming fungal genus <em class="EmphasisTypeItalic">Cladonia</em>. Fungal Genet Biol. 2005;42:554â63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.fgb.2005.03.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15893256"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Breeding%20systems%20in%20the%20lichen-forming%20fungal%20genus%20Cladonia%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=FA.%20Seymour&amp;author=PD.%20Crittenden&amp;author=MJ.%20Dickinson&amp;author=M.%20Paoletti&amp;author=D.%20Montiel&amp;author=L.%20Cho&amp;author=PS.%20Dyer&amp;journal=Fungal%20Genet%20Biol&amp;volume=42&amp;pages=554-63&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shaner G, Finney RE. Effect of nitrogen-fertilization on expression of slow-mildewing resistance in Knox wheat. Phytopathology. 1977;67:1051â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1094/Phyto-67-1051"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20nitrogen-fertilization%20on%20expression%20of%20slow-mildewing%20resistance%20in%20Knox%20wheat&amp;author=G.%20Shaner&amp;author=RE.%20Finney&amp;journal=Phytopathology&amp;volume=67&amp;pages=1051-6&amp;publication_year=1977"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun X, Kang S, Zhang Y, Tan X, Yu Y, He H, Zhang X, Liu Y, Wang S, Sun W. Genetic diversity and population structure of rice pathogen <em class="EmphasisTypeItalic">Ustilaginoidea virens</em> in China. PLoS One. 2013;8:e76879.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0076879"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24098811"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3786968"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20population%20structure%20of%20rice%20pathogen%20Ustilaginoidea%20virens%20in%20China&amp;author=X.%20Sun&amp;author=S.%20Kang&amp;author=Y.%20Zhang&amp;author=X.%20Tan&amp;author=Y.%20Yu&amp;author=H.%20He&amp;author=X.%20Zhang&amp;author=Y.%20Liu&amp;author=S.%20Wang&amp;author=W.%20Sun&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e76879&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tamura K, Dudley J, Nei M, Kumar S. MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0. Mol Biol Evol. 2007;24:1596â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msm092"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17488738"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MEGA4%3A%20molecular%20evolutionary%20genetics%20analysis%20%28MEGA%29%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=1596-9&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van LÃª A, Gladieux P, Lemaire C, Cornille A, Giraud T, Durel CE, Caffier V, Le Cam B.Evolution of pathogenicity traits in the apple scab fungal pathogen in response to the domestication of its host. Evol Appl. 2012;5:694â704.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1752-4571.2012.00246.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%20of%20pathogenicity%20traits%20in%20the%20apple%20scab%20fungal%20pathogen%20in%20response%20to%20the%20domestication%20of%20its%20host&amp;author=A.%20L%C3%AA&amp;author=P.%20Gladieux&amp;author=C.%20Lemaire&amp;author=A.%20Cornille&amp;author=T.%20Giraud&amp;author=CE.%20Durel&amp;author=V.%20Caffier&amp;author=B.%20Cam&amp;journal=Evol%20Appl&amp;volume=5&amp;pages=694-704&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Walser J, Gugerli F, Holderegger R, Kuonen D, Scheidegger C. Recombination and clonal propagation in different populations of the lichen <em class="EmphasisTypeItalic">Lobaria pulmonaria</em>. Heredity. 2004;93:322â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/sj.hdy.6800505"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15241450"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recombination%20and%20clonal%20propagation%20in%20different%20populations%20of%20the%20lichen%20Lobaria%20pulmonaria%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=J.%20Walser&amp;author=F.%20Gugerli&amp;author=R.%20Holderegger&amp;author=D.%20Kuonen&amp;author=C.%20Scheidegger&amp;journal=Heredity&amp;volume=93&amp;pages=322-9&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Weber RWS. Biology and control of the apple canker fungus <em class="EmphasisTypeItalic">Neonectria ditissima</em> (syn. <em class="EmphasisTypeItalic">N. galligena</em>) from a Northwestern European perspective. Erwerbs-Obstbau. 2014;56:95â107.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10341-014-0210-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biology%20and%20control%20of%20the%20apple%20canker%20fungus%20Neonectria%20ditissima%20%28syn.%20N.%20galligena%29%20from%20a%20Northwestern%20European%20perspective&amp;author=RWS.%20Weber&amp;journal=Erwerbs-Obstbau&amp;volume=56&amp;pages=95-107&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Weising K, Nybom H, Pfenninger M, Wolff K, Kahl G. DNA fingerprinting in plants: principles, methods, and applications. Boca Raton: CRC press; 2005.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1201/9781420040043"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20fingerprinting%20in%20plants%3A%20principles%2C%20methods%2C%20and%20applications&amp;author=K.%20Weising&amp;author=H.%20Nybom&amp;author=M.%20Pfenninger&amp;author=K.%20Wolff&amp;author=G.%20Kahl&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu X, Harvey N, Roberts A, Barbara D. Population variation of apple scab (<em class="EmphasisTypeItalic">Venturia inaequalis</em>) within mixed orchards in the UK. Eur J Plant Pathol. 2013;135:97â104.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10658-012-0068-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20variation%20of%20apple%20scab%20%28Venturia%20inaequalis%29%20within%20mixed%20orchards%20in%20the%20UK&amp;author=X.%20Xu&amp;author=N.%20Harvey&amp;author=A.%20Roberts&amp;author=D.%20Barbara&amp;journal=Eur%20J%20Plant%20Pathol&amp;volume=135&amp;pages=97-104&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yeh FC, Yang R, Boyle TB, Ye Z, Mao JX. POPGENE, the user-friendly shareware for population genetic analysis. Canada: Molecular Biology and Biotechnology Centre, University of Alberta; 1997.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=POPGENE%2C%20the%20user-friendly%20shareware%20for%20population%20genetic%20analysis&amp;author=FC.%20Yeh&amp;author=R.%20Yang&amp;author=TB.%20Boyle&amp;author=Z.%20Ye&amp;author=JX.%20Mao&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© The Author(s) 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0011-3" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-016-0011-3">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0011-3" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0011-3?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0011-3 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0011-3 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0011-3 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0011-3">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0011-3">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0011-3"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0011-3">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0011-3">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0011-3"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0011-3">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0011-3">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0011-3"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0011-3">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0011-3">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0011-3"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0011-3">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0011-3">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0011-3"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-016-0011-3/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-016-0011-3">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0011-3">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0011-3" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-016-0011-3&amp;volume=153&amp;issue=1&amp;title=Assessment+of+diversity+and+genetic+relationships+of+Neonectria+ditissima+%3A+the+causal+agent+of+fruit+tree+canker&amp;author_name=Marjan+Ghasemkhani&amp;start_page=1&amp;end_page=11">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-016-0011-3" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
