<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>A C-banded karyotype of mitotic chromosomes in diploid purple coneflower (Echinacea purpurea L.) | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Aneuploid ermpglasm is an important resource for genetic studies and identification of individual chromosomes in the cells of the aneuploid is an important step. The karyotype has already been established for purple coneflower (Echinacea purpurea L.), but due to the high similarity in the morphology of several pairs of chromosomes in this species, it cannot be used to identify individual chromosomes in its own complement. The objectives of this study are to develop and evaluate the Giemsa C-banding technique for the purpose of identifying the individual chromosomes in Echinacea purpurea. The established karyotype with C-bands showed that all the 11 pairs of chromosomes possessed centromeric bands. Telomeric bands appeared most frequently in almost all the chromosomes with only two exceptions, the short arm of the chromosome 9 and the long arm of the chromosome 10. Intercalary bands were found mainly in the long arm of some chromosomes with only two exceptions, the chromosomes 1 and 2 that had intercalary bands on both arms. The chromosome 4 was the only chromosome where intercalary bands were absent. Chromosomes in E. purpurea could be stained with Giemsa to bear C-bands. By classifying the chromosomes into groups and judging the C-bands, each chromosome could be identified. The methods established in this study might be used for the identification of chromosome constitution in aneuploid E. purpurea created in a breeding program.">
<meta name="description" content="Aneuploid ermpglasm is an important resource for genetic studies and identification of individual chromosomes in the cells of the aneuploid is an important step. The karyotype has already been established for purple coneflower (Echinacea purpurea L.), but due to the high similarity in the morphology of several pairs of chromosomes in this species, it cannot be used to identify individual chromosomes in its own complement. The objectives of this study are to develop and evaluate the Giemsa C-banding technique for the purpose of identifying the individual chromosomes in Echinacea purpurea. The established karyotype with C-bands showed that all the 11 pairs of chromosomes possessed centromeric bands. Telomeric bands appeared most frequently in almost all the chromosomes with only two exceptions, the short arm of the chromosome 9 and the long arm of the chromosome 10. Intercalary bands were found mainly in the long arm of some chromosomes with only two exceptions, the chromosomes 1 and 2 that had intercalary bands on both arms. The chromosome 4 was the only chromosome where intercalary bands were absent. Chromosomes in E. purpurea could be stained with Giemsa to bear C-bands. By classifying the chromosomes into groups and judging the C-bands, each chromosome could be identified. The methods established in this study might be used for the identification of chromosome constitution in aneuploid E. purpurea created in a breeding program.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="A C-banded karyotype of mitotic chromosomes in diploid purple coneflower ( Echinacea purpurea L.)">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-11-22">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Aneuploid ermpglasm is an important resource for genetic studies and identification of individual chromosomes in the cells of the aneuploid is an important step. The karyotype has already been established for purple coneflower (Echinacea purpurea L.), but due to the high similarity in the morphology of several pairs of chromosomes in this species, it cannot be used to identify individual chromosomes in its own complement. The objectives of this study are to develop and evaluate the Giemsa C-banding technique for the purpose of identifying the individual chromosomes in Echinacea purpurea. The established karyotype with C-bands showed that all the 11 pairs of chromosomes possessed centromeric bands. Telomeric bands appeared most frequently in almost all the chromosomes with only two exceptions, the short arm of the chromosome 9 and the long arm of the chromosome 10. Intercalary bands were found mainly in the long arm of some chromosomes with only two exceptions, the chromosomes 1 and 2 that had intercalary bands on both arms. The chromosome 4 was the only chromosome where intercalary bands were absent. Chromosomes in E. purpurea could be stained with Giemsa to bear C-bands. By classifying the chromosomes into groups and judging the C-bands, each chromosome could be identified. The methods established in this study might be used for the identification of chromosome constitution in aneuploid E. purpurea created in a breeding program.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-11-22">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="14">
<meta name="prism.copyright" content="2016 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0016-y">
<meta name="prism.doi" content="doi:10.1186/s41065-016-0016-y">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0016-y">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0016-y">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="A C-banded karyotype of mitotic chromosomes in diploid purple coneflower ( Echinacea purpurea L.)">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/11/22">
<meta name="citation_firstpage" content="14">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-016-0016-y">
<meta name="DOI" content="10.1186/s41065-016-0016-y">
<meta name="citation_doi" content="10.1186/s41065-016-0016-y">
<meta name="dc.creator" content="Weizhen Jiang">
<meta name="dc.creator" content="Qingling Li">
<meta name="dc.creator" content="Xiaolu Chen">
<meta name="dc.creator" content="Yi Ren">
<meta name="dc.creator" content="Rong Chen">
<meta name="dc.creator" content="Hong Wu">
<meta name="dc.creator" content="Yuesheng Yang">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_title=
                              Echinacea exalted! The botany, culture, history and medicinal uses of the purple conflowers; citation_publication_date=1984; citation_id=CR1; citation_author=S Foster; citation_publisher=New Life Farm. Inc.">
<meta name="citation_reference" content="citation_journal_title=Herbalgram; citation_title=
                              Echinacea-a literature review; citation_author=CR Hobbs; citation_volume=30; citation_publication_date=1994; citation_pages=33-49; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=J Pharm Pharmacol; citation_title=
                              Echinacea species (Echinacea angustifolia (DC.) Hell., Echinacea pallida (Nutt.) Nutt., Echinacea purpurea (L.) Moench): a review of their chemistry, pharmacology and clinical properties; citation_author=J Barnes, LA Anderson, S Gibbons, JD Phillipson; citation_volume=57; citation_publication_date=2005; citation_pages=929-54; citation_doi=10.1211/0022357056127; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=Inter Immunoph; citation_title=
                              Echinacea purpurea (L.) Moench modulates human T-cell cytokine response; citation_author=FN Fonseca, G Papanicolaou, H Lin, E Kennelly, BR Cassileth, S Cunningham-Rundles; citation_volume=19; citation_publication_date=2014; citation_pages=94-102; citation_doi=10.1016/j.intimp.2013.12.019; citation_id=CR4">
<meta name="citation_reference" content="citation_title=Coneflower, Echinacea species; citation_inbook_title=Flower breeding and genetics: issues, challenges and opportunities for the 21 century; citation_publication_date=2007; citation_pages=801-24; citation_id=CR5; citation_author=JR Ault; citation_publisher=Springer">
<meta name="citation_reference" content="citation_journal_title=Univ Kansas Sci Bul; citation_title=The taxonomy of the genus Echinacea (Compositae); citation_author=RL McGregor; citation_volume=48; citation_publication_date=1968; citation_pages=113-42; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=Organ Cult; citation_title=Anther culture and haploid plant regeneration in purple coneflower (Echinacea purpurea L.). Plant Cell Tis; citation_author=FC Zhao, D Nilanthi, YS Yang, H Wu; citation_volume=86; citation_publication_date=2006; citation_pages=55-62; citation_doi=10.1007/s11240-006-9096-0; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=HortSci; citation_title=Chromosome Karyotypes of Echinacea angustifolia var. Angustifolia and E. Purpurea; citation_author=LP Qu, XP Wang, E Hood, M Wang, R Scalzo; citation_volume=39; citation_issue=2; citation_publication_date=2004; citation_pages=368-70; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Enhancing the productivity of secondary metabolites via induced polyploidy: a review; citation_author=OP Dhawan, UC Lavania; citation_volume=87; citation_publication_date=1996; citation_pages=81-9; citation_doi=10.1007/BF00021879; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Colchicine mutagenesis is associated with substantial heritable variation in cotton; citation_author=DJ Luckett; citation_volume=42; citation_publication_date=1989; citation_pages=177-82; citation_doi=10.1007/BF00042630; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Sex Plant Reprod; citation_title=Primary trisomics obtained from autotriploid by diploid reciprocal crosses in cucumber; citation_author=WP Diao, SY Bao, B Jiang, L Cui, JF Chen; citation_volume=22; citation_publication_date=2009; citation_pages=45-51; citation_doi=10.1007/s00497-008-0090-z; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Plant Breed; citation_title=Identification of primary trisomics and other aneuploids in foxtail millet; citation_author=R Wang, J Gao, GH Liang; citation_volume=118; citation_publication_date=1999; citation_pages=59-62; citation_doi=10.1046/j.1439-0523.1999.118001059.x; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=J Biomed Biotech; citation_title=Induction of tetraploids from petiole explants through colchicine treatments in Echinacea purpurea L; citation_author=D Nilanthi, XL Chen, FC Zhao, YS Yang, H Wu; citation_publication_date=2009; citation_id=CR13">
<meta name="citation_reference" content="citation_title=Comparison among diploid, its colchicine-induced tetraploid and their crossed descendent triploid in purple coneflower (Echinacea purpurea L.); citation_publication_date=2014; citation_id=CR14; citation_author=XL Chen; citation_author=JJ Zhang; citation_author=R Chen; citation_author=QL Li; citation_author=YS Yang; citation_author=H Wu; citation_publisher=International Conference on Biological Engineering and Biomedicine">
<meta name="citation_reference" content="citation_journal_title=Curr Sci; citation_title=Accessory chromosomes in garden chrysanthemum; citation_author=MA Nazeer; citation_volume=50; citation_publication_date=1981; citation_pages=461-2; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=J Syst Evol; citation_title=Analysis of karyotype diversity of 40 Chinese chrysanthemum cultivars; citation_author=Y Zhang, ML Zhu, SL Dai; citation_volume=51; citation_publication_date=2013; citation_pages=335-52; citation_doi=10.1111/j.1759-6831.2012.00235.x; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Phenotypic consequences of aneuploidy in Arabidopsis thaliana
                           ; citation_author=IM Henry, BP Dilkes, ES Miller, D Burkart-Waco, L Comai; citation_volume=186; citation_issue=4; citation_publication_date=2010; citation_pages=1231-45; citation_doi=10.1534/genetics.110.121079; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=Hered; citation_title=Tests of six cotton (Gossypium hirsutum L.) mutants for association with aneuploids. J; citation_author=RJ Kohel, DM Stelly, J Yu; citation_volume=93; citation_issue=2; citation_publication_date=2002; citation_pages=130-2; citation_doi=10.1093/jhered/93.2.130; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Genome-wide gene expression perturbation induced by loss of C2 chromosome in allotetraploid Brassica rapus L; citation_author=B Zhu, YJ Shao, Q Pan, XH Ge, ZY Li; citation_volume=6; citation_publication_date=2015; citation_pages=1-16; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Nat New Biol; citation_title=Quinacrine fluorescence and Giemsa staining in plants; citation_author=CG Vosa, P Marchi; citation_volume=237; citation_issue=75; citation_publication_date=1972; citation_pages=191-2; citation_doi=10.1038/newbio237191a0; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=G-banded chromosomes in Pinus-resinosa
                           ; citation_author=A Drewry; citation_volume=73; citation_issue=4; citation_publication_date=1982; citation_pages=305-6; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Acta Genet Sin; citation_title=Chromosome Giemsa N-banding in rice; citation_author=Z Yao; citation_volume=11; citation_publication_date=1984; citation_pages=207-10; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Molecular hybridization of radioactive DNA to the DNA of cytological preparations; citation_author=ML Pardue, JG Gall; citation_volume=64; citation_publication_date=1969; citation_pages=600-4; citation_doi=10.1073/pnas.64.2.600; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=Chromosoma; citation_title=R-banding produced by DNase I digestion of chromomycin-stained chromosomes; citation_author=D Schweizer; citation_volume=64; citation_issue=2; citation_publication_date=1977; citation_pages=117-24; citation_doi=10.1007/BF00327051; citation_id=CR24">
<meta name="citation_reference" content="citation_journal_title=Plant Breed; citation_title=Karyotype and C-banding pattern of mitotic chromosomes in alfalfa, Medicago sativa L; citation_author=E Falistocco, M Falcinelli, F Veronesi; citation_volume=114; citation_publication_date=1995; citation_pages=451-3; citation_doi=10.1111/j.1439-0523.1995.tb00831.x; citation_id=CR25">
<meta name="citation_reference" content="Gill BS, Sears RG. The current status of chromosome analysis in wheat. Chr. Str. Fun. Part of the series Stadler Genetics Symposia Series. 1988. p. 299–321.">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Karyotype and C-banding patterns of mitotic chromosomes in diploid bromegrass (Bromus riparius Rehm); citation_author=M Tuna, KS Gill, KP Vogel; citation_volume=41; citation_publication_date=2001; citation_pages=831-4; citation_doi=10.2135/cropsci2001.413831x; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Curr Adv Plant Sci; citation_title=The use of Giemsa and other staining techniques in karyotype analysis; citation_author=CG Vosa; citation_volume=14; citation_publication_date=1975; citation_pages=495-510; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=J Wuhan Bot Res; citation_title=A suggestion on the standardization of karyotype analysis in plants; citation_author=MX Li, RY Chen; citation_volume=3; citation_publication_date=1985; citation_pages=297-302; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=J Plant Res; citation_title=High-resolution HKG-banding in maize mitotic chromosomes; citation_author=CR Carlos, LS Saraiva; citation_volume=110; citation_issue=4; citation_publication_date=1997; citation_pages=417-20; citation_doi=10.1007/BF02506801; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Caryologia; citation_title=Karyomorphological analyses and chromatin characterization by banding techniques in Euphorbia characias L. and E. Wulfenii Hoppe (=E.Veneta Willd) (Euphorbiaceae); citation_author=S D’Emerico, D Pignone, F Vita, A Scrugli; citation_volume=56; citation_issue=4; citation_publication_date=2003; citation_pages=501-8; citation_doi=10.1080/00087114.2003.10589363; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Chinensis Acta Horti Sinica; citation_title=A study on the karyotype and Giemsa C-banding patterns in Narcissus tazetta var; citation_author=M Li, D Chen, L Wang; citation_volume=7; citation_issue=2; citation_publication_date=1980; citation_pages=29-39; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=A Giemsa technique for rapid identification of chromosomes in Triticale
                           ; citation_author=A Merker; citation_volume=75; citation_issue=2; citation_publication_date=1973; citation_pages=280-2; citation_doi=10.1111/j.1601-5223.1973.tb01171.x; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=Heterochromatin polymorphism in the rye karyotype as detected by the Giemsa C-banding technique; citation_author=A Weimarck; citation_volume=79; citation_publication_date=1975; citation_pages=293-300; citation_doi=10.1111/j.1601-5223.1975.tb01486.x; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=Herba Polonica; citation_title=Studies on the biology of flowering and fruiting of purple coneflower (Echinacea purpurea Moench). Part 1. Biology of flowering and fruiting; citation_author=LK Sejdler, J Dabrowska; citation_volume=42; citation_publication_date=1998; citation_pages=83-7; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=HortSci; citation_title=Self-incompatibility in Echinacea purpurea
                           ; citation_author=LC Stephens; citation_volume=43; citation_publication_date=2008; citation_pages=1350-4; citation_id=CR36">
<meta name="citation_author" content="Weizhen Jiang">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author" content="Qingling Li">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author" content="Xiaolu Chen">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author" content="Yi Ren">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author" content="Rong Chen">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author" content="Hong Wu">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author_institution" content="Research Center of South China Medicinal Plants, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author_institution" content="Guangdong Technology Research Center for Traditional Chinese Veterinary Medicine and Natural Medicine, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author" content="Yuesheng Yang">
<meta name="citation_author_institution" content="Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author_institution" content="Research Center of South China Medicinal Plants, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta name="citation_author_institution" content="Guangdong Technology Research Center for Traditional Chinese Veterinary Medicine and Natural Medicine, South China Agricultural University, Guangzhou, People’s Republic of China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0016-y',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0016-y","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Karyotype;C-banding technique;Aneuploid;E. Purpurea;Chromosomes"},"contentInfo":{"imprint":"BioMed Central","title":"A C-banded karyotype of mitotic chromosomes in diploid purple coneflower ( Echinacea purpurea L.)","publishedAt":1479772800000,"publishedAtDate":"2016-11-22","author":["Weizhen Jiang","Qingling Li","Xiaolu Chen","Yi Ren","Rong Chen","Hong Wu","Yuesheng Yang"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0016-y">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0016-y">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="A C-banded karyotype of mitotic chromosomes in diploid purple coneflower ( Echinacea purpurea L.)">
<meta property="og:description" content="Aneuploid ermpglasm is an important resource for genetic studies and identification of individual chromosomes in the cells of the aneuploid is an important step. The karyotype has already been established for purple coneflower (Echinacea purpurea L.), but due to the high similarity in the morphology of several pairs of chromosomes in this species, it cannot be used to identify individual chromosomes in its own complement. The objectives of this study are to develop and evaluate the Giemsa C-banding technique for the purpose of identifying the individual chromosomes in Echinacea purpurea. The established karyotype with C-bands showed that all the 11 pairs of chromosomes possessed centromeric bands. Telomeric bands appeared most frequently in almost all the chromosomes with only two exceptions, the short arm of the chromosome 9 and the long arm of the chromosome 10. Intercalary bands were found mainly in the long arm of some chromosomes with only two exceptions, the chromosomes 1 and 2 that had intercalary bands on both arms. The chromosome 4 was the only chromosome where intercalary bands were absent. Chromosomes in E. purpurea could be stained with Giemsa to bear C-bands. By classifying the chromosomes into groups and judging the C-bands, each chromosome could be identified. The methods established in this study might be used for the identification of chromosome constitution in aneuploid E. purpurea created in a breeding program.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0016-y;kwrd=Karyotype,C-banding technique,Aneuploid,E. Purpurea,Chromosomes;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0016-y&amp;kwrd=Karyotype,C-banding%20technique,Aneuploid,E.%20Purpurea,Chromosomes&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0016-y&amp;kwrd=Karyotype,C-banding%20technique,Aneuploid,E.%20Purpurea,Chromosomes&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Method">Method</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec3" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and discussion">Results and discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec4" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">A C-banded karyotype of mitotic chromosomes in diploid purple coneflower (<em xmlns="" class="EmphasisTypeItalic">Echinacea purpurea</em> L.)</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Weizhen Jiang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qingling Li</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaolu Chen</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yi Ren</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Rong Chen</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Hong Wu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yuesheng Yang</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:ysyang@scau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>
</li>
</ul></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:14</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0016-y">https://doi.org/10.1186/s41065-016-0016-y</a></p>
<p class="Copyright">©  The Author(s) 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>3 March 2016</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>20 October 2016</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">22 November 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Aneuploid ermpglasm is an important resource for genetic studies and identification of individual chromosomes in the cells of the aneuploid is an important step. The karyotype has already been established for purple coneflower (<em xmlns="" class="EmphasisTypeItalic">Echinacea purpurea</em> L.), but due to the high similarity in the morphology of several pairs of chromosomes in this species, it cannot be used to identify individual chromosomes in its own complement. The objectives of this study are to develop and evaluate the Giemsa C-banding technique for the purpose of identifying the individual chromosomes in <em xmlns="" class="EmphasisTypeItalic">Echinacea purpurea</em>.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">The established karyotype with C-bands showed that all the 11 pairs of chromosomes possessed centromeric bands. Telomeric bands appeared most frequently in almost all the chromosomes with only two exceptions, the short arm of the chromosome 9 and the long arm of the chromosome 10. Intercalary bands were found mainly in the long arm of some chromosomes with only two exceptions, the chromosomes 1 and 2 that had intercalary bands on both arms. The chromosome 4 was the only chromosome where intercalary bands were absent.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Chromosomes in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> could be stained with Giemsa to bear C-bands. By classifying the chromosomes into groups and judging the C-bands, each chromosome could be identified. The methods established in this study might be used for the identification of chromosome constitution in aneuploid <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> created in a breeding program.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Karyotype</li>
<li class="c-keywords__item" itemprop="about">C-banding technique</li>
<li class="c-keywords__item" itemprop="about">Aneuploid</li>
<li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">E. Purpurea</em>
              </li>
<li class="c-keywords__item" itemprop="about">Chromosomes</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Purple coneflower (<em xmlns="" class="EmphasisTypeItalic">Echinacea purpurea</em> L.) is native to North America [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>] and has important pharmaceutical [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>] and ornamental [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] values. Wild <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> plants are diploids with 22 chromosomes in somatic cells [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. The karyotypes of <em xmlns="" class="EmphasisTypeItalic">Echinacea</em> species have been reported to be quite similar [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. In 2004, the karyotype of <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> was first established by Qu et al.; it has been used successfully to differentiate <em xmlns="" class="EmphasisTypeItalic">E. Purpurea</em> from another <em xmlns="" class="EmphasisTypeItalic">Echinacea</em> species, <em xmlns="" class="EmphasisTypeItalic">E. angustifoli</em>, based on the difference in the centromere’s position of chromosome pair 10 between the two species. However, due to the high similarity in the morphology of some pairs of the chromosomes in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em>, the karyotype established by Qu et al. [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] cannot be used to identify every chromosome in the species.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Ploidy breeding which includes polyploids [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] and aneuploids [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] has been proved to be a feasible method for many plant species. Following the success in obtaining tetraploid <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> by in vitro colchicine treatment of diploid explants (Nilanthi et al. [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], triploid plants were obtained by conventional crossing method between diploid and tetraploid plants [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>] while aneuploids were obtained by crossing between diploid and triploid plants (unpublished results). To obtain aneuploid in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> seems easy because aneuploid individuals are known to exist naturally in Asteraceae [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], the family to which <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> belongs. Aneuploids are different from euploids based on the possession of more or less number of chromosomes and are a unique germplasm resource for genetic studies [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>–<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. Before an effective study can take place, clarification of the constitution of the chromosomes in the material aneuploid is a prerequisite.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Similarity in morphology among some pairs of chromosomes in cells seems a common feature in many plant species. In order to distinguish individual chromosomes from each other in the complement, many chromosome banding techniques such as C-banding [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>], G-banding [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], N-banding [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>], Q-banding [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] and R-banding [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] have been attempted. Among these banding techniques, C-banding staining of the constitutive heterochromatin of the chromosomes with Giemsa is the most frequently used one in a range of plant species [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>–<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. In this report, we describe an effective karyotype based on chromosome length, arm ratio, and the C-banding patterns for identifying individual chromosomes in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em>.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Method" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Method">Method</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Method" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Seeds of purple coneflower (<em xmlns="" class="EmphasisTypeItalic">Echinacea purpurea</em> L.) were originally purchased at a supermarket provided by the Company of Plantation Products (Norton, MA 02766, USA) and maintained by harvesting seeds from the plants cultivated in the campus of South China Agricultural University. The seeds were stored in a refrigerator at 6 °C after harvest. Before germination, the seeds were treated with 2 mg/l gibberellin acid (GA<sub>3</sub>) for 24 h for the purpose of breaking the dormancy.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Seedlings with 2 tiny real leaves which developed from the seeds sown in sandy soil were collected at noon. Root tips of about 10 mm were dissected after washing with pure water. The root tips were kept in a glass bottle filled with 0.05 % (w/v) colchicine water solution for 3–4 h at 4 °C to <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> plants accumulate metaphase chromosomes in the cells, washed with running tap water for about 15 min, dipped in pure water for 8 min, and then put in Carnoy’s solution containing acetic acid and 95 % ethanol in a ratio of 1:3 (v/v) for at least 24 h at 4 °C for fixation. The fixed root tips were then washed again with running tap water for about 15 min, dipped in pure water for 8 min, and hydrolyzed in 1 <em xmlns="" class="EmphasisTypeItalic">M</em> HCl for 5–6 min at 60 °C. After hydrolysis, the root tips were washed again with running tap water for 15 min and dipped in pure water for 8 min. Subsequently, these root tips were stained with 20 % (v/v) carbolfuchsin solution for 1–2 min, squashed on slides under a cover glass and observed under a microscope for the selection of images of well spread metaphase chromosomes.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">The selected slides with well spread metaphase chromosomes were transferred to a freezer at 20 °C for 1 day and then the cover glass was removed. Immediately, the slides were dropped into 95 % (v/v) ethanol for 30 min to discolor the stain made by the carbolfuchsin solution. After discoloration, the slides were transferred into absolute ethanol twice, first for 30 min and then overnight for dehydration. Subsequently, the slides were put in an oven at 37 °C for 1 h, after which they were kept in a room and were ready for C-banding with Giemsa.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">The C-banding method adopted was that provided by Tuna et al. [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] with minor modifications. Dehydrated slides were placed in 0.2 <em xmlns="" class="EmphasisTypeItalic">M</em> HCl water solution and incubated in water bath at 60 °C for 2, 4, 6, 8 and 10 min, after which they were immediately washed with pure water. The slides were put into freshly prepared saturated Ba(OH)<sub>2</sub> solution at room temperature for 4, 8, 12, 16, and 20 min and washed carefully in distilled water kept at 45 °C. The water was changed three times at 2 min intervals. Then, the slides were washed three times with pure water of normal room temperature to ensure that all the barium crystals were removed. After the above treatments, the slides were dried in an oven at 37 °C for about 30–45 min and incubated in freshly prepared 2 × SSC solution (1× is 0.15 <em xmlns="" class="EmphasisTypeItalic">M</em> NaCl plus 0.015 <em xmlns="" class="EmphasisTypeItalic">M</em> citric acid) (pH 7.0) kept in water bath at 60 °C for 0.5, 1.0, 1.5 and 2.0 h. After the incubation, the slides were washed with pure water very carefully this time because the chromosomes adhering to the slide glass might drop off. The slides were dried again in an oven for 30–45 min at 37 °C and stained with 10 % (v/v) Giemsa in phosphate buffer (consisting of 62 % 0.07 <em xmlns="" class="EmphasisTypeItalic">M</em> Na<sub>2</sub>HPO<sub>4</sub> and 38 % 0.07 <em xmlns="" class="EmphasisTypeItalic">M</em> KH<sub>2</sub>PO<sub>4</sub>) for 10, 20, 30, 40, 50 and 60 min. After the staining, the slides were quickly rinsed with distilled water and dried for several hours at room temperature. For observations, slides were mounted in Permount.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Images of well-spread chromosomes were screened under a microscope (LeicaDLMB2) with 1000 times magnification and the photographs were taken. Eight best images with all the 11 pairs of chromosomes in one cell were selected, and the karyotype was analyzed with the software Adobe Photoshop CS5. Data for the long arm and short arm of each chromosome were statistically analyzed with the software SPSS19.0 (IBM), and significant differences were determined by Duncan’s test at 95 % level. The chromosomes were identified on the basis of their total length, arm length ratio (long/short arm), and C-banding patterns [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. Chromosomes in the karyotype were arranged in the order of decreasing mean chromosome length [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec3" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsanddiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsanddiscussion">Results and discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Resultsanddiscussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">In the experiments of C-banding with Giemsa for comparing various treating parameters, the best result was obtained by directly incubating the dried slides in a saturated Ba(OH)<sub>2</sub> solution for 8–12 min, followed by incubation in 2 × SSC solution at 60 °C for 1.5 h and staining with 10 % Giemsa solution for 20–30 min. The step before Ba(OH)<sub>2</sub> incubation treatment of the dried slides with 0.2 <em xmlns="" class="EmphasisTypeItalic">M</em> HCl at 60 °C for 3 min, as applied in other plant species [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>, <span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>], yielded negative results and was omitted.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The swelling of the chromosomes on the slides is most likely to occur during the Giemsa staining course. On the swollen chromosomes, only the centromeric region could be stained; the telomeric region could seldom be stained while the intercalary region could never be stained. Treatment of Ba(OH)<sub>2</sub> solution for longer than 15 min or at a pH value of 2 × SSC higher than 8.0, was the main cause of the swelling of the chromosomes, leading to very poor staining results. The pH value of the phosphate buffer for preparing Giemsa solution is another important factor that needs to be mentioned. A pH value in the range of 6.8–7.0 was desirable. Higher or lower values could result in complete failure.</p>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">C-banded mitotic metaphase chromosomes of diploid <em xmlns="" class="EmphasisTypeItalic">E. Purpurea</em> are shown in Fig. <span class="InternalRef"><a href="#Fig1">1</a></span> and a detailed C-banded karyotype is presented in Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>, with the accompanying total lengths and arm length ratios in Table <span class="InternalRef"><a href="#Tab1">1</a></span>. Standard deviations were also calculated for each chromosome (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). The relatively small deviation values in arm lengths and arm ratios show the high quality of preparation of the chromosomes in this study.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                  </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">C-banded mitotic metaphase chromosomes of <em class="EmphasisTypeItalic">E. Purpurea</em> (2n = 22). Bar = 10 μm</p>
                    </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0016-y/MediaObjects/41065_2016_16_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                  </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                      <p xmlns="" class="SimplePara">C-banded karyotype of diploid <em class="EmphasisTypeItalic">E. purpurea</em> based on the cell presented in Fig. <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Fig1">1</a></span>. Bar =10 μm</p>
                    </div></figcaption></figure><figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">The chromosomes of the diploid <em class="EmphasisTypeItalic">E. Purpurea</em> based on eight cells<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char="±">
<col char="±">
<col char="±">
<col char="±">
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Code of chromosome</p></th>
<th char="±"><p xmlns="" class="SimplePara">Length (μm) of long arm</p></th>
<th char="±"><p xmlns="" class="SimplePara">Length (μm) of short arm</p></th>
<th char="±"><p xmlns="" class="SimplePara">Total length (μm)</p></th>
<th char="±"><p xmlns="" class="SimplePara">Arm ratio<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></th>
<th><p xmlns="" class="SimplePara">Type of chromosome<sup xmlns="http://www.w3.org/1999/xhtml">c</sup></p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">I</p></td>
<td char="±"><p xmlns="" class="SimplePara">6.54 ± 0.14</p></td>
<td char="±"><p xmlns="" class="SimplePara">4.71 ± 0.18</p></td>
<td char="±"><p xmlns="" class="SimplePara">11.25 ± 0.32</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.40 ± 0.02</p></td>
<td><p xmlns="" class="SimplePara">m</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">II</p></td>
<td char="±"><p xmlns="" class="SimplePara">5.67 ± 0.53</p></td>
<td char="±"><p xmlns="" class="SimplePara">3.80 ± 0.21</p></td>
<td char="±"><p xmlns="" class="SimplePara">9.46 ± 0.73</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.50 ± 0.05</p></td>
<td><p xmlns="" class="SimplePara">m</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">III</p></td>
<td char="±"><p xmlns="" class="SimplePara">5.25 ± 0.22</p></td>
<td char="±"><p xmlns="" class="SimplePara">4.22 ± 0.13</p></td>
<td char="±"><p xmlns="" class="SimplePara">9.46 ± 0.35</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.25 ± 0.02</p></td>
<td><p xmlns="" class="SimplePara">m</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">IV</p></td>
<td char="±"><p xmlns="" class="SimplePara">4.98 ± 0.47</p></td>
<td char="±"><p xmlns="" class="SimplePara">3.76 ± 0.35</p></td>
<td char="±"><p xmlns="" class="SimplePara">8.74 ± 0.81</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.34 ± 0.01</p></td>
<td><p xmlns="" class="SimplePara">m</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">V</p></td>
<td char="±"><p xmlns="" class="SimplePara">6.01 ± 0.13</p></td>
<td char="±"><p xmlns="" class="SimplePara">2.56 ± 0.05</p></td>
<td char="±"><p xmlns="" class="SimplePara">8.59 ± 0.15</p></td>
<td char="±"><p xmlns="" class="SimplePara">2.40 ± 0.02</p></td>
<td><p xmlns="" class="SimplePara">sm</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">VI</p></td>
<td char="±"><p xmlns="" class="SimplePara">6.52 ± 0.91</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.73 ± 0.18</p></td>
<td char="±"><p xmlns="" class="SimplePara">8.25 ± 1.09</p></td>
<td char="±"><p xmlns="" class="SimplePara">3.78 ± 0.11</p></td>
<td><p xmlns="" class="SimplePara">st</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">VII</p></td>
<td char="±"><p xmlns="" class="SimplePara">5.61 ± 0.18</p></td>
<td char="±"><p xmlns="" class="SimplePara">2.16 ± 0.15</p></td>
<td char="±"><p xmlns="" class="SimplePara">7.76 ± 0.32</p></td>
<td char="±"><p xmlns="" class="SimplePara">2.62 ± 0.08</p></td>
<td><p xmlns="" class="SimplePara">sm</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">VIII</p></td>
<td char="±"><p xmlns="" class="SimplePara">5.99 ± 0.14</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.75 ± 0.03</p></td>
<td char="±"><p xmlns="" class="SimplePara">7.74 ± 0.11</p></td>
<td char="±"><p xmlns="" class="SimplePara">3.47 ± 0.15</p></td>
<td><p xmlns="" class="SimplePara">st</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">IX</p></td>
<td char="±"><p xmlns="" class="SimplePara">6.03 ± 0.70</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.49 ± 0.16</p></td>
<td char="±"><p xmlns="" class="SimplePara">7.55 ± 0.89</p></td>
<td char="±"><p xmlns="" class="SimplePara">4.17 ± 0.08</p></td>
<td><p xmlns="" class="SimplePara">st</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">X</p></td>
<td char="±"><p xmlns="" class="SimplePara">5.79 ± 0.70</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.52 ± 0.21</p></td>
<td char="±"><p xmlns="" class="SimplePara">7.30 ± 0.90</p></td>
<td char="±"><p xmlns="" class="SimplePara">3.93 ± 0.01</p></td>
<td><p xmlns="" class="SimplePara">st</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">XI</p></td>
<td char="±"><p xmlns="" class="SimplePara">5.73 ± 0.20</p></td>
<td char="±"><p xmlns="" class="SimplePara">1.48 ± 0.04</p></td>
<td char="±"><p xmlns="" class="SimplePara">7.20 ± 0.23</p></td>
<td char="±"><p xmlns="" class="SimplePara">3.99 ± 0.03</p></td>
<td><p xmlns="" class="SimplePara">st</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">
                                 <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Data were mean ± standard deviation</p>
                    <p xmlns="" class="SimplePara">
                                 <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Armratio: Length of the long arm/length of the short arm</p>
                    <p xmlns="" class="SimplePara">
                                 <sup xmlns="http://www.w3.org/1999/xhtml">c</sup>m: Arm ratio lower than 1.70; sm: arm ratio higher than 1.70 but lower than 3.0; st: arm ratio higher than 3.00 [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]</p>
                  </div></figure>
</div>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The mean lengths of the chromosomes ranged from 11.25 to 7.20 μm while the mean arm ratios ranged from 1.25 to 4.17. Arm ratios of chromosomes have been suggested by Li and Chen [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>] as the only factor for classifying chromosome types in standardized karyotype analysis. In accordance with their method, within the chromosome complement in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em>, four pairs of chromosomes were classified as metacentric (m) (chromosomes 1, 2, 3, and 4, coincides with the longest four chromosomes), two pairs were classified as submetacentric (sm) (chromosomes 5 and 7) while the remaining 5 pairs were classified as subterminal (st) (Table <span class="InternalRef"><a href="#Tab1">1</a></span>).</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">With proper Giemsa staining method, all the chromosomes showed terminal bands with only two exceptions, the short arm of the chromosome 9 and the long arm of the chromosome 10. Intercalary bands were observed mainly on the long arms with only two exceptions, the chromosome 1 and 2 that had intercalary bands on both arms. The chromosome 4 was the only chromosome where intercalary bands were absent.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Differentiating each chromosome among the complement in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> was attempted within each group of m, sm and st because the minimal differences in arm ratio between m and sm types of chromosomes and between sm and st types of chromosomes were large and could be easily observed in the present case (the maximal arm ratio for m is 1.50 while the minimal arm ratio for sm is 2.40, equal to a difference of 37.5 %; the maximal arm ratio for sm is 2.62 while the minimal arm ratio for st is 3.47, equal to a difference of 24.5 %). The first group is type m chromosomes consisting of 1, 2, 3 and 4 chromosomes. Among the four chromosomes, 1 is first identified from the others because 1 is the only one having two intercalary bands on the long arm; 2 can be identified from the others because it has one intercalary on both short arm and long arm; 3 can be identified from 4 because it has an intercalary band on the long arm.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The second group is type sm chromosomes consisting of only two chromosomes 5 and 7. They are very similar in total length and arm ratio (similarity 90.3 and 91.6 %, respectively), and both possess two telomeric bands, one centromeric band and several intercalary bands. The only way to differentiate them from each other is by comparing the number of the intercalary band: clearly, chromosome 5 has two intercalary bands on the long arm, but chromosome 7 has only one.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">The third group is type st chromosomes consisting of five chromosomes 6, 8, 9, 10 and 11. They are quite similar in total length and arm ratio (maximal similarity 98.6 and 98.5 % respectively), but each of them has different patterns of C-bands and can be easily differentiated from each other. Chromosome 6 has telomeric bands and centromeric bands on both arms, different from the other four chromosomes that have two intercalary bands on the long arm. Chromosome 8 is extremely similar to chromosome 11 in length and arm ratio, and in addition both of them possess two bands on the long arm. However, the two bands on chromosome 11 are more separated. Chromosome 9 is the only chromosome without C-bands on the short arm, while chromosome 10 is the only chromosome without telomeric bands on the long arm.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">The C-banded karyotype shown in Fig. <span class="InternalRef"><a href="#Fig2">2</a></span> does not only tell the differences among the 11 pairs of chromosomes, but it also reveals certain heterogeneous characteristics between most of the homologous chromosomes. For example, in the pair of chromosomes 1, the centromeric band and an intercalary band on the long arm near the centromere were clearly different in the intensity of staining. Similar phenomena were also observed on chromosomes 2, 3, 4 and 10. The heterogeneous characteristics in C-bands regarding differences in depth of staining and banding positions between homologous chromosomes have been reported in many cases with explanations such as translocation between homologous chromosomes and differences in the content of DNA repeats in constitutive heterochromatin [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>–<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Regarding the studied plant species, <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> is a highly self-incompatible plant [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>, <span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] pollinated with pollen grains mainly from other genotype plant individuals. For this, the homologous chromosomes are almost of different plant origins and might differ to a certain degree in such as DNA sequence and even chromosome configuration, causing the differences in the banding patterns in some pairs of chromosomes.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec4" class="Section1 RenderAsSection1 SectionTypeConclusion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">In conclusion, chromosomes in <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> could be stained to show bands. By classifying the chromosomes into groups and judging the C-bands, each chromosome could be identified. The methods established in this study might be used for the identification of chromosome constitution in aneuploid <em xmlns="" class="EmphasisTypeItalic">E. purpurea</em> created by crosses between diploid and triploid plants.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Funding</h3>
<p xmlns="" class="SimplePara">This research was funded by two grants from Science and Technology Planning Project of Guangdong Province, China (2011B031700026, 2014B090904074).</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and material</h3>
                <p id="Par23" class="Para">All the data and material are of full availability.</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par24" class="Para">WJ conducted most of the experiments especially the C-bands staining experiments and drawing the C-banded karyotype figure. QL helped in preparing slides for chromosome observation. XC was the previous conductor of the concerned work. YR helped in screening slides with high quality chromosome images. RC helped in preparing slides for chromosome observation. HW gave financial support. YY designed the experiments and prepared the manuscript. All authors read and approved the final manuscript.</p>
              </div>
<div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par25" class="Para">The authors declare that they have no competing interests.</p>
              </div>
<div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par26" class="Para">Not applicable (The study doesn’t involve human participants, human data, or human tissue).</p>
              </div>
<div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par27" class="Para">Not applicable (The study doesn’t involve human participants, human data, or human tissue).</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Genetic Engineering Laboratory, College of Life Sciences, South China Agricultural University, Room 614, Guangzhou, 510642, People’s Republic of China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Research Center of South China Medicinal Plants, South China Agricultural University, Guangzhou, 510642, People’s Republic of China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Guangdong Technology Research Center for Traditional Chinese Veterinary Medicine and Natural Medicine, South China Agricultural University, Guangzhou, 510642, People’s Republic of China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Foster S. <em class="EmphasisTypeItalic">Echinacea</em> exalted! The botany, culture, history and medicinal uses of the purple conflowers. Drury: New Life Farm. Inc.; 1984.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Echinacea%20exalted%21%20The%20botany%2C%20culture%2C%20history%20and%20medicinal%20uses%20of%20the%20purple%20conflowers&amp;author=S.%20Foster&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hobbs CR. <em class="EmphasisTypeItalic">Echinacea</em>-a literature review. Herbalgram. 1994;30:33–49.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Echinacea-a%20literature%20review&amp;author=CR.%20Hobbs&amp;journal=Herbalgram&amp;volume=30&amp;pages=33-49&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Barnes J, Anderson LA, Gibbons S, Phillipson JD. <em class="EmphasisTypeItalic">Echinacea</em> species (<em class="EmphasisTypeItalic">Echinacea angustifolia</em> (DC.) Hell., <em class="EmphasisTypeItalic">Echinacea pallida</em> (Nutt.) Nutt., <em class="EmphasisTypeItalic">Echinacea purpurea</em> (L.) Moench): a review of their chemistry, pharmacology and clinical properties. J Pharm Pharmacol. 2005;57:929–54.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1211/0022357056127"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16102249"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Echinacea%20species%20%28Echinacea%20angustifolia%20%28DC.%29%20Hell.%2C%20Echinacea%20pallida%20%28Nutt.%29%20Nutt.%2C%20Echinacea%20purpurea%20%28L.%29%20Moench%29%3A%20a%20review%20of%20their%20chemistry%2C%20pharmacology%20and%20clinical%20properties&amp;author=J.%20Barnes&amp;author=LA.%20Anderson&amp;author=S.%20Gibbons&amp;author=JD.%20Phillipson&amp;journal=J%20Pharm%20Pharmacol&amp;volume=57&amp;pages=929-54&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fonseca FN, Papanicolaou G, Lin H, Kennelly E, Cassileth BR, Cunningham-Rundles S. <em class="EmphasisTypeItalic">Echinacea purpurea</em> (L.) Moench modulates human T-cell cytokine response. Inter Immunoph. 2014;19:94–102.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.intimp.2013.12.019"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Echinacea%20purpurea%20%28L.%29%20Moench%20modulates%20human%20T-cell%20cytokine%20response&amp;author=FN.%20Fonseca&amp;author=G.%20Papanicolaou&amp;author=H.%20Lin&amp;author=E.%20Kennelly&amp;author=BR.%20Cassileth&amp;author=S.%20Cunningham-Rundles&amp;journal=Inter%20Immunoph&amp;volume=19&amp;pages=94-102&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ault JR. Coneflower, <em class="EmphasisTypeItalic">Echinacea</em> species. In: Anderson NO, editor. Flower breeding and genetics: issues, challenges and opportunities for the 21 century. Dordrecht.: Springer; 2007. p. 801–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Coneflower%2C%20Echinacea%20species&amp;author=JR.%20Ault&amp;pages=801-24&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McGregor RL. The taxonomy of the genus <em class="EmphasisTypeItalic">Echinacea</em> (Compositae). Univ Kansas Sci Bul. 1968;48:113–42.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20taxonomy%20of%20the%20genus%20Echinacea%20%28Compositae%29&amp;author=RL.%20McGregor&amp;journal=Univ%20Kansas%20Sci%20Bul&amp;volume=48&amp;pages=113-42&amp;publication_year=1968"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao FC, Nilanthi D, Yang YS, Wu H. Anther culture and haploid plant regeneration in purple coneflower (<em class="EmphasisTypeItalic">Echinacea purpurea</em> L.). Plant Cell Tis. Organ Cult. 2006;86:55–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11240-006-9096-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Anther%20culture%20and%20haploid%20plant%20regeneration%20in%20purple%20coneflower%20%28Echinacea%20purpurea%20L.%29.%20Plant%20Cell%20Tis&amp;author=FC.%20Zhao&amp;author=D.%20Nilanthi&amp;author=YS.%20Yang&amp;author=H.%20Wu&amp;journal=Organ%20Cult&amp;volume=86&amp;pages=55-62&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qu LP, Wang XP, Hood E, Wang M, Scalzo R. Chromosome Karyotypes of <em class="EmphasisTypeItalic">Echinacea angustifolia</em> var. Angustifolia and E. Purpurea. HortSci. 2004;39(2):368–70.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Chromosome%20Karyotypes%20of%20Echinacea%20angustifolia%20var.%20Angustifolia%20and%20E.%20Purpurea&amp;author=LP.%20Qu&amp;author=XP.%20Wang&amp;author=E.%20Hood&amp;author=M.%20Wang&amp;author=R.%20Scalzo&amp;journal=HortSci&amp;volume=39&amp;issue=2&amp;pages=368-70&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dhawan OP, Lavania UC. Enhancing the productivity of secondary metabolites via induced polyploidy: a review. Euphytica. 1996;87:81–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00021879"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Enhancing%20the%20productivity%20of%20secondary%20metabolites%20via%20induced%20polyploidy%3A%20a%20review&amp;author=OP.%20Dhawan&amp;author=UC.%20Lavania&amp;journal=Euphytica&amp;volume=87&amp;pages=81-9&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luckett DJ. Colchicine mutagenesis is associated with substantial heritable variation in cotton. Euphytica. 1989;42:177–82.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00042630"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Colchicine%20mutagenesis%20is%20associated%20with%20substantial%20heritable%20variation%20in%20cotton&amp;author=DJ.%20Luckett&amp;journal=Euphytica&amp;volume=42&amp;pages=177-82&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Diao WP, Bao SY, Jiang B, Cui L, Chen JF. Primary trisomics obtained from autotriploid by diploid reciprocal crosses in cucumber. Sex Plant Reprod. 2009;22:45–51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00497-008-0090-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20033455"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Primary%20trisomics%20obtained%20from%20autotriploid%20by%20diploid%20reciprocal%20crosses%20in%20cucumber&amp;author=WP.%20Diao&amp;author=SY.%20Bao&amp;author=B.%20Jiang&amp;author=L.%20Cui&amp;author=JF.%20Chen&amp;journal=Sex%20Plant%20Reprod&amp;volume=22&amp;pages=45-51&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang R, Gao J, Liang GH. Identification of primary trisomics and other aneuploids in foxtail millet. Plant Breed. 1999;118:59–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1439-0523.1999.118001059.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20primary%20trisomics%20and%20other%20aneuploids%20in%20foxtail%20millet&amp;author=R.%20Wang&amp;author=J.%20Gao&amp;author=GH.%20Liang&amp;journal=Plant%20Breed&amp;volume=118&amp;pages=59-62&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nilanthi D, Chen XL, Zhao FC, Yang YS, Wu H. Induction of tetraploids from petiole explants through colchicine treatments in <em class="EmphasisTypeItalic">Echinacea purpurea</em> L. J Biomed Biotech. 2009. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1155/2009/343485" target="_blank" class="ext-link"><span class="RefSource">10.1155/2009/343485</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Induction%20of%20tetraploids%20from%20petiole%20explants%20through%20colchicine%20treatments%20in%20Echinacea%20purpurea%20L&amp;author=D.%20Nilanthi&amp;author=XL.%20Chen&amp;author=FC.%20Zhao&amp;author=YS.%20Yang&amp;author=H.%20Wu&amp;journal=J%20Biomed%20Biotech&amp;publication_year=2009&amp;doi=10.1155%2F2009%2F343485"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen XL, Zhang JJ, Chen R, Li QL, Yang YS, Wu H. Comparison among diploid, its colchicine-induced tetraploid and their crossed descendent triploid in purple coneflower (<em class="EmphasisTypeItalic">Echinacea purpurea</em> L.). Yichang: International Conference on Biological Engineering and Biomedicine; 2014. p. 159–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20among%20diploid%2C%20its%20colchicine-induced%20tetraploid%20and%20their%20crossed%20descendent%20triploid%20in%20purple%20coneflower%20%28Echinacea%20purpurea%20L.%29&amp;author=XL.%20Chen&amp;author=JJ.%20Zhang&amp;author=R.%20Chen&amp;author=QL.%20Li&amp;author=YS.%20Yang&amp;author=H.%20Wu&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nazeer MA. Accessory chromosomes in garden chrysanthemum. Curr Sci. 1981;50:461–2.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Accessory%20chromosomes%20in%20garden%20chrysanthemum&amp;author=MA.%20Nazeer&amp;journal=Curr%20Sci&amp;volume=50&amp;pages=461-2&amp;publication_year=1981"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang Y, Zhu ML, Dai SL. Analysis of karyotype diversity of 40 Chinese chrysanthemum cultivars. J Syst Evol. 2013;51:335–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1759-6831.2012.00235.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20karyotype%20diversity%20of%2040%20Chinese%20chrysanthemum%20cultivars&amp;author=Y.%20Zhang&amp;author=ML.%20Zhu&amp;author=SL.%20Dai&amp;journal=J%20Syst%20Evol&amp;volume=51&amp;pages=335-52&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Henry IM, Dilkes BP, Miller ES, Burkart-Waco D, Comai L. Phenotypic consequences of aneuploidy in <em class="EmphasisTypeItalic">Arabidopsis thaliana</em>. Genetics. 2010;186(4):1231–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.110.121079"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20876566"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2998307"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phenotypic%20consequences%20of%20aneuploidy%20in%20Arabidopsis%20thaliana%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=IM.%20Henry&amp;author=BP.%20Dilkes&amp;author=ES.%20Miller&amp;author=D.%20Burkart-Waco&amp;author=L.%20Comai&amp;journal=Genetics&amp;volume=186&amp;issue=4&amp;pages=1231-45&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kohel RJ, Stelly DM, Yu J. Tests of six cotton (<em class="EmphasisTypeItalic">Gossypium hirsutum</em> L.) mutants for association with aneuploids. J. Hered. 2002;93(2):130–2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/93.2.130"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tests%20of%20six%20cotton%20%28Gossypium%20hirsutum%20L.%29%20mutants%20for%20association%20with%20aneuploids.%20J&amp;author=RJ.%20Kohel&amp;author=DM.%20Stelly&amp;author=J.%20Yu&amp;journal=Hered&amp;volume=93&amp;issue=2&amp;pages=130-2&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhu B, Shao YJ, Pan Q, Ge XH, Li ZY. Genome-wide gene expression perturbation induced by loss of C2 chromosome in allotetraploid <em class="EmphasisTypeItalic">Brassica rapus</em> L. Front Plant Sci. 2015;6:1–16.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20gene%20expression%20perturbation%20induced%20by%20loss%20of%20C2%20chromosome%20in%20allotetraploid%20Brassica%20rapus%20L&amp;author=B.%20Zhu&amp;author=YJ.%20Shao&amp;author=Q.%20Pan&amp;author=XH.%20Ge&amp;author=ZY.%20Li&amp;journal=Front%20Plant%20Sci&amp;volume=6&amp;pages=1-16&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vosa CG, Marchi P. Quinacrine fluorescence and Giemsa staining in plants. Nat New Biol. 1972;237(75):191–2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/newbio237191a0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4113833"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Quinacrine%20fluorescence%20and%20Giemsa%20staining%20in%20plants&amp;author=CG.%20Vosa&amp;author=P.%20Marchi&amp;journal=Nat%20New%20Biol&amp;volume=237&amp;issue=75&amp;pages=191-2&amp;publication_year=1972"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Drewry A. G-banded chromosomes in <em class="EmphasisTypeItalic">Pinus-resinosa</em>. J Hered. 1982;73(4):305–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=G-banded%20chromosomes%20in%20Pinus-resinosa%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=A.%20Drewry&amp;journal=J%20Hered&amp;volume=73&amp;issue=4&amp;pages=305-6&amp;publication_year=1982"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yao Z. Chromosome Giemsa N-banding in rice. Acta Genet Sin. 1984;11:207–10.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Chromosome%20Giemsa%20N-banding%20in%20rice&amp;author=Z.%20Yao&amp;journal=Acta%20Genet%20Sin&amp;volume=11&amp;pages=207-10&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pardue ML, Gall JG. Molecular hybridization of radioactive DNA to the DNA of cytological preparations. Proc Natl Acad Sci U S A. 1969;64:600–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.64.2.600"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=5261036"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC223386"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20hybridization%20of%20radioactive%20DNA%20to%20the%20DNA%20of%20cytological%20preparations&amp;author=ML.%20Pardue&amp;author=JG.%20Gall&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=64&amp;pages=600-4&amp;publication_year=1969"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schweizer D. R-banding produced by DNase I digestion of chromomycin-stained chromosomes. Chromosoma. 1977;64(2):117–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00327051"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=72638"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=R-banding%20produced%20by%20DNase%20I%20digestion%20of%20chromomycin-stained%20chromosomes&amp;author=D.%20Schweizer&amp;journal=Chromosoma&amp;volume=64&amp;issue=2&amp;pages=117-24&amp;publication_year=1977"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Falistocco E, Falcinelli M, Veronesi F. Karyotype and C-banding pattern of mitotic chromosomes in alfalfa, <em class="EmphasisTypeItalic">Medicago sativa</em> L. Plant Breed. 1995;114:451–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1439-0523.1995.tb00831.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Karyotype%20and%20C-banding%20pattern%20of%20mitotic%20chromosomes%20in%20alfalfa%2C%20Medicago%20sativa%20L&amp;author=E.%20Falistocco&amp;author=M.%20Falcinelli&amp;author=F.%20Veronesi&amp;journal=Plant%20Breed&amp;volume=114&amp;pages=451-3&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gill BS, Sears RG. The current status of chromosome analysis in wheat. Chr. Str. Fun. Part of the series Stadler Genetics Symposia Series. 1988. p. 299–321.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Gill%20BS%2C%20Sears%20RG.%20The%20current%20status%20of%20chromosome%20analysis%20in%20wheat.%20Chr.%20Str.%20Fun.%20Part%20of%20the%20series%20Stadler%20Genetics%20Symposia%20Series.%201988.%20p.%20299%E2%80%93321."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tuna M, Gill KS, Vogel KP. Karyotype and C-banding patterns of mitotic chromosomes in diploid bromegrass (<em class="EmphasisTypeItalic">Bromus riparius</em> Rehm). Crop Sci. 2001;41:831–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2001.413831x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Karyotype%20and%20C-banding%20patterns%20of%20mitotic%20chromosomes%20in%20diploid%20bromegrass%20%28Bromus%20riparius%20Rehm%29&amp;author=M.%20Tuna&amp;author=KS.%20Gill&amp;author=KP.%20Vogel&amp;journal=Crop%20Sci&amp;volume=41&amp;pages=831-4&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vosa CG. The use of Giemsa and other staining techniques in karyotype analysis. Curr Adv Plant Sci. 1975;14:495–510.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20use%20of%20Giemsa%20and%20other%20staining%20techniques%20in%20karyotype%20analysis&amp;author=CG.%20Vosa&amp;journal=Curr%20Adv%20Plant%20Sci&amp;volume=14&amp;pages=495-510&amp;publication_year=1975"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li MX, Chen RY. A suggestion on the standardization of karyotype analysis in plants. J Wuhan Bot Res. 1985;3:297–302.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20suggestion%20on%20the%20standardization%20of%20karyotype%20analysis%20in%20plants&amp;author=MX.%20Li&amp;author=RY.%20Chen&amp;journal=J%20Wuhan%20Bot%20Res&amp;volume=3&amp;pages=297-302&amp;publication_year=1985"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Carlos CR, Saraiva LS. High-resolution HKG-banding in maize mitotic chromosomes. J Plant Res. 1997;110(4):417–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02506801"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-resolution%20HKG-banding%20in%20maize%20mitotic%20chromosomes&amp;author=CR.%20Carlos&amp;author=LS.%20Saraiva&amp;journal=J%20Plant%20Res&amp;volume=110&amp;issue=4&amp;pages=417-20&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">D’Emerico S, Pignone D, Vita F, Scrugli A. Karyomorphological analyses and chromatin characterization by banding techniques in <em class="EmphasisTypeItalic">Euphorbia characias</em> L. and <em class="EmphasisTypeItalic">E. Wulfenii</em> Hoppe (=<em class="EmphasisTypeItalic">E.Veneta</em> Willd) (Euphorbiaceae). Caryologia. 2003;56(4):501–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/00087114.2003.10589363"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Karyomorphological%20analyses%20and%20chromatin%20characterization%20by%20banding%20techniques%20in%20Euphorbia%20characias%20L.%20and%20E.%20Wulfenii%20Hoppe%20%28%3DE.Veneta%20Willd%29%20%28Euphorbiaceae%29&amp;author=S.%20D%E2%80%99Emerico&amp;author=D.%20Pignone&amp;author=F.%20Vita&amp;author=A.%20Scrugli&amp;journal=Caryologia&amp;volume=56&amp;issue=4&amp;pages=501-8&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li M, Chen D, Wang L. A study on the karyotype and Giemsa C-banding patterns in <em class="EmphasisTypeItalic">Narcissus tazetta</em> var. Chinensis Acta Horti Sinica. 1980;7(2):29–39.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20study%20on%20the%20karyotype%20and%20Giemsa%20C-banding%20patterns%20in%20Narcissus%20tazetta%20var&amp;author=M.%20Li&amp;author=D.%20Chen&amp;author=L.%20Wang&amp;journal=Chinensis%20Acta%20Horti%20Sinica&amp;volume=7&amp;issue=2&amp;pages=29-39&amp;publication_year=1980"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Merker A. A Giemsa technique for rapid identification of chromosomes in <em class="EmphasisTypeItalic">Triticale</em>. Hereditas. 1973;75(2):280–2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1601-5223.1973.tb01171.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20Giemsa%20technique%20for%20rapid%20identification%20of%20chromosomes%20in%20Triticale%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=A.%20Merker&amp;journal=Hereditas&amp;volume=75&amp;issue=2&amp;pages=280-2&amp;publication_year=1973"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Weimarck A. Heterochromatin polymorphism in the rye karyotype as detected by the Giemsa C-banding technique. Hereditas. 1975;79:293–300.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1601-5223.1975.tb01486.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1141006"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Heterochromatin%20polymorphism%20in%20the%20rye%20karyotype%20as%20detected%20by%20the%20Giemsa%20C-banding%20technique&amp;author=A.%20Weimarck&amp;journal=Hereditas&amp;volume=79&amp;pages=293-300&amp;publication_year=1975"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sejdler LK, Dabrowska J. Studies on the biology of flowering and fruiting of purple coneflower (<em class="EmphasisTypeItalic">Echinacea purpurea</em> Moench). Part 1. Biology of flowering and fruiting. Herba Polonica. 1998;42:83–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Studies%20on%20the%20biology%20of%20flowering%20and%20fruiting%20of%20purple%20coneflower%20%28Echinacea%20purpurea%20Moench%29.%20Part%201.%20Biology%20of%20flowering%20and%20fruiting&amp;author=LK.%20Sejdler&amp;author=J.%20Dabrowska&amp;journal=Herba%20Polonica&amp;volume=42&amp;pages=83-7&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stephens LC. Self-incompatibility in <em class="EmphasisTypeItalic">Echinacea purpurea</em>. HortSci. 2008;43:1350–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Self-incompatibility%20in%20Echinacea%20purpurea%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=LC.%20Stephens&amp;journal=HortSci&amp;volume=43&amp;pages=1350-4&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">© The Author(s) 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0016-y" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-016-0016-y">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0016-y" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0016-y?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0016-y #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0016-y #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0016-y #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0016-y">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0016-y">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0016-y"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0016-y">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0016-y">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0016-y"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0016-y">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0016-y">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0016-y"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0016-y">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0016-y">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0016-y"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0016-y">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0016-y">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0016-y"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-016-0016-y/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-016-0016-y">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0016-y">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0016-y" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-016-0016-y&amp;volume=153&amp;issue=1&amp;title=A+C-banded+karyotype+of+mitotic+chromosomes+in+diploid+purple+coneflower+%28+Echinacea+purpurea+L.%29&amp;author_name=Weizhen+Jiang&amp;start_page=1&amp;end_page=5">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-016-0016-y" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
