<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Assessment of genetic diversity in Nordic timothy (Phleum pratense L.) | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Timothy (Phleum pratense L.), a cool-season hexaploid perennial, is the most important forage grass species in Nordic countries. Earlier analyses of genetic diversity in a collection of 96 genebank accessions of timothy with SSR markers demonstrated high levels of diversity but could not resolve population structure. Therefore, we examined a subset of 51 accessions with REMAP markers, which are based on retrotransposons, and compared the diversity results with those obtained with SSR markers. Using four primer combinations, 533 REMAP markers were analyzed, compared with 464 polymorphic alleles in the 13 SSR loci previously. The average marker index, which describes information obtained per experiment (per primer combination or locus) was over six times higher with REMAPs. Most of the variation found was within accessions, with somewhat less, 89Â %, for REMAPs, than for SSR, with 93Â %. SSRs revealed differences in the level of diversity slightly better than REMAPs but neither marker type could reveal any clear clustering of accessions based on countries, vegetation zones, or different cultivar types. In our study, reliable evaluation of SSR allele dosages was not possible, so each allele had to be handled as a dominant marker. SSR and REMAP, which report from different mechanisms of generating genetic diversity and from different genomic regions, together indicate a lack of population structure. Taken together, this likely reflects the outcrossing and hexaploid nature of timothy rather than failures of either marker system.">
<meta name="description" content="Timothy (Phleum pratense L.), a cool-season hexaploid perennial, is the most important forage grass species in Nordic countries. Earlier analyses of genetic diversity in a collection of 96 genebank accessions of timothy with SSR markers demonstrated high levels of diversity but could not resolve population structure. Therefore, we examined a subset of 51 accessions with REMAP markers, which are based on retrotransposons, and compared the diversity results with those obtained with SSR markers. Using four primer combinations, 533 REMAP markers were analyzed, compared with 464 polymorphic alleles in the 13 SSR loci previously. The average marker index, which describes information obtained per experiment (per primer combination or locus) was over six times higher with REMAPs. Most of the variation found was within accessions, with somewhat less, 89Â %, for REMAPs, than for SSR, with 93Â %. SSRs revealed differences in the level of diversity slightly better than REMAPs but neither marker type could reveal any clear clustering of accessions based on countries, vegetation zones, or different cultivar types. In our study, reliable evaluation of SSR allele dosages was not possible, so each allele had to be handled as a dominant marker. SSR and REMAP, which report from different mechanisms of generating genetic diversity and from different genomic regions, together indicate a lack of population structure. Taken together, this likely reflects the outcrossing and hexaploid nature of timothy rather than failures of either marker system.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Assessment of genetic diversity in Nordic timothy ( Phleum pratense L . )">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-04-26">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 Tanhuanpää et al.">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Timothy (Phleum pratense L.), a cool-season hexaploid perennial, is the most important forage grass species in Nordic countries. Earlier analyses of genetic diversity in a collection of 96 genebank accessions of timothy with SSR markers demonstrated high levels of diversity but could not resolve population structure. Therefore, we examined a subset of 51 accessions with REMAP markers, which are based on retrotransposons, and compared the diversity results with those obtained with SSR markers. Using four primer combinations, 533 REMAP markers were analyzed, compared with 464 polymorphic alleles in the 13 SSR loci previously. The average marker index, which describes information obtained per experiment (per primer combination or locus) was over six times higher with REMAPs. Most of the variation found was within accessions, with somewhat less, 89 %, for REMAPs, than for SSR, with 93 %. SSRs revealed differences in the level of diversity slightly better than REMAPs but neither marker type could reveal any clear clustering of accessions based on countries, vegetation zones, or different cultivar types. In our study, reliable evaluation of SSR allele dosages was not possible, so each allele had to be handled as a dominant marker. SSR and REMAP, which report from different mechanisms of generating genetic diversity and from different genomic regions, together indicate a lack of population structure. Taken together, this likely reflects the outcrossing and hexaploid nature of timothy rather than failures of either marker system.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-04-26">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="5">
<meta name="prism.copyright" content="2016 Tanhuanpää et al.">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0009-x">
<meta name="prism.doi" content="doi:10.1186/s41065-016-0009-x">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0009-x">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0009-x">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Assessment of genetic diversity in Nordic timothy ( Phleum pratense L . )">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/04/26">
<meta name="citation_firstpage" content="5">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-016-0009-x">
<meta name="DOI" content="10.1186/s41065-016-0009-x">
<meta name="citation_doi" content="10.1186/s41065-016-0009-x">
<meta name="dc.creator" content="Pirjo Tanhuanpää">
<meta name="dc.creator" content="Maria Erkkilä">
<meta name="dc.creator" content="Ruslan Kalendar">
<meta name="dc.creator" content="Alan Howard Schulman">
<meta name="dc.creator" content="Outi Manninen">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=High SSR diversity but little differentiation between accessions of Nordic timothy (Phleum pratense L.); citation_author=P Tanhuanpää, O Manninen; citation_volume=149; citation_publication_date=2012; citation_pages=114-127; citation_doi=10.1111/j.1601-5223.2012.02244.x; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=IRAP and REMAP: two new retrotransposon-based DNA fingerprinting techniques; citation_author=R Kalendar, T Grob, M Regina, A Suoniemi, A Schulman; citation_volume=98; citation_publication_date=1999; citation_pages=704-711; citation_doi=10.1007/s001220051124; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=Nat protocols; citation_title=IRAP and REMAP for retrotransposon-based genotyping and fingerprinting; citation_author=R Kalendar, AH Schulman; citation_volume=1; citation_publication_date=2006; citation_pages=2478-2484; citation_doi=10.1038/nprot.2006.377; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=Curr Opin Virol; citation_title=Retrotransposon replication in plants; citation_author=AH Schulman; citation_volume=3; citation_publication_date=2013; citation_pages=604-614; citation_doi=10.1016/j.coviro.2013.08.009; citation_id=CR4">
<meta name="citation_reference" content="citation_title=Organization of retrotransposons and microsatellites in cereal genomes; citation_inbook_title=Cereal Genomics; citation_publication_date=2004; citation_pages=83-118; citation_id=CR5; citation_author=AH Schulman; citation_author=PK Gupta; citation_author=RK Varshney; citation_publisher=Kluwer Academic Publishers">
<meta name="citation_reference" content="citation_journal_title=Biochim Biophys Acta; citation_title=LTR retrotransposons, handy hitchhikers of plant regulation and stress response; citation_author=MA Grandbastien; citation_volume=2015; citation_publication_date=1849; citation_pages=403-416; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=A matter of life or death: how microsatellites emerge in and vanish from the human genome; citation_author=YD Kelkar, KA Eckert, F Chiaromonte, KD Makova; citation_volume=21; citation_publication_date=2011; citation_pages=2038-2048; citation_doi=10.1101/gr.122937.111; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=Mol Phylogenet Evol; citation_title=Assessment of genetic diversity among Indian potato (Solanum tuberosum L.) collection using microsatellite and retrotransposon based marker systems; citation_author=V Sharma, R Nandineni; citation_volume=73; citation_publication_date=2014; citation_pages=10-17; citation_doi=10.1016/j.ympev.2014.01.003; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=J Agr Sci Tech; citation_title=Comparative assessment of IRAP, REMAP, ISSR, and SSR markers for evaluation of genetic diversity of alfalfa (Medicago sativa L.); citation_author=BA Mandoulakani, P Sadigh, H Azizi, Y Piri, S Nasri, S Arzhangh; citation_volume=17; citation_publication_date=2015; citation_pages=999-1010; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Mol Biotechnol; citation_title=Use of SSR and retrotransposon-based markers to interpret the population structure of native grapevines from southern Italy; citation_author=C Villano, D Carputo, L Frusciante, X Santoro, R Aversano; citation_volume=56; citation_publication_date=2014; citation_pages=1011-1020; citation_doi=10.1007/s12033-014-9780-y; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Random amplified polymorphic DNA and pedigree relationships in spring barley; citation_author=NA Tinker, MG Fortin, DE Mather; citation_volume=85; citation_publication_date=1993; citation_pages=976-984; citation_doi=10.1007/BF00215037; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=iPBS: A universal method for DNA fingerprinting and retrotransposon isolation; citation_author=R Kalendar, K Antonius, P Smykal, AH Schulman; citation_volume=121; citation_publication_date=2010; citation_pages=1419-1430; citation_doi=10.1007/s00122-010-1398-2; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Methods Mol Biol; citation_title=FastPCR software for PCR, in silico PCR, and oligonucleotide assembly and analysis; citation_author=R Kalendar, D Lee, AH Schulman; citation_volume=1116; citation_publication_date=2014; citation_pages=271-302; citation_doi=10.1007/978-1-62703-764-8_18; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Isolation and characterization of simple sequence repeat markers in the hexaploid forage grass timothy (Phleum pratense L.); citation_author=HW Cai, N Yuyama, H Tamaki, A Yoshizawa; citation_volume=107; citation_publication_date=2003; citation_pages=1337-1349; citation_doi=10.1007/s00122-003-1386-x; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Powermarker: integrated analysis environment for genetic marker data; citation_author=K Liu, SV Muse; citation_volume=21; citation_publication_date=2005; citation_pages=2128-2129; citation_doi=10.1093/bioinformatics/bti282; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=The comparison of RFLP, RAPD, AFLP and SSR (microsatellite) markers for germplasm analysis; citation_author=W Powell, M Morgante, C Andre, M Hanafey, J Vogel, S Tingey; citation_volume=2; citation_publication_date=1996; citation_pages=225-238; citation_doi=10.1007/BF00564200; citation_id=CR16">
<meta name="citation_reference" content="Schneider S, Roessli D, Excoffier L. Arlequin: a software for population genetics data analysis. Ver 2.000. Genetics and Biometry Lab, Department of Anthropology, University of Geneva, Switzerland; 2000.">
<meta name="citation_reference" content="citation_journal_title=Bell System Tech J; citation_title=A mathematical theory of communication; citation_author=CE Shannon; citation_volume=27; citation_publication_date=1948; citation_pages=379–423 and 623–56; citation_doi=10.1002/j.1538-7305.1948.tb01338.x; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol Notes; citation_title=GENALEX 6: genetic analysis in Excel. Population genetic software for teaching and research; citation_author=R Peakall, PE Smouse; citation_volume=6; citation_publication_date=2006; citation_pages=288-295; citation_doi=10.1111/j.1471-8286.2005.01155.x; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research – an update; citation_author=R Peakall, PE Smouse; citation_volume=28; citation_publication_date=2012; citation_pages=2537-2539; citation_doi=10.1093/bioinformatics/bts460; citation_id=CR20">
<meta name="citation_reference" content="citation_title=National Atlas of Norway: Vegetation; citation_publication_date=1999; citation_id=CR21; citation_author=A Moen; citation_publisher=Norwegian Mapping Authority">
<meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction sites; citation_author=L Excoffier, PE Smouse, JM Quattro; citation_volume=131; citation_publication_date=1992; citation_pages=479-491; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=Am Nat; citation_title=Genetic distance between populations; citation_author=M Nei; citation_volume=106; citation_publication_date=1972; citation_pages=283-292; citation_doi=10.1086/282771; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=Cancer Res; citation_title=The detection of disease clustering and a generalized regression approach; citation_author=NA Mantel; citation_volume=27; citation_publication_date=1967; citation_pages=209-220; citation_id=CR24">
<meta name="citation_reference" content="Stewart AW, Joachimiak AJ, Ellison NW. Phleum. In: Kole C, editor. Wild crop relatives: genomic and breeding resources. Millets and grasses. Springer Berlin Heidelberg; 2011. p. 257–74.">
<meta name="citation_reference" content="citation_journal_title=Curr Opin Genet Dev; citation_title=Polyploidy and genome evolution in plants; citation_author=PS Soltis, DB Marchant, Y Peer, DE Soltis; citation_volume=35; citation_publication_date=2015; citation_pages=119-125; citation_doi=10.1016/j.gde.2015.11.003; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check; citation_author=A Blenda, DD Fang, J-F Rami, O Garsmeur, F Luo, J-M Lacape; citation_volume=7; citation_publication_date=2012; citation_doi=10.1371/journal.pone.0045739; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=Solving the problem of ambiguous paralogy for marker loci: microsatellite markers with diploid inheritance in allohexaploid Mercurialis annua (Euphorbiaceae); citation_author=G Korbecka, PD Rymer, SA Harris, JR Pannell; citation_volume=101; citation_publication_date=2010; citation_pages=504-511; citation_doi=10.1093/jhered/esq026; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Mol Genet Genomics; citation_title=Molecular markers based on LTR retrotransposons BARE-1 and Jeli uncover different strata of evolutionary relationships in diploid wheats; citation_author=FA Konovalov, NP Goncharov, S Goryunova, A Shaturova, T Proshlyakova, A Kudryavtsev; citation_volume=283; citation_publication_date=2010; citation_pages=551-563; citation_doi=10.1007/s00438-010-0539-2; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=J Exp Bot; citation_title=Using quantitative PCR with retrotransposon-based insertion polymorphisms as markers in sugarcane; citation_author=CJ Metcalfe, SG Oliveira, JW Gaiarsa, KS Aitken, MS Carneiro, F Zatti; citation_volume=66; citation_publication_date=2015; citation_pages=4239-4250; citation_doi=10.1093/jxb/erv283; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Comparison of different nuclear DNA markers for estimating intraspecific genetic diversity in plants; citation_author=H Nybom; citation_volume=13; citation_publication_date=2004; citation_pages=1143-1155; citation_doi=10.1111/j.1365-294X.2004.02141.x; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Comparative analysis of genetic diversity in the mangrove species Avicennia marina (Forsk.) Vierh. (Avicenniaceae) detected by AFLPs and SSRs; citation_author=TL Maguire, R Peakall, P Saenger; citation_volume=104; citation_publication_date=2002; citation_pages=388-398; citation_doi=10.1007/s001220100724; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=
                              In silico comparative analysis of SSR markers in plants; citation_author=FC Victoria, LC Maia, AC Oliveira; citation_volume=11; citation_publication_date=2011; citation_pages=15; citation_doi=10.1186/1471-2229-11-15; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Methods Mol Biol; citation_title=The application of LTR retrotransposons as molecular markers in plants; citation_author=AH Schulman, A Flavell, E Paux, THN Ellis; citation_volume=859; citation_publication_date=2012; citation_pages=115-153; citation_doi=10.1007/978-1-61779-603-6_7; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids; citation_author=AAF Garcia, M Mollinari, TG Marconi, OR Serang, RR Silva, MLC Vieira; citation_volume=3; citation_publication_date=2013; citation_pages=3399; citation_doi=10.1038/srep03399; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species; citation_author=RJ Elshire, JC Glaubitz, Q Sun, JA Poland, K Kawamoto, ES Buckler; citation_volume=6; citation_publication_date=2011; citation_doi=10.1371/journal.pone.0019379; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Genome wide allele frequency fingerprints (GWAFFs) of populations via genotyping by sequencing; citation_author=S Byrne, A Czaban, B Studer, F Panitz, C Bendixen, T Asp; citation_volume=8; citation_publication_date=2013; citation_doi=10.1371/journal.pone.0057438; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Phenotypic or molecular diversity screening for conservation of genetic resources? An example from a genebank collection of the temperate forage grass timothy; citation_author=S Fjellheim, P Tanhuanpää, P Marum, O Manninen, OA Rognli; citation_volume=55; citation_publication_date=2015; citation_pages=1646-1659; citation_doi=10.2135/cropsci2014.12.0825; citation_id=CR38">
<meta name="citation_author" content="Pirjo Tanhuanpää">
<meta name="citation_author_institution" content="Green Technology, Natural Resources Institute Finland (Luke), Jokioinen, Finland">
<meta name="citation_author" content="Maria Erkkilä">
<meta name="citation_author_institution" content="Internal Expert Services, Natural Resources Institute Finland (Luke), Jokioinen, Finland">
<meta name="citation_author" content="Ruslan Kalendar">
<meta name="citation_author_institution" content="Internal Expert Services, Natural Resources Institute Finland (Luke), Jokioinen, Finland">
<meta name="citation_author" content="Alan Howard Schulman">
<meta name="citation_author_institution" content="Green Technology, Natural Resources Institute Finland (Luke), Jokioinen, Finland">
<meta name="citation_author_institution" content="Luke/BI Plant Genome Dynamics Laboratory, Institute of Biotechnology, Viikki Biocenter, University of Helsinki, Helsinki, Finland">
<meta name="citation_author" content="Outi Manninen">
<meta name="citation_author_institution" content="Boreal Plant Breeding Ltd, Jokioinen, Finland">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0009-x',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0009-x","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Genetic diversity;Genetic structure;Phleum pratense L;REMAP;Retrotransposon marker;SSR;Microsatellite;Timothy"},"contentInfo":{"imprint":"BioMed Central","title":"Assessment of genetic diversity in Nordic timothy ( Phleum pratense L . )","publishedAt":1461628800000,"publishedAtDate":"2016-04-26","author":["Pirjo TanhuanpÃ¤Ã¤","Maria ErkkilÃ¤","Ruslan Kalendar","Alan Howard Schulman","Outi Manninen"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0009-x">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0009-x">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Assessment of genetic diversity in Nordic timothy ( Phleum pratense L . )">
<meta property="og:description" content="Timothy (Phleum pratense L.), a cool-season hexaploid perennial, is the most important forage grass species in Nordic countries. Earlier analyses of genetic diversity in a collection of 96 genebank accessions of timothy with SSR markers demonstrated high levels of diversity but could not resolve population structure. Therefore, we examined a subset of 51 accessions with REMAP markers, which are based on retrotransposons, and compared the diversity results with those obtained with SSR markers. Using four primer combinations, 533 REMAP markers were analyzed, compared with 464 polymorphic alleles in the 13 SSR loci previously. The average marker index, which describes information obtained per experiment (per primer combination or locus) was over six times higher with REMAPs. Most of the variation found was within accessions, with somewhat less, 89&amp;nbsp;%, for REMAPs, than for SSR, with 93&amp;nbsp;%. SSRs revealed differences in the level of diversity slightly better than REMAPs but neither marker type could reveal any clear clustering of accessions based on countries, vegetation zones, or different cultivar types. In our study, reliable evaluation of SSR allele dosages was not possible, so each allele had to be handled as a dominant marker. SSR and REMAP, which report from different mechanisms of generating genetic diversity and from different genomic regions, together indicate a lack of population structure. Taken together, this likely reflects the outcrossing and hexaploid nature of timothy rather than failures of either marker system.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0009-x;kwrd=Genetic diversity,Genetic structure,Phleum pratense L,REMAP,Retrotransposon marker,SSR,Microsatellite,Timothy;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0009-x&amp;kwrd=Genetic%20diversity,Genetic%20structure,Phleum%20pratense%20L,REMAP,Retrotransposon%20marker,SSR,Microsatellite,Timothy&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0009-x&amp;kwrd=Genetic%20diversity,Genetic%20structure,Phleum%20pratense%20L,REMAP,Retrotransposon%20marker,SSR,Microsatellite,Timothy&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Assessment of genetic diversity in Nordic timothy (<em xmlns="" class="EmphasisTypeItalic">Phleum pratense</em> L<em xmlns="" class="EmphasisTypeItalic">.</em>)</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Pirjo TanhuanpÃ¤Ã¤</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:pirjo.tanhuanpaa@luke.fi" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Maria ErkkilÃ¤</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ruslan Kalendar</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Alan Howard Schulman</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff3" class="AffiliationID">3</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Outi Manninen</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup>
</li>
</ul></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:5</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0009-x">https://doi.org/10.1186/s41065-016-0009-x</a></p>
<p class="Copyright">Â©  TanhuanpÃ¤Ã¤ et al. 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>18 December 2015</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>19 April 2016</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">26 April 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Timothy (<em xmlns="" class="EmphasisTypeItalic">Phleum pratense</em> L.), a cool-season hexaploid perennial, is the most important forage grass species in Nordic countries. Earlier analyses of genetic diversity in a collection of 96 genebank accessions of timothy with SSR markers demonstrated high levels of diversity but could not resolve population structure. Therefore, we examined a subset of 51 accessions with REMAP markers, which are based on retrotransposons, and compared the diversity results with those obtained with SSR markers.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Using four primer combinations, 533 REMAP markers were analyzed, compared with 464 polymorphic alleles in the 13 SSR loci previously. The average marker index, which describes information obtained per experiment (per primer combination or locus) was over six times higher with REMAPs. Most of the variation found was within accessions, with somewhat less, 89 %, for REMAPs, than for SSR, with 93 %.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">SSRs revealed differences in the level of diversity slightly better than REMAPs but neither marker type could reveal any clear clustering of accessions based on countries, vegetation zones, or different cultivar types. In our study, reliable evaluation of SSR allele dosages was not possible, so each allele had to be handled as a dominant marker. SSR and REMAP, which report from different mechanisms of generating genetic diversity and from different genomic regions, together indicate a lack of population structure. Taken together, this likely reflects the outcrossing and hexaploid nature of timothy rather than failures of either marker system.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Genetic diversity</li>
<li class="c-keywords__item" itemprop="about">Genetic structure</li>
<li class="c-keywords__item" itemprop="about">
                        <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Phleum pratense</em> L</li>
<li class="c-keywords__item" itemprop="about">REMAP</li>
<li class="c-keywords__item" itemprop="about">Retrotransposon marker</li>
<li class="c-keywords__item" itemprop="about">SSR</li>
<li class="c-keywords__item" itemprop="about">Microsatellite</li>
<li class="c-keywords__item" itemprop="about">Timothy</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 SectionTypeIntroduction c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Timothy (<em xmlns="" class="EmphasisTypeItalic">Phleum pratense</em> L.), a cool-season perennial, is the most important forage grass species in Nordic countries. Genetic diversity has been previously assessed [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] in a collection of 96 timothy accessions, of which 88 were of Nordic origin. Simple sequence repeat (SSR) markers revealed Nordic timothy accessions to be very polymorphic, having significant differences in the levels of diversity between countries, vegetation zones, and different cultivar types. However, most of the variation (94 %) existed within accessions, and no clear clustering of accessions based on any grouping was observed. This lack of resolution may either reflect the outcrossing and hexaploid nature of timothy or that SSR markers are not suitable for resolving population structure in timothy.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">A wide range of DNA markers are available for diversity studies, which all have their advantages and disadvantages. SSRs are amplified from single loci, but are multiallelic and highly polymorphic. Although they are inherited codominantly, separation of different genotypes may not be possible in a polyploid species such as timothy. Therefore, each allele has to be treated as a dominant marker [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]; consequently, the markers that are amplified from the same SSR locus are not independent of each other, and consequentially information is lost. In the REMAP (retrotransposon-microsatellite amplified polymorphism) markers [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>] assay, the diversity is generated by the integration of retrotransposons, which move in the genome by a copy-and-paste mechanism but are fixed in position upon insertion [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. They are ubiquitous and abundant in plant genomes, where they are dispersed on all chromosomes [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. REMAP markers are amplified using a primer designed to a conserved retrotransposon region and another anchored to a simple sequence repeat. Products from multiple loci are produced in one PCR reaction, each with only two allele alternatives, a dominant one (amplification) and a recessive one (non-amplification). Because the mechanisms that activate retrotransposons [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] and thereby generate insertional polymorphisms are fully different than that generating SSR allelic variation (polymerase slippage) [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], the two marker systems assay different components of genetic diversity. For potato [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>], alfalfa [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>], and grapevine [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>], retrotransposon and SSR markers in combination were shown to be highly discriminatory and effective.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">In the previous study [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], a collection of 96 timothy accessions was analyzed using 13 SSRs, thus describing diversity only at this number of loci. On the other hand, these 13 SSR loci harbored as many as 499 alleles. In the present study, we used REMAP markers for studying diversity in a subset of 51 accessions and compared the results with those obtained with SSR markers. We wanted to determine if another type of marker, which would report from many more locations in the genome and assess different genomic regions where diversity is generated by a different mechanism, could describe diversity more efficiently and also reveal population structure, particularly for a polyploid species. Especially the autonomous nature of retrotransposon diversity generation and display, which is independent of the syntenic organization of polyploids, appeared suited to clonal polyploid species such as timothy. We expected that the retrotransposon markers should thereby be more likely to find genetic structure in timothy, should it exist.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Plant material</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">In the previous study [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], SSR markers were analyzed in a collection of 96 timothy accessions. Fifty-one of these were selected for the present study to be screened also with REMAP markers (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Fifteen to twenty randomly selected individuals per accession were investigated, in total 945 individuals. The number of individuals analyzed from each accession in the two studies was not exactly the same because 20 individuals had to be omitted due to their poor amplification in REMAP analysis.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Fifty-one <em class="EmphasisTypeItalic">Phleum pratense</em> ssp. <em class="EmphasisTypeItalic">pratense</em> accessions analyzed in the study</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Number code</p></th>
<th><p xmlns="" class="SimplePara">Accession no.</p></th>
<th><p xmlns="" class="SimplePara">Genebank</p></th>
<th><p xmlns="" class="SimplePara">Name</p></th>
<th><p xmlns="" class="SimplePara">Country</p></th>
<th><p xmlns="" class="SimplePara">Cultivar type<sup xmlns="http://www.w3.org/1999/xhtml">1</sup></p></th>
<th><p xmlns="" class="SimplePara">Veg. zone<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">NGB10830</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">VA88119</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">NGB10831</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">HF88266</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">NGB15461</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">Vildbjerg AC0103</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">6</p></td>
<td><p xmlns="" class="SimplePara">NGB16650</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">Ejsing</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">NGB1672</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">BILBO</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">NGB1675</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">POTA</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">NGB4053</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">SR SALTUM MH0202</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">NGB4548</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">NR FARUP MH0202</p></td>
<td><p xmlns="" class="SimplePara">Denmark</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">NGB132</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">LIPINLAHTI ME0901 SEP A</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">NGB14394</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">KÃRKÃLÃ HM0102</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">NGB14404</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">PAATTINEN MH0201</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">NGB14417</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">MEDVASTÃ MH0101</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">NGB747</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">NUVVUS AK0401</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">NGB1095</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">LAITASAARI ME0201</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">27</p></td>
<td><p xmlns="" class="SimplePara">NGB1111</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">MÃLÃSKÃ ME0101</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">30</p></td>
<td><p xmlns="" class="SimplePara">NGB1119</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">KATERMA ME0401</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">32</p></td>
<td><p xmlns="" class="SimplePara">NGB2791</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">NORRGÃRD AP0101</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">35</p></td>
<td><p xmlns="" class="SimplePara">NGB4066</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">TAMMISTO</p></td>
<td><p xmlns="" class="SimplePara">Finland</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">36</p></td>
<td><p xmlns="" class="SimplePara">NGB4140</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">KORPA</p></td>
<td><p xmlns="" class="SimplePara">Iceland</p></td>
<td><p xmlns="" class="SimplePara">L</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">37</p></td>
<td><p xmlns="" class="SimplePara">NGB4141</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">ADDA</p></td>
<td><p xmlns="" class="SimplePara">Iceland</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">42</p></td>
<td><p xmlns="" class="SimplePara">NGB7592</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">SKJÃLSVIK 01-5-46-5</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">45</p></td>
<td><p xmlns="" class="SimplePara">NGB10785</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">SANDBU 01-6-49-4</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">47</p></td>
<td><p xmlns="" class="SimplePara">NGB17194</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">Ifjord 1-1-2-2</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">48</p></td>
<td><p xmlns="" class="SimplePara">NGB17198</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">Karasjok 1-1-3-2</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">49</p></td>
<td><p xmlns="" class="SimplePara">NGB2169</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">BODIN</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">51</p></td>
<td><p xmlns="" class="SimplePara">NGB2180</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">GRINDSTAD</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">53</p></td>
<td><p xmlns="" class="SimplePara">NGB2918</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">HUSETER 01-9-70-1</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">57</p></td>
<td><p xmlns="" class="SimplePara">NGB4226</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">HATLESTAD 01-7-56-3</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">59</p></td>
<td><p xmlns="" class="SimplePara">NGB4231</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">GJERDÃKER 01-7-58-1</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">62</p></td>
<td><p xmlns="" class="SimplePara">NGB7548</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">NAMSVATN 01-5-40-1</p></td>
<td><p xmlns="" class="SimplePara">Norway</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">64</p></td>
<td><p xmlns="" class="SimplePara">NGB722</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">KUOSSENJARKA JP0404</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">65</p></td>
<td><p xmlns="" class="SimplePara">NGB728</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">PJESKER PH0405</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">66</p></td>
<td><p xmlns="" class="SimplePara">NGB11428</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">JONATHAN</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">CV</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">69</p></td>
<td><p xmlns="" class="SimplePara">NGB14224</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">SÃNDRARP IB0101</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">71</p></td>
<td><p xmlns="" class="SimplePara">NGB731</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">RÃRMYRBERG JP0204</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">73</p></td>
<td><p xmlns="" class="SimplePara">NGB16975</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">NORRA KYLSÃTER FO0103</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">76</p></td>
<td><p xmlns="" class="SimplePara">NGB16981</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">BRÃCKETORP FO0501</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">78</p></td>
<td><p xmlns="" class="SimplePara">NGB1306</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">BRATTÃKER GB0101</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">81</p></td>
<td><p xmlns="" class="SimplePara">NGB1327</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">HAMMARN PR0401</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">83</p></td>
<td><p xmlns="" class="SimplePara">NGB1331</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">VÃSTANSJÃ SH0102</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">85</p></td>
<td><p xmlns="" class="SimplePara">NGB1537</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">ESKELHEM TL0104</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">86</p></td>
<td><p xmlns="" class="SimplePara">NGB2530</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">RÃMNE GJ0301</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">87</p></td>
<td><p xmlns="" class="SimplePara">NGB4349</p></td>
<td><p xmlns="" class="SimplePara">Nordgen</p></td>
<td><p xmlns="" class="SimplePara">BENESTAD JK1506</p></td>
<td><p xmlns="" class="SimplePara">Sweden</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">89</p></td>
<td><p xmlns="" class="SimplePara">PI381926</p></td>
<td><p xmlns="" class="SimplePara">GRIN</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">France</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">90</p></td>
<td><p xmlns="" class="SimplePara">PI406317</p></td>
<td><p xmlns="" class="SimplePara">GRIN</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Russia</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">91</p></td>
<td><p xmlns="" class="SimplePara">IHAR151908</p></td>
<td><p xmlns="" class="SimplePara">IHAR</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Germany</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">92</p></td>
<td><p xmlns="" class="SimplePara">PI210426</p></td>
<td><p xmlns="" class="SimplePara">GRIN</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Greece</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">93</p></td>
<td><p xmlns="" class="SimplePara">PI325461</p></td>
<td><p xmlns="" class="SimplePara">GRIN</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Russia</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">94</p></td>
<td><p xmlns="" class="SimplePara">PI204480</p></td>
<td><p xmlns="" class="SimplePara">GRIN</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Turkey</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">95</p></td>
<td><p xmlns="" class="SimplePara">14G2400116</p></td>
<td><p xmlns="" class="SimplePara">RICP</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Czech Republic</p></td>
<td><p xmlns="" class="SimplePara">P</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">96</p></td>
<td><p xmlns="" class="SimplePara">RCAT040682</p></td>
<td><p xmlns="" class="SimplePara">RCAT</p></td>
<td> </td>
<td><p xmlns="" class="SimplePara">Hungary</p></td>
<td><p xmlns="" class="SimplePara">W</p></td>
<td> </td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">1</sup><em class="EmphasisTypeItalic">CV</em> advanced cultivar, <em class="EmphasisTypeItalic">L</em> traditional cultivar, landrace, <em class="EmphasisTypeItalic">P</em> pending, unknown cultivar type, <em class="EmphasisTypeItalic">W</em> wild population, weedy</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>vegetation zones, according to [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]</p>
                    </div></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">The 51 accessions were mostly wild (30, locations in Fig. <span class="InternalRef"><a href="#Fig1">1</a></span> in [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]); seven each were classified as landraces, cultivars, and of unknown cultivar types. Accessions were derived from all Scandinavian countries (Denmark, 8; Finland, 10; Iceland, 2; Norway, 10; Sweden, 13). In addition, eight gene bank accessions (so-called exotics) originating from non-Scandinavian countries were included in the study.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0009-x/MediaObjects/41065_2016_9_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0009-x/MediaObjects/41065_2016_9_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0009-x/MediaObjects/41065_2016_9_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0009-x/MediaObjects/41065_2016_9_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Principal coordinate analysis based on Neiâs genetic distances between accessions based on: <strong class="EmphasisTypeBold">a</strong> 533 REMAP markers; <strong class="EmphasisTypeBold">b</strong> 464 SSR markers; <strong class="EmphasisTypeBold">c</strong> 997 REMAP and SSR markers</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">Marker analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">DNA was extracted using the method of Tinker et al. [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] with some modifications as described in TanhuanpÃ¤Ã¤ and Manninen [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Using the iPBS (inter- primer binding site) method, retrotransposon segments were isolated from the timothy genome, sequenced, and long terminal repeats (LTRs) identified [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. LTR primers were designed to match conserved motifs at or near their termini, according to the methods of Kalendar et al. [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. For REMAP marker amplification, four different retrotransposon primers (TIM1 - 4) for grasses were used. These were combined with 19 microsatellite-based primers (ISSRâ+ânumber) that contain repeat units (composed of two or three bases); the 3â² ends of the primers were anchored by a single nucleotide. Because analyzing markers by gel electrophoresis is very laborious, the retrotransposon primers were labelled with a fluorescent dye, FAM (5-carboxyfluorescein), HEX (hexachloro-6-carboxyfluorescein), or TET (6-carboxytetrachlorofluorescein) to enable resolution and visualization of amplification products with a MegaBACE<sup>TM</sup> 500 Sequencer (GE Healthcare, Buckinghamshire, UK).</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Fifty-nine REMAP primer combinations were first tested in a small set of individuals for their functionality and efficiency to produce polymorphic bands. The four best primer combinations were chosen for final analyses (TIM1 with ISSR1, 15 and 20, and TIM2 with ISSR5). These primers, together with their sequences and properties, are shown in Table <span class="InternalRef"><a href="#Tab2">2</a></span>. The REMAP markers were amplified in a reaction volume of 10 Î¼l, using 0.25 U of FIREPolÂ® DNA polymerase I (Solis BioDyne OU, Tartu, Estonia), buffer B with 2.5 mM MgCl<sub>2</sub> as supplied by the enzyme manufacturer, 200 Î¼mol/L each dNTP, 10 ng of DNA, and 500 nmol/L each primer. The PCR program was run on a PTC-220 DNA Engine Dyad<sup>TM</sup> Peltier Thermal Cycler (MJ Research, Waltham, MA, USA) and consisted of an initial denaturation step of 2 min at 94 Â°C; 32 cycles of 30 s at 94 Â°C, 30 s at 60 Â°C and 2 min at 72 Â°C; a final extension step of 10 min at 72 Â°C. After PCR, the amplified products with different labels were combined for MegaBACE runs. SSRs were developed for timothy [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], and analyses were run as described previously [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>].<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">REMAP primers that were used in the analysis of timothy diversity, their sequences and properties</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col char=".">
<col char=".">
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Name</p></th>
<th><p xmlns="" class="SimplePara">Sequence</p></th>
<th><p xmlns="" class="SimplePara">nt</p></th>
<th char="."><p xmlns="" class="SimplePara">Tm (Â°C)</p></th>
<th char="."><p xmlns="" class="SimplePara">CG %</p></th>
<th><p xmlns="" class="SimplePara">Linguistic complexity (%)</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">TIM1</p></td>
<td><p xmlns="" class="SimplePara">GGTGCCGGCATCGATCCTTTCA</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td char="."><p xmlns="" class="SimplePara">62.4</p></td>
<td char="."><p xmlns="" class="SimplePara">59.1</p></td>
<td><p xmlns="" class="SimplePara">88</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">TIM2</p></td>
<td><p xmlns="" class="SimplePara">ACGAGTGAGGACAAAGTGCGCAGA</p></td>
<td><p xmlns="" class="SimplePara">24</p></td>
<td char="."><p xmlns="" class="SimplePara">61.9</p></td>
<td char="."><p xmlns="" class="SimplePara">54.2</p></td>
<td><p xmlns="" class="SimplePara">79</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR1</p></td>
<td><p xmlns="" class="SimplePara">ACCACCACCACCACCACCC</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td char="."><p xmlns="" class="SimplePara">63.2</p></td>
<td char="."><p xmlns="" class="SimplePara">68.4</p></td>
<td><p xmlns="" class="SimplePara">24</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR5</p></td>
<td><p xmlns="" class="SimplePara">AGCAGCAGCAGCAGCAGCG</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td char="."><p xmlns="" class="SimplePara">64.4</p></td>
<td char="."><p xmlns="" class="SimplePara">68.4</p></td>
<td><p xmlns="" class="SimplePara">30</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR15</p></td>
<td><p xmlns="" class="SimplePara">GTGGTGGTGGTGGTGGTGGTGA</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td char="."><p xmlns="" class="SimplePara">64.2</p></td>
<td char="."><p xmlns="" class="SimplePara">63.6</p></td>
<td><p xmlns="" class="SimplePara">28</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ISSR20</p></td>
<td><p xmlns="" class="SimplePara">TGCTGCTGCTGCTGCTGCC</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td char="."><p xmlns="" class="SimplePara">64.6</p></td>
<td char="."><p xmlns="" class="SimplePara">68.4</p></td>
<td><p xmlns="" class="SimplePara">30</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">
                                    <em class="EmphasisTypeItalic">nt</em> nucleotides, <em class="EmphasisTypeItalic">Tm</em> melting temperature, <em class="EmphasisTypeItalic">CG %</em> percentage of C and G bases</p>
                    </div></figure>
</div>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Data analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Each REMAP fragment represents a separate locus, and the presence and absence of the fragment was scored in a binary code (1/0). Likewise, each SSR allele was treated as a separate locus and scored in a binary code, even though SSRs are codominant markers. This was because we found the evaluation of allele dosages very unreliable in hexaploid timothy. Diversity indices for markers, including polymorphic information content (PIC), gene diversity, and major allele frequency, were calculated with the program Powermarker v3.0 [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. A marker index (MI) for each REMAP primer combination and each SSR locus was determined by multiplying the number of polymorphic markers generated (EMFâ=âEffective multiplex ratio) by average PIC value [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. It illustrates the amount of information obtained per experiment (per primer combination or locus).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Genetic diversity in each accession was described with five different diversity indices: 1) the number of all markers observed (A<sub>A</sub>), corrected to a sample size of nâ=â15 with 1000 resamplings without replacement; 2) the mean number of all markers observed in each individual (A<sub>I</sub>); 3) the mean number of pairwise differences (PWD) (Euclidean distances) between individuals, which was counted with the program ARLEQUIN version 2.000 [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]; 4) Shannonâs diversity index <em xmlns="" class="EmphasisTypeItalic">I</em> [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]; 5) the percentage of polymorphic loci. The last two were calculated using the program GenAlex 6.4 [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Correlations between diversity indices based on REMAP and SSR markers, and differences in the level of diversity between different groups such as countries, vegetation zones [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], or cultivar types were determined by ANOVA Proc GLM (SAS Enterprise Guide 4.3).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The program GenAlex 6.4 [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] was used to perform analysis of molecular variance (AMOVA) [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] which partitions total genetic variation to within- and among-accession variance components. The significance of the results was tested by permuting the data 999 times. Principal coordinate analyses (PCA) based on Neiâs genetic distances [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] between accessions, and a Mantel test [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], which was used to compare Neiâs distances based on REMAP or SSR data, were carried out with the software GenAlex.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 SectionTypeResults c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Diversity at marker loci</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Four REMAP primer combinations were used for studying diversity of the 51 accessions. Because not all fragments could be read as marker peaks, selections were made on the basis of the size and shape of the peaks. The numbers of scored polymorphic markers produced by different primer combinations were as follows: TIM2â+âISSR5, 91; TIM1â+âISSR20, 84; TIM1â+âISSR1, 209; TIM1â+âISSR15, 149. A total of 533 REMAP markers were analyzed, ranging in size from 80 to 650 bp. A total of 464 polymorphic alleles in the 13 SSR loci were amplified from the 51 accessions, the number varying from 13 to 71 per accession [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. The average diversity indices of REMAP markers were higher than those of SSR markers (Table <span class="InternalRef"><a href="#Tab3">3</a></span>) leading to a six-fold higher MI for REMAPs.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Comparison of diversity measures of REMAP and SSR markers in the analysis of 51 timothy accessions</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th> </th>
<th><p xmlns="" class="SimplePara">REMAP</p></th>
<th><p xmlns="" class="SimplePara">SSR</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">No. of primer combinations or loci</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total no. of markers</p></td>
<td><p xmlns="" class="SimplePara">533</p></td>
<td><p xmlns="" class="SimplePara">464</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">No. of markers per primer combination or locusâ=âEMF<sup xmlns="http://www.w3.org/1999/xhtml">1</sup></p></td>
<td><p xmlns="" class="SimplePara">133.3</p></td>
<td><p xmlns="" class="SimplePara">35.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">PIC, average</p></td>
<td><p xmlns="" class="SimplePara">0.131</p></td>
<td><p xmlns="" class="SimplePara">0.086</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Markers with PICâ&gt;â0.1</p></td>
<td><p xmlns="" class="SimplePara">258â=â48 %</p></td>
<td><p xmlns="" class="SimplePara">148â=â32 %</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Markers with MAFâ&lt;â0.1</p></td>
<td><p xmlns="" class="SimplePara">365â=â68 %</p></td>
<td><p xmlns="" class="SimplePara">371â=â80 %</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Average gene diversity</p></td>
<td><p xmlns="" class="SimplePara">0.152</p></td>
<td><p xmlns="" class="SimplePara">0.098</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Marker index (MI)â=âEMF x PIC</p></td>
<td><p xmlns="" class="SimplePara">17.4</p></td>
<td><p xmlns="" class="SimplePara">3.1</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">1</sup>effective multiplex ratio</p>
                    </div></figure>
</div>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic diversity within accessions</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The observed number of REMAP markers per accession varied from 195 (PL204480) to 352 (NGB1672) (Table <span class="InternalRef"><a href="#Tab4">4</a></span>), and the number of SSR alleles from 95 (NGB10785) to 194 (NGB1111). There was only one private REMAP marker (in accession PL325461), but 43 private SSR alleles were found [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Diversity indices of accessions studied with REMAP or SSR markers, respectively, varied as follows: A<sub>I</sub> from 47.5 (PL204480) to 84.8 (NGB1672) and from 28.4 (NGB10831) to 35.2 (NGB7592); PWD from 53.1 (PL204480) to 100.2 (NGB1672) and from 28.9 (NGB10785) to 44.9 (NGB7592); <em xmlns="" class="EmphasisTypeItalic">I</em> from 0.159 (RCAT040682) to 0.280 (NGB1672), with mean of 0.203âÂ±â0.029, and from 0.109 (NGB722) to 0.156 (NGB1111), with mean of 0.138âÂ±â0.014; the percentage of polymorphic loci from 35.8 % (PL204480) to 64.9 % (NGB1672), with a mean of 49.0âÂ±â7.3 %, and from 19.8 % (NGB10785) to 41.4 % (NGB1095), with a mean of 34.4âÂ±â5.1 % (Table <span class="InternalRef"><a href="#Tab4">4</a></span>). The A<sub>I</sub> values based on SSRs changed slightly from the previous results [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] due to exclusion of 20 individuals (see <span class="InternalRef"><a href="#Sec2">Methods</a></span>).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">REMAP and SSR diversity in 51 timothy accessions</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead>
<tr class="header">
<th> </th>
<th> </th>
<th colspan="4"><p xmlns="" class="SimplePara">REMAP (total no. of markersâ=â533)</p></th>
<th> </th>
<th> </th>
<th colspan="4"><p xmlns="" class="SimplePara">SSR (total no. of markers<sup xmlns="http://www.w3.org/1999/xhtml">1</sup>â=â464)</p></th>
<th> </th>
<th> </th>
</tr>
<tr class="header">
<th><p xmlns="" class="SimplePara">Accession</p></th>
<th><p xmlns="" class="SimplePara">No. of ind.</p></th>
<th><p xmlns="" class="SimplePara">No. markers</p></th>
<th><p xmlns="" class="SimplePara">A<sub xmlns="http://www.w3.org/1999/xhtml">A</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></th>
<th><p xmlns="" class="SimplePara">A<sub xmlns="http://www.w3.org/1999/xhtml">I</sub><sup xmlns="http://www.w3.org/1999/xhtml">3</sup></p></th>
<th><p xmlns="" class="SimplePara">PWD<sup xmlns="http://www.w3.org/1999/xhtml">4</sup></p></th>
<th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">I</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">5</em>
                              </sup></p></th>
<th><p xmlns="" class="SimplePara">% polymorphic loci</p></th>
<th><p xmlns="" class="SimplePara">No. markers</p></th>
<th><p xmlns="" class="SimplePara">A<sub xmlns="http://www.w3.org/1999/xhtml">A</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></th>
<th><p xmlns="" class="SimplePara">A<sub xmlns="http://www.w3.org/1999/xhtml">I</sub><sup xmlns="http://www.w3.org/1999/xhtml">3</sup></p></th>
<th><p xmlns="" class="SimplePara">PWD<sup xmlns="http://www.w3.org/1999/xhtml">4</sup></p></th>
<th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">I</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">5</em>
                              </sup></p></th>
<th><p xmlns="" class="SimplePara">% polymorphic loci</p></th>
</tr>
</thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB10830</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">213</p></td>
<td><p xmlns="" class="SimplePara">200.5</p></td>
<td><p xmlns="" class="SimplePara">55.8</p></td>
<td><p xmlns="" class="SimplePara">61.0</p></td>
<td><p xmlns="" class="SimplePara">0.170</p></td>
<td><p xmlns="" class="SimplePara">38.6</p></td>
<td><p xmlns="" class="SimplePara">124</p></td>
<td><p xmlns="" class="SimplePara">116.7</p></td>
<td><p xmlns="" class="SimplePara">28.8</p></td>
<td><p xmlns="" class="SimplePara">30.7</p></td>
<td><p xmlns="" class="SimplePara">0.114</p></td>
<td><p xmlns="" class="SimplePara">26.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB10831</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">221</p></td>
<td><p xmlns="" class="SimplePara">207.3</p></td>
<td><p xmlns="" class="SimplePara">50.1</p></td>
<td><p xmlns="" class="SimplePara">56.2</p></td>
<td><p xmlns="" class="SimplePara">0.161</p></td>
<td><p xmlns="" class="SimplePara">40.5</p></td>
<td><p xmlns="" class="SimplePara">148</p></td>
<td><p xmlns="" class="SimplePara">137.9</p></td>
<td><p xmlns="" class="SimplePara">28.4</p></td>
<td><p xmlns="" class="SimplePara">32.5</p></td>
<td><p xmlns="" class="SimplePara">0.126</p></td>
<td><p xmlns="" class="SimplePara">31.3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB15461</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">323</p></td>
<td><p xmlns="" class="SimplePara">302.2</p></td>
<td><p xmlns="" class="SimplePara">73.1</p></td>
<td><p xmlns="" class="SimplePara">85.7</p></td>
<td><p xmlns="" class="SimplePara">0.244</p></td>
<td><p xmlns="" class="SimplePara">59.3</p></td>
<td><p xmlns="" class="SimplePara">188</p></td>
<td><p xmlns="" class="SimplePara">171.5</p></td>
<td><p xmlns="" class="SimplePara">33.3</p></td>
<td><p xmlns="" class="SimplePara">44.0</p></td>
<td><p xmlns="" class="SimplePara">0.155</p></td>
<td><p xmlns="" class="SimplePara">40.5</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB16650</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">281</p></td>
<td><p xmlns="" class="SimplePara">264.9</p></td>
<td><p xmlns="" class="SimplePara">60.6</p></td>
<td><p xmlns="" class="SimplePara">71.8</p></td>
<td><p xmlns="" class="SimplePara">0.206</p></td>
<td><p xmlns="" class="SimplePara">52.0</p></td>
<td><p xmlns="" class="SimplePara">142</p></td>
<td><p xmlns="" class="SimplePara">132.9</p></td>
<td><p xmlns="" class="SimplePara">31.3</p></td>
<td><p xmlns="" class="SimplePara">36.2</p></td>
<td><p xmlns="" class="SimplePara">0.127</p></td>
<td><p xmlns="" class="SimplePara">30.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1672</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">352</p></td>
<td><p xmlns="" class="SimplePara">328.7</p></td>
<td><p xmlns="" class="SimplePara">84.8</p></td>
<td><p xmlns="" class="SimplePara">100.2</p></td>
<td><p xmlns="" class="SimplePara">0.280</p></td>
<td><p xmlns="" class="SimplePara">64.9</p></td>
<td><p xmlns="" class="SimplePara">162</p></td>
<td><p xmlns="" class="SimplePara">149.7</p></td>
<td><p xmlns="" class="SimplePara">31.7</p></td>
<td><p xmlns="" class="SimplePara">40.1</p></td>
<td><p xmlns="" class="SimplePara">0.137</p></td>
<td><p xmlns="" class="SimplePara">34.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1675</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">297</p></td>
<td><p xmlns="" class="SimplePara">277.8</p></td>
<td><p xmlns="" class="SimplePara">78.4</p></td>
<td><p xmlns="" class="SimplePara">87.2</p></td>
<td><p xmlns="" class="SimplePara">0.244</p></td>
<td><p xmlns="" class="SimplePara">54.8</p></td>
<td><p xmlns="" class="SimplePara">119</p></td>
<td><p xmlns="" class="SimplePara">110.1</p></td>
<td><p xmlns="" class="SimplePara">30.0</p></td>
<td><p xmlns="" class="SimplePara">32.5</p></td>
<td><p xmlns="" class="SimplePara">0.111</p></td>
<td><p xmlns="" class="SimplePara">25.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4053</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">239</p></td>
<td><p xmlns="" class="SimplePara">226.9</p></td>
<td><p xmlns="" class="SimplePara">58.8</p></td>
<td><p xmlns="" class="SimplePara">65.3</p></td>
<td><p xmlns="" class="SimplePara">0.185</p></td>
<td><p xmlns="" class="SimplePara">44.7</p></td>
<td><p xmlns="" class="SimplePara">145</p></td>
<td><p xmlns="" class="SimplePara">135.2</p></td>
<td><p xmlns="" class="SimplePara">30.7</p></td>
<td><p xmlns="" class="SimplePara">36.2</p></td>
<td><p xmlns="" class="SimplePara">0.135</p></td>
<td><p xmlns="" class="SimplePara">31.3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4548</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">236</p></td>
<td><p xmlns="" class="SimplePara">219.2</p></td>
<td><p xmlns="" class="SimplePara">54.8</p></td>
<td><p xmlns="" class="SimplePara">61.3</p></td>
<td><p xmlns="" class="SimplePara">0.177</p></td>
<td><p xmlns="" class="SimplePara">43.3</p></td>
<td><p xmlns="" class="SimplePara">154</p></td>
<td><p xmlns="" class="SimplePara">139.4</p></td>
<td><p xmlns="" class="SimplePara">30.1</p></td>
<td><p xmlns="" class="SimplePara">35.1</p></td>
<td><p xmlns="" class="SimplePara">0.137</p></td>
<td><p xmlns="" class="SimplePara">33.2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB132</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">313</p></td>
<td><p xmlns="" class="SimplePara">293.0</p></td>
<td><p xmlns="" class="SimplePara">75.6</p></td>
<td><p xmlns="" class="SimplePara">84.7</p></td>
<td><p xmlns="" class="SimplePara">0.242</p></td>
<td><p xmlns="" class="SimplePara">57.2</p></td>
<td><p xmlns="" class="SimplePara">186</p></td>
<td><p xmlns="" class="SimplePara">167.5</p></td>
<td><p xmlns="" class="SimplePara">31.1</p></td>
<td><p xmlns="" class="SimplePara">39.1</p></td>
<td><p xmlns="" class="SimplePara">0.144</p></td>
<td><p xmlns="" class="SimplePara">39.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB14394</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">276</p></td>
<td><p xmlns="" class="SimplePara">255.6</p></td>
<td><p xmlns="" class="SimplePara">59.3</p></td>
<td><p xmlns="" class="SimplePara">71.2</p></td>
<td><p xmlns="" class="SimplePara">0.204</p></td>
<td><p xmlns="" class="SimplePara">50.8</p></td>
<td><p xmlns="" class="SimplePara">175</p></td>
<td><p xmlns="" class="SimplePara">162.2</p></td>
<td><p xmlns="" class="SimplePara">32.7</p></td>
<td><p xmlns="" class="SimplePara">42.6</p></td>
<td><p xmlns="" class="SimplePara">0.148</p></td>
<td><p xmlns="" class="SimplePara">37.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB14404</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">295</p></td>
<td><p xmlns="" class="SimplePara">266.0</p></td>
<td><p xmlns="" class="SimplePara">65.7</p></td>
<td><p xmlns="" class="SimplePara">74.5</p></td>
<td><p xmlns="" class="SimplePara">0.214</p></td>
<td><p xmlns="" class="SimplePara">54.6</p></td>
<td><p xmlns="" class="SimplePara">176</p></td>
<td><p xmlns="" class="SimplePara">158.4</p></td>
<td><p xmlns="" class="SimplePara">33.6</p></td>
<td><p xmlns="" class="SimplePara">40.5</p></td>
<td><p xmlns="" class="SimplePara">0.144</p></td>
<td><p xmlns="" class="SimplePara">37.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB14417</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">213</p></td>
<td><p xmlns="" class="SimplePara">201.4</p></td>
<td><p xmlns="" class="SimplePara">60.7</p></td>
<td><p xmlns="" class="SimplePara">59.4</p></td>
<td><p xmlns="" class="SimplePara">0.166</p></td>
<td><p xmlns="" class="SimplePara">38.8</p></td>
<td><p xmlns="" class="SimplePara">121</p></td>
<td><p xmlns="" class="SimplePara">115.2</p></td>
<td><p xmlns="" class="SimplePara">28.4</p></td>
<td><p xmlns="" class="SimplePara">32.5</p></td>
<td><p xmlns="" class="SimplePara">0.115</p></td>
<td><p xmlns="" class="SimplePara">25.9</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB747</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">310</p></td>
<td><p xmlns="" class="SimplePara">285.9</p></td>
<td><p xmlns="" class="SimplePara">75.2</p></td>
<td><p xmlns="" class="SimplePara">81.5</p></td>
<td><p xmlns="" class="SimplePara">0.235</p></td>
<td><p xmlns="" class="SimplePara">56.8</p></td>
<td><p xmlns="" class="SimplePara">144</p></td>
<td><p xmlns="" class="SimplePara">131.4</p></td>
<td><p xmlns="" class="SimplePara">30.3</p></td>
<td><p xmlns="" class="SimplePara">34.7</p></td>
<td><p xmlns="" class="SimplePara">0.126</p></td>
<td><p xmlns="" class="SimplePara">30.8</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1095</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">330</p></td>
<td><p xmlns="" class="SimplePara">300.6</p></td>
<td><p xmlns="" class="SimplePara">72.9</p></td>
<td><p xmlns="" class="SimplePara">85.0</p></td>
<td><p xmlns="" class="SimplePara">0.245</p></td>
<td><p xmlns="" class="SimplePara">61.2</p></td>
<td><p xmlns="" class="SimplePara">192</p></td>
<td><p xmlns="" class="SimplePara">170.2</p></td>
<td><p xmlns="" class="SimplePara">34.2</p></td>
<td><p xmlns="" class="SimplePara">43.1</p></td>
<td><p xmlns="" class="SimplePara">0.153</p></td>
<td><p xmlns="" class="SimplePara">41.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1111</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">308</p></td>
<td><p xmlns="" class="SimplePara">289.2</p></td>
<td><p xmlns="" class="SimplePara">64.3</p></td>
<td><p xmlns="" class="SimplePara">78.1</p></td>
<td><p xmlns="" class="SimplePara">0.229</p></td>
<td><p xmlns="" class="SimplePara">56.7</p></td>
<td><p xmlns="" class="SimplePara">194</p></td>
<td><p xmlns="" class="SimplePara">172.7</p></td>
<td><p xmlns="" class="SimplePara">33.8</p></td>
<td><p xmlns="" class="SimplePara">44.3</p></td>
<td><p xmlns="" class="SimplePara">0.156</p></td>
<td><p xmlns="" class="SimplePara">41.2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1119</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">226</p></td>
<td><p xmlns="" class="SimplePara">210.1</p></td>
<td><p xmlns="" class="SimplePara">52.8</p></td>
<td><p xmlns="" class="SimplePara">58.6</p></td>
<td><p xmlns="" class="SimplePara">0.170</p></td>
<td><p xmlns="" class="SimplePara">41.5</p></td>
<td><p xmlns="" class="SimplePara">189</p></td>
<td><p xmlns="" class="SimplePara">170.9</p></td>
<td><p xmlns="" class="SimplePara">31.9</p></td>
<td><p xmlns="" class="SimplePara">40.5</p></td>
<td><p xmlns="" class="SimplePara">0.149</p></td>
<td><p xmlns="" class="SimplePara">40.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB2791</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">315</p></td>
<td><p xmlns="" class="SimplePara">294.5</p></td>
<td><p xmlns="" class="SimplePara">67.7</p></td>
<td><p xmlns="" class="SimplePara">80.7</p></td>
<td><p xmlns="" class="SimplePara">0.238</p></td>
<td><p xmlns="" class="SimplePara">58.2</p></td>
<td><p xmlns="" class="SimplePara">186</p></td>
<td><p xmlns="" class="SimplePara">169.8</p></td>
<td><p xmlns="" class="SimplePara">33.3</p></td>
<td><p xmlns="" class="SimplePara">42.5</p></td>
<td><p xmlns="" class="SimplePara">0.152</p></td>
<td><p xmlns="" class="SimplePara">40.1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4066</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">259</p></td>
<td><p xmlns="" class="SimplePara">239.8</p></td>
<td><p xmlns="" class="SimplePara">53.6</p></td>
<td><p xmlns="" class="SimplePara">65.4</p></td>
<td><p xmlns="" class="SimplePara">0.188</p></td>
<td><p xmlns="" class="SimplePara">47.5</p></td>
<td><p xmlns="" class="SimplePara">180</p></td>
<td><p xmlns="" class="SimplePara">164.5</p></td>
<td><p xmlns="" class="SimplePara">32.7</p></td>
<td><p xmlns="" class="SimplePara">38.6</p></td>
<td><p xmlns="" class="SimplePara">0.149</p></td>
<td><p xmlns="" class="SimplePara">38.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4140</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">263</p></td>
<td><p xmlns="" class="SimplePara">248.3</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">70.5</p></td>
<td><p xmlns="" class="SimplePara">0.201</p></td>
<td><p xmlns="" class="SimplePara">49.0</p></td>
<td><p xmlns="" class="SimplePara">193</p></td>
<td><p xmlns="" class="SimplePara">176.2</p></td>
<td><p xmlns="" class="SimplePara">33.4</p></td>
<td><p xmlns="" class="SimplePara">43.9</p></td>
<td><p xmlns="" class="SimplePara">0.160</p></td>
<td><p xmlns="" class="SimplePara">40.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4141</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">260</p></td>
<td><p xmlns="" class="SimplePara">246.6</p></td>
<td><p xmlns="" class="SimplePara">66.5</p></td>
<td><p xmlns="" class="SimplePara">71.6</p></td>
<td><p xmlns="" class="SimplePara">0.206</p></td>
<td><p xmlns="" class="SimplePara">48.0</p></td>
<td><p xmlns="" class="SimplePara">117</p></td>
<td><p xmlns="" class="SimplePara">111.2</p></td>
<td><p xmlns="" class="SimplePara">31.3</p></td>
<td><p xmlns="" class="SimplePara">34.6</p></td>
<td><p xmlns="" class="SimplePara">0.118</p></td>
<td><p xmlns="" class="SimplePara">25.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB7592</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">244</p></td>
<td><p xmlns="" class="SimplePara">239.2</p></td>
<td><p xmlns="" class="SimplePara">64.5</p></td>
<td><p xmlns="" class="SimplePara">68.6</p></td>
<td><p xmlns="" class="SimplePara">0.192</p></td>
<td><p xmlns="" class="SimplePara">44.7</p></td>
<td><p xmlns="" class="SimplePara">182</p></td>
<td><p xmlns="" class="SimplePara">168.4</p></td>
<td><p xmlns="" class="SimplePara">35.2</p></td>
<td><p xmlns="" class="SimplePara">44.9</p></td>
<td><p xmlns="" class="SimplePara">0.155</p></td>
<td><p xmlns="" class="SimplePara">37.1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB10785</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">202</p></td>
<td><p xmlns="" class="SimplePara">192.0</p></td>
<td><p xmlns="" class="SimplePara">67.0</p></td>
<td><p xmlns="" class="SimplePara">59.9</p></td>
<td><p xmlns="" class="SimplePara">0.165</p></td>
<td><p xmlns="" class="SimplePara">36.6</p></td>
<td><p xmlns="" class="SimplePara">95</p></td>
<td><p xmlns="" class="SimplePara">86.8</p></td>
<td><p xmlns="" class="SimplePara">32.7</p></td>
<td><p xmlns="" class="SimplePara">28.9</p></td>
<td><p xmlns="" class="SimplePara">0.092</p></td>
<td><p xmlns="" class="SimplePara">19.8</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB17194</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">286</p></td>
<td><p xmlns="" class="SimplePara">260.7</p></td>
<td><p xmlns="" class="SimplePara">62.1</p></td>
<td><p xmlns="" class="SimplePara">72.7</p></td>
<td><p xmlns="" class="SimplePara">0.213</p></td>
<td><p xmlns="" class="SimplePara">52.7</p></td>
<td><p xmlns="" class="SimplePara">166</p></td>
<td><p xmlns="" class="SimplePara">149.5</p></td>
<td><p xmlns="" class="SimplePara">31.4</p></td>
<td><p xmlns="" class="SimplePara">37.2</p></td>
<td><p xmlns="" class="SimplePara">0.134</p></td>
<td><p xmlns="" class="SimplePara">35.8</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB17198</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">336</p></td>
<td><p xmlns="" class="SimplePara">313.7</p></td>
<td><p xmlns="" class="SimplePara">70.1</p></td>
<td><p xmlns="" class="SimplePara">85.7</p></td>
<td><p xmlns="" class="SimplePara">0.245</p></td>
<td><p xmlns="" class="SimplePara">61.9</p></td>
<td><p xmlns="" class="SimplePara">179</p></td>
<td><p xmlns="" class="SimplePara">166.4</p></td>
<td><p xmlns="" class="SimplePara">32.4</p></td>
<td><p xmlns="" class="SimplePara">42.5</p></td>
<td><p xmlns="" class="SimplePara">0.150</p></td>
<td><p xmlns="" class="SimplePara">38.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB2169</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">290</p></td>
<td><p xmlns="" class="SimplePara">269.0</p></td>
<td><p xmlns="" class="SimplePara">62.9</p></td>
<td><p xmlns="" class="SimplePara">76.5</p></td>
<td><p xmlns="" class="SimplePara">0.219</p></td>
<td><p xmlns="" class="SimplePara">53.8</p></td>
<td><p xmlns="" class="SimplePara">167</p></td>
<td><p xmlns="" class="SimplePara">153.4</p></td>
<td><p xmlns="" class="SimplePara">30.3</p></td>
<td><p xmlns="" class="SimplePara">38.8</p></td>
<td><p xmlns="" class="SimplePara">0.140</p></td>
<td><p xmlns="" class="SimplePara">35.8</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB2180</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">304</p></td>
<td><p xmlns="" class="SimplePara">284.6</p></td>
<td><p xmlns="" class="SimplePara">67.7</p></td>
<td><p xmlns="" class="SimplePara">77.4</p></td>
<td><p xmlns="" class="SimplePara">0.226</p></td>
<td><p xmlns="" class="SimplePara">56.1</p></td>
<td><p xmlns="" class="SimplePara">185</p></td>
<td><p xmlns="" class="SimplePara">169.2</p></td>
<td><p xmlns="" class="SimplePara">31.2</p></td>
<td><p xmlns="" class="SimplePara">39.5</p></td>
<td><p xmlns="" class="SimplePara">0.146</p></td>
<td><p xmlns="" class="SimplePara">39.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB2918</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">332</p></td>
<td><p xmlns="" class="SimplePara">305.3</p></td>
<td><p xmlns="" class="SimplePara">73.4</p></td>
<td><p xmlns="" class="SimplePara">88.2</p></td>
<td><p xmlns="" class="SimplePara">0.252</p></td>
<td><p xmlns="" class="SimplePara">61.7</p></td>
<td><p xmlns="" class="SimplePara">164</p></td>
<td><p xmlns="" class="SimplePara">150.2</p></td>
<td><p xmlns="" class="SimplePara">32.1</p></td>
<td><p xmlns="" class="SimplePara">39.0</p></td>
<td><p xmlns="" class="SimplePara">0.140</p></td>
<td><p xmlns="" class="SimplePara">35.3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4226</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">236</p></td>
<td><p xmlns="" class="SimplePara">228.4</p></td>
<td><p xmlns="" class="SimplePara">64.4</p></td>
<td><p xmlns="" class="SimplePara">67.9</p></td>
<td><p xmlns="" class="SimplePara">0.191</p></td>
<td><p xmlns="" class="SimplePara">43.3</p></td>
<td><p xmlns="" class="SimplePara">158</p></td>
<td><p xmlns="" class="SimplePara">151.6</p></td>
<td><p xmlns="" class="SimplePara">34.9</p></td>
<td><p xmlns="" class="SimplePara">42.2</p></td>
<td><p xmlns="" class="SimplePara">0.142</p></td>
<td><p xmlns="" class="SimplePara">34.1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4231</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">230</p></td>
<td><p xmlns="" class="SimplePara">216.4</p></td>
<td><p xmlns="" class="SimplePara">56.3</p></td>
<td><p xmlns="" class="SimplePara">63.0</p></td>
<td><p xmlns="" class="SimplePara">0.183</p></td>
<td><p xmlns="" class="SimplePara">42.2</p></td>
<td><p xmlns="" class="SimplePara">156</p></td>
<td><p xmlns="" class="SimplePara">143.6</p></td>
<td><p xmlns="" class="SimplePara">31.3</p></td>
<td><p xmlns="" class="SimplePara">38.4</p></td>
<td><p xmlns="" class="SimplePara">0.137</p></td>
<td><p xmlns="" class="SimplePara">33.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB7548</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">272</p></td>
<td><p xmlns="" class="SimplePara">260.1</p></td>
<td><p xmlns="" class="SimplePara">61.4</p></td>
<td><p xmlns="" class="SimplePara">70.0</p></td>
<td><p xmlns="" class="SimplePara">0.201</p></td>
<td><p xmlns="" class="SimplePara">49.7</p></td>
<td><p xmlns="" class="SimplePara">150</p></td>
<td><p xmlns="" class="SimplePara">140.8</p></td>
<td><p xmlns="" class="SimplePara">29.8</p></td>
<td><p xmlns="" class="SimplePara">35.4</p></td>
<td><p xmlns="" class="SimplePara">0.131</p></td>
<td><p xmlns="" class="SimplePara">31.9</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB722</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">256</p></td>
<td><p xmlns="" class="SimplePara">241.9</p></td>
<td><p xmlns="" class="SimplePara">77.4</p></td>
<td><p xmlns="" class="SimplePara">71.7</p></td>
<td><p xmlns="" class="SimplePara">0.202</p></td>
<td><p xmlns="" class="SimplePara">45.8</p></td>
<td><p xmlns="" class="SimplePara">115</p></td>
<td><p xmlns="" class="SimplePara">108.8</p></td>
<td><p xmlns="" class="SimplePara">29.5</p></td>
<td><p xmlns="" class="SimplePara">32.7</p></td>
<td><p xmlns="" class="SimplePara">0.109</p></td>
<td><p xmlns="" class="SimplePara">24.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB728</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">265</p></td>
<td><p xmlns="" class="SimplePara">246.3</p></td>
<td><p xmlns="" class="SimplePara">56.7</p></td>
<td><p xmlns="" class="SimplePara">65.5</p></td>
<td><p xmlns="" class="SimplePara">0.196</p></td>
<td><p xmlns="" class="SimplePara">49.5</p></td>
<td><p xmlns="" class="SimplePara">182</p></td>
<td><p xmlns="" class="SimplePara">171.2</p></td>
<td><p xmlns="" class="SimplePara">33.9</p></td>
<td><p xmlns="" class="SimplePara">43.4</p></td>
<td><p xmlns="" class="SimplePara">0.155</p></td>
<td><p xmlns="" class="SimplePara">39.2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB11428</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">231</p></td>
<td><p xmlns="" class="SimplePara">226.2</p></td>
<td><p xmlns="" class="SimplePara">58.9</p></td>
<td><p xmlns="" class="SimplePara">64.4</p></td>
<td><p xmlns="" class="SimplePara">0.180</p></td>
<td><p xmlns="" class="SimplePara">42.0</p></td>
<td><p xmlns="" class="SimplePara">139</p></td>
<td><p xmlns="" class="SimplePara">138</p></td>
<td><p xmlns="" class="SimplePara">34.2</p></td>
<td><p xmlns="" class="SimplePara">39.8</p></td>
<td><p xmlns="" class="SimplePara">0.135</p></td>
<td><p xmlns="" class="SimplePara">29.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB14224</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">293</p></td>
<td><p xmlns="" class="SimplePara">273.6</p></td>
<td><p xmlns="" class="SimplePara">65.7</p></td>
<td><p xmlns="" class="SimplePara">79.2</p></td>
<td><p xmlns="" class="SimplePara">0.224</p></td>
<td><p xmlns="" class="SimplePara">54.0</p></td>
<td><p xmlns="" class="SimplePara">175</p></td>
<td><p xmlns="" class="SimplePara">158.4</p></td>
<td><p xmlns="" class="SimplePara">33.4</p></td>
<td><p xmlns="" class="SimplePara">42.2</p></td>
<td><p xmlns="" class="SimplePara">0.146</p></td>
<td><p xmlns="" class="SimplePara">36.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB731</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">311</p></td>
<td><p xmlns="" class="SimplePara">288.5</p></td>
<td><p xmlns="" class="SimplePara">79.8</p></td>
<td><p xmlns="" class="SimplePara">84.6</p></td>
<td><p xmlns="" class="SimplePara">0.241</p></td>
<td><p xmlns="" class="SimplePara">57.0</p></td>
<td><p xmlns="" class="SimplePara">182</p></td>
<td><p xmlns="" class="SimplePara">163.2</p></td>
<td><p xmlns="" class="SimplePara">31.9</p></td>
<td><p xmlns="" class="SimplePara">41.1</p></td>
<td><p xmlns="" class="SimplePara">0.150</p></td>
<td><p xmlns="" class="SimplePara">39.2</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB16975</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">258</p></td>
<td><p xmlns="" class="SimplePara">237.0</p></td>
<td><p xmlns="" class="SimplePara">56.2</p></td>
<td><p xmlns="" class="SimplePara">66.4</p></td>
<td><p xmlns="" class="SimplePara">0.189</p></td>
<td><p xmlns="" class="SimplePara">47.5</p></td>
<td><p xmlns="" class="SimplePara">176</p></td>
<td><p xmlns="" class="SimplePara">161.8</p></td>
<td><p xmlns="" class="SimplePara">33.5</p></td>
<td><p xmlns="" class="SimplePara">42.2</p></td>
<td><p xmlns="" class="SimplePara">0.150</p></td>
<td><p xmlns="" class="SimplePara">37.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB16981</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">245</p></td>
<td><p xmlns="" class="SimplePara">245.0</p></td>
<td><p xmlns="" class="SimplePara">59.2</p></td>
<td><p xmlns="" class="SimplePara">69.6</p></td>
<td><p xmlns="" class="SimplePara">0.193</p></td>
<td><p xmlns="" class="SimplePara">44.7</p></td>
<td><p xmlns="" class="SimplePara">153</p></td>
<td><p xmlns="" class="SimplePara">149.8</p></td>
<td><p xmlns="" class="SimplePara">33.6</p></td>
<td><p xmlns="" class="SimplePara">41.7</p></td>
<td><p xmlns="" class="SimplePara">0.138</p></td>
<td><p xmlns="" class="SimplePara">32.1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1306</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">298</p></td>
<td><p xmlns="" class="SimplePara">280.5</p></td>
<td><p xmlns="" class="SimplePara">64.0</p></td>
<td><p xmlns="" class="SimplePara">76.0</p></td>
<td><p xmlns="" class="SimplePara">0.224</p></td>
<td><p xmlns="" class="SimplePara">55.0</p></td>
<td><p xmlns="" class="SimplePara">184</p></td>
<td><p xmlns="" class="SimplePara">173.6</p></td>
<td><p xmlns="" class="SimplePara">32.9</p></td>
<td><p xmlns="" class="SimplePara">41.4</p></td>
<td><p xmlns="" class="SimplePara">0.152</p></td>
<td><p xmlns="" class="SimplePara">39.7</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1327</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">284</p></td>
<td><p xmlns="" class="SimplePara">267.1</p></td>
<td><p xmlns="" class="SimplePara">70.6</p></td>
<td><p xmlns="" class="SimplePara">79.8</p></td>
<td><p xmlns="" class="SimplePara">0.225</p></td>
<td><p xmlns="" class="SimplePara">52.2</p></td>
<td><p xmlns="" class="SimplePara">162</p></td>
<td><p xmlns="" class="SimplePara">151.4</p></td>
<td><p xmlns="" class="SimplePara">33.6</p></td>
<td><p xmlns="" class="SimplePara">41.2</p></td>
<td><p xmlns="" class="SimplePara">0.144</p></td>
<td><p xmlns="" class="SimplePara">34.9</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1331</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">223</p></td>
<td><p xmlns="" class="SimplePara">208.5</p></td>
<td><p xmlns="" class="SimplePara">57.1</p></td>
<td><p xmlns="" class="SimplePara">60.7</p></td>
<td><p xmlns="" class="SimplePara">0.172</p></td>
<td><p xmlns="" class="SimplePara">40.3</p></td>
<td><p xmlns="" class="SimplePara">170</p></td>
<td><p xmlns="" class="SimplePara">158.4</p></td>
<td><p xmlns="" class="SimplePara">32.8</p></td>
<td><p xmlns="" class="SimplePara">42.1</p></td>
<td><p xmlns="" class="SimplePara">0.144</p></td>
<td><p xmlns="" class="SimplePara">36.4</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB1537</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">296</p></td>
<td><p xmlns="" class="SimplePara">290.1</p></td>
<td><p xmlns="" class="SimplePara">68.2</p></td>
<td><p xmlns="" class="SimplePara">82.4</p></td>
<td><p xmlns="" class="SimplePara">0.231</p></td>
<td><p xmlns="" class="SimplePara">54.6</p></td>
<td><p xmlns="" class="SimplePara">139</p></td>
<td><p xmlns="" class="SimplePara">131.7</p></td>
<td><p xmlns="" class="SimplePara">30.5</p></td>
<td><p xmlns="" class="SimplePara">36.1</p></td>
<td><p xmlns="" class="SimplePara">0.125</p></td>
<td><p xmlns="" class="SimplePara">28.9</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB2530</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">235</p></td>
<td><p xmlns="" class="SimplePara">218.0</p></td>
<td><p xmlns="" class="SimplePara">57.9</p></td>
<td><p xmlns="" class="SimplePara">61.8</p></td>
<td><p xmlns="" class="SimplePara">0.177</p></td>
<td><p xmlns="" class="SimplePara">42.8</p></td>
<td><p xmlns="" class="SimplePara">165</p></td>
<td><p xmlns="" class="SimplePara">152.8</p></td>
<td><p xmlns="" class="SimplePara">31.9</p></td>
<td><p xmlns="" class="SimplePara">39.6</p></td>
<td><p xmlns="" class="SimplePara">0.141</p></td>
<td><p xmlns="" class="SimplePara">35.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NGB4349</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">267</p></td>
<td><p xmlns="" class="SimplePara">244.0</p></td>
<td><p xmlns="" class="SimplePara">62.7</p></td>
<td><p xmlns="" class="SimplePara">70.7</p></td>
<td><p xmlns="" class="SimplePara">0.203</p></td>
<td><p xmlns="" class="SimplePara">49.3</p></td>
<td><p xmlns="" class="SimplePara">171</p></td>
<td><p xmlns="" class="SimplePara">153.4</p></td>
<td><p xmlns="" class="SimplePara">32.4</p></td>
<td><p xmlns="" class="SimplePara">40.6</p></td>
<td><p xmlns="" class="SimplePara">0.141</p></td>
<td><p xmlns="" class="SimplePara">36.9</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">PL381926</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">240</p></td>
<td><p xmlns="" class="SimplePara">222.2</p></td>
<td><p xmlns="" class="SimplePara">67.4</p></td>
<td><p xmlns="" class="SimplePara">66.5</p></td>
<td><p xmlns="" class="SimplePara">0.187</p></td>
<td><p xmlns="" class="SimplePara">43.9</p></td>
<td><p xmlns="" class="SimplePara">131</p></td>
<td><p xmlns="" class="SimplePara">122.3</p></td>
<td><p xmlns="" class="SimplePara">31.1</p></td>
<td><p xmlns="" class="SimplePara">34.6</p></td>
<td><p xmlns="" class="SimplePara">0.125</p></td>
<td><p xmlns="" class="SimplePara">28.0</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">PL406317</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">243</p></td>
<td><p xmlns="" class="SimplePara">226.8</p></td>
<td><p xmlns="" class="SimplePara">56.5</p></td>
<td><p xmlns="" class="SimplePara">64.1</p></td>
<td><p xmlns="" class="SimplePara">0.183</p></td>
<td><p xmlns="" class="SimplePara">44.5</p></td>
<td><p xmlns="" class="SimplePara">165</p></td>
<td><p xmlns="" class="SimplePara">151.7</p></td>
<td><p xmlns="" class="SimplePara">31.8</p></td>
<td><p xmlns="" class="SimplePara">38.5</p></td>
<td><p xmlns="" class="SimplePara">0.145</p></td>
<td><p xmlns="" class="SimplePara">35.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">IHAR151908</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">259</p></td>
<td><p xmlns="" class="SimplePara">249.0</p></td>
<td><p xmlns="" class="SimplePara">60.3</p></td>
<td><p xmlns="" class="SimplePara">66.4</p></td>
<td><p xmlns="" class="SimplePara">0.192</p></td>
<td><p xmlns="" class="SimplePara">48.0</p></td>
<td><p xmlns="" class="SimplePara">150</p></td>
<td><p xmlns="" class="SimplePara">137.4</p></td>
<td><p xmlns="" class="SimplePara">31.1</p></td>
<td><p xmlns="" class="SimplePara">34.8</p></td>
<td><p xmlns="" class="SimplePara">0.126</p></td>
<td><p xmlns="" class="SimplePara">30.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">PL210426</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">245</p></td>
<td><p xmlns="" class="SimplePara">237.0</p></td>
<td><p xmlns="" class="SimplePara">65.5</p></td>
<td><p xmlns="" class="SimplePara">69.4</p></td>
<td><p xmlns="" class="SimplePara">0.194</p></td>
<td><p xmlns="" class="SimplePara">44.8</p></td>
<td><p xmlns="" class="SimplePara">146</p></td>
<td><p xmlns="" class="SimplePara">138.9</p></td>
<td><p xmlns="" class="SimplePara">31.8</p></td>
<td><p xmlns="" class="SimplePara">38.3</p></td>
<td><p xmlns="" class="SimplePara">0.131</p></td>
<td><p xmlns="" class="SimplePara">30.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">PL325461</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">233</p></td>
<td><p xmlns="" class="SimplePara">223.4</p></td>
<td><p xmlns="" class="SimplePara">55.5</p></td>
<td><p xmlns="" class="SimplePara">61.2</p></td>
<td><p xmlns="" class="SimplePara">0.175</p></td>
<td><p xmlns="" class="SimplePara">43.2</p></td>
<td><p xmlns="" class="SimplePara">170</p></td>
<td><p xmlns="" class="SimplePara">157.8</p></td>
<td><p xmlns="" class="SimplePara">30.8</p></td>
<td><p xmlns="" class="SimplePara">39.8</p></td>
<td><p xmlns="" class="SimplePara">0.138</p></td>
<td><p xmlns="" class="SimplePara">34.3</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">PL204480</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">195</p></td>
<td><p xmlns="" class="SimplePara">182.4</p></td>
<td><p xmlns="" class="SimplePara">47.5</p></td>
<td><p xmlns="" class="SimplePara">53.1</p></td>
<td><p xmlns="" class="SimplePara">0.151</p></td>
<td><p xmlns="" class="SimplePara">35.8</p></td>
<td><p xmlns="" class="SimplePara">158</p></td>
<td><p xmlns="" class="SimplePara">144.3</p></td>
<td><p xmlns="" class="SimplePara">31.6</p></td>
<td><p xmlns="" class="SimplePara">37.6</p></td>
<td><p xmlns="" class="SimplePara">0.133</p></td>
<td><p xmlns="" class="SimplePara">33.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">14G2400116</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">234</p></td>
<td><p xmlns="" class="SimplePara">220.2</p></td>
<td><p xmlns="" class="SimplePara">57.1</p></td>
<td><p xmlns="" class="SimplePara">65.8</p></td>
<td><p xmlns="" class="SimplePara">0.185</p></td>
<td><p xmlns="" class="SimplePara">42.8</p></td>
<td><p xmlns="" class="SimplePara">186</p></td>
<td><p xmlns="" class="SimplePara">170.5</p></td>
<td><p xmlns="" class="SimplePara">34.2</p></td>
<td><p xmlns="" class="SimplePara">44.0</p></td>
<td><p xmlns="" class="SimplePara">0.153</p></td>
<td><p xmlns="" class="SimplePara">39.9</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">RCAT040682</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">209</p></td>
<td><p xmlns="" class="SimplePara">191.5</p></td>
<td><p xmlns="" class="SimplePara">50.8</p></td>
<td><p xmlns="" class="SimplePara">55.9</p></td>
<td><p xmlns="" class="SimplePara">0.159</p></td>
<td><p xmlns="" class="SimplePara">38.6</p></td>
<td><p xmlns="" class="SimplePara">157</p></td>
<td><p xmlns="" class="SimplePara">143.5</p></td>
<td><p xmlns="" class="SimplePara">30.5</p></td>
<td><p xmlns="" class="SimplePara">39.1</p></td>
<td><p xmlns="" class="SimplePara">0.141</p></td>
<td><p xmlns="" class="SimplePara">33.8</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">1</sup>each SSR allele treated as a separate marker</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>corrected number of all markers in each accession</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">3</sup>mean number of all markers observed in each individual</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">4</sup>mean number of pairwise differences (Euclidean distances) between individuals in each accession</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">5</sup>Shannonâs diversity index</p>
                    </div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">The strength of correlation between diversity indices based on REMAP or SSR markers varied depending on the index. No correlation existed in the level of A<sub>I</sub>. PWD and <em xmlns="" class="EmphasisTypeItalic">I</em> correlated weakly at <em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.27 (<em xmlns="" class="EmphasisTypeItalic">Pâ=</em>â0.059<em xmlns="" class="EmphasisTypeItalic">)</em> and <em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.25 (<em xmlns="" class="EmphasisTypeItalic">Pâ=</em>â0.073), respectively. The number of markers per accession (A<sub>A</sub>) correlated moderately at <em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.37 (<em xmlns="" class="EmphasisTypeItalic">Pâ=</em>â0.0075) and the percentage of polymorphic loci strongly with <em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.44 (<em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.0012). Neiâs genetic distances between accessions based on REMAP and SSR data correlated strongly (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.67, <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.001) with each other.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">When studying levels of diversity between countries, vegetation zones, or different cultivar types, we found no significant differences in A<sub>A</sub> and PWD based on REMAP markers (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). On the other hand, statistically significant (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.05) differences in A<sub>A</sub> and PWD between different vegetation zones and in A<sub>A</sub> between different cultivar types were found with SSR markers (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). In the previous study with 96 accessions analyzed with SSR markers, we found significant differences (<em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.05) in levels of diversity in all groups [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. When the total number of markers was studied on an individual rather than accession level (A<sub>I</sub>), significant differences for each grouping and with both marker types were discovered (Table <span class="InternalRef"><a href="#Tab5">5</a></span>). However, these differences explained only a minor fraction of variation between individuals (1 to 5 %).<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">ANOVA tables indicating <em class="EmphasisTypeItalic">F</em>-values, significance levels <em class="EmphasisTypeItalic">P,</em> and R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup> for comparisons between different groups for their levels of REMAP and SSR diversity</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">REMAP</p></td>
<td> </td>
<td colspan="3"><p xmlns="" class="SimplePara">Total no. of markers (A<sub xmlns="http://www.w3.org/1999/xhtml">A)</sub></p></td>
<td colspan="3"><p xmlns="" class="SimplePara">No. of pairwise differences (PWD)</p></td>
<td colspan="3"><p xmlns="" class="SimplePara">Number of markers per individual</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âDiversity index</p></td>
<td><p xmlns="" class="SimplePara">df</p></td>
<td><p xmlns="" class="SimplePara">F</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td>
<td><p xmlns="" class="SimplePara">F</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td>
<td><p xmlns="" class="SimplePara">F</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âGrouping</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââAccession</p></td>
<td><p xmlns="" class="SimplePara">50</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">6.11</p></td>
<td><p xmlns="" class="SimplePara">&lt;0.001</p></td>
<td><p xmlns="" class="SimplePara">0.25</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââCountry</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">1.73</p></td>
<td><p xmlns="" class="SimplePara">0.147</p></td>
<td><p xmlns="" class="SimplePara">0.16</p></td>
<td><p xmlns="" class="SimplePara">1.49</p></td>
<td><p xmlns="" class="SimplePara">0.210</p></td>
<td><p xmlns="" class="SimplePara">0.14</p></td>
<td><p xmlns="" class="SimplePara">5.08</p></td>
<td><p xmlns="" class="SimplePara">&lt;0.001</p></td>
<td><p xmlns="" class="SimplePara">0.03</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââVegetation zone</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">0.74</p></td>
<td><p xmlns="" class="SimplePara">0.602</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
<td><p xmlns="" class="SimplePara">0.74</p></td>
<td><p xmlns="" class="SimplePara">0.600</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
<td><p xmlns="" class="SimplePara">5.08</p></td>
<td><p xmlns="" class="SimplePara">&lt;0.001</p></td>
<td><p xmlns="" class="SimplePara">0.04</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââCultivar type</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">2.03</p></td>
<td><p xmlns="" class="SimplePara">0.144</p></td>
<td><p xmlns="" class="SimplePara">0.09</p></td>
<td><p xmlns="" class="SimplePara">1.96</p></td>
<td><p xmlns="" class="SimplePara">0.153</p></td>
<td><p xmlns="" class="SimplePara">0.09</p></td>
<td><p xmlns="" class="SimplePara">3.98</p></td>
<td><p xmlns="" class="SimplePara">0.019</p></td>
<td><p xmlns="" class="SimplePara">0.01</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SSR</p></td>
<td> </td>
<td colspan="3"><p xmlns="" class="SimplePara">Total no. of markers (A<sub xmlns="http://www.w3.org/1999/xhtml">A)</sub></p></td>
<td colspan="3"><p xmlns="" class="SimplePara">No. of pairwise differences (PWD)</p></td>
<td colspan="3"><p xmlns="" class="SimplePara">Number of markers per individual</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âDiversity index</p></td>
<td><p xmlns="" class="SimplePara">df</p></td>
<td><p xmlns="" class="SimplePara">F</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td>
<td><p xmlns="" class="SimplePara">F</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td>
<td><p xmlns="" class="SimplePara">F</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âGrouping</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââAccession</p></td>
<td><p xmlns="" class="SimplePara">50</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">3.18</p></td>
<td><p xmlns="" class="SimplePara">&lt;0.001</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââCountry</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">1.15</p></td>
<td><p xmlns="" class="SimplePara">0.348</p></td>
<td><p xmlns="" class="SimplePara">0.11</p></td>
<td><p xmlns="" class="SimplePara">1.53</p></td>
<td><p xmlns="" class="SimplePara">0.200</p></td>
<td><p xmlns="" class="SimplePara">0.15</p></td>
<td><p xmlns="" class="SimplePara">5.28</p></td>
<td><p xmlns="" class="SimplePara">&lt;0.001</p></td>
<td><p xmlns="" class="SimplePara">0.03</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââVegetation zone</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">3.90</p></td>
<td><p xmlns="" class="SimplePara">0.008</p></td>
<td><p xmlns="" class="SimplePara">0.40</p></td>
<td><p xmlns="" class="SimplePara">3.49</p></td>
<td><p xmlns="" class="SimplePara">0.014</p></td>
<td><p xmlns="" class="SimplePara">0.38</p></td>
<td><p xmlns="" class="SimplePara">6.67</p></td>
<td><p xmlns="" class="SimplePara">&lt;0.001</p></td>
<td><p xmlns="" class="SimplePara">0.05</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">ââCultivar type</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">4.70</p></td>
<td><p xmlns="" class="SimplePara">0.015</p></td>
<td><p xmlns="" class="SimplePara">0.19</p></td>
<td><p xmlns="" class="SimplePara">1.71</p></td>
<td><p xmlns="" class="SimplePara">0.194</p></td>
<td><p xmlns="" class="SimplePara">0.08</p></td>
<td><p xmlns="" class="SimplePara">4.98</p></td>
<td><p xmlns="" class="SimplePara">0.007</p></td>
<td><p xmlns="" class="SimplePara">0.01</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
              </section><section id="Sec9" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic divergence between accessions and groups</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">AMOVA was performed in order to divide the total genetic variation into three components: variation within accessions, among accessions, and among countries. Most of the variation in the studied material was found within accessions: 89 % when based on REMAP markers, 93 % when based on SSR markers, and 91 % when based on both marker types (Table <span class="InternalRef"><a href="#Tab6">6</a></span>).<figure xmlns="" id="Tab6" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 6</span>
                        <p xmlns="" class="SimplePara">Analysis of molecular variance in 51 timothy accessions based on 533 REMAP markers, 464 SSRs or both</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead>
<tr class="header">
<th> </th>
<th> </th>
<th colspan="4"><p xmlns="" class="SimplePara">REMAP</p></th>
<th colspan="4"><p xmlns="" class="SimplePara">SSR</p></th>
<th colspan="4"><p xmlns="" class="SimplePara">REMAP and SSR</p></th>
</tr>
<tr class="header">
<th><p xmlns="" class="SimplePara">Source</p></th>
<th><p xmlns="" class="SimplePara">df</p></th>
<th><p xmlns="" class="SimplePara">SS</p></th>
<th><p xmlns="" class="SimplePara">MS</p></th>
<th><p xmlns="" class="SimplePara">Variance components</p></th>
<th><p xmlns="" class="SimplePara">% total</p></th>
<th><p xmlns="" class="SimplePara">SS</p></th>
<th><p xmlns="" class="SimplePara">MS</p></th>
<th><p xmlns="" class="SimplePara">Variance components</p></th>
<th><p xmlns="" class="SimplePara">% total</p></th>
<th><p xmlns="" class="SimplePara">SS</p></th>
<th><p xmlns="" class="SimplePara">MS</p></th>
<th><p xmlns="" class="SimplePara">Variance components</p></th>
<th><p xmlns="" class="SimplePara">% total</p></th>
</tr>
</thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Among countries</p></td>
<td char="."><p xmlns="" class="SimplePara">5</p></td>
<td char="."><p xmlns="" class="SimplePara">896.84</p></td>
<td char="."><p xmlns="" class="SimplePara">179.37</p></td>
<td char="."><p xmlns="" class="SimplePara">0.42</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">574.09</p></td>
<td char="."><p xmlns="" class="SimplePara">114.82</p></td>
<td char="."><p xmlns="" class="SimplePara">0.21</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
<td char="."><p xmlns="" class="SimplePara">1471.68</p></td>
<td char="."><p xmlns="" class="SimplePara">294.34</p></td>
<td char="."><p xmlns="" class="SimplePara">0.63</p></td>
<td char="."><p xmlns="" class="SimplePara">1</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Among accessions/countries</p></td>
<td char="."><p xmlns="" class="SimplePara">45</p></td>
<td char="."><p xmlns="" class="SimplePara">5181.97</p></td>
<td char="."><p xmlns="" class="SimplePara">115.15</p></td>
<td char="."><p xmlns="" class="SimplePara">4.21</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">3719.53</p></td>
<td char="."><p xmlns="" class="SimplePara">82.66</p></td>
<td char="."><p xmlns="" class="SimplePara">2.45</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">8901.50</p></td>
<td char="."><p xmlns="" class="SimplePara">197.81</p></td>
<td char="."><p xmlns="" class="SimplePara">6.66</p></td>
<td char="."><p xmlns="" class="SimplePara">8</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Within accessions</p></td>
<td char="."><p xmlns="" class="SimplePara">894</p></td>
<td char="."><p xmlns="" class="SimplePara">33312.58</p></td>
<td char="."><p xmlns="" class="SimplePara">37.26</p></td>
<td char="."><p xmlns="" class="SimplePara">37.26</p></td>
<td char="."><p xmlns="" class="SimplePara">89</p></td>
<td char="."><p xmlns="" class="SimplePara">33271.32</p></td>
<td char="."><p xmlns="" class="SimplePara">37.22</p></td>
<td char="."><p xmlns="" class="SimplePara">37.22</p></td>
<td char="."><p xmlns="" class="SimplePara">93</p></td>
<td char="."><p xmlns="" class="SimplePara">66583.90</p></td>
<td char="."><p xmlns="" class="SimplePara">74.48</p></td>
<td char="."><p xmlns="" class="SimplePara">74.48</p></td>
<td char="."><p xmlns="" class="SimplePara">91</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">944</p></td>
<td char="."><p xmlns="" class="SimplePara">39391.39</p></td>
<td> </td>
<td char="."><p xmlns="" class="SimplePara">41.89</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">37564.94</p></td>
<td> </td>
<td char="."><p xmlns="" class="SimplePara">39.88</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">76957.08</p></td>
<td> </td>
<td char="."><p xmlns="" class="SimplePara">81.77</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Stat</p></td>
<td><p xmlns="" class="SimplePara">Value</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">Value</p></td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">Value</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âPhiRT</p></td>
<td><p xmlns="" class="SimplePara">0.010</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">0.005</p></td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">0.008</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âPhiPR</p></td>
<td><p xmlns="" class="SimplePara">0.101</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">0.062</p></td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">0.082</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">âPhiPT</p></td>
<td><p xmlns="" class="SimplePara">0.110</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">0.067</p></td>
<td> </td>
<td> </td>
<td> </td>
<td><p xmlns="" class="SimplePara">0.089</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Probability, P(randââ¥âdata), for PhiRT, PhiPR and PhiPTâ=â0.001, and is based on permutation across the full data set</p>
                      <p xmlns="" class="SimplePara">PhiRTâ=âAC / (WAâ+âAAâ+âAC)â=âAC / TOT</p>
                      <p xmlns="" class="SimplePara">PhiPRâ=âAA / (WAâ+âAA)</p>
                      <p xmlns="" class="SimplePara">PhiPTâ=â(AAâ+âAC) / (WAâ+âAAâ+âAC)â=â(AAâ+âAC) / TOT</p>
                      <p xmlns="" class="SimplePara">Key: ACâ=âest. var. among countries, AAâ=âest. var. among accessions, WAâ=âest. var. within accessions</p>
                    </div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">No genetic divergence was observed between vegetation zones or cultivar types either using SSR or REMAP markers or both (AMOVA, <em xmlns="" class="EmphasisTypeItalic">P</em>â&lt;â0.05), which might be due to the small numbers of members in different classes. However, the same result was obtained with SSR markers when 96 accessions were studied [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. In PCA analysis as well, no clustering of accessions based on countries, vegetation zones, or cultivar types was seen (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). The first two axes respectively explained 44.1 %, 45.8 %, or 41.1 % of the variation when REMAPs, SSRs, or both marker types were used in the analysis.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Previously, SSR markers revealed timothy to be very diverse both on the individual and accession level when studied in a collection of 96 accessions. Because it was impossible with SSRs to resolve any population or geographical structure [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], we here have applied a very different kind of neutral marker, REMAPs, which are based on displaying retrotransposon insertions.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Both REMAPs and SSRs were highly polymorphic. Variation was observed mostly within accessions but with slightly smaller proportion for REMAPs (89 % vs. 93 %). This difference may be due to the biology of how SSR and retrotransposon polymorphisms are generated. SSRs are generated by replication slippage [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], a process expected to be independent of the environment. In contrast, retrotransposons are known to be activated by both biotic and abiotic stresses [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], conditions which may well be greater in some populations compared with others. Population-level stress would thereby lower the proportion of polymorphism on the individual level and increase it on population or geographic levels.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Diversity indices in accessions were lower for SSR than for REMAP markers. This is likely because SSR markers (i.e., alleles) are not independent of each other; there is a theoretical maximum number of markers that can exist in one individual. If all SSR loci would amplify from all three genomes of <em xmlns="" class="EmphasisTypeItalic">Phleum,</em> the maximum number of markers would be 78 (13 loci, 6 alleles in each). However, there is evidence that timothy is an allopolyploid [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. Allopolyploidy is consistent with our earlier results [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], with some SSR loci found only in one genome whereas others were present in all three. Therefore, the real maximum number of SSR alleles in any one individual lies somewhere between 26 and 78. In the present study, the observed maximum was 45.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Polyploids represent about 50 % of flowering plants [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. In polyploids, the problem of lack of independence between SSR loci is particularly a problem, but given a very high number of loci developed from the genome sequences of major crops such as cotton or wheat, chromosome-specific markers can be recovered [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. For agricultural species without reference genomes such as timothy or for many wild species [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>], selection of markers with diploid inheritance can reduce the usable loci to very low numbers.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">In contrast to SSRs, no limit exists for the maximum amount of REMAPs in one individual because retrotransposon insertions are independent of each other. Moreover, different retrotransposon families, such as in the hexaploid wheat genomes [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>], show different evolutionary histories, enabling discrimination between homeologues. Retrotransposon markers have been deployed effectively for even the highly polyploid sugarcane [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Although codominant REMAPs also exist, codominance does not restrict the possibility of co-existence of markers in one individual. The maximum amount for REMAPs observed in one individual in the present study was 121. Correlations between diversity indices based on REMAP or SSR markers were mostly low or moderate because the two marker systems report from different genomic regions where polymorphisms are generated by different processes. On the other hand, even though SSRs could be treated as codominant markers, it has been suggested that large similarities between diversity indices with dominant markers but somewhat lower between dominant markers and SSRs are due to insufficient numbers of analyzed SSR loci [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">When using markers for measuring distances, PWD between individuals correlated weakly (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.26) but genetic distances between accessions strongly (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.67) between the two marker types. PWD is based on the Euclidean distances between individuals whereas distances between accessions are based on marker frequencies. The same sort of result â poor or nonexistent individual-by-individual correlations but moderate correlation between accessions â was obtained when amplified fragment length polymorphisms (AFLPs), which are comparable to REMAPs by being a multilocus and dominant marker type, and SSRs were compared [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. In potato, a low correlation of SSR and REMAP markers (<em xmlns="" class="EmphasisTypeItalic">r</em>â=â0.17) in the Mantelâs matrix correspondence test was found [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Comparing the two marker types, REMAP markers were more cost-efficient. The PCRs of four REMAP primer combinations were made separately, and products from two different combinations with different fluorescent labels were combined for MegaBACE runs. As a consequence, for the whole diversity analysis study (945 samples), 40 PCRs on 96-well microtitre plates were made and analyzed in 20 Megabace runs. A total of 533 polymorphic markers was produced. On the other hand, the 13 SSR loci were multiplexed into 5 PCR reactions and analyzed in 5 MegaBACE runs, requiring in total 50 PCR plates and 50 MegaBACE runs. In addition, some planning and optimization was required in order to multiplex the PCR reactions for the various SSR loci. A total of 464 SSR markers (i.e., alleles) was amplified. Accordingly, more REMAP markers (i.e., loci) were produced with less labor, money, and time. The MI was over six-fold higher with REMAPs, which is not due only to the need to interpret SSR alleles as separate markers but is also typical for markers with an effective multiplex ratio, and has been detected also when AFLPs have been compared with SSRs [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Knowledge of genetic variation and relationships between individuals and accessions is essential when conserving and using genetic resources. Evaluation of genetic diversity requires analysis of multiple markers as efficiently as possible. Choosing a suitable marker type, several aspects have to be taken into account, not only expected heterozygosity and marker index, but also technical difficulty, ease of genotyping, cost, and availability. Technically, there were no differences between REMAPs and SSRs and we encountered analysis difficulties with both marker types. All SSR peaks contained some degree of stutter, which complicated the identification of alleles. On the other hand, interpretation of REMAP markers was very slow because several markers were amplified in one PCR reaction and there was a wide variation in peak heights. All peaks could not be analyzed and selections had to be made according to peak heights and frequency. Difficulties in scoring hindered the use of automated analysis programs for both marker types. Regarding availability, there are universal retrotransposon primers that can be used in any species, and primers specific for Graminae also have a vast range of use. Moreover, SSRs have not been developed for every species, and transform rates from one species to another depends on the genetic distance of the taxa [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. These general conclusions regarding the utility of SSR and retrotransposon markers alone and in combination are consistent with those for four diverse dicot species, distant from the monocot timothy [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>â<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 SectionTypeConclusion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">When diversity in a polyploid species is examined, where the codominant nature of SSRs is of no use, dominant REMAP markers, as analyzed by size on a sequencer, were more cost-efficient. REMAPs also described diversity from a larger segment of the genome compared to the same number of SSR alleles. On the other hand, SSRs detected differences in the level of diversity in different groups better than REMAPs. Furthermore, private SSR alleles were found, making SSRs better for accession identification. Private alleles, however, can be developed from retrotransposon markers using the RBIP (retrotransposon-based insertion polymorphism) and ISBP (insertion site-based polymorphism) methods, which are locus-specific [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Genetic distances between accessions were similar with REMAP or SSR markers, but neither marker type could reveal any clear divergence between vegetation zones, cultivar types or countries in the polyploid, very polymorphic and heterozygous timothy species. SSR and REMAP polymorphisms derive from very different mechanisms. Variations in SSR numbers at individual loci derive from polymerase slippage during replication. In contrast, retrotransposon insertions, which can be stress-driven, generate the priming sites for retrotransposon-based marker methods. Given the vastly different numbers of microsatellite and retrotransposon loci queried by the marker systems used, which report from very different genomic regions, the fact that they together show a lack of structure likely reflects the outcrossing and hexaploid nature of timothy rather than failures of either marker system. Both retrotransposons and SSRs, however, are neutral markers; patterns of variation in the gene space of timothy, such as through single nucleotide polymorphism (SNP) genotyping, remain to be explored. These would allow the possibility to evaluate allele dosages, thereby increasing the information embodied in each locus. SNP markers have been used in sugarcane, a complex autopolyploid species, to estimate ploidy level and also the dosage of SNPs [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The availability of SNP markers has increased with the invention of the genotyping by sequencing strategy (GBS) [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] and a recent study presents its use to evaluate allele frequencies in populations in an outbreeding species, perennial ryegrass [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Such techniques could be applied to timothy as well to study the structure of accessions. On the other hand, the importance of using both molecular and phenotypic markers for assessing diversity especially when evaluating adaptive potential has been emphasized in a study where timothy accessions were characterized with SSRs, chloroplast DNA sequences, as well as by morphological and phenological traits [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>].</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgements</h3>
<p xmlns="" class="SimplePara">We want to thank Nordgen, Mika Isolahti and Siri Fjellheim for selecting and providing plant material for the study. We appreciate the excellent technical assistance of Marja-Riitta ArajÃ¤rvi, Kirsti MÃ¤kelÃ¤, Ulla Kojonsaari, Sirpa Moisander, and Anneli Virta. The help of Kristiina Antonius, Marjo Serenius, and Miika Tapio in data analyses is also greatly acknowledged. We are grateful for The Ministry of Agriculture and Forestry in Finland, as well as The Nordic Joint Committee for Agricultural Research for their financial support.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3>
<p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authorsâ contributions</h3>
<p xmlns="" class="SimplePara">OM designed the research; PT analyzed the SSR marker data, carried out all data analyses and wrote the manuscript; RK developed the REMAP primers; ME analyzed the REMAP marker data; OM and ME gave comments on the manuscript; AS revised the manuscript. All authors read and approved the final manuscript.</p></aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Green Technology, Natural Resources Institute Finland (Luke), Myllytie 1, FI-31600 Jokioinen, Finland</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Internal Expert Services, Natural Resources Institute Finland (Luke), Humppilantie 14, FI-31600 Jokioinen, Finland</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Luke/BI Plant Genome Dynamics Laboratory, Institute of Biotechnology, Viikki Biocenter, University of Helsinki, P.O. Box 56, Viikinkaari 1, FI-00014 Helsinki, Finland</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4">
<span class="AffiliationNumber">(4)</span><div class="AffiliationText">Boreal Plant Breeding Ltd, Myllytie 10, FI-31600 Jokioinen, Finland</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">TanhuanpÃ¤Ã¤ P, Manninen O. High SSR diversity but little differentiation between accessions of Nordic timothy (<em class="EmphasisTypeItalic">Phleum pratense</em> L.). Hereditas. 2012;149:114â27.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1601-5223.2012.02244.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22967141"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High%20SSR%20diversity%20but%20little%20differentiation%20between%20accessions%20of%20Nordic%20timothy%20%28Phleum%20pratense%20L.%29&amp;author=P.%20Tanhuanp%C3%A4%C3%A4&amp;author=O.%20Manninen&amp;journal=Hereditas&amp;volume=149&amp;pages=114-127&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalendar R, Grob T, Regina M, Suoniemi A, Schulman A. IRAP and REMAP: two new retrotransposon-based DNA fingerprinting techniques. Theor Appl Genet. 1999;98:704â11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051124"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=IRAP%20and%20REMAP%3A%20two%20new%20retrotransposon-based%20DNA%20fingerprinting%20techniques&amp;author=R.%20Kalendar&amp;author=T.%20Grob&amp;author=M.%20Regina&amp;author=A.%20Suoniemi&amp;author=A.%20Schulman&amp;journal=Theor%20Appl%20Genet&amp;volume=98&amp;pages=704-711&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalendar R, Schulman AH. IRAP and REMAP for retrotransposon-based genotyping and fingerprinting. Nat protocols. 2006;1:2478â84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2006.377"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17406494"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=IRAP%20and%20REMAP%20for%20retrotransposon-based%20genotyping%20and%20fingerprinting&amp;author=R.%20Kalendar&amp;author=AH.%20Schulman&amp;journal=Nat%20protocols&amp;volume=1&amp;pages=2478-2484&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schulman AH. Retrotransposon replication in plants. Curr Opin Virol. 2013;3:604â14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.coviro.2013.08.009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24035277"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Retrotransposon%20replication%20in%20plants&amp;author=AH.%20Schulman&amp;journal=Curr%20Opin%20Virol&amp;volume=3&amp;pages=604-614&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schulman AH, Gupta PK, Varshney RK. Organization of retrotransposons and microsatellites in cereal genomes. In: Gupta PK, Varshney RK, editors. Cereal Genomics. Dordrecht: Kluwer Academic Publishers; 2004. p. 83â118.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Organization%20of%20retrotransposons%20and%20microsatellites%20in%20cereal%20genomes&amp;author=AH.%20Schulman&amp;author=PK.%20Gupta&amp;author=RK.%20Varshney&amp;pages=83-118&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grandbastien MA. LTR retrotransposons, handy hitchhikers of plant regulation and stress response. Biochim Biophys Acta. 1849;2015:403â16.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=LTR%20retrotransposons%2C%20handy%20hitchhikers%20of%20plant%20regulation%20and%20stress%20response&amp;author=MA.%20Grandbastien&amp;journal=Biochim%20Biophys%20Acta&amp;volume=2015&amp;pages=403-416&amp;publication_year=1849"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kelkar YD, Eckert KA, Chiaromonte F, Makova KD. A matter of life or death: how microsatellites emerge in and vanish from the human genome. Genome Res. 2011;21:2038â48.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.122937.111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21994250"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3227094"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20matter%20of%20life%20or%20death%3A%20how%20microsatellites%20emerge%20in%20and%20vanish%20from%20the%20human%20genome&amp;author=YD.%20Kelkar&amp;author=KA.%20Eckert&amp;author=F.%20Chiaromonte&amp;author=KD.%20Makova&amp;journal=Genome%20Res&amp;volume=21&amp;pages=2038-2048&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sharma V, Nandineni R. Assessment of genetic diversity among Indian potato (<em class="EmphasisTypeItalic">Solanum tuberosum</em> L.) collection using microsatellite and retrotransposon based marker systems. Mol Phylogenet Evol. 2014;73:10â7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ympev.2014.01.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24440815"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20among%20Indian%20potato%20%28Solanum%20tuberosum%20L.%29%20collection%20using%20microsatellite%20and%20retrotransposon%20based%20marker%20systems&amp;author=V.%20Sharma&amp;author=R.%20Nandineni&amp;journal=Mol%20Phylogenet%20Evol&amp;volume=73&amp;pages=10-17&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mandoulakani BA, Sadigh P, Azizi H, Piri Y, Nasri S, Arzhangh S. Comparative assessment of IRAP, REMAP, ISSR, and SSR markers for evaluation of genetic diversity of alfalfa (<em class="EmphasisTypeItalic">Medicago sativa</em> L.). J Agr Sci Tech. 2015;17:999â1010.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20assessment%20of%20IRAP%2C%20REMAP%2C%20ISSR%2C%20and%20SSR%20markers%20for%20evaluation%20of%20genetic%20diversity%20of%20alfalfa%20%28Medicago%20sativa%20L.%29&amp;author=BA.%20Mandoulakani&amp;author=P.%20Sadigh&amp;author=H.%20Azizi&amp;author=Y.%20Piri&amp;author=S.%20Nasri&amp;author=S.%20Arzhangh&amp;journal=J%20Agr%20Sci%20Tech&amp;volume=17&amp;pages=999-1010&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Villano C, Carputo D, Frusciante L, Santoro X, Aversano R. Use of SSR and retrotransposon-based markers to interpret the population structure of native grapevines from southern Italy. Mol Biotechnol. 2014;56:1011â20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12033-014-9780-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24973024"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Use%20of%20SSR%20and%20retrotransposon-based%20markers%20to%20interpret%20the%20population%20structure%20of%20native%20grapevines%20from%20southern%20Italy&amp;author=C.%20Villano&amp;author=D.%20Carputo&amp;author=L.%20Frusciante&amp;author=X.%20Santoro&amp;author=R.%20Aversano&amp;journal=Mol%20Biotechnol&amp;volume=56&amp;pages=1011-1020&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tinker NA, Fortin MG, Mather DE. Random amplified polymorphic DNA and pedigree relationships in spring barley. Theor Appl Genet. 1993;85:976â84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00215037"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24196148"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Random%20amplified%20polymorphic%20DNA%20and%20pedigree%20relationships%20in%20spring%20barley&amp;author=NA.%20Tinker&amp;author=MG.%20Fortin&amp;author=DE.%20Mather&amp;journal=Theor%20Appl%20Genet&amp;volume=85&amp;pages=976-984&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalendar R, Antonius K, Smykal P, Schulman AH. iPBS: A universal method for DNA fingerprinting and retrotransposon isolation. Theor Appl Genet. 2010;121:1419â30.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-010-1398-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20623102"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=iPBS%3A%20A%20universal%20method%20for%20DNA%20fingerprinting%20and%20retrotransposon%20isolation&amp;author=R.%20Kalendar&amp;author=K.%20Antonius&amp;author=P.%20Smykal&amp;author=AH.%20Schulman&amp;journal=Theor%20Appl%20Genet&amp;volume=121&amp;pages=1419-1430&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalendar R, Lee D, Schulman AH. FastPCR software for PCR, in silico PCR, and oligonucleotide assembly and analysis. Methods Mol Biol. 2014;1116:271â302.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-1-62703-764-8_18"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24395370"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=FastPCR%20software%20for%20PCR%2C%20in%20silico%20PCR%2C%20and%20oligonucleotide%20assembly%20and%20analysis&amp;author=R.%20Kalendar&amp;author=D.%20Lee&amp;author=AH.%20Schulman&amp;journal=Methods%20Mol%20Biol&amp;volume=1116&amp;pages=271-302&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cai HW, Yuyama N, Tamaki H, Yoshizawa A. Isolation and characterization of simple sequence repeat markers in the hexaploid forage grass timothy (<em class="EmphasisTypeItalic">Phleum pratense</em> L.). Theor Appl Genet. 2003;107:1337â49.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-003-1386-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12920514"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20simple%20sequence%20repeat%20markers%20in%20the%20hexaploid%20forage%20grass%20timothy%20%28Phleum%20pratense%20L.%29&amp;author=HW.%20Cai&amp;author=N.%20Yuyama&amp;author=H.%20Tamaki&amp;author=A.%20Yoshizawa&amp;journal=Theor%20Appl%20Genet&amp;volume=107&amp;pages=1337-1349&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu K, Muse SV. Powermarker: integrated analysis environment for genetic marker data. Bioinformatics. 2005;21:2128â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bti282"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15705655"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Powermarker%3A%20integrated%20analysis%20environment%20for%20genetic%20marker%20data&amp;author=K.%20Liu&amp;author=SV.%20Muse&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2128-2129&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Powell W, Morgante M, Andre C, Hanafey M, Vogel J, Tingey S, et al. The comparison of RFLP, RAPD, AFLP and SSR (microsatellite) markers for germplasm analysis. Mol Breeding. 1996;2:225â38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00564200"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20comparison%20of%20RFLP%2C%20RAPD%2C%20AFLP%20and%20SSR%20%28microsatellite%29%20markers%20for%20germplasm%20analysis&amp;author=W.%20Powell&amp;author=M.%20Morgante&amp;author=C.%20Andre&amp;author=M.%20Hanafey&amp;author=J.%20Vogel&amp;author=S.%20Tingey&amp;journal=Mol%20Breeding&amp;volume=2&amp;pages=225-238&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schneider S, Roessli D, Excoffier L. Arlequin: a software for population genetics data analysis. Ver 2.000. Genetics and Biometry Lab, Department of Anthropology, University of Geneva, Switzerland; 2000.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Schneider%20S%2C%20Roessli%20D%2C%20Excoffier%20L.%20Arlequin%3A%20a%20software%20for%20population%20genetics%20data%20analysis.%20Ver%202.000.%20Genetics%20and%20Biometry%20Lab%2C%20Department%20of%20Anthropology%2C%20University%20of%20Geneva%2C%20Switzerland%3B%202000."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shannon CE. A mathematical theory of communication. Bell System Tech J. 1948;27:379â423 and 623â56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/j.1538-7305.1948.tb01338.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20mathematical%20theory%20of%20communication&amp;author=CE.%20Shannon&amp;journal=Bell%20System%20Tech%20J&amp;volume=27&amp;pages=379%E2%80%93423%20and%20623%E2%80%9356&amp;publication_year=1948"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peakall R, Smouse PE. GENALEX 6: genetic analysis in Excel. Population genetic software for teaching and research. Mol Ecol Notes. 2006;6:288â95.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2005.01155.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GENALEX%206%3A%20genetic%20analysis%20in%20Excel.%20Population%20genetic%20software%20for%20teaching%20and%20research&amp;author=R.%20Peakall&amp;author=PE.%20Smouse&amp;journal=Mol%20Ecol%20Notes&amp;volume=6&amp;pages=288-295&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peakall R, Smouse PE. GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research â an update. Bioinformatics. 2012;28:2537â9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts460"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22820204"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463245"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GenAlEx%206.5%3A%20genetic%20analysis%20in%20Excel.%20Population%20genetic%20software%20for%20teaching%20and%20research%20%E2%80%93%20an%20update&amp;author=R.%20Peakall&amp;author=PE.%20Smouse&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2537-2539&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moen A. National Atlas of Norway: Vegetation. HÃ¸nefoss: Norwegian Mapping Authority; 1999.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=National%20Atlas%20of%20Norway%3A%20Vegetation&amp;author=A.%20Moen&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Excoffier L, Smouse PE, Quattro JM. Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction sites. Genetics. 1992;131:479â91.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1644282"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205020"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20molecular%20variance%20inferred%20from%20metric%20distances%20among%20DNA%20haplotypes%3A%20application%20to%20human%20mitochondrial%20DNA%20restriction%20sites&amp;author=L.%20Excoffier&amp;author=PE.%20Smouse&amp;author=JM.%20Quattro&amp;journal=Genetics&amp;volume=131&amp;pages=479-491&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei M. Genetic distance between populations. Am Nat. 1972;106:283â92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/282771"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20distance%20between%20populations&amp;author=M.%20Nei&amp;journal=Am%20Nat&amp;volume=106&amp;pages=283-292&amp;publication_year=1972"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mantel NA. The detection of disease clustering and a generalized regression approach. Cancer Res. 1967;27:209â20.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6018555"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20detection%20of%20disease%20clustering%20and%20a%20generalized%20regression%20approach&amp;author=NA.%20Mantel&amp;journal=Cancer%20Res&amp;volume=27&amp;pages=209-220&amp;publication_year=1967"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stewart AW, Joachimiak AJ, Ellison NW. Phleum. In: Kole C, editor. Wild crop relatives: genomic and breeding resources. Millets and grasses. Springer Berlin Heidelberg; 2011. p. 257â74.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Stewart%20AW%2C%20Joachimiak%20AJ%2C%20Ellison%20NW.%20Phleum.%20In%3A%20Kole%20C%2C%20editor.%20Wild%20crop%20relatives%3A%20genomic%20and%20breeding%20resources.%20Millets%20and%20grasses.%20Springer%20Berlin%20Heidelberg%3B%202011.%20p.%20257%E2%80%9374."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Soltis PS, Marchant DB, Van de Peer Y, Soltis DE. Polyploidy and genome evolution in plants. Curr Opin Genet Dev. 2015;35:119â25.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gde.2015.11.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26656231"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Polyploidy%20and%20genome%20evolution%20in%20plants&amp;author=PS.%20Soltis&amp;author=DB.%20Marchant&amp;author=Y.%20Peer&amp;author=DE.%20Soltis&amp;journal=Curr%20Opin%20Genet%20Dev&amp;volume=35&amp;pages=119-125&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Blenda A, Fang DD, Rami J-F, Garsmeur O, Luo F, Lacape J-M. A high density consensus genetic map of tetraploid cotton that integrates multiple component maps through molecular marker redundancy check. PLoS One. 2012;7, e45739.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0045739"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23029214"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3454346"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20high%20density%20consensus%20genetic%20map%20of%20tetraploid%20cotton%20that%20integrates%20multiple%20component%20maps%20through%20molecular%20marker%20redundancy%20check&amp;author=A.%20Blenda&amp;author=DD.%20Fang&amp;author=J-F.%20Rami&amp;author=O.%20Garsmeur&amp;author=F.%20Luo&amp;author=J-M.%20Lacape&amp;journal=PLoS%20One&amp;volume=7&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Korbecka G, Rymer PD, Harris SA, Pannell JR. Solving the problem of ambiguous paralogy for marker loci: microsatellite markers with diploid inheritance in allohexaploid <em class="EmphasisTypeItalic">Mercurialis annua</em> (Euphorbiaceae). J Hered. 2010;101:504â11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/esq026"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20219887"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Solving%20the%20problem%20of%20ambiguous%20paralogy%20for%20marker%20loci%3A%20microsatellite%20markers%20with%20diploid%20inheritance%20in%20allohexaploid%20Mercurialis%20annua%20%28Euphorbiaceae%29&amp;author=G.%20Korbecka&amp;author=PD.%20Rymer&amp;author=SA.%20Harris&amp;author=JR.%20Pannell&amp;journal=J%20Hered&amp;volume=101&amp;pages=504-511&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Konovalov FA, Goncharov NP, Goryunova S, Shaturova A, Proshlyakova T, Kudryavtsev A. Molecular markers based on LTR retrotransposons <em class="EmphasisTypeItalic">BARE-1</em> and <em class="EmphasisTypeItalic">Jeli</em> uncover different strata of evolutionary relationships in diploid wheats. Mol Genet Genomics. 2010;283:551â63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00438-010-0539-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20407790"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20markers%20based%20on%20LTR%20retrotransposons%20BARE-1%20and%20Jeli%20uncover%20different%20strata%20of%20evolutionary%20relationships%20in%20diploid%20wheats&amp;author=FA.%20Konovalov&amp;author=NP.%20Goncharov&amp;author=S.%20Goryunova&amp;author=A.%20Shaturova&amp;author=T.%20Proshlyakova&amp;author=A.%20Kudryavtsev&amp;journal=Mol%20Genet%20Genomics&amp;volume=283&amp;pages=551-563&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Metcalfe CJ, Oliveira SG, Gaiarsa JW, Aitken KS, Carneiro MS, Zatti F, et al. Using quantitative PCR with retrotransposon-based insertion polymorphisms as markers in sugarcane. J Exp Bot. 2015;66:4239â50.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jxb/erv283"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26093024"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4493790"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Using%20quantitative%20PCR%20with%20retrotransposon-based%20insertion%20polymorphisms%20as%20markers%20in%20sugarcane&amp;author=CJ.%20Metcalfe&amp;author=SG.%20Oliveira&amp;author=JW.%20Gaiarsa&amp;author=KS.%20Aitken&amp;author=MS.%20Carneiro&amp;author=F.%20Zatti&amp;journal=J%20Exp%20Bot&amp;volume=66&amp;pages=4239-4250&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nybom H. Comparison of different nuclear DNA markers for estimating intraspecific genetic diversity in plants. Mol Ecol. 2004;13:1143â55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2004.02141.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15078452"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20different%20nuclear%20DNA%20markers%20for%20estimating%20intraspecific%20genetic%20diversity%20in%20plants&amp;author=H.%20Nybom&amp;journal=Mol%20Ecol&amp;volume=13&amp;pages=1143-1155&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maguire TL, Peakall R, Saenger P. Comparative analysis of genetic diversity in the mangrove species <em class="EmphasisTypeItalic">Avicennia marina</em> (Forsk.) Vierh. (Avicenniaceae) detected by AFLPs and SSRs. Theor Appl Genet. 2002;104:388â98.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220100724"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12582711"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20genetic%20diversity%20in%20the%20mangrove%20species%20Avicennia%20marina%20%28Forsk.%29%20Vierh.%20%28Avicenniaceae%29%20detected%20by%20AFLPs%20and%20SSRs&amp;author=TL.%20Maguire&amp;author=R.%20Peakall&amp;author=P.%20Saenger&amp;journal=Theor%20Appl%20Genet&amp;volume=104&amp;pages=388-398&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Victoria FC, da Maia LC, de Oliveira AC. <em class="EmphasisTypeItalic">In silico</em> comparative analysis of SSR markers in plants. BMC Plant Biol. 2011;11:15.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-11-15"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21247422"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037304"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20In%20silico%20comparative%20analysis%20of%20SSR%20markers%20in%20plants&amp;author=FC.%20Victoria&amp;author=LC.%20Maia&amp;author=AC.%20Oliveira&amp;journal=BMC%20Plant%20Biol&amp;volume=11&amp;pages=15&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schulman AH, Flavell A, Paux E, Ellis THN. The application of LTR retrotransposons as molecular markers in plants. Methods Mol Biol. 2012;859:115â53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-1-61779-603-6_7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22367869"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20application%20of%20LTR%20retrotransposons%20as%20molecular%20markers%20in%20plants&amp;author=AH.%20Schulman&amp;author=A.%20Flavell&amp;author=E.%20Paux&amp;author=THN.%20Ellis&amp;journal=Methods%20Mol%20Biol&amp;volume=859&amp;pages=115-153&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Garcia AAF, Mollinari M, Marconi TG, Serang OR, Silva RR, Vieira MLC, et al. SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids. Sci Rep. 2013;3:3399.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/srep03399"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24292365"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844970"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SNP%20genotyping%20allows%20an%20in-depth%20characterisation%20of%20the%20genome%20of%20sugarcane%20and%20other%20complex%20autopolyploids&amp;author=AAF.%20Garcia&amp;author=M.%20Mollinari&amp;author=TG.%20Marconi&amp;author=OR.%20Serang&amp;author=RR.%20Silva&amp;author=MLC.%20Vieira&amp;journal=Sci%20Rep&amp;volume=3&amp;pages=3399&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Elshire RJ, Glaubitz JC, Sun Q, Poland JA, Kawamoto K, Buckler ES, et al. A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. PLoS One. 2011;6, e19379.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0019379"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21573248"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3087801"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20%28GBS%29%20approach%20for%20high%20diversity%20species&amp;author=RJ.%20Elshire&amp;author=JC.%20Glaubitz&amp;author=Q.%20Sun&amp;author=JA.%20Poland&amp;author=K.%20Kawamoto&amp;author=ES.%20Buckler&amp;journal=PLoS%20One&amp;volume=6&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Byrne S, Czaban A, Studer B, Panitz F, Bendixen C, Asp T. Genome wide allele frequency fingerprints (GWAFFs) of populations via genotyping by sequencing. PLoS One. 2013;8, e57438.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0057438"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23469194"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3587605"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20wide%20allele%20frequency%20fingerprints%20%28GWAFFs%29%20of%20populations%20via%20genotyping%20by%20sequencing&amp;author=S.%20Byrne&amp;author=A.%20Czaban&amp;author=B.%20Studer&amp;author=F.%20Panitz&amp;author=C.%20Bendixen&amp;author=T.%20Asp&amp;journal=PLoS%20One&amp;volume=8&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fjellheim S, TanhuanpÃ¤Ã¤ P, Marum P, Manninen O, Rognli OA. Phenotypic or molecular diversity screening for conservation of genetic resources? An example from a genebank collection of the temperate forage grass timothy. Crop Sci. 2015;55:1646â59.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2014.12.0825"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phenotypic%20or%20molecular%20diversity%20screening%20for%20conservation%20of%20genetic%20resources%3F%20An%20example%20from%20a%20genebank%20collection%20of%20the%20temperate%20forage%20grass%20timothy&amp;author=S.%20Fjellheim&amp;author=P.%20Tanhuanp%C3%A4%C3%A4&amp;author=P.%20Marum&amp;author=O.%20Manninen&amp;author=OA.%20Rognli&amp;journal=Crop%20Sci&amp;volume=55&amp;pages=1646-1659&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Tanhuanpetal" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Tanhuanpetal">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Tanhuanpetal"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© TanhuanpÃ¤Ã¤ et al. 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0009-x" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-016-0009-x">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0009-x" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0009-x?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0009-x #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0009-x #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0009-x #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0009-x">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0009-x">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0009-x"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0009-x">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0009-x">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0009-x"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0009-x">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0009-x">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0009-x"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0009-x">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0009-x">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0009-x"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0009-x">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0009-x">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0009-x"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-016-0009-x/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-016-0009-x">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0009-x">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0009-x" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-016-0009-x&amp;volume=153&amp;issue=1&amp;title=Assessment+of+genetic+diversity+in+Nordic+timothy+%28+Phleum+pratense+L+.+%29&amp;author_name=Pirjo+Tanhuanp%C3%A4%C3%A4&amp;start_page=1&amp;end_page=10">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-016-0009-x" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
