<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>A preliminary phylogeny of the South African Lentulidae | Hereditas | Full Text</title>
<meta name="citation_abstract" content="The grasshopper family Lentulidae is endemic to eastern and southern Africa, with its center of diversity situated in South Africa, the highest diversity being found in the Cape Floristic Region, which is one of the global biodiversity hotspots. The family consists of 35 genera sorted in two subfamilies. This study provides first insights into the phylogeny of Lentulidae. Two mitochondrial genes (12S and NDS) were sequenced and the phylogeny was inferred through Maximum Likelihood and Bayesian Inference. Our results indicate that the current classification into the subfamilies Lentulinae and Shelforditinae may be incorrect as Uvarovidium, Leatettix (Shelforditinae) and Devylderia (Lentulinae) clustered together in one main clade, while Betiscoides, Basutacris and Gymnidium (all Lentulinae) formed the second main clade. The genera Uvarovidium and Leatettix, which had been assigned to the Acrididae (subfamily Hemiacridinae) in the past, grouped within the Lentulidae, confirming their current assignment to this family. The East African Usambilla group is likely to represent a sister clade to the south African Lentula and Eremidium. Diversification patterns in the genus Devylderia and Betiscoides suggest a higher number of species than currently known. Our phylogeny is not in line with the current systematics of Lentulidae, suggesting that a broader sampling and a study of the genitalia would be useful to clarify the taxonomy. Furthermore, some genera (particularly Betiscoides and Devylderia) are in need of taxonomic revision, as the number of species within these genera is likely to be higher than the current taxonomy suggests.">
<meta name="description" content="The grasshopper family Lentulidae is endemic to eastern and southern Africa, with its center of diversity situated in South Africa, the highest diversity being found in the Cape Floristic Region, which is one of the global biodiversity hotspots. The family consists of 35 genera sorted in two subfamilies. This study provides first insights into the phylogeny of Lentulidae. Two mitochondrial genes (12S and NDS) were sequenced and the phylogeny was inferred through Maximum Likelihood and Bayesian Inference. Our results indicate that the current classification into the subfamilies Lentulinae and Shelforditinae may be incorrect as Uvarovidium, Leatettix (Shelforditinae) and Devylderia (Lentulinae) clustered together in one main clade, while Betiscoides, Basutacris and Gymnidium (all Lentulinae) formed the second main clade. The genera Uvarovidium and Leatettix, which had been assigned to the Acrididae (subfamily Hemiacridinae) in the past, grouped within the Lentulidae, confirming their current assignment to this family. The East African Usambilla group is likely to represent a sister clade to the south African Lentula and Eremidium. Diversification patterns in the genus Devylderia and Betiscoides suggest a higher number of species than currently known. Our phylogeny is not in line with the current systematics of Lentulidae, suggesting that a broader sampling and a study of the genitalia would be useful to clarify the taxonomy. Furthermore, some genera (particularly Betiscoides and Devylderia) are in need of taxonomic revision, as the number of species within these genera is likely to be higher than the current taxonomy suggests.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="A preliminary phylogeny of the South African Lentulidae">
<meta name="dc.source" content="Hereditas 2016 153:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2016-01-15">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2016 Matenaar et al.">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="The grasshopper family Lentulidae is endemic to eastern and southern Africa, with its center of diversity situated in South Africa, the highest diversity being found in the Cape Floristic Region, which is one of the global biodiversity hotspots. The family consists of 35 genera sorted in two subfamilies. This study provides first insights into the phylogeny of Lentulidae. Two mitochondrial genes (12S and NDS) were sequenced and the phylogeny was inferred through Maximum Likelihood and Bayesian Inference. Our results indicate that the current classification into the subfamilies Lentulinae and Shelforditinae may be incorrect as Uvarovidium, Leatettix (Shelforditinae) and Devylderia (Lentulinae) clustered together in one main clade, while Betiscoides, Basutacris and Gymnidium (all Lentulinae) formed the second main clade. The genera Uvarovidium and Leatettix, which had been assigned to the Acrididae (subfamily Hemiacridinae) in the past, grouped within the Lentulidae, confirming their current assignment to this family. The East African Usambilla group is likely to represent a sister clade to the south African Lentula and Eremidium. Diversification patterns in the genus Devylderia and Betiscoides suggest a higher number of species than currently known. Our phylogeny is not in line with the current systematics of Lentulidae, suggesting that a broader sampling and a study of the genitalia would be useful to clarify the taxonomy. Furthermore, some genera (particularly Betiscoides and Devylderia) are in need of taxonomic revision, as the number of species within these genera is likely to be higher than the current taxonomy suggests.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2016-01-15">
<meta name="prism.volume" content="153">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="1">
<meta name="prism.copyright" content="2016 Matenaar et al.">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-015-0005-6">
<meta name="prism.doi" content="doi:10.1186/s41065-015-0005-6">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-015-0005-6">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-015-0005-6">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="A preliminary phylogeny of the South African Lentulidae">
<meta name="citation_volume" content="153">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2016/12">
<meta name="citation_online_date" content="2016/01/15">
<meta name="citation_firstpage" content="1">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-015-0005-6">
<meta name="DOI" content="10.1186/s41065-015-0005-6">
<meta name="citation_doi" content="10.1186/s41065-015-0005-6">
<meta name="dc.creator" content="Daniela Matenaar">
<meta name="dc.creator" content="Linda Bröder">
<meta name="dc.creator" content="Axel Hochkirch">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Biodiversity hotspots for conservation priorities; citation_author=N Myers, RA Mittermeier, CG Mittermeier, GA Fonseca, J Kent; citation_volume=403; citation_publication_date=2000; citation_pages=853-8; citation_doi=10.1038/35002501; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Mol Phylogenet Evol; citation_title=Radiation in the Cape flora and the phylogeny of peacock irises Moraea (Iridaceae) based on four plastid DNA regions; citation_author=P Goldblatt, V Savolainen, O Porteous, I Sostaric, M Powell, G Reeves; citation_volume=25; citation_publication_date=2002; citation_pages=341-60; citation_doi=10.1016/S1055-7903(02)00235-X; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=Biol Rev; citation_title=The radiation of the Cape flora, southern Africa; citation_author=HP Linder; citation_volume=78; citation_publication_date=2003; citation_pages=597-638; citation_doi=10.1017/S1464793103006171; citation_id=CR3">
<meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Rapid and recent origin of species richness in the Cape flora of South Africa; citation_author=JE Richardson, FM Weitz, MF Fay, QC Cronk, HP Linder, G Reeves; citation_volume=412; citation_publication_date=2001; citation_pages=181-3; citation_doi=10.1038/35084067; citation_id=CR4">
<meta name="citation_reference" content="citation_title=Floristic and faunal Cape biochoria: do they exist?; citation_inbook_title=Fynbos: Ecology, evolution, and conservation of a megadiverse region; citation_publication_date=2014; citation_pages=73-92; citation_id=CR5; citation_author=JF Colville; citation_author=AJ Potts; citation_author=PL Bradshaw; citation_author=GJ Measey; citation_author=D Snijman; citation_author=MD Picker; citation_publisher=Oxford University Press">
<meta name="citation_reference" content="citation_journal_title=Biol Conserv; citation_title=Simple tools for the evaluation of protected areas for the conservation of grasshoppers; citation_author=D Matenaar, CS Bazelet, A Hochkirch; citation_volume=192; citation_publication_date=2015; citation_pages=192-9; citation_doi=10.1016/j.biocon.2015.09.023; citation_id=CR6">
<meta name="citation_reference" content="Eades DC, Otte D, Cigliano MM, Braun H. Orthoptera Species File. Version 5.0/5.0; 2015.">
<meta name="citation_reference" content="citation_journal_title=Entomology; citation_title=A revision of the genus Usambilla Sjostedt (Orthoptera: Acridoidea) and its allies. Bulletin of the British Museum. Natural History; citation_author=ND Jago; citation_volume=43; citation_publication_date=1981; citation_pages=1-38; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=J Insect Conserv; citation_title=Persisting in a windy habitat: population ecology and behavioral adaptations of two endemic grasshopper species in the Cape region (South Africa); citation_author=D Matenaar, L Bröder, CS Bazelet, A Hochkirch; citation_volume=18; citation_publication_date=2014; citation_pages=447-56; citation_doi=10.1007/s10841-014-9654-4; citation_id=CR9">
<meta name="citation_reference" content="Spearman LA. Taxonomic Revision of the South African Grasshopper Genus Euloryma (Orthoptera: Acrididae). 2013;139:1–111. doi:
                    10.3157/061.139.0101
                    
                  .">
<meta name="citation_reference" content="citation_journal_title=Trans Am Entomol Soc; citation_title=Twenty-Six New Species of Grasshoppers from the Southern African Arid Zone (Orthoptera Acridoidea:Lentulidae); citation_author=D Otte; citation_volume=140; citation_publication_date=2014; citation_pages=293-347; citation_doi=10.3157/061.140.0116; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Proc Biol Sci; citation_title=Character-based DNA barcoding allows discrimination of genera, species and populations in Odonata; citation_author=J Rach, R Desalle, IN Sarkar, B Schierwater, H Hadrys; citation_volume=275; citation_publication_date=2008; citation_pages=237-47; citation_doi=10.1098/rspb.2007.1290; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Annals of the Ditsong National Museum of Natural History; citation_title=Armstrongium - an unusual new sylvan genus of Lentulidae (Orthoptera: Acridoidea) from KwaZulu-Natal Province, South Africa; citation_author=HD Brown; citation_volume=49; citation_publication_date=2014; citation_pages=33-50; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Invert Systematics; citation_title=Phylogeny, historical biogeography and divergence time estimates of the genus Colophon Gray (Coleoptera: Lucanidae); citation_author=AK Switala, CL Sole, CH Scholtz; citation_volume=28; citation_publication_date=2014; citation_pages=326; citation_doi=10.1071/IS13054; citation_id=CR14">
<meta name="citation_reference" content="citation_title=Speciation and extinction in the Greater Cape Floristic Region; citation_inbook_title=Fynbos: Ecology, evolution, and conservation of a megadiverse region; citation_publication_date=2014; citation_pages=119-41; citation_id=CR15; citation_author=AG Ellis; citation_author=G Anthony Verboom; citation_author=T Niet; citation_author=SD Johnson; citation_author=LH Peter; citation_publisher=Oxford University Press">
<meta name="citation_reference" content="citation_journal_title=Biol J Linn Soc; citation_title=Unravelling evolutionary lineages among South African velvet worms (Onychophora: Peripatopsis) provides evidence for widespread cryptic speciation; citation_author=SR Daniels, MD Picker, RM Cowlin, ML Hammer; citation_volume=97; citation_publication_date=2009; citation_pages=200-16; citation_doi=10.1111/j.1095-8312.2009.01205.x; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=The influence of pollinator phylogeography and mate preference on floral divergence in a sexually deceptive daisy; citation_author=ML Jager, AG Ellis; citation_volume=67; citation_publication_date=2013; citation_pages=1706-14; citation_doi=10.1111/evo.12070; citation_id=CR17">
<meta name="citation_reference" content="citation_title=The shifting landscapes of genes since the Pliocene: terrestrial phylogeography in the Greater Cape Floristic Region; citation_inbook_title=Fynbos: Ecology, evolution, and conservation of a megadiverse region; citation_publication_date=2014; citation_pages=142-63; citation_id=CR18; citation_author=K Tolley; citation_author=R Bowie; citation_author=GJ Measey; citation_author=BJ Price; citation_author=F Forest; citation_publisher=Oxford University Press">
<meta name="citation_reference" content="citation_title=Fynbos: Ecology, evolution, and conservation of a megadiverse region; citation_publication_date=2014; citation_id=CR19; citation_publisher=Oxford University Press">
<meta name="citation_reference" content="Song H, Amédégnato C, Cigliano MM, Desutter-Grandcolas L, Heads SW, Huang Y, et al. 300 million years of diversification: Elucidating the patterns of orthopteran evolution based on comprehensive taxon and gene sampling. Cladistics. 2015:1–31. doi:
                    10.1111/cla.12116
                    
                  .">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0; citation_author=K Tamura, G Stecher, D Peterson, A Filipski, S Kumar; citation_volume=30; citation_publication_date=2013; citation_pages=2725-9; citation_doi=10.1093/molbev/mst197; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Partitionfinder: combined selection of partitioning schemes and substitution models for phylogenetic analyses; citation_author=R Lanfear, B Calcott, SYW Ho, S Guindon; citation_volume=29; citation_publication_date=2012; citation_pages=1695-701; citation_doi=10.1093/molbev/mss020; citation_id=CR22">
<meta name="citation_reference" content="Huelsenbeck JP, Ronquist F. MRBAYES: Bayesian inference of phylogenetic trees. 2001;17:754–5.">
<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models; citation_author=F Ronquist, JP Huelsenbeck; citation_volume=19; citation_publication_date=2003; citation_pages=1572-4; citation_doi=10.1093/bioinformatics/btg180; citation_id=CR24">
<meta name="citation_reference" content="Rambaut A. FigTree 1.4.2. 2015. 
                    http://tree.bio.ed.ac.uk/software/figtree
                    
                  ; 2015.">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; citation_author=K Tamura, D Peterson, N Peterson, G Stecher, M Nei, S Kumar; citation_volume=28; citation_publication_date=2011; citation_pages=2731-9; citation_doi=10.1093/molbev/msr121; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Dynamics of mitochondrial DNA evolution in animals: Amplification and sequencing with conserved primers; citation_author=TD Kocher, WK Thomas, A Meyer, SV Edwards, S Pääbo, FX Villablanca; citation_volume=86; citation_publication_date=1989; citation_pages=6196-200; citation_doi=10.1073/pnas.86.16.6196; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Phylogeographic Patterns in Coastal North American Tiger Beetles (Cicindela dorsalis Say) Inferred from Mitochondrial DNA Sequences; citation_author=R Vogler, R Desalle; citation_volume=47; citation_publication_date=1993; citation_pages=1192-201; citation_doi=10.2307/2409985; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Sequence evolution in the ribosomal and ND-1 genes in Lepidoptera: implications for phylogenetic analyses; citation_author=D Pashley, LD Ke; citation_volume=9; citation_publication_date=1992; citation_pages=1061-75; citation_id=CR29">
<meta name="citation_author" content="Daniela Matenaar">
<meta name="citation_author_institution" content="Department of Biogeography, Trier University, Trier, Germany">
<meta name="citation_author_institution" content="State Museum of Natural History Stuttgart, Department of Entomology, Stuttgart, Germany">
<meta name="citation_author" content="Linda Bröder">
<meta name="citation_author_institution" content="Department of Biogeography, Trier University, Trier, Germany">
<meta name="citation_author" content="Axel Hochkirch">
<meta name="citation_author_institution" content="Department of Biogeography, Trier University, Trier, Germany">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-015-0005-6',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-015-0005-6","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Biodiversity hotspot;Cape Floristic Region;Invertebrates;Orthoptera;Taxonomy"},"contentInfo":{"imprint":"BioMed Central","title":"A preliminary phylogeny of the South African Lentulidae","publishedAt":1452816000000,"publishedAtDate":"2016-01-15","author":["Daniela Matenaar","Linda Bröder","Axel Hochkirch"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"153","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-015-0005-6">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-015-0005-6">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="A preliminary phylogeny of the South African Lentulidae">
<meta property="og:description" content="The grasshopper family Lentulidae is endemic to eastern and southern Africa, with its center of diversity situated in South Africa, the highest diversity being found in the Cape Floristic Region, which is one of the global biodiversity hotspots. The family consists of 35 genera sorted in two subfamilies. This study provides first insights into the phylogeny of Lentulidae. Two mitochondrial genes (12S and NDS) were sequenced and the phylogeny was inferred through Maximum Likelihood and Bayesian Inference. Our results indicate that the current classification into the subfamilies Lentulinae and Shelforditinae may be incorrect as Uvarovidium, Leatettix (Shelforditinae) and Devylderia (Lentulinae) clustered together in one main clade, while Betiscoides, Basutacris and Gymnidium (all Lentulinae) formed the second main clade. The genera Uvarovidium and Leatettix, which had been assigned to the Acrididae (subfamily Hemiacridinae) in the past, grouped within the Lentulidae, confirming their current assignment to this family. The East African Usambilla group is likely to represent a sister clade to the south African Lentula and Eremidium. Diversification patterns in the genus Devylderia and Betiscoides suggest a higher number of species than currently known. Our phylogeny is not in line with the current systematics of Lentulidae, suggesting that a broader sampling and a study of the genitalia would be useful to clarify the taxonomy. Furthermore, some genera (particularly Betiscoides and Devylderia) are in need of taxonomic revision, as the number of species within these genera is likely to be higher than the current taxonomy suggests.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-015-0005-6;kwrd=Biodiversity hotspot,Cape Floristic Region,Invertebrates,Orthoptera,Taxonomy;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-015-0005-6&amp;kwrd=Biodiversity%20hotspot,Cape%20Floristic%20Region,Invertebrates,Orthoptera,Taxonomy&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-015-0005-6&amp;kwrd=Biodiversity%20hotspot,Cape%20Floristic%20Region,Invertebrates,Orthoptera,Taxonomy&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec3" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec4" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec5" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">A preliminary phylogeny of the South African Lentulidae</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Daniela Matenaar</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:daniela.matenaar@gmail.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Linda Bröder</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Axel Hochkirch</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>
</li>
</ul></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">153</strong>:1</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-015-0005-6">https://doi.org/10.1186/s41065-015-0005-6</a></p>
<p class="Copyright">©  Matenaar et al. 2016</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>6 October 2015</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>23 December 2015</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">15 January 2016</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">The grasshopper family Lentulidae is endemic to eastern and southern Africa, with its center of diversity situated in South Africa, the highest diversity being found in the Cape Floristic Region, which is one of the global biodiversity hotspots. The family consists of 35 genera sorted in two subfamilies. This study provides first insights into the phylogeny of Lentulidae. Two mitochondrial genes (12S and NDS) were sequenced and the phylogeny was inferred through Maximum Likelihood and Bayesian Inference.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Our results indicate that the current classification into the subfamilies Lentulinae and Shelforditinae may be incorrect as <em xmlns="" class="EmphasisTypeItalic">Uvarovidium, Leatettix</em> (Shelforditinae) and <em xmlns="" class="EmphasisTypeItalic">Devylderia</em> (Lentulinae) clustered together in one main clade, while <em xmlns="" class="EmphasisTypeItalic">Betiscoides, Basutacris</em> and <em xmlns="" class="EmphasisTypeItalic">Gymnidium</em> (all Lentulinae) formed the second main clade. The genera <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em> and <em xmlns="" class="EmphasisTypeItalic">Leatettix</em>, which had been assigned to the Acrididae (subfamily Hemiacridinae) in the past, grouped within the Lentulidae, confirming their current assignment to this family. The East African <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group is likely to represent a sister clade to the south African <em xmlns="" class="EmphasisTypeItalic">Lentula</em> and <em xmlns="" class="EmphasisTypeItalic">Eremidium</em>. Diversification patterns in the genus <em xmlns="" class="EmphasisTypeItalic">Devylderia</em> and <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em> suggest a higher number of species than currently known.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Our phylogeny is not in line with the current systematics of Lentulidae, suggesting that a broader sampling and a study of the genitalia would be useful to clarify the taxonomy. Furthermore, some genera (particularly <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em> and <em xmlns="" class="EmphasisTypeItalic">Devylderia</em>) are in need of taxonomic revision, as the number of species within these genera is likely to be higher than the current taxonomy suggests.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Biodiversity hotspot</li>
<li class="c-keywords__item" itemprop="about">Cape Floristic Region</li>
<li class="c-keywords__item" itemprop="about">Invertebrates</li>
<li class="c-keywords__item" itemprop="about">Orthoptera</li>
<li class="c-keywords__item" itemprop="about">Taxonomy</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">The Cape Floristic Region in South Africa is one of the global biodiversity hotspots [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Numerous studies have dealt with the enormous plant diversity and its origin and a number of phylogenetic studies on certain plant families aimed at unravelling the drivers for diversification in this region [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>–<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The fauna of the Cape Floristic Region, though not less unique, has not been investigated as intensely as the flora, especially research on invertebrate diversity remains still scarce [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Recent evidence suggests that invertebrate diversity and endemism in the Cape Floristic Region might be comparable to the pattern found in plants [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. The Cape Floristic Region also maintains a variety of endemic families, subfamilies and genera, suggesting that it provided refugia also over a longer time-span. One of these subfamilies endemic to the Cape Floristic Region is the subfamily Shelforditinae Ritchie, 1982 within the family Lentulidae Dirsh, 1956. Within the family Lentulidae as a whole, the majority of taxa (70 %) are also endemic to South Africa [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], but some genera occur in East Africa, particularly the <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group, which shows a radiation in the East African mountain systems [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">All Lentulidae are completely wingless, and therefore expected to show high levels of genetic differentiation at a small geographic scale, as they have a low mobility and are often adapted to certain vegetation structures. For example, the Lentulidae genus <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em> Sjöstedt, 1923 is adapted to Restionaceae, which also show a high level of differentiation in the Cape Floristic Region [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Studies on the phylogenetics of grasshoppers from the Cape Floristic Region are lacking so far. However, several recent taxonomic studies suggest a high level of differentiation of flightless grasshopper species in the Cape Floristic Region with the identification of tens of hitherto undescribed species [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Given the enormous number of unidentified species in times of increasing efforts to halt the loss of biodiversity, there have been approaches to record and assess biodiversity parameters also on higher taxonomic level [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. Thus, it is of increasing interest to investigate the systematic relationships of endemic taxa to understand the importance of biodiversity hotspots.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">The aim of our study was to investigate and reconstruct the phylogenetic relationships among genera within the family Lentulidae. We were particularly interested in the systematic position of the genera <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> Dirsh, 1956 and <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em> Dirsh, 1956, which are currently assigned to the subfamily Shelforditinae. Furthermore, we aimed to clarify the phylogenetic relationships of the East African Usambilla group (here represented by the genera <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> Sjöstedt, 1910 and <em xmlns="" class="EmphasisTypeItalic">Rhainopomma</em> Jago, 1981) with respect to the South African Lentulidae. We present the results of a first phylogenetic study on Lentulidae based upon two mitochondrial genes (12S rRNA and NDS – a fragment containing parts of the 16S rRNA, t-Leu and ND1), covering ten genera from both subfamilies, in order to provide new insights in evolutionary history of this family.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">The complete alignment contained 808–823 bp. The 12S fragment was between 318 and 327 bp (when including the outgroups between 316 and 327 bp) and NDS (consisting of a fragment of 16S, t-Leu and ND1) between 490 and 497 bp long (with outgroup: 487–497). NDS had more variable sites (254) compared to 12S with 178 variable sites (see Table <span class="InternalRef"><a href="#Tab1">1</a></span>), but the percentage of variable sites was slightly larger in 12S (54 % vs. 51 %). The highest genetic distance (p-Distance for the 12S data set) between genera within Lentulidae was detected between <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> and <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em> with <em xmlns="" class="EmphasisTypeItalic">p</em> = 0.224, whereas the lowest was found between <em xmlns="" class="EmphasisTypeItalic">Basutacris</em> and <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em> (<em xmlns="" class="EmphasisTypeItalic">p</em> = 0.057, Table <span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">Overview on gene specific parameters for genes 12S and NDS (16S + t-Leu + ND1) of the combined data set</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col char=".">
<col char=".">
<col char=".">
<col char=".">
</colgroup>
<thead>
<tr class="header">
<th rowspan="2"><p xmlns="" class="SimplePara">Sequence</p></th>
<th colspan="3"><p xmlns="" class="SimplePara">Number of</p></th>
<th colspan="4" char="."><p xmlns="" class="SimplePara">Nucleotide frequency [%]</p></th>
</tr>
<tr class="header">
<th><p xmlns="" class="SimplePara">bp</p></th>
<th><p xmlns="" class="SimplePara">indels</p></th>
<th><p xmlns="" class="SimplePara">variable positions</p></th>
<th char="."><p xmlns="" class="SimplePara">T</p></th>
<th char="."><p xmlns="" class="SimplePara">C</p></th>
<th char="."><p xmlns="" class="SimplePara">A</p></th>
<th char="."><p xmlns="" class="SimplePara">G</p></th>
</tr>
</thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">12S</p></td>
<td><p xmlns="" class="SimplePara">327</p></td>
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">178 (54.4 %)</p></td>
<td char="."><p xmlns="" class="SimplePara">41.4</p></td>
<td char="."><p xmlns="" class="SimplePara">10.6</p></td>
<td char="."><p xmlns="" class="SimplePara">31.3</p></td>
<td char="."><p xmlns="" class="SimplePara">16.6</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NDS</p></td>
<td><p xmlns="" class="SimplePara">497</p></td>
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">254 (51.1 %)</p></td>
<td char="."><p xmlns="" class="SimplePara">45.6</p></td>
<td char="."><p xmlns="" class="SimplePara">10.4</p></td>
<td char="."><p xmlns="" class="SimplePara">30.0</p></td>
<td char="."><p xmlns="" class="SimplePara">14.0</p></td>
</tr>
</tbody>
</table></div></figure><figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                      <p xmlns="" class="SimplePara">Genetic distances (p-Distance) between the analyzed genera of Lentulidae based upon the 12S data set</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col char=".">
<col>
<col>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th> </th>
<th char="."><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Devylderia</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Betiscoides</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Leatettix</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Uvarovidium</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Gymnidium</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Basutacris</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Usambilla</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Lentula</em>
                          </p></th>
<th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Eremidium</em>
                          </p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Betiscoides</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.115</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Leatettix</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.129</p></td>
<td><p xmlns="" class="SimplePara">0.142</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Uvarovidium</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.124</p></td>
<td><p xmlns="" class="SimplePara">0.147</p></td>
<td><p xmlns="" class="SimplePara">0.104</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Gymnidium</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.118</p></td>
<td><p xmlns="" class="SimplePara">0.078</p></td>
<td><p xmlns="" class="SimplePara">0.151</p></td>
<td><p xmlns="" class="SimplePara">0.155</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Basutacris</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.116</p></td>
<td><p xmlns="" class="SimplePara">0.057</p></td>
<td><p xmlns="" class="SimplePara">0.137</p></td>
<td><p xmlns="" class="SimplePara">0.142</p></td>
<td><p xmlns="" class="SimplePara">0.069</p></td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Usambilla</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.193</p></td>
<td><p xmlns="" class="SimplePara">0.176</p></td>
<td><p xmlns="" class="SimplePara">0.217</p></td>
<td><p xmlns="" class="SimplePara">0.224</p></td>
<td><p xmlns="" class="SimplePara">0.172</p></td>
<td><p xmlns="" class="SimplePara">0.172</p></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Lentula</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.169</p></td>
<td><p xmlns="" class="SimplePara">0.140</p></td>
<td><p xmlns="" class="SimplePara">0.169</p></td>
<td><p xmlns="" class="SimplePara">0.170</p></td>
<td><p xmlns="" class="SimplePara">0.153</p></td>
<td><p xmlns="" class="SimplePara">0.151</p></td>
<td><p xmlns="" class="SimplePara">0.204</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Eremidium</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.136</p></td>
<td><p xmlns="" class="SimplePara">0.105</p></td>
<td><p xmlns="" class="SimplePara">0.152</p></td>
<td><p xmlns="" class="SimplePara">0.143</p></td>
<td><p xmlns="" class="SimplePara">0.113</p></td>
<td><p xmlns="" class="SimplePara">0.114</p></td>
<td><p xmlns="" class="SimplePara">0.182</p></td>
<td><p xmlns="" class="SimplePara">0.137</p></td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Rhainopomma</em>
                          </p></td>
<td char="."><p xmlns="" class="SimplePara">0.172</p></td>
<td><p xmlns="" class="SimplePara">0.135</p></td>
<td><p xmlns="" class="SimplePara">0.192</p></td>
<td><p xmlns="" class="SimplePara">0.214</p></td>
<td><p xmlns="" class="SimplePara">0.135</p></td>
<td><p xmlns="" class="SimplePara">0.144</p></td>
<td><p xmlns="" class="SimplePara">0.092</p></td>
<td><p xmlns="" class="SimplePara">0.165</p></td>
<td><p xmlns="" class="SimplePara">0.137</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">The phylogenetic reconstruction showed similar results for both methods, i.e. Maximum Likelihood and Bayesian Inference. Thus, only the results derived from the Bayesian Inference are presented and discussed here in detail. The Maximum Likelihood tree is provided in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figure S1. The consensus tree from the Bayesian analysis showed a monophyly of the Lentulidae with respect to the chosen outgroups <em xmlns="" class="EmphasisTypeItalic">Sphingonotus rubescens</em> and <em xmlns="" class="EmphasisTypeItalic">Frontifissia elegans</em> and <em xmlns="" class="EmphasisTypeItalic">F. laevata</em> (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). Within the family Lentulidae, a clear basal split between two main clades was revealed, which was supported by a BPP value of 0.98. The first main clade consisted of the genera <em xmlns="" class="EmphasisTypeItalic">Gymnidium</em>, <em xmlns="" class="EmphasisTypeItalic">Basutacris</em>, <em xmlns="" class="EmphasisTypeItalic">Betiscoides, Lentula, Eremidium, Rhainopomma</em> and <em xmlns="" class="EmphasisTypeItalic">Usambilla.</em> Specimens collected in the same region clustered together, e.g. Table Mountain (<em xmlns="" class="EmphasisTypeItalic">Betiscoides</em>), Hottentots Holland (<em xmlns="" class="EmphasisTypeItalic">Betiscoides</em>), Jonaskop (<em xmlns="" class="EmphasisTypeItalic">Betiscoides</em>), Baviaanskloof (<em xmlns="" class="EmphasisTypeItalic">Betiscoides</em>), and Kogelberg (<em xmlns="" class="EmphasisTypeItalic">Gymnidium</em>). The East African <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group (genera <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> and <em xmlns="" class="EmphasisTypeItalic">Rhainopomma</em>) formed a monophyletic group, related to the South African genera <em xmlns="" class="EmphasisTypeItalic">Eremidium</em> and <em xmlns="" class="EmphasisTypeItalic">Lentula</em>.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                  </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">Shows the inferred phylogeny of the analyzed genera belonging to Lentulidae. Consensus tree of the Bayesian Inference for the genes 12S and NDS; 824 bp in total, 20 million generations with sample frequency of 2000. <em class="EmphasisTypeItalic">Frontifissia</em> and <em class="EmphasisTypeItalic">Sphingonotus</em> were defined as outgroups</p>
                    </div></figcaption></figure>
</div>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">The second main clade, consisting of the genera <em xmlns="" class="EmphasisTypeItalic">Devylderia</em>, <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em> and <em xmlns="" class="EmphasisTypeItalic">Leatettix</em>, revealed a rather distinct pattern regarding their geographical distribution. <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> specimens from Cederberg clustered in a single clade, strongly separated from the clades in Swartberg and Baviaanskloof, as a sister clade to <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em>. However, this relationship was supported by a BPP value of only 0.84. <em xmlns="" class="EmphasisTypeItalic">Devylderia</em> formed four subclades, one including individuals from Jonaskop, one from Cederberg and Groot Winterhoek, one represented in Jonaskop, Table Mountain, Limietberg, Hottentots Holland, Groot Winterhoek and De Hoop, and the fourth subclade included only one specimen from Cederberg.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec3" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">The results provide first insights into the phylogeny of the family Lentulidae. The clear split between the two main clades was highly supported. However, this split does not reflect the current subdivision into the two subfamilies Lentulinae and Shelforditinae. Even though the two Shelforditinae genera <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> and <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em> clustered as a monophylum, they showed a clear sister group relationship to the genus <em xmlns="" class="EmphasisTypeItalic">Devylderia</em> (Lentulinae). This could mean that either <em xmlns="" class="EmphasisTypeItalic">Devylderia</em> belongs to Shelforditinae as well or that the current taxonomy within the complete family needs to be revised. A broader taxon sampling across all described lentulid genera would provide more precise information on the systematic relationships of the genera and clarify the assignments to the respective subfamilies. The East African <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group is likely to represent a sister clade to <em xmlns="" class="EmphasisTypeItalic">Lentula</em> and <em xmlns="" class="EmphasisTypeItalic">Eremidium</em>, confirming their systematic position within the subfamily Lentulinae. This also suggests that the Lentulidae as a whole originated in South Africa and started to diversify here, before one group (the ancestor of the <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group) spread to the north to radiate within the East African Mountain systems.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Although our sampling only covers a small fraction of the Lentulidae genera, a strong genetic differentiation between and within genera becomes already evident. In some cases, our phylogeny does not reflect the current taxonomy. This is the case for the genus <em xmlns="" class="EmphasisTypeItalic">Leatettix</em>, which does not represent a monophylum, but a paraphyletic group. Otte [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] mentioned in his revision of <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> that further division of <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> species into different genera might be necessary. As <em xmlns="" class="EmphasisTypeItalic">Leatettix emota</em> from Cederberg are closer related to <em xmlns="" class="EmphasisTypeItalic">U. peninsulare</em> than to <em xmlns="" class="EmphasisTypeItalic">Leatettix moraki</em> from Swartberg and Baviaanskloof, his suggestion seems to be supported by our findings. Further morphological and genetic research is required to clarify these findings.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Recent revisions of several lentulid genera from South Africa showed that the number of Lentulidae species is much higher than currently known. In total, 44 new species and even three new genera have recently been described (<em xmlns="" class="EmphasisTypeItalic">Armstrongium</em>, <em xmlns="" class="EmphasisTypeItalic">Tanquata</em>, <em xmlns="" class="EmphasisTypeItalic">Tsautettix,</em> [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]). The center of species richness of Lentulidae is in South Africa with 103 of the 146 described species occurring in the Cape Floristic Region [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Recent studies show that insect species diversity in the Cape Floristic Region is generally much higher than current taxonomy suggests [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. This seems to be particularly true for the genus <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em> (Matenaar et al. unpubl.), which is also confirmed in our study. Although this genus represents a monophylum, specimens from different localities showed a high genetic differentiation. All specimens used in our study would morphologically be assigned to <em xmlns="" class="EmphasisTypeItalic">Betiscoides meridionalis</em>. However, the results indicate four clades of this species with high genetic differentiation, suggesting the existence of cryptic species (which is confirmed by first morphological inspection).</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The reasons for the unique diversity in the Cape Floristic Region and potential drivers of differentiation and speciation have been discussed before [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Concerning insect diversity, there is a consensus about the long-term isolation of populations starting in the early Miocene [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>–<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. The Cape Floristic Region experienced climatic changes during late Mio- and Pliocene and repeated orographic changes through the uplift of the Cape Fold Belt as well as oceanic regression. These changing environmental conditions probably triggered and influenced dispersal as well as survival of insect taxa in refugia. Coastal regions and mountains of the Cape Floristic Region probably functioned as refugia during unfavorable periods, whereas ocean regression repeatedly enabled taxa to disperse into lowland habitats and the interior of the Cape Floristic Region [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. Climatic stability in mountain ranges throughout the Pleistocene is believed to have had positive effects on diversity as it supported the origin of new species while keeping extinction rates low [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. As a result, species were able to persist in montane or coastal refugia. The lack of gene flow caused by low dispersal capabilities led to high genetic differentiation within genera. As all Lentulidae are flightless, they are likely to have limited dispersal capabilities, as has been shown in the genus <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em> [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Consequently, this might also explain diversity patterns within the Lentulidae in general, even though the initial splits into several main clades or genera probably occurred much earlier. Several palaeorelictual insects of the Cape Floristic Region are known [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], suggesting that parts of the Cape Floristic Region may have served as refugia for quite a long time. Some represent ancient Gondwanan lineages. Some Lentulidae genera (<em xmlns="" class="EmphasisTypeItalic">Betisocoides, Devylderia</em>) seem to be adapted to plants of typical fynbos plants, which radiated 70 my ago. The phylogeny of Orthoptera indicates that Lentulidae started radiating during that period as well [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. It is thus unlikely that Lentulidae include ancient Gondwanan genera despite their high endemicity.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec4" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Our results provide first insights in the phylogeny of the grasshopper family Lentulidae, indicating that the current subdivision into two subfamilies needs to be revised and that the East African <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group represents a sister clade to the genera <em xmlns="" class="EmphasisTypeItalic">Lentula</em> and <em xmlns="" class="EmphasisTypeItalic">Eremidium</em>. Furthermore, species diversity within the genera seems to be higher than the current taxonomy suggests. Further research is needed including a broader taxon sampling and sequencing of additional genes.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec5" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Study objects</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Members of the family Lentulidae are flightless and lack tympana. Further morphological features, such as shape of head or antenna, vary within this family. Lentulidae occur in South and East Africa with the center of diversity situated in South Africa. In total, 70 % of the Lentulidae are endemic to South Africa. The family is grouped into two subfamilies Lentulinae and Shelforditinae, the latter one only occurring in South Africa. Lentulinae currently comprise 88 species within 25 genera, Shelforditinae consist of ten genera with 44 species. Genera such as <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> and <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em> had been classified as Hemiacridinae (family Acrididae) for some time, but were assigned to the Shelforditinae by Ritchie (1982). The East African <em xmlns="" class="EmphasisTypeItalic">Usambilla</em> group belongs to the subfamily Lentulinae.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">Sampling</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">A total of 50 Lentulidae specimens from six genera (<em xmlns="" class="EmphasisTypeItalic">Basutacris</em>, <em xmlns="" class="EmphasisTypeItalic">Betiscoides</em>, <em xmlns="" class="EmphasisTypeItalic">Devylderia</em>, <em xmlns="" class="EmphasisTypeItalic">Gymnidium</em>, <em xmlns="" class="EmphasisTypeItalic">Leatettix</em> and <em xmlns="" class="EmphasisTypeItalic">Uvarovidium</em>) were collected during four field trips from February 2012 to December 2013. Most specimens were collected in the eight reserves forming the UNESCO World Heritage Site “Cape Floral Region Protected Areas” and some in a private nature reserve “Jonaskop”. If possible, 3–5 specimens of each species were collected at each locality. Specimens were caught by sweep net or by hand and killed in a freezer. Afterwards they were stored in 99 % ethanol p.a. or dried and pinned. Information on specimens and localities and GenBank reference numbers is given in Fig. <span class="InternalRef"><a href="#Fig2">2</a></span> and Table <span class="InternalRef"><a href="#Tab3">3</a></span>. Specimens are stored at the State Museum of Natural History in Stuttgart. <em xmlns="" class="EmphasisTypeItalic">Sphingonotus rubescens</em> (Acrididae: Oedipodinae), <em xmlns="" class="EmphasisTypeItalic">Frontifissia elegans</em> and <em xmlns="" class="EmphasisTypeItalic">F. laevata</em> (Acrididae; Catantopinae) were chosen as outgroups.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Shows the respective collection sites</p>
                      </div></figcaption></figure><figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Information on specimen, locality (coordinates and reserve) and collection site. The site numbers are included in Fig. <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Fig2">2</a></span>
                                    </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Specimen</p></th>
<th><p xmlns="" class="SimplePara">Site</p></th>
<th><p xmlns="" class="SimplePara">Reserve</p></th>
<th><p xmlns="" class="SimplePara">Coordinates</p></th>
<th><p xmlns="" class="SimplePara">Genbank 12S</p></th>
<th><p xmlns="" class="SimplePara">Genbank NDS</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Uvarovidium peninsulare</em> 37</p></td>
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">Jonkershoek</p></td>
<td><p xmlns="" class="SimplePara">S33 58.005 E18 55.195</p></td>
<td><p xmlns="" class="SimplePara">KU206353</p></td>
<td><p xmlns="" class="SimplePara">KU214623</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Uvarovidum peninsulare</em> 62</p></td>
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.341 E19 07.709</p></td>
<td><p xmlns="" class="SimplePara">KU206364</p></td>
<td><p xmlns="" class="SimplePara">KU214634</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix cf emota</em> 15</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.917 E19 11.052</p></td>
<td><p xmlns="" class="SimplePara">KU206335</p></td>
<td><p xmlns="" class="SimplePara">KU214605</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix cf emota</em> 34</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.704 E19 11.010</p></td>
<td><p xmlns="" class="SimplePara">KU206351</p></td>
<td><p xmlns="" class="SimplePara">KU214621</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix cf emota</em> 58</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.917 E19 11.052</p></td>
<td><p xmlns="" class="SimplePara">KU206362</p></td>
<td><p xmlns="" class="SimplePara">KU214632</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix moraki</em> 36</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">Swartberg</p></td>
<td><p xmlns="" class="SimplePara">S33 21.698 E22 05.094</p></td>
<td><p xmlns="" class="SimplePara">KU206352</p></td>
<td><p xmlns="" class="SimplePara">KU214622</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix moraki</em> 48</p></td>
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">Swartberg</p></td>
<td><p xmlns="" class="SimplePara">S33 21.698 E22 05.094</p></td>
<td><p xmlns="" class="SimplePara">KU206356</p></td>
<td><p xmlns="" class="SimplePara">KU214626</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix moraki</em> 38</p></td>
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">Swartberg</p></td>
<td><p xmlns="" class="SimplePara">S33 20.777 E21 58.702</p></td>
<td><p xmlns="" class="SimplePara">KU234555</p></td>
<td><p xmlns="" class="SimplePara">n/a</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Leatettix moraki</em> 18</p></td>
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">Baviaanskloof</p></td>
<td><p xmlns="" class="SimplePara">S33 38.085 E24 28.425</p></td>
<td><p xmlns="" class="SimplePara">KU206338</p></td>
<td><p xmlns="" class="SimplePara">KU214608</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 1</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 58.163 E19 30.300</p></td>
<td><p xmlns="" class="SimplePara">KU206324</p></td>
<td><p xmlns="" class="SimplePara">KU214594</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 5</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 58.163 E19 30.300</p></td>
<td><p xmlns="" class="SimplePara">KU206327</p></td>
<td><p xmlns="" class="SimplePara">KU214597</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 8</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 58.163 E19 30.300</p></td>
<td><p xmlns="" class="SimplePara">KU206329</p></td>
<td><p xmlns="" class="SimplePara">KU214599</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 7</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 58.163 E19 30.300</p></td>
<td><p xmlns="" class="SimplePara">KU206328</p></td>
<td><p xmlns="" class="SimplePara">KU214598</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 3</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 58.163 E19 30.300</p></td>
<td><p xmlns="" class="SimplePara">KU206325</p></td>
<td><p xmlns="" class="SimplePara">KU214595</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 4</p></td>
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 58.163 E19 30.300</p></td>
<td><p xmlns="" class="SimplePara">KU206326</p></td>
<td><p xmlns="" class="SimplePara">KU214596</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 14</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">Groot Winterhoek</p></td>
<td><p xmlns="" class="SimplePara">S33 00.197 E19 04.348</p></td>
<td><p xmlns="" class="SimplePara">KU206334</p></td>
<td><p xmlns="" class="SimplePara">KU214604</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia</em> spec. 60</p></td>
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">Hottentots Holland</p></td>
<td><p xmlns="" class="SimplePara">S34 04.454 E19 03.702</p></td>
<td><p xmlns="" class="SimplePara">KU206363</p></td>
<td><p xmlns="" class="SimplePara">KU214633</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 47</p></td>
<td><p xmlns="" class="SimplePara">6</p></td>
<td><p xmlns="" class="SimplePara">Kogelberg</p></td>
<td><p xmlns="" class="SimplePara">S34 17.765 E19 07.470</p></td>
<td><p xmlns="" class="SimplePara">KU234556</p></td>
<td><p xmlns="" class="SimplePara">n/a</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia</em> spec. 55</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">Kogelberg</p></td>
<td><p xmlns="" class="SimplePara">S34 21.355 E18 51.543</p></td>
<td><p xmlns="" class="SimplePara">KU234557</p></td>
<td><p xmlns="" class="SimplePara">n/a</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 29</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S33 57.536 E18 23.268</p></td>
<td><p xmlns="" class="SimplePara">KU206347</p></td>
<td><p xmlns="" class="SimplePara">KU214617</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia coryphistoides</em> 49</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S34 19.038 E18 25.211</p></td>
<td><p xmlns="" class="SimplePara">KU206357</p></td>
<td><p xmlns="" class="SimplePara">KU214627</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 51</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S34 19.038 E18 25.211</p></td>
<td><p xmlns="" class="SimplePara">KU206359</p></td>
<td><p xmlns="" class="SimplePara">KU214629</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 45</p></td>
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">Limietberg</p></td>
<td><p xmlns="" class="SimplePara">S33 41.272 E19 05.750</p></td>
<td><p xmlns="" class="SimplePara">KU206354</p></td>
<td><p xmlns="" class="SimplePara">KU214624</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 52</p></td>
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S33 57.536 E18 23.268</p></td>
<td><p xmlns="" class="SimplePara">KU206360</p></td>
<td><p xmlns="" class="SimplePara">KU214630</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 67</p></td>
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">Kogelberg</p></td>
<td><p xmlns="" class="SimplePara">S34 19.604 E18 50.224</p></td>
<td><p xmlns="" class="SimplePara">KU234559</p></td>
<td><p xmlns="" class="SimplePara">n/a</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia bothai</em> 46</p></td>
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">De Hoop</p></td>
<td><p xmlns="" class="SimplePara">S34 26.644 E20 25.301</p></td>
<td><p xmlns="" class="SimplePara">KU206355</p></td>
<td><p xmlns="" class="SimplePara">KU214625</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia coryphistoides</em> 17</p></td>
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 26.168 E19 10.929</p></td>
<td><p xmlns="" class="SimplePara">KU206337</p></td>
<td><p xmlns="" class="SimplePara">KU214607</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia coryphistoides</em> 31</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.704 E19 11.010</p></td>
<td><p xmlns="" class="SimplePara">KU206348</p></td>
<td><p xmlns="" class="SimplePara">KU214618</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia capensis</em> 32</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.704 E19 11.010</p></td>
<td><p xmlns="" class="SimplePara">KU206349</p></td>
<td><p xmlns="" class="SimplePara">KU214619</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia cf capensis</em> 33</p></td>
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.704 E19 11.010</p></td>
<td><p xmlns="" class="SimplePara">KU206350</p></td>
<td><p xmlns="" class="SimplePara">KU214620</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia</em> spec<em class="EmphasisTypeItalic">.</em> 50</p></td>
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">Groot Winterhoek</p></td>
<td><p xmlns="" class="SimplePara">S33 00.197 E19 04.348</p></td>
<td><p xmlns="" class="SimplePara">KU206358</p></td>
<td><p xmlns="" class="SimplePara">KU214628</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia</em> spec<em class="EmphasisTypeItalic">.</em> 54</p></td>
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">Groot Winterhoek</p></td>
<td><p xmlns="" class="SimplePara">S32 59.589 E19 03.548</p></td>
<td><p xmlns="" class="SimplePara">KU206361</p></td>
<td><p xmlns="" class="SimplePara">KU214631</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Devylderia bothai</em> 16</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.917 E19 11.052</p></td>
<td><p xmlns="" class="SimplePara">KU206336</p></td>
<td><p xmlns="" class="SimplePara">KU214606</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 9</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S34 19.038 E18 25.211</p></td>
<td><p xmlns="" class="SimplePara">KU206330</p></td>
<td><p xmlns="" class="SimplePara">KU214600</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 10</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S34 19.038 E18 25.211</p></td>
<td><p xmlns="" class="SimplePara">KU206331</p></td>
<td><p xmlns="" class="SimplePara">KU214601</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 12</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S34 19.038 E18 25.211</p></td>
<td><p xmlns="" class="SimplePara">KU206333</p></td>
<td><p xmlns="" class="SimplePara">KU214603</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 11</p></td>
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Table Mountain</p></td>
<td><p xmlns="" class="SimplePara">S34 19.038 E18 25.211</p></td>
<td><p xmlns="" class="SimplePara">KU206332</p></td>
<td><p xmlns="" class="SimplePara">KU214602</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 21</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">Hottentots Holland</p></td>
<td><p xmlns="" class="SimplePara">S33 58.832 E19 07.903</p></td>
<td><p xmlns="" class="SimplePara">KU206341</p></td>
<td><p xmlns="" class="SimplePara">KU214611</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 22</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">Hottentots Holland</p></td>
<td><p xmlns="" class="SimplePara">S33 58.832 E19 07.903</p></td>
<td><p xmlns="" class="SimplePara">KU206342</p></td>
<td><p xmlns="" class="SimplePara">KU214612</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 23</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">Hottentots Holland</p></td>
<td><p xmlns="" class="SimplePara">S33 58.832 E19 07.903</p></td>
<td><p xmlns="" class="SimplePara">KU206343</p></td>
<td><p xmlns="" class="SimplePara">KU214613</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 24</p></td>
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">Hottentots Holland</p></td>
<td><p xmlns="" class="SimplePara">S33 58.832 E19 07.903</p></td>
<td><p xmlns="" class="SimplePara">KU206344</p></td>
<td><p xmlns="" class="SimplePara">KU214614</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 26</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 57.622 E19 31.168</p></td>
<td><p xmlns="" class="SimplePara">KU206345</p></td>
<td><p xmlns="" class="SimplePara">KU214615</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 28</p></td>
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">Jonaskop</p></td>
<td><p xmlns="" class="SimplePara">S33 57.622 E19 31.168</p></td>
<td><p xmlns="" class="SimplePara">KU206346</p></td>
<td><p xmlns="" class="SimplePara">KU214616</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 19</p></td>
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">Baviaanskloof</p></td>
<td><p xmlns="" class="SimplePara">S33 38.085 E24 28.425</p></td>
<td><p xmlns="" class="SimplePara">KU206339</p></td>
<td><p xmlns="" class="SimplePara">KU214609</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Betiscoides meridionalis</em> 20</p></td>
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">Baviaanskloof</p></td>
<td><p xmlns="" class="SimplePara">S33 38.085 E24 28.425</p></td>
<td><p xmlns="" class="SimplePara">KU206340</p></td>
<td><p xmlns="" class="SimplePara">KU214610</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Basutacris</em> spec<em class="EmphasisTypeItalic">.</em> 65</p></td>
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">Cederberg</p></td>
<td><p xmlns="" class="SimplePara">S32 25.917 E19 11.052</p></td>
<td><p xmlns="" class="SimplePara">KU206366</p></td>
<td><p xmlns="" class="SimplePara">KU214636</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Gymnidium turbinatum</em> 61</p></td>
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">De Hoop</p></td>
<td><p xmlns="" class="SimplePara">S34 28.320 E20 27.120</p></td>
<td><p xmlns="" class="SimplePara">KU234558</p></td>
<td><p xmlns="" class="SimplePara">n/a</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Gymnidium turbinatum</em> 64</p></td>
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">Kogelberg</p></td>
<td><p xmlns="" class="SimplePara">S34 21.355 E18 51.543</p></td>
<td><p xmlns="" class="SimplePara">KU206365</p></td>
<td><p xmlns="" class="SimplePara">KU214635</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Gymnidium turbinatum</em> 70</p></td>
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">Kogelberg</p></td>
<td><p xmlns="" class="SimplePara">S34 19.528 E18 50.763</p></td>
<td><p xmlns="" class="SimplePara">KU206367</p></td>
<td><p xmlns="" class="SimplePara">KU214637</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Sequences for 12S from the genera <em xmlns="" class="EmphasisTypeItalic">Eremidium nr equuleus</em> (AY569277.1), <em xmlns="" class="EmphasisTypeItalic">Lentula callani</em> (NC_020774.1) <em xmlns="" class="EmphasisTypeItalic">Lentula obtusa</em> (AY569276.1), <em xmlns="" class="EmphasisTypeItalic">Usambilla sagonai</em> (AY569279.1) and <em xmlns="" class="EmphasisTypeItalic">Rhainopomma montanum</em> (Z97601.1) were obtained from Genbank for an extended data set for 12S. The data sets for 12S and NDS were concatenated, data for NDS was coded missing when sequences were not available (Table <span class="InternalRef"><a href="#Tab3">3</a></span>).</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2"><h3 class="Heading">DNA analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">DNA was isolated from the insects’ hind leg muscles using the Qiagen DNeasy Blood and Tissue Kit (Qiagen GmbH, Hilden, Germany). We amplified two mitochondrial gene fragments: 12S rRNA and the gene fragment NDS (a combination of 16S rRNA, t-Leu and NADH-Dehydrogenase subunit 1 (ND1); see Table <span class="InternalRef"><a href="#Tab4">4</a></span> for primer sequences). The gene fragment 12S was chosen as it typically amplifies well also in old samples (i.e. the pinned specimens used), whereas NDS was used because it is more variable and typically shows a better resolution than 12S. Two different polymerases were used for PCR reactions (HotMasterMix by 5Prime and HotStarTaq Master Mix by Qiagen). The reactions for both polymerases were compiled as follows: 26 μl of diH<sub>2</sub>O, 20 μl of HotStarMasterMix or HotStarTaq Master Mix respectively, 0.3 pmol of each primer and 50 ng of DNA template (total Vol. = 50 μl). HotStarMasterMix was used for amplifying 12S under the following conditions: 94 °C for 2 min., 37 Cycles of denaturation at 94 °C for 30 s., annealing at 45 °C for 30 s and elongation at 65 °C for 1 min and a final elongation step for 10 min at 65 °C. NDS was amplified with the HotStarTaq Master Mix under the following conditions: Initial denaturation at 96 °C for 20 min., 33 cycles of denaturation at 96 °C, annealing at 45 °C for and elongation at 68 °C for 1.30 min each following a final elongation step at 65 °C for 3 min. PCR products were visualized on an 1 % agarose gel stained with SYBR Green I (Biozym, Hessisch, Oldendorf, Germany). The products were purified using Roche High Pure PCR Product Purification Kit (Roche Deutschland Holding GmbH, Germany) and sequenced afterwards at MacroGen Cooperation (Amsterdam, The Netherlands) or on a MEGABACE 1000 automated sequencer at the University of Trier with the DYEnamic ET Terminator Cycle Sequencing Premixkit (GE Healthcare, Munich, Germany).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Primer information for analyzed gene fragments 12S, 16S, t-Leu and ND1</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Name</p></th>
<th><p xmlns="" class="SimplePara">Gene</p></th>
<th><p xmlns="" class="SimplePara">Sequence (5'-3')</p></th>
<th><p xmlns="" class="SimplePara">Reference</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">12sai</p></td>
<td><p xmlns="" class="SimplePara">12S rRNA</p></td>
<td><p xmlns="" class="SimplePara">AAA CTA GGA TTA GAT ACC CTA TTA T</p></td>
<td><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">12sbi</p></td>
<td><p xmlns="" class="SimplePara">12S rRNA</p></td>
<td><p xmlns="" class="SimplePara">AAG AGC GAC GGG CGA TGT GT</p></td>
<td><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NDII</p></td>
<td><p xmlns="" class="SimplePara">16S rRNA</p></td>
<td><p xmlns="" class="SimplePara">ACA TGA TCT GAG TTC AAA CCG G</p></td>
<td><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">NDS</p></td>
<td><p xmlns="" class="SimplePara">ND1</p></td>
<td><p xmlns="" class="SimplePara">TAG AAT TAG AAG ATC AAC CAG C</p></td>
<td><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
              </section><section id="Sec9" class="Section2 RenderAsSection2"><h3 class="Heading">Sequence analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Sequences were inspected in MEGA 6.0 [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] and aligned using ClustalW. The Gap opening penalty was set to 15 for pairwise and multiple alignments respectively and the gap extension penalty was set to 6.66 for both pairwise and multiple alignment. IUB was chosen as DNA Weight Matrix with a transition weight of 0.5. Delay Divergent cutoff was set to 30 %. The genetic distance (p-Distance) between the genera was calculated only for the 12S sequences, as NDS was missing in GenBank for the genera <em xmlns="" class="EmphasisTypeItalic">Lentula</em>, <em xmlns="" class="EmphasisTypeItalic">Eremidium</em>, <em xmlns="" class="EmphasisTypeItalic">Rhainopomma</em> and <em xmlns="" class="EmphasisTypeItalic">Usambilla</em>. For tree reconstruction with Bayesian Inference we first used PartitionFinder in order to detect possible partitions in the data set and to find the best-fitting substitution models applicable in MrBayes [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Subsets were defined before Partitionfinder was run by dividing NDS in the components 16S (including t-Leu) and ND1 as well as the coding gene ND1 into its coding positions. In total, five different partitions with different substitution models were calculated. The model GTR + I + G was the best-fitting model for 12S, 16S and first coding position of ND1; HKY + I was calculated for the second coding position of ND1 and HKY + G for the third coding position of ND1.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">We performed a Bayesian analyses on the datasets using MrBayes v.3.1.2 [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. The analysis was run for 20 million generations, sampling trees every 2000 generations. The first 2500 trees were discarded before a consensus tree was calculated and visualized in FIGTREE v. 1.4.2 [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The Maximum Likelihood tree reconstruction was run in MEGA 6.0. In addition to PartitionFinder, we applied the goodness-of-fit test for each nucleotide model in order to find the best substitution model applicable in MEGA [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. Based on the BIC (Bayesian Information Criterion) and AICc (corrected Akaike Information Criterion), we chose the best-fitting models. The best fitting model for the Maximum likelihood analysis was GTR + G (BIC: 6153.091). The Bootstrap method was chosen as phylogenetic test and the number of replications of bootstraps was set to 500. Gaps or missing data were treated with partial deletion with a cutoff of 95 %. The heuristic method was set to Nearest Neighbor Interchange with the first tree being calculated automatically. The Branch swap filter was set to very strong.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgements</h3>
<p xmlns="" class="SimplePara">We would like to thank SANParks, Eastern Cape Parks and Tourism Agency and CapeNature for providing the permits to work in the Cape Floral Region Protected Areas. In particular we thank the managers of the reserves for their continuous support during our studies. We are also grateful to Eva Heym for her helpful assistance in the field. This study was funded by German Research Foundation (DFG, GRK 1319) as part of the PhD of D.M.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside class="Appendix" id="App1"><h3 class="Heading">Additional files</h3>
<div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-015-0005-6/MediaObjects/41065_2015_5_MOESM1_ESM.jpg"><span class="caption-number">Additional file 1: Figure S1.</span></a>
                          <span xmlns="" class="SimplePara">Phylogeny of the analyzed genera belonging to Lentulidae. Maximum likelihood tree for the genes 12S and NDS. <em class="EmphasisTypeItalic">Frontifissia</em> and <em class="EmphasisTypeItalic">Sphingonotus</em> were defined as outgroups. (JPG 1183 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3>
<p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors’ contributions</h3>
<p xmlns="" class="SimplePara">DM, LB and AH conceived the ideas; DM and LB conducted the field work and assessed the data; DM and AH wrote the manuscript. All authors read and approved the final manuscript.</p></aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Biogeography, Trier University, D-54286 Trier, Germany</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">State Museum of Natural History Stuttgart, Department of Entomology, D-70191 Stuttgart, Germany</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Myers N, Mittermeier RA, Mittermeier CG, da Fonseca GA, Kent J. Biodiversity hotspots for conservation priorities. Nature. 2000;403:853–8. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1038/35002501" target="_blank" class="ext-link"><span class="RefSource">10.1038/35002501</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/35002501"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10706275"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biodiversity%20hotspots%20for%20conservation%20priorities&amp;author=N.%20Myers&amp;author=RA.%20Mittermeier&amp;author=CG.%20Mittermeier&amp;author=GA.%20Fonseca&amp;author=J.%20Kent&amp;journal=Nature&amp;volume=403&amp;pages=853-8&amp;publication_year=2000&amp;doi=10.1038%2F35002501"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goldblatt P, Savolainen V, Porteous O, Sostaric I, Powell M, Reeves G, et al. Radiation in the Cape flora and the phylogeny of peacock irises Moraea (Iridaceae) based on four plastid DNA regions. Mol Phylogenet Evol. 2002;25:341–60. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1016/S1055-7903(02)00235-X" target="_blank" class="ext-link"><span class="RefSource">10.1016/S1055-7903(02)00235-X</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1055-7903(02)00235-X"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12414315"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Radiation%20in%20the%20Cape%20flora%20and%20the%20phylogeny%20of%20peacock%20irises%20Moraea%20%28Iridaceae%29%20based%20on%20four%20plastid%20DNA%20regions&amp;author=P.%20Goldblatt&amp;author=V.%20Savolainen&amp;author=O.%20Porteous&amp;author=I.%20Sostaric&amp;author=M.%20Powell&amp;author=G.%20Reeves&amp;journal=Mol%20Phylogenet%20Evol&amp;volume=25&amp;pages=341-60&amp;publication_year=2002&amp;doi=10.1016%2FS1055-7903%2802%2900235-X"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Linder HP. The radiation of the Cape flora, southern Africa. Biol Rev. 2003;78:597–638. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1017/S1464793103006171" target="_blank" class="ext-link"><span class="RefSource">10.1017/S1464793103006171</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/S1464793103006171"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14700393"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20radiation%20of%20the%20Cape%20flora%2C%20southern%20Africa&amp;author=HP.%20Linder&amp;journal=Biol%20Rev&amp;volume=78&amp;pages=597-638&amp;publication_year=2003&amp;doi=10.1017%2FS1464793103006171"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Richardson JE, Weitz FM, Fay MF, Cronk QC, Linder HP, Reeves G, et al. Rapid and recent origin of species richness in the Cape flora of South Africa. Nature. 2001;412:181–3. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1038/35084067" target="_blank" class="ext-link"><span class="RefSource">10.1038/35084067</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/35084067"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11449273"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20and%20recent%20origin%20of%20species%20richness%20in%20the%20Cape%20flora%20of%20South%20Africa&amp;author=JE.%20Richardson&amp;author=FM.%20Weitz&amp;author=MF.%20Fay&amp;author=QC.%20Cronk&amp;author=HP.%20Linder&amp;author=G.%20Reeves&amp;journal=Nature&amp;volume=412&amp;pages=181-3&amp;publication_year=2001&amp;doi=10.1038%2F35084067"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Colville JF, Potts AJ, Bradshaw PL, Measey GJ, Snijman D, Picker MD, et al. Floristic and faunal Cape biochoria: do they exist? In: Allsopp N, Colville JF, Verboom GA, editors. Fynbos: Ecology, evolution, and conservation of a megadiverse region. Oxford, New York: Oxford University Press; 2014. p. 73–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/acprof%3Aoso/9780199679584.003.0004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Floristic%20and%20faunal%20Cape%20biochoria%3A%20do%20they%20exist%3F&amp;author=JF.%20Colville&amp;author=AJ.%20Potts&amp;author=PL.%20Bradshaw&amp;author=GJ.%20Measey&amp;author=D.%20Snijman&amp;author=MD.%20Picker&amp;pages=73-92&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Matenaar D, Bazelet CS, Hochkirch A. Simple tools for the evaluation of protected areas for the conservation of grasshoppers. Biol Conserv. 2015;192:192–9. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1016/j.biocon.2015.09.023" target="_blank" class="ext-link"><span class="RefSource">10.1016/j.biocon.2015.09.023</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.biocon.2015.09.023"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Simple%20tools%20for%20the%20evaluation%20of%20protected%20areas%20for%20the%20conservation%20of%20grasshoppers&amp;author=D.%20Matenaar&amp;author=CS.%20Bazelet&amp;author=A.%20Hochkirch&amp;journal=Biol%20Conserv&amp;volume=192&amp;pages=192-9&amp;publication_year=2015&amp;doi=10.1016%2Fj.biocon.2015.09.023"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Eades DC, Otte D, Cigliano MM, Braun H. Orthoptera Species File. Version 5.0/5.0; 2015.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Eades%20DC%2C%20Otte%20D%2C%20Cigliano%20MM%2C%20Braun%20H.%20Orthoptera%20Species%20File.%20Version%205.0%2F5.0%3B%202015."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jago ND. A revision of the genus Usambilla Sjostedt (Orthoptera: Acridoidea) and its allies. Bulletin of the British Museum. Natural History. Entomology. 1981;43:1–38.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20revision%20of%20the%20genus%20Usambilla%20Sjostedt%20%28Orthoptera%3A%20Acridoidea%29%20and%20its%20allies.%20Bulletin%20of%20the%20British%20Museum.%20Natural%20History&amp;author=ND.%20Jago&amp;journal=Entomology&amp;volume=43&amp;pages=1-38&amp;publication_year=1981"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Matenaar D, Bröder L, Bazelet CS, Hochkirch A. Persisting in a windy habitat: population ecology and behavioral adaptations of two endemic grasshopper species in the Cape region (South Africa). J Insect Conserv. 2014;18:447–56. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1007/s10841-014-9654-4" target="_blank" class="ext-link"><span class="RefSource">10.1007/s10841-014-9654-4</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10841-014-9654-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Persisting%20in%20a%20windy%20habitat%3A%20population%20ecology%20and%20behavioral%20adaptations%20of%20two%20endemic%20grasshopper%20species%20in%20the%20Cape%20region%20%28South%20Africa%29&amp;author=D.%20Matenaar&amp;author=L.%20Br%C3%B6der&amp;author=CS.%20Bazelet&amp;author=A.%20Hochkirch&amp;journal=J%20Insect%20Conserv&amp;volume=18&amp;pages=447-56&amp;publication_year=2014&amp;doi=10.1007%2Fs10841-014-9654-4"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Spearman LA. Taxonomic Revision of the South African Grasshopper Genus Euloryma (Orthoptera: Acrididae). 2013;139:1–111. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.3157/061.139.0101" target="_blank" class="ext-link"><span class="RefSource">10.3157/061.139.0101</span></a></span>.</span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Otte D. Twenty-Six New Species of Grasshoppers from the Southern African Arid Zone (Orthoptera Acridoidea:Lentulidae). Trans Am Entomol Soc. 2014;140:293–347. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.3157/061.140.0116" target="_blank" class="ext-link"><span class="RefSource">10.3157/061.140.0116</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3157/061.140.0116"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Twenty-Six%20New%20Species%20of%20Grasshoppers%20from%20the%20Southern%20African%20Arid%20Zone%20%28Orthoptera%20Acridoidea%3ALentulidae%29&amp;author=D.%20Otte&amp;journal=Trans%20Am%20Entomol%20Soc&amp;volume=140&amp;pages=293-347&amp;publication_year=2014&amp;doi=10.3157%2F061.140.0116"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rach J, Desalle R, Sarkar IN, Schierwater B, Hadrys H. Character-based DNA barcoding allows discrimination of genera, species and populations in Odonata. Proc Biol Sci. 2008;275:237–47. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1098/rspb.2007.1290" target="_blank" class="ext-link"><span class="RefSource">10.1098/rspb.2007.1290</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rspb.2007.1290"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17999953"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2212734"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Character-based%20DNA%20barcoding%20allows%20discrimination%20of%20genera%2C%20species%20and%20populations%20in%20Odonata&amp;author=J.%20Rach&amp;author=R.%20Desalle&amp;author=IN.%20Sarkar&amp;author=B.%20Schierwater&amp;author=H.%20Hadrys&amp;journal=Proc%20Biol%20Sci&amp;volume=275&amp;pages=237-47&amp;publication_year=2008&amp;doi=10.1098%2Frspb.2007.1290"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brown HD. Armstrongium - an unusual new sylvan genus of Lentulidae (Orthoptera: Acridoidea) from KwaZulu-Natal Province, South Africa. Annals of the Ditsong National Museum of Natural History. 2014;49:33–50.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Armstrongium%20-%20an%20unusual%20new%20sylvan%20genus%20of%20Lentulidae%20%28Orthoptera%3A%20Acridoidea%29%20from%20KwaZulu-Natal%20Province%2C%20South%20Africa&amp;author=HD.%20Brown&amp;journal=Annals%20of%20the%20Ditsong%20National%20Museum%20of%20Natural%20History&amp;volume=49&amp;pages=33-50&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Switala AK, Sole CL, Scholtz CH. Phylogeny, historical biogeography and divergence time estimates of the genus Colophon Gray (Coleoptera: Lucanidae). Invert Systematics. 2014;28:326. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1071/IS13054" target="_blank" class="ext-link"><span class="RefSource">10.1071/IS13054</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1071/IS13054"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeny%2C%20historical%20biogeography%20and%20divergence%20time%20estimates%20of%20the%20genus%20Colophon%20Gray%20%28Coleoptera%3A%20Lucanidae%29&amp;author=AK.%20Switala&amp;author=CL.%20Sole&amp;author=CH.%20Scholtz&amp;journal=Invert%20Systematics&amp;volume=28&amp;pages=326&amp;publication_year=2014&amp;doi=10.1071%2FIS13054"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ellis AG, Anthony Verboom G, van der Niet T, Johnson SD, Peter LH. Speciation and extinction in the Greater Cape Floristic Region. In: Allsopp N, Colville JF, Verboom GA, editors. Fynbos: Ecology, evolution, and conservation of a megadiverse region. Oxford, New York: Oxford University Press; 2014. p. 119–41. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/acprof%3Aoso/9780199679584.003.0006" target="_blank" class="ext-link"><span class="RefSource">10.1093/acprof:oso/9780199679584.003.0006</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/acprof%3Aoso/9780199679584.003.0006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Speciation%20and%20extinction%20in%20the%20Greater%20Cape%20Floristic%20Region&amp;author=AG.%20Ellis&amp;author=G.%20Anthony%20Verboom&amp;author=T.%20Niet&amp;author=SD.%20Johnson&amp;author=LH.%20Peter&amp;pages=119-41&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Daniels SR, Picker MD, Cowlin RM, Hammer ML. Unravelling evolutionary lineages among South African velvet worms (Onychophora: Peripatopsis) provides evidence for widespread cryptic speciation. Biol J Linn Soc. 2009;97:200–16. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1111/j.1095-8312.2009.01205.x" target="_blank" class="ext-link"><span class="RefSource">10.1111/j.1095-8312.2009.01205.x</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1095-8312.2009.01205.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Unravelling%20evolutionary%20lineages%20among%20South%20African%20velvet%20worms%20%28Onychophora%3A%20Peripatopsis%29%20provides%20evidence%20for%20widespread%20cryptic%20speciation&amp;author=SR.%20Daniels&amp;author=MD.%20Picker&amp;author=RM.%20Cowlin&amp;author=ML.%20Hammer&amp;journal=Biol%20J%20Linn%20Soc&amp;volume=97&amp;pages=200-16&amp;publication_year=2009&amp;doi=10.1111%2Fj.1095-8312.2009.01205.x"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">de Jager ML, Ellis AG. The influence of pollinator phylogeography and mate preference on floral divergence in a sexually deceptive daisy. Evolution. 2013;67:1706–14. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1111/evo.12070" target="_blank" class="ext-link"><span class="RefSource">10.1111/evo.12070</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/evo.12070"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23730763"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20influence%20of%20pollinator%20phylogeography%20and%20mate%20preference%20on%20floral%20divergence%20in%20a%20sexually%20deceptive%20daisy&amp;author=ML.%20Jager&amp;author=AG.%20Ellis&amp;journal=Evolution&amp;volume=67&amp;pages=1706-14&amp;publication_year=2013&amp;doi=10.1111%2Fevo.12070"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tolley K, Bowie R, Measey GJ, Price BJ, Forest F. The shifting landscapes of genes since the Pliocene: terrestrial phylogeography in the Greater Cape Floristic Region. In: Allsopp N, Colville JF, Verboom GA, editors. Fynbos: Ecology, evolution, and conservation of a megadiverse region. Oxford, New York: Oxford University Press; 2014. p. 142–63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/acprof%3Aoso/9780199679584.003.0007"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20shifting%20landscapes%20of%20genes%20since%20the%20Pliocene%3A%20terrestrial%20phylogeography%20in%20the%20Greater%20Cape%20Floristic%20Region&amp;author=K.%20Tolley&amp;author=R.%20Bowie&amp;author=GJ.%20Measey&amp;author=BJ.%20Price&amp;author=F.%20Forest&amp;pages=142-63&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Allsopp N, Colville JF, Verboom GA, editors. Fynbos: Ecology, evolution, and conservation of a megadiverse region. Oxford, New York: Oxford University Press; 2014.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fynbos%3A%20Ecology%2C%20evolution%2C%20and%20conservation%20of%20a%20megadiverse%20region&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Song H, Amédégnato C, Cigliano MM, Desutter-Grandcolas L, Heads SW, Huang Y, et al. 300 million years of diversification: Elucidating the patterns of orthopteran evolution based on comprehensive taxon and gene sampling. Cladistics. 2015:1–31. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1111/cla.12116" target="_blank" class="ext-link"><span class="RefSource">10.1111/cla.12116</span></a></span>.</span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0. Mol Biol Evol. 2013;30:2725–9. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/molbev/mst197" target="_blank" class="ext-link"><span class="RefSource">10.1093/molbev/mst197</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/mst197"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24132122"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3840312"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MEGA6%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;journal=Mol%20Biol%20Evol&amp;volume=30&amp;pages=2725-9&amp;publication_year=2013&amp;doi=10.1093%2Fmolbev%2Fmst197"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lanfear R, Calcott B, Ho SYW, Guindon S. Partitionfinder: combined selection of partitioning schemes and substitution models for phylogenetic analyses. Mol Biol Evol. 2012;29:1695–701. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/molbev/mss020" target="_blank" class="ext-link"><span class="RefSource">10.1093/molbev/mss020</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/mss020"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22319168"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Partitionfinder%3A%20combined%20selection%20of%20partitioning%20schemes%20and%20substitution%20models%20for%20phylogenetic%20analyses&amp;author=R.%20Lanfear&amp;author=B.%20Calcott&amp;author=SYW.%20Ho&amp;author=S.%20Guindon&amp;journal=Mol%20Biol%20Evol&amp;volume=29&amp;pages=1695-701&amp;publication_year=2012&amp;doi=10.1093%2Fmolbev%2Fmss020"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huelsenbeck JP, Ronquist F. MRBAYES: Bayesian inference of phylogenetic trees. 2001;17:754–5.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Huelsenbeck%20JP%2C%20Ronquist%20F.%20MRBAYES%3A%20Bayesian%20inference%20of%20phylogenetic%20trees.%202001%3B17%3A754%E2%80%935."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ronquist F, Huelsenbeck JP. MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics. 2003;19:1572–4. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/bioinformatics/btg180" target="_blank" class="ext-link"><span class="RefSource">10.1093/bioinformatics/btg180</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg180"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12912839"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MrBayes%203%3A%20Bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;author=F.%20Ronquist&amp;author=JP.%20Huelsenbeck&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1572-4&amp;publication_year=2003&amp;doi=10.1093%2Fbioinformatics%2Fbtg180"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rambaut A. FigTree 1.4.2. 2015. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://tree.bio.ed.ac.uk/software/figtree" target="_blank" class="ext-link"><span class="RefSource">http://tree.bio.ed.ac.uk/software/figtree</span></a></span>; 2015.</span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S. MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. Mol Biol Evol. 2011;28:2731–9. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/molbev/msr121" target="_blank" class="ext-link"><span class="RefSource">10.1093/molbev/msr121</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msr121"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21546353"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3203626"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;journal=Mol%20Biol%20Evol&amp;volume=28&amp;pages=2731-9&amp;publication_year=2011&amp;doi=10.1093%2Fmolbev%2Fmsr121"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kocher TD, Thomas WK, Meyer A, Edwards SV, Pääbo S, Villablanca FX, et al. Dynamics of mitochondrial DNA evolution in animals: Amplification and sequencing with conserved primers. Proc Natl Acad Sci. 1989;86:6196–200.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.86.16.6196"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2762322"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC297804"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Dynamics%20of%20mitochondrial%20DNA%20evolution%20in%20animals%3A%20Amplification%20and%20sequencing%20with%20conserved%20primers&amp;author=TD.%20Kocher&amp;author=WK.%20Thomas&amp;author=A.%20Meyer&amp;author=SV.%20Edwards&amp;author=S.%20P%C3%A4%C3%A4bo&amp;author=FX.%20Villablanca&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=86&amp;pages=6196-200&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vogler R, Desalle R. Phylogeographic Patterns in Coastal North American Tiger Beetles (<em class="EmphasisTypeItalic">Cicindela dorsalis</em> Say) Inferred from Mitochondrial DNA Sequences. Evolution. 1993;47:1192–201.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2409985"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeographic%20Patterns%20in%20Coastal%20North%20American%20Tiger%20Beetles%20%28Cicindela%20dorsalis%20Say%29%20Inferred%20from%20Mitochondrial%20DNA%20Sequences&amp;author=R.%20Vogler&amp;author=R.%20Desalle&amp;journal=Evolution&amp;volume=47&amp;pages=1192-201&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pashley D, Ke LD. Sequence evolution in the ribosomal and ND-1 genes in Lepidoptera: implications for phylogenetic analyses. Mol Biol Evol. 1992;9:1061–75.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1435234"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequence%20evolution%20in%20the%20ribosomal%20and%20ND-1%20genes%20in%20Lepidoptera%3A%20implications%20for%20phylogenetic%20analyses&amp;author=D.%20Pashley&amp;author=LD.%20Ke&amp;journal=Mol%20Biol%20Evol&amp;volume=9&amp;pages=1061-75&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Matenaaretal" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Matenaaretal">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Matenaaretal"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">© Matenaar et al. 2016</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-015-0005-6" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-015-0005-6">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-015-0005-6" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-015-0005-6?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-015-0005-6 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-015-0005-6 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-015-0005-6 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-015-0005-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-015-0005-6">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-015-0005-6"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-015-0005-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-015-0005-6">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-015-0005-6"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-015-0005-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-015-0005-6">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-015-0005-6"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-015-0005-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-015-0005-6">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-015-0005-6"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-015-0005-6">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-015-0005-6">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-015-0005-6"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-015-0005-6/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-015-0005-6">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-015-0005-6">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-015-0005-6" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-015-0005-6&amp;volume=153&amp;issue=1&amp;title=A+preliminary+phylogeny+of+the+South+African+Lentulidae&amp;author_name=Daniela+Matenaar&amp;start_page=1&amp;end_page=8">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-015-0005-6" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
