<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Genetic diversity of Bletilla striata assessed by SCoT and IRAP markers | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Bletilla striata is a well-known traditional Chinese herb with varieties of functions. In China, the natural resources of Bletilla striata have been severely damaged because of the excessive exploitation and destruction of natural habitats. The aim of present study was to provide a reference for fully exploiting and utilizing the germplasm resources of Bletilla striata. The genetic diversity of 50 varieties of Bletilla striata from different area in China was analyzed by SCoT and IRAP molecular marker technique. A total of 209 bands were amplified by 20 groups of SCoT primers, of which 201 (96.17%) were polymorphic, and 47 polymorphic bands (94%) were observed in 50 bands amplified by 8 groups of IRAP primers. The 50 populations of Bletilla striata were divided into two major groups by SCoT and IRAP at the genetic similarity coefficient value of 0.60 and 0.68 individually. The partition of clusters in the unweighted pair group method with arithmetic mean dendrogram and principal coordinate analysis plot based on the SCoT and IRAP markers was similar. Results indicated the abundant genetic diversity of Bletilla striata among different areas. Our results will provide useful information for resource protection.">
<meta name="description" content="Bletilla striata is a well-known traditional Chinese herb with varieties of functions. In China, the natural resources of Bletilla striata have been severely damaged because of the excessive exploitation and destruction of natural habitats. The aim of present study was to provide a reference for fully exploiting and utilizing the germplasm resources of Bletilla striata. The genetic diversity of 50 varieties of Bletilla striata from different area in China was analyzed by SCoT and IRAP molecular marker technique. A total of 209 bands were amplified by 20 groups of SCoT primers, of which 201 (96.17%) were polymorphic, and 47 polymorphic bands (94%) were observed in 50 bands amplified by 8 groups of IRAP primers. The 50 populations of Bletilla striata were divided into two major groups by SCoT and IRAP at the genetic similarity coefficient value of 0.60 and 0.68 individually. The partition of clusters in the unweighted pair group method with arithmetic mean dendrogram and principal coordinate analysis plot based on the SCoT and IRAP markers was similar. Results indicated the abundant genetic diversity of Bletilla striata among different areas. Our results will provide useful information for resource protection.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Genetic diversity of Bletilla striata assessed by SCoT and IRAP markers">
<meta name="dc.source" content="Hereditas 2018 155:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2018-11-12">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2018 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="Bletilla striata is a well-known traditional Chinese herb with varieties of functions. In China, the natural resources of Bletilla striata have been severely damaged because of the excessive exploitation and destruction of natural habitats. The aim of present study was to provide a reference for fully exploiting and utilizing the germplasm resources of Bletilla striata. The genetic diversity of 50 varieties of Bletilla striata from different area in China was analyzed by SCoT and IRAP molecular marker technique. A total of 209 bands were amplified by 20 groups of SCoT primers, of which 201 (96.17%) were polymorphic, and 47 polymorphic bands (94%) were observed in 50 bands amplified by 8 groups of IRAP primers. The 50 populations of Bletilla striata were divided into two major groups by SCoT and IRAP at the genetic similarity coefficient value of 0.60 and 0.68 individually. The partition of clusters in the unweighted pair group method with arithmetic mean dendrogram and principal coordinate analysis plot based on the SCoT and IRAP markers was similar. Results indicated the abundant genetic diversity of Bletilla striata among different areas. Our results will provide useful information for resource protection.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2018-11-12">
<meta name="prism.volume" content="155">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="35">
<meta name="prism.copyright" content="2018 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0074-4">
<meta name="prism.doi" content="doi:10.1186/s41065-018-0074-4">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0074-4">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0074-4">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Genetic diversity of Bletilla striata assessed by SCoT and IRAP markers">
<meta name="citation_volume" content="155">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2018/12">
<meta name="citation_online_date" content="2018/11/12">
<meta name="citation_firstpage" content="35">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-018-0074-4">
<meta name="DOI" content="10.1186/s41065-018-0074-4">
<meta name="citation_doi" content="10.1186/s41065-018-0074-4">
<meta name="dc.creator" content="Yan Guo">
<meta name="dc.creator" content="Lina Zhai">
<meta name="dc.creator" content="Hao Long">
<meta name="dc.creator" content="Nipi Chen">
<meta name="dc.creator" content="Chengxian Gao">
<meta name="dc.creator" content="Zhishan Ding">
<meta name="dc.creator" content="Bo Jin">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=Am J Pot Res; citation_title=Efficiency of arbitrarily amplified dominant markers (SCOT, ISSR and RAPD) for diagnostic fingerprinting in tetraploid potato; citation_author=MG Ahmad, P Peter, P Zsolt, T Janos; citation_volume=88; citation_publication_date=2011; citation_pages=226-237; citation_doi=10.1007/s12230-011-9187-2; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Genetic variability and structure of Quercus brantii assessed by ISSR, IRAP and SCoT markers; citation_author=L Alikhani, M-S Rahmani, S Naghi, B Hedieh, KK Abdollah; citation_volume=552; citation_publication_date=2014; citation_pages=176-183; citation_doi=10.1016/j.gene.2014.09.034; citation_id=CR2">
<meta name="citation_reference" content="citation_journal_title=AJHG; citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; citation_author=D Botstein, RL White, M Skolnick, RW Davis; citation_volume=32; citation_publication_date=1980; citation_pages=314-331; citation_id=CR3">
<meta name="citation_reference" content="Chinese Herbalism Editorial Board. Zhonghua Bencao. Shanghai: Shanghai Scientific &amp; Technical Publishers; 1999. p. 7803.">
<meta name="citation_reference" content="citation_journal_title=Plant Mol Biol Rep; citation_title=Start codon targeted (SCoT) polymorphism: a simple, novel DNA marker technique for generating gene-targeted markers in plants; citation_author=BCY Collard, DJ Mackill; citation_volume=27; citation_publication_date=2009; citation_pages=86-93; citation_doi=10.1007/s11105-008-0060-5; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=IRAP and REMAP: two new retrotransposon-based DNA fingerprinting techniques; citation_author=R Kalendar, T Grob, M Regina, A Suoniemi, A Schulman; citation_volume=98; citation_publication_date=1999; citation_pages=704-711; citation_doi=10.1007/s001220051124; citation_id=CR6">
<meta name="citation_reference" content="citation_title=Ethnobotany of the orchidaceae; citation_inbook_title=Orchid Biology; citation_publication_date=1984; citation_pages=256; citation_id=CR7; citation_author=LJ Lawler; citation_publisher=Reviews and Perspectives. Cornell University Press, NY">
<meta name="citation_reference" content="citation_journal_title=Biotechnol Bull; citation_title=Analysis of genetic diversity in Saccharina japonica germplasm resources; citation_author=Y Li, YL Liu, CJ Cui, XJ Li, GJ Liang, J Peng, PPRAPD Tian; citation_volume=32; citation_publication_date=2016; citation_pages=151-158; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Chin Tradit Herb Drugs; citation_title=Research survey and countermeasure on resources utilization in stem tuber of Bletilla striata; citation_author=R Li, ZZ Wang; citation_volume=37; citation_publication_date=2006; citation_pages=1751-1755; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Conservation genetics: beyond the maintenance of marker diversity; citation_author=BG Milligan, J Leebens-Mack, AE Strand; citation_volume=12; citation_publication_date=1994; citation_pages=844-855; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Breed Sci; citation_title=Development of genomic and EST-SSR markers in radish (Raphanus sativus L.); citation_author=R Nakatsuji, T Hashida, N Matsumoto, M Tsuro, N Kubo, M Hirai; citation_volume=61; citation_publication_date=2011; citation_pages=413-419; citation_doi=10.1270/jsbbs.61.413; citation_id=CR11">
<meta name="citation_reference" content="citation_title=Medicinal plants of east and Southeast Asia: attributed properties and uses; citation_publication_date=1990; citation_id=CR12; citation_author=LM Perry; citation_author=J Metzger; citation_publisher=MIT Press">
<meta name="citation_reference" content="citation_journal_title=Biol Plant; citation_title=Genetic fidelity assessment of in vitro-regenerated plants of Albizia julibrissin using SCoT and IRAP fingerprinting; citation_author=M-S Rahmani, M Paula, NS Pijut, N Mona; citation_volume=51; citation_publication_date=2015; citation_pages=407-419; citation_id=CR13">
<meta name="citation_reference" content="citation_title=Resources and medicinal utilization of Bletilla (orchidaceae); citation_publication_date=2009; citation_id=CR14; citation_author=J Shi; citation_publisher=Dissertation">
<meta name="citation_reference" content="citation_journal_title=Chiese Horti Culture Abstracts; citation_title=To Surve and analyze the market of Bletilla striata in China; citation_author=J Shi, YB Luo, XQ Song; citation_volume=8; citation_publication_date=2010; citation_pages=48-50; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=Biochem Genet; citation_title=Potential start codon targeted (SCoT) and inter-retrotransposon amplified polymorphism (IRAP) markers for evaluation of genetic diversity and conservation of wild Pistacia species population; citation_author=K Sorkheh, A Nazanin, E Sezai; citation_volume=54; citation_publication_date=2016; citation_pages=368-387; citation_doi=10.1007/s10528-016-9725-1; citation_id=CR16">
<meta name="citation_reference" content="Wu JN. An illustrated Chinese Materia Medica. New York: Oxford University Press; 2005.">
<meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Genetic diversity analysis among and within populations of Pogostemon cablin from China with ISSR and SRAP markers; citation_author=YG Wu, QS Guo, JC He, YF Lin, LJ Luo, GD Liu; citation_volume=38; citation_publication_date=2010; citation_pages=63-72; citation_doi=10.1016/j.bse.2009.12.006; citation_id=CR18">
<meta name="citation_reference" content="citation_title=“POPGENE, the user friendly shareware for population genetic analysis version1. 31,” University of Alberta and Centre For International Forestry Research; citation_publication_date=1999; citation_id=CR19; citation_author=FC Yeh; citation_author=RC Yang; citation_author=TBJ Boyle; citation_author=ZH Ye; citation_author=JX Mao">
<meta name="citation_reference" content="citation_journal_title=Boreal -OcidentSin; citation_title=SSR fingerprinting of Bletilla striata Rchb.f. Germplasm and its relatives or congeners. Acta bot; citation_author=TH Zhou, J Li, JX Ding, H Yang, SL Chen; citation_volume=37; citation_publication_date=2017; citation_pages=0673-0681; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=Mol Plant Breed; citation_title=Study on DNA isolation from polysaccharides-rich transgenic dendrobium; citation_author=MB Zhang, GQ Fan, JM Chen, P Chen; citation_volume=7; citation_publication_date=2009; citation_pages=209-214; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Chin Arch Tradit Chin Med; citation_title=Present status and sustainable development of Rhizoma Bletillae industry; citation_author=YW Zhang, FS Jiang, Y Wang, ZS Ding; citation_volume=30; citation_publication_date=2012; citation_pages=2264-2267; citation_id=CR22">
<meta name="citation_author" content="Yan Guo">
<meta name="citation_author_institution" content="College of Life Science, Zhejiang Chinese Medical University, Hangzhou, China">
<meta name="citation_author" content="Lina Zhai">
<meta name="citation_author_institution" content="College of Life Science, Zhejiang Chinese Medical University, Hangzhou, China">
<meta name="citation_author" content="Hao Long">
<meta name="citation_author_institution" content="College of Life Science, Zhejiang Chinese Medical University, Hangzhou, China">
<meta name="citation_author" content="Nipi Chen">
<meta name="citation_author_institution" content="College of Life Science, Zhejiang Chinese Medical University, Hangzhou, China">
<meta name="citation_author" content="Chengxian Gao">
<meta name="citation_author_institution" content="College of Life Science, Zhejiang Chinese Medical University, Hangzhou, China">
<meta name="citation_author" content="Zhishan Ding">
<meta name="citation_author_institution" content="College of Medical Technology, Zhejiang Chinese Medical University, Hangzhou, China">
<meta name="citation_author" content="Bo Jin">
<meta name="citation_author_institution" content="College of Life Science, Zhejiang Chinese Medical University, Hangzhou, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0074-4',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0074-4","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Bletilla striata;Genetic diversity;SCoT;IRAP;Cluster analysis"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic diversity of Bletilla striata assessed by SCoT and IRAP markers","publishedAt":1541980800000,"publishedAtDate":"2018-11-12","author":["Yan Guo","Lina Zhai","Hao Long","Nipi Chen","Chengxian Gao","Zhishan Ding","Bo Jin"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0074-4">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0074-4">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Genetic diversity of Bletilla striata assessed by SCoT and IRAP markers">
<meta property="og:description" content="Bletilla striata is a well-known traditional Chinese herb with varieties of functions. In China, the natural resources of Bletilla striata have been severely damaged because of the excessive exploitation and destruction of natural habitats. The aim of present study was to provide a reference for fully exploiting and utilizing the germplasm resources of Bletilla striata. The genetic diversity of 50 varieties of Bletilla striata from different area in China was analyzed by SCoT and IRAP molecular marker technique. A total of 209 bands were amplified by 20 groups of SCoT primers, of which 201 (96.17%) were polymorphic, and 47 polymorphic bands (94%) were observed in 50 bands amplified by 8 groups of IRAP primers. The 50 populations of Bletilla striata were divided into two major groups by SCoT and IRAP at the genetic similarity coefficient value of 0.60 and 0.68 individually. The partition of clusters in the unweighted pair group method with arithmetic mean dendrogram and principal coordinate analysis plot based on the SCoT and IRAP markers was similar. Results indicated the abundant genetic diversity of Bletilla striata among different areas. Our results will provide useful information for resource protection.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0074-4;kwrd=Bletilla striata,Genetic diversity,SCoT,IRAP,Cluster analysis;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0074-4&amp;kwrd=Bletilla%20striata,Genetic%20diversity,SCoT,IRAP,Cluster%20analysis&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0074-4&amp;kwrd=Bletilla%20striata,Genetic%20diversity,SCoT,IRAP,Cluster%20analysis&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec8" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Material and methods">Material and methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic diversity of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> assessed by SCoT and IRAP markers</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true">
<ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yan Guo</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lina Zhai</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Hao Long</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Nipi Chen</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Chengxian Gao</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zhishan Ding</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Bo Jin</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:jinbo@zcmu.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-7701-6102" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a>
</li>
</ul>
<div class="AffiliationLegend"><div class="AffiliationLegendLine">
<span class="EqualContributionSymbol">†</span>Contributed equally</div></div>
</div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:35</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0074-4">https://doi.org/10.1186/s41065-018-0074-4</a></p>
<p class="Copyright">©  The Author(s) 2018</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>21 July 2018</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>28 October 2018</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">12 November 2018</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para"><em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> is a well-known traditional Chinese herb with varieties of functions. In China, the natural resources of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> have been severely damaged because of the excessive exploitation and destruction of natural habitats. The aim of present study was to provide a reference for fully exploiting and utilizing the germplasm resources of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em>.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">The genetic diversity of 50 varieties of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> from different area in China was analyzed by SCoT and IRAP molecular marker technique. A total of 209 bands were amplified by 20 groups of SCoT primers, of which 201 (96.17%) were polymorphic, and 47 polymorphic bands (94%) were observed in 50 bands amplified by 8 groups of IRAP primers. The 50 populations of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> were divided into two major groups by SCoT and IRAP at the genetic similarity coefficient value of 0.60 and 0.68 individually. The partition of clusters in the unweighted pair group method with arithmetic mean dendrogram and principal coordinate analysis plot based on the SCoT and IRAP markers was similar.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Results indicated the abundant genetic diversity of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> among different areas. Our results will provide useful information for resource protection.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Bletilla striata</em>
              </li>
<li class="c-keywords__item" itemprop="about">Genetic diversity</li>
<li class="c-keywords__item" itemprop="about">SCoT</li>
<li class="c-keywords__item" itemprop="about">IRAP</li>
<li class="c-keywords__item" itemprop="about">Cluster analysis</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para"><em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> is a well-known traditional Chinese herb, which was first described in Shennong BenCao Jing (Shennong’s Materia Medica) 2000 years ago. In China, there are four unique populations: <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em>, <em xmlns="" class="EmphasisTypeItalic">Bletilla formosana</em>, <em xmlns="" class="EmphasisTypeItalic">Bletilla ochracea</em> and <em xmlns="" class="EmphasisTypeItalic">Bletilla sinensis</em> [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Traditional Chinese Medicine holds that <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> is capable of restraining blood leakage, stopping bleeding, dispersing swelling and promoting tissue regeneration. Thus it could be effectively applied in the treatment of hematemesis, hemoptysis, traumatic bleeding, chapped kin, and ulcerative carbuncle [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Besides, <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> can be added in medicated diets or drinks when stewed together with chicken or duck and extracted by boiling water or brewed as wine material [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Additionally, the non-medical uses of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> include rubbing its mucilaginous roots in inkstones with vermilion for writing [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], and another use is as an insecticide [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. In industry, Bletilla extract is used as a coating agent and cosmetic additive [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Moreover, the plant also has very high decorative value [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. The wide uses of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> results in demand exceeding supply.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">In China, the natural resources of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> have been severely damaged because of the excessive exploitation and destruction of natural habitats [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]<em xmlns="" class="EmphasisTypeItalic">.</em> More than 10 years ago, <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> was listed as one of the key protected wild medicinal plants (<span class="ExternalRef"><a href="http://rep.iplant.cn" target="_blank" class="ext-link"><span class="RefSource">http://rep.iplant.cn</span></a></span>/). The price of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> has soared 20-fold in the past 10 years [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. In order to solve the problem of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> resources, the artificial cultivation of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> and its related species has been developed in the majority of regions in China [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. More importantly, research on strengthening the selection and breeding of excellent varieties of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> also has been launched [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. As part of the efforts to protect the precious plant and explore the best way to utilize the entire plant, our laboratory established good practice for the genetic relationships of <em xmlns="" class="EmphasisTypeItalic">Bletilla</em> until now.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Molecular markers are one of the most important methods to allow cultivar identification. They are widespread used on account of their simple operation and high detection efficiency [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. The different DNA-based molecular markers unfold plant variability directly at genetic levels with reliable data required for the estimation of genetic diversity [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>], such as inter-simple sequence repeats (ISSR), inter-retrotransposon amplified polymorphism (IRAP), or start codon targeted (SCoT) polymorphism [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. In this investigation, SCoT and IRAP were used to assess genetic constancy of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata.</em></p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">The objectives of this study are to investigate the genetic relationships among some <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> species in China and construct a molecular phylogenetic tree, which provides a theoretical basis for protection, selection and development.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">The oligo nucleotide sequences of SCoT and IRAP primers are summarized in Table <span class="InternalRef"><a href="#Tab1">1</a></span>.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">All molecular marker primers sequence of SCoT and IRAP used in this research</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<thead><tr class="header">
<th colspan="4"><p xmlns="" class="SimplePara">SCoT primer</p></th>
<th colspan="2"><p xmlns="" class="SimplePara">IRAP primer</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Code</p></td>
<td><p xmlns="" class="SimplePara">Sequence(5’to3’)</p></td>
<td><p xmlns="" class="SimplePara">Code</p></td>
<td><p xmlns="" class="SimplePara">Sequence(5’to3’)</p></td>
<td><p xmlns="" class="SimplePara">Code</p></td>
<td><p xmlns="" class="SimplePara">Sequence(5’to3’)</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC3</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCG</p></td>
<td><p xmlns="" class="SimplePara">SC19</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGGC</p></td>
<td><p xmlns="" class="SimplePara">LIRE3-F</p></td>
<td><p xmlns="" class="SimplePara">CCTCAAAGCTCTCTTCTCCTTC</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC4</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCT</p></td>
<td><p xmlns="" class="SimplePara">SC20</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCG</p></td>
<td><p xmlns="" class="SimplePara">LIRE3-R</p></td>
<td><p xmlns="" class="SimplePara">CCTAGAGTTGTACATTTACATT</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC5</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGA</p></td>
<td><p xmlns="" class="SimplePara">SC21</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCCACA</p></td>
<td><p xmlns="" class="SimplePara">LIRE-Tar-F</p></td>
<td><p xmlns="" class="SimplePara">CATYATCRGTGGAGCTCT</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC6</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGC</p></td>
<td><p xmlns="" class="SimplePara">SC22</p></td>
<td><p xmlns="" class="SimplePara">AACCATGGCTACCACCAC</p></td>
<td><p xmlns="" class="SimplePara">LIRE-Tar-R</p></td>
<td><p xmlns="" class="SimplePara">AATCATYYYGAGAGCTCTCC</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC7</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGG</p></td>
<td><p xmlns="" class="SimplePara">SC23</p></td>
<td><p xmlns="" class="SimplePara">CACCATGGCTACCACCAG</p></td>
<td><p xmlns="" class="SimplePara">LIRE-del-F1</p></td>
<td><p xmlns="" class="SimplePara">TTGAAACCACYAGCTCAAGGTA</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC9</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCAGCA</p></td>
<td><p xmlns="" class="SimplePara">SC24</p></td>
<td><p xmlns="" class="SimplePara">CACCATGGCTACCACCAT</p></td>
<td><p xmlns="" class="SimplePara">LIRE-del-F2</p></td>
<td><p xmlns="" class="SimplePara">TATAAAATGTCRGGTCGTGA</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC10</p></td>
<td><p xmlns="" class="SimplePara">CAACAATGGCTACCAGCC</p></td>
<td><p xmlns="" class="SimplePara">SC25</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGGG</p></td>
<td><p xmlns="" class="SimplePara">LIRE-del-R</p></td>
<td><p xmlns="" class="SimplePara">TCATGRAGRTGATARAGWYTAACC</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC11</p></td>
<td><p xmlns="" class="SimplePara">AAGCAATGGCTACCACCA</p></td>
<td><p xmlns="" class="SimplePara">SC26</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGTC</p></td>
<td><p xmlns="" class="SimplePara">DO-F</p></td>
<td><p xmlns="" class="SimplePara">TCCACAAGGCTATCTATG</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC12</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCAACG</p></td>
<td><p xmlns="" class="SimplePara">SC27</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGTG</p></td>
<td><p xmlns="" class="SimplePara">DO-R</p></td>
<td><p xmlns="" class="SimplePara">GAGGGATTGGACTTACTG</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC13</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCATCG</p></td>
<td><p xmlns="" class="SimplePara">SC28</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCA</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC14</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCACGC</p></td>
<td><p xmlns="" class="SimplePara">SC30</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGGCG</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC15</p></td>
<td><p xmlns="" class="SimplePara">ACGACATGGCGACCGCGA</p></td>
<td><p xmlns="" class="SimplePara">SC31</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCT</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC16</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGAC</p></td>
<td><p xmlns="" class="SimplePara">SC32</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCAC</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC17</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGAG</p></td>
<td><p xmlns="" class="SimplePara">SC33</p></td>
<td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCAG</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC18</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCC</p></td>
<td><p xmlns="" class="SimplePara">SC34</p></td>
<td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCA</p></td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table></div></figure>
</div>
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic diversity revealed by SCoT polymorphic markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Twenty groups of primers were used to detect genetic polymorphisms in the samples. The primers produced 209 clear SCoT bands, including 201 polymorphism bands. The total number of scored bands varied from 6 (for SC8 + SC11) to 14 (for SC16 + SC21, SC17 + SC20) with an average of 10.5 bands per primer (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The number of the polymorphic bands ranging from 6 (for SC8 + SC11) to 13 (for SC16 + SC21, SC17 + SC20, SC14 + SC23) with a mean of 96.17% was polymorphic bands per primer (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The PIC value was ranging from 0.92 to 0.99, with a mean value of 0.96, indicated that the SCoT primers used in the study were effective and polymorphic (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The Nei’s index varied from 0.28 (for SC18 + SC19) to 0.42 (for SC14 + SC33) with a mean value of 0.37 (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). A dendrogram (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>) was constructed based on the similarity matrix (data not shown). The highest genetic similarity coefficient (0.87) was found between 38 and 39. With a genetic similarity coefficient value of 0.60, the 50 varieties were divided into two major groups. Group I included twenty-five varieties; the all other varieties were classified into Group II. And the results of cophenetic correlation analysis are shown below (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>) (the correlation coefficient is <em xmlns="" class="EmphasisTypeItalic">r</em> = 0.752, indicating that the clustering results are good).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Characteristics of SCoT and IRAP banding profiles produced in the studied <em class="EmphasisTypeItalic">Bletilla striata</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup><col></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Primer name</p></th>
<th char="."><p xmlns="" class="SimplePara">TB</p></th>
<th char="."><p xmlns="" class="SimplePara">PB</p></th>
<th char="."><p xmlns="" class="SimplePara">PP(%)</p></th>
<th char="."><p xmlns="" class="SimplePara">PIC</p></th>
<th char="."><p xmlns="" class="SimplePara">Nei’s</p></th>
<th char="×"><p xmlns="" class="SimplePara">Fst</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">IRAP</p></td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Do-F + tar-R</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">5</p></td>
<td char="."><p xmlns="" class="SimplePara">71.43</p></td>
<td char="."><p xmlns="" class="SimplePara">0.87</p></td>
<td char="."><p xmlns="" class="SimplePara">0.45</p></td>
<td char="."><p xmlns="" class="SimplePara">0.009</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">DO-F + LIRE-del-F2</p></td>
<td char="."><p xmlns="" class="SimplePara">5</p></td>
<td char="."><p xmlns="" class="SimplePara">5</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.94</p></td>
<td char="."><p xmlns="" class="SimplePara">0.40</p></td>
<td char="."><p xmlns="" class="SimplePara">0.018</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">DO-F + LIRE-del-F1</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">8</p></td>
<td char="."><p xmlns="" class="SimplePara">88.89</p></td>
<td char="."><p xmlns="" class="SimplePara">0.96</p></td>
<td char="."><p xmlns="" class="SimplePara">0.23</p></td>
<td char="."><p xmlns="" class="SimplePara">0.012</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">LIRE-del-F1 + LIRE-Tar-R</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.98</p></td>
<td char="."><p xmlns="" class="SimplePara">0.35</p></td>
<td char="."><p xmlns="" class="SimplePara">0.023</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">LIRE3-R + LIRE-del-R</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.95</p></td>
<td char="."><p xmlns="" class="SimplePara">0.25</p></td>
<td char="."><p xmlns="" class="SimplePara">0.011</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">LIRE3-R + LIRE-TAR-R</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.97</p></td>
<td char="."><p xmlns="" class="SimplePara">0.36</p></td>
<td char="."><p xmlns="" class="SimplePara">0.023</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">LIRE3R + DO-R</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.91</p></td>
<td char="."><p xmlns="" class="SimplePara">0.38</p></td>
<td char="."><p xmlns="" class="SimplePara">0.019</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">LIRE-Tar-F + LIRE-del-F1</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td char="."><p xmlns="" class="SimplePara">4</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.85</p></td>
<td char="."><p xmlns="" class="SimplePara">0.42</p></td>
<td char="."><p xmlns="" class="SimplePara">0.012</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mean</p></td>
<td char="."><p xmlns="" class="SimplePara">50</p></td>
<td char="."><p xmlns="" class="SimplePara">47</p></td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SCOT</p></td>
<td char="."><p xmlns="" class="SimplePara">6.25</p></td>
<td char="."><p xmlns="" class="SimplePara">5.88</p></td>
<td char="."><p xmlns="" class="SimplePara">95.04</p></td>
<td char="."><p xmlns="" class="SimplePara">0.93</p></td>
<td char="."><p xmlns="" class="SimplePara">0.36</p></td>
<td char="."><p xmlns="" class="SimplePara">0.016</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC17 + SC30</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.98</p></td>
<td char="."><p xmlns="" class="SimplePara">0.36</p></td>
<td char="."><p xmlns="" class="SimplePara">0.007</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC5 + SC14</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.97</p></td>
<td char="."><p xmlns="" class="SimplePara">0.42</p></td>
<td char="."><p xmlns="" class="SimplePara">0.009</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC8 + SC11</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">6</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.92</p></td>
<td char="."><p xmlns="" class="SimplePara">0.39</p></td>
<td char="."><p xmlns="" class="SimplePara">0.012</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC14 + SC33</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">90</p></td>
<td char="."><p xmlns="" class="SimplePara">0.93</p></td>
<td char="."><p xmlns="" class="SimplePara">0.42</p></td>
<td char="."><p xmlns="" class="SimplePara">0.016</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC15 + SC32</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.97</p></td>
<td char="."><p xmlns="" class="SimplePara">0.33</p></td>
<td char="."><p xmlns="" class="SimplePara">0.016</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC16 + SC21</p></td>
<td char="."><p xmlns="" class="SimplePara">14</p></td>
<td char="."><p xmlns="" class="SimplePara">13</p></td>
<td char="."><p xmlns="" class="SimplePara">92.86</p></td>
<td char="."><p xmlns="" class="SimplePara">0.96</p></td>
<td char="."><p xmlns="" class="SimplePara">0.41</p></td>
<td char="."><p xmlns="" class="SimplePara">0.015</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC17 + SC20</p></td>
<td char="."><p xmlns="" class="SimplePara">14</p></td>
<td char="."><p xmlns="" class="SimplePara">13</p></td>
<td char="."><p xmlns="" class="SimplePara">92.86</p></td>
<td char="."><p xmlns="" class="SimplePara">0.92</p></td>
<td char="."><p xmlns="" class="SimplePara">0.33</p></td>
<td char="."><p xmlns="" class="SimplePara">0.013</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC18 + SC19</p></td>
<td char="."><p xmlns="" class="SimplePara">12</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">91.67</p></td>
<td char="."><p xmlns="" class="SimplePara">0.97</p></td>
<td char="."><p xmlns="" class="SimplePara">0.28</p></td>
<td char="."><p xmlns="" class="SimplePara">0.017</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC3 + SC34</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">7</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.94</p></td>
<td char="."><p xmlns="" class="SimplePara">0.34</p></td>
<td char="."><p xmlns="" class="SimplePara">0.021</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC4 + SC33</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">8</p></td>
<td char="."><p xmlns="" class="SimplePara">88.89</p></td>
<td char="."><p xmlns="" class="SimplePara">0.94</p></td>
<td char="."><p xmlns="" class="SimplePara">0.39</p></td>
<td char="."><p xmlns="" class="SimplePara">0.023</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC5 + SC32</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">90</p></td>
<td char="."><p xmlns="" class="SimplePara">0.97</p></td>
<td char="."><p xmlns="" class="SimplePara">0.38</p></td>
<td char="."><p xmlns="" class="SimplePara">0.017</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC6 + SC31</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.98</p></td>
<td char="."><p xmlns="" class="SimplePara">0.35</p></td>
<td char="."><p xmlns="" class="SimplePara">0.019</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC7 + SC30</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.98</p></td>
<td char="."><p xmlns="" class="SimplePara">0.41</p></td>
<td char="."><p xmlns="" class="SimplePara">0.017</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC9 + SC28</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">9</p></td>
<td char="."><p xmlns="" class="SimplePara">90</p></td>
<td char="."><p xmlns="" class="SimplePara">0.98</p></td>
<td char="."><p xmlns="" class="SimplePara">0.33</p></td>
<td char="."><p xmlns="" class="SimplePara">0.015</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC10 + SC27</p></td>
<td char="."><p xmlns="" class="SimplePara">12</p></td>
<td char="."><p xmlns="" class="SimplePara">12</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.96</p></td>
<td char="."><p xmlns="" class="SimplePara">0.39</p></td>
<td char="."><p xmlns="" class="SimplePara">0.017</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC11 + SC26</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">10</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.96</p></td>
<td char="."><p xmlns="" class="SimplePara">0.29</p></td>
<td char="."><p xmlns="" class="SimplePara">0.016</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC12 + SC25</p></td>
<td char="."><p xmlns="" class="SimplePara">12</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">91.67</p></td>
<td char="."><p xmlns="" class="SimplePara">0.95</p></td>
<td char="."><p xmlns="" class="SimplePara">0.38</p></td>
<td char="."><p xmlns="" class="SimplePara">0.019</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC13 + SC24</p></td>
<td char="."><p xmlns="" class="SimplePara">8</p></td>
<td char="."><p xmlns="" class="SimplePara">8</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.99</p></td>
<td char="."><p xmlns="" class="SimplePara">0.37</p></td>
<td char="."><p xmlns="" class="SimplePara">0.021</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC14 + SC23</p></td>
<td char="."><p xmlns="" class="SimplePara">13</p></td>
<td char="."><p xmlns="" class="SimplePara">13</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.99</p></td>
<td char="."><p xmlns="" class="SimplePara">0.34</p></td>
<td char="."><p xmlns="" class="SimplePara">0.023</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">SC15 + SC22</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">11</p></td>
<td char="."><p xmlns="" class="SimplePara">100</p></td>
<td char="."><p xmlns="" class="SimplePara">0.93</p></td>
<td char="."><p xmlns="" class="SimplePara">0.36</p></td>
<td char="."><p xmlns="" class="SimplePara">0.021</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">209</p></td>
<td char="."><p xmlns="" class="SimplePara">201</p></td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Mean</p></td>
<td char="."><p xmlns="" class="SimplePara">10.45</p></td>
<td char="."><p xmlns="" class="SimplePara">10.05</p></td>
<td char="."><p xmlns="" class="SimplePara">96.40</p></td>
<td char="."><p xmlns="" class="SimplePara">0.96</p></td>
<td char="."><p xmlns="" class="SimplePara">0.37</p></td>
<td char="."><p xmlns="" class="SimplePara">0.017</p></td>
</tr>
</tbody>
</table></div>
<div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Abbreviation</em>: <em class="EmphasisTypeItalic">TB</em> Total bands, <em class="EmphasisTypeItalic">PB</em> Polymorphic bands, <em class="EmphasisTypeItalic">pp</em> Polymorphic percentage, <em class="EmphasisTypeItalic">PIC</em> Polymorphic Information Content, <em class="EmphasisTypeItalic">Fst</em> Differentiation between populations index</p>
                    </div></figure><figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig1_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig1_HTML.png" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">UPGMA dendrogram based on SCoT data showing genetic relationship among the 50 <em class="EmphasisTypeItalic">Bletilla striata</em></p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig2_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig2_HTML.png" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">The results of cophenetic correlation analysis among 50 <em class="EmphasisTypeItalic">Bletilla striata</em> accessions based on SCoT markers</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">Genetic diversity revealed by IRAP polymorphic markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Eight groups of primers were used to detect genetic polymorphisms in the samples. The primers produced 50 clear IRAP bands, including 47 polymorphism bands. The total number of scored bands varied from 4 (for LIRE-Tar-F + LIRE-del-F1) to 9 (for DO-F + LIRE-del-F1) with an average of 6.3 bands per primer (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The number of the polymorphic bands ranging from 4 (for LIRE-Tar-F + LIRE-del-F1) to 8 (for DO-F + LIRE-del-F1) with a mean of 94% was polymorphic bands per primer (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The PIC value was ranging from 0.85 to 0.98, with a mean value of 0.93, indicated that the IRAP primers used in the study were effective and polymorphic (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The Nei’s index varied from 0.23 (for DO-F + LIRE-del-F1) to 0.43 (for Do-F + tar-R) with a mean value of 0.36 (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). A dendrogram (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>) was constructed based on the similarity matrix (data not shown). The highest genetic similarity coefficient (0.94) was found between 12, 26, 32 and 38. With a genetic similarity coefficient value of 0.78, the 50 varieties were divided into two major groups. Group I included two varieties: 19 and 42; the all other varieties were classified into Group II. And the results of cophenetic correlation analysis are shown below (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>) (the correlation coefficient is <em xmlns="" class="EmphasisTypeItalic">r</em> = 0.707, indicating that the clustering results are good).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig3_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig3_HTML.png" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig3_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig3_HTML.png" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">UPGMA dendrogram based on IRAP data showing genetic relationship among the 50 <em class="EmphasisTypeItalic">Bletilla striata</em></p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig4_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig4_HTML.png" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig4_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig4_HTML.png" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">The results of cophenetic correlation analysis among 50 <em class="EmphasisTypeItalic">Bletilla striata</em> accessions based on IRAP markers</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Analysis of PCoA</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">PCoA analysis was done to see the displacement of the accessions and to further confirm the clustering pattern obtained from the dendrogram. The result of the three-dimensional (3D) plots of PCoA analysis for SCoT marker analysis was also similar to that of clustering in the dendrogram (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>). PCoA analysis indicated the first three axes accounted for 26.2% of total variation among all the accessions studied. The first, second and third axes represented 12.9, 7.2 and 6.1% of the variation respectively. The result of the three-dimensional (3D) plots of PCoA analysis for IRAP marker analysis was similar to that of clustering in the dendrogram (Fig. <span class="InternalRef"><a href="#Fig6">6</a></span>). PCoA analysis indicated the first three axes accounted for 30.4% of total variation among all the accessions studied. The first, second and third axes represented 11.7, 10.9 and 7.8% of the variation respectively.<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig5_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig5_HTML.png" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig5_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig5_HTML.png" alt="Fig. 5"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Three-dimensional plots of PCoA indicating relationships among 50 <em class="EmphasisTypeItalic">Bletilla striata</em> accessions based on SCoT markers</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig6_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig6_HTML.png" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig6_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig6_HTML.png" alt="Fig. 6"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Three-dimensional plots of PCoA indicating relationships among 50 <em class="EmphasisTypeItalic">Bletilla striata</em> accessions based on IRAP markers</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Analysis of molecular variance (AMOVA)</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">The analysis of molecular variance (AMOVA) of the SCoT data revealed that out of the total recorded variation, 79.38% was recorded within the species level whereas 20.62% was compartmentalized amongst the species. Similar in distribution to that of SCoT, IRAP marker data accounted for 77.36% variability at inter-specific level and 22.64% at intra-specific level. Which indicated high genetic variation within populations, and using SCoT marker reveled higher variation within and among populations of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> (Table <span class="InternalRef"><a href="#Tab3">3</a></span>)<em xmlns="" class="EmphasisTypeItalic">.</em><figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Analysis of molecular variance (AMOVA) among and within species based on SCoT marker (a) and IRAP marker (b)</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup><col></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col char="."></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">Source of variation</p></th>
<th char="."><p xmlns="" class="SimplePara">df</p></th>
<th char="."><p xmlns="" class="SimplePara">Sum of squares</p></th>
<th char="."><p xmlns="" class="SimplePara">Variance components</p></th>
<th><p xmlns="" class="SimplePara">Variation (%)</p></th>
<th><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">P</em>-value</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">a</p></td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Among populations</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td char="."><p xmlns="" class="SimplePara">385.11</p></td>
<td char="."><p xmlns="" class="SimplePara">1.87054</p></td>
<td char="."><p xmlns="" class="SimplePara">20.62</p></td>
<td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">P</em> &lt; 0.001</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Within populations</p></td>
<td char="."><p xmlns="" class="SimplePara">46</p></td>
<td char="."><p xmlns="" class="SimplePara">783.35</p></td>
<td char="."><p xmlns="" class="SimplePara">21.3307</p></td>
<td char="."><p xmlns="" class="SimplePara">79.38</p></td>
<td><p xmlns="" class="SimplePara">P &lt; 0.001</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">49</p></td>
<td char="."><p xmlns="" class="SimplePara">2049.35</p></td>
<td char="."><p xmlns="" class="SimplePara">23.52211</p></td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">b</p></td>
<td char="."> </td>
<td char="."> </td>
<td char="."> </td>
<td> </td>
<td> </td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Among populations</p></td>
<td char="."><p xmlns="" class="SimplePara">3</p></td>
<td char="."><p xmlns="" class="SimplePara">365.49</p></td>
<td char="."><p xmlns="" class="SimplePara">2.66702</p></td>
<td char="."><p xmlns="" class="SimplePara">22.64</p></td>
<td><p xmlns="" class="SimplePara">P &lt; 0.001</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Within populations</p></td>
<td char="."><p xmlns="" class="SimplePara">46</p></td>
<td char="."><p xmlns="" class="SimplePara">667.90</p></td>
<td char="."><p xmlns="" class="SimplePara">27.3087</p></td>
<td char="."><p xmlns="" class="SimplePara">77.36</p></td>
<td><p xmlns="" class="SimplePara">P &lt; 0.001</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">Total</p></td>
<td char="."><p xmlns="" class="SimplePara">49</p></td>
<td char="."><p xmlns="" class="SimplePara">1329.72</p></td>
<td char="."><p xmlns="" class="SimplePara">30.41602</p></td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table></div></figure>
</div>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">In general, the genetic characteristics of a species are affected primarily by various ecological and biological traits, modes of reproduction and breeding and also by the various human mediated anthropogenic activities (Hamrick and Godt 1996; Nybom 2004). The genetic background of wild <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> becomes complex because of prolonged cross pollination and natural selection. In this study, we collected a variety of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> from China. The genetic diversity and phylogenetic relationships of the samples were analyzed using SCoT and IRAP markers. The SCoT marker systems are proved to be efficient and inexpensive ways to provide molecular data to assess genetic diversity, and it has been used successfully to determine genetic relationships for many plants ([<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]; Bhattacharyya et al. 2013; [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]) whereby primers target the short-conserved region flanking the ATG translation start codon of plant genes [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. IRAP is developed by Kalendar et al. [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] and initially designed to identify different barley (<em xmlns="" class="EmphasisTypeItalic">H. vulgare</em>) cultivars. IRAP markers involve the PCR amplification of DNA sequences between two nearby retrotransposons, using a primer designed from the LTR sequence of a retrotransposon, which were recently used extensively in studying genetic diversity, genetic relationship and germplasm management [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. In the ATG flanking region, 20 groups of SCoT primers obtained a ratio of 96.17% polymorphism. In the LTR region, 8 groups of IRAP primer combinations of polymorphic fragments obtained a ratio of 94%. The results indicated that both the SCoT and IRAP markers are equally efficient in the detection of polymorphism. However, SCoT proved to have slightly higher detection capacity i.e. in comparison to IRAP.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Sometimes, making use of only one marker for evaluating the genetic diversities among species may ignore a lot of unanswered questions [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. However, combining polymorphic information derived from different marker systems was expected to decrease the effects of their independent inaccuracies [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Genetic relationship evaluation using a combination of SCoT+IRAP would result in a more accurate and reliable analysis of genetic diversities compared with a single marker. Our report is the first to integrate SCoT and IRAP data to elucidate the genetic relationships among Bletilla species, which can serve as a basis for the establishment of an effective system for species identification and genetic diversity analysis of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em>.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The UPGMA dendrograms indicated the abundant genetic diversity of the samples. <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> collected from one province can be clustered into nearly 2 clusters. The genetic distances of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> from different provinces can be very close regardless of the flowers’ color. Therefore, combining the current status of online trade prevalence, we suspect that exchanging universally results in the serious situation of transplanting and cross planting, which may cause high genetic variation among the populations from different areas. At the same time, we found that <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> collected from the same province are also far apart. The reason for this separation can be its growth background. <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> from different region may change the genetic information due to the hybridization between different parental, with the coaction of the effect of the soil, nutrients, temperature, and moisture resulting in differences in genetic traits, making the genetic diversity of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> from the same area more complicated. Furthermore, the UPGMA clustering indicated that there was no significant relationship between genetic and geographic distances. This result was confirmed by the PCoA which indicates that the geographical distribution is not the main factor that shaped the current population genetic structure. For those populations with high levels of genetic variation of different regions, we suggest that their habitats should be protected and the exploitation of wild resources be controlled.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Experiments have shown that the genetic relationships among different geographical origins of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> are complex. As a valuable Chinese herbal medicine, to protect and take artificial breeding of wild resources are very important. We hope that this research provides some references and theoretical basis for the identification and selection of good seeds of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> resources.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec8" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">The primary objective in nature conservation is to preserve the evolutionary potential of species through maintaining as much genetic diversity as possible. Thus, knowledge of the genetic variation between and within different populations of plant species plays a significant role in the formulation of appropriate strategies for their conservation [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. In summary, our study revealed that 50 samples of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> depicted a high level of genetic diversity. SCoT and IRAP markers can provide highly accurate and efficient information on population genetic diversity. This study provides valuable baseline data on the population genetics of <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> from China, which is useful for germplasm appraisal and resource protection, including the construction of a core collection and regional variety distribution and subrogation. Finally, it is noted that the agarose gel electrophoresis detection is simple and easy to operate, but it is often difficult to get a clear, easily distinguishable photo. Even a large number of repeated tests are required to meet the final test requirements. Capillary electrophoresis may be used in the future to obtain more accurate analysis results.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Materialandmethods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Materialandmethods">Material and methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Materialandmethods" data-test="collapse-section">
<section id="Sec10" class="Section2 RenderAsSection2"><h3 class="Heading">Materials</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The <em xmlns="" class="EmphasisTypeItalic">Bletilla striata</em> collected in this experiment were identified by professor Zhishan Ding of Zhejiang Chinese Medical University. A voucher specimen of the plant material used in this study has been deposited in molecular biology laboratory of Zhejiang Chinese Medical University (From NO.BJ-20160301-001 to NO.BJ-20160301-050). Specific information is in Table <span class="InternalRef"><a href="#Tab4">4</a></span>, Fig. <span class="InternalRef"><a href="#Fig7">7</a></span>.<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Sources and description of materials</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<colgroup><col></colgroup>
<thead><tr class="header">
<th><p xmlns="" class="SimplePara">No</p></th>
<th><p xmlns="" class="SimplePara">Location</p></th>
<th><p xmlns="" class="SimplePara">Species name</p></th>
<th><p xmlns="" class="SimplePara">Flower color</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">1</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">2</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B1etilla sinensis</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">White</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">3</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">4</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla ochracea</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Yellow</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">5</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">6</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">7</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">8</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">9</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B1etilla sinensis</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">White</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">10</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">11</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">12</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B1etilla sinensis</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">White</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">13</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">14</p></td>
<td><p xmlns="" class="SimplePara">Hubei Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">15</p></td>
<td><p xmlns="" class="SimplePara">Anhui Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">16</p></td>
<td><p xmlns="" class="SimplePara">Anhui Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">17</p></td>
<td><p xmlns="" class="SimplePara">Anhui Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">18</p></td>
<td><p xmlns="" class="SimplePara">Anhui Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">19</p></td>
<td><p xmlns="" class="SimplePara">Anhui Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">20</p></td>
<td><p xmlns="" class="SimplePara">Anhui Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">21</p></td>
<td><p xmlns="" class="SimplePara">Sichuan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">22</p></td>
<td><p xmlns="" class="SimplePara">Sichuan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">23</p></td>
<td><p xmlns="" class="SimplePara">Sichuan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">24</p></td>
<td><p xmlns="" class="SimplePara">Sichuan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B1etilla sinensis</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">White</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">25</p></td>
<td><p xmlns="" class="SimplePara">Henan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">26</p></td>
<td><p xmlns="" class="SimplePara">Henan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">27</p></td>
<td><p xmlns="" class="SimplePara">Henan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">28</p></td>
<td><p xmlns="" class="SimplePara">Guizhou Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">29</p></td>
<td><p xmlns="" class="SimplePara">Guizhou Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">30</p></td>
<td><p xmlns="" class="SimplePara">Yunnan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">31</p></td>
<td><p xmlns="" class="SimplePara">Yunnan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">32</p></td>
<td><p xmlns="" class="SimplePara">Yunnan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">33</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">34</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">35</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">36</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">37</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">38</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">39</p></td>
<td><p xmlns="" class="SimplePara">Zhejiang Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">40</p></td>
<td><p xmlns="" class="SimplePara">Shanxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">41</p></td>
<td><p xmlns="" class="SimplePara">Shanxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B1etilla sinensis</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">White</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">42</p></td>
<td><p xmlns="" class="SimplePara">Shanxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">43</p></td>
<td><p xmlns="" class="SimplePara">Shanxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">44</p></td>
<td><p xmlns="" class="SimplePara">Shanxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">45</p></td>
<td><p xmlns="" class="SimplePara">Gansu Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">46</p></td>
<td><p xmlns="" class="SimplePara">Hunan Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">47</p></td>
<td><p xmlns="" class="SimplePara">Fujian Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">48</p></td>
<td><p xmlns="" class="SimplePara">Jiangxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">49</p></td>
<td><p xmlns="" class="SimplePara">Jiangxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">50</p></td>
<td><p xmlns="" class="SimplePara">Guangxi Province, China</p></td>
<td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bletilla striata</em>
                            </p></td>
<td><p xmlns="" class="SimplePara">Purple</p></td>
</tr>
</tbody>
</table></div></figure><figure xmlns="" class="Figure" id="Fig7"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO7">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig7_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig7_HTML.png" alt="Fig. 7" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig7_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0074-4/MediaObjects/41065_2018_74_Fig7_HTML.png" alt="Fig. 7"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 7</span>
                        <p xmlns="" class="SimplePara">Collecting sites for the <em class="EmphasisTypeItalic">Bletilla striata</em> species in China are marked</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec11" class="Section2 RenderAsSection2"><h3 class="Heading">Leaf DNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">A modified cetyltrimethyl ammonium bromide (CTAB) method was used for DNA extraction [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. DNA samples were stored at − 20 °C and the quality verified by electrophoresis on ethidium bromide stained 1% agarose gel.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Determination of IRAP and SCoT systems</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Karim Sorkheh et al. used IRAP for germplasm analysis of wild <em xmlns="" class="EmphasisTypeItalic">Pistacia chinensis</em> [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], and Ahmad Mousapour Gorji et al. used SCoT to analyze the genetic fingerprint of tetraploid potatoes [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. We used 20 μL of the IRAP and SCoT molecular markers in both studies.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2"><h3 class="Heading">SCoT analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">PCR amplification was performed in a final reaction volume of 20 μL containing 10× buffer, 0.1 mM dNTPs, 0.3 μM of each primer, 1 U Taq polymerase, and 20 ng of template DNA. The amplification was performed with the following thermal program: 94 °C for 4 min; followed by 36 cycles of denaturation at 95 °C for 50s, annealing at 50 °C for 40 s, elongation at 72 °C for 2 min; and extension at 72 °C for 5 min. All PCR reactions were performed twice. The products were kept at 4 °C.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2"><h3 class="Heading">IRAP analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">PCR amplification was performed in a final reaction volume of 20 μL containing 10× buffer, 800 mM dNTPs, 0.3 nM of each primer, 1.25 U Taq polymerase, and 20 ng of template DNA. The amplification was performed with the following thermal program: 94 °C for 4 min, 36 °C for 1 min, 72 °C for2 min; followed by 40 cycles of denaturation at 94 °C for 1 min, annealing at 36 °C for 1 min, elongation at 72 °C for 2 min; and extension at 94 °C for 4 min, 36 °C for 1 min, 72 °C for2 min. All PCR reactions were performed twice. The products were kept at 4 °C.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2"><h3 class="Heading">Electrophoresis detection and genotypes of reaction products</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">DNA amplification products for SCoT and IRAP markers were analyzed by electrophoresis in 1.80% agarose gels. The electrophoretic patterns of the products were recorded using a gel imaging system (Tanon 3500R, Shanghai, China). Genotypes were built based on different DNA band patterns on gels.</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2"><h3 class="Heading">Data analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Data matrices of Inter-simple sequence repeat and start codon targeted polymorphic marker profiles were generated by scoring (1) for presence and (0) for absence of individual allele. Polymorphism information content (PIC) is a property value of a marker based on its allelic number and distribution frequency in a population. PIC for marker <em xmlns="" class="EmphasisTypeItalic">ί</em> was calculated using PIC = 1 − Σ<em xmlns="" class="EmphasisTypeItalic">Pί</em><sup>2</sup> according to Botstein et al. [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>], where <em xmlns="" class="EmphasisTypeItalic">Pί</em> is the allele frequency at locus <em xmlns="" class="EmphasisTypeItalic">ί</em>. Nei’s genetic diversity index (<span class="stix">ℎ</span>) was calculated using POPGENE 1.31[<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. The resulting matrices were used to construct the dendrogram through unweighted pair-group method with arithmetic mean (UPGMA) using SAHN module. The cluster, cophenetic correlation and PCoA analysis was conducted by NTSYSpc software version 2.1. Analysis of molecular variance (AMOVA) was performed in order to determine the differences existent amongst and within population levels. F-statistics (Fst) was also calculated using Arlequin v. 3.01 (Miller 1998).</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Notes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Notes">Notes</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Notes"><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteAuthorContribution"><p xmlns="" class="SimplePara">Yan Guo and Lina Zhai contributed equally to this work.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgements</h3>
<p xmlns="" class="SimplePara">We thank professor Zhishan Ding for technical support.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par25" class="Para">This study was supported by the Study on Quality Guarantee System of Chinese Herbal Pieces in the State Administration of Traditional Chinese Medicine (No.201507002), National Natural Science Foundation of China (No.81503329) and Zhejiang Province Public Welfare Technology Application Research Project (2015C32084).</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par26" class="Para">The data that support the findings of this study are available from the corresponding author (Bo Jin) upon reasonable request.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1">
<h3 xmlns="" class="Heading">Authors’ contributions</h3>
<p xmlns="" class="SimplePara">BJ and ZD conceived and designed the study. YG, LZ, and HL performed the experiments. YG wrote the paper. YG, NC, and CG reviewed and edited the manuscript. All authors read and approved the manuscript.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par27" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par28" class="Para">All authors agree to publication.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par29" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par30" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">College of Life Science, Zhejiang Chinese Medical University, Hangzhou, 310053, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">College of Medical Technology, Zhejiang Chinese Medical University, Hangzhou, 310053, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ahmad MG, Peter P, Zsolt P, Janos T. Efficiency of arbitrarily amplified dominant markers (SCOT, ISSR and RAPD) for diagnostic fingerprinting in tetraploid potato. Am J Pot Res. 2011;88:226–37.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12230-011-9187-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Efficiency%20of%20arbitrarily%20amplified%20dominant%20markers%20%28SCOT%2C%20ISSR%20and%20RAPD%29%20for%20diagnostic%20fingerprinting%20in%20tetraploid%20potato&amp;author=MG.%20Ahmad&amp;author=P.%20Peter&amp;author=P.%20Zsolt&amp;author=T.%20Janos&amp;journal=Am%20J%20Pot%20Res&amp;volume=88&amp;pages=226-237&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Alikhani L, Rahmani M-S, Naghi S, Hedieh B, Abdollah KK. Genetic variability and structure of Quercus brantii assessed by ISSR, IRAP and SCoT markers. Gene. 2014;552:176–83.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2014.09.034"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variability%20and%20structure%20of%20Quercus%20brantii%20assessed%20by%20ISSR%2C%20IRAP%20and%20SCoT%20markers&amp;author=L.%20Alikhani&amp;author=M-S.%20Rahmani&amp;author=S.%20Naghi&amp;author=B.%20Hedieh&amp;author=KK.%20Abdollah&amp;journal=Gene&amp;volume=552&amp;pages=176-183&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Botstein D, White RL, Skolnick M, Davis RW. Construction of a genetic linkage map in man using restriction fragment length polymorphisms. AJHG. 1980;32:314–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=RL.%20White&amp;author=M.%20Skolnick&amp;author=RW.%20Davis&amp;journal=AJHG&amp;volume=32&amp;pages=314-331&amp;publication_year=1980"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chinese Herbalism Editorial Board. Zhonghua Bencao. Shanghai: Shanghai Scientific &amp; Technical Publishers; 1999. p. 7803.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Chinese%20Herbalism%20Editorial%20Board.%20Zhonghua%20Bencao.%20Shanghai%3A%20Shanghai%20Scientific%20%26%20Technical%20Publishers%3B%201999.%20p.%207803."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Collard BCY, Mackill DJ. Start codon targeted (SCoT) polymorphism: a simple, novel DNA marker technique for generating gene-targeted markers in plants. Plant Mol Biol Rep. 2009;27:86–93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11105-008-0060-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Start%20codon%20targeted%20%28SCoT%29%20polymorphism%3A%20a%20simple%2C%20novel%20DNA%20marker%20technique%20for%20generating%20gene-targeted%20markers%20in%20plants&amp;author=BCY.%20Collard&amp;author=DJ.%20Mackill&amp;journal=Plant%20Mol%20Biol%20Rep&amp;volume=27&amp;pages=86-93&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalendar R, Grob T, Regina M, Suoniemi A, Schulman A. IRAP and REMAP: two new retrotransposon-based DNA fingerprinting techniques. Theor Appl Genet. 1999;98:704–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051124"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=IRAP%20and%20REMAP%3A%20two%20new%20retrotransposon-based%20DNA%20fingerprinting%20techniques&amp;author=R.%20Kalendar&amp;author=T.%20Grob&amp;author=M.%20Regina&amp;author=A.%20Suoniemi&amp;author=A.%20Schulman&amp;journal=Theor%20Appl%20Genet&amp;volume=98&amp;pages=704-711&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lawler LJ. Ethnobotany of the orchidaceae. In: Arditti J, editor. Orchid Biology, vol. 3. America: Reviews and Perspectives. Cornell University Press, NY; 1984. p. 256.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ethnobotany%20of%20the%20orchidaceae&amp;author=LJ.%20Lawler&amp;pages=256&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Liu YL, Cui CJ, Li XJ, Liang GJ, Peng J, Tian PPRAPD. Analysis of genetic diversity in Saccharina japonica germplasm resources. Biotechnol Bull. 2016;32:151–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20in%20Saccharina%20japonica%20germplasm%20resources&amp;author=Y.%20Li&amp;author=YL.%20Liu&amp;author=CJ.%20Cui&amp;author=XJ.%20Li&amp;author=GJ.%20Liang&amp;author=J.%20Peng&amp;author=PPRAPD.%20Tian&amp;journal=Biotechnol%20Bull&amp;volume=32&amp;pages=151-158&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li R, Wang ZZ. Research survey and countermeasure on resources utilization in stem tuber of Bletilla striata. Chin Tradit Herb Drugs. 2006;37:1751–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Research%20survey%20and%20countermeasure%20on%20resources%20utilization%20in%20stem%20tuber%20of%20Bletilla%20striata&amp;author=R.%20Li&amp;author=ZZ.%20Wang&amp;journal=Chin%20Tradit%20Herb%20Drugs&amp;volume=37&amp;pages=1751-1755&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Milligan BG, Leebens-Mack J, Strand AE. Conservation genetics: beyond the maintenance of marker diversity. Mol Ecol. 1994;12:844–55.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Conservation%20genetics%3A%20beyond%20the%20maintenance%20of%20marker%20diversity&amp;author=BG.%20Milligan&amp;author=J.%20Leebens-Mack&amp;author=AE.%20Strand&amp;journal=Mol%20Ecol&amp;volume=12&amp;pages=844-855&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nakatsuji R, Hashida T, Matsumoto N, Tsuro M, Kubo N, Hirai M. Development of genomic and EST-SSR markers in radish (Raphanus sativus L.). Breed Sci. 2011;61:413–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1270/jsbbs.61.413"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20genomic%20and%20EST-SSR%20markers%20in%20radish%20%28Raphanus%20sativus%20L.%29&amp;author=R.%20Nakatsuji&amp;author=T.%20Hashida&amp;author=N.%20Matsumoto&amp;author=M.%20Tsuro&amp;author=N.%20Kubo&amp;author=M.%20Hirai&amp;journal=Breed%20Sci&amp;volume=61&amp;pages=413-419&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Perry LM, Metzger J. Medicinal plants of east and Southeast Asia: attributed properties and uses. Cambridge: MIT Press; 1990. p. 168.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Medicinal%20plants%20of%20east%20and%20Southeast%20Asia%3A%20attributed%20properties%20and%20uses&amp;author=LM.%20Perry&amp;author=J.%20Metzger&amp;publication_year=1990"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rahmani M-S, Paula M, Pijut NS, Mona N. Genetic fidelity assessment of in vitro-regenerated plants of Albizia julibrissin using SCoT and IRAP fingerprinting. Biol Plant. 2015;51:407–19.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20fidelity%20assessment%20of%20in%20vitro-regenerated%20plants%20of%20Albizia%20julibrissin%20using%20SCoT%20and%20IRAP%20fingerprinting&amp;author=M-S.%20Rahmani&amp;author=M.%20Paula&amp;author=NS.%20Pijut&amp;author=N.%20Mona&amp;journal=Biol%20Plant&amp;volume=51&amp;pages=407-419&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi J. Resources and medicinal utilization of Bletilla (orchidaceae). Hainan University: Dissertation; 2009.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Resources%20and%20medicinal%20utilization%20of%20Bletilla%20%28orchidaceae%29&amp;author=J.%20Shi&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi J, Luo YB, Song XQ. To Surve and analyze the market of Bletilla striata in China. Chiese Horti Culture Abstracts. 2010;8:48–50.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=To%20Surve%20and%20analyze%20the%20market%20of%20Bletilla%20striata%20in%20China&amp;author=J.%20Shi&amp;author=YB.%20Luo&amp;author=XQ.%20Song&amp;journal=Chiese%20Horti%20Culture%20Abstracts&amp;volume=8&amp;pages=48-50&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sorkheh K, Nazanin A, Sezai E. Potential start codon targeted (SCoT) and inter-retrotransposon amplified polymorphism (IRAP) markers for evaluation of genetic diversity and conservation of wild Pistacia species population. Biochem Genet. 2016;54:368–87.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10528-016-9725-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Potential%20start%20codon%20targeted%20%28SCoT%29%20and%20inter-retrotransposon%20amplified%20polymorphism%20%28IRAP%29%20markers%20for%20evaluation%20of%20genetic%20diversity%20and%20conservation%20of%20wild%20Pistacia%20species%20population&amp;author=K.%20Sorkheh&amp;author=A.%20Nazanin&amp;author=E.%20Sezai&amp;journal=Biochem%20Genet&amp;volume=54&amp;pages=368-387&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu JN. An illustrated Chinese Materia Medica. New York: Oxford University Press; 2005.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Wu%20JN.%20An%20illustrated%20Chinese%20Materia%20Medica.%20New%20York%3A%20Oxford%20University%20Press%3B%202005."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu YG, Guo QS, He JC, Lin YF, Luo LJ, Liu GD. Genetic diversity analysis among and within populations of Pogostemon cablin from China with ISSR and SRAP markers. Biochem Syst Ecol. 2010;38:63–72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2009.12.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20analysis%20among%20and%20within%20populations%20of%20Pogostemon%20cablin%20from%20China%20with%20ISSR%20and%20SRAP%20markers&amp;author=YG.%20Wu&amp;author=QS.%20Guo&amp;author=JC.%20He&amp;author=YF.%20Lin&amp;author=LJ.%20Luo&amp;author=GD.%20Liu&amp;journal=Biochem%20Syst%20Ecol&amp;volume=38&amp;pages=63-72&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yeh FC, Yang RC, Boyle TBJ, Ye ZH, Mao JX. “POPGENE, the user friendly shareware for population genetic analysis version1. 31,” University of Alberta and Centre For International Forestry Research, vol. 28; 1999. p. 45–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%E2%80%9CPOPGENE%2C%20the%20user%20friendly%20shareware%20for%20population%20genetic%20analysis%20version1.%2031%2C%E2%80%9D%20University%20of%20Alberta%20and%20Centre%20For%20International%20Forestry%20Research&amp;author=FC.%20Yeh&amp;author=RC.%20Yang&amp;author=TBJ.%20Boyle&amp;author=ZH.%20Ye&amp;author=JX.%20Mao&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhou TH, Li J, Ding JX, Yang H, Chen SL. SSR fingerprinting of <em class="EmphasisTypeItalic">Bletilla striata Rchb.f. Germplasm</em> and its relatives or congeners. Acta bot. Boreal -OcidentSin. 2017;37:0673–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SSR%20fingerprinting%20of%20Bletilla%20striata%20Rchb.f.%20Germplasm%20and%20its%20relatives%20or%20congeners.%20Acta%20bot&amp;author=TH.%20Zhou&amp;author=J.%20Li&amp;author=JX.%20Ding&amp;author=H.%20Yang&amp;author=SL.%20Chen&amp;journal=Boreal%20-OcidentSin&amp;volume=37&amp;pages=0673-0681&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang MB, Fan GQ, Chen JM, Chen P. Study on DNA isolation from polysaccharides-rich transgenic dendrobium. Mol Plant Breed. 2009;7:209–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Study%20on%20DNA%20isolation%20from%20polysaccharides-rich%20transgenic%20dendrobium&amp;author=MB.%20Zhang&amp;author=GQ.%20Fan&amp;author=JM.%20Chen&amp;author=P.%20Chen&amp;journal=Mol%20Plant%20Breed&amp;volume=7&amp;pages=209-214&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang YW, Jiang FS, Wang Y, Ding ZS. Present status and sustainable development of Rhizoma Bletillae industry. Chin Arch Tradit Chin Med. 2012;30:2264–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Present%20status%20and%20sustainable%20development%20of%20Rhizoma%20Bletillae%20industry&amp;author=YW.%20Zhang&amp;author=FS.%20Jiang&amp;author=Y.%20Wang&amp;author=ZS.%20Ding&amp;journal=Chin%20Arch%20Tradit%20Chin%20Med&amp;volume=30&amp;pages=2264-2267&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">© The Author(s) 2018</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0074-4" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-018-0074-4">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0074-4" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0074-4?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0074-4 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0074-4 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0074-4 #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0074-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0074-4">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0074-4"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0074-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0074-4">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0074-4"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0074-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0074-4">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0074-4"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0074-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0074-4">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0074-4"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0074-4">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0074-4">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0074-4"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-018-0074-4/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-018-0074-4">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0074-4">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0074-4" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-018-0074-4&amp;volume=155&amp;issue=1&amp;title=Genetic+diversity+of+Bletilla+striata+assessed+by+SCoT+and+IRAP+markers&amp;author_name=Yan+Guo&amp;start_page=1&amp;end_page=10">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-018-0074-4" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
