<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Global mapping of binding sites for phic31 integrase in transgenic maden-darby bovine kidney cells using ChIP-seq | Hereditas | Full Text</title>
<meta name="citation_abstract" content="Î¦C31 integrase, a site-specific recombinase, can efficiently target attB-bearing transgenes to endogenous pseudo attP sites within mammalian genomes. The sequence features of endogenous binding sites will help us to fully understand the site-specific recognition function by Î¦C31 integrase. The present study was aimed to uncover the global map of Î¦C31 integrase binding sites in bovine cells and analysis the features of these binding sites by comprehensive bioinformatics methods. In this study, we constructed a ChIP-seq method that can be used to uncover the global binding sites by phiC31 integrase. 6740 potential Î¦C31 integrase binding sites were identified. A sequence motif was found that contains inverted repeats and has similarities to wild-type attP site. Using REPEATMASKER, we identified a total of 20,183 repeat-regions distributed in 50 repeat types for the 6740 binding sites. These sites enriched in âregulation of GTPase activityâ of in the GO category of biological process and KEGG pathway of signal transmembrane transporter activity. This study is the first time to uncover the global map of binding sites for Î¦C31 integrase using ChIP-sequencing method and analysis the features of these binding sites. This method will help us to fully understand the mechanism of the site-specific integration function by phiC31 integrase and will potentially boost its genetic manipulations in both gene therapy and generation of transgenic animals.">
<meta name="description" content="Î¦C31 integrase, a site-specific recombinase, can efficiently target attB-bearing transgenes to endogenous pseudo attP sites within mammalian genomes. The sequence features of endogenous binding sites will help us to fully understand the site-specific recognition function by Î¦C31 integrase. The present study was aimed to uncover the global map of Î¦C31 integrase binding sites in bovine cells and analysis the features of these binding sites by comprehensive bioinformatics methods. In this study, we constructed a ChIP-seq method that can be used to uncover the global binding sites by phiC31 integrase. 6740 potential Î¦C31 integrase binding sites were identified. A sequence motif was found that contains inverted repeats and has similarities to wild-type attP site. Using REPEATMASKER, we identified a total of 20,183 repeat-regions distributed in 50 repeat types for the 6740 binding sites. These sites enriched in âregulation of GTPase activityâ of in the GO category of biological process and KEGG pathway of signal transmembrane transporter activity. This study is the first time to uncover the global map of binding sites for Î¦C31 integrase using ChIP-sequencing method and analysis the features of these binding sites. This method will help us to fully understand the mechanism of the site-specific integration function by phiC31 integrase and will potentially boost its genetic manipulations in both gene therapy and generation of transgenic animals.">
<meta name="journal_id" content="41065">
<meta name="dc.title" content="Global mapping of binding sites for phic31 integrase in transgenic maden-darby bovine kidney cells using ChIP-seq">
<meta name="dc.source" content="Hereditas 2019 156:1">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central">
<meta name="dc.date" content="2019-01-14">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2019 The Author(s)">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">
<meta name="dc.description" content="ΦC31 integrase, a site-specific recombinase, can efficiently target attB-bearing transgenes to endogenous pseudo attP sites within mammalian genomes. The sequence features of endogenous binding sites will help us to fully understand the site-specific recognition function by ΦC31 integrase. The present study was aimed to uncover the global map of ΦC31 integrase binding sites in bovine cells and analysis the features of these binding sites by comprehensive bioinformatics methods. In this study, we constructed a ChIP-seq method that can be used to uncover the global binding sites by phiC31 integrase. 6740 potential ΦC31 integrase binding sites were identified. A sequence motif was found that contains inverted repeats and has similarities to wild-type attP site. Using REPEATMASKER, we identified a total of 20,183 repeat-regions distributed in 50 repeat types for the 6740 binding sites. These sites enriched in “regulation of GTPase activity” of in the GO category of biological process and KEGG pathway of signal transmembrane transporter activity. This study is the first time to uncover the global map of binding sites for ΦC31 integrase using ChIP-sequencing method and analysis the features of these binding sites. This method will help us to fully understand the mechanism of the site-specific integration function by phiC31 integrase and will potentially boost its genetic manipulations in both gene therapy and generation of transgenic animals.">
<meta name="prism.issn" content="1601-5223">
<meta name="prism.publicationName" content="Hereditas">
<meta name="prism.publicationDate" content="2019-01-14">
<meta name="prism.volume" content="156">
<meta name="prism.number" content="1">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="3">
<meta name="prism.copyright" content="2019 The Author(s)">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">
<meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0079-z">
<meta name="prism.doi" content="doi:10.1186/s41065-018-0079-z">
<meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0079-z">
<meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0079-z">
<meta name="citation_journal_title" content="Hereditas">
<meta name="citation_publisher" content="BioMed Central">
<meta name="citation_issn" content="1601-5223">
<meta name="citation_title" content="Global mapping of binding sites for phic31 integrase in transgenic maden-darby bovine kidney cells using ChIP-seq">
<meta name="citation_volume" content="156">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/12">
<meta name="citation_online_date" content="2019/01/14">
<meta name="citation_firstpage" content="3">
<meta name="citation_article_type" content="Research">
<meta name="citation_fulltext_world_readable" content="">
<meta name="dc.identifier" content="doi:10.1186/s41065-018-0079-z">
<meta name="DOI" content="10.1186/s41065-018-0079-z">
<meta name="citation_doi" content="10.1186/s41065-018-0079-z">
<meta name="dc.creator" content="Lijuan Qu">
<meta name="dc.creator" content="Lei Wang">
<meta name="dc.creator" content="Xueyuan Zhu">
<meta name="dc.creator" content="Yan Zhang">
<meta name="dc.creator" content="Qiang Ou">
<meta name="dc.creator" content="Aying Ma">
<meta name="dc.creator" content="Fengying Sheng">
<meta name="dc.creator" content="Xiaoqing Wei">
<meta name="dc.creator" content="Yue Dai">
<meta name="dc.creator" content="Guoting Li">
<meta name="dc.creator" content="Shuwu Xie">
<meta name="dc.subject" content="Animal Genetics and Genomics">
<meta name="dc.subject" content="Evolutionary Biology">
<meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Analysis of the integration function of the streptomycetebacteriophage phiC31; citation_author=S Kuhstoss, RN Rao; citation_volume=222; citation_publication_date=1991; citation_pages=897-908; citation_doi=10.1016/0022-2836(91)90584-S; citation_id=CR1">
<meta name="citation_reference" content="citation_journal_title=Adv Genet; citation_title=Site-specific integration with phiC31 integrase for prolonged expression of therapeutic genes; citation_author=DS Ginsburg, MP Calos; citation_volume=54; citation_publication_date=2005; citation_pages=179-187; citation_doi=10.1016/S0065-2660(05)54008-2; citation_id=CR2">
<meta name="citation_reference" content="Hillman RT, Calos MP. Site-specific integration with bacteriophage PhiC31 integrase. Cold Spring Harb Protoc. 2012. 
                    https://doi.org/10.1101/pdb.prot069211
                    
                  .">
<meta name="citation_reference" content="citation_journal_title=Biosci Biotechnol Biochem; citation_title=ΦC31integrase mediates efficient site-specific integration in sheep fibroblasts; citation_author=W Ni, S Hu, J Qiao, Y Wang, H Shi, Y Wang, Z He, G Li, C Chen; citation_volume=76; citation_publication_date=2012; citation_pages=2093-2095; citation_doi=10.1271/bbb.120439; citation_id=CR4">
<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=One-step generation of multiple transgenic mouse lines using an improved pronuclear injection-based targeted Transgenesis (i-PITT); citation_author=M Ohtsuka, H Miura, K Mochida, M Hirose, A Hasegawa, A Ogura, R Mizutani, M Kimura, A Isotani, M Ikawa, M Sato, CB Gurumurthy; citation_volume=16; citation_publication_date=2015; citation_pages=274; citation_doi=10.1186/s12864-015-1432-5; citation_id=CR5">
<meta name="citation_reference" content="citation_journal_title=FEBS J; citation_title=A phiC31 integrase-mediated integration hotspot in favor of transgene expression exists in the bovine genome; citation_author=HL Ou, Y Huang, LJ Qu, M Xu, JB Yan, ZR Ren, SZ Huang, YT Zeng; citation_volume=276; citation_publication_date=2009; citation_pages=155-163; citation_doi=10.1111/j.1742-4658.2008.06762.x; citation_id=CR6">
<meta name="citation_reference" content="citation_journal_title=Mol Cell Bio; citation_title=Site-specific genomic integration in mammalian cells mediated by phage phiC31 integrase; citation_author=B Thyagarajan, EC Olivares, RP Hollis, DS Ginsburg, MP Calos; citation_volume=21; citation_publication_date=2001; citation_pages=3926-3934; citation_doi=10.1128/MCB.21.12.3926-3934.2001; citation_id=CR7">
<meta name="citation_reference" content="citation_journal_title=Biotechnol J; citation_title=Efficient reversal of phiC31 integrase recombination in mammalian cells; citation_author=AP Farruggio, CL Chavez, CL Mikell, MP Calos; citation_volume=7; citation_publication_date=2012; citation_pages=1332-1336; citation_doi=10.1002/biot.201200283; citation_id=CR8">
<meta name="citation_reference" content="citation_journal_title=Hum Gene Ther; citation_title=Long-term expression of human coagulation factor VIII in a tolerant mouse model using the phiC31 integrase system; citation_author=CL Chavez, A Keravala, JN Chu, AP Farruggio, VE Cuellar, J Voorberg, MP Calos; citation_volume=23; citation_publication_date=2012; citation_pages=390-398; citation_doi=10.1089/hum.2011.110; citation_id=CR9">
<meta name="citation_reference" content="citation_journal_title=Gene Ther; citation_title=Long-term phenotypic correction in factor IX knockout mice by using phiC31 integrase-mediated gene therapy; citation_author=A Keravala, CL Chavez, G Hu, LE Woodard, PE Monahan, MP Calos; citation_volume=18; citation_publication_date=2011; citation_pages=842-848; citation_doi=10.1038/gt.2011.31; citation_id=CR10">
<meta name="citation_reference" content="citation_journal_title=Gene Ther; citation_title=Intragenic integration in DLC1 sustains factor VIII expression in primary human cells without insertional oncogenicity; citation_author=J Sivalingam, TT Phan, OL Kon; citation_volume=21; citation_publication_date=2014; citation_pages=402-412; citation_doi=10.1038/gt.2014.11; citation_id=CR11">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Targeted microRNA expression in dairy cattle directs production of beta-lactoglobulin-free, high-casein milk; citation_author=A Jabed, S Wagner, J McCracken, DN Wells, G Laible; citation_volume=109; citation_publication_date=2012; citation_pages=16811-16816; citation_doi=10.1073/pnas.1210057109; citation_id=CR12">
<meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Establishment of a rapid and scalable gene expression system in livestock by site-specific integration; citation_author=H Yu, X Wang, L Zhu, Z He, G Liu, X Xu, J Chen, G Cheng; citation_volume=515; citation_publication_date=2013; citation_pages=367-371; citation_doi=10.1016/j.gene.2012.10.017; citation_id=CR13">
<meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=Using phiC31 integrase to make transgenic Xenopus laevis embryos; citation_author=BG Allen, DL Weeks; citation_volume=1; citation_publication_date=2006; citation_pages=1248-1257; citation_doi=10.1038/nprot.2006.183; citation_id=CR14">
<meta name="citation_reference" content="citation_journal_title=Neurosci Res; citation_title=In vivo evaluation of phiC31 recombinase activity in transgenic mice; citation_author=I Imayoshi, K Hirano, S Kitano, H Miyachi, R Kageyama; citation_volume=73; citation_publication_date=2012; citation_pages=106-114; citation_doi=10.1016/j.neures.2012.02.008; citation_id=CR15">
<meta name="citation_reference" content="citation_journal_title=Methods Mol Biol; citation_title=Using phiC31 integrase to mediate insertion of DNA in Xenopus embryos; citation_author=YE Li, BG Allen, DL Weeks; citation_volume=917; citation_publication_date=2012; citation_pages=219-230; citation_doi=10.1007/978-1-61779-992-1_13; citation_id=CR16">
<meta name="citation_reference" content="citation_journal_title=BMC Biotechnol; citation_title=phiC31 integrase induces a DNA damage response and chromosomal rearrangements in human adult fibroblasts; citation_author=J Liu, T Skjorringe, T Gjetting, TG Jensen; citation_volume=9; citation_publication_date=2009; citation_pages=31; citation_doi=10.1186/1472-6750-9-31; citation_id=CR17">
<meta name="citation_reference" content="citation_journal_title=J Genet Genomics; citation_title=A profile of native integration sites used by phiC31 integrase in the bovine genome; citation_author=L Qu, Q Ma, Z Zhou, H Ma, Y Huang, S Huang, F Zeng, Y Zeng; citation_volume=39; citation_publication_date=2012; citation_pages=217-224; citation_doi=10.1016/j.jgg.2012.03.004; citation_id=CR18">
<meta name="citation_reference" content="citation_journal_title=Stem Cells; citation_title=Creation of engineered human embryonic stem cell lines using phiC31 integrase; citation_author=B Thyagarajan, Y Liu, S Shin, U Lakshmipathy, K Scheyhing, H Xue, C Ellerstrom, R Strehl, J Hyllner, MS Rao, JD Chesnut; citation_volume=26; citation_publication_date=2008; citation_pages=119-126; citation_doi=10.1634/stemcells.2007-0283; citation_id=CR19">
<meta name="citation_reference" content="citation_journal_title=Mol Ther; citation_title=Biosafety assessment of site-directed transgene integration in human umbilical cord-lining cells; citation_author=J Sivalingam, S Krishnan, WH Ng, SS Lee, TT Phan, OL Kon; citation_volume=18; citation_publication_date=2010; citation_pages=1346-1356; citation_doi=10.1038/mt.2010.61; citation_id=CR20">
<meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Integration specificity of phage phiC31 integrase in the human genome; citation_author=TW Chalberg, JL Portlock, EC Olivares, B Thyagarajan, PJ Kirby, RT Hillman, J Hoelters, MP Calos; citation_volume=357; citation_publication_date=2006; citation_pages=28-48; citation_doi=10.1016/j.jmb.2005.11.098; citation_id=CR21">
<meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=High-throughput chromatin immunoprecipitation for genome-wide mapping of in vivo protein-DNA interactions and epigenomic states; citation_author=R Blecher-Gonen, Z Barnett-Itzhaki, D Jaitin, D Amann-Zalcenstein, D Lara-Astiaso, I Amit; citation_volume=8; citation_publication_date=2013; citation_pages=539-554; citation_doi=10.1038/nprot.2013.023; citation_id=CR22">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Hybrid lentivirus-phiC31-int-NLS vector allows site-specific recombination in murine and human cells but induces DNA damage; citation_author=N Grandchamp, D Altemir, S philippe, S Ursulet, H Pilet, MC Serre, A Lenain, C Serguera, J Mallet, C Sarkis; citation_volume=9; citation_publication_date=2014; citation_pages=e99649; citation_doi=10.1371/journal.pone.0099649; citation_id=CR23">
<meta name="citation_reference" content="citation_journal_title=J Gene Med.; citation_title=Phage phiC31 integrase-mediated genomic integration and long-term gene expression in the lung after nonviral gene delivery; citation_author=MK Aneja, R Imker, C Rudolph; citation_volume=9; citation_publication_date=2007; citation_pages=967-975; citation_doi=10.1002/jgm.1090; citation_id=CR24">
<meta name="citation_reference" content="citation_journal_title=Invest Ophthalmol Vis Sci; citation_title=phiC31 integrase confers genomic integration and long-term transgene expression in rat retina; citation_author=TW Chalberg, HL Genise, D Vollrath, MP Calos; citation_volume=46; citation_publication_date=2005; citation_pages=2140-2146; citation_doi=10.1167/iovs.04-1252; citation_id=CR25">
<meta name="citation_reference" content="citation_journal_title=Mol Ther; citation_title=In vivo correction of murine hereditary tyrosinemia type I by phiC31 integrase-mediated gene delivery; citation_author=PK Held, EC Olivares, CP Aguilar, M Finegold, MP Calos, M Grompe; citation_volume=11; citation_publication_date=2005; citation_pages=399-408; citation_doi=10.1016/j.ymthe.2004.11.001; citation_id=CR26">
<meta name="citation_reference" content="citation_journal_title=J Gene Med; citation_title=PhiC31 integrase mediates integration in cultured synovial cells and enhances gene expression in rabbit joints; citation_author=A Keravala, JL Portlock, JA Nash, DG Vitrant, PD Robbins, MP Calos; citation_volume=8; citation_publication_date=2006; citation_pages=1008-1017; citation_doi=10.1002/jgm.928; citation_id=CR27">
<meta name="citation_reference" content="citation_journal_title=Transgenic research Transgenic Res; citation_title=Production of transgenic cattle highly expressing human serum albumin in milk by phiC31 integrase-mediated gene delivery; citation_author=Y Luo, Y Wang, J Liu, H Lan, M Shao, Y Yu, F Quan, Y Zhang; citation_volume=24; citation_publication_date=2015; citation_pages=875-883; citation_doi=10.1007/s11248-015-9898-0; citation_id=CR28">
<meta name="citation_reference" content="citation_journal_title=J Biotechnol; citation_title=Targeted gene integration using the combination of a sequence-specific DNA-binding protein and phiC31 integrase; citation_author=H Nakanishi, Y Higuchi, F Yamashita, M Hashida; citation_volume=186; citation_publication_date=2014; citation_pages=139-147; citation_doi=10.1016/j.jbiotec.2014.07.012; citation_id=CR29">
<meta name="citation_reference" content="citation_journal_title=J Drug Target; citation_title=Long-term in vivo gene expression in mouse kidney using phiC31 integrase and electroporation; citation_author=Y Otani, S Kawakami, H Mukai, Y Fuchigami, F Yamashita, M Hashida; citation_volume=23; citation_publication_date=2015; citation_pages=427-435; citation_doi=10.3109/1061186X.2014.1002788; citation_id=CR30">
<meta name="citation_reference" content="citation_journal_title=Drug News Perspect; citation_title=Insertional mutagenesis and development of malignancies induced by integrating gene delivery systems: implications for the design of safer gene-based interventions in patients; citation_author=G Romano, IR Marino, F Pentimalli, V Adamo, A Giordano; citation_volume=22; citation_publication_date=2009; citation_pages=185-196; citation_doi=10.1358/dnp.2009.22.4.1367704; citation_id=CR31">
<meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=A novel system for simultaneous or sequential integration of multiple gene-loading vectors into a defined site of a human artificial chromosome; citation_author=T Suzuki, Y Kazuki, M Oshimura, T Hara; citation_volume=9; citation_publication_date=2014; citation_pages=e110404; citation_doi=10.1371/journal.pone.0110404; citation_id=CR32">
<meta name="citation_reference" content="citation_journal_title=Gene Ther; citation_title=Effect of nuclear localization and hydrodynamic delivery-induced cell division on phiC31 integrase activity; citation_author=LE Woodard, RT Hillman, A Keravala, S Lee, MP Calos; citation_volume=17; citation_publication_date=2010; citation_pages=217-226; citation_doi=10.1038/gt.2009.136; citation_id=CR33">
<meta name="citation_reference" content="citation_journal_title=Biochem Biophys Res Commun; citation_title=The systemic effects of sclerostin overexpression using phiC31 integrase in mice; citation_author=D Zhang, BM Park, M Kang, H Nam, EJ Kim, C Bae, SK Lim; citation_volume=472; citation_publication_date=2016; citation_pages=471-476; citation_doi=10.1016/j.bbrc.2016.01.178; citation_id=CR34">
<meta name="citation_reference" content="citation_journal_title=Gene Ther; citation_title=Long-term and efficient expression of human beta-globin gene in a hematopoietic cell line using a new site-specific integrating non-viral system; citation_author=K Dormiani, H Mir Mohammad Sadeghi, H Sadeghi-Aliabadi, K Ghaedi, M Forouzanfar, H Baharvand, MH Nasr-Esfahani; citation_volume=22; citation_publication_date=2015; citation_pages=663-674; citation_doi=10.1038/gt.2015.30; citation_id=CR35">
<meta name="citation_reference" content="citation_journal_title=Gene Ther; citation_title=A versatile nonviral vector system for tetracycline-dependent one-step conditional induction of transgene expression; citation_author=K Inoue, T Sone, C Oneyama, F Nishiumi, H Kishine, Y Sasaki, T Andoh, M Okada, JD Chesnut, F Imamoto; citation_volume=16; citation_publication_date=2009; citation_pages=1383-1394; citation_doi=10.1038/gt.2009.105; citation_id=CR36">
<meta name="citation_reference" content="citation_journal_title=J Gene Med.; citation_title=alpha- L-iduronidase gene-based therapy using the phiC31 system to treat mucopolysaccharidose type I mice; citation_author=RS Stilhano, PK Martin, SM Melo, VY Samoto, GB Peres, YM Silva Michelacci, FH Silva, VG Pereira, V D'Almeida, AT Cruz, MG Jasiulionis, SW Han; citation_volume=17; citation_publication_date=2015; citation_pages=1-13; citation_doi=10.1002/jgm.2818; citation_id=CR37">
<meta name="citation_reference" content="citation_journal_title=Biometrics; citation_title=A Bayesian extension of the hypergeometric test for functional enrichment analysis; citation_author=J Cao, S Zhang; citation_volume=70; citation_publication_date=2014; citation_pages=84-94; citation_doi=10.1111/biom.12122; citation_id=CR38">
<meta name="citation_author" content="Lijuan Qu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Lei Wang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Xueyuan Zhu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Yan Zhang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Qiang Ou">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Aying Ma">
<meta name="citation_author_institution" content="Department of Respiratory Medicine, Shanghai First People’s Hospital, Shanghai Jiaotong University School of Medicine, Shanghai, China">
<meta name="citation_author" content="Fengying Sheng">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Xiaoqing Wei">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Yue Dai">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, Shanghai Eighth People’s Hospital, Shanghai, China">
<meta name="citation_author" content="Guoting Li">
<meta name="citation_author_institution" content="Lab of Reproductive Pharmacology, NHC Key Lab of Reproduction Regulation, Shanghai Institute of Planned Parenthood Research, Fudan University, Shanghai, China">
<meta name="citation_author" content="Shuwu Xie">
<meta name="citation_author_institution" content="Lab of Reproductive Pharmacology, NHC Key Lab of Reproduction Regulation, Shanghai Institute of Planned Parenthood Research, Fudan University, Shanghai, China">
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- For IE 9 and below --><!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]--><!-- Touch Icons - iOS and Android 2.1+ --><link rel="apple-touch-icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera --><link rel="icon" href="/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><link rel="stylesheet" media="screen" href="/static/styles/bmc/basic-0600efa430.css">
<link rel="stylesheet" media="print" href="/static/styles/bmc/print-7680b1a80b.css">
<link rel="stylesheet" id="js-mustard" href="/static/styles/bmc/main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script><script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0079-z',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script><!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker --><script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script><!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js--><script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script><script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0079-z","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Î¦C31 integrase;Binding sites;ChIP-seq;Sequence motif"},"contentInfo":{"imprint":"BioMed Central","title":"Global mapping of binding sites for phic31 integrase in transgenic maden-darby bovine kidney cells using ChIP-seq","publishedAt":1547424000000,"publishedAtDate":"2019-01-14","author":["Lijuan Qu","Lei Wang","Xueyuan Zhu","Yan Zhang","Qiang Ou","Aying Ma","Fengying Sheng","Xiaoqing Wei","Yue Dai","Guoting Li","Shuwu Xie"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"156","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script><script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script><script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><!-- Google Tag Manager --><script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script><!-- End Google Tag Manager --><link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0079-z">
<meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0079-z">
<meta property="og:type" content="article">
<meta property="og:site_name" content="Hereditas">
<meta property="og:title" content="Global mapping of binding sites for phic31 integrase in transgenic maden-darby bovine kidney cells using ChIP-seq">
<meta property="og:description" content="Î¦C31 integrase, a site-specific recombinase, can efficiently target attB-bearing transgenes to endogenous pseudo attP sites within mammalian genomes. The sequence features of endogenous binding sites will help us to fully understand the site-specific recognition function by Î¦C31 integrase. The present study was aimed to uncover the global map of Î¦C31 integrase binding sites in bovine cells and analysis the features of these binding sites by comprehensive bioinformatics methods. In this study, we constructed a ChIP-seq method that can be used to uncover the global binding sites by phiC31 integrase. 6740 potential Î¦C31 integrase binding sites were identified. A sequence motif was found that contains inverted repeats and has similarities to wild-type attP site. Using REPEATMASKER, we identified a total of 20,183 repeat-regions distributed in 50 repeat types for the 6740 binding sites. These sites enriched in âregulation of GTPase activityâ of in the GO category of biological process and KEGG pathway of signal transmembrane transporter activity. This study is the first time to uncover the global map of binding sites for Î¦C31 integrase using ChIP-sequencing method and analysis the features of these binding sites. This method will help us to fully understand the mechanism of the site-specific integration function by phiC31 integrase and will potentially boost its genetic manipulations in both gene therapy and generation of transgenic animals.">
<meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg">
</head>
<body class="journal journal-fulltext">
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK" height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><fecolormatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></fecolormatrix></filter></defs><symbol id="icon-arrow-down" viewbox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewbox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewbox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewbox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewbox="0 0 60.643 31.01"><mask maskunits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewbox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewbox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewbox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewbox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewbox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewbox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewbox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewbox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewbox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewbox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewbox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewbox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewbox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewbox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewbox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewbox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewbox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewbox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewbox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewbox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewbox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0079-z;kwrd=Î¦C31 integrase,Binding sites,ChIP-seq,Sequence motif;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0079-z&amp;kwrd=%C3%8E%C2%A6C31%20integrase,Binding%20sites,ChIP-seq,Sequence%20motif&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img" src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0079-z&amp;kwrd=%C3%8E%C2%A6C31%20integrase,Binding%20sites,ChIP-seq,Sequence%20motif&amp;pmc=L32030,L21001&amp;" alt="Advertisement" width="728" height="90"></a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header"><nav class="c-navbar"><div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
<div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src="/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo"><div class="c-logo__strapline">
            <img src="/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
</div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
<li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>
<ul class="c-navbar__nav c-navbar__nav--right">
<li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
</div>
        </div>
    </nav><div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""><div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true"><use xlink:href="#icon-search"></use></svg></button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch">
</form>
</div>

            </div>
        </div>
    </div>
</header><header class="c-journal-header c-journal-header--hereditas ctx-journal-header"><div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav"><ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
<li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul></nav>
</div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header><div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3>
<ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc="">
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec8" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li>
<li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li>
</ol></nav>
</div>

                <main class="c-content-layout__main"><article itemscope itemtype="http://schema.org/ScholarlyArticle"><div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
<li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
</div>

                        <div class="FulltextWrapper">
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Global mapping of binding sites for phic31 integrase in transgenic maden-darby bovine kidney cells using ChIP-seq</h1></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset">
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lijuan Qu</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lei Wang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xueyuan Zhu</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yan Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qiang Ou</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Aying Ma</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Fengying Sheng</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaoqing Wei</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yue Dai</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Guoting Li</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup> and </li>
<li class="Author hasAffil" data-jumpto="Aff" data-test="author-name">
<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Shuwu Xie</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:xieshw@fudan.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>
</li>
</ul></div>
<div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation">
<div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2019</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">156</strong>:3</span></span></div>
<p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0079-z">https://doi.org/10.1186/s41065-018-0079-z</a></p>
<p class="Copyright">Â©  The Author(s) 2019</p>
</div></div>
<ul class="ArticleHistory u-reset-list">
<li class="History HistoryReceived">
<strong>Received: </strong><span>15 October 2018</span>
</li>
<li class="History HistoryAccepted">
<strong>Accepted: </strong><span>25 December 2018</span>
</li>
<li class="History HistoryOnlineDate">
<strong>Published: </strong><span itemprop="datePublished">14 January 2019</span>
</li>
</ul>
</div>
                        
                        
                        
    

                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2>
<div class="c-section__content" id="Abstract">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Î¦C31 integrase, a site-specific recombinase, can efficiently target attB-bearing transgenes to endogenous pseudo attP sites within mammalian genomes. The sequence features of endogenous binding sites will help us to fully understand the site-specific recognition function by Î¦C31 integrase. The present study was aimed to uncover the global map of Î¦C31 integrase binding sites in bovine cells and analysis the features of these binding sites by comprehensive bioinformatics methods.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">In this study, we constructed a ChIP-seq method that can be used to uncover the global binding sites by phiC31 integrase. 6740 potential Î¦C31 integrase binding sites were identified. A sequence motif was found that contains inverted repeats and has similarities to wild-type attP site. Using REPEATMASKER, we identified a total of 20,183 repeat-regions distributed in 50 repeat types for the 6740 binding sites. These sites enriched in âregulation of GTPase activityâ of in the GO category of biological process and KEGG pathway of signal transmembrane transporter activity.</p>
              </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">This study is the first time to uncover the global map of binding sites for Î¦C31 integrase using ChIP-sequencing method and analysis the features of these binding sites. This method will help us to fully understand the mechanism of the site-specific integration function by phiC31 integrase and will potentially boost its genetic manipulations in both gene therapy and generation of transgenic animals.</p>
              </div>
</div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords">
<li class="c-keywords__item" itemprop="about">Î¦C31 integrase</li>
<li class="c-keywords__item" itemprop="about">Binding sites</li>
<li class="c-keywords__item" itemprop="about">ChIP-seq</li>
<li class="c-keywords__item" itemprop="about">Sequence motif</li>
</ul></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">The Streptomyces phage Î¦C31 integrase is a site-specific recombinase that catalyzes recombination between two short DNA sequences, a bacterial attachment site (attB) and a phage attachment site (attP) [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. As prokaryotic attachment sites, attP and attB are unlikely to occur in the genomes of mammals and most plants, but pseudo attP sites are present in the eukaryotic genome [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>â<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], Î¦C31 integrase can efficiently target attB-bearing transgenes to endogenous pseudo attP sites within mammalian genomes [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Because Î¦C31 integrase-mediated recombination is stable, irreversible and does not require external chemical energy and cofactors, Î¦C31 integrase has become a powerful tool for both gene therapy animals [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>â<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] and generation of transgenic animals [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>â<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. However, with the use of Î¦C31 integrase system in genetic manipulations, an increasing number of problems is being encountered. On one hand, the presence of a large number of pseudo attP sites in the genome increases the risk of chromosomal rearrangements and DNA damage [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. On the other hand, due to the high activity of the integrase system in the genome, the specificity does not appear to be as high as expected. Therefore, it is necessary to further study the mechanism of Î¦C31 integrase system in mammalian cells, in order to further optimize the system, improve its integration specificity and reduce the non-specific recombination.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Studies on sequence characteristics of pseudo attP sites in many species revealed that sequence characteristic is one of the most important factors for Î¦C31 integrase to recognize the pseudo attP sites and mediate site-specific integration in eukaryotic cells [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>â<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Meanwhile the pseudo attP sites in different cell types of the same species are cell-specific. For example, the BpsM1 site we found in bovine maden-darby bovine kidney (MDBK)cell line was not detected in bovine ear fibroblasts [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. The 19q13.31 site in the human genome is detectable in all cell types, and 3q26.31 appears only in HepG2 cells [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. These phenomena indicate that, in addition to sequence features, the recognition and binding of the integrase to the sites are also affected by other factors such as chromatin structure. However, this is still not fully studied and understood. The research on Î¦C31 integrase function in mammalian cells is still in its infancy. Indepth studies will accelerate its application in biological engineering. Especially, the main problems that need to be investigated include several aspects. First of all, although about 10<sup>2</sup>â10<sup>3</sup> pseudo attP sites have been predicted in mammalian cells [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], the actual number of potential sites we didnât know. Thus the data on Î¦C31 integrase is too small to analyze its function and mechanism. Secondly, at present, common methods for identification of integration sites are plasmid rescue, reverse nested PCR and other methods. These methods have limitations and provide only a small amount of binding sites information.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Chromatin immunoprecipitation assay (ChIP) is a powerful tool for studying the interaction between protein and DNA in vivo. The ChIP-Seq technology, which combines ChIP with second-generation sequencing technology, can efficiently detect genome-wide DNA segments that interact with functional proteins [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>].</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">In this study we establish an expression platform for Î¦C31 integrase in bovine cells and identify the whole set of Î¦C31 integrase binding sites in these cells by using ChIP-Seq technology. We equally aimed to analyze the characteristics of these binding sites using bioinformatics.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section">
<section id="Sec3" class="Section2 RenderAsSection2"><h3 class="Heading">Generation of MDKB cells expressing Î¦C31 integrase</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">In this study, the Î¦C31 integrase gene sequence was isolated from pCMV-INT plasmid and inserted into the pcDNA3.1-neo plasmid in order to construct a eukaryotic selection vector for generating Î¦C31 integrase cell lines. The constructed expression vector pcDNA3.1-int-neo (7272âbp) was validated by enzyme digestion and sequencing. MDKB cells were transfected with the constructed plasmid pcDNA3.1-int-neo. After G418 selection, PCR analysis using specific primers for Î¦C31 integrase gene and only clones harboring the Î¦C31 integrase gene were used for further analysis. The presence of Î¦C31 integrase gene was further confirmed by sequencing. To verify the effective expression of Î¦C31 integrase, western blot was performed for Î¦C31 integrase in wild type and engineered MDKB cells. As shown in Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>, Î¦C31 integrase was expressed, indicating that a MDKB cell line expressing the Î¦C31 integrase was effectively established.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig1_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig1_HTML.png" alt="Fig. 1"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Verification of ÏC31 integraseexpression in the established MDBKcells. <strong class="EmphasisTypeBold">a</strong> Western blot analysis of ÏC31 integrase expression in the MDBK cells. <strong class="EmphasisTypeBold">b</strong> densitometric analysis of the representative bands obtained from thewestern blot analysis. <strong class="EmphasisTypeBold">c</strong> Real time PCR analysis of ÏC31 integrase expressionin the MDBK cells. Î²-actin was used as endogenous control for western blotanalysis while the GAPDHgene was employed as housekeeper gene in thereal time PCR analysis</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec4" class="Section2 RenderAsSection2"><h3 class="Heading">Global mapping of binding sites for Î¦C31 integrase</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">We performed the ChIP-Seq experiments in order to uncover the whole genome binding sites for Î¦C31 integrase. The depth of sequencing data of the genome was obtained by analyzing the coverage of the reference sequence and the depth information of the genome while the gene profiling and the depth distribution of the upstream and downstream intervals were obtained by analyzing the depth distribution of the gene in the interval between the gene ontology and the upstream and downstream 2âk intervals (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figures S1-S4). The statistics of the ChIP-Seq data before and after quality filtering is summarized in Table <span class="InternalRef"><a href="#Tab1">1</a></span>. A total of 42,233,088 and 36,809,270 clean short reads were generated from the raw ChIP-Seq data.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Statistics of the ChIP-Seq data before and after quality filtering</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1">
<colgroup>
<col>
<col>
<col>
<col>
<col>
<col>
</colgroup>
<thead><tr class="header">
<th> </th>
<th><p xmlns="" class="SimplePara">Sample</p></th>
<th><p xmlns="" class="SimplePara">Number of sequences</p></th>
<th><p xmlns="" class="SimplePara">Base number (bp)</p></th>
<th><p xmlns="" class="SimplePara">Q20 (%)</p></th>
<th><p xmlns="" class="SimplePara">Q30 (%)</p></th>
</tr></thead>
<tbody>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">Raw reads</p></td>
<td><p xmlns="" class="SimplePara">INTMDBK_input</p></td>
<td><p xmlns="" class="SimplePara">46106506</p></td>
<td><p xmlns="" class="SimplePara">6915975900</p></td>
<td><p xmlns="" class="SimplePara">96.19</p></td>
<td><p xmlns="" class="SimplePara">92.37</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">INTMDBK_treat</p></td>
<td><p xmlns="" class="SimplePara">40239504</p></td>
<td><p xmlns="" class="SimplePara">6035925600</p></td>
<td><p xmlns="" class="SimplePara">96.02</p></td>
<td><p xmlns="" class="SimplePara">91.84</p></td>
</tr>
<tr class="noclass">
<td rowspan="2"><p xmlns="" class="SimplePara">Clean reads</p></td>
<td><p xmlns="" class="SimplePara">INTMDBK_input</p></td>
<td><p xmlns="" class="SimplePara">42233088</p></td>
<td><p xmlns="" class="SimplePara">6127524993</p></td>
<td><p xmlns="" class="SimplePara">98.34</p></td>
<td><p xmlns="" class="SimplePara">95.56</p></td>
</tr>
<tr class="noclass">
<td><p xmlns="" class="SimplePara">INTMDBK_treat</p></td>
<td><p xmlns="" class="SimplePara">36809270</p></td>
<td><p xmlns="" class="SimplePara">5411758088</p></td>
<td><p xmlns="" class="SimplePara">98.17</p></td>
<td><p xmlns="" class="SimplePara">95.08</p></td>
</tr>
</tbody>
</table></div></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Following quality control, we mapped the reads to the Reference Genome Consortium database. From the 42,233,088 reads obtained from INTMDBK_input 38,407,502 were mapped to the Reference Genome Consortium database (mapping rate of 90.94%) whereas 30,675,258 were mapped from the 36,809,270 clean reads generated from the INTMDBK_treat sample (mapping rate of83.34%). These reads with unique alignment to a unique position on the genome was used for subsequent information analysis. The 66 captured sites (markers) information and location was reported in Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Based on a certain analysis model in the entire genome range, the peaks (ChIP-Seq enrichment regions) were scanned to obtain the data on peak location in the genome and the peak region sequence information. The peak search output file was saved into wiggle file format and uploaded to the UCSC Genome Browser for visualization. We identified a total of 6740 ChIP-Seq peaks with total peak length of 4,136,812 and peak mean length of 613.77. The total peak depth was 2,035,208 while the peak mean depth was 301.96 for the INTMDBK_treat sample.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2"><h3 class="Heading">Features of the binding sites</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The distribution characteristics of gene functional elements of peaks showed that peaks were composed of exons, introns, upstream, downstream, intergenic and other functional elements (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>). The peak calling output file and peak annotation are reported in Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig2_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig2_HTML.png" alt="Fig. 2"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">The distribution characteristics of gene functional elements of peaks. Peaks were distributed into exons, introns, upstream, downstream, intergenic and other functional elements</p>
                      </div></figcaption></figure>
</div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">To determine whether our ChIP-seq experiment recovered the DNA motif assumed to be responsible for sequence-specific Î¦C31 integraseâDNA binding, we applied de novo motif discovery to the ChIP-seq peak regions. Using the MEME motif discovery tool, we searched all regions for enriched sequence patterns without any assumption. A highly significant motif (Eâ=â1.4e-043) was found in 236 sequences (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). The motif was 21âbp long and contained inverted repeats around the core.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig3_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig3_HTML.png" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig3_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig3_HTML.png" alt="Fig. 3"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">ChIP-seq peaks contain ÏC31 integrase-binding motifs. The multilevel consensus derived from the motif predicted by the MEME finder. The consensus is symmetric around the core and contains inverted repeats that extend over the length of the consensus</p>
                      </div></figcaption></figure>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Using REPEATMASKER, we identified a total of 21,065 repeat-regions distributed in 50 repeat types for the 6740 binding sites (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>). The repeat-types included SINE/tRNA-Core-RTE, SINE/MIR, LINE/L1, LINE/L2, SINE/tRNA and DNA/hAT-Ac. We found that 6740 repeat-regions were located in the upstream regions of Î¦C31 integrase binding sites, 6740 in the downstream and 7585 were overlapped with the binding sites.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2"><h3 class="Heading">Molecular pathways of ChIP-Seq-based Î¦C31 binding sites</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">To uncover the molecular pathways in which are involved the set of 6740 peaks, we performed GO and KEGG analysesby using bioinformatics tools. In GO functional analysis (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>, Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>), the most significant GO terms included âregulation of GTPase activityâ (GO:0043087; <em xmlns="" class="EmphasisTypeItalic">p-</em>FDRâ=â0) for biological process, âmetal ion transmembrane transporter activityâ (GO:0046873; <em xmlns="" class="EmphasisTypeItalic">p-</em>FDRâ=â0) for molecular function, and âmembrane regionâ (GO:0098589; <em xmlns="" class="EmphasisTypeItalic">p-</em>FDR <em xmlns="" class="EmphasisTypeItalic">=â0</em>) for cellular component. The Directed Acyclic Graph (DAG) of biological processes, molecular functions and cellular components were subsequently performed in this study to generate the graphical representation of the results of enrichment analysis showing the inclusion relation and the range of functions (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figures S5-S7). KEGG pathway analysis was performed using the KOBAS software (<span class="ExternalRef"><a href="http://kobas.cbi.pku.edu.cn" target="_blank" class="ext-link"><span class="RefSource">http://kobas.cbi.pku.edu.cn</span></a></span>). The results (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>, Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>) showed that Î¦C31 integrase target genes were significantly associated with âcGMP-PKG signaling pathwayâ (ko04022, 43 genes,<em xmlns="" class="EmphasisTypeItalic">P</em>â=â2.16E-05), âFocal adhesionâ (ko04510,49genes, <em xmlns="" class="EmphasisTypeItalic">P</em>â=â3.95E-05) and âAdrenergic signaling in cardiomyocytesâ (ko04261, 37 genes, <em xmlns="" class="EmphasisTypeItalic">P</em>â=â0.000122399).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig4_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig4_HTML.png" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig4_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig4_HTML.png" alt="Fig. 4"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">GO enrichment analysis of the ÏC31 integrase target genes. The GO term which satisfied the condition was defined as the GO term which was significantly enriched in the peak-related gene if the Bonferroni corrected <em class="EmphasisTypeItalic">p</em>-value was &lt;â0.05</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig5_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig5_HTML.png" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig5_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig5_HTML.png" alt="Fig. 5"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">KEGG pathway enrichment analysis of the ÏC31 integrase target genes. KEGG pathway analysis was performed using the KOBAS software The KO term which satisfied the condition was defined as the KO term which was significantly enriched in the peak-related gene if the Bonferroni corrected p-value was &lt;â0.05</p>
                      </div></figcaption></figure>
</div>
              </section><section id="Sec7" class="Section2 RenderAsSection2"><h3 class="Heading">qPCR validation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">To validate the functional Î¦C31 integrase binding to the genome, we performed qRT-PCR assay to verify the binding of Î¦C31 integrase to candidate hotspot binding sites, namely BF27, BpsF1, BpsM1 and BF4a that we had found [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. Indeed, we found that BF27, BpsF1, BpsM1 and BF4a were respectively enriched by 13.2, 13.05, 2.41 and 1.76 fold in the Int-MDKB cells (Fig. <span class="InternalRef"><a href="#Fig6">6</a></span>). These results demonstrated the effective binding of Î¦C31 integrase onto target binding sites in the engineered MDBK cells, thus confirming the validity of the ChIP-Seq data at some extent.<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig6_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig6_HTML.png" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig6_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_Fig6_HTML.png" alt="Fig. 6"></noscript></a>
                    </div>
<figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent">
<span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Real time PCR analysis of the ChIP-PCR products of ÏC31 integrase expression for previously studied binding sites</p>
                      </div></figcaption></figure>
</div>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec8" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section">
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">The development of genetic engineering techniques for the introduction of gene sequences into mammalian cells has many potential important applications, such as gene therapy, mammary gland bioreactor and mechanism study on gene function [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] The Î¦C31 integrase-mediated gene delivery system has the ability to integrate transgenes into mammalian genomes at specific locations, and guarantees high expression levels of recombinant protein [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>â<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. Building non-viral expression vectors for Î¦C31 integrase will provide an important and safe tool for gene therapy and the generation of transgenic organisms [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>â<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Furthermore, though Î¦C31 integrase has been applied for site-directed gene integration in the mammalian genome, the features of Î¦C31 integrase binding sites are not well defined. Indeed, although about 10<sup>2</sup>â10<sup>3</sup> pseudo attP sites have been predicted in mammalian cells [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], the number found in humans and mice is very small. In addition, though we have found 32 integration sites in the bovine genome using reverse nested PCR [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>], the actual number of these sites in the genome is very different. Thus, the data on Î¦C31 integrase has been found too small to analyze its function and mechanism. Therefore, in this study, we constructed a Î¦C31 integrase eukaryotic expression vector and generated a stable MDBK cell line expressing Î¦C31 integrase to characterize the binding sites of Î¦C31 integrase in the whole bovine genome.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">After having confirmed the effective expression of Î¦C31 integrase, we applied the ChIP-Seq method to uncover the global map of Î¦C31 integrase binding sites. A total of 6740 binding sites were identified to be significantly enriched Î¦C31 integrase binding regions in the MDBK cell genome. This study is the first of its kind to uncover the global map of binding sites for Î¦C31 integrase in the bovine genome. By analyzing genomic DNA sequences, we identified a highly significant sequence motif among the binding sites. This motif contains inverted repeats and has similarities to wild-type attP site. We also have found that these sites contain repetitive DNA. These results strongly suggest that there are sequence characteristics that define pseudo attP sites in the bovine genome, and that the recognition of these sequences is an important feature of site-specific Î¦C31 integrase activity in MDBK cell. Repetitive elements are presumably recognized as a suitable substitute for attP sequences.</p>
<p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Through GO analysis, the most significant GO terms of these genes around the Î¦C31 integrase binding sites were âregulation of GTPase activityâ in the GO category of biological process, âmetal ion transmembrane transporter activityâ in the GO category of molecular function, and âmembrane regionâ in the GO category of cellular component. Using KEGG pathway analysis with KOBAS, we found that the genes located in the core significant pathways were associated with âcGMP-PKG signaling pathwayâ and âFocal adhesionâ. These results may explain why the Î¦C31 integrase does not necessitate external energy and cofactors as it is able to catalyze the metabolism of energy and microelement pathways. Maybe this hypothesis is wrong because we didnât have enough direct evidences. Also we donât know what influence will happen to the location genes expression if a foreign gene is integrated nearby by Î¦C31 integrase.The present study has some limitations since we analyzed only a single ChIP-Seq dataset without animal in vivo experimental validation following ChIP-Seq. However, due to the fact that the engineered cells were well characterized and the ChIP-Seq results were validated by real time PCR, it is probable that sequences obtained here are potential Î¦C31 integrase target binding sites. This time, our purpose was to validate that the ChIP-seq method can be used to uncover the global binding sites by phiC31 integrase and next time we will take some strategies to optimize our selecting system, in order to get rid of the random binding sites of phiC31 integrase in the bovine genome.</p>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Our study uncovered a large set of Î¦C31 integrase binding sites in the bovine genome and analysis the features of these binding sites. This method will help us to fully understand the mechanism of the site-specific integration function by phiC31 integrase and will potentially boost its genetic manipulations in both gene therapy and generation of transgenic animals.</p></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section">
<section id="Sec11" class="Section2 RenderAsSection2"><h3 class="Heading">Plasmid construction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">The pCMV-Int plasmid harboring Î¦C31 integrase was a kind gift of Prof. M. P. Calos (Stanford University School of Medicine, Stanford, USA). The plasmid pCMV-Int was amplified by PCR using specific forward (5-AAGCTTGGTACCGGTCCGGAATTCCC-3) and reverse (5-CTCGCTGGATCCGGGTGTCGCTAC-3) primers. After digestion by HindIII and xhoI restriction enzymes, the Î¦C31 gene fragment was isolated from the PCR product and cloned into the recipient plasmid pcDNA3.1-neo digested by the same restriction enzymes after overnight connection at 16âÂ°C to generate Î¦C31 integrase eukaryotic expression vector pcDNA3.1-int-neo.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2"><h3 class="Heading">Establishment of MDKB cells expressing Î¦C31 and verification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">MDKB cells were obtained from the Cell Bank of the Chinese Academy of Sciences. All reagents were obtained from Invitrogen unless otherwise indicated. The cells were maintained on dulbeccoâs modified eagle medium (DMEM). Fresh medium was provided to the cells every day, and the cells were passaged every 4â5âdays. One day prior to transfection with Lipofectamine 2000 (Invitrogen), cells were treated with Accutase (Sigma-Aldrich, St. Louis) and plated on Matrigel in conditioned medium. Lipofectamine 2000 transfection was carried out according to the manufacturerâs protocol. We typically used 4âÎ¼g of the Î¦C31 integrase expression vector pcDNA3.1-int-neo to transfect 2 million cells. After 14â21âdays of selection, individual colonies of MDKB cells transformed with the expression vector were manually picked and expanded for further analysis. DNA was extracted from the transgenic colonies and negative MDKB cells with the phenol chloroform procedure.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The Î¦C31 integrase expression was validated by western blot analysis. MDBK cells were washed with 3âml 4âÂ°C precooled PBS and heated at 80âÂ°C for 30âmin in RIPA buffer(150âmM Nacl, 50âmM Tris-HCL pHâ8, 0.1% SDS, 0.5% Deoxycholate, 1âmM EDTA, 1% NP-40), and then centrifuged for 5âmin at 10000âg. Next, 20âÎ¼g total proteins were eledectrophoresed in a 10% SDS-PAGE gel, and then electrically transferred onto PVDF membranes using a semidry method. Ã-actin was used as endogenous control. Detection was performed using the ECL-plus enhanced chemiluminescence system (ECL Amersham Biosciences) after auto radiographic exposure to HyperfilmTM.ECL (Amersham Biosciences). The program ImageMaster I-D (Pharmacia, Sweden) was used for quantitation of bands by densitometry.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2"><h3 class="Heading">Chromatin immunoprecipitation, ChIP-sequencing and ChIP-qPCR</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">MDBK or Int-MDKB cells (expressing Î¦C31 integrase) (1âÃâ10<sup>7</sup> cells) were inoculated into 10âmL DMEM medium and cultured to reach 80â90% confluence. Thereafter, cells were fixed with 1% formaldehyde for 15âmin cross-linking and quenched with 450âÎ¼L 2.5âM glycine. After washing cells three times with ice-cold PBS, chromatin was isolated by adding 1âml SDS lysis buffer (150âMm Nacl, 1% Triton X-100, 25âmM Tris pHâ7.5, 0.1%SDS, 0.5% Deoxycholate) followed by disruption with a Dounce homogenizer. Lysates were sonicated (VCX750) to shear the DNA to an average length of 200â500âbp, followed by centrifugation at 15000âg for 10âmin to remove cell debris. 300âÎ¼L of supernatant were divided into 3 aliquots of 100âÎ¼L: 100âÎ¼L for the experimental group into which was added 1â10âÎ¼g anti-Î¦C31 antibodies, 100âÎ¼L for the control group without addition of antibodies and 100âÎ¼L to which was added 4âÎ¼L 5âM NaCl. The 3 aliquots were submitted to 3âh cross-linking at 65âÂ°C followed by agarose gel electrophoresis for the evaluation of ultrasonic disruption results. Thereafter, 900âÎ¼L ChIP Dilution Buffer (0.01%SDS, 1% Triton X-100, 1% Triton X-100, 1.2âmM EDTA, 16.7âmM Tris-Hcl, 167âmM Nacl pHâ8.1) and 20âÎ¼L 50âÃâPIC were added in the experimental and control samples, and cleared with 60 Î¼lProtein AAgarose/Salmon Sperm DNA. 20âÎ¼L genomic DNA (input) was purified from both aliquots of chromatin and quantified on a Nanodrop spectrophotometer. Extrapolation to the original chromatin volume allowed quantitation of the total chromatin yield. ChIP assays were carried out as follows: An aliquot of chromatin (50âmg) was precleared with protein Aagarose beads (Invitrogen). Î¦C31-bound genomic DNA fragments were isolated using an anti-bacteriophage Î¦C31 integrase antibody (ab93248). After incubation at 4âÂ°C overnight, protein Aagarose beads were used to isolate the immune complexes. Complexes were washed, eluted from the beads using SDS buffer (1%SDS, 0.1âM NaHCO<sub>3</sub>), and subjected to RNase and proteinase K treatment. Crosslinks were reversed by incubation overnight at 65âÂ°C, and ChIP DNA was purified by PCR Purification Kit (QIAGEN).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">We generated one DNA library using the Î¦C31 integrase-specific antibodies incubated with crosslinked and sonicated DNA from Int-MDKB cells. A second control library containing the input DNA was prepared from MDBK cells. About 20â30âng of the ChIP DNA from each library was prepared for sequencing using the Illumina TruSeq Nano DNA Sample Preparation Kit according to the kit manual. The DNA libraries were sequenced with HiSeq 2500 machine using TruSeq Rapid SBS Kit-HS reagents (Illumina).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">To ensure the accuracy of subsequent bioinformatics analysis, the original sequencing data was filtered to obtain high-quality sequencing data (clean data). Quality control of the sequencing data was performed using Sickle (<span class="ExternalRef"><a href="https://github.com/najoshi/sickle" target="_blank" class="ext-link"><span class="RefSource">https://github.com/najoshi/sickle</span></a></span>) and SeqPrep (<span class="ExternalRef"><a href="https://github.com/jstjohn/SeqPrep" target="_blank" class="ext-link"><span class="RefSource">https://github.com/jstjohn/SeqPrep</span></a></span>). The sequencing output raw reads were trimmed by stripping the adaptor sequences and ambiguous nucleotides and reads with quality scores less than 20 and lengths below 20âbp were removed.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">Equal amounts of DNA for each sample were combined with a SYBR Green PCR master mix (Qiagen) and specific primers. Amplification reactions were then performed with an Applied Biosystems 7900 real-time PCR system. The primers used for BF27 binding site were 5-GTACTTATTTTTGTATTTGAAACCACCT and TCTGGGTGTTTTTACATT-3; for BF4a, 5-CCATAAAAGGAATATACTTGAAA-3 and 5-ACCAACATGGCAATCGGGGACAT-3; for BPSF1, 5-GGACCTAGAAGGGGCTCATAACT-3 and 5-TGTATACTCACACCACTGTCCTA-3; for BPSM1, 5-GCCGTTTGTTGGGTTAGCTTTTCAGAA-3 and 5-GTCATCCTGCCTTGCAGGCTGTTTCTT-3; for GAPDH, 5-TTCACCACCATGGAGAAGG-3 and 5-GGCATGGACTGTGGTCATGA-3. Use AppliedBiosystems SDS2.4 and RQ manager software to analyze data.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2"><h3 class="Heading">ChIP-Seq library peak finding</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">To identify significant Î¦C31 binding sites in the sequencing data, the whole genome peak scan was performed using the UCSC Genome Browser following the UCSC Genome Browser Instructions (<span class="ExternalRef"><a href="http://genome.ucsc.edu" target="_blank" class="ext-link"><span class="RefSource">http://genome.ucsc.edu</span></a></span>). UCSC provides genomic reference sequences for a large number of species, including a large number of gene annotation information and comparative genomics information.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2"><h3 class="Heading">MEME motif discovery analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">The MEME web service (<span class="ExternalRef"><a href="http://meme-suite.org" target="_blank" class="ext-link"><span class="RefSource">http://meme-suite.org</span></a></span>) was ran with the command line summary below: meme INTMDBK_treat.peaks.fa -dna -mod zoops -revcomp -nmotifs3 -maxw30 -minw 6 -maxsize 26,000,000 -oc. /INTMDBK_treat_meme.GO functional enrichment analysis of Peak related genes.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">Gene Ontology (GO) is an internationally standardized gene function classification system that provides a set of dynamically updated standard vocabularies to comprehensively describe the properties of genes and gene products in organisms. GO has a total of three ontologies describing the molecular function of the gene, the cellular component in which it is located, and the biological process involved. GO is the basic unit (term, node), each term corresponds to a property.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">The GO enrichment analysis was used to map all the peak-related genes to terms in the Gene Ontology database (<span class="ExternalRef"><a href="http://www.geneontology.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.geneontology.org/</span></a></span>) and calculate the number of genes for each term. The hypergeometric test was applied to find out significant GO enrichment of peak-related genes based on the entire genome background using the formula [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]:<div id="Equa" class="Equation EquationMathjax"><div class="EquationContent">$$ P=1-\sum \limits_{i=0}^{m-1}\frac{\left(\begin{array}{c}M\\ {}i\end{array}\right)\left(\begin{array}{c}N-M\\ {}n-i\end{array}\right)}{\left(\begin{array}{c}N\\ {}n\end{array}\right)} $$</div></div>
</div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">Where N is the number of genes with GO annotation in the genome; n is the number of peak-related genes in N; M is the number of genes annotated for a particular GO term in the genome; m is the number of peak-associated genes annotated for a particular GO term. The GO term which satisfied the condition was defined as the GO term which was significantly enriched in the peak-related gene if the Bonferroni corrected <em xmlns="" class="EmphasisTypeItalic">p</em>-value was &lt;â0.05. The Directed Acyclic Graph (DAG), a graphical representation of the results of enrichment analysis of peak-related genes was also equally performed.</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2"><h3 class="Heading">KEGG pathway enrichment analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">This study used KOBAS (<span class="ExternalRef"><a href="http://kobas.cbi.pku.edu.cn" target="_blank" class="ext-link"><span class="RefSource">http://kobas.cbi.pku.edu.cn</span></a></span>) for KEGG pathway enrichment analysis. The calculation principle was the same as GO enrichment analysis. Terms with corrected p-value of &lt;â0.05 was considered significant.</p>
              </section>
</div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry">
<dt class="Term"><dfn>Chip-seq: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Chromatin immunoprecipitation-DNA sequencing</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>GO: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Gene Ontology</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>KEGG: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Kyoto Encyclopedia of Genes and Genomes</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>MDBK: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Maden-darby bovine kidney</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>MEME: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Multiple Em for Motif Elicitation</p>
                  </dd>
</dl>
<dl class="DefinitionListEntry">
<dt class="Term"><dfn>Î¦C31: </dfn></dt>
<dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">PhiC31</p>
                  </dd>
</dl>
</div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments">
<h3 xmlns="" class="Heading">Acknowledgments</h3>
<p xmlns="" class="SimplePara">We thank Mr. Noel Komi for valuable advice on the manuscript.</p>
<div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par40" class="Para">This work was supported by the National Natural Science Foundation of China (31301009, 81100463).</p>
              </div>
<div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par41" class="Para">The datasets used and/or analysed during the current study are available from the corresponding author on reasonable request.</p>
              </div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1">
<h3 xmlns="" class="Heading">Authorsâ contributions</h3>
<p xmlns="" class="SimplePara">LQ and SW lead the experimental design, facilitated data analysis and wrote the manuscript. LW and XZ conceived and guided the execution of the study. QO and YZ were responsible for sequencing data analysis. AM and GL were in charge of cell culture and chromatin immunoprecipitation. FS, XW and YD revised the manuscript. All authors read and approved the final manuscript.</p>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par42" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par43" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par44" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisherâs Note</h3>
                <p id="Par45" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div>
<aside class="Appendix" id="App1"><h3 class="Heading">Additional files</h3>
<div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_MOESM1_ESM.zip"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">GO functional analysis output file. <strong class="EmphasisTypeBold">Figure S1.</strong> Genome sequencing depth cumulative distribution obtained from the INPUT sample. <strong class="EmphasisTypeBold">Figure S2.</strong> Gene and upstream and downstream sequences depth distribution map obtained from the INPUT sample. <strong class="EmphasisTypeBold">Figure S3.</strong> Genome sequencing depth cumulative distribution obtained from the INTMDBK_treat sample. <strong class="EmphasisTypeBold">Figure S4.</strong> Gene and upstream and downstream sequences depth distribution mapobtained from the INTMDBK_treat sample. <strong class="EmphasisTypeBold">Figure S5.</strong> Directed Acyclic Graph (DAG) of âBiological Processesâ obtained from GO enrichment analysis. <strong class="EmphasisTypeBold">Figure S6.</strong> Directed Acyclic Graph (DAG) of âCellular Componentsâ obtained from GO enrichment analysis. <strong class="EmphasisTypeBold">Figure S7.</strong> Directed Acyclic Graph (DAG) of âMolecular Functionsâ obtained from GO enrichment analysis. (ZIP 726 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_MOESM2_ESM.xlsx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">Captured ÏC31 integrase binding sites (markers) information and location. (XLSX 15 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_MOESM3_ESM.xlsx"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">The peak calling output file and peak annotation are reported. (XLSX 109 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_MOESM4_ESM.xlsx"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara">KEGG functional analysis output file. (XLSX 855 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container">
<a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0079-z/MediaObjects/41065_2018_79_MOESM5_ESM.xlsx"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara">GO functional analysis output file. (XLSX 29 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside>
</div></section>
</div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authorsâ Affiliations</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations">
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1">
<span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Laboratory Medicine, Shanghai Eighth Peopleâs Hospital, Shanghai, 200040, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2">
<span class="AffiliationNumber">(2)</span><div class="AffiliationText">Department of Respiratory Medicine, Shanghai First Peopleâs Hospital, Shanghai Jiaotong University School of Medicine, Shanghai, 201620, China</div>
</div>
<div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3">
<span class="AffiliationNumber">(3)</span><div class="AffiliationText">Lab of Reproductive Pharmacology, NHC Key Lab of Reproduction Regulation, Shanghai Institute of Planned Parenthood Research, Fudan University, Shanghai, 200032, China</div>
</div>
</div></section></div>
                        <div class="FulltextWrapper">
<section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper">
<li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kuhstoss S, Rao RN. Analysis of the integration function of the streptomycetebacteriophage phiC31. J Mol Biol. 1991;222:897â908.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0022-2836(91)90584-S"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20integration%20function%20of%20the%20streptomycetebacteriophage%20phiC31&amp;author=S.%20Kuhstoss&amp;author=RN.%20Rao&amp;journal=J%20Mol%20Biol&amp;volume=222&amp;pages=897-908&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ginsburg DS, Calos MP. Site-specific integration with phiC31 integrase for prolonged expression of therapeutic genes. Adv Genet. 2005;54:179â87.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0065-2660(05)54008-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Site-specific%20integration%20with%20phiC31%20integrase%20for%20prolonged%20expression%20of%20therapeutic%20genes&amp;author=DS.%20Ginsburg&amp;author=MP.%20Calos&amp;journal=Adv%20Genet&amp;volume=54&amp;pages=179-187&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hillman RT, Calos MP. Site-specific integration with bacteriophage PhiC31 integrase. Cold Spring Harb Protoc. 2012. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1101/pdb.prot069211" target="_blank" class="ext-link"><span class="RefSource">https://doi.org/10.1101/pdb.prot069211</span></a></span>.</span><span class="Occurrences"></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ni W, Hu S, Qiao J, Wang Y, Shi H, Wang Y, He Z, Li G, Chen C. Î¦C31integrase mediates efficient site-specific integration in sheep fibroblasts. Biosci Biotechnol Biochem. 2012;76:2093â5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1271/bbb.120439"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%CE%A6C31integrase%20mediates%20efficient%20site-specific%20integration%20in%20sheep%20fibroblasts&amp;author=W.%20Ni&amp;author=S.%20Hu&amp;author=J.%20Qiao&amp;author=Y.%20Wang&amp;author=H.%20Shi&amp;author=Y.%20Wang&amp;author=Z.%20He&amp;author=G.%20Li&amp;author=C.%20Chen&amp;journal=Biosci%20Biotechnol%20Biochem&amp;volume=76&amp;pages=2093-2095&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ohtsuka M, Miura H, Mochida K, Hirose M, Hasegawa A, Ogura A, Mizutani R, Kimura M, Isotani A, Ikawa M, Sato M, Gurumurthy CB. One-step generation of multiple transgenic mouse lines using an improved pronuclear injection-based targeted Transgenesis (i-PITT). BMC Genomics. 2015;16:274.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-015-1432-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=One-step%20generation%20of%20multiple%20transgenic%20mouse%20lines%20using%20an%20improved%20pronuclear%20injection-based%20targeted%20Transgenesis%20%28i-PITT%29&amp;author=M.%20Ohtsuka&amp;author=H.%20Miura&amp;author=K.%20Mochida&amp;author=M.%20Hirose&amp;author=A.%20Hasegawa&amp;author=A.%20Ogura&amp;author=R.%20Mizutani&amp;author=M.%20Kimura&amp;author=A.%20Isotani&amp;author=M.%20Ikawa&amp;author=M.%20Sato&amp;author=CB.%20Gurumurthy&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=274&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ou HL, Huang Y, Qu LJ, Xu M, Yan JB, Ren ZR, Huang SZ, Zeng YT. A phiC31 integrase-mediated integration hotspot in favor of transgene expression exists in the bovine genome. FEBS J. 2009;276:155â63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1742-4658.2008.06762.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20phiC31%20integrase-mediated%20integration%20hotspot%20in%20favor%20of%20transgene%20expression%20exists%20in%20the%20bovine%20genome&amp;author=HL.%20Ou&amp;author=Y.%20Huang&amp;author=LJ.%20Qu&amp;author=M.%20Xu&amp;author=JB.%20Yan&amp;author=ZR.%20Ren&amp;author=SZ.%20Huang&amp;author=YT.%20Zeng&amp;journal=FEBS%20J&amp;volume=276&amp;pages=155-163&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thyagarajan B, Olivares EC, Hollis RP, Ginsburg DS, Calos MP. Site-specific genomic integration in mammalian cells mediated by phage phiC31 integrase. Mol Cell Bio. 2001;21:3926â34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1128/MCB.21.12.3926-3934.2001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Site-specific%20genomic%20integration%20in%20mammalian%20cells%20mediated%20by%20phage%20phiC31%20integrase&amp;author=B.%20Thyagarajan&amp;author=EC.%20Olivares&amp;author=RP.%20Hollis&amp;author=DS.%20Ginsburg&amp;author=MP.%20Calos&amp;journal=Mol%20Cell%20Bio&amp;volume=21&amp;pages=3926-3934&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Farruggio AP, Chavez CL, Mikell CL, Calos MP. Efficient reversal of phiC31 integrase recombination in mammalian cells. Biotechnol J. 2012;7:1332â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/biot.201200283"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Efficient%20reversal%20of%20phiC31%20integrase%20recombination%20in%20mammalian%20cells&amp;author=AP.%20Farruggio&amp;author=CL.%20Chavez&amp;author=CL.%20Mikell&amp;author=MP.%20Calos&amp;journal=Biotechnol%20J&amp;volume=7&amp;pages=1332-1336&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chavez CL, Keravala A, Chu JN, Farruggio AP, Cuellar VE, Voorberg J, Calos MP. Long-term expression of human coagulation factor VIII in a tolerant mouse model using the phiC31 integrase system. Hum Gene Ther. 2012;23:390â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/hum.2011.110"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Long-term%20expression%20of%20human%20coagulation%20factor%20VIII%20in%20a%20tolerant%20mouse%20model%20using%20the%20phiC31%20integrase%20system&amp;author=CL.%20Chavez&amp;author=A.%20Keravala&amp;author=JN.%20Chu&amp;author=AP.%20Farruggio&amp;author=VE.%20Cuellar&amp;author=J.%20Voorberg&amp;author=MP.%20Calos&amp;journal=Hum%20Gene%20Ther&amp;volume=23&amp;pages=390-398&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Keravala A, Chavez CL, Hu G, Woodard LE, Monahan PE, Calos MP. Long-term phenotypic correction in factor IX knockout mice by using phiC31 integrase-mediated gene therapy. Gene Ther. 2011;18:842â8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gt.2011.31"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Long-term%20phenotypic%20correction%20in%20factor%20IX%20knockout%20mice%20by%20using%20phiC31%20integrase-mediated%20gene%20therapy&amp;author=A.%20Keravala&amp;author=CL.%20Chavez&amp;author=G.%20Hu&amp;author=LE.%20Woodard&amp;author=PE.%20Monahan&amp;author=MP.%20Calos&amp;journal=Gene%20Ther&amp;volume=18&amp;pages=842-848&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sivalingam J, Phan TT, Kon OL. Intragenic integration in DLC1 sustains factor VIII expression in primary human cells without insertional oncogenicity. Gene Ther. 2014;21:402â12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gt.2014.11"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Intragenic%20integration%20in%20DLC1%20sustains%20factor%20VIII%20expression%20in%20primary%20human%20cells%20without%20insertional%20oncogenicity&amp;author=J.%20Sivalingam&amp;author=TT.%20Phan&amp;author=OL.%20Kon&amp;journal=Gene%20Ther&amp;volume=21&amp;pages=402-412&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jabed A, Wagner S, McCracken J, Wells DN, Laible G. Targeted microRNA expression in dairy cattle directs production of beta-lactoglobulin-free, high-casein milk. Proc Natl Acad Sci U S A. 2012;109:16811â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1210057109"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Targeted%20microRNA%20expression%20in%20dairy%20cattle%20directs%20production%20of%20beta-lactoglobulin-free%2C%20high-casein%20milk&amp;author=A.%20Jabed&amp;author=S.%20Wagner&amp;author=J.%20McCracken&amp;author=DN.%20Wells&amp;author=G.%20Laible&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=109&amp;pages=16811-16816&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu H, Wang X, Zhu L, He Z, Liu G, Xu X, Chen J, Cheng G. Establishment of a rapid and scalable gene expression system in livestock by site-specific integration. Gene. 2013;515:367â71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2012.10.017"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Establishment%20of%20a%20rapid%20and%20scalable%20gene%20expression%20system%20in%20livestock%20by%20site-specific%20integration&amp;author=H.%20Yu&amp;author=X.%20Wang&amp;author=L.%20Zhu&amp;author=Z.%20He&amp;author=G.%20Liu&amp;author=X.%20Xu&amp;author=J.%20Chen&amp;author=G.%20Cheng&amp;journal=Gene&amp;volume=515&amp;pages=367-371&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Allen BG, Weeks DL. Using phiC31 integrase to make transgenic Xenopus laevis embryos. Nat Protoc. 2006;1:1248â57.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2006.183"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Using%20phiC31%20integrase%20to%20make%20transgenic%20Xenopus%20laevis%20embryos&amp;author=BG.%20Allen&amp;author=DL.%20Weeks&amp;journal=Nat%20Protoc&amp;volume=1&amp;pages=1248-1257&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Imayoshi I, Hirano K, Kitano S, Miyachi H, Kageyama R. In vivo evaluation of phiC31 recombinase activity in transgenic mice. Neurosci Res. 2012;73:106â14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.neures.2012.02.008"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=In%20vivo%20evaluation%20of%20phiC31%20recombinase%20activity%20in%20transgenic%20mice&amp;author=I.%20Imayoshi&amp;author=K.%20Hirano&amp;author=S.%20Kitano&amp;author=H.%20Miyachi&amp;author=R.%20Kageyama&amp;journal=Neurosci%20Res&amp;volume=73&amp;pages=106-114&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li YE, Allen BG, Weeks DL. Using phiC31 integrase to mediate insertion of DNA in Xenopus embryos. Methods Mol Biol. 2012;917:219â30.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-1-61779-992-1_13"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Using%20phiC31%20integrase%20to%20mediate%20insertion%20of%20DNA%20in%20Xenopus%20embryos&amp;author=YE.%20Li&amp;author=BG.%20Allen&amp;author=DL.%20Weeks&amp;journal=Methods%20Mol%20Biol&amp;volume=917&amp;pages=219-230&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu J, Skjorringe T, Gjetting T, Jensen TG. phiC31 integrase induces a DNA damage response and chromosomal rearrangements in human adult fibroblasts. BMC Biotechnol. 2009;9:31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1472-6750-9-31"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=phiC31%20integrase%20induces%20a%20DNA%20damage%20response%20and%20chromosomal%20rearrangements%20in%20human%20adult%20fibroblasts&amp;author=J.%20Liu&amp;author=T.%20Skjorringe&amp;author=T.%20Gjetting&amp;author=TG.%20Jensen&amp;journal=BMC%20Biotechnol&amp;volume=9&amp;pages=31&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qu L, Ma Q, Zhou Z, Ma H, Huang Y, Huang S, Zeng F, Zeng Y. A profile of native integration sites used by phiC31 integrase in the bovine genome. J Genet Genomics. 2012;39:217â24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jgg.2012.03.004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20profile%20of%20native%20integration%20sites%20used%20by%20phiC31%20integrase%20in%20the%20bovine%20genome&amp;author=L.%20Qu&amp;author=Q.%20Ma&amp;author=Z.%20Zhou&amp;author=H.%20Ma&amp;author=Y.%20Huang&amp;author=S.%20Huang&amp;author=F.%20Zeng&amp;author=Y.%20Zeng&amp;journal=J%20Genet%20Genomics&amp;volume=39&amp;pages=217-224&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thyagarajan B, Liu Y, Shin S, Lakshmipathy U, Scheyhing K, Xue H, Ellerstrom C, Strehl R, Hyllner J, Rao MS, Chesnut JD. Creation of engineered human embryonic stem cell lines using phiC31 integrase. Stem Cells. 2008;26:119â26.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1634/stemcells.2007-0283"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Creation%20of%20engineered%20human%20embryonic%20stem%20cell%20lines%20using%20phiC31%20integrase&amp;author=B.%20Thyagarajan&amp;author=Y.%20Liu&amp;author=S.%20Shin&amp;author=U.%20Lakshmipathy&amp;author=K.%20Scheyhing&amp;author=H.%20Xue&amp;author=C.%20Ellerstrom&amp;author=R.%20Strehl&amp;author=J.%20Hyllner&amp;author=MS.%20Rao&amp;author=JD.%20Chesnut&amp;journal=Stem%20Cells&amp;volume=26&amp;pages=119-126&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sivalingam J, Krishnan S, Ng WH, Lee SS, Phan TT, Kon OL. Biosafety assessment of site-directed transgene integration in human umbilical cord-lining cells. Mol Ther. 2010;18:1346â56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/mt.2010.61"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biosafety%20assessment%20of%20site-directed%20transgene%20integration%20in%20human%20umbilical%20cord-lining%20cells&amp;author=J.%20Sivalingam&amp;author=S.%20Krishnan&amp;author=WH.%20Ng&amp;author=SS.%20Lee&amp;author=TT.%20Phan&amp;author=OL.%20Kon&amp;journal=Mol%20Ther&amp;volume=18&amp;pages=1346-1356&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chalberg TW, Portlock JL, Olivares EC, Thyagarajan B, Kirby PJ, Hillman RT, Hoelters J, Calos MP. Integration specificity of phage phiC31 integrase in the human genome. J Mol Biol. 2006;357:28â48.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jmb.2005.11.098"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integration%20specificity%20of%20phage%20phiC31%20integrase%20in%20the%20human%20genome&amp;author=TW.%20Chalberg&amp;author=JL.%20Portlock&amp;author=EC.%20Olivares&amp;author=B.%20Thyagarajan&amp;author=PJ.%20Kirby&amp;author=RT.%20Hillman&amp;author=J.%20Hoelters&amp;author=MP.%20Calos&amp;journal=J%20Mol%20Biol&amp;volume=357&amp;pages=28-48&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Blecher-Gonen R, Barnett-Itzhaki Z, Jaitin D, Amann-Zalcenstein D, Lara-Astiaso D, Amit I. High-throughput chromatin immunoprecipitation for genome-wide mapping of in vivo protein-DNA interactions and epigenomic states. Nat Protoc. 2013;8:539â54.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2013.023"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-throughput%20chromatin%20immunoprecipitation%20for%20genome-wide%20mapping%20of%20in%20vivo%20protein-DNA%20interactions%20and%20epigenomic%20states&amp;author=R.%20Blecher-Gonen&amp;author=Z.%20Barnett-Itzhaki&amp;author=D.%20Jaitin&amp;author=D.%20Amann-Zalcenstein&amp;author=D.%20Lara-Astiaso&amp;author=I.%20Amit&amp;journal=Nat%20Protoc&amp;volume=8&amp;pages=539-554&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grandchamp N, Altemir D, philippe S, Ursulet S, Pilet H, Serre MC, Lenain A, Serguera C, Mallet J, Sarkis C. Hybrid lentivirus-phiC31-int-NLS vector allows site-specific recombination in murine and human cells but induces DNA damage. PLoS One. 2014;9:e99649.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0099649"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hybrid%20lentivirus-phiC31-int-NLS%20vector%20allows%20site-specific%20recombination%20in%20murine%20and%20human%20cells%20but%20induces%20DNA%20damage&amp;author=N.%20Grandchamp&amp;author=D.%20Altemir&amp;author=S.%20philippe&amp;author=S.%20Ursulet&amp;author=H.%20Pilet&amp;author=MC.%20Serre&amp;author=A.%20Lenain&amp;author=C.%20Serguera&amp;author=J.%20Mallet&amp;author=C.%20Sarkis&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e99649&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Aneja MK, Imker R, Rudolph C. Phage phiC31 integrase-mediated genomic integration and long-term gene expression in the lung after nonviral gene delivery. J Gene Med. 2007;9:967â75.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/jgm.1090"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phage%20phiC31%20integrase-mediated%20genomic%20integration%20and%20long-term%20gene%20expression%20in%20the%20lung%20after%20nonviral%20gene%20delivery&amp;author=MK.%20Aneja&amp;author=R.%20Imker&amp;author=C.%20Rudolph&amp;journal=J%20Gene%20Med.&amp;volume=9&amp;pages=967-975&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chalberg TW, Genise HL, Vollrath D, Calos MP. phiC31 integrase confers genomic integration and long-term transgene expression in rat retina. Invest Ophthalmol Vis Sci. 2005;46:2140â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1167/iovs.04-1252"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=phiC31%20integrase%20confers%20genomic%20integration%20and%20long-term%20transgene%20expression%20in%20rat%20retina&amp;author=TW.%20Chalberg&amp;author=HL.%20Genise&amp;author=D.%20Vollrath&amp;author=MP.%20Calos&amp;journal=Invest%20Ophthalmol%20Vis%20Sci&amp;volume=46&amp;pages=2140-2146&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Held PK, Olivares EC, Aguilar CP, Finegold M, Calos MP, Grompe M. In vivo correction of murine hereditary tyrosinemia type I by phiC31 integrase-mediated gene delivery. Mol Ther. 2005;11:399â408.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ymthe.2004.11.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=In%20vivo%20correction%20of%20murine%20hereditary%20tyrosinemia%20type%20I%20by%20phiC31%20integrase-mediated%20gene%20delivery&amp;author=PK.%20Held&amp;author=EC.%20Olivares&amp;author=CP.%20Aguilar&amp;author=M.%20Finegold&amp;author=MP.%20Calos&amp;author=M.%20Grompe&amp;journal=Mol%20Ther&amp;volume=11&amp;pages=399-408&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Keravala A, Portlock JL, Nash JA, Vitrant DG, Robbins PD, Calos MP. PhiC31 integrase mediates integration in cultured synovial cells and enhances gene expression in rabbit joints. J Gene Med. 2006;8:1008â17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/jgm.928"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PhiC31%20integrase%20mediates%20integration%20in%20cultured%20synovial%20cells%20and%20enhances%20gene%20expression%20in%20rabbit%20joints&amp;author=A.%20Keravala&amp;author=JL.%20Portlock&amp;author=JA.%20Nash&amp;author=DG.%20Vitrant&amp;author=PD.%20Robbins&amp;author=MP.%20Calos&amp;journal=J%20Gene%20Med&amp;volume=8&amp;pages=1008-1017&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luo Y, Wang Y, Liu J, Lan H, Shao M, Yu Y, Quan F, Zhang Y. Production of transgenic cattle highly expressing human serum albumin in milk by phiC31 integrase-mediated gene delivery. Transgenic research Transgenic Res. 2015;24:875â83.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11248-015-9898-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Production%20of%20transgenic%20cattle%20highly%20expressing%20human%20serum%20albumin%20in%20milk%20by%20phiC31%20integrase-mediated%20gene%20delivery&amp;author=Y.%20Luo&amp;author=Y.%20Wang&amp;author=J.%20Liu&amp;author=H.%20Lan&amp;author=M.%20Shao&amp;author=Y.%20Yu&amp;author=F.%20Quan&amp;author=Y.%20Zhang&amp;journal=Transgenic%20research%20Transgenic%20Res&amp;volume=24&amp;pages=875-883&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nakanishi H, Higuchi Y, Yamashita F, Hashida M. Targeted gene integration using the combination of a sequence-specific DNA-binding protein and phiC31 integrase. J Biotechnol. 2014;186:139â47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jbiotec.2014.07.012"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Targeted%20gene%20integration%20using%20the%20combination%20of%20a%20sequence-specific%20DNA-binding%20protein%20and%20phiC31%20integrase&amp;author=H.%20Nakanishi&amp;author=Y.%20Higuchi&amp;author=F.%20Yamashita&amp;author=M.%20Hashida&amp;journal=J%20Biotechnol&amp;volume=186&amp;pages=139-147&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Otani Y, Kawakami S, Mukai H, Fuchigami Y, Yamashita F, Hashida M. Long-term in vivo gene expression in mouse kidney using phiC31 integrase and electroporation. J Drug Target. 2015;23:427â35.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3109/1061186X.2014.1002788"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Long-term%20in%20vivo%20gene%20expression%20in%20mouse%20kidney%20using%20phiC31%20integrase%20and%20electroporation&amp;author=Y.%20Otani&amp;author=S.%20Kawakami&amp;author=H.%20Mukai&amp;author=Y.%20Fuchigami&amp;author=F.%20Yamashita&amp;author=M.%20Hashida&amp;journal=J%20Drug%20Target&amp;volume=23&amp;pages=427-435&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Romano G, Marino IR, Pentimalli F, Adamo V, Giordano A. Insertional mutagenesis and development of malignancies induced by integrating gene delivery systems: implications for the design of safer gene-based interventions in patients. Drug News Perspect. 2009;22:185â96.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1358/dnp.2009.22.4.1367704"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Insertional%20mutagenesis%20and%20development%20of%20malignancies%20induced%20by%20integrating%20gene%20delivery%20systems%3A%20implications%20for%20the%20design%20of%20safer%20gene-based%20interventions%20in%20patients&amp;author=G.%20Romano&amp;author=IR.%20Marino&amp;author=F.%20Pentimalli&amp;author=V.%20Adamo&amp;author=A.%20Giordano&amp;journal=Drug%20News%20Perspect&amp;volume=22&amp;pages=185-196&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Suzuki T, Kazuki Y, Oshimura M, Hara T. A novel system for simultaneous or sequential integration of multiple gene-loading vectors into a defined site of a human artificial chromosome. PLoS One. 2014;9:e110404.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0110404"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20novel%20system%20for%20simultaneous%20or%20sequential%20integration%20of%20multiple%20gene-loading%20vectors%20into%20a%20defined%20site%20of%20a%20human%20artificial%20chromosome&amp;author=T.%20Suzuki&amp;author=Y.%20Kazuki&amp;author=M.%20Oshimura&amp;author=T.%20Hara&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e110404&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Woodard LE, Hillman RT, Keravala A, Lee S, Calos MP. Effect of nuclear localization and hydrodynamic delivery-induced cell division on phiC31 integrase activity. Gene Ther. 2010;17:217â26.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gt.2009.136"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20nuclear%20localization%20and%20hydrodynamic%20delivery-induced%20cell%20division%20on%20phiC31%20integrase%20activity&amp;author=LE.%20Woodard&amp;author=RT.%20Hillman&amp;author=A.%20Keravala&amp;author=S.%20Lee&amp;author=MP.%20Calos&amp;journal=Gene%20Ther&amp;volume=17&amp;pages=217-226&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang D, Park BM, Kang M, Nam H, Kim EJ, Bae C, Lim SK. The systemic effects of sclerostin overexpression using phiC31 integrase in mice. Biochem Biophys Res Commun. 2016;472:471â6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bbrc.2016.01.178"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20systemic%20effects%20of%20sclerostin%20overexpression%20using%20phiC31%20integrase%20in%20mice&amp;author=D.%20Zhang&amp;author=BM.%20Park&amp;author=M.%20Kang&amp;author=H.%20Nam&amp;author=EJ.%20Kim&amp;author=C.%20Bae&amp;author=SK.%20Lim&amp;journal=Biochem%20Biophys%20Res%20Commun&amp;volume=472&amp;pages=471-476&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dormiani K, Mir Mohammad Sadeghi H, Sadeghi-Aliabadi H, Ghaedi K, Forouzanfar M, Baharvand H, Nasr-Esfahani MH. Long-term and efficient expression of human beta-globin gene in a hematopoietic cell line using a new site-specific integrating non-viral system. Gene Ther. 2015;22:663â74.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gt.2015.30"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Long-term%20and%20efficient%20expression%20of%20human%20beta-globin%20gene%20in%20a%20hematopoietic%20cell%20line%20using%20a%20new%20site-specific%20integrating%20non-viral%20system&amp;author=K.%20Dormiani&amp;author=H.%20Mir%20Mohammad%20Sadeghi&amp;author=H.%20Sadeghi-Aliabadi&amp;author=K.%20Ghaedi&amp;author=M.%20Forouzanfar&amp;author=H.%20Baharvand&amp;author=MH.%20Nasr-Esfahani&amp;journal=Gene%20Ther&amp;volume=22&amp;pages=663-674&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Inoue K, Sone T, Oneyama C, Nishiumi F, Kishine H, Sasaki Y, Andoh T, Okada M, Chesnut JD, Imamoto F. A versatile nonviral vector system for tetracycline-dependent one-step conditional induction of transgene expression. Gene Ther. 2009;16:1383â94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gt.2009.105"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20versatile%20nonviral%20vector%20system%20for%20tetracycline-dependent%20one-step%20conditional%20induction%20of%20transgene%20expression&amp;author=K.%20Inoue&amp;author=T.%20Sone&amp;author=C.%20Oneyama&amp;author=F.%20Nishiumi&amp;author=H.%20Kishine&amp;author=Y.%20Sasaki&amp;author=T.%20Andoh&amp;author=M.%20Okada&amp;author=JD.%20Chesnut&amp;author=F.%20Imamoto&amp;journal=Gene%20Ther&amp;volume=16&amp;pages=1383-1394&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stilhano RS, Martin PK, de Melo SM, Samoto VY, Peres GB, da Silva Michelacci YM, da Silva FH, Pereira VG, D'Almeida V, da Cruz AT, Jasiulionis MG, Han SW. alpha- L-iduronidase gene-based therapy using the phiC31 system to treat mucopolysaccharidose type I mice. J Gene Med. 2015;17:1â13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/jgm.2818"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=alpha-%20L-iduronidase%20gene-based%20therapy%20using%20the%20phiC31%20system%20to%20treat%20mucopolysaccharidose%20type%20I%20mice&amp;author=RS.%20Stilhano&amp;author=PK.%20Martin&amp;author=SM.%20Melo&amp;author=VY.%20Samoto&amp;author=GB.%20Peres&amp;author=YM.%20Silva%20Michelacci&amp;author=FH.%20Silva&amp;author=VG.%20Pereira&amp;author=V.%20D%27Almeida&amp;author=AT.%20Cruz&amp;author=MG.%20Jasiulionis&amp;author=SW.%20Han&amp;journal=J%20Gene%20Med.&amp;volume=17&amp;pages=1-13&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li>
<li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cao J, Zhang S. A Bayesian extension of the hypergeometric test for functional enrichment analysis. Biometrics. 2014;70:84â94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/biom.12122"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20Bayesian%20extension%20of%20the%20hypergeometric%20test%20for%20functional%20enrichment%20analysis&amp;author=J.%20Cao&amp;author=S.%20Zhang&amp;journal=Biometrics&amp;volume=70&amp;pages=84-94&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li>
</ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2>
<div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â© The Author(s) 2019</div></div></section>
</div>
                        


                    </article></main><div class="c-content-layout__side u-text-sm">
                    <aside><div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0079-z" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s41065-018-0079-z">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
<li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0079-z" href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0079-z?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true"><use xlink:href="#icon-download"></use></svg><span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
</div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
<div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0079-z #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0079-z #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0079-z #altmetric-container"></div>
                            </ul>
</div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
<li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0079-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0079-z">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0079-z"><use xlink:href="#icon-twitter"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0079-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0079-z">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0079-z"><use xlink:href="#icon-facebook"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0079-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0079-z">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0079-z"><use xlink:href="#icon-linkedin"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0079-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0079-z">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0079-z"><use xlink:href="#icon-weibo"></use></svg></button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0079-z">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0079-z">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0079-z"><use xlink:href="#icon-reddit"></use></svg></button>
                </li>
            </ul>
</div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="/articles/10.1186/s41065-018-0079-z/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/s41065-018-0079-z">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0079-z">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo"></button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
<li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0079-z" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs41065-018-0079-z&amp;volume=156&amp;issue=1&amp;title=Global+mapping+of+binding+sites+for+phic31+integrase+in+transgenic+maden-darby+bovine+kidney+cells+using+ChIP-seq&amp;author_name=Lijuan+Qu&amp;start_page=1&amp;end_page=10">Order reprint</a>

                                </li>
                            </ul>
</div>
                    </aside><div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
<li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a>
</li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a>
</li>
                                
                            </ul>
</div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="/track/article/10.1186/s41065-018-0079-z" alt=""><footer><div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
<ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
<li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
</div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
<li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-twitter-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-facebook-bordered"></use></svg></a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true"><use xlink:href="#icon-weibo-bordered"></use></svg></a>
                        </li>
                    
                </ul>
</div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo"><p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>
</div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script><noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>
</body>
</html>
